
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_47_14"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 28 (31/01/17 at 14:47:14)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_47_14_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_47_14_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 54
No. of loci              : 71
No. of usable loci       : 71 loci with less than  5.00 % missing data
No. of polymorphic sites : 71


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.02649
     2       150         2   0.08957
     3       150         2   0.01333
     4       150         2   0.03946
     5       150         2   0.01333
     6       150         2   0.01333
     7       150         2   0.02649
     8       150         2   0.01333
     9       150         2   0.06488
    10       150         2   0.01333
    11       150         2   0.01333
    12       150         2   0.01333
    13       150         2   0.03946
    14       150         2   0.10166
    15       150         2   0.02649
    16       150         2   0.01333
    17       150         2   0.05226
    18       150         2   0.02649
    19       150         2   0.01333
    20       150         2   0.03946
    21       150         2   0.11356
    22       150         2   0.10166
    23       150         2   0.06488
    24       150         2   0.01333
    25       150         2   0.02649
    26       150         2   0.49253
    27       150         2   0.02649
    28       150         2   0.49253
    29       150         2   0.03946
    30       150         2   0.02649
    31       150         2   0.07732
    32       150         2   0.03946
    33       150         2   0.06488
    34       150         2   0.01333
    35       150         2   0.01333
    36       150         2   0.23266
    37       150         2   0.01333
    38       150         2   0.02649
    39       150         2   0.03946
    40       150         2   0.01333
    41       150         2   0.02649
    42       150         2   0.06488
    43       150         2   0.06488
    44       150         2   0.01333
    45       150         2   0.02649
    46       150         2   0.01333
    47       150         2   0.12528
    48       150         2   0.01333
    49       150         2   0.01333
    50       150         2   0.03946
    51       150         2   0.07732
    52       150         2   0.07732
    53       150         2   0.05226
    54       150         2   0.02649
    55       150         2   0.03946
    56       150         2   0.02649
    57       150         2   0.01333
    58       150         2   0.06488
    59       150         2   0.05226
    60       150         2   0.02649
    61       150         2   0.02649
    62       150         2   0.01333
    63       150         2   0.01333
    64       150         2   0.03946
    65       150         2   0.02649
    66       150         2   0.01333
    67       150         2   0.01333
    68       150         2   0.02649
    69       150         2   0.01333
    70       150         2   0.06488
    71       150         2   0.07732
-------------------------------------------------
  Mean   150.000     2.000   0.05308
  s.d.     0.000     0.000   0.08346
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0446
Gene diversity           :  0.9618 +/-  0.0066

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 54

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 71
Number of usable nucleotide sites            : 71


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.768591 +/-     1.910454

Average gene diversity over loci              :     0.053079 +/-     0.029784

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    23.467286
  S.D. Theta(Hom) :     4.476141

         Theta(k) :    29.820320
95 % confidence interval limits for theta(k) : [  20.964943,  42.104679 ]

         Theta(S) :    12.713730
    S.D. Theta(S) :     3.198456

        Theta(Pi) :     3.768591
   S.D. Theta(Pi) :     2.114688

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     427       0.000
       1    1023       0.000
       2    1527       0.000
       3    2055       0.000
       4    2191       0.000
       5    1961       0.000
       6    1104       0.000
       7     517       0.000
       8     290       0.000
       9      60       0.000
      10      20       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.769
Mismatch observed variance          :        3.713
Tau                                 :        4.062
Theta0                              :        0.009
Theta1                              :       51.953

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00114242
          P(Sim. Ssd >= Obs. Ssd): 0.60000000
    Harpending's Raggedness index: 0.01717527
        P(Sim. Rag. >= Obs. Rag.): 0.60000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        427.0        314.7        383.9
          1.0       1023.0        709.1        907.7
          2.0       1527.0       1354.7       1619.6
          3.0       2055.0       2029.5       2100.7
          4.0       2191.0       2187.7       2095.7
          5.0       1961.0       1890.8       1688.5
          6.0       1104.0       1272.4       1138.5
          7.0        517.0        734.7        659.5
          8.0        290.0        380.6        334.7
          9.0         60.0        187.2        151.1
         10.0         20.0         78.2         61.4
         11.0          0.0         22.8         22.7
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.062           3.043           5.262           4.070
                   Theta0:           0.009           0.000           0.656           0.090               
                   Theta1:          51.953          28.418       99999.000       60019.513
Mean No. of pairwise diff.           3.769           3.218           4.912           4.153
           No. pol. sites:              71              71             108          89.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        427.0        174.0        640.0        383.9
          1.0       1023.0        369.0       1530.0        907.7
          2.0       1527.0        895.0       2046.0       1619.6
          3.0       2055.0       1335.0       2685.0       2100.7
          4.0       2191.0       1803.0       2627.0       2095.7
          5.0       1961.0       1386.0       2447.0       1688.5
          6.0       1104.0        786.0       1723.0       1138.5
          7.0        517.0        366.0       1349.0        659.5
          8.0        290.0        125.0        828.0        334.7
          9.0         60.0         28.0        617.0        151.1
         10.0         20.0          6.0        419.0         61.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.062           3.043           5.262           4.070
                   Theta0:           0.009           0.000           0.656           0.090               
                   Theta1:          51.953          28.418       99999.000       60019.513
Mean No. of pairwise diff.           3.769           3.218           4.912           4.153
           No. pol. sites:              71              71             108          89.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        427.0        174.0        640.0        383.9
          1.0       1023.0        369.0       1530.0        907.7
          2.0       1527.0        895.0       2046.0       1619.6
          3.0       2055.0       1335.0       2685.0       2100.7
          4.0       2191.0       1803.0       2627.0       2095.7
          5.0       1961.0       1386.0       2447.0       1688.5
          6.0       1104.0        786.0       1723.0       1138.5
          7.0        517.0        366.0       1349.0        659.5
          8.0        290.0        125.0        828.0        334.7
          9.0         60.0         28.0        617.0        151.1
         10.0         20.0          6.0        419.0         61.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.062           3.043           5.262           4.070
                   Theta0:           0.009           0.000           0.656           0.090               
                   Theta1:          51.953          28.418       99999.000       60019.513
Mean No. of pairwise diff.           3.769           3.218           4.912           4.153
           No. pol. sites:              71              71             108          89.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        427.0        174.0        640.0        383.9
          1.0       1023.0        369.0       1530.0        907.7
          2.0       1527.0        895.0       2046.0       1619.6
          3.0       2055.0       1335.0       2685.0       2100.7
          4.0       2191.0       1803.0       2627.0       2095.7
          5.0       1961.0       1386.0       2447.0       1688.5
          6.0       1104.0        786.0       1723.0       1138.5
          7.0        517.0        366.0       1349.0        659.5
          8.0        290.0        125.0        828.0        334.7
          9.0         60.0         28.0        617.0        151.1
         10.0         20.0          6.0        419.0         61.4
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_47_14_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            71      71.000       0.000
  No. of usable loci            71      71.000       0.000
  No. of polym. loci            71      71.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_47_14_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.02649     0.02649     0.00000     0.02649
     2       0.08957     0.08957     0.00000     0.08957
     3       0.01333     0.01333     0.00000     0.01333
     4       0.03946     0.03946     0.00000     0.03946
     5       0.01333     0.01333     0.00000     0.01333
     6       0.01333     0.01333     0.00000     0.01333
     7       0.02649     0.02649     0.00000     0.02649
     8       0.01333     0.01333     0.00000     0.01333
     9       0.06488     0.06488     0.00000     0.06488
    10       0.01333     0.01333     0.00000     0.01333
    11       0.01333     0.01333     0.00000     0.01333
    12       0.01333     0.01333     0.00000     0.01333
    13       0.03946     0.03946     0.00000     0.03946
    14       0.10166     0.10166     0.00000     0.10166
    15       0.02649     0.02649     0.00000     0.02649
    16       0.01333     0.01333     0.00000     0.01333
    17       0.05226     0.05226     0.00000     0.05226
    18       0.02649     0.02649     0.00000     0.02649
    19       0.01333     0.01333     0.00000     0.01333
    20       0.03946     0.03946     0.00000     0.03946
    21       0.11356     0.11356     0.00000     0.11356
    22       0.10166     0.10166     0.00000     0.10166
    23       0.06488     0.06488     0.00000     0.06488
    24       0.01333     0.01333     0.00000     0.01333
    25       0.02649     0.02649     0.00000     0.02649
    26       0.49253     0.49253     0.00000     0.49253
    27       0.02649     0.02649     0.00000     0.02649
    28       0.49253     0.49253     0.00000     0.49253
    29       0.03946     0.03946     0.00000     0.03946
    30       0.02649     0.02649     0.00000     0.02649
    31       0.07732     0.07732     0.00000     0.07732
    32       0.03946     0.03946     0.00000     0.03946
    33       0.06488     0.06488     0.00000     0.06488
    34       0.01333     0.01333     0.00000     0.01333
    35       0.01333     0.01333     0.00000     0.01333
    36       0.23266     0.23266     0.00000     0.23266
    37       0.01333     0.01333     0.00000     0.01333
    38       0.02649     0.02649     0.00000     0.02649
    39       0.03946     0.03946     0.00000     0.03946
    40       0.01333     0.01333     0.00000     0.01333
    41       0.02649     0.02649     0.00000     0.02649
    42       0.06488     0.06488     0.00000     0.06488
    43       0.06488     0.06488     0.00000     0.06488
    44       0.01333     0.01333     0.00000     0.01333
    45       0.02649     0.02649     0.00000     0.02649
    46       0.01333     0.01333     0.00000     0.01333
    47       0.12528     0.12528     0.00000     0.12528
    48       0.01333     0.01333     0.00000     0.01333
    49       0.01333     0.01333     0.00000     0.01333
    50       0.03946     0.03946     0.00000     0.03946
    51       0.07732     0.07732     0.00000     0.07732
    52       0.07732     0.07732     0.00000     0.07732
    53       0.05226     0.05226     0.00000     0.05226
    54       0.02649     0.02649     0.00000     0.02649
    55       0.03946     0.03946     0.00000     0.03946
    56       0.02649     0.02649     0.00000     0.02649
    57       0.01333     0.01333     0.00000     0.01333
    58       0.06488     0.06488     0.00000     0.06488
    59       0.05226     0.05226     0.00000     0.05226
    60       0.02649     0.02649     0.00000     0.02649
    61       0.02649     0.02649     0.00000     0.02649
    62       0.01333     0.01333     0.00000     0.01333
    63       0.01333     0.01333     0.00000     0.01333
    64       0.03946     0.03946     0.00000     0.03946
    65       0.02649     0.02649     0.00000     0.02649
    66       0.01333     0.01333     0.00000     0.01333
    67       0.01333     0.01333     0.00000     0.01333
    68       0.02649     0.02649     0.00000     0.02649
    69       0.01333     0.01333     0.00000     0.01333
    70       0.06488     0.06488     0.00000     0.06488
    71       0.07732     0.07732     0.00000     0.07732
--------------------------------------------------------
  Mean       0.05308     0.05308     0.00000     0.05308
  s.d.       0.08346     0.08346     0.00000     0.08346
--------------------------------------------------------
<A NAME="31_01_17at14_47_14_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_47_14_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      29.82032    29.82032     0.00000
            Theta_k_lower      20.96494    20.96494     0.00000
            Theta_k_upper      42.10468    42.10468     0.00000
                  Theta_H      23.46729    23.46729     0.00000
             s.d. Theta_H       4.47614     4.47614     0.00000
                  Theta_S      12.71373    12.71373     0.00000
             s.d. Theta_S       3.19846     3.19846     0.00000
                 Theta_pi       3.76859     3.76859     0.00000
            s.d. Theta_pi       2.11469     2.11469     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_47_14_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.06250     4.06250     0.00000
              Tau qt 2.5%       3.04297     3.04297     0.00000
                Tau qt 5%       3.04297     3.04297     0.00000
               Tau qt 95%       4.71289     4.71289     0.00000
             Tau qt 97.5%       4.71289     4.71289     0.00000
                   Theta0       0.00879     0.00879     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.21270     0.21270     0.00000
          Theta0 qt 97.5%       0.21270     0.21270     0.00000
                   Theta1      51.95312    51.95312     0.00000
           Theta1 qt 2.5%      28.41797    28.41797     0.00000
             Theta1 qt 5%      28.41797    28.41797     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00114     0.00114     0.00000
      Model (SSD) p-value       0.60000     0.60000     0.00000
         Raggedness index       0.01718     0.01718     0.00000
       Raggedness p-value       0.60000     0.60000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 28 (31/01/17 at 14:47:14))
Total computing time for this run : 0h 0m 0s 134 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

