
<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_36_07"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 161 (31/01/17 at 16:36:07)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_36_07_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_36_07_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 68
No. of loci              : 21
No. of usable loci       : 21 loci with less than  5.00 % missing data
No. of polymorphic sites : 21


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.41172
     3       150         2   0.21262
     4       150         2   0.07732
     5       150         2   0.29718
     6       150         2   0.43812
     7       150         2   0.01333
     8       150         2   0.28850
     9       150         2   0.30568
    10       150         2   0.06488
    11       150         2   0.01333
    12       150         2   0.29718
    13       150         2   0.31400
    14       150         2   0.07732
    15       150         2   0.01333
    16       150         2   0.27964
    17       150         2   0.44743
    18       150         2   0.17038
    19       150         2   0.05226
    20       150         2   0.30568
    21       150         2   0.27060
-------------------------------------------------
  Mean   150.000     2.000   0.20780
  s.d.     0.000     0.000   0.14924
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0242
Gene diversity           :  0.9824 +/-  0.0031

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 68

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 21
Number of usable nucleotide sites            : 21


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.363848 +/-     2.168837

Average gene diversity over loci              :     0.207802 +/-     0.114319

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    53.865313
  S.D. Theta(Hom) :     9.826868

         Theta(k) :    47.395630
95 % confidence interval limits for theta(k) : [  33.909879,  66.029618 ]

         Theta(S) :     3.760399
    S.D. Theta(S) :     1.159953

        Theta(Pi) :     4.363848
   S.D. Theta(Pi) :     2.400693

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     197       0.000
       1     620       0.000
       2    1219       0.000
       3    1687       0.000
       4    2199       0.000
       5    2116       0.000
       6    1686       0.000
       7     783       0.000
       8     438       0.000
       9     130       0.000
      10     100       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.364
Mismatch observed variance          :        3.900
Tau                                 :        4.564
Theta0                              :        0.000
Theta1                              :     1367.500

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00158608
          P(Sim. Ssd >= Obs. Ssd): 0.20000000
    Harpending's Raggedness index: 0.01802429
        P(Sim. Rag. >= Obs. Rag.): 0.30000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        197.0        103.3        124.1
          1.0        620.0        459.3        537.2
          2.0       1219.0       1124.2       1215.2
          3.0       1687.0       1806.9       1844.1
          4.0       2199.0       2160.2       2102.0
          5.0       2116.0       2010.5       1917.8
          6.0       1686.0       1525.4       1458.5
          7.0        783.0        967.6        950.9
          8.0        438.0        540.9        542.4
          9.0        130.0        262.0        275.1
         10.0        100.0        121.2        125.5
         11.0          0.0         55.4         52.1
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.564           3.770           4.914           4.549
                   Theta0:           0.000           0.000           0.603           0.074               
                   Theta1:        1367.500         108.125       99999.000       70352.113
Mean No. of pairwise diff.           4.364           4.237           5.011           4.630
           No. pol. sites:              21             211             230         217.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        197.0         34.0        260.0        124.1
          1.0        620.0        284.0        719.0        537.2
          2.0       1219.0        901.0       1485.0       1215.2
          3.0       1687.0       1643.0       2122.0       1844.1
          4.0       2199.0       1991.0       2410.0       2102.0
          5.0       2116.0       1746.0       2246.0       1917.8
          6.0       1686.0       1228.0       1658.0       1458.5
          7.0        783.0        827.0       1135.0        950.9
          8.0        438.0        391.0        675.0        542.4
          9.0        130.0        141.0        338.0        275.1
         10.0        100.0         43.0        197.0        125.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.564           3.770           4.914           4.549
                   Theta0:           0.000           0.000           0.603           0.074               
                   Theta1:        1367.500         108.125       99999.000       70352.113
Mean No. of pairwise diff.           4.364           4.237           5.011           4.630
           No. pol. sites:              21             211             230         217.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        197.0         34.0        260.0        124.1
          1.0        620.0        284.0        719.0        537.2
          2.0       1219.0        901.0       1485.0       1215.2
          3.0       1687.0       1643.0       2122.0       1844.1
          4.0       2199.0       1991.0       2410.0       2102.0
          5.0       2116.0       1746.0       2246.0       1917.8
          6.0       1686.0       1228.0       1658.0       1458.5
          7.0        783.0        827.0       1135.0        950.9
          8.0        438.0        391.0        675.0        542.4
          9.0        130.0        141.0        338.0        275.1
         10.0        100.0         43.0        197.0        125.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.564           3.770           4.914           4.549
                   Theta0:           0.000           0.000           0.603           0.074               
                   Theta1:        1367.500         108.125       99999.000       70352.113
Mean No. of pairwise diff.           4.364           4.237           5.011           4.630
           No. pol. sites:              21             211             230         217.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        197.0         34.0        260.0        124.1
          1.0        620.0        284.0        719.0        537.2
          2.0       1219.0        901.0       1485.0       1215.2
          3.0       1687.0       1643.0       2122.0       1844.1
          4.0       2199.0       1991.0       2410.0       2102.0
          5.0       2116.0       1746.0       2246.0       1917.8
          6.0       1686.0       1228.0       1658.0       1458.5
          7.0        783.0        827.0       1135.0        950.9
          8.0        438.0        391.0        675.0        542.4
          9.0        130.0        141.0        338.0        275.1
         10.0        100.0         43.0        197.0        125.5
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_36_07_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            21      21.000       0.000
  No. of usable loci            21      21.000       0.000
  No. of polym. loci            21      21.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_36_07_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.41172     0.41172     0.00000     0.41172
     3       0.21262     0.21262     0.00000     0.21262
     4       0.07732     0.07732     0.00000     0.07732
     5       0.29718     0.29718     0.00000     0.29718
     6       0.43812     0.43812     0.00000     0.43812
     7       0.01333     0.01333     0.00000     0.01333
     8       0.28850     0.28850     0.00000     0.28850
     9       0.30568     0.30568     0.00000     0.30568
    10       0.06488     0.06488     0.00000     0.06488
    11       0.01333     0.01333     0.00000     0.01333
    12       0.29718     0.29718     0.00000     0.29718
    13       0.31400     0.31400     0.00000     0.31400
    14       0.07732     0.07732     0.00000     0.07732
    15       0.01333     0.01333     0.00000     0.01333
    16       0.27964     0.27964     0.00000     0.27964
    17       0.44743     0.44743     0.00000     0.44743
    18       0.17038     0.17038     0.00000     0.17038
    19       0.05226     0.05226     0.00000     0.05226
    20       0.30568     0.30568     0.00000     0.30568
    21       0.27060     0.27060     0.00000     0.27060
--------------------------------------------------------
  Mean       0.20780     0.20780     0.00000     0.20780
  s.d.       0.14924     0.14924     0.00000     0.14924
--------------------------------------------------------
<A NAME="31_01_17at16_36_07_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_36_07_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      47.39563    47.39563     0.00000
            Theta_k_lower      33.90988    33.90988     0.00000
            Theta_k_upper      66.02962    66.02962     0.00000
                  Theta_H      53.86531    53.86531     0.00000
             s.d. Theta_H       9.82687     9.82687     0.00000
                  Theta_S       3.76040     3.76040     0.00000
             s.d. Theta_S       1.15995     1.15995     0.00000
                 Theta_pi       4.36385     4.36385     0.00000
            s.d. Theta_pi       2.40069     2.40069     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_36_07_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.56445     4.56445     0.00000
              Tau qt 2.5%       3.76953     3.76953     0.00000
                Tau qt 5%       3.76953     3.76953     0.00000
               Tau qt 95%       4.86523     4.86523     0.00000
             Tau qt 97.5%       4.86523     4.86523     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.07910     0.07910     0.00000
          Theta0 qt 97.5%       0.07910     0.07910     0.00000
                   Theta1    1367.50000  1367.50000     0.00000
           Theta1 qt 2.5%     108.12500   108.12500     0.00000
             Theta1 qt 5%     108.12500   108.12500     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00159     0.00159     0.00000
      Model (SSD) p-value       0.20000     0.20000     0.00000
         Raggedness index       0.01802     0.01802     0.00000
       Raggedness p-value       0.30000     0.30000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 161 (31/01/17 at 16:36:07))
Total computing time for this run : 0h 0m 0s 154 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

