
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_29_24"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 58 (31/01/17 at 16:29:24)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_29_24_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_29_24_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 58
No. of loci              : 75
No. of usable loci       : 75 loci with less than  5.00 % missing data
No. of polymorphic sites : 75


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.01333
     3       150         2   0.01333
     4       150         2   0.02649
     5       150         2   0.02649
     6       150         2   0.01333
     7       150         2   0.02649
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.02649
    11       150         2   0.02649
    12       150         2   0.02649
    13       150         2   0.01333
    14       150         2   0.01333
    15       150         2   0.03946
    16       150         2   0.01333
    17       150         2   0.05226
    18       150         2   0.05226
    19       150         2   0.01333
    20       150         2   0.03946
    21       150         2   0.02649
    22       150         2   0.01333
    23       150         2   0.01333
    24       150         2   0.03946
    25       150         2   0.01333
    26       150         2   0.01333
    27       150         2   0.01333
    28       150         2   0.01333
    29       150         2   0.01333
    30       150         2   0.01333
    31       150         2   0.01333
    32       150         2   0.02649
    33       150         2   0.01333
    34       150         2   0.01333
    35       150         2   0.02649
    36       150         2   0.05226
    37       150         2   0.01333
    38       150         2   0.01333
    39       150         2   0.01333
    40       150         2   0.02649
    41       150         2   0.01333
    42       150         2   0.01333
    43       150         2   0.02649
    44       150         2   0.05226
    45       150         2   0.02649
    46       150         2   0.03946
    47       150         2   0.01333
    48       150         2   0.01333
    49       150         2   0.01333
    50       150         2   0.01333
    51       150         2   0.02649
    52       150         2   0.03946
    53       150         2   0.01333
    54       150         2   0.03946
    55       150         2   0.01333
    56       150         2   0.01333
    57       150         2   0.05226
    58       150         2   0.01333
    59       150         2   0.01333
    60       150         2   0.01333
    61       150         2   0.03946
    62       150         2   0.01333
    63       150         2   0.01333
    64       150         2   0.01333
    65       150         2   0.01333
    66       150         2   0.01333
    67       150         2   0.01333
    68       150         2   0.01333
    69       150         2   0.05226
    70       150         2   0.01333
    71       150         2   0.01333
    72       150         2   0.01333
    73       150         2   0.02649
    74       150         2   0.03946
    75       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.02169
  s.d.     0.000     0.000   0.01259
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.2123
Gene diversity           :  0.7930 +/-  0.0359

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 58

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 75
Number of usable nucleotide sites            : 75


Distance method : Pairwise difference
Mean number of pairwise differences           :     1.626756 +/-     0.966947

Average gene diversity over loci              :     0.021690 +/-     0.014271

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     3.026765
  S.D. Theta(Hom) :     0.715868

         Theta(k) :    34.205353
95 % confidence interval limits for theta(k) : [  24.201985,  48.048972 ]

         Theta(S) :    13.429996
    S.D. Theta(S) :     3.359547

        Theta(Pi) :     1.626756
   S.D. Theta(Pi) :     1.070317

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    2313    2248.987
       1    3480    3559.501
       2    2819    2857.001
       3    1573    1554.712
       4     683     647.403
       5     240     220.911
       6      59      64.644
       7       8      16.776
       8       0       3.965
       9       0       0.872
      10       0       0.182
      11       0       0.037
      12       0       0.007
      13       0       0.001
      14       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        1.627
Mismatch observed variance          :        1.681
Tau                                 :        1.695
Theta0                              :        0.000
Theta1                              :       20.572

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00001096
          P(Sim. Ssd >= Obs. Ssd): 1.00000000
    Harpending's Raggedness index: 0.03503427
        P(Sim. Rag. >= Obs. Rag.): 0.70000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       2313.0       2744.9       2319.3
          1.0       3480.0       3681.6       3464.2
          2.0       2819.0       2689.8       2838.4
          3.0       1573.0       1391.5       1581.1
          4.0        683.0        503.0        665.0
          5.0        240.0        136.4        224.4
          6.0         59.0         22.4         63.2
          7.0          8.0          4.7         15.3
          8.0          0.0          0.7          3.2
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.695           0.924           2.559           1.592
                   Theta0:           0.000           0.000           0.181           0.027               
                   Theta1:          20.572           3.296       99999.000       60002.065
Mean No. of pairwise diff.           1.627           0.989           1.800           1.441
           No. pol. sites:              75              29              50          39.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2313.0       1891.0       4261.0       2319.3
          1.0       3480.0       2742.0       4411.0       3464.2
          2.0       2819.0       2095.0       3386.0       2838.4
          3.0       1573.0        768.0       2019.0       1581.1
          4.0        683.0        152.0       1024.0        665.0
          5.0        240.0         14.0        376.0        224.4
          6.0         59.0          0.0        126.0         63.2
          7.0          8.0          0.0         44.0         15.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.695           0.924           2.559           1.592
                   Theta0:           0.000           0.000           0.181           0.027               
                   Theta1:          20.572           3.296       99999.000       60002.065
Mean No. of pairwise diff.           1.627           0.989           1.800           1.441
           No. pol. sites:              75              29              50          39.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2313.0       1891.0       4261.0       2319.3
          1.0       3480.0       2742.0       4411.0       3464.2
          2.0       2819.0       2095.0       3386.0       2838.4
          3.0       1573.0        768.0       2019.0       1581.1
          4.0        683.0        152.0       1024.0        665.0
          5.0        240.0         14.0        376.0        224.4
          6.0         59.0          0.0        126.0         63.2
          7.0          8.0          0.0         44.0         15.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.695           0.924           2.559           1.592
                   Theta0:           0.000           0.000           0.181           0.027               
                   Theta1:          20.572           3.296       99999.000       60002.065
Mean No. of pairwise diff.           1.627           0.989           1.800           1.441
           No. pol. sites:              75              29              50          39.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2313.0       1891.0       4261.0       2319.3
          1.0       3480.0       2742.0       4411.0       3464.2
          2.0       2819.0       2095.0       3386.0       2838.4
          3.0       1573.0        768.0       2019.0       1581.1
          4.0        683.0        152.0       1024.0        665.0
          5.0        240.0         14.0        376.0        224.4
          6.0         59.0          0.0        126.0         63.2
          7.0          8.0          0.0         44.0         15.3
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_29_24_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            75      75.000       0.000
  No. of usable loci            75      75.000       0.000
  No. of polym. loci            75      75.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_29_24_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.01333     0.01333     0.00000     0.01333
     3       0.01333     0.01333     0.00000     0.01333
     4       0.02649     0.02649     0.00000     0.02649
     5       0.02649     0.02649     0.00000     0.02649
     6       0.01333     0.01333     0.00000     0.01333
     7       0.02649     0.02649     0.00000     0.02649
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.02649     0.02649     0.00000     0.02649
    11       0.02649     0.02649     0.00000     0.02649
    12       0.02649     0.02649     0.00000     0.02649
    13       0.01333     0.01333     0.00000     0.01333
    14       0.01333     0.01333     0.00000     0.01333
    15       0.03946     0.03946     0.00000     0.03946
    16       0.01333     0.01333     0.00000     0.01333
    17       0.05226     0.05226     0.00000     0.05226
    18       0.05226     0.05226     0.00000     0.05226
    19       0.01333     0.01333     0.00000     0.01333
    20       0.03946     0.03946     0.00000     0.03946
    21       0.02649     0.02649     0.00000     0.02649
    22       0.01333     0.01333     0.00000     0.01333
    23       0.01333     0.01333     0.00000     0.01333
    24       0.03946     0.03946     0.00000     0.03946
    25       0.01333     0.01333     0.00000     0.01333
    26       0.01333     0.01333     0.00000     0.01333
    27       0.01333     0.01333     0.00000     0.01333
    28       0.01333     0.01333     0.00000     0.01333
    29       0.01333     0.01333     0.00000     0.01333
    30       0.01333     0.01333     0.00000     0.01333
    31       0.01333     0.01333     0.00000     0.01333
    32       0.02649     0.02649     0.00000     0.02649
    33       0.01333     0.01333     0.00000     0.01333
    34       0.01333     0.01333     0.00000     0.01333
    35       0.02649     0.02649     0.00000     0.02649
    36       0.05226     0.05226     0.00000     0.05226
    37       0.01333     0.01333     0.00000     0.01333
    38       0.01333     0.01333     0.00000     0.01333
    39       0.01333     0.01333     0.00000     0.01333
    40       0.02649     0.02649     0.00000     0.02649
    41       0.01333     0.01333     0.00000     0.01333
    42       0.01333     0.01333     0.00000     0.01333
    43       0.02649     0.02649     0.00000     0.02649
    44       0.05226     0.05226     0.00000     0.05226
    45       0.02649     0.02649     0.00000     0.02649
    46       0.03946     0.03946     0.00000     0.03946
    47       0.01333     0.01333     0.00000     0.01333
    48       0.01333     0.01333     0.00000     0.01333
    49       0.01333     0.01333     0.00000     0.01333
    50       0.01333     0.01333     0.00000     0.01333
    51       0.02649     0.02649     0.00000     0.02649
    52       0.03946     0.03946     0.00000     0.03946
    53       0.01333     0.01333     0.00000     0.01333
    54       0.03946     0.03946     0.00000     0.03946
    55       0.01333     0.01333     0.00000     0.01333
    56       0.01333     0.01333     0.00000     0.01333
    57       0.05226     0.05226     0.00000     0.05226
    58       0.01333     0.01333     0.00000     0.01333
    59       0.01333     0.01333     0.00000     0.01333
    60       0.01333     0.01333     0.00000     0.01333
    61       0.03946     0.03946     0.00000     0.03946
    62       0.01333     0.01333     0.00000     0.01333
    63       0.01333     0.01333     0.00000     0.01333
    64       0.01333     0.01333     0.00000     0.01333
    65       0.01333     0.01333     0.00000     0.01333
    66       0.01333     0.01333     0.00000     0.01333
    67       0.01333     0.01333     0.00000     0.01333
    68       0.01333     0.01333     0.00000     0.01333
    69       0.05226     0.05226     0.00000     0.05226
    70       0.01333     0.01333     0.00000     0.01333
    71       0.01333     0.01333     0.00000     0.01333
    72       0.01333     0.01333     0.00000     0.01333
    73       0.02649     0.02649     0.00000     0.02649
    74       0.03946     0.03946     0.00000     0.03946
    75       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.02169     0.02169     0.00000     0.02169
  s.d.       0.01259     0.01259     0.00000     0.01259
--------------------------------------------------------
<A NAME="31_01_17at16_29_24_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
    72             2       2.000       0.000           2
    73             2       2.000       0.000           2
    74             2       2.000       0.000           2
    75             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_29_24_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      34.20535    34.20535     0.00000
            Theta_k_lower      24.20198    24.20198     0.00000
            Theta_k_upper      48.04897    48.04897     0.00000
                  Theta_H       3.02677     3.02677     0.00000
             s.d. Theta_H       0.71587     0.71587     0.00000
                  Theta_S      13.43000    13.43000     0.00000
             s.d. Theta_S       3.35955     3.35955     0.00000
                 Theta_pi       1.62676     1.62676     0.00000
            s.d. Theta_pi       1.07032     1.07032     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_29_24_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.69531     1.69531     0.00000
              Tau qt 2.5%       0.92383     0.92383     0.00000
                Tau qt 5%       0.92383     0.92383     0.00000
               Tau qt 95%       2.10547     2.10547     0.00000
             Tau qt 97.5%       2.10547     2.10547     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.06152     0.06152     0.00000
          Theta0 qt 97.5%       0.06152     0.06152     0.00000
                   Theta1      20.57188    20.57188     0.00000
           Theta1 qt 2.5%       3.29648     3.29648     0.00000
             Theta1 qt 5%       3.29648     3.29648     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00001     0.00001     0.00000
      Model (SSD) p-value       1.00000     1.00000     0.00000
         Raggedness index       0.03503     0.03503     0.00000
       Raggedness p-value       0.70000     0.70000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 58 (31/01/17 at 16:29:24))
Total computing time for this run : 0h 0m 0s 106 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

