
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_51_22"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 25 (31/01/17 at 14:51:22)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_51_22_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_51_22_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 53
No. of loci              : 22
No. of usable loci       : 22 loci with less than  5.00 % missing data
No. of polymorphic sites : 22


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.06488
     2       150         2   0.44743
     3       150         2   0.29718
     4       150         2   0.05226
     5       150         2   0.07732
     6       150         2   0.41736
     7       150         2   0.40591
     8       150         2   0.10166
     9       150         2   0.33790
    10       150         2   0.35293
    11       150         2   0.14819
    12       150         2   0.01333
    13       150         2   0.06488
    14       150         2   0.22273
    15       150         2   0.01333
    16       150         2   0.14819
    17       150         2   0.31400
    18       150         2   0.07732
    19       150         2   0.07732
    20       150         2   0.30568
    21       150         2   0.28850
    22       150         2   0.13682
-------------------------------------------------
  Mean   150.000     2.000   0.19841
  s.d.     0.000     0.000   0.14266
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0357
Gene diversity           :  0.9707 +/-  0.0048

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 53

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 22
Number of usable nucleotide sites            : 22


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.365101 +/-     2.169380

Average gene diversity over loci              :     0.198414 +/-     0.109150

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    31.403367
  S.D. Theta(Hom) :     5.523366

         Theta(k) :    28.791891
95 % confidence interval limits for theta(k) : [  20.205559,  40.712272 ]

         Theta(S) :     3.939466
    S.D. Theta(S) :     1.201985

        Theta(Pi) :     4.365101
   S.D. Theta(Pi) :     2.401294

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     327       0.000
       1     517       0.000
       2    1239       0.000
       3    1879       0.000
       4    1857       0.000
       5    2103       0.000
       6    1597       0.000
       7     933       0.000
       8     477       0.000
       9     184       0.000
      10      62       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.365
Mismatch observed variance          :        4.169
Tau                                 :        4.629
Theta0                              :        0.011
Theta1                              :       98.906

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00123734
          P(Sim. Ssd >= Obs. Ssd): 0.40000000
    Harpending's Raggedness index: 0.01631501
        P(Sim. Rag. >= Obs. Rag.): 0.50000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        327.0        179.5        213.9
          1.0        517.0        493.9        583.0
          2.0       1239.0       1094.4       1201.7
          3.0       1879.0       1764.5       1789.6
          4.0       1857.0       2105.7       2043.0
          5.0       2103.0       1946.5       1880.6
          6.0       1597.0       1533.3       1447.5
          7.0        933.0        972.4        956.6
          8.0        477.0        611.8        553.6
          9.0        184.0        281.6        285.0
         10.0         62.0        114.4        132.1
         11.0          0.0         51.5         55.7
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.629           3.697           5.480           4.534
                   Theta0:           0.011           0.000           1.758           0.242               
                   Theta1:          98.906          36.133       99999.000       50047.777
Mean No. of pairwise diff.           4.365           4.130           5.207           4.618
           No. pol. sites:              22             120             140         130.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        327.0         82.0        302.0        213.9
          1.0        517.0        166.0        800.0        583.0
          2.0       1239.0        644.0       1539.0       1201.7
          3.0       1879.0       1278.0       2291.0       1789.6
          4.0       1857.0       1484.0       2518.0       2043.0
          5.0       2103.0       1506.0       2272.0       1880.6
          6.0       1597.0       1213.0       2160.0       1447.5
          7.0        933.0        667.0       1421.0        956.6
          8.0        477.0        374.0        922.0        553.6
          9.0        184.0        112.0        511.0        285.0
         10.0         62.0         27.0        287.0        132.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.629           3.697           5.480           4.534
                   Theta0:           0.011           0.000           1.758           0.242               
                   Theta1:          98.906          36.133       99999.000       50047.777
Mean No. of pairwise diff.           4.365           4.130           5.207           4.618
           No. pol. sites:              22             120             140         130.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        327.0         82.0        302.0        213.9
          1.0        517.0        166.0        800.0        583.0
          2.0       1239.0        644.0       1539.0       1201.7
          3.0       1879.0       1278.0       2291.0       1789.6
          4.0       1857.0       1484.0       2518.0       2043.0
          5.0       2103.0       1506.0       2272.0       1880.6
          6.0       1597.0       1213.0       2160.0       1447.5
          7.0        933.0        667.0       1421.0        956.6
          8.0        477.0        374.0        922.0        553.6
          9.0        184.0        112.0        511.0        285.0
         10.0         62.0         27.0        287.0        132.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.629           3.697           5.480           4.534
                   Theta0:           0.011           0.000           1.758           0.242               
                   Theta1:          98.906          36.133       99999.000       50047.777
Mean No. of pairwise diff.           4.365           4.130           5.207           4.618
           No. pol. sites:              22             120             140         130.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        327.0         82.0        302.0        213.9
          1.0        517.0        166.0        800.0        583.0
          2.0       1239.0        644.0       1539.0       1201.7
          3.0       1879.0       1278.0       2291.0       1789.6
          4.0       1857.0       1484.0       2518.0       2043.0
          5.0       2103.0       1506.0       2272.0       1880.6
          6.0       1597.0       1213.0       2160.0       1447.5
          7.0        933.0        667.0       1421.0        956.6
          8.0        477.0        374.0        922.0        553.6
          9.0        184.0        112.0        511.0        285.0
         10.0         62.0         27.0        287.0        132.1
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_51_22_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            22      22.000       0.000
  No. of usable loci            22      22.000       0.000
  No. of polym. loci            22      22.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_51_22_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.06488     0.06488     0.00000     0.06488
     2       0.44743     0.44743     0.00000     0.44743
     3       0.29718     0.29718     0.00000     0.29718
     4       0.05226     0.05226     0.00000     0.05226
     5       0.07732     0.07732     0.00000     0.07732
     6       0.41736     0.41736     0.00000     0.41736
     7       0.40591     0.40591     0.00000     0.40591
     8       0.10166     0.10166     0.00000     0.10166
     9       0.33790     0.33790     0.00000     0.33790
    10       0.35293     0.35293     0.00000     0.35293
    11       0.14819     0.14819     0.00000     0.14819
    12       0.01333     0.01333     0.00000     0.01333
    13       0.06488     0.06488     0.00000     0.06488
    14       0.22273     0.22273     0.00000     0.22273
    15       0.01333     0.01333     0.00000     0.01333
    16       0.14819     0.14819     0.00000     0.14819
    17       0.31400     0.31400     0.00000     0.31400
    18       0.07732     0.07732     0.00000     0.07732
    19       0.07732     0.07732     0.00000     0.07732
    20       0.30568     0.30568     0.00000     0.30568
    21       0.28850     0.28850     0.00000     0.28850
    22       0.13682     0.13682     0.00000     0.13682
--------------------------------------------------------
  Mean       0.19841     0.19841     0.00000     0.19841
  s.d.       0.14266     0.14266     0.00000     0.14266
--------------------------------------------------------
<A NAME="31_01_17at14_51_22_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_51_22_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      28.79189    28.79189     0.00000
            Theta_k_lower      20.20556    20.20556     0.00000
            Theta_k_upper      40.71227    40.71227     0.00000
                  Theta_H      31.40337    31.40337     0.00000
             s.d. Theta_H       5.52337     5.52337     0.00000
                  Theta_S       3.93947     3.93947     0.00000
             s.d. Theta_S       1.20198     1.20198     0.00000
                 Theta_pi       4.36510     4.36510     0.00000
            s.d. Theta_pi       2.40129     2.40129     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_51_22_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.62891     4.62891     0.00000
              Tau qt 2.5%       3.69727     3.69727     0.00000
                Tau qt 5%       3.69727     3.69727     0.00000
               Tau qt 95%       5.12109     5.12109     0.00000
             Tau qt 97.5%       5.12109     5.12109     0.00000
                   Theta0       0.01055     0.01055     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.51504     0.51504     0.00000
          Theta0 qt 97.5%       0.51504     0.51504     0.00000
                   Theta1      98.90625    98.90625     0.00000
           Theta1 qt 2.5%      36.13281    36.13281     0.00000
             Theta1 qt 5%      36.13281    36.13281     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00124     0.00124     0.00000
      Model (SSD) p-value       0.40000     0.40000     0.00000
         Raggedness index       0.01632     0.01632     0.00000
       Raggedness p-value       0.50000     0.50000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 25 (31/01/17 at 14:51:23))
Total computing time for this run : 0h 0m 0s 127 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

