
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_30_28"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 73 (31/01/17 at 16:30:28)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_30_28_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_30_28_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 45
No. of loci              : 19
No. of usable loci       : 19 loci with less than  5.00 % missing data
No. of polymorphic sites : 19


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.21262
     3       150         2   0.05226
     4       150         2   0.21262
     5       150         2   0.12528
     6       150         2   0.08957
     7       150         2   0.13682
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.17038
    11       150         2   0.24242
    12       150         2   0.02649
    13       150         2   0.22273
    14       150         2   0.14819
    15       150         2   0.01333
    16       150         2   0.12528
    17       150         2   0.19186
    18       150         2   0.12528
    19       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.11308
  s.d.     0.000     0.000   0.08226
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.1837
Gene diversity           :  0.8217 +/-  0.0316

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 45

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 19
Number of usable nucleotide sites            : 19


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.148456 +/-     1.200367

Average gene diversity over loci              :     0.113077 +/-     0.069931

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     3.699327
  S.D. Theta(Hom) :     0.870277

         Theta(k) :    21.427915
95 % confidence interval limits for theta(k) : [  14.772787,  30.751806 ]

         Theta(S) :     3.402266
    S.D. Theta(S) :     1.075424

        Theta(Pi) :     2.148456
   S.D. Theta(Pi) :     1.328691

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1992    1481.436
       1    2128    2854.795
       2    2818    2853.542
       3    1928    1990.701
       4    1417    1102.217
       5     621     522.791
       6     218     223.854
       7      46      89.870
       8       5      34.745
       9       2      13.165
      10       0       4.940
      11       0       1.846
      12       0       0.689
      13       0       0.257
      14       0       0.096
      15       0       0.036
      16       0       0.013
      17       0       0.005
      18       0       0.002


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.148
Mismatch observed variance          :        2.501
Tau                                 :        2.686
Theta0                              :        0.000
Theta1                              :        6.953

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00178495
          P(Sim. Ssd >= Obs. Ssd): 0.70000000
    Harpending's Raggedness index: 0.01901912
        P(Sim. Rag. >= Obs. Rag.): 0.90000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1992.0       2478.5       1857.8
          1.0       2128.0       2704.0       2387.3
          2.0       2818.0       2792.8       2503.9
          3.0       1928.0       1820.8       2017.9
          4.0       1417.0        862.2       1284.0
          5.0        621.0        350.3        668.3
          6.0        218.0        136.4        293.2
          7.0         46.0         25.4        110.9
          8.0          5.0          3.9         36.9
          9.0          2.0          0.7         10.9
         10.0          0.0          0.0          2.9
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.686           0.686           3.482           2.101
                   Theta0:           0.000           0.000           0.299           0.032               
                   Theta1:           6.953           3.738       99999.000       30021.880
Mean No. of pairwise diff.           2.148           0.747           2.999           1.788
           No. pol. sites:              19              17              32          23.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1992.0       1080.0       5447.0       1857.8
          1.0       2128.0       1223.0       4121.0       2387.3
          2.0       2818.0       1212.0       4306.0       2503.9
          3.0       1928.0        498.0       2758.0       2017.9
          4.0       1417.0        114.0       1678.0       1284.0
          5.0        621.0         14.0       1468.0        668.3
          6.0        218.0          0.0        866.0        293.2
          7.0         46.0          0.0        116.0        110.9
          8.0          5.0          0.0         39.0         36.9
          9.0          2.0          0.0          7.0         10.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.686           0.686           3.482           2.101
                   Theta0:           0.000           0.000           0.299           0.032               
                   Theta1:           6.953           3.738       99999.000       30021.880
Mean No. of pairwise diff.           2.148           0.747           2.999           1.788
           No. pol. sites:              19              17              32          23.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1992.0       1080.0       5447.0       1857.8
          1.0       2128.0       1223.0       4121.0       2387.3
          2.0       2818.0       1212.0       4306.0       2503.9
          3.0       1928.0        498.0       2758.0       2017.9
          4.0       1417.0        114.0       1678.0       1284.0
          5.0        621.0         14.0       1468.0        668.3
          6.0        218.0          0.0        866.0        293.2
          7.0         46.0          0.0        116.0        110.9
          8.0          5.0          0.0         39.0         36.9
          9.0          2.0          0.0          7.0         10.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.686           0.686           3.482           2.101
                   Theta0:           0.000           0.000           0.299           0.032               
                   Theta1:           6.953           3.738       99999.000       30021.880
Mean No. of pairwise diff.           2.148           0.747           2.999           1.788
           No. pol. sites:              19              17              32          23.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1992.0       1080.0       5447.0       1857.8
          1.0       2128.0       1223.0       4121.0       2387.3
          2.0       2818.0       1212.0       4306.0       2503.9
          3.0       1928.0        498.0       2758.0       2017.9
          4.0       1417.0        114.0       1678.0       1284.0
          5.0        621.0         14.0       1468.0        668.3
          6.0        218.0          0.0        866.0        293.2
          7.0         46.0          0.0        116.0        110.9
          8.0          5.0          0.0         39.0         36.9
          9.0          2.0          0.0          7.0         10.9
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_30_28_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            19      19.000       0.000
  No. of usable loci            19      19.000       0.000
  No. of polym. loci            19      19.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_30_28_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.21262     0.21262     0.00000     0.21262
     3       0.05226     0.05226     0.00000     0.05226
     4       0.21262     0.21262     0.00000     0.21262
     5       0.12528     0.12528     0.00000     0.12528
     6       0.08957     0.08957     0.00000     0.08957
     7       0.13682     0.13682     0.00000     0.13682
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.17038     0.17038     0.00000     0.17038
    11       0.24242     0.24242     0.00000     0.24242
    12       0.02649     0.02649     0.00000     0.02649
    13       0.22273     0.22273     0.00000     0.22273
    14       0.14819     0.14819     0.00000     0.14819
    15       0.01333     0.01333     0.00000     0.01333
    16       0.12528     0.12528     0.00000     0.12528
    17       0.19186     0.19186     0.00000     0.19186
    18       0.12528     0.12528     0.00000     0.12528
    19       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.11308     0.11308     0.00000     0.11308
  s.d.       0.08226     0.08226     0.00000     0.08226
--------------------------------------------------------
<A NAME="31_01_17at16_30_28_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_30_28_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      21.42791    21.42791     0.00000
            Theta_k_lower      14.77279    14.77279     0.00000
            Theta_k_upper      30.75181    30.75181     0.00000
                  Theta_H       3.69933     3.69933     0.00000
             s.d. Theta_H       0.87028     0.87028     0.00000
                  Theta_S       3.40227     3.40227     0.00000
             s.d. Theta_S       1.07542     1.07542     0.00000
                 Theta_pi       2.14846     2.14846     0.00000
            s.d. Theta_pi       1.32869     1.32869     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_30_28_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       2.68555     2.68555     0.00000
              Tau qt 2.5%       0.68555     0.68555     0.00000
                Tau qt 5%       0.68555     0.68555     0.00000
               Tau qt 95%       3.36914     3.36914     0.00000
             Tau qt 97.5%       3.36914     3.36914     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.01406     0.01406     0.00000
          Theta0 qt 97.5%       0.01406     0.01406     0.00000
                   Theta1       6.95312     6.95312     0.00000
           Theta1 qt 2.5%       3.73779     3.73779     0.00000
             Theta1 qt 5%       3.73779     3.73779     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00178     0.00178     0.00000
      Model (SSD) p-value       0.70000     0.70000     0.00000
         Raggedness index       0.01902     0.01902     0.00000
       Raggedness p-value       0.90000     0.90000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 73 (31/01/17 at 16:30:28))
Total computing time for this run : 0h 0m 0s 112 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

