
<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_42_22"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 10 (31/01/17 at 14:42:22)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_42_22_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_42_22_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 33
No. of loci              : 37
No. of usable loci       : 37 loci with less than  5.00 % missing data
No. of polymorphic sites : 37


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.13682
     2       150         2   0.03946
     3       150         2   0.01333
     4       150         2   0.01333
     5       150         2   0.01333
     6       150         2   0.08957
     7       150         2   0.03946
     8       150         2   0.01333
     9       150         2   0.02649
    10       150         2   0.02649
    11       150         2   0.02649
    12       150         2   0.01333
    13       150         2   0.01333
    14       150         2   0.01333
    15       150         2   0.02649
    16       150         2   0.05226
    17       150         2   0.05226
    18       150         2   0.08957
    19       150         2   0.01333
    20       150         2   0.01333
    21       150         2   0.01333
    22       150         2   0.02649
    23       150         2   0.01333
    24       150         2   0.01333
    25       150         2   0.01333
    26       150         2   0.01333
    27       150         2   0.13682
    28       150         2   0.03946
    29       150         2   0.01333
    30       150         2   0.01333
    31       150         2   0.02649
    32       150         2   0.01333
    33       150         2   0.01333
    34       150         2   0.08957
    35       150         2   0.01333
    36       150         2   0.03946
    37       150         2   0.22273
-------------------------------------------------
  Mean   150.000     2.000   0.03891
  s.d.     0.000     0.000   0.04545
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.2311
Gene diversity           :  0.7740 +/-  0.0336

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 33

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 37
Number of usable nucleotide sites            : 37


Distance method : Pairwise difference
Mean number of pairwise differences           :     1.439732 +/-     0.882040

Average gene diversity over loci              :     0.038912 +/-     0.026387

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     2.683126
  S.D. Theta(Hom) :     0.552950

         Theta(k) :    12.788545
95 % confidence interval limits for theta(k) : [   8.448358,  19.034782 ]

         Theta(S) :     6.625465
    S.D. Theta(S) :     1.821540

        Theta(Pi) :     1.439732
   S.D. Theta(Pi) :     0.976333

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    2525       0.000
       1    3713       0.000
       2    2948       0.000
       3    1534       0.000
       4     399       0.000
       5      54       0.000
       6       2       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        1.440
Mismatch observed variance          :        1.249
Tau                                 :        1.492
Theta0                              :        0.000
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00056292
          P(Sim. Ssd >= Obs. Ssd): 0.30000000
    Harpending's Raggedness index: 0.04328872
        P(Sim. Rag. >= Obs. Rag.): 0.60000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       2525.0       2366.6       2513.1
          1.0       3713.0       3848.1       3749.9
          2.0       2948.0       2828.1       2797.8
          3.0       1534.0       1401.6       1391.6
          4.0        399.0        540.1        519.1
          5.0         54.0        151.9        154.9
          6.0          2.0         33.7         38.5
          7.0          0.0          4.4          8.2
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.492           1.248           1.707           1.454
                   Theta0:           0.000           0.000           0.585           0.083               
                   Theta1:       99999.000          75.172       99999.000       80032.180
Mean No. of pairwise diff.           1.440           1.253           1.798           1.509
           No. pol. sites:              37              85             116          98.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2525.0       1830.0       3003.0       2513.1
          1.0       3713.0       3334.0       4413.0       3749.9
          2.0       2948.0       2565.0       3049.0       2797.8
          3.0       1534.0       1092.0       1868.0       1391.6
          4.0        399.0        257.0        891.0        519.1
          5.0         54.0         34.0        354.0        154.9
          6.0          2.0          1.0        100.0         38.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.492           1.248           1.707           1.454
                   Theta0:           0.000           0.000           0.585           0.083               
                   Theta1:       99999.000          75.172       99999.000       80032.180
Mean No. of pairwise diff.           1.440           1.253           1.798           1.509
           No. pol. sites:              37              85             116          98.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2525.0       1830.0       3003.0       2513.1
          1.0       3713.0       3334.0       4413.0       3749.9
          2.0       2948.0       2565.0       3049.0       2797.8
          3.0       1534.0       1092.0       1868.0       1391.6
          4.0        399.0        257.0        891.0        519.1
          5.0         54.0         34.0        354.0        154.9
          6.0          2.0          1.0        100.0         38.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.492           1.248           1.707           1.454
                   Theta0:           0.000           0.000           0.585           0.083               
                   Theta1:       99999.000          75.172       99999.000       80032.180
Mean No. of pairwise diff.           1.440           1.253           1.798           1.509
           No. pol. sites:              37              85             116          98.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2525.0       1830.0       3003.0       2513.1
          1.0       3713.0       3334.0       4413.0       3749.9
          2.0       2948.0       2565.0       3049.0       2797.8
          3.0       1534.0       1092.0       1868.0       1391.6
          4.0        399.0        257.0        891.0        519.1
          5.0         54.0         34.0        354.0        154.9
          6.0          2.0          1.0        100.0         38.5
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_42_22_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            37      37.000       0.000
  No. of usable loci            37      37.000       0.000
  No. of polym. loci            37      37.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_42_22_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.13682     0.13682     0.00000     0.13682
     2       0.03946     0.03946     0.00000     0.03946
     3       0.01333     0.01333     0.00000     0.01333
     4       0.01333     0.01333     0.00000     0.01333
     5       0.01333     0.01333     0.00000     0.01333
     6       0.08957     0.08957     0.00000     0.08957
     7       0.03946     0.03946     0.00000     0.03946
     8       0.01333     0.01333     0.00000     0.01333
     9       0.02649     0.02649     0.00000     0.02649
    10       0.02649     0.02649     0.00000     0.02649
    11       0.02649     0.02649     0.00000     0.02649
    12       0.01333     0.01333     0.00000     0.01333
    13       0.01333     0.01333     0.00000     0.01333
    14       0.01333     0.01333     0.00000     0.01333
    15       0.02649     0.02649     0.00000     0.02649
    16       0.05226     0.05226     0.00000     0.05226
    17       0.05226     0.05226     0.00000     0.05226
    18       0.08957     0.08957     0.00000     0.08957
    19       0.01333     0.01333     0.00000     0.01333
    20       0.01333     0.01333     0.00000     0.01333
    21       0.01333     0.01333     0.00000     0.01333
    22       0.02649     0.02649     0.00000     0.02649
    23       0.01333     0.01333     0.00000     0.01333
    24       0.01333     0.01333     0.00000     0.01333
    25       0.01333     0.01333     0.00000     0.01333
    26       0.01333     0.01333     0.00000     0.01333
    27       0.13682     0.13682     0.00000     0.13682
    28       0.03946     0.03946     0.00000     0.03946
    29       0.01333     0.01333     0.00000     0.01333
    30       0.01333     0.01333     0.00000     0.01333
    31       0.02649     0.02649     0.00000     0.02649
    32       0.01333     0.01333     0.00000     0.01333
    33       0.01333     0.01333     0.00000     0.01333
    34       0.08957     0.08957     0.00000     0.08957
    35       0.01333     0.01333     0.00000     0.01333
    36       0.03946     0.03946     0.00000     0.03946
    37       0.22273     0.22273     0.00000     0.22273
--------------------------------------------------------
  Mean       0.03891     0.03891     0.00000     0.03891
  s.d.       0.04545     0.04545     0.00000     0.04545
--------------------------------------------------------
<A NAME="31_01_17at14_42_22_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_42_22_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      12.78854    12.78854     0.00000
            Theta_k_lower       8.44836     8.44836     0.00000
            Theta_k_upper      19.03478    19.03478     0.00000
                  Theta_H       2.68313     2.68313     0.00000
             s.d. Theta_H       0.55295     0.55295     0.00000
                  Theta_S       6.62546     6.62546     0.00000
             s.d. Theta_S       1.82154     1.82154     0.00000
                 Theta_pi       1.43973     1.43973     0.00000
            s.d. Theta_pi       0.97633     0.97633     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_42_22_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.49219     1.49219     0.00000
              Tau qt 2.5%       1.24805     1.24805     0.00000
                Tau qt 5%       1.24805     1.24805     0.00000
               Tau qt 95%       1.57031     1.57031     0.00000
             Tau qt 97.5%       1.57031     1.57031     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.20039     0.20039     0.00000
          Theta0 qt 97.5%       0.20039     0.20039     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      75.17188    75.17188     0.00000
             Theta1 qt 5%      75.17188    75.17188     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00056     0.00056     0.00000
      Model (SSD) p-value       0.30000     0.30000     0.00000
         Raggedness index       0.04329     0.04329     0.00000
       Raggedness p-value       0.60000     0.60000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 10 (31/01/17 at 14:42:22))
Total computing time for this run : 0h 0m 0s 78 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

