
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_13"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 194 (31/01/17 at 14:44:13)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_13_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_13_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 52
No. of loci              : 71
No. of usable loci       : 71 loci with less than  5.00 % missing data
No. of polymorphic sites : 71


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.05226
     3       150         2   0.02649
     4       150         2   0.01333
     5       150         2   0.07732
     6       150         2   0.03946
     7       150         2   0.07732
     8       150         2   0.03946
     9       150         2   0.01333
    10       150         2   0.02649
    11       150         2   0.01333
    12       150         2   0.17038
    13       150         2   0.02649
    14       150         2   0.01333
    15       150         2   0.01333
    16       150         2   0.02649
    17       150         2   0.01333
    18       150         2   0.01333
    19       150         2   0.05226
    20       150         2   0.02649
    21       150         2   0.11356
    22       150         2   0.02649
    23       150         2   0.03946
    24       150         2   0.01333
    25       150         2   0.01333
    26       150         2   0.07732
    27       150         2   0.01333
    28       150         2   0.03946
    29       150         2   0.02649
    30       150         2   0.02649
    31       150         2   0.01333
    32       150         2   0.02649
    33       150         2   0.01333
    34       150         2   0.02649
    35       150         2   0.02649
    36       150         2   0.06488
    37       150         2   0.02649
    38       150         2   0.02649
    39       150         2   0.01333
    40       150         2   0.07732
    41       150         2   0.01333
    42       150         2   0.02649
    43       150         2   0.02649
    44       150         2   0.01333
    45       150         2   0.02649
    46       150         2   0.12528
    47       150         2   0.07732
    48       150         2   0.02649
    49       150         2   0.01333
    50       150         2   0.02649
    51       150         2   0.01333
    52       150         2   0.03946
    53       150         2   0.05226
    54       150         2   0.01333
    55       150         2   0.01333
    56       150         2   0.02649
    57       150         2   0.01333
    58       150         2   0.03946
    59       150         2   0.01333
    60       150         2   0.01333
    61       150         2   0.01333
    62       150         2   0.10166
    63       150         2   0.01333
    64       150         2   0.25199
    65       150         2   0.05226
    66       150         2   0.01333
    67       150         2   0.01333
    68       150         2   0.25199
    69       150         2   0.08957
    70       150         2   0.01333
    71       150         2   0.10166
-------------------------------------------------
  Mean   150.000     2.000   0.04197
  s.d.     0.000     0.000   0.04795
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0734
Gene diversity           :  0.9328 +/-  0.0140

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 52

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 71
Number of usable nucleotide sites            : 71


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.979955 +/-     1.566536

Average gene diversity over loci              :     0.041971 +/-     0.024423

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    12.382162
  S.D. Theta(Hom) :     3.012640

         Theta(k) :    27.789032
95 % confidence interval limits for theta(k) : [  19.465082,  39.354984 ]

         Theta(S) :    12.713730
    S.D. Theta(S) :     3.198456

        Theta(Pi) :     2.979955
   S.D. Theta(Pi) :     1.734003

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     751     578.972
       1    1771    1703.792
       2    2063    2515.823
       3    2391    2486.293
       4    2059    1850.864
       5    1190    1107.626
       6     656     555.378
       7     250     240.151
       8      37      91.490
       9       7      31.224
      10       0       9.675
      11       0       2.753
      12       0       0.726
      13       0       0.179
      14       0       0.042
      15       0       0.009
      16       0       0.002
      17       0       0.000
      18       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.980
Mismatch observed variance          :        3.025
Tau                                 :        3.246
Theta0                              :        0.026
Theta1                              :       25.508

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00102914
          P(Sim. Ssd >= Obs. Ssd): 0.70000000
    Harpending's Raggedness index: 0.02077939
        P(Sim. Rag. >= Obs. Rag.): 0.80000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        751.0        723.8        780.4
          1.0       1771.0       1351.6       1566.0
          2.0       2063.0       2221.7       2253.4
          3.0       2391.0       2651.6       2342.5
          4.0       2059.0       1969.4       1873.0
          5.0       1190.0       1250.4       1210.4
          6.0        656.0        619.4        655.0
          7.0        250.0        263.3        304.6
          8.0         37.0         88.9        124.2
          9.0          7.0         22.6         45.0
         10.0          0.0          8.9         14.7
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.246           2.084           3.861           3.161
                   Theta0:           0.026           0.000           1.018           0.194               
                   Theta1:          25.508           9.587       99999.000       50033.409
Mean No. of pairwise diff.           2.980           2.591           3.699           3.086
           No. pol. sites:              71              46              67          57.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        751.0        396.0       1435.0        780.4
          1.0       1771.0        715.0       2083.0       1566.0
          2.0       2063.0       1702.0       3412.0       2253.4
          3.0       2391.0       2068.0       3578.0       2342.5
          4.0       2059.0       1607.0       2604.0       1873.0
          5.0       1190.0        387.0       1802.0       1210.4
          6.0        656.0         91.0       1075.0        655.0
          7.0        250.0         10.0        635.0        304.6
          8.0         37.0          0.0        319.0        124.2
          9.0          7.0          0.0        117.0         45.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.246           2.084           3.861           3.161
                   Theta0:           0.026           0.000           1.018           0.194               
                   Theta1:          25.508           9.587       99999.000       50033.409
Mean No. of pairwise diff.           2.980           2.591           3.699           3.086
           No. pol. sites:              71              46              67          57.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        751.0        396.0       1435.0        780.4
          1.0       1771.0        715.0       2083.0       1566.0
          2.0       2063.0       1702.0       3412.0       2253.4
          3.0       2391.0       2068.0       3578.0       2342.5
          4.0       2059.0       1607.0       2604.0       1873.0
          5.0       1190.0        387.0       1802.0       1210.4
          6.0        656.0         91.0       1075.0        655.0
          7.0        250.0         10.0        635.0        304.6
          8.0         37.0          0.0        319.0        124.2
          9.0          7.0          0.0        117.0         45.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.246           2.084           3.861           3.161
                   Theta0:           0.026           0.000           1.018           0.194               
                   Theta1:          25.508           9.587       99999.000       50033.409
Mean No. of pairwise diff.           2.980           2.591           3.699           3.086
           No. pol. sites:              71              46              67          57.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        751.0        396.0       1435.0        780.4
          1.0       1771.0        715.0       2083.0       1566.0
          2.0       2063.0       1702.0       3412.0       2253.4
          3.0       2391.0       2068.0       3578.0       2342.5
          4.0       2059.0       1607.0       2604.0       1873.0
          5.0       1190.0        387.0       1802.0       1210.4
          6.0        656.0         91.0       1075.0        655.0
          7.0        250.0         10.0        635.0        304.6
          8.0         37.0          0.0        319.0        124.2
          9.0          7.0          0.0        117.0         45.0
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_13_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            71      71.000       0.000
  No. of usable loci            71      71.000       0.000
  No. of polym. loci            71      71.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_13_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.05226     0.05226     0.00000     0.05226
     3       0.02649     0.02649     0.00000     0.02649
     4       0.01333     0.01333     0.00000     0.01333
     5       0.07732     0.07732     0.00000     0.07732
     6       0.03946     0.03946     0.00000     0.03946
     7       0.07732     0.07732     0.00000     0.07732
     8       0.03946     0.03946     0.00000     0.03946
     9       0.01333     0.01333     0.00000     0.01333
    10       0.02649     0.02649     0.00000     0.02649
    11       0.01333     0.01333     0.00000     0.01333
    12       0.17038     0.17038     0.00000     0.17038
    13       0.02649     0.02649     0.00000     0.02649
    14       0.01333     0.01333     0.00000     0.01333
    15       0.01333     0.01333     0.00000     0.01333
    16       0.02649     0.02649     0.00000     0.02649
    17       0.01333     0.01333     0.00000     0.01333
    18       0.01333     0.01333     0.00000     0.01333
    19       0.05226     0.05226     0.00000     0.05226
    20       0.02649     0.02649     0.00000     0.02649
    21       0.11356     0.11356     0.00000     0.11356
    22       0.02649     0.02649     0.00000     0.02649
    23       0.03946     0.03946     0.00000     0.03946
    24       0.01333     0.01333     0.00000     0.01333
    25       0.01333     0.01333     0.00000     0.01333
    26       0.07732     0.07732     0.00000     0.07732
    27       0.01333     0.01333     0.00000     0.01333
    28       0.03946     0.03946     0.00000     0.03946
    29       0.02649     0.02649     0.00000     0.02649
    30       0.02649     0.02649     0.00000     0.02649
    31       0.01333     0.01333     0.00000     0.01333
    32       0.02649     0.02649     0.00000     0.02649
    33       0.01333     0.01333     0.00000     0.01333
    34       0.02649     0.02649     0.00000     0.02649
    35       0.02649     0.02649     0.00000     0.02649
    36       0.06488     0.06488     0.00000     0.06488
    37       0.02649     0.02649     0.00000     0.02649
    38       0.02649     0.02649     0.00000     0.02649
    39       0.01333     0.01333     0.00000     0.01333
    40       0.07732     0.07732     0.00000     0.07732
    41       0.01333     0.01333     0.00000     0.01333
    42       0.02649     0.02649     0.00000     0.02649
    43       0.02649     0.02649     0.00000     0.02649
    44       0.01333     0.01333     0.00000     0.01333
    45       0.02649     0.02649     0.00000     0.02649
    46       0.12528     0.12528     0.00000     0.12528
    47       0.07732     0.07732     0.00000     0.07732
    48       0.02649     0.02649     0.00000     0.02649
    49       0.01333     0.01333     0.00000     0.01333
    50       0.02649     0.02649     0.00000     0.02649
    51       0.01333     0.01333     0.00000     0.01333
    52       0.03946     0.03946     0.00000     0.03946
    53       0.05226     0.05226     0.00000     0.05226
    54       0.01333     0.01333     0.00000     0.01333
    55       0.01333     0.01333     0.00000     0.01333
    56       0.02649     0.02649     0.00000     0.02649
    57       0.01333     0.01333     0.00000     0.01333
    58       0.03946     0.03946     0.00000     0.03946
    59       0.01333     0.01333     0.00000     0.01333
    60       0.01333     0.01333     0.00000     0.01333
    61       0.01333     0.01333     0.00000     0.01333
    62       0.10166     0.10166     0.00000     0.10166
    63       0.01333     0.01333     0.00000     0.01333
    64       0.25199     0.25199     0.00000     0.25199
    65       0.05226     0.05226     0.00000     0.05226
    66       0.01333     0.01333     0.00000     0.01333
    67       0.01333     0.01333     0.00000     0.01333
    68       0.25199     0.25199     0.00000     0.25199
    69       0.08957     0.08957     0.00000     0.08957
    70       0.01333     0.01333     0.00000     0.01333
    71       0.10166     0.10166     0.00000     0.10166
--------------------------------------------------------
  Mean       0.04197     0.04197     0.00000     0.04197
  s.d.       0.04795     0.04795     0.00000     0.04795
--------------------------------------------------------
<A NAME="31_01_17at14_44_13_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_13_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      27.78903    27.78903     0.00000
            Theta_k_lower      19.46508    19.46508     0.00000
            Theta_k_upper      39.35498    39.35498     0.00000
                  Theta_H      12.38216    12.38216     0.00000
             s.d. Theta_H       3.01264     3.01264     0.00000
                  Theta_S      12.71373    12.71373     0.00000
             s.d. Theta_S       3.19846     3.19846     0.00000
                 Theta_pi       2.97996     2.97996     0.00000
            s.d. Theta_pi       1.73400     1.73400     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_13_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.24609     3.24609     0.00000
              Tau qt 2.5%       2.08398     2.08398     0.00000
                Tau qt 5%       2.08398     2.08398     0.00000
               Tau qt 95%       3.68164     3.68164     0.00000
             Tau qt 97.5%       3.68164     3.68164     0.00000
                   Theta0       0.02617     0.02617     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.90527     0.90527     0.00000
          Theta0 qt 97.5%       0.90527     0.90527     0.00000
                   Theta1      25.50781    25.50781     0.00000
           Theta1 qt 2.5%       9.58740     9.58740     0.00000
             Theta1 qt 5%       9.58740     9.58740     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00103     0.00103     0.00000
      Model (SSD) p-value       0.70000     0.70000     0.00000
         Raggedness index       0.02078     0.02078     0.00000
       Raggedness p-value       0.80000     0.80000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 194 (31/01/17 at 14:44:13))
Total computing time for this run : 0h 0m 0s 130 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

