
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_47_11"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 8 (31/01/17 at 14:47:11)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_47_11_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_47_11_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 54
No. of loci              : 72
No. of usable loci       : 72 loci with less than  5.00 % missing data
No. of polymorphic sites : 72


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.02649
     2       150         2   0.01333
     3       150         2   0.17038
     4       150         2   0.01333
     5       150         2   0.08957
     6       150         2   0.01333
     7       150         2   0.03946
     8       150         2   0.06488
     9       150         2   0.03946
    10       150         2   0.01333
    11       150         2   0.01333
    12       150         2   0.03946
    13       150         2   0.17038
    14       150         2   0.05226
    15       150         2   0.01333
    16       150         2   0.20233
    17       150         2   0.13682
    18       150         2   0.01333
    19       150         2   0.05226
    20       150         2   0.01333
    21       150         2   0.01333
    22       150         2   0.01333
    23       150         2   0.03946
    24       150         2   0.01333
    25       150         2   0.10166
    26       150         2   0.18121
    27       150         2   0.02649
    28       150         2   0.07732
    29       150         2   0.05226
    30       150         2   0.03946
    31       150         2   0.02649
    32       150         2   0.01333
    33       150         2   0.01333
    34       150         2   0.07732
    35       150         2   0.01333
    36       150         2   0.03946
    37       150         2   0.22273
    38       150         2   0.01333
    39       150         2   0.02649
    40       150         2   0.02649
    41       150         2   0.03946
    42       150         2   0.01333
    43       150         2   0.07732
    44       150         2   0.01333
    45       150         2   0.02649
    46       150         2   0.15937
    47       150         2   0.01333
    48       150         2   0.03946
    49       150         2   0.01333
    50       150         2   0.15937
    51       150         2   0.01333
    52       150         2   0.06488
    53       150         2   0.01333
    54       150         2   0.01333
    55       150         2   0.08957
    56       150         2   0.03946
    57       150         2   0.01333
    58       150         2   0.06488
    59       150         2   0.33790
    60       150         2   0.01333
    61       150         2   0.32215
    62       150         2   0.02649
    63       150         2   0.02649
    64       150         2   0.10166
    65       150         2   0.17038
    66       150         2   0.01333
    67       150         2   0.01333
    68       150         2   0.02649
    69       150         2   0.02649
    70       150         2   0.02649
    71       150         2   0.03946
    72       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.05896
  s.d.     0.000     0.000   0.07034
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0365
Gene diversity           :  0.9699 +/-  0.0049

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 54

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 72
Number of usable nucleotide sites            : 72


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.244832 +/-     2.117238

Average gene diversity over loci              :     0.058956 +/-     0.032550

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    30.494101
  S.D. Theta(Hom) :     5.357828

         Theta(k) :    29.820320
95 % confidence interval limits for theta(k) : [  20.964943,  42.104679 ]

         Theta(S) :    12.892796
    S.D. Theta(S) :     3.238739

        Theta(Pi) :     4.244832
   S.D. Theta(Pi) :     2.343577

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     336       0.000
       1     572       0.000
       2    1183       0.000
       3    1814       0.000
       4    2180       0.000
       5    2248       0.000
       6    1538       0.000
       7     716       0.000
       8     487       0.000
       9      50       0.000
      10      51       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.245
Mismatch observed variance          :        3.770
Tau                                 :        4.496
Theta0                              :        0.000
Theta1                              :      515.625

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00220161
          P(Sim. Ssd >= Obs. Ssd): 0.10000000
    Harpending's Raggedness index: 0.01915072
        P(Sim. Rag. >= Obs. Rag.): 0.40000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        336.0        142.4        144.9
          1.0        572.0        530.9        575.8
          2.0       1183.0       1175.7       1266.6
          3.0       1814.0       1854.7       1885.6
          4.0       2180.0       2169.0       2113.7
          5.0       2248.0       1961.1       1898.0
          6.0       1538.0       1472.6       1421.1
          7.0        716.0        929.1        912.3
          8.0        487.0        493.6        512.5
          9.0         50.0        247.6        256.0
         10.0         51.0        123.9        115.1
         11.0          0.0         45.7         47.0
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.496           4.111           5.389           4.535
                   Theta0:           0.000           0.000           0.294           0.047               
                   Theta1:         515.625          46.592       99999.000       60057.348
Mean No. of pairwise diff.           4.245           3.871           5.340           4.523
           No. pol. sites:              72             169             214         189.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        336.0         53.0        393.0        144.9
          1.0        572.0        327.0        991.0        575.8
          2.0       1183.0        761.0       1495.0       1266.6
          3.0       1814.0       1271.0       2238.0       1885.6
          4.0       2180.0       1726.0       2467.0       2113.7
          5.0       2248.0       1732.0       2375.0       1898.0
          6.0       1538.0       1220.0       1901.0       1421.1
          7.0        716.0        675.0       1351.0        912.3
          8.0        487.0        268.0        981.0        512.5
          9.0         50.0         94.0        486.0        256.0
         10.0         51.0         19.0        315.0        115.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.496           4.111           5.389           4.535
                   Theta0:           0.000           0.000           0.294           0.047               
                   Theta1:         515.625          46.592       99999.000       60057.348
Mean No. of pairwise diff.           4.245           3.871           5.340           4.523
           No. pol. sites:              72             169             214         189.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        336.0         53.0        393.0        144.9
          1.0        572.0        327.0        991.0        575.8
          2.0       1183.0        761.0       1495.0       1266.6
          3.0       1814.0       1271.0       2238.0       1885.6
          4.0       2180.0       1726.0       2467.0       2113.7
          5.0       2248.0       1732.0       2375.0       1898.0
          6.0       1538.0       1220.0       1901.0       1421.1
          7.0        716.0        675.0       1351.0        912.3
          8.0        487.0        268.0        981.0        512.5
          9.0         50.0         94.0        486.0        256.0
         10.0         51.0         19.0        315.0        115.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.496           4.111           5.389           4.535
                   Theta0:           0.000           0.000           0.294           0.047               
                   Theta1:         515.625          46.592       99999.000       60057.348
Mean No. of pairwise diff.           4.245           3.871           5.340           4.523
           No. pol. sites:              72             169             214         189.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        336.0         53.0        393.0        144.9
          1.0        572.0        327.0        991.0        575.8
          2.0       1183.0        761.0       1495.0       1266.6
          3.0       1814.0       1271.0       2238.0       1885.6
          4.0       2180.0       1726.0       2467.0       2113.7
          5.0       2248.0       1732.0       2375.0       1898.0
          6.0       1538.0       1220.0       1901.0       1421.1
          7.0        716.0        675.0       1351.0        912.3
          8.0        487.0        268.0        981.0        512.5
          9.0         50.0         94.0        486.0        256.0
         10.0         51.0         19.0        315.0        115.1
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_47_11_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            72      72.000       0.000
  No. of usable loci            72      72.000       0.000
  No. of polym. loci            72      72.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_47_11_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.02649     0.02649     0.00000     0.02649
     2       0.01333     0.01333     0.00000     0.01333
     3       0.17038     0.17038     0.00000     0.17038
     4       0.01333     0.01333     0.00000     0.01333
     5       0.08957     0.08957     0.00000     0.08957
     6       0.01333     0.01333     0.00000     0.01333
     7       0.03946     0.03946     0.00000     0.03946
     8       0.06488     0.06488     0.00000     0.06488
     9       0.03946     0.03946     0.00000     0.03946
    10       0.01333     0.01333     0.00000     0.01333
    11       0.01333     0.01333     0.00000     0.01333
    12       0.03946     0.03946     0.00000     0.03946
    13       0.17038     0.17038     0.00000     0.17038
    14       0.05226     0.05226     0.00000     0.05226
    15       0.01333     0.01333     0.00000     0.01333
    16       0.20233     0.20233     0.00000     0.20233
    17       0.13682     0.13682     0.00000     0.13682
    18       0.01333     0.01333     0.00000     0.01333
    19       0.05226     0.05226     0.00000     0.05226
    20       0.01333     0.01333     0.00000     0.01333
    21       0.01333     0.01333     0.00000     0.01333
    22       0.01333     0.01333     0.00000     0.01333
    23       0.03946     0.03946     0.00000     0.03946
    24       0.01333     0.01333     0.00000     0.01333
    25       0.10166     0.10166     0.00000     0.10166
    26       0.18121     0.18121     0.00000     0.18121
    27       0.02649     0.02649     0.00000     0.02649
    28       0.07732     0.07732     0.00000     0.07732
    29       0.05226     0.05226     0.00000     0.05226
    30       0.03946     0.03946     0.00000     0.03946
    31       0.02649     0.02649     0.00000     0.02649
    32       0.01333     0.01333     0.00000     0.01333
    33       0.01333     0.01333     0.00000     0.01333
    34       0.07732     0.07732     0.00000     0.07732
    35       0.01333     0.01333     0.00000     0.01333
    36       0.03946     0.03946     0.00000     0.03946
    37       0.22273     0.22273     0.00000     0.22273
    38       0.01333     0.01333     0.00000     0.01333
    39       0.02649     0.02649     0.00000     0.02649
    40       0.02649     0.02649     0.00000     0.02649
    41       0.03946     0.03946     0.00000     0.03946
    42       0.01333     0.01333     0.00000     0.01333
    43       0.07732     0.07732     0.00000     0.07732
    44       0.01333     0.01333     0.00000     0.01333
    45       0.02649     0.02649     0.00000     0.02649
    46       0.15937     0.15937     0.00000     0.15937
    47       0.01333     0.01333     0.00000     0.01333
    48       0.03946     0.03946     0.00000     0.03946
    49       0.01333     0.01333     0.00000     0.01333
    50       0.15937     0.15937     0.00000     0.15937
    51       0.01333     0.01333     0.00000     0.01333
    52       0.06488     0.06488     0.00000     0.06488
    53       0.01333     0.01333     0.00000     0.01333
    54       0.01333     0.01333     0.00000     0.01333
    55       0.08957     0.08957     0.00000     0.08957
    56       0.03946     0.03946     0.00000     0.03946
    57       0.01333     0.01333     0.00000     0.01333
    58       0.06488     0.06488     0.00000     0.06488
    59       0.33790     0.33790     0.00000     0.33790
    60       0.01333     0.01333     0.00000     0.01333
    61       0.32215     0.32215     0.00000     0.32215
    62       0.02649     0.02649     0.00000     0.02649
    63       0.02649     0.02649     0.00000     0.02649
    64       0.10166     0.10166     0.00000     0.10166
    65       0.17038     0.17038     0.00000     0.17038
    66       0.01333     0.01333     0.00000     0.01333
    67       0.01333     0.01333     0.00000     0.01333
    68       0.02649     0.02649     0.00000     0.02649
    69       0.02649     0.02649     0.00000     0.02649
    70       0.02649     0.02649     0.00000     0.02649
    71       0.03946     0.03946     0.00000     0.03946
    72       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.05896     0.05896     0.00000     0.05896
  s.d.       0.07034     0.07034     0.00000     0.07034
--------------------------------------------------------
<A NAME="31_01_17at14_47_11_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
    72             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_47_11_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      29.82032    29.82032     0.00000
            Theta_k_lower      20.96494    20.96494     0.00000
            Theta_k_upper      42.10468    42.10468     0.00000
                  Theta_H      30.49410    30.49410     0.00000
             s.d. Theta_H       5.35783     5.35783     0.00000
                  Theta_S      12.89280    12.89280     0.00000
             s.d. Theta_S       3.23874     3.23874     0.00000
                 Theta_pi       4.24483     4.24483     0.00000
            s.d. Theta_pi       2.34358     2.34358     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_47_11_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.49609     4.49609     0.00000
              Tau qt 2.5%       4.11133     4.11133     0.00000
                Tau qt 5%       4.11133     4.11133     0.00000
               Tau qt 95%       4.86328     4.86328     0.00000
             Tau qt 97.5%       4.86328     4.86328     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.07734     0.07734     0.00000
          Theta0 qt 97.5%       0.07734     0.07734     0.00000
                   Theta1     515.62500   515.62500     0.00000
           Theta1 qt 2.5%      46.59180    46.59180     0.00000
             Theta1 qt 5%      46.59180    46.59180     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00220     0.00220     0.00000
      Model (SSD) p-value       0.10000     0.10000     0.00000
         Raggedness index       0.01915     0.01915     0.00000
       Raggedness p-value       0.40000     0.40000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 8 (31/01/17 at 14:47:11))
Total computing time for this run : 0h 0m 0s 138 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

