
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_33_12"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 103 (31/01/17 at 16:33:12)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_33_12_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_33_12_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 59
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.41172
     3       150         2   0.38085
     4       150         2   0.08957
     5       150         2   0.03946
     6       150         2   0.13682
     7       150         2   0.01333
     8       150         2   0.36018
     9       150         2   0.25199
    10       150         2   0.10166
    11       150         2   0.41736
    12       150         2   0.10166
    13       150         2   0.03946
    14       150         2   0.05226
    15       150         2   0.42282
    16       150         2   0.40591
    17       150         2   0.07732
    18       150         2   0.08957
    19       150         2   0.26139
    20       150         2   0.47436
-------------------------------------------------
  Mean   150.000     2.000   0.20705
  s.d.     0.000     0.000   0.16687
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0281
Gene diversity           :  0.9784 +/-  0.0030

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 59

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.141029 +/-     2.072210

Average gene diversity over loci              :     0.207051 +/-     0.114687

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    43.539305
  S.D. Theta(Hom) :     6.409412

         Theta(k) :    35.373655
95 % confidence interval limits for theta(k) : [  25.063940,  49.635064 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     4.141029
   S.D. Theta(Pi) :     2.293736

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     241       0.000
       1     658       0.000
       2    1414       0.000
       3    2132       0.000
       4    2098       0.000
       5    1809       0.000
       6    1480       0.000
       7     853       0.000
       8     336       0.000
       9     123       0.000
      10      30       0.000
      11       1       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.141
Mismatch observed variance          :        3.788
Tau                                 :        4.219
Theta0                              :        0.044
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00058549
          P(Sim. Ssd >= Obs. Ssd): 0.40000000
    Harpending's Raggedness index: 0.01736975
        P(Sim. Rag. >= Obs. Rag.): 0.80000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        241.0        152.8        157.7
          1.0        658.0        689.2        671.4
          2.0       1414.0       1475.7       1430.4
          3.0       2132.0       2120.0       2031.8
          4.0       2098.0       2247.7       2164.8
          5.0       1809.0       1907.1       1845.4
          6.0       1480.0       1258.3       1311.1
          7.0        853.0        710.5        798.5
          8.0        336.0        348.1        425.6
          9.0        123.0        161.4        201.6
         10.0         30.0         66.2         86.0
         11.0          1.0         27.1         33.3
         12.0          0.0          7.8         11.9
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.219           3.641           4.344           4.157
                   Theta0:           0.044           0.000           0.042           0.009               
                   Theta1:       99999.000         171.500       99999.000       90016.250
Mean No. of pairwise diff.           4.141           3.645           4.297           4.151
           No. pol. sites:              20             205             233         217.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        241.0        105.0        190.0        157.7
          1.0        658.0        525.0        982.0        671.4
          2.0       1414.0       1309.0       2009.0       1430.4
          3.0       2132.0       1957.0       2443.0       2031.8
          4.0       2098.0       2093.0       2441.0       2164.8
          5.0       1809.0       1676.0       2070.0       1845.4
          6.0       1480.0        939.0       1396.0       1311.1
          7.0        853.0        420.0        868.0        798.5
          8.0        336.0        195.0        438.0        425.6
          9.0        123.0         72.0        245.0        201.6
         10.0         30.0         18.0        136.0         86.0
         11.0          1.0          4.0         60.0         33.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.219           3.641           4.344           4.157
                   Theta0:           0.044           0.000           0.042           0.009               
                   Theta1:       99999.000         171.500       99999.000       90016.250
Mean No. of pairwise diff.           4.141           3.645           4.297           4.151
           No. pol. sites:              20             205             233         217.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        241.0        105.0        190.0        157.7
          1.0        658.0        525.0        982.0        671.4
          2.0       1414.0       1309.0       2009.0       1430.4
          3.0       2132.0       1957.0       2443.0       2031.8
          4.0       2098.0       2093.0       2441.0       2164.8
          5.0       1809.0       1676.0       2070.0       1845.4
          6.0       1480.0        939.0       1396.0       1311.1
          7.0        853.0        420.0        868.0        798.5
          8.0        336.0        195.0        438.0        425.6
          9.0        123.0         72.0        245.0        201.6
         10.0         30.0         18.0        136.0         86.0
         11.0          1.0          4.0         60.0         33.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.219           3.641           4.344           4.157
                   Theta0:           0.044           0.000           0.042           0.009               
                   Theta1:       99999.000         171.500       99999.000       90016.250
Mean No. of pairwise diff.           4.141           3.645           4.297           4.151
           No. pol. sites:              20             205             233         217.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        241.0        105.0        190.0        157.7
          1.0        658.0        525.0        982.0        671.4
          2.0       1414.0       1309.0       2009.0       1430.4
          3.0       2132.0       1957.0       2443.0       2031.8
          4.0       2098.0       2093.0       2441.0       2164.8
          5.0       1809.0       1676.0       2070.0       1845.4
          6.0       1480.0        939.0       1396.0       1311.1
          7.0        853.0        420.0        868.0        798.5
          8.0        336.0        195.0        438.0        425.6
          9.0        123.0         72.0        245.0        201.6
         10.0         30.0         18.0        136.0         86.0
         11.0          1.0          4.0         60.0         33.3
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_33_12_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_33_12_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.41172     0.41172     0.00000     0.41172
     3       0.38085     0.38085     0.00000     0.38085
     4       0.08957     0.08957     0.00000     0.08957
     5       0.03946     0.03946     0.00000     0.03946
     6       0.13682     0.13682     0.00000     0.13682
     7       0.01333     0.01333     0.00000     0.01333
     8       0.36018     0.36018     0.00000     0.36018
     9       0.25199     0.25199     0.00000     0.25199
    10       0.10166     0.10166     0.00000     0.10166
    11       0.41736     0.41736     0.00000     0.41736
    12       0.10166     0.10166     0.00000     0.10166
    13       0.03946     0.03946     0.00000     0.03946
    14       0.05226     0.05226     0.00000     0.05226
    15       0.42282     0.42282     0.00000     0.42282
    16       0.40591     0.40591     0.00000     0.40591
    17       0.07732     0.07732     0.00000     0.07732
    18       0.08957     0.08957     0.00000     0.08957
    19       0.26139     0.26139     0.00000     0.26139
    20       0.47436     0.47436     0.00000     0.47436
--------------------------------------------------------
  Mean       0.20705     0.20705     0.00000     0.20705
  s.d.       0.16687     0.16687     0.00000     0.16687
--------------------------------------------------------
<A NAME="31_01_17at16_33_12_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_33_12_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      35.37366    35.37366     0.00000
            Theta_k_lower      25.06394    25.06394     0.00000
            Theta_k_upper      49.63506    49.63506     0.00000
                  Theta_H      43.53930    43.53930     0.00000
             s.d. Theta_H       6.40941     6.40941     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       4.14103     4.14103     0.00000
            s.d. Theta_pi       2.29374     2.29374     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_33_12_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.21875     4.21875     0.00000
              Tau qt 2.5%       3.64062     3.64062     0.00000
                Tau qt 5%       3.64062     3.64062     0.00000
               Tau qt 95%       4.34180     4.34180     0.00000
             Tau qt 97.5%       4.34180     4.34180     0.00000
                   Theta0       0.04375     0.04375     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.01738     0.01738     0.00000
          Theta0 qt 97.5%       0.01738     0.01738     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%     171.50000   171.50000     0.00000
             Theta1 qt 5%     171.50000   171.50000     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00059     0.00059     0.00000
      Model (SSD) p-value       0.40000     0.40000     0.00000
         Raggedness index       0.01737     0.01737     0.00000
       Raggedness p-value       0.80000     0.80000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 103 (31/01/17 at 16:33:12))
Total computing time for this run : 0h 0m 0s 147 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

