
<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_42_39"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 190 (31/01/17 at 14:42:39)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_42_39_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_42_39_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 44
No. of loci              : 52
No. of usable loci       : 52 loci with less than  5.00 % missing data
No. of polymorphic sites : 52


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.03946
     3       150         2   0.01333
     4       150         2   0.01333
     5       150         2   0.02649
     6       150         2   0.01333
     7       150         2   0.01333
     8       150         2   0.01333
     9       150         2   0.06488
    10       150         2   0.01333
    11       150         2   0.03946
    12       150         2   0.02649
    13       150         2   0.01333
    14       150         2   0.03946
    15       150         2   0.03946
    16       150         2   0.01333
    17       150         2   0.01333
    18       150         2   0.02649
    19       150         2   0.01333
    20       150         2   0.03946
    21       150         2   0.03946
    22       150         2   0.02649
    23       150         2   0.02649
    24       150         2   0.01333
    25       150         2   0.01333
    26       150         2   0.01333
    27       150         2   0.01333
    28       150         2   0.01333
    29       150         2   0.01333
    30       150         2   0.01333
    31       150         2   0.01333
    32       150         2   0.08957
    33       150         2   0.01333
    34       150         2   0.01333
    35       150         2   0.01333
    36       150         2   0.03946
    37       150         2   0.02649
    38       150         2   0.02649
    39       150         2   0.02649
    40       150         2   0.01333
    41       150         2   0.01333
    42       150         2   0.01333
    43       150         2   0.02649
    44       150         2   0.01333
    45       150         2   0.01333
    46       150         2   0.01333
    47       150         2   0.01333
    48       150         2   0.01333
    49       150         2   0.01333
    50       150         2   0.01333
    51       150         2   0.02649
    52       150         2   0.02649
-------------------------------------------------
  Mean   150.000     2.000   0.02209
  s.d.     0.000     0.000   0.01480
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.3125
Gene diversity           :  0.6921 +/-  0.0438

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 44

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 52
Number of usable nucleotide sites            : 52


Distance method : Pairwise difference
Mean number of pairwise differences           :     1.148725 +/-     0.747857

Average gene diversity over loci              :     0.022091 +/-     0.015919

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     1.715764
  S.D. Theta(Hom) :     0.370005

         Theta(k) :    20.605975
95 % confidence interval limits for theta(k) : [  14.167681,  29.640070 ]

         Theta(S) :     9.311464
    S.D. Theta(S) :     2.431333

        Theta(Pi) :     1.148725
   S.D. Theta(Pi) :     0.827805

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    3441       0.000
       1    4166       0.000
       2    2356       0.000
       3     935       0.000
       4     234       0.000
       5      40       0.000
       6       3       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        1.149
Mismatch observed variance          :        1.084
Tau                                 :        1.139
Theta0                              :        0.030
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00012153
          P(Sim. Ssd >= Obs. Ssd): 1.00000000
    Harpending's Raggedness index: 0.05085962
        P(Sim. Rag. >= Obs. Rag.): 0.50000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       3441.0       3444.5       3474.9
          1.0       4166.0       4137.8       4057.6
          2.0       2356.0       2344.2       2370.4
          3.0        935.0        911.4        923.8
          4.0        234.0        266.8        270.2
          5.0         40.0         59.4         63.3
          6.0          3.0          9.3         12.4
          7.0          0.0          1.5          2.1
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.139           0.938           1.393           1.151
                   Theta0:           0.030           0.000           0.004           0.000               
                   Theta1:       99999.000          39.703       99999.000       60040.250
Mean No. of pairwise diff.           1.149           1.013           1.319           1.163
           No. pol. sites:              52              67              90          78.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       3441.0       2628.0       4186.0       3474.9
          1.0       4166.0       3713.0       4559.0       4057.6
          2.0       2356.0       1831.0       2911.0       2370.4
          3.0        935.0        686.0       1257.0        923.8
          4.0        234.0        113.0        390.0        270.2
          5.0         40.0         11.0        110.0         63.3
          6.0          3.0          0.0         26.0         12.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.139           0.938           1.393           1.151
                   Theta0:           0.030           0.000           0.004           0.000               
                   Theta1:       99999.000          39.703       99999.000       60040.250
Mean No. of pairwise diff.           1.149           1.013           1.319           1.163
           No. pol. sites:              52              67              90          78.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       3441.0       2628.0       4186.0       3474.9
          1.0       4166.0       3713.0       4559.0       4057.6
          2.0       2356.0       1831.0       2911.0       2370.4
          3.0        935.0        686.0       1257.0        923.8
          4.0        234.0        113.0        390.0        270.2
          5.0         40.0         11.0        110.0         63.3
          6.0          3.0          0.0         26.0         12.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.139           0.938           1.393           1.151
                   Theta0:           0.030           0.000           0.004           0.000               
                   Theta1:       99999.000          39.703       99999.000       60040.250
Mean No. of pairwise diff.           1.149           1.013           1.319           1.163
           No. pol. sites:              52              67              90          78.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       3441.0       2628.0       4186.0       3474.9
          1.0       4166.0       3713.0       4559.0       4057.6
          2.0       2356.0       1831.0       2911.0       2370.4
          3.0        935.0        686.0       1257.0        923.8
          4.0        234.0        113.0        390.0        270.2
          5.0         40.0         11.0        110.0         63.3
          6.0          3.0          0.0         26.0         12.4
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_42_39_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            52      52.000       0.000
  No. of usable loci            52      52.000       0.000
  No. of polym. loci            52      52.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_42_39_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.03946     0.03946     0.00000     0.03946
     3       0.01333     0.01333     0.00000     0.01333
     4       0.01333     0.01333     0.00000     0.01333
     5       0.02649     0.02649     0.00000     0.02649
     6       0.01333     0.01333     0.00000     0.01333
     7       0.01333     0.01333     0.00000     0.01333
     8       0.01333     0.01333     0.00000     0.01333
     9       0.06488     0.06488     0.00000     0.06488
    10       0.01333     0.01333     0.00000     0.01333
    11       0.03946     0.03946     0.00000     0.03946
    12       0.02649     0.02649     0.00000     0.02649
    13       0.01333     0.01333     0.00000     0.01333
    14       0.03946     0.03946     0.00000     0.03946
    15       0.03946     0.03946     0.00000     0.03946
    16       0.01333     0.01333     0.00000     0.01333
    17       0.01333     0.01333     0.00000     0.01333
    18       0.02649     0.02649     0.00000     0.02649
    19       0.01333     0.01333     0.00000     0.01333
    20       0.03946     0.03946     0.00000     0.03946
    21       0.03946     0.03946     0.00000     0.03946
    22       0.02649     0.02649     0.00000     0.02649
    23       0.02649     0.02649     0.00000     0.02649
    24       0.01333     0.01333     0.00000     0.01333
    25       0.01333     0.01333     0.00000     0.01333
    26       0.01333     0.01333     0.00000     0.01333
    27       0.01333     0.01333     0.00000     0.01333
    28       0.01333     0.01333     0.00000     0.01333
    29       0.01333     0.01333     0.00000     0.01333
    30       0.01333     0.01333     0.00000     0.01333
    31       0.01333     0.01333     0.00000     0.01333
    32       0.08957     0.08957     0.00000     0.08957
    33       0.01333     0.01333     0.00000     0.01333
    34       0.01333     0.01333     0.00000     0.01333
    35       0.01333     0.01333     0.00000     0.01333
    36       0.03946     0.03946     0.00000     0.03946
    37       0.02649     0.02649     0.00000     0.02649
    38       0.02649     0.02649     0.00000     0.02649
    39       0.02649     0.02649     0.00000     0.02649
    40       0.01333     0.01333     0.00000     0.01333
    41       0.01333     0.01333     0.00000     0.01333
    42       0.01333     0.01333     0.00000     0.01333
    43       0.02649     0.02649     0.00000     0.02649
    44       0.01333     0.01333     0.00000     0.01333
    45       0.01333     0.01333     0.00000     0.01333
    46       0.01333     0.01333     0.00000     0.01333
    47       0.01333     0.01333     0.00000     0.01333
    48       0.01333     0.01333     0.00000     0.01333
    49       0.01333     0.01333     0.00000     0.01333
    50       0.01333     0.01333     0.00000     0.01333
    51       0.02649     0.02649     0.00000     0.02649
    52       0.02649     0.02649     0.00000     0.02649
--------------------------------------------------------
  Mean       0.02209     0.02209     0.00000     0.02209
  s.d.       0.01480     0.01480     0.00000     0.01480
--------------------------------------------------------
<A NAME="31_01_17at14_42_39_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_42_39_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      20.60597    20.60597     0.00000
            Theta_k_lower      14.16768    14.16768     0.00000
            Theta_k_upper      29.64007    29.64007     0.00000
                  Theta_H       1.71576     1.71576     0.00000
             s.d. Theta_H       0.37000     0.37000     0.00000
                  Theta_S       9.31146     9.31146     0.00000
             s.d. Theta_S       2.43133     2.43133     0.00000
                 Theta_pi       1.14872     1.14872     0.00000
            s.d. Theta_pi       0.82780     0.82780     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_42_39_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.13867     1.13867     0.00000
              Tau qt 2.5%       0.93750     0.93750     0.00000
                Tau qt 5%       0.93750     0.93750     0.00000
               Tau qt 95%       1.33203     1.33203     0.00000
             Tau qt 97.5%       1.33203     1.33203     0.00000
                   Theta0       0.02988     0.02988     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00000     0.00000     0.00000
          Theta0 qt 97.5%       0.00000     0.00000     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      39.70312    39.70312     0.00000
             Theta1 qt 5%      39.70312    39.70312     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00012     0.00012     0.00000
      Model (SSD) p-value       1.00000     1.00000     0.00000
         Raggedness index       0.05086     0.05086     0.00000
       Raggedness p-value       0.50000     0.50000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 190 (31/01/17 at 14:42:39))
Total computing time for this run : 0h 0m 0s 79 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

