
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_54_24"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 163 (31/01/17 at 14:54:24)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_54_24_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_54_24_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 52
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.03946
     2       150         2   0.32215
     3       150         2   0.08957
     4       150         2   0.25199
     5       150         2   0.29718
     6       150         2   0.02649
     7       150         2   0.22273
     8       150         2   0.42282
     9       150         2   0.30568
    10       150         2   0.02649
    11       150         2   0.50112
    12       150         2   0.01333
    13       150         2   0.10166
    14       150         2   0.13682
    15       150         2   0.27060
    16       150         2   0.49763
    17       150         2   0.15937
    18       150         2   0.08957
    19       150         2   0.47436
    20       150         2   0.14819
-------------------------------------------------
  Mean   150.000     2.000   0.21986
  s.d.     0.000     0.000   0.16275
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0366
Gene diversity           :  0.9698 +/-  0.0049

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 52

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.397226 +/-     2.183304

Average gene diversity over loci              :     0.219861 +/-     0.120835

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    30.396089
  S.D. Theta(Hom) :     5.391355

         Theta(k) :    27.789032
95 % confidence interval limits for theta(k) : [  19.465082,  39.354984 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     4.397226
   S.D. Theta(Pi) :     2.416706

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     337       0.000
       1     549       0.000
       2    1085       0.000
       3    1788       0.000
       4    2072       0.000
       5    2069       0.000
       6    1493       0.000
       7    1036       0.000
       8     537       0.000
       9     176       0.000
      10      30       0.000
      11       3       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.397
Mismatch observed variance          :        4.142
Tau                                 :        4.668
Theta0                              :        0.016
Theta1                              :      113.281

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00075800
          P(Sim. Ssd >= Obs. Ssd): 0.70000000
    Harpending's Raggedness index: 0.01480706
        P(Sim. Rag. >= Obs. Rag.): 0.60000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        337.0        205.3        196.0
          1.0        549.0        570.2        556.3
          2.0       1085.0       1190.7       1168.9
          3.0       1788.0       1792.8       1763.5
          4.0       2072.0       2112.6       2035.7
          5.0       2069.0       2012.5       1893.3
          6.0       1493.0       1460.8       1471.9
          7.0       1036.0        939.5        982.3
          8.0        537.0        493.9        574.1
          9.0        176.0        230.2        298.4
         10.0         30.0        100.6        139.6
         11.0          3.0         42.9         59.4
         12.0          0.0         15.0         23.2
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.668           4.164           4.947           4.497
                   Theta0:           0.016           0.000           0.737           0.127               
                   Theta1:         113.281          38.945       99999.000       50056.057
Mean No. of pairwise diff.           4.397           4.126           4.959           4.472
           No. pol. sites:              20             124             137         131.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        337.0        100.0        341.0        196.0
          1.0        549.0        350.0        819.0        556.3
          2.0       1085.0        937.0       1435.0       1168.9
          3.0       1788.0       1577.0       2098.0       1763.5
          4.0       2072.0       1847.0       2438.0       2035.7
          5.0       2069.0       1838.0       2201.0       1893.3
          6.0       1493.0       1212.0       1754.0       1471.9
          7.0       1036.0        658.0       1209.0        982.3
          8.0        537.0        288.0        659.0        574.1
          9.0        176.0         91.0        394.0        298.4
         10.0         30.0         10.0        299.0        139.6
         11.0          3.0          0.0        197.0         59.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.668           4.164           4.947           4.497
                   Theta0:           0.016           0.000           0.737           0.127               
                   Theta1:         113.281          38.945       99999.000       50056.057
Mean No. of pairwise diff.           4.397           4.126           4.959           4.472
           No. pol. sites:              20             124             137         131.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        337.0        100.0        341.0        196.0
          1.0        549.0        350.0        819.0        556.3
          2.0       1085.0        937.0       1435.0       1168.9
          3.0       1788.0       1577.0       2098.0       1763.5
          4.0       2072.0       1847.0       2438.0       2035.7
          5.0       2069.0       1838.0       2201.0       1893.3
          6.0       1493.0       1212.0       1754.0       1471.9
          7.0       1036.0        658.0       1209.0        982.3
          8.0        537.0        288.0        659.0        574.1
          9.0        176.0         91.0        394.0        298.4
         10.0         30.0         10.0        299.0        139.6
         11.0          3.0          0.0        197.0         59.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.668           4.164           4.947           4.497
                   Theta0:           0.016           0.000           0.737           0.127               
                   Theta1:         113.281          38.945       99999.000       50056.057
Mean No. of pairwise diff.           4.397           4.126           4.959           4.472
           No. pol. sites:              20             124             137         131.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        337.0        100.0        341.0        196.0
          1.0        549.0        350.0        819.0        556.3
          2.0       1085.0        937.0       1435.0       1168.9
          3.0       1788.0       1577.0       2098.0       1763.5
          4.0       2072.0       1847.0       2438.0       2035.7
          5.0       2069.0       1838.0       2201.0       1893.3
          6.0       1493.0       1212.0       1754.0       1471.9
          7.0       1036.0        658.0       1209.0        982.3
          8.0        537.0        288.0        659.0        574.1
          9.0        176.0         91.0        394.0        298.4
         10.0         30.0         10.0        299.0        139.6
         11.0          3.0          0.0        197.0         59.4
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_54_24_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_54_24_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.03946     0.03946     0.00000     0.03946
     2       0.32215     0.32215     0.00000     0.32215
     3       0.08957     0.08957     0.00000     0.08957
     4       0.25199     0.25199     0.00000     0.25199
     5       0.29718     0.29718     0.00000     0.29718
     6       0.02649     0.02649     0.00000     0.02649
     7       0.22273     0.22273     0.00000     0.22273
     8       0.42282     0.42282     0.00000     0.42282
     9       0.30568     0.30568     0.00000     0.30568
    10       0.02649     0.02649     0.00000     0.02649
    11       0.50112     0.50112     0.00000     0.50112
    12       0.01333     0.01333     0.00000     0.01333
    13       0.10166     0.10166     0.00000     0.10166
    14       0.13682     0.13682     0.00000     0.13682
    15       0.27060     0.27060     0.00000     0.27060
    16       0.49763     0.49763     0.00000     0.49763
    17       0.15937     0.15937     0.00000     0.15937
    18       0.08957     0.08957     0.00000     0.08957
    19       0.47436     0.47436     0.00000     0.47436
    20       0.14819     0.14819     0.00000     0.14819
--------------------------------------------------------
  Mean       0.21986     0.21986     0.00000     0.21986
  s.d.       0.16275     0.16275     0.00000     0.16275
--------------------------------------------------------
<A NAME="31_01_17at14_54_24_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_54_24_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      27.78903    27.78903     0.00000
            Theta_k_lower      19.46508    19.46508     0.00000
            Theta_k_upper      39.35498    39.35498     0.00000
                  Theta_H      30.39609    30.39609     0.00000
             s.d. Theta_H       5.39135     5.39135     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       4.39723     4.39723     0.00000
            s.d. Theta_pi       2.41671     2.41671     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_54_24_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.66797     4.66797     0.00000
              Tau qt 2.5%       4.16406     4.16406     0.00000
                Tau qt 5%       4.16406     4.16406     0.00000
               Tau qt 95%       4.93164     4.93164     0.00000
             Tau qt 97.5%       4.93164     4.93164     0.00000
                   Theta0       0.01582     0.01582     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.46934     0.46934     0.00000
          Theta0 qt 97.5%       0.46934     0.46934     0.00000
                   Theta1     113.28125   113.28125     0.00000
           Theta1 qt 2.5%      38.94531    38.94531     0.00000
             Theta1 qt 5%      38.94531    38.94531     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00076     0.00076     0.00000
      Model (SSD) p-value       0.70000     0.70000     0.00000
         Raggedness index       0.01481     0.01481     0.00000
       Raggedness p-value       0.60000     0.60000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 163 (31/01/17 at 14:54:24))
Total computing time for this run : 0h 0m 0s 160 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

