
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_59_44"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 144 (31/01/17 at 14:59:44)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_59_44_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_59_44_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 44
No. of loci              : 76
No. of usable loci       : 76 loci with less than  5.00 % missing data
No. of polymorphic sites : 76


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.02649
     2       150         2   0.30568
     3       150         2   0.05226
     4       150         2   0.01333
     5       150         2   0.01333
     6       150         2   0.05226
     7       150         2   0.01333
     8       150         2   0.03946
     9       150         2   0.01333
    10       150         2   0.10166
    11       150         2   0.01333
    12       150         2   0.01333
    13       150         2   0.01333
    14       150         2   0.01333
    15       150         2   0.01333
    16       150         2   0.01333
    17       150         2   0.07732
    18       150         2   0.01333
    19       150         2   0.20233
    20       150         2   0.03946
    21       150         2   0.01333
    22       150         2   0.18121
    23       150         2   0.01333
    24       150         2   0.05226
    25       150         2   0.01333
    26       150         2   0.20233
    27       150         2   0.02649
    28       150         2   0.17038
    29       150         2   0.07732
    30       150         2   0.30568
    31       150         2   0.25199
    32       150         2   0.05226
    33       150         2   0.13682
    34       150         2   0.01333
    35       150         2   0.03946
    36       150         2   0.19186
    37       150         2   0.01333
    38       150         2   0.01333
    39       150         2   0.01333
    40       150         2   0.02649
    41       150         2   0.01333
    42       150         2   0.02649
    43       150         2   0.03946
    44       150         2   0.18121
    45       150         2   0.01333
    46       150         2   0.02649
    47       150         2   0.15937
    48       150         2   0.03946
    49       150         2   0.01333
    50       150         2   0.01333
    51       150         2   0.07732
    52       150         2   0.32215
    53       150         2   0.01333
    54       150         2   0.01333
    55       150         2   0.17038
    56       150         2   0.01333
    57       150         2   0.01333
    58       150         2   0.02649
    59       150         2   0.14819
    60       150         2   0.03946
    61       150         2   0.08957
    62       150         2   0.45602
    63       150         2   0.19186
    64       150         2   0.01333
    65       150         2   0.01333
    66       150         2   0.45602
    67       150         2   0.01333
    68       150         2   0.10166
    69       150         2   0.01333
    70       150         2   0.14819
    71       150         2   0.45602
    72       150         2   0.01333
    73       150         2   0.01333
    74       150         2   0.02649
    75       150         2   0.25199
    76       150         2   0.05226
-------------------------------------------------
  Mean   150.000     2.000   0.08585
  s.d.     0.000     0.000   0.11129
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0442
Gene diversity           :  0.9622 +/-  0.0046

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 44

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 76
Number of usable nucleotide sites            : 76


Distance method : Pairwise difference
Mean number of pairwise differences           :     6.524653 +/-     3.102378

Average gene diversity over loci              :     0.085851 +/-     0.045185

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    23.774043
  S.D. Theta(Hom) :     3.212992

         Theta(k) :    20.605975
95 % confidence interval limits for theta(k) : [  14.167681,  29.640070 ]

         Theta(S) :    13.609063
    S.D. Theta(S) :     3.399804

        Theta(Pi) :     6.524653
   S.D. Theta(Pi) :     3.434032

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     422      30.805
       1     306     171.240
       2     488     481.723
       3     739     916.355
       4     787    1329.496
       5    1286    1574.195
       6    1124    1590.489
       7    1451    1416.567
       8    1488    1141.019
       9    1190     848.979
      10     973     594.134
      11     579     397.156
      12     252     256.900
      13      66     162.498
      14      20     101.318
      15       4      62.627
      16       0      38.522
      17       0      23.634
      18       0      14.481
      19       0       8.867
      20       0       5.428
      21       0       3.323
      22       0       2.034
      23       0       1.245
      24       0       0.762
      25       0       0.466
      26       0       0.285
      27       0       0.175
      28       0       0.107
      29       0       0.065
      30       0       0.040


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        6.525
Mismatch observed variance          :        9.014
Tau                                 :        7.660
Theta0                              :        0.007
Theta1                              :       31.445

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00167927
          P(Sim. Ssd >= Obs. Ssd): 0.50000000
    Harpending's Raggedness index: 0.00745088
        P(Sim. Rag. >= Obs. Rag.): 0.70000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        422.0        352.9        348.4
          1.0        306.0        424.9        364.1
          2.0        488.0        599.0        439.2
          3.0        739.0        882.7        607.3
          4.0        787.0       1013.6        862.1
          5.0       1286.0       1131.2       1140.5
          6.0       1124.0       1302.9       1349.0
          7.0       1451.0       1411.4       1414.2
          8.0       1488.0       1298.6       1319.6
          9.0       1190.0        946.2       1104.7
         10.0        973.0        711.3        836.9
         11.0        579.0        448.7        578.3
         12.0        252.0        286.4        367.2
         13.0         66.0        138.4        215.5
         14.0         20.0         93.2        117.5
         15.0          4.0         67.6         59.9
         16.0          0.0         43.3         28.6
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           7.660           3.934           9.504           6.676
                   Theta0:           0.007           0.000           1.923           0.437               
                   Theta1:          31.445          12.754         203.320          85.161
Mean No. of pairwise diff.           6.525           3.830           8.488           6.378
           No. pol. sites:              76              63              99          85.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        422.0        212.0       1085.0        348.4
          1.0        306.0        214.0        840.0        364.1
          2.0        488.0        146.0       1718.0        439.2
          3.0        739.0        239.0       2443.0        607.3
          4.0        787.0        338.0       1949.0        862.1
          5.0       1286.0        505.0       1743.0       1140.5
          6.0       1124.0        802.0       1792.0       1349.0
          7.0       1451.0        618.0       2210.0       1414.2
          8.0       1488.0        241.0       2298.0       1319.6
          9.0       1190.0         92.0       1670.0       1104.7
         10.0        973.0         17.0       1204.0        836.9
         11.0        579.0          0.0       1193.0        578.3
         12.0        252.0          0.0        917.0        367.2
         13.0         66.0          0.0        602.0        215.5
         14.0         20.0          0.0        353.0        117.5
         15.0          4.0          0.0        423.0         59.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           7.660           3.934           9.504           6.676
                   Theta0:           0.007           0.000           1.923           0.437               
                   Theta1:          31.445          12.754         203.320          85.161
Mean No. of pairwise diff.           6.525           3.830           8.488           6.378
           No. pol. sites:              76              63              99          85.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        422.0        212.0       1085.0        348.4
          1.0        306.0        214.0        840.0        364.1
          2.0        488.0        146.0       1718.0        439.2
          3.0        739.0        239.0       2443.0        607.3
          4.0        787.0        338.0       1949.0        862.1
          5.0       1286.0        505.0       1743.0       1140.5
          6.0       1124.0        802.0       1792.0       1349.0
          7.0       1451.0        618.0       2210.0       1414.2
          8.0       1488.0        241.0       2298.0       1319.6
          9.0       1190.0         92.0       1670.0       1104.7
         10.0        973.0         17.0       1204.0        836.9
         11.0        579.0          0.0       1193.0        578.3
         12.0        252.0          0.0        917.0        367.2
         13.0         66.0          0.0        602.0        215.5
         14.0         20.0          0.0        353.0        117.5
         15.0          4.0          0.0        423.0         59.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           7.660           3.934           9.504           6.676
                   Theta0:           0.007           0.000           1.923           0.437               
                   Theta1:          31.445          12.754         203.320          85.161
Mean No. of pairwise diff.           6.525           3.830           8.488           6.378
           No. pol. sites:              76              63              99          85.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        422.0        212.0       1085.0        348.4
          1.0        306.0        214.0        840.0        364.1
          2.0        488.0        146.0       1718.0        439.2
          3.0        739.0        239.0       2443.0        607.3
          4.0        787.0        338.0       1949.0        862.1
          5.0       1286.0        505.0       1743.0       1140.5
          6.0       1124.0        802.0       1792.0       1349.0
          7.0       1451.0        618.0       2210.0       1414.2
          8.0       1488.0        241.0       2298.0       1319.6
          9.0       1190.0         92.0       1670.0       1104.7
         10.0        973.0         17.0       1204.0        836.9
         11.0        579.0          0.0       1193.0        578.3
         12.0        252.0          0.0        917.0        367.2
         13.0         66.0          0.0        602.0        215.5
         14.0         20.0          0.0        353.0        117.5
         15.0          4.0          0.0        423.0         59.9
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_59_44_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            76      76.000       0.000
  No. of usable loci            76      76.000       0.000
  No. of polym. loci            76      76.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_59_44_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.02649     0.02649     0.00000     0.02649
     2       0.30568     0.30568     0.00000     0.30568
     3       0.05226     0.05226     0.00000     0.05226
     4       0.01333     0.01333     0.00000     0.01333
     5       0.01333     0.01333     0.00000     0.01333
     6       0.05226     0.05226     0.00000     0.05226
     7       0.01333     0.01333     0.00000     0.01333
     8       0.03946     0.03946     0.00000     0.03946
     9       0.01333     0.01333     0.00000     0.01333
    10       0.10166     0.10166     0.00000     0.10166
    11       0.01333     0.01333     0.00000     0.01333
    12       0.01333     0.01333     0.00000     0.01333
    13       0.01333     0.01333     0.00000     0.01333
    14       0.01333     0.01333     0.00000     0.01333
    15       0.01333     0.01333     0.00000     0.01333
    16       0.01333     0.01333     0.00000     0.01333
    17       0.07732     0.07732     0.00000     0.07732
    18       0.01333     0.01333     0.00000     0.01333
    19       0.20233     0.20233     0.00000     0.20233
    20       0.03946     0.03946     0.00000     0.03946
    21       0.01333     0.01333     0.00000     0.01333
    22       0.18121     0.18121     0.00000     0.18121
    23       0.01333     0.01333     0.00000     0.01333
    24       0.05226     0.05226     0.00000     0.05226
    25       0.01333     0.01333     0.00000     0.01333
    26       0.20233     0.20233     0.00000     0.20233
    27       0.02649     0.02649     0.00000     0.02649
    28       0.17038     0.17038     0.00000     0.17038
    29       0.07732     0.07732     0.00000     0.07732
    30       0.30568     0.30568     0.00000     0.30568
    31       0.25199     0.25199     0.00000     0.25199
    32       0.05226     0.05226     0.00000     0.05226
    33       0.13682     0.13682     0.00000     0.13682
    34       0.01333     0.01333     0.00000     0.01333
    35       0.03946     0.03946     0.00000     0.03946
    36       0.19186     0.19186     0.00000     0.19186
    37       0.01333     0.01333     0.00000     0.01333
    38       0.01333     0.01333     0.00000     0.01333
    39       0.01333     0.01333     0.00000     0.01333
    40       0.02649     0.02649     0.00000     0.02649
    41       0.01333     0.01333     0.00000     0.01333
    42       0.02649     0.02649     0.00000     0.02649
    43       0.03946     0.03946     0.00000     0.03946
    44       0.18121     0.18121     0.00000     0.18121
    45       0.01333     0.01333     0.00000     0.01333
    46       0.02649     0.02649     0.00000     0.02649
    47       0.15937     0.15937     0.00000     0.15937
    48       0.03946     0.03946     0.00000     0.03946
    49       0.01333     0.01333     0.00000     0.01333
    50       0.01333     0.01333     0.00000     0.01333
    51       0.07732     0.07732     0.00000     0.07732
    52       0.32215     0.32215     0.00000     0.32215
    53       0.01333     0.01333     0.00000     0.01333
    54       0.01333     0.01333     0.00000     0.01333
    55       0.17038     0.17038     0.00000     0.17038
    56       0.01333     0.01333     0.00000     0.01333
    57       0.01333     0.01333     0.00000     0.01333
    58       0.02649     0.02649     0.00000     0.02649
    59       0.14819     0.14819     0.00000     0.14819
    60       0.03946     0.03946     0.00000     0.03946
    61       0.08957     0.08957     0.00000     0.08957
    62       0.45602     0.45602     0.00000     0.45602
    63       0.19186     0.19186     0.00000     0.19186
    64       0.01333     0.01333     0.00000     0.01333
    65       0.01333     0.01333     0.00000     0.01333
    66       0.45602     0.45602     0.00000     0.45602
    67       0.01333     0.01333     0.00000     0.01333
    68       0.10166     0.10166     0.00000     0.10166
    69       0.01333     0.01333     0.00000     0.01333
    70       0.14819     0.14819     0.00000     0.14819
    71       0.45602     0.45602     0.00000     0.45602
    72       0.01333     0.01333     0.00000     0.01333
    73       0.01333     0.01333     0.00000     0.01333
    74       0.02649     0.02649     0.00000     0.02649
    75       0.25199     0.25199     0.00000     0.25199
    76       0.05226     0.05226     0.00000     0.05226
--------------------------------------------------------
  Mean       0.08585     0.08585     0.00000     0.08585
  s.d.       0.11129     0.11129     0.00000     0.11129
--------------------------------------------------------
<A NAME="31_01_17at14_59_44_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
    72             2       2.000       0.000           2
    73             2       2.000       0.000           2
    74             2       2.000       0.000           2
    75             2       2.000       0.000           2
    76             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_59_44_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      20.60597    20.60597     0.00000
            Theta_k_lower      14.16768    14.16768     0.00000
            Theta_k_upper      29.64007    29.64007     0.00000
                  Theta_H      23.77404    23.77404     0.00000
             s.d. Theta_H       3.21299     3.21299     0.00000
                  Theta_S      13.60906    13.60906     0.00000
             s.d. Theta_S       3.39980     3.39980     0.00000
                 Theta_pi       6.52465     6.52465     0.00000
            s.d. Theta_pi       3.43403     3.43403     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_59_44_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       7.66016     7.66016     0.00000
              Tau qt 2.5%       3.93359     3.93359     0.00000
                Tau qt 5%       3.93359     3.93359     0.00000
               Tau qt 95%       8.47461     8.47461     0.00000
             Tau qt 97.5%       8.47461     8.47461     0.00000
                   Theta0       0.00703     0.00703     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.46602     1.46602     0.00000
          Theta0 qt 97.5%       1.46602     1.46602     0.00000
                   Theta1      31.44531    31.44531     0.00000
           Theta1 qt 2.5%      12.75391    12.75391     0.00000
             Theta1 qt 5%      12.75391    12.75391     0.00000
            Theta1 qt 95%     186.44531   186.44531     0.00000
          Theta1 qt 97.5%     186.44531   186.44531     0.00000
                      SSD       0.00168     0.00168     0.00000
      Model (SSD) p-value       0.50000     0.50000     0.00000
         Raggedness index       0.00745     0.00745     0.00000
       Raggedness p-value       0.70000     0.70000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 144 (31/01/17 at 14:59:44))
Total computing time for this run : 0h 0m 0s 255 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

