
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_48_07"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 382 (31/01/17 at 14:48:07)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_48_07_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_48_07_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 42
No. of loci              : 60
No. of usable loci       : 60 loci with less than  5.00 % missing data
No. of polymorphic sites : 60


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.23266
     2       150         2   0.01333
     3       150         2   0.02649
     4       150         2   0.05226
     5       150         2   0.08957
     6       150         2   0.01333
     7       150         2   0.07732
     8       150         2   0.06488
     9       150         2   0.01333
    10       150         2   0.01333
    11       150         2   0.06488
    12       150         2   0.12528
    13       150         2   0.02649
    14       150         2   0.08957
    15       150         2   0.05226
    16       150         2   0.01333
    17       150         2   0.01333
    18       150         2   0.01333
    19       150         2   0.01333
    20       150         2   0.02649
    21       150         2   0.02649
    22       150         2   0.05226
    23       150         2   0.02649
    24       150         2   0.01333
    25       150         2   0.05226
    26       150         2   0.01333
    27       150         2   0.03946
    28       150         2   0.03946
    29       150         2   0.01333
    30       150         2   0.01333
    31       150         2   0.19186
    32       150         2   0.01333
    33       150         2   0.01333
    34       150         2   0.02649
    35       150         2   0.01333
    36       150         2   0.05226
    37       150         2   0.08957
    38       150         2   0.08957
    39       150         2   0.08957
    40       150         2   0.06488
    41       150         2   0.01333
    42       150         2   0.01333
    43       150         2   0.08957
    44       150         2   0.15937
    45       150         2   0.01333
    46       150         2   0.03946
    47       150         2   0.02649
    48       150         2   0.01333
    49       150         2   0.01333
    50       150         2   0.01333
    51       150         2   0.02649
    52       150         2   0.01333
    53       150         2   0.01333
    54       150         2   0.01333
    55       150         2   0.01333
    56       150         2   0.01333
    57       150         2   0.03946
    58       150         2   0.07732
    59       150         2   0.01333
    60       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.04334
  s.d.     0.000     0.000   0.04557
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0820
Gene diversity           :  0.9241 +/-  0.0127

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 42

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 60
Number of usable nucleotide sites            : 60


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.600268 +/-     1.399951

Average gene diversity over loci              :     0.043338 +/-     0.025827

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    10.737552
  S.D. Theta(Hom) :     2.121790

         Theta(k) :    19.021744
95 % confidence interval limits for theta(k) : [  13.002730,  27.496429 ]

         Theta(S) :    10.743997
    S.D. Theta(S) :     2.754809

        Theta(Pi) :     2.600268
   S.D. Theta(Pi) :     1.549611

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     848     859.000
       1    2239    2178.781
       2    2773    2784.462
       3    2236    2393.393
       4    1686    1558.758
       5     844     821.812
       6     328     366.072
       7     166     142.035
       8      30      49.134
       9      19      15.442
      10       6       4.481
      11       0       1.217
      12       0       0.313
      13       0       0.077
      14       0       0.018
      15       0       0.004
      16       0       0.001
      17       0       0.000
      18       0       0.000
      19       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.600
Mismatch observed variance          :        2.682
Tau                                 :        2.615
Theta0                              :        0.000
Theta1                              :      101.562

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00042877
          P(Sim. Ssd >= Obs. Ssd): 0.70000000
    Harpending's Raggedness index: 0.03067883
        P(Sim. Rag. >= Obs. Rag.): 0.60000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        848.0        791.8        897.9
          1.0       2239.0       2201.7       2163.4
          2.0       2773.0       2810.7       2777.0
          3.0       2236.0       2369.8       2403.9
          4.0       1686.0       1565.4       1566.3
          5.0        844.0        818.7        817.6
          6.0        328.0        381.5        355.9
          7.0        166.0        150.8        132.9
          8.0         30.0         58.7         43.4
          9.0         19.0         18.4         12.6
         10.0          6.0          5.1          3.3
         11.0          0.0          2.0          0.8
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.615           1.639           2.787           2.357
                   Theta0:           0.000           0.000           1.090           0.293               
                   Theta1:         101.562          44.375       99999.000       70023.948
Mean No. of pairwise diff.           2.600           2.251           3.170           2.626
           No. pol. sites:              60              83             110          97.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        848.0        435.0       1133.0        897.9
          1.0       2239.0       1597.0       2986.0       2163.4
          2.0       2773.0       2479.0       3240.0       2777.0
          3.0       2236.0       1886.0       3175.0       2403.9
          4.0       1686.0       1002.0       2136.0       1566.3
          5.0        844.0        447.0       1368.0        817.6
          6.0        328.0         84.0        765.0        355.9
          7.0        166.0          8.0        382.0        132.9
          8.0         30.0          0.0        141.0         43.4
          9.0         19.0          0.0         58.0         12.6
         10.0          6.0          0.0         28.0          3.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.615           1.639           2.787           2.357
                   Theta0:           0.000           0.000           1.090           0.293               
                   Theta1:         101.562          44.375       99999.000       70023.948
Mean No. of pairwise diff.           2.600           2.251           3.170           2.626
           No. pol. sites:              60              83             110          97.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        848.0        435.0       1133.0        897.9
          1.0       2239.0       1597.0       2986.0       2163.4
          2.0       2773.0       2479.0       3240.0       2777.0
          3.0       2236.0       1886.0       3175.0       2403.9
          4.0       1686.0       1002.0       2136.0       1566.3
          5.0        844.0        447.0       1368.0        817.6
          6.0        328.0         84.0        765.0        355.9
          7.0        166.0          8.0        382.0        132.9
          8.0         30.0          0.0        141.0         43.4
          9.0         19.0          0.0         58.0         12.6
         10.0          6.0          0.0         28.0          3.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.615           1.639           2.787           2.357
                   Theta0:           0.000           0.000           1.090           0.293               
                   Theta1:         101.562          44.375       99999.000       70023.948
Mean No. of pairwise diff.           2.600           2.251           3.170           2.626
           No. pol. sites:              60              83             110          97.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        848.0        435.0       1133.0        897.9
          1.0       2239.0       1597.0       2986.0       2163.4
          2.0       2773.0       2479.0       3240.0       2777.0
          3.0       2236.0       1886.0       3175.0       2403.9
          4.0       1686.0       1002.0       2136.0       1566.3
          5.0        844.0        447.0       1368.0        817.6
          6.0        328.0         84.0        765.0        355.9
          7.0        166.0          8.0        382.0        132.9
          8.0         30.0          0.0        141.0         43.4
          9.0         19.0          0.0         58.0         12.6
         10.0          6.0          0.0         28.0          3.3
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_48_07_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            60      60.000       0.000
  No. of usable loci            60      60.000       0.000
  No. of polym. loci            60      60.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_48_07_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.23266     0.23266     0.00000     0.23266
     2       0.01333     0.01333     0.00000     0.01333
     3       0.02649     0.02649     0.00000     0.02649
     4       0.05226     0.05226     0.00000     0.05226
     5       0.08957     0.08957     0.00000     0.08957
     6       0.01333     0.01333     0.00000     0.01333
     7       0.07732     0.07732     0.00000     0.07732
     8       0.06488     0.06488     0.00000     0.06488
     9       0.01333     0.01333     0.00000     0.01333
    10       0.01333     0.01333     0.00000     0.01333
    11       0.06488     0.06488     0.00000     0.06488
    12       0.12528     0.12528     0.00000     0.12528
    13       0.02649     0.02649     0.00000     0.02649
    14       0.08957     0.08957     0.00000     0.08957
    15       0.05226     0.05226     0.00000     0.05226
    16       0.01333     0.01333     0.00000     0.01333
    17       0.01333     0.01333     0.00000     0.01333
    18       0.01333     0.01333     0.00000     0.01333
    19       0.01333     0.01333     0.00000     0.01333
    20       0.02649     0.02649     0.00000     0.02649
    21       0.02649     0.02649     0.00000     0.02649
    22       0.05226     0.05226     0.00000     0.05226
    23       0.02649     0.02649     0.00000     0.02649
    24       0.01333     0.01333     0.00000     0.01333
    25       0.05226     0.05226     0.00000     0.05226
    26       0.01333     0.01333     0.00000     0.01333
    27       0.03946     0.03946     0.00000     0.03946
    28       0.03946     0.03946     0.00000     0.03946
    29       0.01333     0.01333     0.00000     0.01333
    30       0.01333     0.01333     0.00000     0.01333
    31       0.19186     0.19186     0.00000     0.19186
    32       0.01333     0.01333     0.00000     0.01333
    33       0.01333     0.01333     0.00000     0.01333
    34       0.02649     0.02649     0.00000     0.02649
    35       0.01333     0.01333     0.00000     0.01333
    36       0.05226     0.05226     0.00000     0.05226
    37       0.08957     0.08957     0.00000     0.08957
    38       0.08957     0.08957     0.00000     0.08957
    39       0.08957     0.08957     0.00000     0.08957
    40       0.06488     0.06488     0.00000     0.06488
    41       0.01333     0.01333     0.00000     0.01333
    42       0.01333     0.01333     0.00000     0.01333
    43       0.08957     0.08957     0.00000     0.08957
    44       0.15937     0.15937     0.00000     0.15937
    45       0.01333     0.01333     0.00000     0.01333
    46       0.03946     0.03946     0.00000     0.03946
    47       0.02649     0.02649     0.00000     0.02649
    48       0.01333     0.01333     0.00000     0.01333
    49       0.01333     0.01333     0.00000     0.01333
    50       0.01333     0.01333     0.00000     0.01333
    51       0.02649     0.02649     0.00000     0.02649
    52       0.01333     0.01333     0.00000     0.01333
    53       0.01333     0.01333     0.00000     0.01333
    54       0.01333     0.01333     0.00000     0.01333
    55       0.01333     0.01333     0.00000     0.01333
    56       0.01333     0.01333     0.00000     0.01333
    57       0.03946     0.03946     0.00000     0.03946
    58       0.07732     0.07732     0.00000     0.07732
    59       0.01333     0.01333     0.00000     0.01333
    60       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.04334     0.04334     0.00000     0.04334
  s.d.       0.04557     0.04557     0.00000     0.04557
--------------------------------------------------------
<A NAME="31_01_17at14_48_07_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_48_07_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      19.02174    19.02174     0.00000
            Theta_k_lower      13.00273    13.00273     0.00000
            Theta_k_upper      27.49643    27.49643     0.00000
                  Theta_H      10.73755    10.73755     0.00000
             s.d. Theta_H       2.12179     2.12179     0.00000
                  Theta_S      10.74400    10.74400     0.00000
             s.d. Theta_S       2.75481     2.75481     0.00000
                 Theta_pi       2.60027     2.60027     0.00000
            s.d. Theta_pi       1.54961     1.54961     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_48_07_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       2.61523     2.61523     0.00000
              Tau qt 2.5%       1.63867     1.63867     0.00000
                Tau qt 5%       1.63867     1.63867     0.00000
               Tau qt 95%       2.58398     2.58398     0.00000
             Tau qt 97.5%       2.58398     2.58398     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.72949     0.72949     0.00000
          Theta0 qt 97.5%       0.72949     0.72949     0.00000
                   Theta1     101.56250   101.56250     0.00000
           Theta1 qt 2.5%      44.37500    44.37500     0.00000
             Theta1 qt 5%      44.37500    44.37500     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00043     0.00043     0.00000
      Model (SSD) p-value       0.70000     0.70000     0.00000
         Raggedness index       0.03068     0.03068     0.00000
       Raggedness p-value       0.60000     0.60000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 382 (31/01/17 at 14:48:07))
Total computing time for this run : 0h 0m 0s 105 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

