
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_47_53"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 288 (31/01/17 at 14:47:53)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_47_53_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_47_53_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 50
No. of loci              : 73
No. of usable loci       : 73 loci with less than  5.00 % missing data
No. of polymorphic sites : 73


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.01333
     3       150         2   0.01333
     4       150         2   0.05226
     5       150         2   0.05226
     6       150         2   0.01333
     7       150         2   0.03946
     8       150         2   0.32215
     9       150         2   0.08957
    10       150         2   0.01333
    11       150         2   0.01333
    12       150         2   0.03946
    13       150         2   0.10166
    14       150         2   0.05226
    15       150         2   0.01333
    16       150         2   0.01333
    17       150         2   0.32215
    18       150         2   0.02649
    19       150         2   0.02649
    20       150         2   0.02649
    21       150         2   0.01333
    22       150         2   0.01333
    23       150         2   0.08957
    24       150         2   0.08957
    25       150         2   0.07732
    26       150         2   0.01333
    27       150         2   0.02649
    28       150         2   0.01333
    29       150         2   0.08957
    30       150         2   0.13682
    31       150         2   0.02649
    32       150         2   0.21262
    33       150         2   0.01333
    34       150         2   0.01333
    35       150         2   0.05226
    36       150         2   0.01333
    37       150         2   0.10166
    38       150         2   0.01333
    39       150         2   0.02649
    40       150         2   0.01333
    41       150         2   0.01333
    42       150         2   0.02649
    43       150         2   0.01333
    44       150         2   0.03946
    45       150         2   0.12528
    46       150         2   0.05226
    47       150         2   0.07732
    48       150         2   0.02649
    49       150         2   0.01333
    50       150         2   0.06488
    51       150         2   0.02649
    52       150         2   0.01333
    53       150         2   0.21262
    54       150         2   0.02649
    55       150         2   0.07732
    56       150         2   0.08957
    57       150         2   0.05226
    58       150         2   0.01333
    59       150         2   0.03946
    60       150         2   0.02649
    61       150         2   0.01333
    62       150         2   0.01333
    63       150         2   0.02649
    64       150         2   0.01333
    65       150         2   0.02649
    66       150         2   0.03946
    67       150         2   0.08957
    68       150         2   0.02649
    69       150         2   0.01333
    70       150         2   0.13682
    71       150         2   0.05226
    72       150         2   0.01333
    73       150         2   0.32215
-------------------------------------------------
  Mean   150.000     2.000   0.05647
  s.d.     0.000     0.000   0.07018
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0485
Gene diversity           :  0.9579 +/-  0.0072

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 50

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 73
Number of usable nucleotide sites            : 73


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.122148 +/-     2.064018

Average gene diversity over loci              :     0.056468 +/-     0.031297

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    21.051519
  S.D. Theta(Hom) :     4.021787

         Theta(k) :    25.857163
95 % confidence interval limits for theta(k) : [  18.038924,  36.741475 ]

         Theta(S) :    13.071863
    S.D. Theta(S) :     3.279015

        Theta(Pi) :     4.122148
   S.D. Theta(Pi) :     2.284668

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     471       0.000
       1     518       0.000
       2     866       0.000
       3    1892       0.000
       4    2582       0.000
       5    2444       0.000
       6    1479       0.000
       7     702       0.000
       8     187       0.000
       9      33       0.000
      10       1       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.122
Mismatch observed variance          :        3.214
Tau                                 :        4.480
Theta0                              :        0.000
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00797744
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.02799516
        P(Sim. Rag. >= Obs. Rag.): 0.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        471.0        116.2        126.7
          1.0        518.0        562.2        567.3
          2.0        866.0       1281.1       1270.7
          3.0       1892.0       1924.5       1897.7
          4.0       2582.0       2155.6       2125.6
          5.0       2444.0       1925.8       1904.7
          6.0       1479.0       1421.1       1422.3
          7.0        702.0        889.1        910.4
          8.0        187.0        485.0        509.9
          9.0         33.0        245.2        253.8
         10.0          1.0        102.4        113.7
         11.0          0.0         46.4         46.3
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.480           4.123           4.760           4.438
                   Theta0:           0.000           0.000           0.025           0.003               
                   Theta1:       99999.000          98.219       99999.000       50093.672
Mean No. of pairwise diff.           4.122           4.149           4.653           4.454
           No. pol. sites:              73             214             240         227.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        471.0         28.0        190.0        126.7
          1.0        518.0        268.0        705.0        567.3
          2.0        866.0        954.0       1615.0       1270.7
          3.0       1892.0       1736.0       2070.0       1897.7
          4.0       2582.0       1997.0       2565.0       2125.6
          5.0       2444.0       1697.0       2299.0       1904.7
          6.0       1479.0       1268.0       1605.0       1422.3
          7.0        702.0        763.0       1089.0        910.4
          8.0        187.0        355.0        587.0        509.9
          9.0         33.0        165.0        360.0        253.8
         10.0          1.0         51.0        198.0        113.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.480           4.123           4.760           4.438
                   Theta0:           0.000           0.000           0.025           0.003               
                   Theta1:       99999.000          98.219       99999.000       50093.672
Mean No. of pairwise diff.           4.122           4.149           4.653           4.454
           No. pol. sites:              73             214             240         227.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        471.0         28.0        190.0        126.7
          1.0        518.0        268.0        705.0        567.3
          2.0        866.0        954.0       1615.0       1270.7
          3.0       1892.0       1736.0       2070.0       1897.7
          4.0       2582.0       1997.0       2565.0       2125.6
          5.0       2444.0       1697.0       2299.0       1904.7
          6.0       1479.0       1268.0       1605.0       1422.3
          7.0        702.0        763.0       1089.0        910.4
          8.0        187.0        355.0        587.0        509.9
          9.0         33.0        165.0        360.0        253.8
         10.0          1.0         51.0        198.0        113.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.480           4.123           4.760           4.438
                   Theta0:           0.000           0.000           0.025           0.003               
                   Theta1:       99999.000          98.219       99999.000       50093.672
Mean No. of pairwise diff.           4.122           4.149           4.653           4.454
           No. pol. sites:              73             214             240         227.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        471.0         28.0        190.0        126.7
          1.0        518.0        268.0        705.0        567.3
          2.0        866.0        954.0       1615.0       1270.7
          3.0       1892.0       1736.0       2070.0       1897.7
          4.0       2582.0       1997.0       2565.0       2125.6
          5.0       2444.0       1697.0       2299.0       1904.7
          6.0       1479.0       1268.0       1605.0       1422.3
          7.0        702.0        763.0       1089.0        910.4
          8.0        187.0        355.0        587.0        509.9
          9.0         33.0        165.0        360.0        253.8
         10.0          1.0         51.0        198.0        113.7
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_47_53_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            73      73.000       0.000
  No. of usable loci            73      73.000       0.000
  No. of polym. loci            73      73.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_47_53_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.01333     0.01333     0.00000     0.01333
     3       0.01333     0.01333     0.00000     0.01333
     4       0.05226     0.05226     0.00000     0.05226
     5       0.05226     0.05226     0.00000     0.05226
     6       0.01333     0.01333     0.00000     0.01333
     7       0.03946     0.03946     0.00000     0.03946
     8       0.32215     0.32215     0.00000     0.32215
     9       0.08957     0.08957     0.00000     0.08957
    10       0.01333     0.01333     0.00000     0.01333
    11       0.01333     0.01333     0.00000     0.01333
    12       0.03946     0.03946     0.00000     0.03946
    13       0.10166     0.10166     0.00000     0.10166
    14       0.05226     0.05226     0.00000     0.05226
    15       0.01333     0.01333     0.00000     0.01333
    16       0.01333     0.01333     0.00000     0.01333
    17       0.32215     0.32215     0.00000     0.32215
    18       0.02649     0.02649     0.00000     0.02649
    19       0.02649     0.02649     0.00000     0.02649
    20       0.02649     0.02649     0.00000     0.02649
    21       0.01333     0.01333     0.00000     0.01333
    22       0.01333     0.01333     0.00000     0.01333
    23       0.08957     0.08957     0.00000     0.08957
    24       0.08957     0.08957     0.00000     0.08957
    25       0.07732     0.07732     0.00000     0.07732
    26       0.01333     0.01333     0.00000     0.01333
    27       0.02649     0.02649     0.00000     0.02649
    28       0.01333     0.01333     0.00000     0.01333
    29       0.08957     0.08957     0.00000     0.08957
    30       0.13682     0.13682     0.00000     0.13682
    31       0.02649     0.02649     0.00000     0.02649
    32       0.21262     0.21262     0.00000     0.21262
    33       0.01333     0.01333     0.00000     0.01333
    34       0.01333     0.01333     0.00000     0.01333
    35       0.05226     0.05226     0.00000     0.05226
    36       0.01333     0.01333     0.00000     0.01333
    37       0.10166     0.10166     0.00000     0.10166
    38       0.01333     0.01333     0.00000     0.01333
    39       0.02649     0.02649     0.00000     0.02649
    40       0.01333     0.01333     0.00000     0.01333
    41       0.01333     0.01333     0.00000     0.01333
    42       0.02649     0.02649     0.00000     0.02649
    43       0.01333     0.01333     0.00000     0.01333
    44       0.03946     0.03946     0.00000     0.03946
    45       0.12528     0.12528     0.00000     0.12528
    46       0.05226     0.05226     0.00000     0.05226
    47       0.07732     0.07732     0.00000     0.07732
    48       0.02649     0.02649     0.00000     0.02649
    49       0.01333     0.01333     0.00000     0.01333
    50       0.06488     0.06488     0.00000     0.06488
    51       0.02649     0.02649     0.00000     0.02649
    52       0.01333     0.01333     0.00000     0.01333
    53       0.21262     0.21262     0.00000     0.21262
    54       0.02649     0.02649     0.00000     0.02649
    55       0.07732     0.07732     0.00000     0.07732
    56       0.08957     0.08957     0.00000     0.08957
    57       0.05226     0.05226     0.00000     0.05226
    58       0.01333     0.01333     0.00000     0.01333
    59       0.03946     0.03946     0.00000     0.03946
    60       0.02649     0.02649     0.00000     0.02649
    61       0.01333     0.01333     0.00000     0.01333
    62       0.01333     0.01333     0.00000     0.01333
    63       0.02649     0.02649     0.00000     0.02649
    64       0.01333     0.01333     0.00000     0.01333
    65       0.02649     0.02649     0.00000     0.02649
    66       0.03946     0.03946     0.00000     0.03946
    67       0.08957     0.08957     0.00000     0.08957
    68       0.02649     0.02649     0.00000     0.02649
    69       0.01333     0.01333     0.00000     0.01333
    70       0.13682     0.13682     0.00000     0.13682
    71       0.05226     0.05226     0.00000     0.05226
    72       0.01333     0.01333     0.00000     0.01333
    73       0.32215     0.32215     0.00000     0.32215
--------------------------------------------------------
  Mean       0.05647     0.05647     0.00000     0.05647
  s.d.       0.07018     0.07018     0.00000     0.07018
--------------------------------------------------------
<A NAME="31_01_17at14_47_53_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
    72             2       2.000       0.000           2
    73             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_47_53_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      25.85716    25.85716     0.00000
            Theta_k_lower      18.03892    18.03892     0.00000
            Theta_k_upper      36.74148    36.74148     0.00000
                  Theta_H      21.05152    21.05152     0.00000
             s.d. Theta_H       4.02179     4.02179     0.00000
                  Theta_S      13.07186    13.07186     0.00000
             s.d. Theta_S       3.27901     3.27901     0.00000
                 Theta_pi       4.12215     4.12215     0.00000
            s.d. Theta_pi       2.28467     2.28467     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_47_53_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.48047     4.48047     0.00000
              Tau qt 2.5%       4.12305     4.12305     0.00000
                Tau qt 5%       4.12305     4.12305     0.00000
               Tau qt 95%       4.67773     4.67773     0.00000
             Tau qt 97.5%       4.67773     4.67773     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00176     0.00176     0.00000
          Theta0 qt 97.5%       0.00176     0.00176     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      98.21875    98.21875     0.00000
             Theta1 qt 5%      98.21875    98.21875     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00798     0.00798     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.02800     0.02800     0.00000
       Raggedness p-value       0.00000     0.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 288 (31/01/17 at 14:47:54))
Total computing time for this run : 0h 0m 0s 107 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

