
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_29_35"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 138 (31/01/17 at 16:29:35)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_29_35_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_29_35_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 48
No. of loci              : 56
No. of usable loci       : 56 loci with less than  5.00 % missing data
No. of polymorphic sites : 56


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.03946
     2       150         2   0.01333
     3       150         2   0.01333
     4       150         2   0.08957
     5       150         2   0.01333
     6       150         2   0.02649
     7       150         2   0.01333
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.01333
    11       150         2   0.01333
    12       150         2   0.01333
    13       150         2   0.01333
    14       150         2   0.02649
    15       150         2   0.01333
    16       150         2   0.01333
    17       150         2   0.03946
    18       150         2   0.01333
    19       150         2   0.01333
    20       150         2   0.01333
    21       150         2   0.01333
    22       150         2   0.01333
    23       150         2   0.01333
    24       150         2   0.03946
    25       150         2   0.01333
    26       150         2   0.01333
    27       150         2   0.01333
    28       150         2   0.01333
    29       150         2   0.02649
    30       150         2   0.01333
    31       150         2   0.02649
    32       150         2   0.01333
    33       150         2   0.01333
    34       150         2   0.01333
    35       150         2   0.02649
    36       150         2   0.01333
    37       150         2   0.01333
    38       150         2   0.01333
    39       150         2   0.02649
    40       150         2   0.02649
    41       150         2   0.01333
    42       150         2   0.01333
    43       150         2   0.01333
    44       150         2   0.01333
    45       150         2   0.02649
    46       150         2   0.01333
    47       150         2   0.01333
    48       150         2   0.03946
    49       150         2   0.02649
    50       150         2   0.02649
    51       150         2   0.02649
    52       150         2   0.01333
    53       150         2   0.01333
    54       150         2   0.02649
    55       150         2   0.01333
    56       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.01938
  s.d.     0.000     0.000   0.01248
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.3121
Gene diversity           :  0.6925 +/-  0.0439

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 48

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 56
Number of usable nucleotide sites            : 56


Distance method : Pairwise difference
Mean number of pairwise differences           :     1.085280 +/-     0.718151

Average gene diversity over loci              :     0.019380 +/-     0.014195

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     1.719546
  S.D. Theta(Hom) :     0.371867

         Theta(k) :    24.019287
95 % confidence interval limits for theta(k) : [  16.682825,  34.256022 ]

         Theta(S) :    10.027730
    S.D. Theta(S) :     2.593178

        Theta(Pi) :     1.085280
   S.D. Theta(Pi) :     0.794924

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    3436       0.000
       1    4492       0.000
       2    2313       0.000
       3     750       0.000
       4     161       0.000
       5      22       0.000
       6       1       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        1.085
Mismatch observed variance          :        0.939
Tau                                 :        1.133
Theta0                              :        0.007
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00176780
          P(Sim. Ssd >= Obs. Ssd): 0.20000000
    Harpending's Raggedness index: 0.06944896
        P(Sim. Rag. >= Obs. Rag.): 0.10000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       3436.0       3619.4       3574.7
          1.0       4492.0       4054.0       4074.3
          2.0       2313.0       2292.9       2322.0
          3.0        750.0        891.1        882.3
          4.0        161.0        253.4        251.4
          5.0         22.0         56.7         57.3
          6.0          1.0          6.5         10.9
          7.0          0.0          1.0          1.8
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.133           1.008           1.715           1.170
                   Theta0:           0.007           0.000           0.174           0.018               
                   Theta1:       99999.000           2.340       99999.000       60023.646
Mean No. of pairwise diff.           1.085           1.039           1.266           1.133
           No. pol. sites:              56              71              84          77.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       3436.0       3081.0       4095.0       3574.7
          1.0       4492.0       3185.0       4484.0       4074.3
          2.0       2313.0       1892.0       2509.0       2322.0
          3.0        750.0        692.0       1060.0        882.3
          4.0        161.0        119.0        403.0        251.4
          5.0         22.0         12.0        117.0         57.3
          6.0          1.0          0.0         21.0         10.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.133           1.008           1.715           1.170
                   Theta0:           0.007           0.000           0.174           0.018               
                   Theta1:       99999.000           2.340       99999.000       60023.646
Mean No. of pairwise diff.           1.085           1.039           1.266           1.133
           No. pol. sites:              56              71              84          77.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       3436.0       3081.0       4095.0       3574.7
          1.0       4492.0       3185.0       4484.0       4074.3
          2.0       2313.0       1892.0       2509.0       2322.0
          3.0        750.0        692.0       1060.0        882.3
          4.0        161.0        119.0        403.0        251.4
          5.0         22.0         12.0        117.0         57.3
          6.0          1.0          0.0         21.0         10.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.133           1.008           1.715           1.170
                   Theta0:           0.007           0.000           0.174           0.018               
                   Theta1:       99999.000           2.340       99999.000       60023.646
Mean No. of pairwise diff.           1.085           1.039           1.266           1.133
           No. pol. sites:              56              71              84          77.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       3436.0       3081.0       4095.0       3574.7
          1.0       4492.0       3185.0       4484.0       4074.3
          2.0       2313.0       1892.0       2509.0       2322.0
          3.0        750.0        692.0       1060.0        882.3
          4.0        161.0        119.0        403.0        251.4
          5.0         22.0         12.0        117.0         57.3
          6.0          1.0          0.0         21.0         10.9
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_29_35_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            56      56.000       0.000
  No. of usable loci            56      56.000       0.000
  No. of polym. loci            56      56.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_29_35_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.03946     0.03946     0.00000     0.03946
     2       0.01333     0.01333     0.00000     0.01333
     3       0.01333     0.01333     0.00000     0.01333
     4       0.08957     0.08957     0.00000     0.08957
     5       0.01333     0.01333     0.00000     0.01333
     6       0.02649     0.02649     0.00000     0.02649
     7       0.01333     0.01333     0.00000     0.01333
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.01333     0.01333     0.00000     0.01333
    11       0.01333     0.01333     0.00000     0.01333
    12       0.01333     0.01333     0.00000     0.01333
    13       0.01333     0.01333     0.00000     0.01333
    14       0.02649     0.02649     0.00000     0.02649
    15       0.01333     0.01333     0.00000     0.01333
    16       0.01333     0.01333     0.00000     0.01333
    17       0.03946     0.03946     0.00000     0.03946
    18       0.01333     0.01333     0.00000     0.01333
    19       0.01333     0.01333     0.00000     0.01333
    20       0.01333     0.01333     0.00000     0.01333
    21       0.01333     0.01333     0.00000     0.01333
    22       0.01333     0.01333     0.00000     0.01333
    23       0.01333     0.01333     0.00000     0.01333
    24       0.03946     0.03946     0.00000     0.03946
    25       0.01333     0.01333     0.00000     0.01333
    26       0.01333     0.01333     0.00000     0.01333
    27       0.01333     0.01333     0.00000     0.01333
    28       0.01333     0.01333     0.00000     0.01333
    29       0.02649     0.02649     0.00000     0.02649
    30       0.01333     0.01333     0.00000     0.01333
    31       0.02649     0.02649     0.00000     0.02649
    32       0.01333     0.01333     0.00000     0.01333
    33       0.01333     0.01333     0.00000     0.01333
    34       0.01333     0.01333     0.00000     0.01333
    35       0.02649     0.02649     0.00000     0.02649
    36       0.01333     0.01333     0.00000     0.01333
    37       0.01333     0.01333     0.00000     0.01333
    38       0.01333     0.01333     0.00000     0.01333
    39       0.02649     0.02649     0.00000     0.02649
    40       0.02649     0.02649     0.00000     0.02649
    41       0.01333     0.01333     0.00000     0.01333
    42       0.01333     0.01333     0.00000     0.01333
    43       0.01333     0.01333     0.00000     0.01333
    44       0.01333     0.01333     0.00000     0.01333
    45       0.02649     0.02649     0.00000     0.02649
    46       0.01333     0.01333     0.00000     0.01333
    47       0.01333     0.01333     0.00000     0.01333
    48       0.03946     0.03946     0.00000     0.03946
    49       0.02649     0.02649     0.00000     0.02649
    50       0.02649     0.02649     0.00000     0.02649
    51       0.02649     0.02649     0.00000     0.02649
    52       0.01333     0.01333     0.00000     0.01333
    53       0.01333     0.01333     0.00000     0.01333
    54       0.02649     0.02649     0.00000     0.02649
    55       0.01333     0.01333     0.00000     0.01333
    56       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.01938     0.01938     0.00000     0.01938
  s.d.       0.01248     0.01248     0.00000     0.01248
--------------------------------------------------------
<A NAME="31_01_17at16_29_35_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_29_35_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      24.01929    24.01929     0.00000
            Theta_k_lower      16.68282    16.68282     0.00000
            Theta_k_upper      34.25602    34.25602     0.00000
                  Theta_H       1.71955     1.71955     0.00000
             s.d. Theta_H       0.37187     0.37187     0.00000
                  Theta_S      10.02773    10.02773     0.00000
             s.d. Theta_S       2.59318     2.59318     0.00000
                 Theta_pi       1.08528     1.08528     0.00000
            s.d. Theta_pi       0.79492     0.79492     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_29_35_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.13281     1.13281     0.00000
              Tau qt 2.5%       1.00781     1.00781     0.00000
                Tau qt 5%       1.00781     1.00781     0.00000
               Tau qt 95%       1.23828     1.23828     0.00000
             Tau qt 97.5%       1.23828     1.23828     0.00000
                   Theta0       0.00703     0.00703     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00703     0.00703     0.00000
          Theta0 qt 97.5%       0.00703     0.00703     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%       2.33984     2.33984     0.00000
             Theta1 qt 5%       2.33984     2.33984     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00177     0.00177     0.00000
      Model (SSD) p-value       0.20000     0.20000     0.00000
         Raggedness index       0.06945     0.06945     0.00000
       Raggedness p-value       0.10000     0.10000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 138 (31/01/17 at 16:29:35))
Total computing time for this run : 0h 0m 0s 106 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

