
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_47_56"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 303 (31/01/17 at 14:47:56)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_47_56_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_47_56_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 43
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.11356
     2       150         2   0.37414
     3       150         2   0.27060
     4       150         2   0.01333
     5       150         2   0.03946
     6       150         2   0.20233
     7       150         2   0.06488
     8       150         2   0.02649
     9       150         2   0.07732
    10       150         2   0.36018
    11       150         2   0.26139
    12       150         2   0.01333
    13       150         2   0.08957
    14       150         2   0.27964
    15       150         2   0.01333
    16       150         2   0.22273
    17       150         2   0.08957
    18       150         2   0.01333
    19       150         2   0.36725
    20       150         2   0.28850
-------------------------------------------------
  Mean   150.000     2.000   0.15905
  s.d.     0.000     0.000   0.13264
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0735
Gene diversity           :  0.9327 +/-  0.0118

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 43

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.180940 +/-     1.654410

Average gene diversity over loci              :     0.159047 +/-     0.091564

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    12.362977
  S.D. Theta(Hom) :     2.527538

         Theta(k) :    19.804082
95 % confidence interval limits for theta(k) : [  13.577772,  28.555190 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     3.180940
   S.D. Theta(Pi) :     1.831272

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     752     495.501
       1    1561    1518.217
       2    1862    2346.526
       3    2120    2442.661
       4    2225    1929.859
       5    1488    1236.831
       6     725     671.403
       7     341     318.423
       8      85     135.133
       9      15      52.328
      10       1      18.799
      11       0       6.357
      12       0       2.050
      13       0       0.638
      14       0       0.193
      15       0       0.057
      16       0       0.017
      17       0       0.005
      18       0       0.001
      19       0       0.000
      20       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.181
Mismatch observed variance          :        3.345
Tau                                 :        3.619
Theta0                              :        0.000
Theta1                              :       20.234

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00157349
          P(Sim. Ssd >= Obs. Ssd): 0.20000000
    Harpending's Raggedness index: 0.01734535
        P(Sim. Rag. >= Obs. Rag.): 0.70000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        752.0        778.3        765.0
          1.0       1561.0       1485.4       1354.1
          2.0       1862.0       2137.5       1989.7
          3.0       2120.0       2207.8       2218.6
          4.0       2225.0       1884.4       1934.6
          5.0       1488.0       1323.7       1372.3
          6.0        725.0        737.3        817.5
          7.0        341.0        378.4        419.2
          8.0         85.0        156.6        188.6
          9.0         15.0         62.2         75.5
         10.0          1.0         19.6         27.2
         11.0          0.0          2.2          8.9
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.619           0.785           4.004           2.992
                   Theta0:           0.000           0.000           1.238           0.274               
                   Theta1:          20.234          15.225       99999.000       30710.265
Mean No. of pairwise diff.           3.181           2.674           3.856           3.191
           No. pol. sites:              20              42              64          50.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        752.0        501.0       1610.0        765.0
          1.0       1561.0       1027.0       2566.0       1354.1
          2.0       1862.0       1596.0       2726.0       1989.7
          3.0       2120.0       1773.0       2850.0       2218.6
          4.0       2225.0       1409.0       2149.0       1934.6
          5.0       1488.0        702.0       1649.0       1372.3
          6.0        725.0        408.0       1232.0        817.5
          7.0        341.0        108.0        684.0        419.2
          8.0         85.0          8.0        384.0        188.6
          9.0         15.0          0.0        164.0         75.5
         10.0          1.0          0.0         63.0         27.2
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.619           0.785           4.004           2.992
                   Theta0:           0.000           0.000           1.238           0.274               
                   Theta1:          20.234          15.225       99999.000       30710.265
Mean No. of pairwise diff.           3.181           2.674           3.856           3.191
           No. pol. sites:              20              42              64          50.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        752.0        501.0       1610.0        765.0
          1.0       1561.0       1027.0       2566.0       1354.1
          2.0       1862.0       1596.0       2726.0       1989.7
          3.0       2120.0       1773.0       2850.0       2218.6
          4.0       2225.0       1409.0       2149.0       1934.6
          5.0       1488.0        702.0       1649.0       1372.3
          6.0        725.0        408.0       1232.0        817.5
          7.0        341.0        108.0        684.0        419.2
          8.0         85.0          8.0        384.0        188.6
          9.0         15.0          0.0        164.0         75.5
         10.0          1.0          0.0         63.0         27.2
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.619           0.785           4.004           2.992
                   Theta0:           0.000           0.000           1.238           0.274               
                   Theta1:          20.234          15.225       99999.000       30710.265
Mean No. of pairwise diff.           3.181           2.674           3.856           3.191
           No. pol. sites:              20              42              64          50.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        752.0        501.0       1610.0        765.0
          1.0       1561.0       1027.0       2566.0       1354.1
          2.0       1862.0       1596.0       2726.0       1989.7
          3.0       2120.0       1773.0       2850.0       2218.6
          4.0       2225.0       1409.0       2149.0       1934.6
          5.0       1488.0        702.0       1649.0       1372.3
          6.0        725.0        408.0       1232.0        817.5
          7.0        341.0        108.0        684.0        419.2
          8.0         85.0          8.0        384.0        188.6
          9.0         15.0          0.0        164.0         75.5
         10.0          1.0          0.0         63.0         27.2
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_47_56_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_47_56_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.11356     0.11356     0.00000     0.11356
     2       0.37414     0.37414     0.00000     0.37414
     3       0.27060     0.27060     0.00000     0.27060
     4       0.01333     0.01333     0.00000     0.01333
     5       0.03946     0.03946     0.00000     0.03946
     6       0.20233     0.20233     0.00000     0.20233
     7       0.06488     0.06488     0.00000     0.06488
     8       0.02649     0.02649     0.00000     0.02649
     9       0.07732     0.07732     0.00000     0.07732
    10       0.36018     0.36018     0.00000     0.36018
    11       0.26139     0.26139     0.00000     0.26139
    12       0.01333     0.01333     0.00000     0.01333
    13       0.08957     0.08957     0.00000     0.08957
    14       0.27964     0.27964     0.00000     0.27964
    15       0.01333     0.01333     0.00000     0.01333
    16       0.22273     0.22273     0.00000     0.22273
    17       0.08957     0.08957     0.00000     0.08957
    18       0.01333     0.01333     0.00000     0.01333
    19       0.36725     0.36725     0.00000     0.36725
    20       0.28850     0.28850     0.00000     0.28850
--------------------------------------------------------
  Mean       0.15905     0.15905     0.00000     0.15905
  s.d.       0.13264     0.13264     0.00000     0.13264
--------------------------------------------------------
<A NAME="31_01_17at14_47_56_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_47_56_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      19.80408    19.80408     0.00000
            Theta_k_lower      13.57777    13.57777     0.00000
            Theta_k_upper      28.55519    28.55519     0.00000
                  Theta_H      12.36298    12.36298     0.00000
             s.d. Theta_H       2.52754     2.52754     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       3.18094     3.18094     0.00000
            s.d. Theta_pi       1.83127     1.83127     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_47_56_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.61914     3.61914     0.00000
              Tau qt 2.5%       0.78516     0.78516     0.00000
                Tau qt 5%       0.78516     0.78516     0.00000
               Tau qt 95%       3.54102     3.54102     0.00000
             Tau qt 97.5%       3.54102     3.54102     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.07578     1.07578     0.00000
          Theta0 qt 97.5%       1.07578     1.07578     0.00000
                   Theta1      20.23438    20.23438     0.00000
           Theta1 qt 2.5%      15.22461    15.22461     0.00000
             Theta1 qt 5%      15.22461    15.22461     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00157     0.00157     0.00000
      Model (SSD) p-value       0.20000     0.20000     0.00000
         Raggedness index       0.01735     0.01735     0.00000
       Raggedness p-value       0.70000     0.70000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 303 (31/01/17 at 14:47:56))
Total computing time for this run : 0h 0m 0s 127 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

