
<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_42_42"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 220 (31/01/17 at 14:42:42)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_42_42_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_42_42_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 35
No. of loci              : 39
No. of usable loci       : 39 loci with less than  5.00 % missing data
No. of polymorphic sites : 39


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.02649
     3       150         2   0.02649
     4       150         2   0.03946
     5       150         2   0.01333
     6       150         2   0.01333
     7       150         2   0.02649
     8       150         2   0.01333
     9       150         2   0.02649
    10       150         2   0.05226
    11       150         2   0.03946
    12       150         2   0.02649
    13       150         2   0.03946
    14       150         2   0.01333
    15       150         2   0.01333
    16       150         2   0.01333
    17       150         2   0.03946
    18       150         2   0.02649
    19       150         2   0.01333
    20       150         2   0.01333
    21       150         2   0.03946
    22       150         2   0.01333
    23       150         2   0.01333
    24       150         2   0.05226
    25       150         2   0.02649
    26       150         2   0.05226
    27       150         2   0.01333
    28       150         2   0.02649
    29       150         2   0.03946
    30       150         2   0.03946
    31       150         2   0.02649
    32       150         2   0.01333
    33       150         2   0.05226
    34       150         2   0.01333
    35       150         2   0.13682
    36       150         2   0.02649
    37       150         2   0.01333
    38       150         2   0.07732
    39       150         2   0.03946
-------------------------------------------------
  Mean   150.000     2.000   0.03087
  s.d.     0.000     0.000   0.02329
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.2548
Gene diversity           :  0.7502 +/-  0.0386

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 35

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 39
Number of usable nucleotide sites            : 39


Distance method : Pairwise difference
Mean number of pairwise differences           :     1.203758 +/-     0.773474

Average gene diversity over loci              :     0.030866 +/-     0.021953

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     2.330105
  S.D. Theta(Hom) :     0.511008

         Theta(k) :    14.055440
95 % confidence interval limits for theta(k) : [   9.368737,  20.760299 ]

         Theta(S) :     6.983598
    S.D. Theta(S) :     1.903203

        Theta(Pi) :     1.203758
   S.D. Theta(Pi) :     0.856161

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    2792       0.000
       1    4542       0.000
       2    2784       0.000
       3     900       0.000
       4     143       0.000
       5      14       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        1.204
Mismatch observed variance          :        0.915
Tau                                 :        1.303
Theta0                              :        0.000
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00439700
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.08241796
        P(Sim. Rag. >= Obs. Rag.): 0.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       2792.0       2784.2       3037.3
          1.0       4542.0       4079.5       3956.7
          2.0       2784.0       2622.3       2577.2
          3.0        900.0       1196.6       1119.2
          4.0        143.0        389.0        364.5
          5.0         14.0         87.7         95.0
          6.0          0.0         14.4         20.6
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.303           1.055           1.498           1.337
                   Theta0:           0.000           0.000           0.033           0.007               
                   Theta1:       99999.000          23.688       99999.000       90001.469
Mean No. of pairwise diff.           1.204           1.173           1.513           1.343
           No. pol. sites:              39              77              98          86.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2792.0       2316.0       3486.0       3037.3
          1.0       4542.0       3576.0       4409.0       3956.7
          2.0       2784.0       2184.0       2888.0       2577.2
          3.0        900.0        923.0       1534.0       1119.2
          4.0        143.0        274.0        570.0        364.5
          5.0         14.0         42.0        163.0         95.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.303           1.055           1.498           1.337
                   Theta0:           0.000           0.000           0.033           0.007               
                   Theta1:       99999.000          23.688       99999.000       90001.469
Mean No. of pairwise diff.           1.204           1.173           1.513           1.343
           No. pol. sites:              39              77              98          86.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2792.0       2316.0       3486.0       3037.3
          1.0       4542.0       3576.0       4409.0       3956.7
          2.0       2784.0       2184.0       2888.0       2577.2
          3.0        900.0        923.0       1534.0       1119.2
          4.0        143.0        274.0        570.0        364.5
          5.0         14.0         42.0        163.0         95.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.303           1.055           1.498           1.337
                   Theta0:           0.000           0.000           0.033           0.007               
                   Theta1:       99999.000          23.688       99999.000       90001.469
Mean No. of pairwise diff.           1.204           1.173           1.513           1.343
           No. pol. sites:              39              77              98          86.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2792.0       2316.0       3486.0       3037.3
          1.0       4542.0       3576.0       4409.0       3956.7
          2.0       2784.0       2184.0       2888.0       2577.2
          3.0        900.0        923.0       1534.0       1119.2
          4.0        143.0        274.0        570.0        364.5
          5.0         14.0         42.0        163.0         95.0
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_42_42_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            39      39.000       0.000
  No. of usable loci            39      39.000       0.000
  No. of polym. loci            39      39.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_42_42_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.02649     0.02649     0.00000     0.02649
     3       0.02649     0.02649     0.00000     0.02649
     4       0.03946     0.03946     0.00000     0.03946
     5       0.01333     0.01333     0.00000     0.01333
     6       0.01333     0.01333     0.00000     0.01333
     7       0.02649     0.02649     0.00000     0.02649
     8       0.01333     0.01333     0.00000     0.01333
     9       0.02649     0.02649     0.00000     0.02649
    10       0.05226     0.05226     0.00000     0.05226
    11       0.03946     0.03946     0.00000     0.03946
    12       0.02649     0.02649     0.00000     0.02649
    13       0.03946     0.03946     0.00000     0.03946
    14       0.01333     0.01333     0.00000     0.01333
    15       0.01333     0.01333     0.00000     0.01333
    16       0.01333     0.01333     0.00000     0.01333
    17       0.03946     0.03946     0.00000     0.03946
    18       0.02649     0.02649     0.00000     0.02649
    19       0.01333     0.01333     0.00000     0.01333
    20       0.01333     0.01333     0.00000     0.01333
    21       0.03946     0.03946     0.00000     0.03946
    22       0.01333     0.01333     0.00000     0.01333
    23       0.01333     0.01333     0.00000     0.01333
    24       0.05226     0.05226     0.00000     0.05226
    25       0.02649     0.02649     0.00000     0.02649
    26       0.05226     0.05226     0.00000     0.05226
    27       0.01333     0.01333     0.00000     0.01333
    28       0.02649     0.02649     0.00000     0.02649
    29       0.03946     0.03946     0.00000     0.03946
    30       0.03946     0.03946     0.00000     0.03946
    31       0.02649     0.02649     0.00000     0.02649
    32       0.01333     0.01333     0.00000     0.01333
    33       0.05226     0.05226     0.00000     0.05226
    34       0.01333     0.01333     0.00000     0.01333
    35       0.13682     0.13682     0.00000     0.13682
    36       0.02649     0.02649     0.00000     0.02649
    37       0.01333     0.01333     0.00000     0.01333
    38       0.07732     0.07732     0.00000     0.07732
    39       0.03946     0.03946     0.00000     0.03946
--------------------------------------------------------
  Mean       0.03087     0.03087     0.00000     0.03087
  s.d.       0.02329     0.02329     0.00000     0.02329
--------------------------------------------------------
<A NAME="31_01_17at14_42_42_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_42_42_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      14.05544    14.05544     0.00000
            Theta_k_lower       9.36874     9.36874     0.00000
            Theta_k_upper      20.76030    20.76030     0.00000
                  Theta_H       2.33010     2.33010     0.00000
             s.d. Theta_H       0.51101     0.51101     0.00000
                  Theta_S       6.98360     6.98360     0.00000
             s.d. Theta_S       1.90320     1.90320     0.00000
                 Theta_pi       1.20376     1.20376     0.00000
            s.d. Theta_pi       0.85616     0.85616     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_42_42_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.30273     1.30273     0.00000
              Tau qt 2.5%       1.05469     1.05469     0.00000
                Tau qt 5%       1.05469     1.05469     0.00000
               Tau qt 95%       1.42383     1.42383     0.00000
             Tau qt 97.5%       1.42383     1.42383     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.01406     0.01406     0.00000
          Theta0 qt 97.5%       0.01406     0.01406     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      23.68750    23.68750     0.00000
             Theta1 qt 5%      23.68750    23.68750     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00440     0.00440     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.08242     0.08242     0.00000
       Raggedness p-value       0.00000     0.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 220 (31/01/17 at 14:42:42))
Total computing time for this run : 0h 0m 0s 73 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

