
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_49_49"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 227 (31/01/17 at 14:49:49)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_49_49_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_49_49_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 39
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.13682
     2       150         2   0.19186
     3       150         2   0.10166
     4       150         2   0.01333
     5       150         2   0.15937
     6       150         2   0.08957
     7       150         2   0.01333
     8       150         2   0.19186
     9       150         2   0.34550
    10       150         2   0.08957
    11       150         2   0.05226
    12       150         2   0.05226
    13       150         2   0.28850
    14       150         2   0.21262
    15       150         2   0.06488
    16       150         2   0.37414
    17       150         2   0.06488
    18       150         2   0.42810
    19       150         2   0.07732
    20       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.14806
  s.d.     0.000     0.000   0.12506
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0555
Gene diversity           :  0.9509 +/-  0.0066

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 39

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.961163 +/-     1.558310

Average gene diversity over loci              :     0.148058 +/-     0.086245

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    17.731303
  S.D. Theta(Hom) :     2.695807

         Theta(k) :    16.787407
95 % confidence interval limits for theta(k) : [  11.363701,  24.469906 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     2.961163
   S.D. Theta(Pi) :     1.724898

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     549     689.006
       1    1801    1835.536
       2    3069    2504.704
       3    2180    2346.885
       4    1393    1709.939
       5     999    1041.392
       6     614     557.025
       7     281     271.567
       8     228     124.214
       9      40      54.504
      10      21      23.321
      11       0       9.839
      12       0       4.121
      13       0       1.720
      14       0       0.717
      15       0       0.299
      16       0       0.124
      17       0       0.052
      18       0       0.022
      19       0       0.009
      20       0       0.004


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.961
Mismatch observed variance          :        3.470
Tau                                 :        2.734
Theta0                              :        0.000
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00362094
          P(Sim. Ssd >= Obs. Ssd): 0.10000000
    Harpending's Raggedness index: 0.04034511
        P(Sim. Rag. >= Obs. Rag.): 0.20000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        549.0        665.1        725.7
          1.0       1801.0       1905.0       1984.2
          2.0       3069.0       2748.6       2712.7
          3.0       2180.0       2540.7       2472.5
          4.0       1393.0       1716.8       1690.2
          5.0        999.0        905.3        924.3
          6.0        614.0        429.1        421.2
          7.0        281.0        174.2        164.5
          8.0        228.0         62.9         56.2
          9.0         40.0         20.9         17.1
         10.0         21.0          5.5          4.7
         11.0          0.0          0.8          1.2
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.734           2.537           3.209           2.727
                   Theta0:           0.000           0.000           0.199           0.028               
                   Theta1:       99999.000          27.398       99999.000       90001.840
Mean No. of pairwise diff.           2.961           2.556           3.180           2.771
           No. pol. sites:              20             151             179         163.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        549.0        253.0       1035.0        725.7
          1.0       1801.0       1176.0       2206.0       1984.2
          2.0       3069.0       2506.0       3027.0       2712.7
          3.0       2180.0       2351.0       2984.0       2472.5
          4.0       1393.0       1513.0       2188.0       1690.2
          5.0        999.0        613.0       1138.0        924.3
          6.0        614.0        232.0        647.0        421.2
          7.0        281.0        105.0        263.0        164.5
          8.0        228.0         30.0         86.0         56.2
          9.0         40.0          6.0         36.0         17.1
         10.0         21.0          0.0         15.0          4.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.734           2.537           3.209           2.727
                   Theta0:           0.000           0.000           0.199           0.028               
                   Theta1:       99999.000          27.398       99999.000       90001.840
Mean No. of pairwise diff.           2.961           2.556           3.180           2.771
           No. pol. sites:              20             151             179         163.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        549.0        253.0       1035.0        725.7
          1.0       1801.0       1176.0       2206.0       1984.2
          2.0       3069.0       2506.0       3027.0       2712.7
          3.0       2180.0       2351.0       2984.0       2472.5
          4.0       1393.0       1513.0       2188.0       1690.2
          5.0        999.0        613.0       1138.0        924.3
          6.0        614.0        232.0        647.0        421.2
          7.0        281.0        105.0        263.0        164.5
          8.0        228.0         30.0         86.0         56.2
          9.0         40.0          6.0         36.0         17.1
         10.0         21.0          0.0         15.0          4.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.734           2.537           3.209           2.727
                   Theta0:           0.000           0.000           0.199           0.028               
                   Theta1:       99999.000          27.398       99999.000       90001.840
Mean No. of pairwise diff.           2.961           2.556           3.180           2.771
           No. pol. sites:              20             151             179         163.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        549.0        253.0       1035.0        725.7
          1.0       1801.0       1176.0       2206.0       1984.2
          2.0       3069.0       2506.0       3027.0       2712.7
          3.0       2180.0       2351.0       2984.0       2472.5
          4.0       1393.0       1513.0       2188.0       1690.2
          5.0        999.0        613.0       1138.0        924.3
          6.0        614.0        232.0        647.0        421.2
          7.0        281.0        105.0        263.0        164.5
          8.0        228.0         30.0         86.0         56.2
          9.0         40.0          6.0         36.0         17.1
         10.0         21.0          0.0         15.0          4.7
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_49_49_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_49_49_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.13682     0.13682     0.00000     0.13682
     2       0.19186     0.19186     0.00000     0.19186
     3       0.10166     0.10166     0.00000     0.10166
     4       0.01333     0.01333     0.00000     0.01333
     5       0.15937     0.15937     0.00000     0.15937
     6       0.08957     0.08957     0.00000     0.08957
     7       0.01333     0.01333     0.00000     0.01333
     8       0.19186     0.19186     0.00000     0.19186
     9       0.34550     0.34550     0.00000     0.34550
    10       0.08957     0.08957     0.00000     0.08957
    11       0.05226     0.05226     0.00000     0.05226
    12       0.05226     0.05226     0.00000     0.05226
    13       0.28850     0.28850     0.00000     0.28850
    14       0.21262     0.21262     0.00000     0.21262
    15       0.06488     0.06488     0.00000     0.06488
    16       0.37414     0.37414     0.00000     0.37414
    17       0.06488     0.06488     0.00000     0.06488
    18       0.42810     0.42810     0.00000     0.42810
    19       0.07732     0.07732     0.00000     0.07732
    20       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.14806     0.14806     0.00000     0.14806
  s.d.       0.12506     0.12506     0.00000     0.12506
--------------------------------------------------------
<A NAME="31_01_17at14_49_49_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_49_49_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      16.78741    16.78741     0.00000
            Theta_k_lower      11.36370    11.36370     0.00000
            Theta_k_upper      24.46991    24.46991     0.00000
                  Theta_H      17.73130    17.73130     0.00000
             s.d. Theta_H       2.69581     2.69581     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       2.96116     2.96116     0.00000
            s.d. Theta_pi       1.72490     1.72490     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_49_49_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       2.73438     2.73438     0.00000
              Tau qt 2.5%       2.53711     2.53711     0.00000
                Tau qt 5%       2.53711     2.53711     0.00000
               Tau qt 95%       2.89648     2.89648     0.00000
             Tau qt 97.5%       2.89648     2.89648     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.02461     0.02461     0.00000
          Theta0 qt 97.5%       0.02461     0.02461     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      27.39844    27.39844     0.00000
             Theta1 qt 5%      27.39844    27.39844     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00362     0.00362     0.00000
      Model (SSD) p-value       0.10000     0.10000     0.00000
         Raggedness index       0.04035     0.04035     0.00000
       Raggedness p-value       0.20000     0.20000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 227 (31/01/17 at 14:49:49))
Total computing time for this run : 0h 0m 0s 106 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

