
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_49_54"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 255 (31/01/17 at 14:49:54)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_49_54_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_49_54_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 49
No. of loci              : 21
No. of usable loci       : 21 loci with less than  5.00 % missing data
No. of polymorphic sites : 21


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.11356
     2       150         2   0.18121
     3       150         2   0.03946
     4       150         2   0.11356
     5       150         2   0.30568
     6       150         2   0.23266
     7       150         2   0.05226
     8       150         2   0.13682
     9       150         2   0.32215
    10       150         2   0.13682
    11       150         2   0.12528
    12       150         2   0.15937
    13       150         2   0.20233
    14       150         2   0.02649
    15       150         2   0.05226
    16       150         2   0.36725
    17       150         2   0.20233
    18       150         2   0.05226
    19       150         2   0.26139
    20       150         2   0.36725
    21       150         2   0.11356
-------------------------------------------------
  Mean   150.000     2.000   0.16971
  s.d.     0.000     0.000   0.10653
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0545
Gene diversity           :  0.9519 +/-  0.0095

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 49

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 21
Number of usable nucleotide sites            : 21


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.563937 +/-     1.821415

Average gene diversity over loci              :     0.169711 +/-     0.096006

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    18.140403
  S.D. Theta(Hom) :     4.035815

         Theta(k) :    24.926798
95 % confidence interval limits for theta(k) : [  17.352333,  35.483223 ]

         Theta(S) :     3.760399
    S.D. Theta(S) :     1.159953

        Theta(Pi) :     3.563937
   S.D. Theta(Pi) :     2.016130

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     538       0.000
       1     976       0.000
       2    1838       0.000
       3    2249       0.000
       4    2225       0.000
       5    1672       0.000
       6     887       0.000
       7     529       0.000
       8     206       0.000
       9      55       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.564
Mismatch observed variance          :        3.537
Tau                                 :        3.793
Theta0                              :        0.000
Theta1                              :       57.812

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00050588
          P(Sim. Ssd >= Obs. Ssd): 0.50000000
    Harpending's Raggedness index: 0.01829538
        P(Sim. Rag. >= Obs. Rag.): 0.60000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        538.0        390.4        421.8
          1.0        976.0       1093.6       1061.9
          2.0       1838.0       1873.4       1831.9
          3.0       2249.0       2200.9       2241.4
          4.0       2225.0       2053.9       2094.2
          5.0       1672.0       1610.4       1576.1
          6.0        887.0       1034.2        991.6
          7.0        529.0        539.7        535.6
          8.0        206.0        250.3        253.4
          9.0         55.0         95.1        106.6
         10.0          0.0         25.3         40.4
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.793           2.766           4.744           3.634
                   Theta0:           0.000           0.000           0.619           0.166               
                   Theta1:          57.812          37.031       99999.000       50026.812
Mean No. of pairwise diff.           3.564           2.732           4.601           3.659
           No. pol. sites:              21              75             107          92.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        538.0        162.0        720.0        421.8
          1.0        976.0        472.0       2169.0       1061.9
          2.0       1838.0       1039.0       2649.0       1831.9
          3.0       2249.0       1674.0       2853.0       2241.4
          4.0       2225.0       1726.0       2442.0       2094.2
          5.0       1672.0       1101.0       1979.0       1576.1
          6.0        887.0        430.0       1775.0        991.6
          7.0        529.0        162.0        991.0        535.6
          8.0        206.0         57.0        591.0        253.4
          9.0         55.0          0.0        275.0        106.6
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.793           2.766           4.744           3.634
                   Theta0:           0.000           0.000           0.619           0.166               
                   Theta1:          57.812          37.031       99999.000       50026.812
Mean No. of pairwise diff.           3.564           2.732           4.601           3.659
           No. pol. sites:              21              75             107          92.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        538.0        162.0        720.0        421.8
          1.0        976.0        472.0       2169.0       1061.9
          2.0       1838.0       1039.0       2649.0       1831.9
          3.0       2249.0       1674.0       2853.0       2241.4
          4.0       2225.0       1726.0       2442.0       2094.2
          5.0       1672.0       1101.0       1979.0       1576.1
          6.0        887.0        430.0       1775.0        991.6
          7.0        529.0        162.0        991.0        535.6
          8.0        206.0         57.0        591.0        253.4
          9.0         55.0          0.0        275.0        106.6
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.793           2.766           4.744           3.634
                   Theta0:           0.000           0.000           0.619           0.166               
                   Theta1:          57.812          37.031       99999.000       50026.812
Mean No. of pairwise diff.           3.564           2.732           4.601           3.659
           No. pol. sites:              21              75             107          92.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        538.0        162.0        720.0        421.8
          1.0        976.0        472.0       2169.0       1061.9
          2.0       1838.0       1039.0       2649.0       1831.9
          3.0       2249.0       1674.0       2853.0       2241.4
          4.0       2225.0       1726.0       2442.0       2094.2
          5.0       1672.0       1101.0       1979.0       1576.1
          6.0        887.0        430.0       1775.0        991.6
          7.0        529.0        162.0        991.0        535.6
          8.0        206.0         57.0        591.0        253.4
          9.0         55.0          0.0        275.0        106.6
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_49_54_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            21      21.000       0.000
  No. of usable loci            21      21.000       0.000
  No. of polym. loci            21      21.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_49_54_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.11356     0.11356     0.00000     0.11356
     2       0.18121     0.18121     0.00000     0.18121
     3       0.03946     0.03946     0.00000     0.03946
     4       0.11356     0.11356     0.00000     0.11356
     5       0.30568     0.30568     0.00000     0.30568
     6       0.23266     0.23266     0.00000     0.23266
     7       0.05226     0.05226     0.00000     0.05226
     8       0.13682     0.13682     0.00000     0.13682
     9       0.32215     0.32215     0.00000     0.32215
    10       0.13682     0.13682     0.00000     0.13682
    11       0.12528     0.12528     0.00000     0.12528
    12       0.15937     0.15937     0.00000     0.15937
    13       0.20233     0.20233     0.00000     0.20233
    14       0.02649     0.02649     0.00000     0.02649
    15       0.05226     0.05226     0.00000     0.05226
    16       0.36725     0.36725     0.00000     0.36725
    17       0.20233     0.20233     0.00000     0.20233
    18       0.05226     0.05226     0.00000     0.05226
    19       0.26139     0.26139     0.00000     0.26139
    20       0.36725     0.36725     0.00000     0.36725
    21       0.11356     0.11356     0.00000     0.11356
--------------------------------------------------------
  Mean       0.16971     0.16971     0.00000     0.16971
  s.d.       0.10653     0.10653     0.00000     0.10653
--------------------------------------------------------
<A NAME="31_01_17at14_49_54_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_49_54_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      24.92680    24.92680     0.00000
            Theta_k_lower      17.35233    17.35233     0.00000
            Theta_k_upper      35.48322    35.48322     0.00000
                  Theta_H      18.14040    18.14040     0.00000
             s.d. Theta_H       4.03582     4.03582     0.00000
                  Theta_S       3.76040     3.76040     0.00000
             s.d. Theta_S       1.15995     1.15995     0.00000
                 Theta_pi       3.56394     3.56394     0.00000
            s.d. Theta_pi       2.01613     2.01613     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_49_54_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.79297     3.79297     0.00000
              Tau qt 2.5%       2.76562     2.76562     0.00000
                Tau qt 5%       2.76562     2.76562     0.00000
               Tau qt 95%       4.67578     4.67578     0.00000
             Tau qt 97.5%       4.67578     4.67578     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.57305     0.57305     0.00000
          Theta0 qt 97.5%       0.57305     0.57305     0.00000
                   Theta1      57.81250    57.81250     0.00000
           Theta1 qt 2.5%      37.03125    37.03125     0.00000
             Theta1 qt 5%      37.03125    37.03125     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00051     0.00051     0.00000
      Model (SSD) p-value       0.50000     0.50000     0.00000
         Raggedness index       0.01830     0.01830     0.00000
       Raggedness p-value       0.60000     0.60000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 255 (31/01/17 at 14:49:54))
Total computing time for this run : 0h 0m 0s 121 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

