
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_30_37"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 124 (31/01/17 at 16:30:37)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_30_37_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_30_37_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 58
No. of loci              : 77
No. of usable loci       : 77 loci with less than  5.00 % missing data
No. of polymorphic sites : 77


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.06488
     2       150         2   0.03946
     3       150         2   0.01333
     4       150         2   0.01333
     5       150         2   0.02649
     6       150         2   0.03946
     7       150         2   0.03946
     8       150         2   0.03946
     9       150         2   0.05226
    10       150         2   0.01333
    11       150         2   0.02649
    12       150         2   0.01333
    13       150         2   0.02649
    14       150         2   0.02649
    15       150         2   0.03946
    16       150         2   0.01333
    17       150         2   0.01333
    18       150         2   0.02649
    19       150         2   0.02649
    20       150         2   0.01333
    21       150         2   0.03946
    22       150         2   0.06488
    23       150         2   0.03946
    24       150         2   0.01333
    25       150         2   0.01333
    26       150         2   0.01333
    27       150         2   0.02649
    28       150         2   0.01333
    29       150         2   0.01333
    30       150         2   0.06488
    31       150         2   0.01333
    32       150         2   0.02649
    33       150         2   0.01333
    34       150         2   0.01333
    35       150         2   0.06488
    36       150         2   0.01333
    37       150         2   0.01333
    38       150         2   0.01333
    39       150         2   0.02649
    40       150         2   0.02649
    41       150         2   0.03946
    42       150         2   0.02649
    43       150         2   0.05226
    44       150         2   0.01333
    45       150         2   0.01333
    46       150         2   0.01333
    47       150         2   0.01333
    48       150         2   0.03946
    49       150         2   0.03946
    50       150         2   0.01333
    51       150         2   0.03946
    52       150         2   0.01333
    53       150         2   0.07732
    54       150         2   0.03946
    55       150         2   0.06488
    56       150         2   0.01333
    57       150         2   0.03946
    58       150         2   0.03946
    59       150         2   0.01333
    60       150         2   0.01333
    61       150         2   0.02649
    62       150         2   0.01333
    63       150         2   0.05226
    64       150         2   0.01333
    65       150         2   0.08957
    66       150         2   0.02649
    67       150         2   0.01333
    68       150         2   0.06488
    69       150         2   0.01333
    70       150         2   0.03946
    71       150         2   0.03946
    72       150         2   0.01333
    73       150         2   0.05226
    74       150         2   0.03946
    75       150         2   0.03946
    76       150         2   0.02649
    77       150         2   0.06488
-------------------------------------------------
  Mean   150.000     2.000   0.03036
  s.d.     0.000     0.000   0.01883
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.1581
Gene diversity           :  0.8475 +/-  0.0297

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 58

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 77
Number of usable nucleotide sites            : 77


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.337897 +/-     1.284270

Average gene diversity over loci              :     0.030362 +/-     0.018462

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     4.538201
  S.D. Theta(Hom) :     1.144823

         Theta(k) :    34.205353
95 % confidence interval limits for theta(k) : [  24.201985,  48.048972 ]

         Theta(S) :    13.788129
    S.D. Theta(S) :     3.440055

        Theta(Pi) :     2.337897
   S.D. Theta(Pi) :     1.421563

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1704    1316.250
       1    2240    2636.383
       2    2614    2764.815
       3    1818    2048.350
       4    1519    1222.531
       5     738     635.764
       6     399     303.731
       7     100     138.212
       8      43      61.317
       9       0      26.882
      10       0      11.727
      11       0       5.106
      12       0       2.222
      13       0       0.967
      14       0       0.420
      15       0       0.183
      16       0       0.080


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.338
Mismatch observed variance          :        2.931
Tau                                 :        2.264
Theta0                              :        0.520
Theta1                              :        8.359

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00085877
          P(Sim. Ssd >= Obs. Ssd): 0.90000000
    Harpending's Raggedness index: 0.01577162
        P(Sim. Rag. >= Obs. Rag.): 1.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1704.0       1706.0       1682.2
          1.0       2240.0       2751.1       2286.5
          2.0       2614.0       3033.4       2456.3
          3.0       1818.0       1985.0       2028.5
          4.0       1519.0       1089.8       1345.5
          5.0        738.0        445.7        749.8
          6.0        399.0        119.3        364.4
          7.0        100.0         33.2        159.3
          8.0         43.0          9.5         64.2
          9.0          0.0          2.0         24.4
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.264           1.531           2.482           2.143
                   Theta0:           0.520           0.000           1.211           0.124               
                   Theta1:           8.359           2.863       99999.000       21858.914
Mean No. of pairwise diff.           2.338           1.415           2.688           2.005
           No. pol. sites:              77              17              29          24.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1704.0       1026.0       3475.0       1682.2
          1.0       2240.0       2210.0       4405.0       2286.5
          2.0       2614.0       1962.0       3750.0       2456.3
          3.0       1818.0        874.0       2704.0       2028.5
          4.0       1519.0        177.0       1741.0       1345.5
          5.0        738.0         16.0       1199.0        749.8
          6.0        399.0          0.0        506.0        364.4
          7.0        100.0          0.0        220.0        159.3
          8.0         43.0          0.0         92.0         64.2
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.264           1.531           2.482           2.143
                   Theta0:           0.520           0.000           1.211           0.124               
                   Theta1:           8.359           2.863       99999.000       21858.914
Mean No. of pairwise diff.           2.338           1.415           2.688           2.005
           No. pol. sites:              77              17              29          24.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1704.0       1026.0       3475.0       1682.2
          1.0       2240.0       2210.0       4405.0       2286.5
          2.0       2614.0       1962.0       3750.0       2456.3
          3.0       1818.0        874.0       2704.0       2028.5
          4.0       1519.0        177.0       1741.0       1345.5
          5.0        738.0         16.0       1199.0        749.8
          6.0        399.0          0.0        506.0        364.4
          7.0        100.0          0.0        220.0        159.3
          8.0         43.0          0.0         92.0         64.2
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.264           1.531           2.482           2.143
                   Theta0:           0.520           0.000           1.211           0.124               
                   Theta1:           8.359           2.863       99999.000       21858.914
Mean No. of pairwise diff.           2.338           1.415           2.688           2.005
           No. pol. sites:              77              17              29          24.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1704.0       1026.0       3475.0       1682.2
          1.0       2240.0       2210.0       4405.0       2286.5
          2.0       2614.0       1962.0       3750.0       2456.3
          3.0       1818.0        874.0       2704.0       2028.5
          4.0       1519.0        177.0       1741.0       1345.5
          5.0        738.0         16.0       1199.0        749.8
          6.0        399.0          0.0        506.0        364.4
          7.0        100.0          0.0        220.0        159.3
          8.0         43.0          0.0         92.0         64.2
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_30_37_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            77      77.000       0.000
  No. of usable loci            77      77.000       0.000
  No. of polym. loci            77      77.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_30_37_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.06488     0.06488     0.00000     0.06488
     2       0.03946     0.03946     0.00000     0.03946
     3       0.01333     0.01333     0.00000     0.01333
     4       0.01333     0.01333     0.00000     0.01333
     5       0.02649     0.02649     0.00000     0.02649
     6       0.03946     0.03946     0.00000     0.03946
     7       0.03946     0.03946     0.00000     0.03946
     8       0.03946     0.03946     0.00000     0.03946
     9       0.05226     0.05226     0.00000     0.05226
    10       0.01333     0.01333     0.00000     0.01333
    11       0.02649     0.02649     0.00000     0.02649
    12       0.01333     0.01333     0.00000     0.01333
    13       0.02649     0.02649     0.00000     0.02649
    14       0.02649     0.02649     0.00000     0.02649
    15       0.03946     0.03946     0.00000     0.03946
    16       0.01333     0.01333     0.00000     0.01333
    17       0.01333     0.01333     0.00000     0.01333
    18       0.02649     0.02649     0.00000     0.02649
    19       0.02649     0.02649     0.00000     0.02649
    20       0.01333     0.01333     0.00000     0.01333
    21       0.03946     0.03946     0.00000     0.03946
    22       0.06488     0.06488     0.00000     0.06488
    23       0.03946     0.03946     0.00000     0.03946
    24       0.01333     0.01333     0.00000     0.01333
    25       0.01333     0.01333     0.00000     0.01333
    26       0.01333     0.01333     0.00000     0.01333
    27       0.02649     0.02649     0.00000     0.02649
    28       0.01333     0.01333     0.00000     0.01333
    29       0.01333     0.01333     0.00000     0.01333
    30       0.06488     0.06488     0.00000     0.06488
    31       0.01333     0.01333     0.00000     0.01333
    32       0.02649     0.02649     0.00000     0.02649
    33       0.01333     0.01333     0.00000     0.01333
    34       0.01333     0.01333     0.00000     0.01333
    35       0.06488     0.06488     0.00000     0.06488
    36       0.01333     0.01333     0.00000     0.01333
    37       0.01333     0.01333     0.00000     0.01333
    38       0.01333     0.01333     0.00000     0.01333
    39       0.02649     0.02649     0.00000     0.02649
    40       0.02649     0.02649     0.00000     0.02649
    41       0.03946     0.03946     0.00000     0.03946
    42       0.02649     0.02649     0.00000     0.02649
    43       0.05226     0.05226     0.00000     0.05226
    44       0.01333     0.01333     0.00000     0.01333
    45       0.01333     0.01333     0.00000     0.01333
    46       0.01333     0.01333     0.00000     0.01333
    47       0.01333     0.01333     0.00000     0.01333
    48       0.03946     0.03946     0.00000     0.03946
    49       0.03946     0.03946     0.00000     0.03946
    50       0.01333     0.01333     0.00000     0.01333
    51       0.03946     0.03946     0.00000     0.03946
    52       0.01333     0.01333     0.00000     0.01333
    53       0.07732     0.07732     0.00000     0.07732
    54       0.03946     0.03946     0.00000     0.03946
    55       0.06488     0.06488     0.00000     0.06488
    56       0.01333     0.01333     0.00000     0.01333
    57       0.03946     0.03946     0.00000     0.03946
    58       0.03946     0.03946     0.00000     0.03946
    59       0.01333     0.01333     0.00000     0.01333
    60       0.01333     0.01333     0.00000     0.01333
    61       0.02649     0.02649     0.00000     0.02649
    62       0.01333     0.01333     0.00000     0.01333
    63       0.05226     0.05226     0.00000     0.05226
    64       0.01333     0.01333     0.00000     0.01333
    65       0.08957     0.08957     0.00000     0.08957
    66       0.02649     0.02649     0.00000     0.02649
    67       0.01333     0.01333     0.00000     0.01333
    68       0.06488     0.06488     0.00000     0.06488
    69       0.01333     0.01333     0.00000     0.01333
    70       0.03946     0.03946     0.00000     0.03946
    71       0.03946     0.03946     0.00000     0.03946
    72       0.01333     0.01333     0.00000     0.01333
    73       0.05226     0.05226     0.00000     0.05226
    74       0.03946     0.03946     0.00000     0.03946
    75       0.03946     0.03946     0.00000     0.03946
    76       0.02649     0.02649     0.00000     0.02649
    77       0.06488     0.06488     0.00000     0.06488
--------------------------------------------------------
  Mean       0.03036     0.03036     0.00000     0.03036
  s.d.       0.01883     0.01883     0.00000     0.01883
--------------------------------------------------------
<A NAME="31_01_17at16_30_37_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
    72             2       2.000       0.000           2
    73             2       2.000       0.000           2
    74             2       2.000       0.000           2
    75             2       2.000       0.000           2
    76             2       2.000       0.000           2
    77             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_30_37_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      34.20535    34.20535     0.00000
            Theta_k_lower      24.20198    24.20198     0.00000
            Theta_k_upper      48.04897    48.04897     0.00000
                  Theta_H       4.53820     4.53820     0.00000
             s.d. Theta_H       1.14482     1.14482     0.00000
                  Theta_S      13.78813    13.78813     0.00000
             s.d. Theta_S       3.44006     3.44006     0.00000
                 Theta_pi       2.33790     2.33790     0.00000
            s.d. Theta_pi       1.42156     1.42156     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_30_37_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       2.26367     2.26367     0.00000
              Tau qt 2.5%       1.53125     1.53125     0.00000
                Tau qt 5%       1.53125     1.53125     0.00000
               Tau qt 95%       2.33984     2.33984     0.00000
             Tau qt 97.5%       2.33984     2.33984     0.00000
                   Theta0       0.52031     0.52031     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.01406     0.01406     0.00000
          Theta0 qt 97.5%       0.01406     0.01406     0.00000
                   Theta1       8.35938     8.35938     0.00000
           Theta1 qt 2.5%       2.86270     2.86270     0.00000
             Theta1 qt 5%       2.86270     2.86270     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00086     0.00086     0.00000
      Model (SSD) p-value       0.90000     0.90000     0.00000
         Raggedness index       0.01577     0.01577     0.00000
       Raggedness p-value       1.00000     1.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 124 (31/01/17 at 16:30:37))
Total computing time for this run : 0h 0m 0s 110 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

