
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_03"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 119 (31/01/17 at 14:44:03)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_03_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_03_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 29
No. of loci              : 17
No. of usable loci       : 17 loci with less than  5.00 % missing data
No. of polymorphic sites : 17


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.20233
     2       150         2   0.33011
     3       150         2   0.06488
     4       150         2   0.06488
     5       150         2   0.21262
     6       150         2   0.27964
     7       150         2   0.08957
     8       150         2   0.13682
     9       150         2   0.19186
    10       150         2   0.02649
    11       150         2   0.14819
    12       150         2   0.33011
    13       150         2   0.15937
    14       150         2   0.11356
    15       150         2   0.13682
    16       150         2   0.21262
    17       150         2   0.17038
-------------------------------------------------
  Mean   150.000     2.000   0.16884
  s.d.     0.000     0.000   0.08788
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.1178
Gene diversity           :  0.8881 +/-  0.0198

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 29

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 17
Number of usable nucleotide sites            : 17


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.870246 +/-     1.518486

Average gene diversity over loci              :     0.168838 +/-     0.098872

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     6.716140
  S.D. Theta(Hom) :     1.473809

         Theta(k) :    10.439799
95 % confidence interval limits for theta(k) : [   6.753332,  15.822185 ]

         Theta(S) :     3.044132
    S.D. Theta(S) :     0.990144

        Theta(Pi) :     2.870246
   S.D. Theta(Pi) :     1.680817

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1250     825.021
       1    2068    2008.653
       2    1713    2539.141
       3    1999    2243.010
       4    1756    1575.065
       5    1207     949.208
       6     790     517.424
       7     351     264.940
       8      41     130.840
       9       0      63.396
      10       0      30.440
      11       0      14.560
      12       0       6.953
      13       0       3.319
      14       0       1.584
      15       0       0.756
      16       0       0.361


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.870
Mismatch observed variance          :        3.703
Tau                                 :        3.535
Theta0                              :        0.160
Theta1                              :        8.062

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00226157
          P(Sim. Ssd >= Obs. Ssd): 0.90000000
    Harpending's Raggedness index: 0.01362730
        P(Sim. Rag. >= Obs. Rag.): 0.80000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1250.0       1303.6       1391.2
          1.0       2068.0       1624.7       1659.9
          2.0       1713.0       1670.9       1963.6
          3.0       1999.0       1902.4       1990.2
          4.0       1756.0       1813.6       1668.9
          5.0       1207.0       1215.9       1169.5
          6.0        790.0        746.2        698.4
          7.0        351.0        444.6        362.2
          8.0         41.0        171.9        165.8
          9.0          0.0        192.7         67.9
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.535           1.865           5.223           3.779
                   Theta0:           0.160           0.000           2.273           0.280               
                   Theta1:           8.062           6.516       99999.000       10059.716
Mean No. of pairwise diff.           2.870           1.737           4.465           3.187
           No. pol. sites:              17              21              38          27.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1250.0        610.0       1896.0       1391.2
          1.0       2068.0        486.0       3400.0       1659.9
          2.0       1713.0        445.0       3754.0       1963.6
          3.0       1999.0       1262.0       2796.0       1990.2
          4.0       1756.0        552.0       3356.0       1668.9
          5.0       1207.0        104.0       2561.0       1169.5
          6.0        790.0          0.0       1559.0        698.4
          7.0        351.0          0.0       1306.0        362.2
          8.0         41.0          0.0        590.0        165.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.535           1.865           5.223           3.779
                   Theta0:           0.160           0.000           2.273           0.280               
                   Theta1:           8.062           6.516       99999.000       10059.716
Mean No. of pairwise diff.           2.870           1.737           4.465           3.187
           No. pol. sites:              17              21              38          27.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1250.0        610.0       1896.0       1391.2
          1.0       2068.0        486.0       3400.0       1659.9
          2.0       1713.0        445.0       3754.0       1963.6
          3.0       1999.0       1262.0       2796.0       1990.2
          4.0       1756.0        552.0       3356.0       1668.9
          5.0       1207.0        104.0       2561.0       1169.5
          6.0        790.0          0.0       1559.0        698.4
          7.0        351.0          0.0       1306.0        362.2
          8.0         41.0          0.0        590.0        165.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.535           1.865           5.223           3.779
                   Theta0:           0.160           0.000           2.273           0.280               
                   Theta1:           8.062           6.516       99999.000       10059.716
Mean No. of pairwise diff.           2.870           1.737           4.465           3.187
           No. pol. sites:              17              21              38          27.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1250.0        610.0       1896.0       1391.2
          1.0       2068.0        486.0       3400.0       1659.9
          2.0       1713.0        445.0       3754.0       1963.6
          3.0       1999.0       1262.0       2796.0       1990.2
          4.0       1756.0        552.0       3356.0       1668.9
          5.0       1207.0        104.0       2561.0       1169.5
          6.0        790.0          0.0       1559.0        698.4
          7.0        351.0          0.0       1306.0        362.2
          8.0         41.0          0.0        590.0        165.8
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_03_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            17      17.000       0.000
  No. of usable loci            17      17.000       0.000
  No. of polym. loci            17      17.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_03_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.20233     0.20233     0.00000     0.20233
     2       0.33011     0.33011     0.00000     0.33011
     3       0.06488     0.06488     0.00000     0.06488
     4       0.06488     0.06488     0.00000     0.06488
     5       0.21262     0.21262     0.00000     0.21262
     6       0.27964     0.27964     0.00000     0.27964
     7       0.08957     0.08957     0.00000     0.08957
     8       0.13682     0.13682     0.00000     0.13682
     9       0.19186     0.19186     0.00000     0.19186
    10       0.02649     0.02649     0.00000     0.02649
    11       0.14819     0.14819     0.00000     0.14819
    12       0.33011     0.33011     0.00000     0.33011
    13       0.15937     0.15937     0.00000     0.15937
    14       0.11356     0.11356     0.00000     0.11356
    15       0.13682     0.13682     0.00000     0.13682
    16       0.21262     0.21262     0.00000     0.21262
    17       0.17038     0.17038     0.00000     0.17038
--------------------------------------------------------
  Mean       0.16884     0.16884     0.00000     0.16884
  s.d.       0.08788     0.08788     0.00000     0.08788
--------------------------------------------------------
<A NAME="31_01_17at14_44_03_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_03_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      10.43980    10.43980     0.00000
            Theta_k_lower       6.75333     6.75333     0.00000
            Theta_k_upper      15.82219    15.82219     0.00000
                  Theta_H       6.71614     6.71614     0.00000
             s.d. Theta_H       1.47381     1.47381     0.00000
                  Theta_S       3.04413     3.04413     0.00000
             s.d. Theta_S       0.99014     0.99014     0.00000
                 Theta_pi       2.87025     2.87025     0.00000
            s.d. Theta_pi       1.68082     1.68082     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_03_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.53516     3.53516     0.00000
              Tau qt 2.5%       1.86523     1.86523     0.00000
                Tau qt 5%       1.86523     1.86523     0.00000
               Tau qt 95%       5.09375     5.09375     0.00000
             Tau qt 97.5%       5.09375     5.09375     0.00000
                   Theta0       0.15996     0.15996     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.45176     0.45176     0.00000
          Theta0 qt 97.5%       0.45176     0.45176     0.00000
                   Theta1       8.06152     8.06152     0.00000
           Theta1 qt 2.5%       6.51641     6.51641     0.00000
             Theta1 qt 5%       6.51641     6.51641     0.00000
            Theta1 qt 95%     485.56152   485.56152     0.00000
          Theta1 qt 97.5%     485.56152   485.56152     0.00000
                      SSD       0.00226     0.00226     0.00000
      Model (SSD) p-value       0.90000     0.90000     0.00000
         Raggedness index       0.01363     0.01363     0.00000
       Raggedness p-value       0.80000     0.80000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 119 (31/01/17 at 14:44:03))
Total computing time for this run : 0h 0m 0s 85 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

