
<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_42_35"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 146 (31/01/17 at 14:42:35)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_42_35_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_42_35_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 39
No. of loci              : 43
No. of usable loci       : 43 loci with less than  5.00 % missing data
No. of polymorphic sites : 43


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.02649
     3       150         2   0.01333
     4       150         2   0.02649
     5       150         2   0.01333
     6       150         2   0.06488
     7       150         2   0.01333
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.03946
    11       150         2   0.02649
    12       150         2   0.01333
    13       150         2   0.01333
    14       150         2   0.05226
    15       150         2   0.01333
    16       150         2   0.07732
    17       150         2   0.06488
    18       150         2   0.01333
    19       150         2   0.02649
    20       150         2   0.01333
    21       150         2   0.07732
    22       150         2   0.03946
    23       150         2   0.12528
    24       150         2   0.03946
    25       150         2   0.01333
    26       150         2   0.03946
    27       150         2   0.01333
    28       150         2   0.01333
    29       150         2   0.01333
    30       150         2   0.10166
    31       150         2   0.01333
    32       150         2   0.01333
    33       150         2   0.08957
    34       150         2   0.08957
    35       150         2   0.01333
    36       150         2   0.01333
    37       150         2   0.02649
    38       150         2   0.01333
    39       150         2   0.02649
    40       150         2   0.01333
    41       150         2   0.21262
    42       150         2   0.01333
    43       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.03672
  s.d.     0.000     0.000   0.03992
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.1540
Gene diversity           :  0.8517 +/-  0.0258

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 39

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 43
Number of usable nucleotide sites            : 43


Distance method : Pairwise difference
Mean number of pairwise differences           :     1.578792 +/-     0.945250

Average gene diversity over loci              :     0.036716 +/-     0.024333

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     4.704979
  S.D. Theta(Hom) :     1.055190

         Theta(k) :    16.787407
95 % confidence interval limits for theta(k) : [  11.363701,  24.469906 ]

         Theta(S) :     7.699864
    S.D. Theta(S) :     2.066141

        Theta(Pi) :     1.578792
   S.D. Theta(Pi) :     1.046301

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1657       0.000
       1    4061       0.000
       2    3430       0.000
       3    1471       0.000
       4     475       0.000
       5      77       0.000
       6       4       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        1.579
Mismatch observed variance          :        1.149
Tau                                 :        1.668
Theta0                              :        0.000
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00673756
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.08945201
        P(Sim. Rag. >= Obs. Rag.): 0.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1657.0       1947.1       2108.0
          1.0       4061.0       3441.0       3516.0
          2.0       3430.0       2940.2       2932.2
          3.0       1471.0       1705.9       1630.3
          4.0        475.0        770.1        679.8
          5.0         77.0        270.5        226.8
          6.0          4.0         79.0         63.0
          7.0          0.0         17.7         15.0
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.668           1.543           2.102           1.690
                   Theta0:           0.000           0.000           0.037           0.004               
                   Theta1:       99999.000          37.516       99999.000       70026.145
Mean No. of pairwise diff.           1.579           1.505           2.024           1.745
           No. pol. sites:              43              96             128         111.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1657.0       1156.0       2346.0       2108.0
          1.0       4061.0       3008.0       4032.0       3516.0
          2.0       3430.0       2440.0       3301.0       2932.2
          3.0       1471.0       1371.0       2284.0       1630.3
          4.0        475.0        411.0       1038.0        679.8
          5.0         77.0        104.0        406.0        226.8
          6.0          4.0         22.0        167.0         63.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.668           1.543           2.102           1.690
                   Theta0:           0.000           0.000           0.037           0.004               
                   Theta1:       99999.000          37.516       99999.000       70026.145
Mean No. of pairwise diff.           1.579           1.505           2.024           1.745
           No. pol. sites:              43              96             128         111.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1657.0       1156.0       2346.0       2108.0
          1.0       4061.0       3008.0       4032.0       3516.0
          2.0       3430.0       2440.0       3301.0       2932.2
          3.0       1471.0       1371.0       2284.0       1630.3
          4.0        475.0        411.0       1038.0        679.8
          5.0         77.0        104.0        406.0        226.8
          6.0          4.0         22.0        167.0         63.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.668           1.543           2.102           1.690
                   Theta0:           0.000           0.000           0.037           0.004               
                   Theta1:       99999.000          37.516       99999.000       70026.145
Mean No. of pairwise diff.           1.579           1.505           2.024           1.745
           No. pol. sites:              43              96             128         111.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1657.0       1156.0       2346.0       2108.0
          1.0       4061.0       3008.0       4032.0       3516.0
          2.0       3430.0       2440.0       3301.0       2932.2
          3.0       1471.0       1371.0       2284.0       1630.3
          4.0        475.0        411.0       1038.0        679.8
          5.0         77.0        104.0        406.0        226.8
          6.0          4.0         22.0        167.0         63.0
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_42_35_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            43      43.000       0.000
  No. of usable loci            43      43.000       0.000
  No. of polym. loci            43      43.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_42_35_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.02649     0.02649     0.00000     0.02649
     3       0.01333     0.01333     0.00000     0.01333
     4       0.02649     0.02649     0.00000     0.02649
     5       0.01333     0.01333     0.00000     0.01333
     6       0.06488     0.06488     0.00000     0.06488
     7       0.01333     0.01333     0.00000     0.01333
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.03946     0.03946     0.00000     0.03946
    11       0.02649     0.02649     0.00000     0.02649
    12       0.01333     0.01333     0.00000     0.01333
    13       0.01333     0.01333     0.00000     0.01333
    14       0.05226     0.05226     0.00000     0.05226
    15       0.01333     0.01333     0.00000     0.01333
    16       0.07732     0.07732     0.00000     0.07732
    17       0.06488     0.06488     0.00000     0.06488
    18       0.01333     0.01333     0.00000     0.01333
    19       0.02649     0.02649     0.00000     0.02649
    20       0.01333     0.01333     0.00000     0.01333
    21       0.07732     0.07732     0.00000     0.07732
    22       0.03946     0.03946     0.00000     0.03946
    23       0.12528     0.12528     0.00000     0.12528
    24       0.03946     0.03946     0.00000     0.03946
    25       0.01333     0.01333     0.00000     0.01333
    26       0.03946     0.03946     0.00000     0.03946
    27       0.01333     0.01333     0.00000     0.01333
    28       0.01333     0.01333     0.00000     0.01333
    29       0.01333     0.01333     0.00000     0.01333
    30       0.10166     0.10166     0.00000     0.10166
    31       0.01333     0.01333     0.00000     0.01333
    32       0.01333     0.01333     0.00000     0.01333
    33       0.08957     0.08957     0.00000     0.08957
    34       0.08957     0.08957     0.00000     0.08957
    35       0.01333     0.01333     0.00000     0.01333
    36       0.01333     0.01333     0.00000     0.01333
    37       0.02649     0.02649     0.00000     0.02649
    38       0.01333     0.01333     0.00000     0.01333
    39       0.02649     0.02649     0.00000     0.02649
    40       0.01333     0.01333     0.00000     0.01333
    41       0.21262     0.21262     0.00000     0.21262
    42       0.01333     0.01333     0.00000     0.01333
    43       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.03672     0.03672     0.00000     0.03672
  s.d.       0.03992     0.03992     0.00000     0.03992
--------------------------------------------------------
<A NAME="31_01_17at14_42_35_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_42_35_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      16.78741    16.78741     0.00000
            Theta_k_lower      11.36370    11.36370     0.00000
            Theta_k_upper      24.46991    24.46991     0.00000
                  Theta_H       4.70498     4.70498     0.00000
             s.d. Theta_H       1.05519     1.05519     0.00000
                  Theta_S       7.69986     7.69986     0.00000
             s.d. Theta_S       2.06614     2.06614     0.00000
                 Theta_pi       1.57879     1.57879     0.00000
            s.d. Theta_pi       1.04630     1.04630     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_42_35_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.66797     1.66797     0.00000
              Tau qt 2.5%       1.54297     1.54297     0.00000
                Tau qt 5%       1.54297     1.54297     0.00000
               Tau qt 95%       1.84570     1.84570     0.00000
             Tau qt 97.5%       1.84570     1.84570     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00527     0.00527     0.00000
          Theta0 qt 97.5%       0.00527     0.00527     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      37.51562    37.51562     0.00000
             Theta1 qt 5%      37.51562    37.51562     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00674     0.00674     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.08945     0.08945     0.00000
       Raggedness p-value       0.00000     0.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 146 (31/01/17 at 14:42:35))
Total computing time for this run : 0h 0m 0s 74 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

