
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_54_12"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 113 (31/01/17 at 14:54:12)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_54_12_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_54_12_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 40
No. of loci              : 19
No. of usable loci       : 19 loci with less than  5.00 % missing data
No. of polymorphic sites : 19


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.19186
     2       150         2   0.45602
     3       150         2   0.01333
     4       150         2   0.01333
     5       150         2   0.12528
     6       150         2   0.48322
     7       150         2   0.06488
     8       150         2   0.21262
     9       150         2   0.24242
    10       150         2   0.07732
    11       150         2   0.03946
    12       150         2   0.14819
    13       150         2   0.37414
    14       150         2   0.31400
    15       150         2   0.26139
    16       150         2   0.13682
    17       150         2   0.02649
    18       150         2   0.17038
    19       150         2   0.45181
-------------------------------------------------
  Mean   150.000     2.000   0.20016
  s.d.     0.000     0.000   0.15461
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0766
Gene diversity           :  0.9296 +/-  0.0111

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 40

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 19
Number of usable nucleotide sites            : 19


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.802953 +/-     1.925393

Average gene diversity over loci              :     0.200155 +/-     0.112170

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    11.723060
  S.D. Theta(Hom) :     2.161934

         Theta(k) :    17.513851
95 % confidence interval limits for theta(k) : [  11.895992,  25.454453 ]

         Theta(S) :     3.402266
    S.D. Theta(S) :     1.075424

        Theta(Pi) :     3.802953
   S.D. Theta(Pi) :     2.131224

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     787     276.722
       1     892    1001.647
       2    1072    1829.406
       3    1972    2251.128
       4    2242    2103.278
       5    2143    1594.919
       6    1186    1025.039
       7     436     575.997
       8     327     289.887
       9     101     133.264
      10      17      56.906
      11       0      22.905
      12       0       8.804
      13       0       3.268
      14       0       1.183
      15       0       0.421
      16       0       0.148
      17       0       0.052
      18       0       0.018
      19       0       0.006
      20       0       0.002


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.803
Mismatch observed variance          :        4.083
Tau                                 :        4.381
Theta0                              :        0.000
Theta1                              :       27.852

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00487731
          P(Sim. Ssd >= Obs. Ssd): 0.10000000
    Harpending's Raggedness index: 0.01989722
        P(Sim. Rag. >= Obs. Rag.): 0.40000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        787.0        601.9        502.7
          1.0        892.0       1105.4        875.3
          2.0       1072.0       1526.3       1446.5
          3.0       1972.0       1805.1       1905.9
          4.0       2242.0       1793.4       1993.7
          5.0       2143.0       1521.1       1705.4
          6.0       1186.0       1151.5       1227.7
          7.0        436.0        729.2        761.4
          8.0        327.0        464.6        414.4
          9.0        101.0        228.0        200.8
         10.0         17.0        153.6         87.7
         11.0          0.0         64.1         34.8
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.381           2.268           7.605           4.073
                   Theta0:           0.000           0.000           2.213           0.397               
                   Theta1:          27.852          10.098       99999.000       20030.720
Mean No. of pairwise diff.           3.803           2.942           6.550           4.005
           No. pol. sites:              19              50              79          63.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        787.0        263.0       1350.0        502.7
          1.0        892.0        336.0       1863.0        875.3
          2.0       1072.0        397.0       2227.0       1446.5
          3.0       1972.0        710.0       2609.0       1905.9
          4.0       2242.0        697.0       2288.0       1993.7
          5.0       2143.0       1027.0       2335.0       1705.4
          6.0       1186.0        568.0       1883.0       1227.7
          7.0        436.0        187.0       1764.0        761.4
          8.0        327.0         28.0       1446.0        414.4
          9.0        101.0          0.0       1192.0        200.8
         10.0         17.0          0.0        898.0         87.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.381           2.268           7.605           4.073
                   Theta0:           0.000           0.000           2.213           0.397               
                   Theta1:          27.852          10.098       99999.000       20030.720
Mean No. of pairwise diff.           3.803           2.942           6.550           4.005
           No. pol. sites:              19              50              79          63.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        787.0        263.0       1350.0        502.7
          1.0        892.0        336.0       1863.0        875.3
          2.0       1072.0        397.0       2227.0       1446.5
          3.0       1972.0        710.0       2609.0       1905.9
          4.0       2242.0        697.0       2288.0       1993.7
          5.0       2143.0       1027.0       2335.0       1705.4
          6.0       1186.0        568.0       1883.0       1227.7
          7.0        436.0        187.0       1764.0        761.4
          8.0        327.0         28.0       1446.0        414.4
          9.0        101.0          0.0       1192.0        200.8
         10.0         17.0          0.0        898.0         87.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.381           2.268           7.605           4.073
                   Theta0:           0.000           0.000           2.213           0.397               
                   Theta1:          27.852          10.098       99999.000       20030.720
Mean No. of pairwise diff.           3.803           2.942           6.550           4.005
           No. pol. sites:              19              50              79          63.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        787.0        263.0       1350.0        502.7
          1.0        892.0        336.0       1863.0        875.3
          2.0       1072.0        397.0       2227.0       1446.5
          3.0       1972.0        710.0       2609.0       1905.9
          4.0       2242.0        697.0       2288.0       1993.7
          5.0       2143.0       1027.0       2335.0       1705.4
          6.0       1186.0        568.0       1883.0       1227.7
          7.0        436.0        187.0       1764.0        761.4
          8.0        327.0         28.0       1446.0        414.4
          9.0        101.0          0.0       1192.0        200.8
         10.0         17.0          0.0        898.0         87.7
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_54_12_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            19      19.000       0.000
  No. of usable loci            19      19.000       0.000
  No. of polym. loci            19      19.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_54_12_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.19186     0.19186     0.00000     0.19186
     2       0.45602     0.45602     0.00000     0.45602
     3       0.01333     0.01333     0.00000     0.01333
     4       0.01333     0.01333     0.00000     0.01333
     5       0.12528     0.12528     0.00000     0.12528
     6       0.48322     0.48322     0.00000     0.48322
     7       0.06488     0.06488     0.00000     0.06488
     8       0.21262     0.21262     0.00000     0.21262
     9       0.24242     0.24242     0.00000     0.24242
    10       0.07732     0.07732     0.00000     0.07732
    11       0.03946     0.03946     0.00000     0.03946
    12       0.14819     0.14819     0.00000     0.14819
    13       0.37414     0.37414     0.00000     0.37414
    14       0.31400     0.31400     0.00000     0.31400
    15       0.26139     0.26139     0.00000     0.26139
    16       0.13682     0.13682     0.00000     0.13682
    17       0.02649     0.02649     0.00000     0.02649
    18       0.17038     0.17038     0.00000     0.17038
    19       0.45181     0.45181     0.00000     0.45181
--------------------------------------------------------
  Mean       0.20016     0.20016     0.00000     0.20016
  s.d.       0.15461     0.15461     0.00000     0.15461
--------------------------------------------------------
<A NAME="31_01_17at14_54_12_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_54_12_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      17.51385    17.51385     0.00000
            Theta_k_lower      11.89599    11.89599     0.00000
            Theta_k_upper      25.45445    25.45445     0.00000
                  Theta_H      11.72306    11.72306     0.00000
             s.d. Theta_H       2.16193     2.16193     0.00000
                  Theta_S       3.40227     3.40227     0.00000
             s.d. Theta_S       1.07542     1.07542     0.00000
                 Theta_pi       3.80295     3.80295     0.00000
            s.d. Theta_pi       2.13122     2.13122     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_54_12_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.38086     4.38086     0.00000
              Tau qt 2.5%       2.26758     2.26758     0.00000
                Tau qt 5%       2.26758     2.26758     0.00000
               Tau qt 95%       5.14648     5.14648     0.00000
             Tau qt 97.5%       5.14648     5.14648     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.21992     1.21992     0.00000
          Theta0 qt 97.5%       1.21992     1.21992     0.00000
                   Theta1      27.85156    27.85156     0.00000
           Theta1 qt 2.5%      10.09766    10.09766     0.00000
             Theta1 qt 5%      10.09766    10.09766     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00488     0.00488     0.00000
      Model (SSD) p-value       0.10000     0.10000     0.00000
         Raggedness index       0.01990     0.01990     0.00000
       Raggedness p-value       0.40000     0.40000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 113 (31/01/17 at 14:54:12))
Total computing time for this run : 0h 0m 0s 130 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

