
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_09"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 164 (31/01/17 at 14:44:09)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_09_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_09_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 48
No. of loci              : 69
No. of usable loci       : 69 loci with less than  5.00 % missing data
No. of polymorphic sites : 69


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.11356
     3       150         2   0.01333
     4       150         2   0.01333
     5       150         2   0.01333
     6       150         2   0.01333
     7       150         2   0.06488
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.05226
    11       150         2   0.11356
    12       150         2   0.02649
    13       150         2   0.01333
    14       150         2   0.03946
    15       150         2   0.02649
    16       150         2   0.11356
    17       150         2   0.02649
    18       150         2   0.02649
    19       150         2   0.08957
    20       150         2   0.10166
    21       150         2   0.03946
    22       150         2   0.08957
    23       150         2   0.07732
    24       150         2   0.05226
    25       150         2   0.06488
    26       150         2   0.12528
    27       150         2   0.01333
    28       150         2   0.01333
    29       150         2   0.03946
    30       150         2   0.01333
    31       150         2   0.05226
    32       150         2   0.01333
    33       150         2   0.01333
    34       150         2   0.02649
    35       150         2   0.01333
    36       150         2   0.03946
    37       150         2   0.02649
    38       150         2   0.01333
    39       150         2   0.03946
    40       150         2   0.23266
    41       150         2   0.01333
    42       150         2   0.02649
    43       150         2   0.02649
    44       150         2   0.03946
    45       150         2   0.01333
    46       150         2   0.01333
    47       150         2   0.03946
    48       150         2   0.23266
    49       150         2   0.01333
    50       150         2   0.01333
    51       150         2   0.01333
    52       150         2   0.01333
    53       150         2   0.05226
    54       150         2   0.02649
    55       150         2   0.01333
    56       150         2   0.10166
    57       150         2   0.06488
    58       150         2   0.03946
    59       150         2   0.01333
    60       150         2   0.02649
    61       150         2   0.02649
    62       150         2   0.01333
    63       150         2   0.01333
    64       150         2   0.03946
    65       150         2   0.05226
    66       150         2   0.02649
    67       150         2   0.13682
    68       150         2   0.02649
    69       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.04411
  s.d.     0.000     0.000   0.04603
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0779
Gene diversity           :  0.9283 +/-  0.0145

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 48

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 69
Number of usable nucleotide sites            : 69


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.043311 +/-     1.594256

Average gene diversity over loci              :     0.044106 +/-     0.025575

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    11.483192
  S.D. Theta(Hom) :     2.725056

         Theta(k) :    24.019287
95 % confidence interval limits for theta(k) : [  16.682825,  34.256022 ]

         Theta(S) :    12.355596
    S.D. Theta(S) :     3.117869

        Theta(Pi) :     3.043311
   S.D. Theta(Pi) :     1.764687

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     801     534.929
       1    1482    1623.787
       2    2065    2466.427
       3    2538    2499.597
       4    2016    1901.544
       5    1335    1158.315
       6     615     588.548
       7     242     256.585
       8      69      97.984
       9      12      33.298
      10       0      10.197
      11       0       2.842
      12       0       0.727
      13       0       0.172
      14       0       0.038
      15       0       0.008
      16       0       0.002
      17       0       0.000
      18       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.043
Mismatch observed variance          :        3.052
Tau                                 :        3.340
Theta0                              :        0.005
Theta1                              :       29.648

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00080061
          P(Sim. Ssd >= Obs. Ssd): 0.70000000
    Harpending's Raggedness index: 0.01965457
        P(Sim. Rag. >= Obs. Rag.): 0.70000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        801.0        825.4        705.1
          1.0       1482.0       1560.7       1480.5
          2.0       2065.0       2293.9       2198.2
          3.0       2538.0       2437.8       2345.6
          4.0       2016.0       1892.9       1921.3
          5.0       1335.0       1184.9       1270.7
          6.0        615.0        576.1        703.4
          7.0        242.0        253.5        334.6
          8.0         69.0         91.5        139.4
          9.0         12.0         39.2         51.7
         10.0          0.0         14.9         17.3
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.340           2.473           4.434           3.047
                   Theta0:           0.005           0.000           0.958           0.156               
                   Theta1:          29.648           6.094       99999.000       40029.123
Mean No. of pairwise diff.           3.043           2.034           4.148           2.995
           No. pol. sites:              69              52              78          62.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        801.0        408.0       2185.0        705.1
          1.0       1482.0        460.0       2323.0       1480.5
          2.0       2065.0        721.0       3350.0       2198.2
          3.0       2538.0       1906.0       3022.0       2345.6
          4.0       2016.0       1099.0       2794.0       1921.3
          5.0       1335.0        550.0       2261.0       1270.7
          6.0        615.0        213.0       1433.0        703.4
          7.0        242.0         66.0        688.0        334.6
          8.0         69.0          8.0        452.0        139.4
          9.0         12.0          0.0        296.0         51.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.340           2.473           4.434           3.047
                   Theta0:           0.005           0.000           0.958           0.156               
                   Theta1:          29.648           6.094       99999.000       40029.123
Mean No. of pairwise diff.           3.043           2.034           4.148           2.995
           No. pol. sites:              69              52              78          62.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        801.0        408.0       2185.0        705.1
          1.0       1482.0        460.0       2323.0       1480.5
          2.0       2065.0        721.0       3350.0       2198.2
          3.0       2538.0       1906.0       3022.0       2345.6
          4.0       2016.0       1099.0       2794.0       1921.3
          5.0       1335.0        550.0       2261.0       1270.7
          6.0        615.0        213.0       1433.0        703.4
          7.0        242.0         66.0        688.0        334.6
          8.0         69.0          8.0        452.0        139.4
          9.0         12.0          0.0        296.0         51.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.340           2.473           4.434           3.047
                   Theta0:           0.005           0.000           0.958           0.156               
                   Theta1:          29.648           6.094       99999.000       40029.123
Mean No. of pairwise diff.           3.043           2.034           4.148           2.995
           No. pol. sites:              69              52              78          62.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        801.0        408.0       2185.0        705.1
          1.0       1482.0        460.0       2323.0       1480.5
          2.0       2065.0        721.0       3350.0       2198.2
          3.0       2538.0       1906.0       3022.0       2345.6
          4.0       2016.0       1099.0       2794.0       1921.3
          5.0       1335.0        550.0       2261.0       1270.7
          6.0        615.0        213.0       1433.0        703.4
          7.0        242.0         66.0        688.0        334.6
          8.0         69.0          8.0        452.0        139.4
          9.0         12.0          0.0        296.0         51.7
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_09_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            69      69.000       0.000
  No. of usable loci            69      69.000       0.000
  No. of polym. loci            69      69.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_09_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.11356     0.11356     0.00000     0.11356
     3       0.01333     0.01333     0.00000     0.01333
     4       0.01333     0.01333     0.00000     0.01333
     5       0.01333     0.01333     0.00000     0.01333
     6       0.01333     0.01333     0.00000     0.01333
     7       0.06488     0.06488     0.00000     0.06488
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.05226     0.05226     0.00000     0.05226
    11       0.11356     0.11356     0.00000     0.11356
    12       0.02649     0.02649     0.00000     0.02649
    13       0.01333     0.01333     0.00000     0.01333
    14       0.03946     0.03946     0.00000     0.03946
    15       0.02649     0.02649     0.00000     0.02649
    16       0.11356     0.11356     0.00000     0.11356
    17       0.02649     0.02649     0.00000     0.02649
    18       0.02649     0.02649     0.00000     0.02649
    19       0.08957     0.08957     0.00000     0.08957
    20       0.10166     0.10166     0.00000     0.10166
    21       0.03946     0.03946     0.00000     0.03946
    22       0.08957     0.08957     0.00000     0.08957
    23       0.07732     0.07732     0.00000     0.07732
    24       0.05226     0.05226     0.00000     0.05226
    25       0.06488     0.06488     0.00000     0.06488
    26       0.12528     0.12528     0.00000     0.12528
    27       0.01333     0.01333     0.00000     0.01333
    28       0.01333     0.01333     0.00000     0.01333
    29       0.03946     0.03946     0.00000     0.03946
    30       0.01333     0.01333     0.00000     0.01333
    31       0.05226     0.05226     0.00000     0.05226
    32       0.01333     0.01333     0.00000     0.01333
    33       0.01333     0.01333     0.00000     0.01333
    34       0.02649     0.02649     0.00000     0.02649
    35       0.01333     0.01333     0.00000     0.01333
    36       0.03946     0.03946     0.00000     0.03946
    37       0.02649     0.02649     0.00000     0.02649
    38       0.01333     0.01333     0.00000     0.01333
    39       0.03946     0.03946     0.00000     0.03946
    40       0.23266     0.23266     0.00000     0.23266
    41       0.01333     0.01333     0.00000     0.01333
    42       0.02649     0.02649     0.00000     0.02649
    43       0.02649     0.02649     0.00000     0.02649
    44       0.03946     0.03946     0.00000     0.03946
    45       0.01333     0.01333     0.00000     0.01333
    46       0.01333     0.01333     0.00000     0.01333
    47       0.03946     0.03946     0.00000     0.03946
    48       0.23266     0.23266     0.00000     0.23266
    49       0.01333     0.01333     0.00000     0.01333
    50       0.01333     0.01333     0.00000     0.01333
    51       0.01333     0.01333     0.00000     0.01333
    52       0.01333     0.01333     0.00000     0.01333
    53       0.05226     0.05226     0.00000     0.05226
    54       0.02649     0.02649     0.00000     0.02649
    55       0.01333     0.01333     0.00000     0.01333
    56       0.10166     0.10166     0.00000     0.10166
    57       0.06488     0.06488     0.00000     0.06488
    58       0.03946     0.03946     0.00000     0.03946
    59       0.01333     0.01333     0.00000     0.01333
    60       0.02649     0.02649     0.00000     0.02649
    61       0.02649     0.02649     0.00000     0.02649
    62       0.01333     0.01333     0.00000     0.01333
    63       0.01333     0.01333     0.00000     0.01333
    64       0.03946     0.03946     0.00000     0.03946
    65       0.05226     0.05226     0.00000     0.05226
    66       0.02649     0.02649     0.00000     0.02649
    67       0.13682     0.13682     0.00000     0.13682
    68       0.02649     0.02649     0.00000     0.02649
    69       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.04411     0.04411     0.00000     0.04411
  s.d.       0.04603     0.04603     0.00000     0.04603
--------------------------------------------------------
<A NAME="31_01_17at14_44_09_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_09_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      24.01929    24.01929     0.00000
            Theta_k_lower      16.68282    16.68282     0.00000
            Theta_k_upper      34.25602    34.25602     0.00000
                  Theta_H      11.48319    11.48319     0.00000
             s.d. Theta_H       2.72506     2.72506     0.00000
                  Theta_S      12.35560    12.35560     0.00000
             s.d. Theta_S       3.11787     3.11787     0.00000
                 Theta_pi       3.04331     3.04331     0.00000
            s.d. Theta_pi       1.76469     1.76469     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_09_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.33984     3.33984     0.00000
              Tau qt 2.5%       2.47266     2.47266     0.00000
                Tau qt 5%       2.47266     2.47266     0.00000
               Tau qt 95%       3.49609     3.49609     0.00000
             Tau qt 97.5%       3.49609     3.49609     0.00000
                   Theta0       0.00527     0.00527     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.56602     0.56602     0.00000
          Theta0 qt 97.5%       0.56602     0.56602     0.00000
                   Theta1      29.64844    29.64844     0.00000
           Theta1 qt 2.5%       6.09375     6.09375     0.00000
             Theta1 qt 5%       6.09375     6.09375     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00080     0.00080     0.00000
      Model (SSD) p-value       0.70000     0.70000     0.00000
         Raggedness index       0.01965     0.01965     0.00000
       Raggedness p-value       0.70000     0.70000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 164 (31/01/17 at 14:44:09))
Total computing time for this run : 0h 0m 0s 112 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

