
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_32_05"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 191 (31/01/17 at 16:32:05)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_32_05_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_32_05_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 65
No. of loci              : 22
No. of usable loci       : 22 loci with less than  5.00 % missing data
No. of polymorphic sites : 22


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.23266
     3       150         2   0.42282
     4       150         2   0.05226
     5       150         2   0.02649
     6       150         2   0.22273
     7       150         2   0.31400
     8       150         2   0.01333
     9       150         2   0.05226
    10       150         2   0.46004
    11       150         2   0.19186
    12       150         2   0.03946
    13       150         2   0.34550
    14       150         2   0.19186
    15       150         2   0.02649
    16       150         2   0.03946
    17       150         2   0.24242
    18       150         2   0.18121
    19       150         2   0.01333
    20       150         2   0.02649
    21       150         2   0.22273
    22       150         2   0.13682
-------------------------------------------------
  Mean   150.000     2.000   0.15762
  s.d.     0.000     0.000   0.13979
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0336
Gene diversity           :  0.9729 +/-  0.0050

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 65

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 22
Number of usable nucleotide sites            : 22


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.467562 +/-     1.779439

Average gene diversity over loci              :     0.157616 +/-     0.089530

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    34.093870
  S.D. Theta(Hom) :     6.799476

         Theta(k) :    43.063016
95 % confidence interval limits for theta(k) : [  30.727796,  60.104460 ]

         Theta(S) :     3.939466
    S.D. Theta(S) :     1.201985

        Theta(Pi) :     3.467562
   S.D. Theta(Pi) :     1.969667

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     303       0.000
       1     907       0.000
       2    1825       0.000
       3    2727       0.000
       4    2590       0.000
       5    1720       0.000
       6     773       0.000
       7     248       0.000
       8      64       0.000
       9      14       0.000
      10       4       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.468
Mismatch observed variance          :        2.544
Tau                                 :        3.635
Theta0                              :        0.000
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00417207
          P(Sim. Ssd >= Obs. Ssd): 0.10000000
    Harpending's Raggedness index: 0.03207638
        P(Sim. Rag. >= Obs. Rag.): 0.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        303.0        279.8        295.0
          1.0        907.0       1050.9       1072.0
          2.0       1825.0       1950.8       1948.1
          3.0       2727.0       2458.3       2360.3
          4.0       2590.0       2243.9       2144.7
          5.0       1720.0       1623.4       1559.1
          6.0        773.0        897.0        944.5
          7.0        248.0        416.9        490.4
          8.0         64.0        168.3        222.8
          9.0         14.0         59.6         90.0
         10.0          4.0         18.3         32.7
         11.0          0.0          6.4         10.8
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.635           3.396           4.016           3.596
                   Theta0:           0.000           0.000           0.000           0.000               
                   Theta1:       99999.000         129.781       99999.000       90012.078
Mean No. of pairwise diff.           3.468           3.327           3.937           3.568
           No. pol. sites:              22             185             205         196.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        303.0        120.0        378.0        295.0
          1.0        907.0        641.0       1199.0       1072.0
          2.0       1825.0       1528.0       2252.0       1948.1
          3.0       2727.0       2171.0       2751.0       2360.3
          4.0       2590.0       2011.0       2638.0       2144.7
          5.0       1720.0       1440.0       2106.0       1559.1
          6.0        773.0        643.0       1184.0        944.5
          7.0        248.0        265.0        669.0        490.4
          8.0         64.0         97.0        310.0        222.8
          9.0         14.0         12.0        120.0         90.0
         10.0          4.0          0.0         55.0         32.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.635           3.396           4.016           3.596
                   Theta0:           0.000           0.000           0.000           0.000               
                   Theta1:       99999.000         129.781       99999.000       90012.078
Mean No. of pairwise diff.           3.468           3.327           3.937           3.568
           No. pol. sites:              22             185             205         196.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        303.0        120.0        378.0        295.0
          1.0        907.0        641.0       1199.0       1072.0
          2.0       1825.0       1528.0       2252.0       1948.1
          3.0       2727.0       2171.0       2751.0       2360.3
          4.0       2590.0       2011.0       2638.0       2144.7
          5.0       1720.0       1440.0       2106.0       1559.1
          6.0        773.0        643.0       1184.0        944.5
          7.0        248.0        265.0        669.0        490.4
          8.0         64.0         97.0        310.0        222.8
          9.0         14.0         12.0        120.0         90.0
         10.0          4.0          0.0         55.0         32.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.635           3.396           4.016           3.596
                   Theta0:           0.000           0.000           0.000           0.000               
                   Theta1:       99999.000         129.781       99999.000       90012.078
Mean No. of pairwise diff.           3.468           3.327           3.937           3.568
           No. pol. sites:              22             185             205         196.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        303.0        120.0        378.0        295.0
          1.0        907.0        641.0       1199.0       1072.0
          2.0       1825.0       1528.0       2252.0       1948.1
          3.0       2727.0       2171.0       2751.0       2360.3
          4.0       2590.0       2011.0       2638.0       2144.7
          5.0       1720.0       1440.0       2106.0       1559.1
          6.0        773.0        643.0       1184.0        944.5
          7.0        248.0        265.0        669.0        490.4
          8.0         64.0         97.0        310.0        222.8
          9.0         14.0         12.0        120.0         90.0
         10.0          4.0          0.0         55.0         32.7
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_32_05_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            22      22.000       0.000
  No. of usable loci            22      22.000       0.000
  No. of polym. loci            22      22.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_32_05_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.23266     0.23266     0.00000     0.23266
     3       0.42282     0.42282     0.00000     0.42282
     4       0.05226     0.05226     0.00000     0.05226
     5       0.02649     0.02649     0.00000     0.02649
     6       0.22273     0.22273     0.00000     0.22273
     7       0.31400     0.31400     0.00000     0.31400
     8       0.01333     0.01333     0.00000     0.01333
     9       0.05226     0.05226     0.00000     0.05226
    10       0.46004     0.46004     0.00000     0.46004
    11       0.19186     0.19186     0.00000     0.19186
    12       0.03946     0.03946     0.00000     0.03946
    13       0.34550     0.34550     0.00000     0.34550
    14       0.19186     0.19186     0.00000     0.19186
    15       0.02649     0.02649     0.00000     0.02649
    16       0.03946     0.03946     0.00000     0.03946
    17       0.24242     0.24242     0.00000     0.24242
    18       0.18121     0.18121     0.00000     0.18121
    19       0.01333     0.01333     0.00000     0.01333
    20       0.02649     0.02649     0.00000     0.02649
    21       0.22273     0.22273     0.00000     0.22273
    22       0.13682     0.13682     0.00000     0.13682
--------------------------------------------------------
  Mean       0.15762     0.15762     0.00000     0.15762
  s.d.       0.13979     0.13979     0.00000     0.13979
--------------------------------------------------------
<A NAME="31_01_17at16_32_05_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_32_05_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      43.06302    43.06302     0.00000
            Theta_k_lower      30.72780    30.72780     0.00000
            Theta_k_upper      60.10446    60.10446     0.00000
                  Theta_H      34.09387    34.09387     0.00000
             s.d. Theta_H       6.79948     6.79948     0.00000
                  Theta_S       3.93947     3.93947     0.00000
             s.d. Theta_S       1.20198     1.20198     0.00000
                 Theta_pi       3.46756     3.46756     0.00000
            s.d. Theta_pi       1.96967     1.96967     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_32_05_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.63477     3.63477     0.00000
              Tau qt 2.5%       3.39648     3.39648     0.00000
                Tau qt 5%       3.39648     3.39648     0.00000
               Tau qt 95%       3.82031     3.82031     0.00000
             Tau qt 97.5%       3.82031     3.82031     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00000     0.00000     0.00000
          Theta0 qt 97.5%       0.00000     0.00000     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%     129.78125   129.78125     0.00000
             Theta1 qt 5%     129.78125   129.78125     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00417     0.00417     0.00000
      Model (SSD) p-value       0.10000     0.10000     0.00000
         Raggedness index       0.03208     0.03208     0.00000
       Raggedness p-value       0.00000     0.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 191 (31/01/17 at 16:32:05))
Total computing time for this run : 0h 0m 0s 115 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

