
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_43_59"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 84 (31/01/17 at 14:43:59)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_43_59_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_43_59_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 43
No. of loci              : 62
No. of usable loci       : 62 loci with less than  5.00 % missing data
No. of polymorphic sites : 62


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.02649
     3       150         2   0.01333
     4       150         2   0.01333
     5       150         2   0.01333
     6       150         2   0.01333
     7       150         2   0.01333
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.02649
    11       150         2   0.03946
    12       150         2   0.03946
    13       150         2   0.01333
    14       150         2   0.01333
    15       150         2   0.02649
    16       150         2   0.03946
    17       150         2   0.01333
    18       150         2   0.01333
    19       150         2   0.02649
    20       150         2   0.02649
    21       150         2   0.03946
    22       150         2   0.10166
    23       150         2   0.02649
    24       150         2   0.11356
    25       150         2   0.01333
    26       150         2   0.02649
    27       150         2   0.14819
    28       150         2   0.02649
    29       150         2   0.02649
    30       150         2   0.18121
    31       150         2   0.13682
    32       150         2   0.01333
    33       150         2   0.06488
    34       150         2   0.01333
    35       150         2   0.01333
    36       150         2   0.01333
    37       150         2   0.01333
    38       150         2   0.01333
    39       150         2   0.01333
    40       150         2   0.01333
    41       150         2   0.03946
    42       150         2   0.01333
    43       150         2   0.11356
    44       150         2   0.11356
    45       150         2   0.11356
    46       150         2   0.06488
    47       150         2   0.01333
    48       150         2   0.06488
    49       150         2   0.02649
    50       150         2   0.02649
    51       150         2   0.01333
    52       150         2   0.03946
    53       150         2   0.01333
    54       150         2   0.06488
    55       150         2   0.05226
    56       150         2   0.01333
    57       150         2   0.01333
    58       150         2   0.01333
    59       150         2   0.03946
    60       150         2   0.01333
    61       150         2   0.01333
    62       150         2   0.03946
-------------------------------------------------
  Mean   150.000     2.000   0.03776
  s.d.     0.000     0.000   0.03886
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.1154
Gene diversity           :  0.8906 +/-  0.0212

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 43

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 62
Number of usable nucleotide sites            : 62


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.340940 +/-     1.285615

Average gene diversity over loci              :     0.037757 +/-     0.022952

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     6.900047
  S.D. Theta(Hom) :     1.648681

         Theta(k) :    19.804082
95 % confidence interval limits for theta(k) : [  13.577772,  28.555190 ]

         Theta(S) :    11.102130
    S.D. Theta(S) :     2.835556

        Theta(Pi) :     2.340940
   S.D. Theta(Pi) :     1.423051

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1223    1147.142
       1    2500    2552.310
       2    2712    2886.765
       3    2342    2220.359
       4    1388    1311.811
       5     624     638.101
       6     239     267.718
       7      94     100.291
       8      34      34.482
       9      16      11.129
      10       3       3.435
      11       0       1.029
      12       0       0.303
      13       0       0.088
      14       0       0.025
      15       0       0.007
      16       0       0.002
      17       0       0.001
      18       0       0.000
      19       0       0.000
      20       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.341
Mismatch observed variance          :        2.504
Tau                                 :        2.312
Theta0                              :        0.135
Theta1                              :       29.648

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00022230
          P(Sim. Ssd >= Obs. Ssd): 1.00000000
    Harpending's Raggedness index: 0.02786451
        P(Sim. Rag. >= Obs. Rag.): 0.80000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1223.0       1221.5       1232.7
          1.0       2500.0       2454.4       2435.2
          2.0       2712.0       2921.3       2817.6
          3.0       2342.0       2152.1       2248.8
          4.0       1388.0       1254.1       1363.8
          5.0        624.0        715.9        666.6
          6.0        239.0        310.9        273.0
          7.0         94.0        102.1         96.3
          8.0         34.0         31.5         29.9
          9.0         16.0          9.9          8.3
         10.0          3.0          1.3          2.1
         11.0          0.0          0.0          0.5
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.312           0.000           3.797           2.115
                   Theta0:           0.135           0.000           0.814           0.084               
                   Theta1:          29.648         209.648       99999.000       80084.130
Mean No. of pairwise diff.           2.341           1.777           3.654           2.352
           No. pol. sites:              62              43              67          51.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1223.0        404.0       3655.0       1232.7
          1.0       2500.0        862.0       3541.0       2435.2
          2.0       2712.0       1718.0       4200.0       2817.6
          3.0       2342.0       1405.0       2759.0       2248.8
          4.0       1388.0        626.0       2250.0       1363.8
          5.0        624.0        278.0       1766.0        666.6
          6.0        239.0         89.0       1024.0        273.0
          7.0         94.0         13.0        504.0         96.3
          8.0         34.0          0.0        176.0         29.9
          9.0         16.0          0.0         36.0          8.3
         10.0          3.0          0.0          9.0          2.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.312           0.000           3.797           2.115
                   Theta0:           0.135           0.000           0.814           0.084               
                   Theta1:          29.648         209.648       99999.000       80084.130
Mean No. of pairwise diff.           2.341           1.777           3.654           2.352
           No. pol. sites:              62              43              67          51.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1223.0        404.0       3655.0       1232.7
          1.0       2500.0        862.0       3541.0       2435.2
          2.0       2712.0       1718.0       4200.0       2817.6
          3.0       2342.0       1405.0       2759.0       2248.8
          4.0       1388.0        626.0       2250.0       1363.8
          5.0        624.0        278.0       1766.0        666.6
          6.0        239.0         89.0       1024.0        273.0
          7.0         94.0         13.0        504.0         96.3
          8.0         34.0          0.0        176.0         29.9
          9.0         16.0          0.0         36.0          8.3
         10.0          3.0          0.0          9.0          2.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.312           0.000           3.797           2.115
                   Theta0:           0.135           0.000           0.814           0.084               
                   Theta1:          29.648         209.648       99999.000       80084.130
Mean No. of pairwise diff.           2.341           1.777           3.654           2.352
           No. pol. sites:              62              43              67          51.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1223.0        404.0       3655.0       1232.7
          1.0       2500.0        862.0       3541.0       2435.2
          2.0       2712.0       1718.0       4200.0       2817.6
          3.0       2342.0       1405.0       2759.0       2248.8
          4.0       1388.0        626.0       2250.0       1363.8
          5.0        624.0        278.0       1766.0        666.6
          6.0        239.0         89.0       1024.0        273.0
          7.0         94.0         13.0        504.0         96.3
          8.0         34.0          0.0        176.0         29.9
          9.0         16.0          0.0         36.0          8.3
         10.0          3.0          0.0          9.0          2.1
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_43_59_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            62      62.000       0.000
  No. of usable loci            62      62.000       0.000
  No. of polym. loci            62      62.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_43_59_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.02649     0.02649     0.00000     0.02649
     3       0.01333     0.01333     0.00000     0.01333
     4       0.01333     0.01333     0.00000     0.01333
     5       0.01333     0.01333     0.00000     0.01333
     6       0.01333     0.01333     0.00000     0.01333
     7       0.01333     0.01333     0.00000     0.01333
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.02649     0.02649     0.00000     0.02649
    11       0.03946     0.03946     0.00000     0.03946
    12       0.03946     0.03946     0.00000     0.03946
    13       0.01333     0.01333     0.00000     0.01333
    14       0.01333     0.01333     0.00000     0.01333
    15       0.02649     0.02649     0.00000     0.02649
    16       0.03946     0.03946     0.00000     0.03946
    17       0.01333     0.01333     0.00000     0.01333
    18       0.01333     0.01333     0.00000     0.01333
    19       0.02649     0.02649     0.00000     0.02649
    20       0.02649     0.02649     0.00000     0.02649
    21       0.03946     0.03946     0.00000     0.03946
    22       0.10166     0.10166     0.00000     0.10166
    23       0.02649     0.02649     0.00000     0.02649
    24       0.11356     0.11356     0.00000     0.11356
    25       0.01333     0.01333     0.00000     0.01333
    26       0.02649     0.02649     0.00000     0.02649
    27       0.14819     0.14819     0.00000     0.14819
    28       0.02649     0.02649     0.00000     0.02649
    29       0.02649     0.02649     0.00000     0.02649
    30       0.18121     0.18121     0.00000     0.18121
    31       0.13682     0.13682     0.00000     0.13682
    32       0.01333     0.01333     0.00000     0.01333
    33       0.06488     0.06488     0.00000     0.06488
    34       0.01333     0.01333     0.00000     0.01333
    35       0.01333     0.01333     0.00000     0.01333
    36       0.01333     0.01333     0.00000     0.01333
    37       0.01333     0.01333     0.00000     0.01333
    38       0.01333     0.01333     0.00000     0.01333
    39       0.01333     0.01333     0.00000     0.01333
    40       0.01333     0.01333     0.00000     0.01333
    41       0.03946     0.03946     0.00000     0.03946
    42       0.01333     0.01333     0.00000     0.01333
    43       0.11356     0.11356     0.00000     0.11356
    44       0.11356     0.11356     0.00000     0.11356
    45       0.11356     0.11356     0.00000     0.11356
    46       0.06488     0.06488     0.00000     0.06488
    47       0.01333     0.01333     0.00000     0.01333
    48       0.06488     0.06488     0.00000     0.06488
    49       0.02649     0.02649     0.00000     0.02649
    50       0.02649     0.02649     0.00000     0.02649
    51       0.01333     0.01333     0.00000     0.01333
    52       0.03946     0.03946     0.00000     0.03946
    53       0.01333     0.01333     0.00000     0.01333
    54       0.06488     0.06488     0.00000     0.06488
    55       0.05226     0.05226     0.00000     0.05226
    56       0.01333     0.01333     0.00000     0.01333
    57       0.01333     0.01333     0.00000     0.01333
    58       0.01333     0.01333     0.00000     0.01333
    59       0.03946     0.03946     0.00000     0.03946
    60       0.01333     0.01333     0.00000     0.01333
    61       0.01333     0.01333     0.00000     0.01333
    62       0.03946     0.03946     0.00000     0.03946
--------------------------------------------------------
  Mean       0.03776     0.03776     0.00000     0.03776
  s.d.       0.03886     0.03886     0.00000     0.03886
--------------------------------------------------------
<A NAME="31_01_17at14_43_59_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_43_59_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      19.80408    19.80408     0.00000
            Theta_k_lower      13.57777    13.57777     0.00000
            Theta_k_upper      28.55519    28.55519     0.00000
                  Theta_H       6.90005     6.90005     0.00000
             s.d. Theta_H       1.64868     1.64868     0.00000
                  Theta_S      11.10213    11.10213     0.00000
             s.d. Theta_S       2.83556     2.83556     0.00000
                 Theta_pi       2.34094     2.34094     0.00000
            s.d. Theta_pi       1.42305     1.42305     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_43_59_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       2.31250     2.31250     0.00000
              Tau qt 2.5%       0.00000     0.00000     0.00000
                Tau qt 5%       0.00000     0.00000     0.00000
               Tau qt 95%       2.68750     2.68750     0.00000
             Tau qt 97.5%       2.68750     2.68750     0.00000
                   Theta0       0.13535     0.13535     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.01230     0.01230     0.00000
          Theta0 qt 97.5%       0.01230     0.01230     0.00000
                   Theta1      29.64844    29.64844     0.00000
           Theta1 qt 2.5%     209.64844   209.64844     0.00000
             Theta1 qt 5%     209.64844   209.64844     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00022     0.00022     0.00000
      Model (SSD) p-value       1.00000     1.00000     0.00000
         Raggedness index       0.02786     0.02786     0.00000
       Raggedness p-value       0.80000     0.80000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 84 (31/01/17 at 14:43:59))
Total computing time for this run : 0h 0m 0s 125 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

