
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_30_18"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 9 (31/01/17 at 16:30:18)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_30_18_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_30_18_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 42
No. of loci              : 19
No. of usable loci       : 19 loci with less than  5.00 % missing data
No. of polymorphic sites : 19


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.08957
     2       150         2   0.12528
     3       150         2   0.03946
     4       150         2   0.05226
     5       150         2   0.27060
     6       150         2   0.01333
     7       150         2   0.27964
     8       150         2   0.11356
     9       150         2   0.01333
    10       150         2   0.10166
    11       150         2   0.27060
    12       150         2   0.05226
    13       150         2   0.13682
    14       150         2   0.24242
    15       150         2   0.05226
    16       150         2   0.01333
    17       150         2   0.03946
    18       150         2   0.27060
    19       150         2   0.06488
-------------------------------------------------
  Mean   150.000     2.000   0.11797
  s.d.     0.000     0.000   0.09831
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.1532
Gene diversity           :  0.8524 +/-  0.0268

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 42

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 19
Number of usable nucleotide sites            : 19


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.241342 +/-     1.241551

Average gene diversity over loci              :     0.117965 +/-     0.072330

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     4.734381
  S.D. Theta(Hom) :     1.107315

         Theta(k) :    19.021744
95 % confidence interval limits for theta(k) : [  13.002730,  27.496429 ]

         Theta(S) :     3.402266
    S.D. Theta(S) :     1.075424

        Theta(Pi) :     2.241342
   S.D. Theta(Pi) :     1.374277

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1649    1418.965
       1    2747    2752.907
       2    2358    2794.713
       3    1809    2002.746
       4    1418    1155.105
       5     797     579.884
       6     308     267.156
       7      68     117.130
       8      13      50.034
       9       8      21.112
      10       0       8.862
      11       0       3.713
      12       0       1.554
      13       0       0.651
      14       0       0.272
      15       0       0.114
      16       0       0.048
      17       0       0.020
      18       0       0.008


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.241
Mismatch observed variance          :        2.759
Tau                                 :        1.406
Theta0                              :        1.102
Theta1                              :       17.891

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00135220
          P(Sim. Ssd >= Obs. Ssd): 0.70000000
    Harpending's Raggedness index: 0.01999256
        P(Sim. Rag. >= Obs. Rag.): 0.90000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1649.0       1680.2       1661.8
          1.0       2747.0       2365.5       2569.8
          2.0       2358.0       2681.9       2509.1
          3.0       1809.0       1733.8       1853.6
          4.0       1418.0       1288.3       1159.8
          5.0        797.0        719.4        660.7
          6.0        308.0        463.7        358.7
          7.0         68.0        197.9        190.5
          8.0         13.0         39.0        100.3
          9.0          8.0          5.1         52.7
         10.0          0.0          0.2         27.6
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.406           0.498           4.766           2.202
                   Theta0:           1.102           0.000           0.350           0.074               
                   Theta1:          17.891           5.352       99999.000       50004.540
Mean No. of pairwise diff.           2.241           1.751           3.580           2.345
           No. pol. sites:              19              31              44          36.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1649.0        912.0       2781.0       1661.8
          1.0       2747.0        834.0       3441.0       2569.8
          2.0       2358.0        995.0       3994.0       2509.1
          3.0       1809.0        467.0       2574.0       1853.6
          4.0       1418.0        474.0       3739.0       1159.8
          5.0        797.0        329.0       1581.0        660.7
          6.0        308.0         80.0       1566.0        358.7
          7.0         68.0          0.0        675.0        190.5
          8.0         13.0          0.0        171.0        100.3
          9.0          8.0          0.0         19.0         52.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.406           0.498           4.766           2.202
                   Theta0:           1.102           0.000           0.350           0.074               
                   Theta1:          17.891           5.352       99999.000       50004.540
Mean No. of pairwise diff.           2.241           1.751           3.580           2.345
           No. pol. sites:              19              31              44          36.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1649.0        912.0       2781.0       1661.8
          1.0       2747.0        834.0       3441.0       2569.8
          2.0       2358.0        995.0       3994.0       2509.1
          3.0       1809.0        467.0       2574.0       1853.6
          4.0       1418.0        474.0       3739.0       1159.8
          5.0        797.0        329.0       1581.0        660.7
          6.0        308.0         80.0       1566.0        358.7
          7.0         68.0          0.0        675.0        190.5
          8.0         13.0          0.0        171.0        100.3
          9.0          8.0          0.0         19.0         52.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.406           0.498           4.766           2.202
                   Theta0:           1.102           0.000           0.350           0.074               
                   Theta1:          17.891           5.352       99999.000       50004.540
Mean No. of pairwise diff.           2.241           1.751           3.580           2.345
           No. pol. sites:              19              31              44          36.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1649.0        912.0       2781.0       1661.8
          1.0       2747.0        834.0       3441.0       2569.8
          2.0       2358.0        995.0       3994.0       2509.1
          3.0       1809.0        467.0       2574.0       1853.6
          4.0       1418.0        474.0       3739.0       1159.8
          5.0        797.0        329.0       1581.0        660.7
          6.0        308.0         80.0       1566.0        358.7
          7.0         68.0          0.0        675.0        190.5
          8.0         13.0          0.0        171.0        100.3
          9.0          8.0          0.0         19.0         52.7
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_30_18_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            19      19.000       0.000
  No. of usable loci            19      19.000       0.000
  No. of polym. loci            19      19.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_30_18_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.08957     0.08957     0.00000     0.08957
     2       0.12528     0.12528     0.00000     0.12528
     3       0.03946     0.03946     0.00000     0.03946
     4       0.05226     0.05226     0.00000     0.05226
     5       0.27060     0.27060     0.00000     0.27060
     6       0.01333     0.01333     0.00000     0.01333
     7       0.27964     0.27964     0.00000     0.27964
     8       0.11356     0.11356     0.00000     0.11356
     9       0.01333     0.01333     0.00000     0.01333
    10       0.10166     0.10166     0.00000     0.10166
    11       0.27060     0.27060     0.00000     0.27060
    12       0.05226     0.05226     0.00000     0.05226
    13       0.13682     0.13682     0.00000     0.13682
    14       0.24242     0.24242     0.00000     0.24242
    15       0.05226     0.05226     0.00000     0.05226
    16       0.01333     0.01333     0.00000     0.01333
    17       0.03946     0.03946     0.00000     0.03946
    18       0.27060     0.27060     0.00000     0.27060
    19       0.06488     0.06488     0.00000     0.06488
--------------------------------------------------------
  Mean       0.11797     0.11797     0.00000     0.11797
  s.d.       0.09831     0.09831     0.00000     0.09831
--------------------------------------------------------
<A NAME="31_01_17at16_30_18_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_30_18_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      19.02174    19.02174     0.00000
            Theta_k_lower      13.00273    13.00273     0.00000
            Theta_k_upper      27.49643    27.49643     0.00000
                  Theta_H       4.73438     4.73438     0.00000
             s.d. Theta_H       1.10732     1.10732     0.00000
                  Theta_S       3.40227     3.40227     0.00000
             s.d. Theta_S       1.07542     1.07542     0.00000
                 Theta_pi       2.24134     2.24134     0.00000
            s.d. Theta_pi       1.37428     1.37428     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_30_18_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.40625     1.40625     0.00000
              Tau qt 2.5%       0.49805     0.49805     0.00000
                Tau qt 5%       0.49805     0.49805     0.00000
               Tau qt 95%       2.65430     2.65430     0.00000
             Tau qt 97.5%       2.65430     2.65430     0.00000
                   Theta0       1.10215     1.10215     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.15820     0.15820     0.00000
          Theta0 qt 97.5%       0.15820     0.15820     0.00000
                   Theta1      17.89062    17.89062     0.00000
           Theta1 qt 2.5%       5.35156     5.35156     0.00000
             Theta1 qt 5%       5.35156     5.35156     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00135     0.00135     0.00000
      Model (SSD) p-value       0.70000     0.70000     0.00000
         Raggedness index       0.01999     0.01999     0.00000
       Raggedness p-value       0.90000     0.90000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 9 (31/01/17 at 16:30:18))
Total computing time for this run : 0h 0m 0s 143 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

