
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_40_43"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 85 (31/01/17 at 16:40:43)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_40_43_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_40_43_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 81
No. of loci              : 27
No. of usable loci       : 27 loci with less than  5.00 % missing data
No. of polymorphic sites : 27


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.07732
     3       150         2   0.38738
     4       150         2   0.27060
     5       150         2   0.02649
     6       150         2   0.02649
     7       150         2   0.23266
     8       150         2   0.47105
     9       150         2   0.37414
    10       150         2   0.11356
    11       150         2   0.01333
    12       150         2   0.10166
    13       150         2   0.41172
    14       150         2   0.36018
    15       150         2   0.01333
    16       150         2   0.15937
    17       150         2   0.43812
    18       150         2   0.23266
    19       150         2   0.03946
    20       150         2   0.25199
    21       150         2   0.47749
    22       150         2   0.14819
    23       150         2   0.01333
    24       150         2   0.40591
    25       150         2   0.34550
    26       150         2   0.22273
    27       150         2   0.10166
-------------------------------------------------
  Mean   150.000     2.000   0.21221
  s.d.     0.000     0.000   0.16226
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0194
Gene diversity           :  0.9872 +/-  0.0028

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 81

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 27
Number of usable nucleotide sites            : 27


Distance method : Pairwise difference
Mean number of pairwise differences           :     5.729664 +/-     2.759471

Average gene diversity over loci              :     0.212210 +/-     0.113128

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    75.248430
  S.D. Theta(Hom) :    17.046614

         Theta(k) :    71.088590
95 % confidence interval limits for theta(k) : [  51.152967,  98.819247 ]

         Theta(S) :     4.834799
    S.D. Theta(S) :     1.410378

        Theta(Pi) :     5.729664
   S.D. Theta(Pi) :     3.054468

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     143       0.000
       1     298       0.000
       2     526       0.000
       3     902       0.000
       4    1397       0.000
       5    1951       0.000
       6    1944       0.000
       7    1630       0.000
       8    1074       0.000
       9     640       0.000
      10     366       0.000
      11     164       0.000
      12      90       0.000
      13      40       0.000
      14      10       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        5.730
Mismatch observed variance          :        5.668
Tau                                 :        5.926
Theta0                              :        0.000
Theta1                              :      542.500

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00111594
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.01193395
        P(Sim. Rag. >= Obs. Rag.): 0.20000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        143.0         63.1         50.0
          1.0        298.0        228.2        195.0
          2.0        526.0        599.5        537.3
          3.0        902.0       1057.7       1040.7
          4.0       1397.0       1505.3       1530.6
          5.0       1951.0       1758.7       1808.0
          6.0       1944.0       1715.1       1782.4
          7.0       1630.0       1478.8       1507.3
          8.0       1074.0       1106.8       1115.8
          9.0        640.0        729.1        734.3
         10.0        366.0        445.6        435.0
         11.0        164.0        242.3        234.3
         12.0         90.0        120.2        115.7
         13.0         40.0         61.5         52.7
         14.0         10.0         30.6         22.3
         15.0          0.0         15.9          8.8
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           5.926           4.887           6.332           5.543
                   Theta0:           0.000           0.000           1.248           0.463               
                   Theta1:         542.500         103.984       99999.000       20167.300
Mean No. of pairwise diff.           5.730           5.360           6.723           5.886
           No. pol. sites:              27             206             240         221.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        143.0         21.0        110.0         50.0
          1.0        298.0        122.0        379.0        195.0
          2.0        526.0        416.0        824.0        537.3
          3.0        902.0        812.0       1346.0       1040.7
          4.0       1397.0       1145.0       1928.0       1530.6
          5.0       1951.0       1396.0       2137.0       1808.0
          6.0       1944.0       1552.0       1884.0       1782.4
          7.0       1630.0       1301.0       1630.0       1507.3
          8.0       1074.0        832.0       1321.0       1115.8
          9.0        640.0        484.0        992.0        734.3
         10.0        366.0        224.0        733.0        435.0
         11.0        164.0        109.0        481.0        234.3
         12.0         90.0         41.0        277.0        115.7
         13.0         40.0         10.0        178.0         52.7
         14.0         10.0          0.0         89.0         22.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           5.926           4.887           6.332           5.543
                   Theta0:           0.000           0.000           1.248           0.463               
                   Theta1:         542.500         103.984       99999.000       20167.300
Mean No. of pairwise diff.           5.730           5.360           6.723           5.886
           No. pol. sites:              27             206             240         221.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        143.0         21.0        110.0         50.0
          1.0        298.0        122.0        379.0        195.0
          2.0        526.0        416.0        824.0        537.3
          3.0        902.0        812.0       1346.0       1040.7
          4.0       1397.0       1145.0       1928.0       1530.6
          5.0       1951.0       1396.0       2137.0       1808.0
          6.0       1944.0       1552.0       1884.0       1782.4
          7.0       1630.0       1301.0       1630.0       1507.3
          8.0       1074.0        832.0       1321.0       1115.8
          9.0        640.0        484.0        992.0        734.3
         10.0        366.0        224.0        733.0        435.0
         11.0        164.0        109.0        481.0        234.3
         12.0         90.0         41.0        277.0        115.7
         13.0         40.0         10.0        178.0         52.7
         14.0         10.0          0.0         89.0         22.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           5.926           4.887           6.332           5.543
                   Theta0:           0.000           0.000           1.248           0.463               
                   Theta1:         542.500         103.984       99999.000       20167.300
Mean No. of pairwise diff.           5.730           5.360           6.723           5.886
           No. pol. sites:              27             206             240         221.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        143.0         21.0        110.0         50.0
          1.0        298.0        122.0        379.0        195.0
          2.0        526.0        416.0        824.0        537.3
          3.0        902.0        812.0       1346.0       1040.7
          4.0       1397.0       1145.0       1928.0       1530.6
          5.0       1951.0       1396.0       2137.0       1808.0
          6.0       1944.0       1552.0       1884.0       1782.4
          7.0       1630.0       1301.0       1630.0       1507.3
          8.0       1074.0        832.0       1321.0       1115.8
          9.0        640.0        484.0        992.0        734.3
         10.0        366.0        224.0        733.0        435.0
         11.0        164.0        109.0        481.0        234.3
         12.0         90.0         41.0        277.0        115.7
         13.0         40.0         10.0        178.0         52.7
         14.0         10.0          0.0         89.0         22.3
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_40_43_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            27      27.000       0.000
  No. of usable loci            27      27.000       0.000
  No. of polym. loci            27      27.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_40_43_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.07732     0.07732     0.00000     0.07732
     3       0.38738     0.38738     0.00000     0.38738
     4       0.27060     0.27060     0.00000     0.27060
     5       0.02649     0.02649     0.00000     0.02649
     6       0.02649     0.02649     0.00000     0.02649
     7       0.23266     0.23266     0.00000     0.23266
     8       0.47105     0.47105     0.00000     0.47105
     9       0.37414     0.37414     0.00000     0.37414
    10       0.11356     0.11356     0.00000     0.11356
    11       0.01333     0.01333     0.00000     0.01333
    12       0.10166     0.10166     0.00000     0.10166
    13       0.41172     0.41172     0.00000     0.41172
    14       0.36018     0.36018     0.00000     0.36018
    15       0.01333     0.01333     0.00000     0.01333
    16       0.15937     0.15937     0.00000     0.15937
    17       0.43812     0.43812     0.00000     0.43812
    18       0.23266     0.23266     0.00000     0.23266
    19       0.03946     0.03946     0.00000     0.03946
    20       0.25199     0.25199     0.00000     0.25199
    21       0.47749     0.47749     0.00000     0.47749
    22       0.14819     0.14819     0.00000     0.14819
    23       0.01333     0.01333     0.00000     0.01333
    24       0.40591     0.40591     0.00000     0.40591
    25       0.34550     0.34550     0.00000     0.34550
    26       0.22273     0.22273     0.00000     0.22273
    27       0.10166     0.10166     0.00000     0.10166
--------------------------------------------------------
  Mean       0.21221     0.21221     0.00000     0.21221
  s.d.       0.16226     0.16226     0.00000     0.16226
--------------------------------------------------------
<A NAME="31_01_17at16_40_43_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_40_43_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      71.08859    71.08859     0.00000
            Theta_k_lower      51.15297    51.15297     0.00000
            Theta_k_upper      98.81925    98.81925     0.00000
                  Theta_H      75.24843    75.24843     0.00000
             s.d. Theta_H      17.04661    17.04661     0.00000
                  Theta_S       4.83480     4.83480     0.00000
             s.d. Theta_S       1.41038     1.41038     0.00000
                 Theta_pi       5.72966     5.72966     0.00000
            s.d. Theta_pi       3.05447     3.05447     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_40_43_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       5.92578     5.92578     0.00000
              Tau qt 2.5%       4.88672     4.88672     0.00000
                Tau qt 5%       4.88672     4.88672     0.00000
               Tau qt 95%       5.94141     5.94141     0.00000
             Tau qt 97.5%       5.94141     5.94141     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.24277     1.24277     0.00000
          Theta0 qt 97.5%       1.24277     1.24277     0.00000
                   Theta1     542.50000   542.50000     0.00000
           Theta1 qt 2.5%     103.98438   103.98438     0.00000
             Theta1 qt 5%     103.98438   103.98438     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00112     0.00112     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.01193     0.01193     0.00000
       Raggedness p-value       0.20000     0.20000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 85 (31/01/17 at 16:40:43))
Total computing time for this run : 0h 0m 0s 286 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

