
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_56_39"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 87 (31/01/17 at 14:56:39)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_56_39_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_56_39_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 40
No. of loci              : 23
No. of usable loci       : 23 loci with less than  5.00 % missing data
No. of polymorphic sites : 23


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.05226
     2       150         2   0.24242
     3       150         2   0.33790
     4       150         2   0.03946
     5       150         2   0.19186
     6       150         2   0.20233
     7       150         2   0.49441
     8       150         2   0.06488
     9       150         2   0.21262
    10       150         2   0.48582
    11       150         2   0.43320
    12       150         2   0.32215
    13       150         2   0.01333
    14       150         2   0.11356
    15       150         2   0.39991
    16       150         2   0.31400
    17       150         2   0.18121
    18       150         2   0.11356
    19       150         2   0.47436
    20       150         2   0.05226
    21       150         2   0.10166
    22       150         2   0.44286
    23       150         2   0.22273
-------------------------------------------------
  Mean   150.000     2.000   0.23951
  s.d.     0.000     0.000   0.15927
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0692
Gene diversity           :  0.9370 +/-  0.0096

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 40

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 23
Number of usable nucleotide sites            : 23


Distance method : Pairwise difference
Mean number of pairwise differences           :     5.508725 +/-     2.664079

Average gene diversity over loci              :     0.239510 +/-     0.128212

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    13.347021
  S.D. Theta(Hom) :     2.349159

         Theta(k) :    17.513851
95 % confidence interval limits for theta(k) : [  11.895992,  25.454453 ]

         Theta(S) :     4.118532
    S.D. Theta(S) :     1.243883

        Theta(Pi) :     5.508725
   S.D. Theta(Pi) :     2.948877

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     704      71.962
       1     497     343.922
       2     467     833.280
       3     808    1368.236
       4     850    1718.274
       5    1658    1767.112
       6    2059    1557.272
       7    1631    1215.886
       8    1119     863.629
       9     833     570.390
      10     320     356.812
      11     186     214.695
      12      22     125.813
      13      10      72.490
      14      10      41.345
      15       1      23.448
      16       0      13.259
      17       0       7.486
      18       0       4.224
      19       0       2.383
      20       0       1.344
      21       0       0.758
      22       0       0.428
      23       0       0.241
      24       0       0.136
      25       0       0.077
      26       0       0.043
      27       0       0.024
      28       0       0.014
      29       0       0.008
      30       0       0.004


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        5.509
Mismatch observed variance          :        7.182
Tau                                 :        6.521
Theta0                              :        0.000
Theta1                              :       32.031

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00561937
          P(Sim. Ssd >= Obs. Ssd): 0.10000000
    Harpending's Raggedness index: 0.01450049
        P(Sim. Rag. >= Obs. Rag.): 0.30000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        704.0        331.8        351.3
          1.0        497.0        478.2        412.5
          2.0        467.0        815.3        591.8
          3.0        808.0       1262.9        898.6
          4.0        850.0       1582.8       1250.5
          5.0       1658.0       1634.0       1510.9
          6.0       2059.0       1450.8       1576.3
          7.0       1631.0       1341.5       1433.6
          8.0       1119.0        960.6       1150.9
          9.0        833.0        643.9        825.4
         10.0        320.0        336.6        534.3
         11.0        186.0        180.8        315.1
         12.0         22.0         99.9        170.5
         13.0         10.0         37.8         85.3
         14.0         10.0         13.9         39.6
         15.0          1.0          3.7         17.2
         16.0          0.0          0.4          7.0
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           6.521           4.373           8.055           5.613
                   Theta0:           0.000           0.000           1.470           0.262               
                   Theta1:          32.031          23.085         159.844          58.473
Mean No. of pairwise diff.           5.509           4.166           6.676           5.304
           No. pol. sites:              23              69             105          81.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        704.0        229.0        471.0        351.3
          1.0        497.0        267.0        699.0        412.5
          2.0        467.0        498.0       1420.0        591.8
          3.0        808.0        857.0       2029.0        898.6
          4.0        850.0        956.0       2170.0       1250.5
          5.0       1658.0        724.0       2236.0       1510.9
          6.0       2059.0        874.0       1768.0       1576.3
          7.0       1631.0        726.0       1737.0       1433.6
          8.0       1119.0        375.0       1556.0       1150.9
          9.0        833.0        175.0       1241.0        825.4
         10.0        320.0         42.0        773.0        534.3
         11.0        186.0          6.0        516.0        315.1
         12.0         22.0          0.0        433.0        170.5
         13.0         10.0          0.0        246.0         85.3
         14.0         10.0          0.0         98.0         39.6
         15.0          1.0          0.0         29.0         17.2
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           6.521           4.373           8.055           5.613
                   Theta0:           0.000           0.000           1.470           0.262               
                   Theta1:          32.031          23.085         159.844          58.473
Mean No. of pairwise diff.           5.509           4.166           6.676           5.304
           No. pol. sites:              23              69             105          81.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        704.0        229.0        471.0        351.3
          1.0        497.0        267.0        699.0        412.5
          2.0        467.0        498.0       1420.0        591.8
          3.0        808.0        857.0       2029.0        898.6
          4.0        850.0        956.0       2170.0       1250.5
          5.0       1658.0        724.0       2236.0       1510.9
          6.0       2059.0        874.0       1768.0       1576.3
          7.0       1631.0        726.0       1737.0       1433.6
          8.0       1119.0        375.0       1556.0       1150.9
          9.0        833.0        175.0       1241.0        825.4
         10.0        320.0         42.0        773.0        534.3
         11.0        186.0          6.0        516.0        315.1
         12.0         22.0          0.0        433.0        170.5
         13.0         10.0          0.0        246.0         85.3
         14.0         10.0          0.0         98.0         39.6
         15.0          1.0          0.0         29.0         17.2
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           6.521           4.373           8.055           5.613
                   Theta0:           0.000           0.000           1.470           0.262               
                   Theta1:          32.031          23.085         159.844          58.473
Mean No. of pairwise diff.           5.509           4.166           6.676           5.304
           No. pol. sites:              23              69             105          81.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        704.0        229.0        471.0        351.3
          1.0        497.0        267.0        699.0        412.5
          2.0        467.0        498.0       1420.0        591.8
          3.0        808.0        857.0       2029.0        898.6
          4.0        850.0        956.0       2170.0       1250.5
          5.0       1658.0        724.0       2236.0       1510.9
          6.0       2059.0        874.0       1768.0       1576.3
          7.0       1631.0        726.0       1737.0       1433.6
          8.0       1119.0        375.0       1556.0       1150.9
          9.0        833.0        175.0       1241.0        825.4
         10.0        320.0         42.0        773.0        534.3
         11.0        186.0          6.0        516.0        315.1
         12.0         22.0          0.0        433.0        170.5
         13.0         10.0          0.0        246.0         85.3
         14.0         10.0          0.0         98.0         39.6
         15.0          1.0          0.0         29.0         17.2
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_56_39_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            23      23.000       0.000
  No. of usable loci            23      23.000       0.000
  No. of polym. loci            23      23.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_56_39_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.05226     0.05226     0.00000     0.05226
     2       0.24242     0.24242     0.00000     0.24242
     3       0.33790     0.33790     0.00000     0.33790
     4       0.03946     0.03946     0.00000     0.03946
     5       0.19186     0.19186     0.00000     0.19186
     6       0.20233     0.20233     0.00000     0.20233
     7       0.49441     0.49441     0.00000     0.49441
     8       0.06488     0.06488     0.00000     0.06488
     9       0.21262     0.21262     0.00000     0.21262
    10       0.48582     0.48582     0.00000     0.48582
    11       0.43320     0.43320     0.00000     0.43320
    12       0.32215     0.32215     0.00000     0.32215
    13       0.01333     0.01333     0.00000     0.01333
    14       0.11356     0.11356     0.00000     0.11356
    15       0.39991     0.39991     0.00000     0.39991
    16       0.31400     0.31400     0.00000     0.31400
    17       0.18121     0.18121     0.00000     0.18121
    18       0.11356     0.11356     0.00000     0.11356
    19       0.47436     0.47436     0.00000     0.47436
    20       0.05226     0.05226     0.00000     0.05226
    21       0.10166     0.10166     0.00000     0.10166
    22       0.44286     0.44286     0.00000     0.44286
    23       0.22273     0.22273     0.00000     0.22273
--------------------------------------------------------
  Mean       0.23951     0.23951     0.00000     0.23951
  s.d.       0.15927     0.15927     0.00000     0.15927
--------------------------------------------------------
<A NAME="31_01_17at14_56_39_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_56_39_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      17.51385    17.51385     0.00000
            Theta_k_lower      11.89599    11.89599     0.00000
            Theta_k_upper      25.45445    25.45445     0.00000
                  Theta_H      13.34702    13.34702     0.00000
             s.d. Theta_H       2.34916     2.34916     0.00000
                  Theta_S       4.11853     4.11853     0.00000
             s.d. Theta_S       1.24388     1.24388     0.00000
                 Theta_pi       5.50872     5.50872     0.00000
            s.d. Theta_pi       2.94888     2.94888     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_56_39_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       6.52148     6.52148     0.00000
              Tau qt 2.5%       4.37305     4.37305     0.00000
                Tau qt 5%       4.37305     4.37305     0.00000
               Tau qt 95%       6.41602     6.41602     0.00000
             Tau qt 97.5%       6.41602     6.41602     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.10215     1.10215     0.00000
          Theta0 qt 97.5%       1.10215     1.10215     0.00000
                   Theta1      32.03125    32.03125     0.00000
           Theta1 qt 2.5%      23.08472    23.08472     0.00000
             Theta1 qt 5%      23.08472    23.08472     0.00000
            Theta1 qt 95%      82.03125    82.03125     0.00000
          Theta1 qt 97.5%      82.03125    82.03125     0.00000
                      SSD       0.00562     0.00562     0.00000
      Model (SSD) p-value       0.10000     0.10000     0.00000
         Raggedness index       0.01450     0.01450     0.00000
       Raggedness p-value       0.30000     0.30000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 87 (31/01/17 at 14:56:39))
Total computing time for this run : 0h 0m 0s 190 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

