
<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_42_32"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 114 (31/01/17 at 14:42:32)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_42_32_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_42_32_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 45
No. of loci              : 50
No. of usable loci       : 50 loci with less than  5.00 % missing data
No. of polymorphic sites : 50


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.01333
     3       150         2   0.01333
     4       150         2   0.10166
     5       150         2   0.01333
     6       150         2   0.05226
     7       150         2   0.01333
     8       150         2   0.03946
     9       150         2   0.03946
    10       150         2   0.01333
    11       150         2   0.03946
    12       150         2   0.01333
    13       150         2   0.06488
    14       150         2   0.02649
    15       150         2   0.02649
    16       150         2   0.03946
    17       150         2   0.03946
    18       150         2   0.01333
    19       150         2   0.01333
    20       150         2   0.01333
    21       150         2   0.07732
    22       150         2   0.01333
    23       150         2   0.06488
    24       150         2   0.03946
    25       150         2   0.01333
    26       150         2   0.02649
    27       150         2   0.01333
    28       150         2   0.01333
    29       150         2   0.10166
    30       150         2   0.01333
    31       150         2   0.02649
    32       150         2   0.01333
    33       150         2   0.03946
    34       150         2   0.07732
    35       150         2   0.01333
    36       150         2   0.02649
    37       150         2   0.01333
    38       150         2   0.01333
    39       150         2   0.02649
    40       150         2   0.01333
    41       150         2   0.01333
    42       150         2   0.01333
    43       150         2   0.01333
    44       150         2   0.01333
    45       150         2   0.01333
    46       150         2   0.01333
    47       150         2   0.05226
    48       150         2   0.18121
    49       150         2   0.06488
    50       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.03267
  s.d.     0.000     0.000   0.03197
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.1932
Gene diversity           :  0.8122 +/-  0.0330

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 45

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 50
Number of usable nucleotide sites            : 50


Distance method : Pairwise difference
Mean number of pairwise differences           :     1.633468 +/-     0.969979

Average gene diversity over loci              :     0.032669 +/-     0.021473

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     3.450419
  S.D. Theta(Hom) :     0.811002

         Theta(k) :    21.427915
95 % confidence interval limits for theta(k) : [  14.772787,  30.751806 ]

         Theta(S) :     8.953331
    S.D. Theta(S) :     2.350317

        Theta(Pi) :     1.633468
   S.D. Theta(Pi) :     1.073673

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    2099       0.000
       1    3295       0.000
       2    3259       0.000
       3    1774       0.000
       4     627       0.000
       5     115       0.000
       6       6       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        1.633
Mismatch observed variance          :        1.396
Tau                                 :        1.732
Theta0                              :        0.000
Theta1                              :      457.525

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00127449
          P(Sim. Ssd >= Obs. Ssd): 0.20000000
    Harpending's Raggedness index: 0.04185283
        P(Sim. Rag. >= Obs. Rag.): 0.30000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       2099.0       1980.2       1989.0
          1.0       3295.0       3316.1       3423.5
          2.0       3259.0       2936.6       2960.7
          3.0       1774.0       1735.4       1708.6
          4.0        627.0        793.4        739.7
          5.0        115.0        302.5        256.3
          6.0          6.0         88.1         74.0
          7.0          0.0         18.4         18.3
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.732           1.197           2.168           1.721
                   Theta0:           0.000           0.000           0.431           0.106               
                   Theta1:         457.525           8.311       99999.000       60051.545
Mean No. of pairwise diff.           1.633           1.345           2.062           1.769
           No. pol. sites:              50              87             122         102.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2099.0       1387.0       3242.0       1989.0
          1.0       3295.0       2781.0       3731.0       3423.5
          2.0       3259.0       2410.0       3337.0       2960.7
          3.0       1774.0       1024.0       2203.0       1708.6
          4.0        627.0        517.0       1124.0        739.7
          5.0        115.0        165.0        433.0        256.3
          6.0          6.0         33.0        159.0         74.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.732           1.197           2.168           1.721
                   Theta0:           0.000           0.000           0.431           0.106               
                   Theta1:         457.525           8.311       99999.000       60051.545
Mean No. of pairwise diff.           1.633           1.345           2.062           1.769
           No. pol. sites:              50              87             122         102.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2099.0       1387.0       3242.0       1989.0
          1.0       3295.0       2781.0       3731.0       3423.5
          2.0       3259.0       2410.0       3337.0       2960.7
          3.0       1774.0       1024.0       2203.0       1708.6
          4.0        627.0        517.0       1124.0        739.7
          5.0        115.0        165.0        433.0        256.3
          6.0          6.0         33.0        159.0         74.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.732           1.197           2.168           1.721
                   Theta0:           0.000           0.000           0.431           0.106               
                   Theta1:         457.525           8.311       99999.000       60051.545
Mean No. of pairwise diff.           1.633           1.345           2.062           1.769
           No. pol. sites:              50              87             122         102.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2099.0       1387.0       3242.0       1989.0
          1.0       3295.0       2781.0       3731.0       3423.5
          2.0       3259.0       2410.0       3337.0       2960.7
          3.0       1774.0       1024.0       2203.0       1708.6
          4.0        627.0        517.0       1124.0        739.7
          5.0        115.0        165.0        433.0        256.3
          6.0          6.0         33.0        159.0         74.0
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_42_32_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            50      50.000       0.000
  No. of usable loci            50      50.000       0.000
  No. of polym. loci            50      50.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_42_32_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.01333     0.01333     0.00000     0.01333
     3       0.01333     0.01333     0.00000     0.01333
     4       0.10166     0.10166     0.00000     0.10166
     5       0.01333     0.01333     0.00000     0.01333
     6       0.05226     0.05226     0.00000     0.05226
     7       0.01333     0.01333     0.00000     0.01333
     8       0.03946     0.03946     0.00000     0.03946
     9       0.03946     0.03946     0.00000     0.03946
    10       0.01333     0.01333     0.00000     0.01333
    11       0.03946     0.03946     0.00000     0.03946
    12       0.01333     0.01333     0.00000     0.01333
    13       0.06488     0.06488     0.00000     0.06488
    14       0.02649     0.02649     0.00000     0.02649
    15       0.02649     0.02649     0.00000     0.02649
    16       0.03946     0.03946     0.00000     0.03946
    17       0.03946     0.03946     0.00000     0.03946
    18       0.01333     0.01333     0.00000     0.01333
    19       0.01333     0.01333     0.00000     0.01333
    20       0.01333     0.01333     0.00000     0.01333
    21       0.07732     0.07732     0.00000     0.07732
    22       0.01333     0.01333     0.00000     0.01333
    23       0.06488     0.06488     0.00000     0.06488
    24       0.03946     0.03946     0.00000     0.03946
    25       0.01333     0.01333     0.00000     0.01333
    26       0.02649     0.02649     0.00000     0.02649
    27       0.01333     0.01333     0.00000     0.01333
    28       0.01333     0.01333     0.00000     0.01333
    29       0.10166     0.10166     0.00000     0.10166
    30       0.01333     0.01333     0.00000     0.01333
    31       0.02649     0.02649     0.00000     0.02649
    32       0.01333     0.01333     0.00000     0.01333
    33       0.03946     0.03946     0.00000     0.03946
    34       0.07732     0.07732     0.00000     0.07732
    35       0.01333     0.01333     0.00000     0.01333
    36       0.02649     0.02649     0.00000     0.02649
    37       0.01333     0.01333     0.00000     0.01333
    38       0.01333     0.01333     0.00000     0.01333
    39       0.02649     0.02649     0.00000     0.02649
    40       0.01333     0.01333     0.00000     0.01333
    41       0.01333     0.01333     0.00000     0.01333
    42       0.01333     0.01333     0.00000     0.01333
    43       0.01333     0.01333     0.00000     0.01333
    44       0.01333     0.01333     0.00000     0.01333
    45       0.01333     0.01333     0.00000     0.01333
    46       0.01333     0.01333     0.00000     0.01333
    47       0.05226     0.05226     0.00000     0.05226
    48       0.18121     0.18121     0.00000     0.18121
    49       0.06488     0.06488     0.00000     0.06488
    50       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.03267     0.03267     0.00000     0.03267
  s.d.       0.03197     0.03197     0.00000     0.03197
--------------------------------------------------------
<A NAME="31_01_17at14_42_32_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_42_32_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      21.42791    21.42791     0.00000
            Theta_k_lower      14.77279    14.77279     0.00000
            Theta_k_upper      30.75181    30.75181     0.00000
                  Theta_H       3.45042     3.45042     0.00000
             s.d. Theta_H       0.81100     0.81100     0.00000
                  Theta_S       8.95333     8.95333     0.00000
             s.d. Theta_S       2.35032     2.35032     0.00000
                 Theta_pi       1.63347     1.63347     0.00000
            s.d. Theta_pi       1.07367     1.07367     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_42_32_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.73242     1.73242     0.00000
              Tau qt 2.5%       1.19727     1.19727     0.00000
                Tau qt 5%       1.19727     1.19727     0.00000
               Tau qt 95%       2.05859     2.05859     0.00000
             Tau qt 97.5%       2.05859     2.05859     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.21621     0.21621     0.00000
          Theta0 qt 97.5%       0.21621     0.21621     0.00000
                   Theta1     457.52500   457.52500     0.00000
           Theta1 qt 2.5%       8.31113     8.31113     0.00000
             Theta1 qt 5%       8.31113     8.31113     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00127     0.00127     0.00000
      Model (SSD) p-value       0.20000     0.20000     0.00000
         Raggedness index       0.04185     0.04185     0.00000
       Raggedness p-value       0.30000     0.30000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 114 (31/01/17 at 14:42:32))
Total computing time for this run : 0h 0m 0s 87 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

