
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_51_47"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 143 (31/01/17 at 14:51:47)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_51_47_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_51_47_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 48
No. of loci              : 23
No. of usable loci       : 23 loci with less than  5.00 % missing data
No. of polymorphic sites : 23


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.39374
     3       150         2   0.17038
     4       150         2   0.25199
     5       150         2   0.08957
     6       150         2   0.39991
     7       150         2   0.38085
     8       150         2   0.19186
     9       150         2   0.01333
    10       150         2   0.05226
    11       150         2   0.20233
    12       150         2   0.44743
    13       150         2   0.35293
    14       150         2   0.11356
    15       150         2   0.07732
    16       150         2   0.07732
    17       150         2   0.33790
    18       150         2   0.39374
    19       150         2   0.07732
    20       150         2   0.03946
    21       150         2   0.36018
    22       150         2   0.15937
    23       150         2   0.05226
-------------------------------------------------
  Mean   150.000     2.000   0.20210
  s.d.     0.000     0.000   0.14865
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0364
Gene diversity           :  0.9700 +/-  0.0038

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 48

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 23
Number of usable nucleotide sites            : 23


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.648322 +/-     2.292068

Average gene diversity over loci              :     0.202101 +/-     0.110309

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    30.592703
  S.D. Theta(Hom) :     4.158184

         Theta(k) :    24.019287
95 % confidence interval limits for theta(k) : [  16.682825,  34.256022 ]

         Theta(S) :     4.118532
    S.D. Theta(S) :     1.243883

        Theta(Pi) :     4.648322
   S.D. Theta(Pi) :     2.537098

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     335     165.980
       1     611     656.340
       2    1100    1323.297
       3    1910    1821.231
       4    1944    1934.571
       5    1606    1701.966
       6    1267    1300.769
       7     922     895.081
       8     594     570.489
       9     443     344.564
      10     247     200.798
      11      92     114.440
      12      59      64.389
      13      21      35.980
      14      20      20.037
      15       4      11.141
      16       0       6.191
      17       0       3.439
      18       0       1.910
      19       0       1.061
      20       0       0.589
      21       0       0.327
      22       0       0.182
      23       0       0.101
      24       0       0.056
      25       0       0.031
      26       0       0.017
      27       0       0.010
      28       0       0.005
      29       0       0.003
      30       0       0.002


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.648
Mismatch observed variance          :        6.209
Tau                                 :        3.590
Theta0                              :        1.195
Theta1                              :       87.969

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00070361
          P(Sim. Ssd >= Obs. Ssd): 0.60000000
    Harpending's Raggedness index: 0.01214260
        P(Sim. Rag. >= Obs. Rag.): 0.50000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        335.0        261.4        257.2
          1.0        611.0        664.3        666.6
          2.0       1100.0       1196.4       1259.1
          3.0       1910.0       1611.0       1738.3
          4.0       1944.0       1685.2       1883.3
          5.0       1606.0       1627.2       1695.0
          6.0       1267.0       1354.8       1322.4
          7.0        922.0       1003.3        924.5
          8.0        594.0        724.1        595.1
          9.0        443.0        481.8        360.5
         10.0        247.0        285.8        209.4
         11.0         92.0        169.4        118.3
         12.0         59.0         69.0         65.7
         13.0         21.0         27.2         36.1
         14.0         20.0         10.0         19.8
         15.0          4.0          3.1         10.8
         16.0          0.0          0.8          5.9
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.590           2.803           6.410           4.593
                   Theta0:           1.195           0.000           1.775           0.555               
                   Theta1:          87.969          27.090       99999.000       20093.536
Mean No. of pairwise diff.           4.648           3.432           6.228           4.846
           No. pol. sites:              23              89             123         110.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        335.0        142.0        443.0        257.2
          1.0        611.0        280.0       1197.0        666.6
          2.0       1100.0        542.0       2280.0       1259.1
          3.0       1910.0        904.0       2579.0       1738.3
          4.0       1944.0       1151.0       2113.0       1883.3
          5.0       1606.0       1179.0       2381.0       1695.0
          6.0       1267.0        910.0       1796.0       1322.4
          7.0        922.0        460.0       1535.0        924.5
          8.0        594.0        174.0       1411.0        595.1
          9.0        443.0         76.0       1195.0        360.5
         10.0        247.0         21.0        793.0        209.4
         11.0         92.0          6.0        574.0        118.3
         12.0         59.0          0.0        216.0         65.7
         13.0         21.0          0.0        105.0         36.1
         14.0         20.0          0.0         46.0         19.8
         15.0          4.0          0.0         16.0         10.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.590           2.803           6.410           4.593
                   Theta0:           1.195           0.000           1.775           0.555               
                   Theta1:          87.969          27.090       99999.000       20093.536
Mean No. of pairwise diff.           4.648           3.432           6.228           4.846
           No. pol. sites:              23              89             123         110.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        335.0        142.0        443.0        257.2
          1.0        611.0        280.0       1197.0        666.6
          2.0       1100.0        542.0       2280.0       1259.1
          3.0       1910.0        904.0       2579.0       1738.3
          4.0       1944.0       1151.0       2113.0       1883.3
          5.0       1606.0       1179.0       2381.0       1695.0
          6.0       1267.0        910.0       1796.0       1322.4
          7.0        922.0        460.0       1535.0        924.5
          8.0        594.0        174.0       1411.0        595.1
          9.0        443.0         76.0       1195.0        360.5
         10.0        247.0         21.0        793.0        209.4
         11.0         92.0          6.0        574.0        118.3
         12.0         59.0          0.0        216.0         65.7
         13.0         21.0          0.0        105.0         36.1
         14.0         20.0          0.0         46.0         19.8
         15.0          4.0          0.0         16.0         10.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.590           2.803           6.410           4.593
                   Theta0:           1.195           0.000           1.775           0.555               
                   Theta1:          87.969          27.090       99999.000       20093.536
Mean No. of pairwise diff.           4.648           3.432           6.228           4.846
           No. pol. sites:              23              89             123         110.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        335.0        142.0        443.0        257.2
          1.0        611.0        280.0       1197.0        666.6
          2.0       1100.0        542.0       2280.0       1259.1
          3.0       1910.0        904.0       2579.0       1738.3
          4.0       1944.0       1151.0       2113.0       1883.3
          5.0       1606.0       1179.0       2381.0       1695.0
          6.0       1267.0        910.0       1796.0       1322.4
          7.0        922.0        460.0       1535.0        924.5
          8.0        594.0        174.0       1411.0        595.1
          9.0        443.0         76.0       1195.0        360.5
         10.0        247.0         21.0        793.0        209.4
         11.0         92.0          6.0        574.0        118.3
         12.0         59.0          0.0        216.0         65.7
         13.0         21.0          0.0        105.0         36.1
         14.0         20.0          0.0         46.0         19.8
         15.0          4.0          0.0         16.0         10.8
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_51_47_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            23      23.000       0.000
  No. of usable loci            23      23.000       0.000
  No. of polym. loci            23      23.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_51_47_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.39374     0.39374     0.00000     0.39374
     3       0.17038     0.17038     0.00000     0.17038
     4       0.25199     0.25199     0.00000     0.25199
     5       0.08957     0.08957     0.00000     0.08957
     6       0.39991     0.39991     0.00000     0.39991
     7       0.38085     0.38085     0.00000     0.38085
     8       0.19186     0.19186     0.00000     0.19186
     9       0.01333     0.01333     0.00000     0.01333
    10       0.05226     0.05226     0.00000     0.05226
    11       0.20233     0.20233     0.00000     0.20233
    12       0.44743     0.44743     0.00000     0.44743
    13       0.35293     0.35293     0.00000     0.35293
    14       0.11356     0.11356     0.00000     0.11356
    15       0.07732     0.07732     0.00000     0.07732
    16       0.07732     0.07732     0.00000     0.07732
    17       0.33790     0.33790     0.00000     0.33790
    18       0.39374     0.39374     0.00000     0.39374
    19       0.07732     0.07732     0.00000     0.07732
    20       0.03946     0.03946     0.00000     0.03946
    21       0.36018     0.36018     0.00000     0.36018
    22       0.15937     0.15937     0.00000     0.15937
    23       0.05226     0.05226     0.00000     0.05226
--------------------------------------------------------
  Mean       0.20210     0.20210     0.00000     0.20210
  s.d.       0.14865     0.14865     0.00000     0.14865
--------------------------------------------------------
<A NAME="31_01_17at14_51_47_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_51_47_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      24.01929    24.01929     0.00000
            Theta_k_lower      16.68282    16.68282     0.00000
            Theta_k_upper      34.25602    34.25602     0.00000
                  Theta_H      30.59270    30.59270     0.00000
             s.d. Theta_H       4.15818     4.15818     0.00000
                  Theta_S       4.11853     4.11853     0.00000
             s.d. Theta_S       1.24388     1.24388     0.00000
                 Theta_pi       4.64832     4.64832     0.00000
            s.d. Theta_pi       2.53710     2.53710     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_51_47_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.58984     3.58984     0.00000
              Tau qt 2.5%       2.80273     2.80273     0.00000
                Tau qt 5%       2.80273     2.80273     0.00000
               Tau qt 95%       6.24023     6.24023     0.00000
             Tau qt 97.5%       6.24023     6.24023     0.00000
                   Theta0       1.19531     1.19531     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.11445     1.11445     0.00000
          Theta0 qt 97.5%       1.11445     1.11445     0.00000
                   Theta1      87.96875    87.96875     0.00000
           Theta1 qt 2.5%      27.08984    27.08984     0.00000
             Theta1 qt 5%      27.08984    27.08984     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00070     0.00070     0.00000
      Model (SSD) p-value       0.60000     0.60000     0.00000
         Raggedness index       0.01214     0.01214     0.00000
       Raggedness p-value       0.50000     0.50000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 143 (31/01/17 at 14:51:47))
Total computing time for this run : 0h 0m 0s 256 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

