
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_12"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 184 (31/01/17 at 14:44:12)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_12_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_12_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 46
No. of loci              : 65
No. of usable loci       : 65 loci with less than  5.00 % missing data
No. of polymorphic sites : 65


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.03946
     3       150         2   0.11356
     4       150         2   0.01333
     5       150         2   0.29718
     6       150         2   0.01333
     7       150         2   0.06488
     8       150         2   0.11356
     9       150         2   0.02649
    10       150         2   0.01333
    11       150         2   0.01333
    12       150         2   0.03946
    13       150         2   0.01333
    14       150         2   0.07732
    15       150         2   0.08957
    16       150         2   0.03946
    17       150         2   0.01333
    18       150         2   0.01333
    19       150         2   0.07732
    20       150         2   0.08957
    21       150         2   0.01333
    22       150         2   0.01333
    23       150         2   0.01333
    24       150         2   0.01333
    25       150         2   0.01333
    26       150         2   0.02649
    27       150         2   0.05226
    28       150         2   0.01333
    29       150         2   0.01333
    30       150         2   0.03946
    31       150         2   0.08957
    32       150         2   0.02649
    33       150         2   0.01333
    34       150         2   0.01333
    35       150         2   0.10166
    36       150         2   0.05226
    37       150         2   0.01333
    38       150         2   0.01333
    39       150         2   0.05226
    40       150         2   0.07732
    41       150         2   0.05226
    42       150         2   0.02649
    43       150         2   0.02649
    44       150         2   0.10166
    45       150         2   0.01333
    46       150         2   0.06488
    47       150         2   0.02649
    48       150         2   0.01333
    49       150         2   0.03946
    50       150         2   0.01333
    51       150         2   0.08957
    52       150         2   0.01333
    53       150         2   0.05226
    54       150         2   0.01333
    55       150         2   0.03946
    56       150         2   0.24242
    57       150         2   0.03946
    58       150         2   0.06488
    59       150         2   0.01333
    60       150         2   0.06488
    61       150         2   0.02649
    62       150         2   0.07732
    63       150         2   0.06488
    64       150         2   0.18121
    65       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.04943
  s.d.     0.000     0.000   0.05284
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0606
Gene diversity           :  0.9457 +/-  0.0096

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 46

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 65
Number of usable nucleotide sites            : 65


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.212796 +/-     1.668322

Average gene diversity over loci              :     0.049428 +/-     0.028410

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    15.823297
  S.D. Theta(Hom) :     3.202128

         Theta(k) :    22.270418
95 % confidence interval limits for theta(k) : [  15.393436,  31.891169 ]

         Theta(S) :    11.639330
    S.D. Theta(S) :     2.956600

        Theta(Pi) :     3.212796
   S.D. Theta(Pi) :     1.846671

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     607       0.000
       1    1194       0.000
       2    2068       0.000
       3    2454       0.000
       4    2376       0.000
       5    1451       0.000
       6     773       0.000
       7     202       0.000
       8      50       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.213
Mismatch observed variance          :        2.812
Tau                                 :        3.457
Theta0                              :        0.000
Theta1                              :      113.906

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00158290
          P(Sim. Ssd >= Obs. Ssd): 0.20000000
    Harpending's Raggedness index: 0.02346624
        P(Sim. Rag. >= Obs. Rag.): 0.50000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        607.0        357.9        436.0
          1.0       1194.0       1193.4       1264.6
          2.0       2068.0       2074.7       2106.8
          3.0       2454.0       2386.4       2396.8
          4.0       2376.0       2103.5       2059.3
          5.0       1451.0       1447.9       1419.3
          6.0        773.0        845.5        816.1
          7.0        202.0        440.4        402.5
          8.0         50.0        199.2        173.8
          9.0          0.0         86.7         66.7
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.457           2.695           4.008           3.220
                   Theta0:           0.000           0.000           1.123           0.347               
                   Theta1:         113.906          49.844       99999.000       50115.688
Mean No. of pairwise diff.           3.213           3.014           3.890           3.499
           No. pol. sites:              65             110             135         120.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        607.0        144.0        461.0        436.0
          1.0       1194.0        682.0       1636.0       1264.6
          2.0       2068.0       1602.0       2723.0       2106.8
          3.0       2454.0       2034.0       2665.0       2396.8
          4.0       2376.0       1701.0       2531.0       2059.3
          5.0       1451.0       1062.0       1758.0       1419.3
          6.0        773.0        572.0       1193.0        816.1
          7.0        202.0        216.0        675.0        402.5
          8.0         50.0         56.0        356.0        173.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.457           2.695           4.008           3.220
                   Theta0:           0.000           0.000           1.123           0.347               
                   Theta1:         113.906          49.844       99999.000       50115.688
Mean No. of pairwise diff.           3.213           3.014           3.890           3.499
           No. pol. sites:              65             110             135         120.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        607.0        144.0        461.0        436.0
          1.0       1194.0        682.0       1636.0       1264.6
          2.0       2068.0       1602.0       2723.0       2106.8
          3.0       2454.0       2034.0       2665.0       2396.8
          4.0       2376.0       1701.0       2531.0       2059.3
          5.0       1451.0       1062.0       1758.0       1419.3
          6.0        773.0        572.0       1193.0        816.1
          7.0        202.0        216.0        675.0        402.5
          8.0         50.0         56.0        356.0        173.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.457           2.695           4.008           3.220
                   Theta0:           0.000           0.000           1.123           0.347               
                   Theta1:         113.906          49.844       99999.000       50115.688
Mean No. of pairwise diff.           3.213           3.014           3.890           3.499
           No. pol. sites:              65             110             135         120.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        607.0        144.0        461.0        436.0
          1.0       1194.0        682.0       1636.0       1264.6
          2.0       2068.0       1602.0       2723.0       2106.8
          3.0       2454.0       2034.0       2665.0       2396.8
          4.0       2376.0       1701.0       2531.0       2059.3
          5.0       1451.0       1062.0       1758.0       1419.3
          6.0        773.0        572.0       1193.0        816.1
          7.0        202.0        216.0        675.0        402.5
          8.0         50.0         56.0        356.0        173.8
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_12_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            65      65.000       0.000
  No. of usable loci            65      65.000       0.000
  No. of polym. loci            65      65.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_12_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.03946     0.03946     0.00000     0.03946
     3       0.11356     0.11356     0.00000     0.11356
     4       0.01333     0.01333     0.00000     0.01333
     5       0.29718     0.29718     0.00000     0.29718
     6       0.01333     0.01333     0.00000     0.01333
     7       0.06488     0.06488     0.00000     0.06488
     8       0.11356     0.11356     0.00000     0.11356
     9       0.02649     0.02649     0.00000     0.02649
    10       0.01333     0.01333     0.00000     0.01333
    11       0.01333     0.01333     0.00000     0.01333
    12       0.03946     0.03946     0.00000     0.03946
    13       0.01333     0.01333     0.00000     0.01333
    14       0.07732     0.07732     0.00000     0.07732
    15       0.08957     0.08957     0.00000     0.08957
    16       0.03946     0.03946     0.00000     0.03946
    17       0.01333     0.01333     0.00000     0.01333
    18       0.01333     0.01333     0.00000     0.01333
    19       0.07732     0.07732     0.00000     0.07732
    20       0.08957     0.08957     0.00000     0.08957
    21       0.01333     0.01333     0.00000     0.01333
    22       0.01333     0.01333     0.00000     0.01333
    23       0.01333     0.01333     0.00000     0.01333
    24       0.01333     0.01333     0.00000     0.01333
    25       0.01333     0.01333     0.00000     0.01333
    26       0.02649     0.02649     0.00000     0.02649
    27       0.05226     0.05226     0.00000     0.05226
    28       0.01333     0.01333     0.00000     0.01333
    29       0.01333     0.01333     0.00000     0.01333
    30       0.03946     0.03946     0.00000     0.03946
    31       0.08957     0.08957     0.00000     0.08957
    32       0.02649     0.02649     0.00000     0.02649
    33       0.01333     0.01333     0.00000     0.01333
    34       0.01333     0.01333     0.00000     0.01333
    35       0.10166     0.10166     0.00000     0.10166
    36       0.05226     0.05226     0.00000     0.05226
    37       0.01333     0.01333     0.00000     0.01333
    38       0.01333     0.01333     0.00000     0.01333
    39       0.05226     0.05226     0.00000     0.05226
    40       0.07732     0.07732     0.00000     0.07732
    41       0.05226     0.05226     0.00000     0.05226
    42       0.02649     0.02649     0.00000     0.02649
    43       0.02649     0.02649     0.00000     0.02649
    44       0.10166     0.10166     0.00000     0.10166
    45       0.01333     0.01333     0.00000     0.01333
    46       0.06488     0.06488     0.00000     0.06488
    47       0.02649     0.02649     0.00000     0.02649
    48       0.01333     0.01333     0.00000     0.01333
    49       0.03946     0.03946     0.00000     0.03946
    50       0.01333     0.01333     0.00000     0.01333
    51       0.08957     0.08957     0.00000     0.08957
    52       0.01333     0.01333     0.00000     0.01333
    53       0.05226     0.05226     0.00000     0.05226
    54       0.01333     0.01333     0.00000     0.01333
    55       0.03946     0.03946     0.00000     0.03946
    56       0.24242     0.24242     0.00000     0.24242
    57       0.03946     0.03946     0.00000     0.03946
    58       0.06488     0.06488     0.00000     0.06488
    59       0.01333     0.01333     0.00000     0.01333
    60       0.06488     0.06488     0.00000     0.06488
    61       0.02649     0.02649     0.00000     0.02649
    62       0.07732     0.07732     0.00000     0.07732
    63       0.06488     0.06488     0.00000     0.06488
    64       0.18121     0.18121     0.00000     0.18121
    65       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.04943     0.04943     0.00000     0.04943
  s.d.       0.05284     0.05284     0.00000     0.05284
--------------------------------------------------------
<A NAME="31_01_17at14_44_12_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_12_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      22.27042    22.27042     0.00000
            Theta_k_lower      15.39344    15.39344     0.00000
            Theta_k_upper      31.89117    31.89117     0.00000
                  Theta_H      15.82330    15.82330     0.00000
             s.d. Theta_H       3.20213     3.20213     0.00000
                  Theta_S      11.63933    11.63933     0.00000
             s.d. Theta_S       2.95660     2.95660     0.00000
                 Theta_pi       3.21280     3.21280     0.00000
            s.d. Theta_pi       1.84667     1.84667     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_12_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.45703     3.45703     0.00000
              Tau qt 2.5%       2.69531     2.69531     0.00000
                Tau qt 5%       2.69531     2.69531     0.00000
               Tau qt 95%       3.75000     3.75000     0.00000
             Tau qt 97.5%       3.75000     3.75000     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.05996     1.05996     0.00000
          Theta0 qt 97.5%       1.05996     1.05996     0.00000
                   Theta1     113.90625   113.90625     0.00000
           Theta1 qt 2.5%      49.84375    49.84375     0.00000
             Theta1 qt 5%      49.84375    49.84375     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00158     0.00158     0.00000
      Model (SSD) p-value       0.20000     0.20000     0.00000
         Raggedness index       0.02347     0.02347     0.00000
       Raggedness p-value       0.50000     0.50000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 184 (31/01/17 at 14:44:12))
Total computing time for this run : 0h 0m 0s 138 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

