
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at15_00_39"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 351 (31/01/17 at 15:00:39)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at15_00_39_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at15_00_39_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 47
No. of loci              : 19
No. of usable loci       : 19 loci with less than  5.00 % missing data
No. of polymorphic sites : 19


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.02649
     2       150         2   0.18121
     3       150         2   0.25199
     4       150         2   0.02649
     5       150         2   0.34550
     6       150         2   0.17038
     7       150         2   0.11356
     8       150         2   0.46004
     9       150         2   0.06488
    10       150         2   0.21262
    11       150         2   0.50300
    12       150         2   0.42282
    13       150         2   0.37414
    14       150         2   0.05226
    15       150         2   0.07732
    16       150         2   0.14819
    17       150         2   0.02649
    18       150         2   0.25199
    19       150         2   0.07732
-------------------------------------------------
  Mean   150.000     2.000   0.19930
  s.d.     0.000     0.000   0.15600
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0712
Gene diversity           :  0.9350 +/-  0.0141

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 47

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 19
Number of usable nucleotide sites            : 19


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.786667 +/-     1.918313

Average gene diversity over loci              :     0.199298 +/-     0.111757

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    12.879429
  S.D. Theta(Hom) :     3.246130

         Theta(k) :    23.134022
95 % confidence interval limits for theta(k) : [  16.029990,  33.058962 ]

         Theta(S) :     3.402266
    S.D. Theta(S) :     1.075424

        Theta(Pi) :     3.786667
   S.D. Theta(Pi) :     2.123387

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     726     329.668
       1    1338    1105.200
       2    1392    1890.027
       3    1590    2207.695
       4    1568    1991.853
       5    1723    1489.831
       6    1627     969.164
       7     941     568.382
       8     244     309.185
       9      24     159.604
      10       2      79.595
      11       0      38.860
      12       0      18.744
      13       0       8.984
      14       0       4.292
      15       0       2.048
      16       0       0.977
      17       0       0.466
      18       0       0.222
      19       0       0.106
      20       0       0.050


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.787
Mismatch observed variance          :        4.616
Tau                                 :        4.812
Theta0                              :        0.000
Theta1                              :       13.125

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00391623
          P(Sim. Ssd >= Obs. Ssd): 0.60000000
    Harpending's Raggedness index: 0.01165659
        P(Sim. Rag. >= Obs. Rag.): 0.70000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        726.0        730.4        849.6
          1.0       1338.0        866.2       1012.5
          2.0       1392.0       1283.1       1336.6
          3.0       1590.0       1632.1       1651.2
          4.0       1568.0       1782.2       1755.0
          5.0       1723.0       1600.0       1581.7
          6.0       1627.0       1268.7       1220.7
          7.0        941.0       1009.8        818.9
          8.0        244.0        583.6        484.5
          9.0         24.0        252.3        256.0
         10.0          2.0        103.8        122.1
         11.0          0.0         32.1         53.1
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.812           2.828           6.262           4.553
                   Theta0:           0.000           0.000           2.626           0.373               
                   Theta1:          13.125          12.168         137.656          39.005
Mean No. of pairwise diff.           3.787           2.475           5.454           4.192
           No. pol. sites:              19              31              53          42.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        726.0        422.0       1259.0        849.6
          1.0       1338.0        448.0       2372.0       1012.5
          2.0       1392.0        548.0       2304.0       1336.6
          3.0       1590.0        630.0       2309.0       1651.2
          4.0       1568.0       1070.0       2427.0       1755.0
          5.0       1723.0        710.0       2308.0       1581.7
          6.0       1627.0        382.0       2332.0       1220.7
          7.0        941.0        102.0       2295.0        818.9
          8.0        244.0          7.0       1171.0        484.5
          9.0         24.0          0.0        601.0        256.0
         10.0          2.0          0.0        390.0        122.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.812           2.828           6.262           4.553
                   Theta0:           0.000           0.000           2.626           0.373               
                   Theta1:          13.125          12.168         137.656          39.005
Mean No. of pairwise diff.           3.787           2.475           5.454           4.192
           No. pol. sites:              19              31              53          42.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        726.0        422.0       1259.0        849.6
          1.0       1338.0        448.0       2372.0       1012.5
          2.0       1392.0        548.0       2304.0       1336.6
          3.0       1590.0        630.0       2309.0       1651.2
          4.0       1568.0       1070.0       2427.0       1755.0
          5.0       1723.0        710.0       2308.0       1581.7
          6.0       1627.0        382.0       2332.0       1220.7
          7.0        941.0        102.0       2295.0        818.9
          8.0        244.0          7.0       1171.0        484.5
          9.0         24.0          0.0        601.0        256.0
         10.0          2.0          0.0        390.0        122.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.812           2.828           6.262           4.553
                   Theta0:           0.000           0.000           2.626           0.373               
                   Theta1:          13.125          12.168         137.656          39.005
Mean No. of pairwise diff.           3.787           2.475           5.454           4.192
           No. pol. sites:              19              31              53          42.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        726.0        422.0       1259.0        849.6
          1.0       1338.0        448.0       2372.0       1012.5
          2.0       1392.0        548.0       2304.0       1336.6
          3.0       1590.0        630.0       2309.0       1651.2
          4.0       1568.0       1070.0       2427.0       1755.0
          5.0       1723.0        710.0       2308.0       1581.7
          6.0       1627.0        382.0       2332.0       1220.7
          7.0        941.0        102.0       2295.0        818.9
          8.0        244.0          7.0       1171.0        484.5
          9.0         24.0          0.0        601.0        256.0
         10.0          2.0          0.0        390.0        122.1
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at15_00_39_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            19      19.000       0.000
  No. of usable loci            19      19.000       0.000
  No. of polym. loci            19      19.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at15_00_39_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.02649     0.02649     0.00000     0.02649
     2       0.18121     0.18121     0.00000     0.18121
     3       0.25199     0.25199     0.00000     0.25199
     4       0.02649     0.02649     0.00000     0.02649
     5       0.34550     0.34550     0.00000     0.34550
     6       0.17038     0.17038     0.00000     0.17038
     7       0.11356     0.11356     0.00000     0.11356
     8       0.46004     0.46004     0.00000     0.46004
     9       0.06488     0.06488     0.00000     0.06488
    10       0.21262     0.21262     0.00000     0.21262
    11       0.50300     0.50300     0.00000     0.50300
    12       0.42282     0.42282     0.00000     0.42282
    13       0.37414     0.37414     0.00000     0.37414
    14       0.05226     0.05226     0.00000     0.05226
    15       0.07732     0.07732     0.00000     0.07732
    16       0.14819     0.14819     0.00000     0.14819
    17       0.02649     0.02649     0.00000     0.02649
    18       0.25199     0.25199     0.00000     0.25199
    19       0.07732     0.07732     0.00000     0.07732
--------------------------------------------------------
  Mean       0.19930     0.19930     0.00000     0.19930
  s.d.       0.15600     0.15600     0.00000     0.15600
--------------------------------------------------------
<A NAME="31_01_17at15_00_39_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at15_00_39_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      23.13402    23.13402     0.00000
            Theta_k_lower      16.02999    16.02999     0.00000
            Theta_k_upper      33.05896    33.05896     0.00000
                  Theta_H      12.87943    12.87943     0.00000
             s.d. Theta_H       3.24613     3.24613     0.00000
                  Theta_S       3.40227     3.40227     0.00000
             s.d. Theta_S       1.07542     1.07542     0.00000
                 Theta_pi       3.78667     3.78667     0.00000
            s.d. Theta_pi       2.12339     2.12339     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at15_00_39_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.81250     4.81250     0.00000
              Tau qt 2.5%       2.82812     2.82812     0.00000
                Tau qt 5%       2.82812     2.82812     0.00000
               Tau qt 95%       5.84961     5.84961     0.00000
             Tau qt 97.5%       5.84961     5.84961     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.94395     0.94395     0.00000
          Theta0 qt 97.5%       0.94395     0.94395     0.00000
                   Theta1      13.12500    13.12500     0.00000
           Theta1 qt 2.5%      12.16797    12.16797     0.00000
             Theta1 qt 5%      12.16797    12.16797     0.00000
            Theta1 qt 95%      47.34375    47.34375     0.00000
          Theta1 qt 97.5%      47.34375    47.34375     0.00000
                      SSD       0.00392     0.00392     0.00000
      Model (SSD) p-value       0.60000     0.60000     0.00000
         Raggedness index       0.01166     0.01166     0.00000
       Raggedness p-value       0.70000     0.70000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 351 (31/01/17 at 15:00:40))
Total computing time for this run : 0h 0m 0s 138 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

