
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_51_45"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 135 (31/01/17 at 14:51:45)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_51_45_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_51_45_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 39
No. of loci              : 19
No. of usable loci       : 19 loci with less than  5.00 % missing data
No. of polymorphic sites : 19


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.24242
     2       150         2   0.08957
     3       150         2   0.01333
     4       150         2   0.13682
     5       150         2   0.17038
     6       150         2   0.30568
     7       150         2   0.23266
     8       150         2   0.05226
     9       150         2   0.02649
    10       150         2   0.31400
    11       150         2   0.03946
    12       150         2   0.17038
    13       150         2   0.41736
    14       150         2   0.27060
    15       150         2   0.01333
    16       150         2   0.01333
    17       150         2   0.13682
    18       150         2   0.22273
    19       150         2   0.14819
-------------------------------------------------
  Mean   150.000     2.000   0.15873
  s.d.     0.000     0.000   0.11885
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0763
Gene diversity           :  0.9299 +/-  0.0115

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 39

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 19
Number of usable nucleotide sites            : 19


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.015839 +/-     1.582238

Average gene diversity over loci              :     0.158728 +/-     0.092178

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    11.793216
  S.D. Theta(Hom) :     2.268071

         Theta(k) :    16.787407
95 % confidence interval limits for theta(k) : [  11.363701,  24.469906 ]

         Theta(S) :     3.402266
    S.D. Theta(S) :     1.075424

        Theta(Pi) :     3.015839
   S.D. Theta(Pi) :     1.751385

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     783     593.379
       1    1566    1704.862
       2    2262    2478.238
       3    2482    2435.218
       4    1834    1824.417
       5    1231    1114.946
       6     640     581.077
       7     236     266.775
       8      99     110.675
       9      29      42.374
      10      13      15.244
      11       0       5.233
      12       0       1.736
      13       0       0.563
      14       0       0.180
      15       0       0.057
      16       0       0.018
      17       0       0.006
      18       0       0.002
      19       0       0.001
      20       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.016
Mismatch observed variance          :        3.224
Tau                                 :        3.191
Theta0                              :        0.026
Theta1                              :       30.488

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00017404
          P(Sim. Ssd >= Obs. Ssd): 0.90000000
    Harpending's Raggedness index: 0.01974693
        P(Sim. Rag. >= Obs. Rag.): 0.90000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        783.0        706.5        744.9
          1.0       1566.0       1642.0       1586.0
          2.0       2262.0       2512.6       2299.3
          3.0       2482.0       2544.9       2372.6
          4.0       1834.0       1886.7       1873.9
          5.0       1231.0       1103.8       1193.7
          6.0        640.0        497.0        636.2
          7.0        236.0        210.8        291.3
          8.0         99.0         51.8        116.8
          9.0         29.0         14.9         41.7
         10.0         13.0          3.5         13.4
         11.0          0.0          0.3          3.9
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.191           1.852           3.502           2.993
                   Theta0:           0.026           0.000           0.058           0.007               
                   Theta1:          30.488          17.559       99999.000       70030.419
Mean No. of pairwise diff.           3.016           1.790           3.423           2.901
           No. pol. sites:              19              47              65          58.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        783.0        392.0       1892.0        744.9
          1.0       1566.0       1015.0       3190.0       1586.0
          2.0       2262.0       1941.0       3072.0       2299.3
          3.0       2482.0       1914.0       3020.0       2372.6
          4.0       1834.0        743.0       2432.0       1873.9
          5.0       1231.0        265.0       1675.0       1193.7
          6.0        640.0         67.0        744.0        636.2
          7.0        236.0         26.0        542.0        291.3
          8.0         99.0          4.0        170.0        116.8
          9.0         29.0          0.0         71.0         41.7
         10.0         13.0          0.0         27.0         13.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.191           1.852           3.502           2.993
                   Theta0:           0.026           0.000           0.058           0.007               
                   Theta1:          30.488          17.559       99999.000       70030.419
Mean No. of pairwise diff.           3.016           1.790           3.423           2.901
           No. pol. sites:              19              47              65          58.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        783.0        392.0       1892.0        744.9
          1.0       1566.0       1015.0       3190.0       1586.0
          2.0       2262.0       1941.0       3072.0       2299.3
          3.0       2482.0       1914.0       3020.0       2372.6
          4.0       1834.0        743.0       2432.0       1873.9
          5.0       1231.0        265.0       1675.0       1193.7
          6.0        640.0         67.0        744.0        636.2
          7.0        236.0         26.0        542.0        291.3
          8.0         99.0          4.0        170.0        116.8
          9.0         29.0          0.0         71.0         41.7
         10.0         13.0          0.0         27.0         13.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.191           1.852           3.502           2.993
                   Theta0:           0.026           0.000           0.058           0.007               
                   Theta1:          30.488          17.559       99999.000       70030.419
Mean No. of pairwise diff.           3.016           1.790           3.423           2.901
           No. pol. sites:              19              47              65          58.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        783.0        392.0       1892.0        744.9
          1.0       1566.0       1015.0       3190.0       1586.0
          2.0       2262.0       1941.0       3072.0       2299.3
          3.0       2482.0       1914.0       3020.0       2372.6
          4.0       1834.0        743.0       2432.0       1873.9
          5.0       1231.0        265.0       1675.0       1193.7
          6.0        640.0         67.0        744.0        636.2
          7.0        236.0         26.0        542.0        291.3
          8.0         99.0          4.0        170.0        116.8
          9.0         29.0          0.0         71.0         41.7
         10.0         13.0          0.0         27.0         13.4
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_51_45_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            19      19.000       0.000
  No. of usable loci            19      19.000       0.000
  No. of polym. loci            19      19.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_51_45_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.24242     0.24242     0.00000     0.24242
     2       0.08957     0.08957     0.00000     0.08957
     3       0.01333     0.01333     0.00000     0.01333
     4       0.13682     0.13682     0.00000     0.13682
     5       0.17038     0.17038     0.00000     0.17038
     6       0.30568     0.30568     0.00000     0.30568
     7       0.23266     0.23266     0.00000     0.23266
     8       0.05226     0.05226     0.00000     0.05226
     9       0.02649     0.02649     0.00000     0.02649
    10       0.31400     0.31400     0.00000     0.31400
    11       0.03946     0.03946     0.00000     0.03946
    12       0.17038     0.17038     0.00000     0.17038
    13       0.41736     0.41736     0.00000     0.41736
    14       0.27060     0.27060     0.00000     0.27060
    15       0.01333     0.01333     0.00000     0.01333
    16       0.01333     0.01333     0.00000     0.01333
    17       0.13682     0.13682     0.00000     0.13682
    18       0.22273     0.22273     0.00000     0.22273
    19       0.14819     0.14819     0.00000     0.14819
--------------------------------------------------------
  Mean       0.15873     0.15873     0.00000     0.15873
  s.d.       0.11885     0.11885     0.00000     0.11885
--------------------------------------------------------
<A NAME="31_01_17at14_51_45_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_51_45_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      16.78741    16.78741     0.00000
            Theta_k_lower      11.36370    11.36370     0.00000
            Theta_k_upper      24.46991    24.46991     0.00000
                  Theta_H      11.79322    11.79322     0.00000
             s.d. Theta_H       2.26807     2.26807     0.00000
                  Theta_S       3.40227     3.40227     0.00000
             s.d. Theta_S       1.07542     1.07542     0.00000
                 Theta_pi       3.01584     3.01584     0.00000
            s.d. Theta_pi       1.75138     1.75138     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_51_45_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.19141     3.19141     0.00000
              Tau qt 2.5%       1.85156     1.85156     0.00000
                Tau qt 5%       1.85156     1.85156     0.00000
               Tau qt 95%       3.43555     3.43555     0.00000
             Tau qt 97.5%       3.43555     3.43555     0.00000
                   Theta0       0.02637     0.02637     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00703     0.00703     0.00000
          Theta0 qt 97.5%       0.00703     0.00703     0.00000
                   Theta1      30.48828    30.48828     0.00000
           Theta1 qt 2.5%      17.55859    17.55859     0.00000
             Theta1 qt 5%      17.55859    17.55859     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00017     0.00017     0.00000
      Model (SSD) p-value       0.90000     0.90000     0.00000
         Raggedness index       0.01975     0.01975     0.00000
       Raggedness p-value       0.90000     0.90000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 135 (31/01/17 at 14:51:45))
Total computing time for this run : 0h 0m 0s 102 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

