
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_43_53"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 30 (31/01/17 at 14:43:53)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_43_53_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_43_53_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 49
No. of loci              : 71
No. of usable loci       : 71 loci with less than  5.00 % missing data
No. of polymorphic sites : 71


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.01333
     3       150         2   0.01333
     4       150         2   0.01333
     5       150         2   0.01333
     6       150         2   0.01333
     7       150         2   0.05226
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.03946
    11       150         2   0.01333
    12       150         2   0.20233
    13       150         2   0.02649
    14       150         2   0.01333
    15       150         2   0.07732
    16       150         2   0.01333
    17       150         2   0.01333
    18       150         2   0.02649
    19       150         2   0.01333
    20       150         2   0.01333
    21       150         2   0.05226
    22       150         2   0.01333
    23       150         2   0.01333
    24       150         2   0.07732
    25       150         2   0.01333
    26       150         2   0.08957
    27       150         2   0.03946
    28       150         2   0.03946
    29       150         2   0.03946
    30       150         2   0.02649
    31       150         2   0.05226
    32       150         2   0.01333
    33       150         2   0.05226
    34       150         2   0.06488
    35       150         2   0.01333
    36       150         2   0.01333
    37       150         2   0.05226
    38       150         2   0.01333
    39       150         2   0.02649
    40       150         2   0.05226
    41       150         2   0.05226
    42       150         2   0.01333
    43       150         2   0.01333
    44       150         2   0.01333
    45       150         2   0.13682
    46       150         2   0.01333
    47       150         2   0.01333
    48       150         2   0.02649
    49       150         2   0.01333
    50       150         2   0.11356
    51       150         2   0.07732
    52       150         2   0.01333
    53       150         2   0.02649
    54       150         2   0.03946
    55       150         2   0.01333
    56       150         2   0.05226
    57       150         2   0.05226
    58       150         2   0.01333
    59       150         2   0.01333
    60       150         2   0.01333
    61       150         2   0.01333
    62       150         2   0.02649
    63       150         2   0.01333
    64       150         2   0.01333
    65       150         2   0.01333
    66       150         2   0.01333
    67       150         2   0.02649
    68       150         2   0.05226
    69       150         2   0.01333
    70       150         2   0.02649
    71       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.03264
  s.d.     0.000     0.000   0.03282
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.1212
Gene diversity           :  0.8847 +/-  0.0227

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 49

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 71
Number of usable nucleotide sites            : 71


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.317405 +/-     1.275211

Average gene diversity over loci              :     0.032640 +/-     0.019881

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     6.464820
  S.D. Theta(Hom) :     1.579371

         Theta(k) :    24.926798
95 % confidence interval limits for theta(k) : [  17.352333,  35.483223 ]

         Theta(S) :    12.713730
    S.D. Theta(S) :     3.198456

        Theta(Pi) :     2.317405
   S.D. Theta(Pi) :     1.411535

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1289    1238.766
       1    2513    2616.566
       2    2890    2844.497
       3    2226    2136.508
       4    1131    1257.609
       5     618     624.680
       6     300     275.521
       7     160     112.069
       8      40      43.259
       9       8      16.179
      10       0       5.947
      11       0       2.166
      12       0       0.786
      13       0       0.285
      14       0       0.103
      15       0       0.037
      16       0       0.013
      17       0       0.005
      18       0       0.002


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.317
Mismatch observed variance          :        2.639
Tau                                 :        1.883
Theta0                              :        0.455
Theta1                              :       70.547

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00029405
          P(Sim. Ssd >= Obs. Ssd): 0.90000000
    Harpending's Raggedness index: 0.02946500
        P(Sim. Rag. >= Obs. Rag.): 0.70000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1289.0       1398.6       1270.7
          1.0       2513.0       2682.7       2585.6
          2.0       2890.0       2845.5       2843.4
          3.0       2226.0       2072.3       2159.9
          4.0       1131.0       1171.5       1272.1
          5.0        618.0        556.0        622.2
          6.0        300.0        251.1        264.9
          7.0        160.0        114.7        101.8
          8.0         40.0         57.2         36.3
          9.0          8.0         18.0         12.3
         10.0          0.0          3.9          4.0
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.883           1.541           4.957           2.356
                   Theta0:           0.455           0.000           0.682           0.081               
                   Theta1:          70.547           7.212       99999.000       60007.323
Mean No. of pairwise diff.           2.317           1.634           3.932           2.243
           No. pol. sites:              71              64              81          74.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1289.0        666.0       2531.0       1270.7
          1.0       2513.0       1295.0       3633.0       2585.6
          2.0       2890.0       1333.0       3325.0       2843.4
          3.0       2226.0       1533.0       2671.0       2159.9
          4.0       1131.0        732.0       1763.0       1272.1
          5.0        618.0        173.0       1733.0        622.2
          6.0        300.0         15.0       1333.0        264.9
          7.0        160.0          0.0        862.0        101.8
          8.0         40.0          0.0        521.0         36.3
          9.0          8.0          0.0        175.0         12.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.883           1.541           4.957           2.356
                   Theta0:           0.455           0.000           0.682           0.081               
                   Theta1:          70.547           7.212       99999.000       60007.323
Mean No. of pairwise diff.           2.317           1.634           3.932           2.243
           No. pol. sites:              71              64              81          74.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1289.0        666.0       2531.0       1270.7
          1.0       2513.0       1295.0       3633.0       2585.6
          2.0       2890.0       1333.0       3325.0       2843.4
          3.0       2226.0       1533.0       2671.0       2159.9
          4.0       1131.0        732.0       1763.0       1272.1
          5.0        618.0        173.0       1733.0        622.2
          6.0        300.0         15.0       1333.0        264.9
          7.0        160.0          0.0        862.0        101.8
          8.0         40.0          0.0        521.0         36.3
          9.0          8.0          0.0        175.0         12.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.883           1.541           4.957           2.356
                   Theta0:           0.455           0.000           0.682           0.081               
                   Theta1:          70.547           7.212       99999.000       60007.323
Mean No. of pairwise diff.           2.317           1.634           3.932           2.243
           No. pol. sites:              71              64              81          74.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1289.0        666.0       2531.0       1270.7
          1.0       2513.0       1295.0       3633.0       2585.6
          2.0       2890.0       1333.0       3325.0       2843.4
          3.0       2226.0       1533.0       2671.0       2159.9
          4.0       1131.0        732.0       1763.0       1272.1
          5.0        618.0        173.0       1733.0        622.2
          6.0        300.0         15.0       1333.0        264.9
          7.0        160.0          0.0        862.0        101.8
          8.0         40.0          0.0        521.0         36.3
          9.0          8.0          0.0        175.0         12.3
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_43_53_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            71      71.000       0.000
  No. of usable loci            71      71.000       0.000
  No. of polym. loci            71      71.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_43_53_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.01333     0.01333     0.00000     0.01333
     3       0.01333     0.01333     0.00000     0.01333
     4       0.01333     0.01333     0.00000     0.01333
     5       0.01333     0.01333     0.00000     0.01333
     6       0.01333     0.01333     0.00000     0.01333
     7       0.05226     0.05226     0.00000     0.05226
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.03946     0.03946     0.00000     0.03946
    11       0.01333     0.01333     0.00000     0.01333
    12       0.20233     0.20233     0.00000     0.20233
    13       0.02649     0.02649     0.00000     0.02649
    14       0.01333     0.01333     0.00000     0.01333
    15       0.07732     0.07732     0.00000     0.07732
    16       0.01333     0.01333     0.00000     0.01333
    17       0.01333     0.01333     0.00000     0.01333
    18       0.02649     0.02649     0.00000     0.02649
    19       0.01333     0.01333     0.00000     0.01333
    20       0.01333     0.01333     0.00000     0.01333
    21       0.05226     0.05226     0.00000     0.05226
    22       0.01333     0.01333     0.00000     0.01333
    23       0.01333     0.01333     0.00000     0.01333
    24       0.07732     0.07732     0.00000     0.07732
    25       0.01333     0.01333     0.00000     0.01333
    26       0.08957     0.08957     0.00000     0.08957
    27       0.03946     0.03946     0.00000     0.03946
    28       0.03946     0.03946     0.00000     0.03946
    29       0.03946     0.03946     0.00000     0.03946
    30       0.02649     0.02649     0.00000     0.02649
    31       0.05226     0.05226     0.00000     0.05226
    32       0.01333     0.01333     0.00000     0.01333
    33       0.05226     0.05226     0.00000     0.05226
    34       0.06488     0.06488     0.00000     0.06488
    35       0.01333     0.01333     0.00000     0.01333
    36       0.01333     0.01333     0.00000     0.01333
    37       0.05226     0.05226     0.00000     0.05226
    38       0.01333     0.01333     0.00000     0.01333
    39       0.02649     0.02649     0.00000     0.02649
    40       0.05226     0.05226     0.00000     0.05226
    41       0.05226     0.05226     0.00000     0.05226
    42       0.01333     0.01333     0.00000     0.01333
    43       0.01333     0.01333     0.00000     0.01333
    44       0.01333     0.01333     0.00000     0.01333
    45       0.13682     0.13682     0.00000     0.13682
    46       0.01333     0.01333     0.00000     0.01333
    47       0.01333     0.01333     0.00000     0.01333
    48       0.02649     0.02649     0.00000     0.02649
    49       0.01333     0.01333     0.00000     0.01333
    50       0.11356     0.11356     0.00000     0.11356
    51       0.07732     0.07732     0.00000     0.07732
    52       0.01333     0.01333     0.00000     0.01333
    53       0.02649     0.02649     0.00000     0.02649
    54       0.03946     0.03946     0.00000     0.03946
    55       0.01333     0.01333     0.00000     0.01333
    56       0.05226     0.05226     0.00000     0.05226
    57       0.05226     0.05226     0.00000     0.05226
    58       0.01333     0.01333     0.00000     0.01333
    59       0.01333     0.01333     0.00000     0.01333
    60       0.01333     0.01333     0.00000     0.01333
    61       0.01333     0.01333     0.00000     0.01333
    62       0.02649     0.02649     0.00000     0.02649
    63       0.01333     0.01333     0.00000     0.01333
    64       0.01333     0.01333     0.00000     0.01333
    65       0.01333     0.01333     0.00000     0.01333
    66       0.01333     0.01333     0.00000     0.01333
    67       0.02649     0.02649     0.00000     0.02649
    68       0.05226     0.05226     0.00000     0.05226
    69       0.01333     0.01333     0.00000     0.01333
    70       0.02649     0.02649     0.00000     0.02649
    71       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.03264     0.03264     0.00000     0.03264
  s.d.       0.03282     0.03282     0.00000     0.03282
--------------------------------------------------------
<A NAME="31_01_17at14_43_53_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_43_53_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      24.92680    24.92680     0.00000
            Theta_k_lower      17.35233    17.35233     0.00000
            Theta_k_upper      35.48322    35.48322     0.00000
                  Theta_H       6.46482     6.46482     0.00000
             s.d. Theta_H       1.57937     1.57937     0.00000
                  Theta_S      12.71373    12.71373     0.00000
             s.d. Theta_S       3.19846     3.19846     0.00000
                 Theta_pi       2.31740     2.31740     0.00000
            s.d. Theta_pi       1.41153     1.41153     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_43_53_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.88281     1.88281     0.00000
              Tau qt 2.5%       1.54102     1.54102     0.00000
                Tau qt 5%       1.54102     1.54102     0.00000
               Tau qt 95%       2.68359     2.68359     0.00000
             Tau qt 97.5%       2.68359     2.68359     0.00000
                   Theta0       0.45527     0.45527     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.05625     0.05625     0.00000
          Theta0 qt 97.5%       0.05625     0.05625     0.00000
                   Theta1      70.54688    70.54688     0.00000
           Theta1 qt 2.5%       7.21191     7.21191     0.00000
             Theta1 qt 5%       7.21191     7.21191     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00029     0.00029     0.00000
      Model (SSD) p-value       0.90000     0.90000     0.00000
         Raggedness index       0.02946     0.02946     0.00000
       Raggedness p-value       0.70000     0.70000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 30 (31/01/17 at 14:43:53))
Total computing time for this run : 0h 0m 0s 115 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

