
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at15_05_45"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 59 (31/01/17 at 15:05:45)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at15_05_45_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at15_05_45_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 52
No. of loci              : 23
No. of usable loci       : 23 loci with less than  5.00 % missing data
No. of polymorphic sites : 23


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.37414
     3       150         2   0.44743
     4       150         2   0.11356
     5       150         2   0.03946
     6       150         2   0.50300
     7       150         2   0.27964
     8       150         2   0.03946
     9       150         2   0.23266
    10       150         2   0.15937
    11       150         2   0.03946
    12       150         2   0.12528
    13       150         2   0.49763
    14       150         2   0.49253
    15       150         2   0.44286
    16       150         2   0.10166
    17       150         2   0.14819
    18       150         2   0.47105
    19       150         2   0.50192
    20       150         2   0.10166
    21       150         2   0.33011
    22       150         2   0.40591
    23       150         2   0.06488
-------------------------------------------------
  Mean   150.000     2.000   0.25762
  s.d.     0.000     0.000   0.18324
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0365
Gene diversity           :  0.9699 +/-  0.0057

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 52

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 23
Number of usable nucleotide sites            : 23


Distance method : Pairwise difference
Mean number of pairwise differences           :     5.925190 +/-     2.843854

Average gene diversity over loci              :     0.257617 +/-     0.136864

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    30.494101
  S.D. Theta(Hom) :     6.221602

         Theta(k) :    27.789032
95 % confidence interval limits for theta(k) : [  19.465082,  39.354984 ]

         Theta(S) :     4.118532
    S.D. Theta(S) :     1.243883

        Theta(Pi) :     5.925190
   S.D. Theta(Pi) :     3.147871

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     336      56.528
       1     639     279.749
       2     724     702.878
       3     799    1198.995
       4    1018    1567.846
       5    1320    1683.668
       6    1298    1554.660
       7    1254    1277.103
       8    1326     958.888
       9    1268     672.904
      10     797     449.624
      11     276     290.425
      12      98     183.499
      13      19     114.389
      14       2      70.764
      15       1      43.600
      16       0      26.809
      17       0      16.470
      18       0      10.114
      19       0       6.210
      20       0       3.813
      21       0       2.341
      22       0       1.437
      23       0       0.882
      24       0       0.542
      25       0       0.333
      26       0       0.204
      27       0       0.125
      28       0       0.077
      29       0       0.047
      30       0       0.029


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        5.925
Mismatch observed variance          :        8.454
Tau                                 :        7.240
Theta0                              :        0.005
Theta1                              :       22.330

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00239764
          P(Sim. Ssd >= Obs. Ssd): 0.60000000
    Harpending's Raggedness index: 0.00629633
        P(Sim. Rag. >= Obs. Rag.): 0.80000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        336.0        482.1        484.5
          1.0        639.0        472.9        498.2
          2.0        724.0        654.2        581.7
          3.0        799.0        842.0        761.6
          4.0       1018.0       1109.0       1012.7
          5.0       1320.0       1427.3       1253.8
          6.0       1298.0       1410.0       1390.6
          7.0       1254.0       1367.6       1369.8
          8.0       1326.0       1079.6       1202.9
          9.0       1268.0        785.3        948.8
         10.0        797.0        569.5        677.7
         11.0        276.0        398.9        441.8
         12.0         98.0        229.5        264.7
         13.0         19.0        152.5        146.6
         14.0          2.0         91.6         75.5
         15.0          1.0         37.0         36.3
         16.0          0.0         15.4         16.4
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           7.240           3.885          10.914           6.753
                   Theta0:           0.005           0.000           1.682           0.339               
                   Theta1:          22.330          15.675          89.205          35.564
Mean No. of pairwise diff.           5.925           4.652           8.833           6.032
           No. pol. sites:              23              61              74          67.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        336.0        288.0        823.0        484.5
          1.0        639.0        309.0        777.0        498.2
          2.0        724.0        382.0       1278.0        581.7
          3.0        799.0        319.0       1603.0        761.6
          4.0       1018.0        485.0       1597.0       1012.7
          5.0       1320.0        507.0       1954.0       1253.8
          6.0       1298.0        705.0       1951.0       1390.6
          7.0       1254.0       1019.0       1826.0       1369.8
          8.0       1326.0        740.0       1490.0       1202.9
          9.0       1268.0        467.0       1438.0        948.8
         10.0        797.0        206.0       1109.0        677.7
         11.0        276.0         80.0       1364.0        441.8
         12.0         98.0          8.0        909.0        264.7
         13.0         19.0          0.0        751.0        146.6
         14.0          2.0          0.0        532.0         75.5
         15.0          1.0          0.0        255.0         36.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           7.240           3.885          10.914           6.753
                   Theta0:           0.005           0.000           1.682           0.339               
                   Theta1:          22.330          15.675          89.205          35.564
Mean No. of pairwise diff.           5.925           4.652           8.833           6.032
           No. pol. sites:              23              61              74          67.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        336.0        288.0        823.0        484.5
          1.0        639.0        309.0        777.0        498.2
          2.0        724.0        382.0       1278.0        581.7
          3.0        799.0        319.0       1603.0        761.6
          4.0       1018.0        485.0       1597.0       1012.7
          5.0       1320.0        507.0       1954.0       1253.8
          6.0       1298.0        705.0       1951.0       1390.6
          7.0       1254.0       1019.0       1826.0       1369.8
          8.0       1326.0        740.0       1490.0       1202.9
          9.0       1268.0        467.0       1438.0        948.8
         10.0        797.0        206.0       1109.0        677.7
         11.0        276.0         80.0       1364.0        441.8
         12.0         98.0          8.0        909.0        264.7
         13.0         19.0          0.0        751.0        146.6
         14.0          2.0          0.0        532.0         75.5
         15.0          1.0          0.0        255.0         36.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           7.240           3.885          10.914           6.753
                   Theta0:           0.005           0.000           1.682           0.339               
                   Theta1:          22.330          15.675          89.205          35.564
Mean No. of pairwise diff.           5.925           4.652           8.833           6.032
           No. pol. sites:              23              61              74          67.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        336.0        288.0        823.0        484.5
          1.0        639.0        309.0        777.0        498.2
          2.0        724.0        382.0       1278.0        581.7
          3.0        799.0        319.0       1603.0        761.6
          4.0       1018.0        485.0       1597.0       1012.7
          5.0       1320.0        507.0       1954.0       1253.8
          6.0       1298.0        705.0       1951.0       1390.6
          7.0       1254.0       1019.0       1826.0       1369.8
          8.0       1326.0        740.0       1490.0       1202.9
          9.0       1268.0        467.0       1438.0        948.8
         10.0        797.0        206.0       1109.0        677.7
         11.0        276.0         80.0       1364.0        441.8
         12.0         98.0          8.0        909.0        264.7
         13.0         19.0          0.0        751.0        146.6
         14.0          2.0          0.0        532.0         75.5
         15.0          1.0          0.0        255.0         36.3
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at15_05_45_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            23      23.000       0.000
  No. of usable loci            23      23.000       0.000
  No. of polym. loci            23      23.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at15_05_45_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.37414     0.37414     0.00000     0.37414
     3       0.44743     0.44743     0.00000     0.44743
     4       0.11356     0.11356     0.00000     0.11356
     5       0.03946     0.03946     0.00000     0.03946
     6       0.50300     0.50300     0.00000     0.50300
     7       0.27964     0.27964     0.00000     0.27964
     8       0.03946     0.03946     0.00000     0.03946
     9       0.23266     0.23266     0.00000     0.23266
    10       0.15937     0.15937     0.00000     0.15937
    11       0.03946     0.03946     0.00000     0.03946
    12       0.12528     0.12528     0.00000     0.12528
    13       0.49763     0.49763     0.00000     0.49763
    14       0.49253     0.49253     0.00000     0.49253
    15       0.44286     0.44286     0.00000     0.44286
    16       0.10166     0.10166     0.00000     0.10166
    17       0.14819     0.14819     0.00000     0.14819
    18       0.47105     0.47105     0.00000     0.47105
    19       0.50192     0.50192     0.00000     0.50192
    20       0.10166     0.10166     0.00000     0.10166
    21       0.33011     0.33011     0.00000     0.33011
    22       0.40591     0.40591     0.00000     0.40591
    23       0.06488     0.06488     0.00000     0.06488
--------------------------------------------------------
  Mean       0.25762     0.25762     0.00000     0.25762
  s.d.       0.18324     0.18324     0.00000     0.18324
--------------------------------------------------------
<A NAME="31_01_17at15_05_45_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at15_05_45_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      27.78903    27.78903     0.00000
            Theta_k_lower      19.46508    19.46508     0.00000
            Theta_k_upper      39.35498    39.35498     0.00000
                  Theta_H      30.49410    30.49410     0.00000
             s.d. Theta_H       6.22160     6.22160     0.00000
                  Theta_S       4.11853     4.11853     0.00000
             s.d. Theta_S       1.24388     1.24388     0.00000
                 Theta_pi       5.92519     5.92519     0.00000
            s.d. Theta_pi       3.14787     3.14787     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at15_05_45_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       7.24023     7.24023     0.00000
              Tau qt 2.5%       3.88477     3.88477     0.00000
                Tau qt 5%       3.88477     3.88477     0.00000
               Tau qt 95%       8.05078     8.05078     0.00000
             Tau qt 97.5%       8.05078     8.05078     0.00000
                   Theta0       0.00527     0.00527     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.65586     1.65586     0.00000
          Theta0 qt 97.5%       1.65586     1.65586     0.00000
                   Theta1      22.33032    22.33032     0.00000
           Theta1 qt 2.5%      15.67505    15.67505     0.00000
             Theta1 qt 5%      15.67505    15.67505     0.00000
            Theta1 qt 95%      47.64282    47.64282     0.00000
          Theta1 qt 97.5%      47.64282    47.64282     0.00000
                      SSD       0.00240     0.00240     0.00000
      Model (SSD) p-value       0.60000     0.60000     0.00000
         Raggedness index       0.00630     0.00630     0.00000
       Raggedness p-value       0.80000     0.80000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 59 (31/01/17 at 15:05:45))
Total computing time for this run : 0h 0m 0s 208 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

