
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_48_00"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 332 (31/01/17 at 14:48:00)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_48_00_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_48_00_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 53
No. of loci              : 66
No. of usable loci       : 66 loci with less than  5.00 % missing data
No. of polymorphic sites : 66


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.03946
     2       150         2   0.02649
     3       150         2   0.01333
     4       150         2   0.18121
     5       150         2   0.01333
     6       150         2   0.01333
     7       150         2   0.03946
     8       150         2   0.02649
     9       150         2   0.05226
    10       150         2   0.01333
    11       150         2   0.01333
    12       150         2   0.12528
    13       150         2   0.02649
    14       150         2   0.01333
    15       150         2   0.02649
    16       150         2   0.03946
    17       150         2   0.01333
    18       150         2   0.01333
    19       150         2   0.02649
    20       150         2   0.01333
    21       150         2   0.01333
    22       150         2   0.05226
    23       150         2   0.06488
    24       150         2   0.01333
    25       150         2   0.06488
    26       150         2   0.12528
    27       150         2   0.27060
    28       150         2   0.01333
    29       150         2   0.01333
    30       150         2   0.36018
    31       150         2   0.01333
    32       150         2   0.01333
    33       150         2   0.05226
    34       150         2   0.08957
    35       150         2   0.26139
    36       150         2   0.03946
    37       150         2   0.01333
    38       150         2   0.15937
    39       150         2   0.06488
    40       150         2   0.01333
    41       150         2   0.03946
    42       150         2   0.01333
    43       150         2   0.03946
    44       150         2   0.01333
    45       150         2   0.01333
    46       150         2   0.01333
    47       150         2   0.01333
    48       150         2   0.01333
    49       150         2   0.01333
    50       150         2   0.02649
    51       150         2   0.01333
    52       150         2   0.01333
    53       150         2   0.01333
    54       150         2   0.02649
    55       150         2   0.26139
    56       150         2   0.06488
    57       150         2   0.02649
    58       150         2   0.02649
    59       150         2   0.02649
    60       150         2   0.02649
    61       150         2   0.03946
    62       150         2   0.24242
    63       150         2   0.01333
    64       150         2   0.02649
    65       150         2   0.02649
    66       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.05303
  s.d.     0.000     0.000   0.07448
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0469
Gene diversity           :  0.9595 +/-  0.0063

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 53

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 66
Number of usable nucleotide sites            : 66


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.500224 +/-     1.793668

Average gene diversity over loci              :     0.053034 +/-     0.030082

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    21.982423
  S.D. Theta(Hom) :     3.794354

         Theta(k) :    28.791891
95 % confidence interval limits for theta(k) : [  20.205559,  40.712272 ]

         Theta(S) :    11.818397
    S.D. Theta(S) :     2.996930

        Theta(Pi) :     3.500224
   S.D. Theta(Pi) :     1.985417

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     453       0.000
       1    1204       0.000
       2    1836       0.000
       3    2290       0.000
       4    2134       0.000
       5    1646       0.000
       6     957       0.000
       7     442       0.000
       8     163       0.000
       9      38       0.000
      10      11       0.000
      11       1       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.500
Mismatch observed variance          :        3.435
Tau                                 :        3.684
Theta0                              :        0.028
Theta1                              :       57.891

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00031046
          P(Sim. Ssd >= Obs. Ssd): 0.80000000
    Harpending's Raggedness index: 0.01814743
        P(Sim. Rag. >= Obs. Rag.): 0.80000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        453.0        483.0        441.6
          1.0       1204.0       1151.6       1116.9
          2.0       1836.0       1930.1       1897.6
          3.0       2290.0       2391.7       2276.6
          4.0       2134.0       2164.8       2083.2
          5.0       1646.0       1530.0       1534.9
          6.0        957.0        893.8        945.3
          7.0        442.0        404.1        499.8
          8.0        163.0        159.0        231.4
          9.0         38.0         47.6         95.3
         10.0         11.0         14.8         35.4
         11.0          1.0          4.3         11.9
         12.0          0.0          0.2          3.7
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.684           2.092           4.289           3.457
                   Theta0:           0.028           0.000           1.540           0.162               
                   Theta1:          57.891          22.559       99999.000       60011.330
Mean No. of pairwise diff.           3.500           2.728           4.236           3.453
           No. pol. sites:              66              64              99          87.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        453.0        197.0        961.0        441.6
          1.0       1204.0        463.0       1809.0       1116.9
          2.0       1836.0       1228.0       2490.0       1897.6
          3.0       2290.0       1941.0       2591.0       2276.6
          4.0       2134.0       1498.0       2685.0       2083.2
          5.0       1646.0        967.0       1928.0       1534.9
          6.0        957.0        532.0       1415.0        945.3
          7.0        442.0         98.0        793.0        499.8
          8.0        163.0         28.0        412.0        231.4
          9.0         38.0          0.0        152.0         95.3
         10.0         11.0          0.0         57.0         35.4
         11.0          1.0          0.0         18.0         11.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.684           2.092           4.289           3.457
                   Theta0:           0.028           0.000           1.540           0.162               
                   Theta1:          57.891          22.559       99999.000       60011.330
Mean No. of pairwise diff.           3.500           2.728           4.236           3.453
           No. pol. sites:              66              64              99          87.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        453.0        197.0        961.0        441.6
          1.0       1204.0        463.0       1809.0       1116.9
          2.0       1836.0       1228.0       2490.0       1897.6
          3.0       2290.0       1941.0       2591.0       2276.6
          4.0       2134.0       1498.0       2685.0       2083.2
          5.0       1646.0        967.0       1928.0       1534.9
          6.0        957.0        532.0       1415.0        945.3
          7.0        442.0         98.0        793.0        499.8
          8.0        163.0         28.0        412.0        231.4
          9.0         38.0          0.0        152.0         95.3
         10.0         11.0          0.0         57.0         35.4
         11.0          1.0          0.0         18.0         11.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.684           2.092           4.289           3.457
                   Theta0:           0.028           0.000           1.540           0.162               
                   Theta1:          57.891          22.559       99999.000       60011.330
Mean No. of pairwise diff.           3.500           2.728           4.236           3.453
           No. pol. sites:              66              64              99          87.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        453.0        197.0        961.0        441.6
          1.0       1204.0        463.0       1809.0       1116.9
          2.0       1836.0       1228.0       2490.0       1897.6
          3.0       2290.0       1941.0       2591.0       2276.6
          4.0       2134.0       1498.0       2685.0       2083.2
          5.0       1646.0        967.0       1928.0       1534.9
          6.0        957.0        532.0       1415.0        945.3
          7.0        442.0         98.0        793.0        499.8
          8.0        163.0         28.0        412.0        231.4
          9.0         38.0          0.0        152.0         95.3
         10.0         11.0          0.0         57.0         35.4
         11.0          1.0          0.0         18.0         11.9
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_48_00_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            66      66.000       0.000
  No. of usable loci            66      66.000       0.000
  No. of polym. loci            66      66.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_48_00_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.03946     0.03946     0.00000     0.03946
     2       0.02649     0.02649     0.00000     0.02649
     3       0.01333     0.01333     0.00000     0.01333
     4       0.18121     0.18121     0.00000     0.18121
     5       0.01333     0.01333     0.00000     0.01333
     6       0.01333     0.01333     0.00000     0.01333
     7       0.03946     0.03946     0.00000     0.03946
     8       0.02649     0.02649     0.00000     0.02649
     9       0.05226     0.05226     0.00000     0.05226
    10       0.01333     0.01333     0.00000     0.01333
    11       0.01333     0.01333     0.00000     0.01333
    12       0.12528     0.12528     0.00000     0.12528
    13       0.02649     0.02649     0.00000     0.02649
    14       0.01333     0.01333     0.00000     0.01333
    15       0.02649     0.02649     0.00000     0.02649
    16       0.03946     0.03946     0.00000     0.03946
    17       0.01333     0.01333     0.00000     0.01333
    18       0.01333     0.01333     0.00000     0.01333
    19       0.02649     0.02649     0.00000     0.02649
    20       0.01333     0.01333     0.00000     0.01333
    21       0.01333     0.01333     0.00000     0.01333
    22       0.05226     0.05226     0.00000     0.05226
    23       0.06488     0.06488     0.00000     0.06488
    24       0.01333     0.01333     0.00000     0.01333
    25       0.06488     0.06488     0.00000     0.06488
    26       0.12528     0.12528     0.00000     0.12528
    27       0.27060     0.27060     0.00000     0.27060
    28       0.01333     0.01333     0.00000     0.01333
    29       0.01333     0.01333     0.00000     0.01333
    30       0.36018     0.36018     0.00000     0.36018
    31       0.01333     0.01333     0.00000     0.01333
    32       0.01333     0.01333     0.00000     0.01333
    33       0.05226     0.05226     0.00000     0.05226
    34       0.08957     0.08957     0.00000     0.08957
    35       0.26139     0.26139     0.00000     0.26139
    36       0.03946     0.03946     0.00000     0.03946
    37       0.01333     0.01333     0.00000     0.01333
    38       0.15937     0.15937     0.00000     0.15937
    39       0.06488     0.06488     0.00000     0.06488
    40       0.01333     0.01333     0.00000     0.01333
    41       0.03946     0.03946     0.00000     0.03946
    42       0.01333     0.01333     0.00000     0.01333
    43       0.03946     0.03946     0.00000     0.03946
    44       0.01333     0.01333     0.00000     0.01333
    45       0.01333     0.01333     0.00000     0.01333
    46       0.01333     0.01333     0.00000     0.01333
    47       0.01333     0.01333     0.00000     0.01333
    48       0.01333     0.01333     0.00000     0.01333
    49       0.01333     0.01333     0.00000     0.01333
    50       0.02649     0.02649     0.00000     0.02649
    51       0.01333     0.01333     0.00000     0.01333
    52       0.01333     0.01333     0.00000     0.01333
    53       0.01333     0.01333     0.00000     0.01333
    54       0.02649     0.02649     0.00000     0.02649
    55       0.26139     0.26139     0.00000     0.26139
    56       0.06488     0.06488     0.00000     0.06488
    57       0.02649     0.02649     0.00000     0.02649
    58       0.02649     0.02649     0.00000     0.02649
    59       0.02649     0.02649     0.00000     0.02649
    60       0.02649     0.02649     0.00000     0.02649
    61       0.03946     0.03946     0.00000     0.03946
    62       0.24242     0.24242     0.00000     0.24242
    63       0.01333     0.01333     0.00000     0.01333
    64       0.02649     0.02649     0.00000     0.02649
    65       0.02649     0.02649     0.00000     0.02649
    66       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.05303     0.05303     0.00000     0.05303
  s.d.       0.07448     0.07448     0.00000     0.07448
--------------------------------------------------------
<A NAME="31_01_17at14_48_00_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_48_00_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      28.79189    28.79189     0.00000
            Theta_k_lower      20.20556    20.20556     0.00000
            Theta_k_upper      40.71227    40.71227     0.00000
                  Theta_H      21.98242    21.98242     0.00000
             s.d. Theta_H       3.79435     3.79435     0.00000
                  Theta_S      11.81840    11.81840     0.00000
             s.d. Theta_S       2.99693     2.99693     0.00000
                 Theta_pi       3.50022     3.50022     0.00000
            s.d. Theta_pi       1.98542     1.98542     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_48_00_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.68359     3.68359     0.00000
              Tau qt 2.5%       2.09180     2.09180     0.00000
                Tau qt 5%       2.09180     2.09180     0.00000
               Tau qt 95%       4.12109     4.12109     0.00000
             Tau qt 97.5%       4.12109     4.12109     0.00000
                   Theta0       0.02813     0.02813     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.02813     0.02813     0.00000
          Theta0 qt 97.5%       0.02813     0.02813     0.00000
                   Theta1      57.89062    57.89062     0.00000
           Theta1 qt 2.5%      22.55859    22.55859     0.00000
             Theta1 qt 5%      22.55859    22.55859     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00031     0.00031     0.00000
      Model (SSD) p-value       0.80000     0.80000     0.00000
         Raggedness index       0.01815     0.01815     0.00000
       Raggedness p-value       0.80000     0.80000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 332 (31/01/17 at 14:48:00))
Total computing time for this run : 0h 0m 0s 134 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

