
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_50_16"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 379 (31/01/17 at 14:50:16)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_50_16_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_50_16_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 44
No. of loci              : 19
No. of usable loci       : 19 loci with less than  5.00 % missing data
No. of polymorphic sites : 19


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.34550
     2       150         2   0.18121
     3       150         2   0.49897
     4       150         2   0.17038
     5       150         2   0.07732
     6       150         2   0.27964
     7       150         2   0.42810
     8       150         2   0.07732
     9       150         2   0.05226
    10       150         2   0.21262
    11       150         2   0.36725
    12       150         2   0.03946
    13       150         2   0.14819
    14       150         2   0.14819
    15       150         2   0.06488
    16       150         2   0.01333
    17       150         2   0.01333
    18       150         2   0.29718
    19       150         2   0.14819
-------------------------------------------------
  Mean   150.000     2.000   0.18754
  s.d.     0.000     0.000   0.14501
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0538
Gene diversity           :  0.9526 +/-  0.0074

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 44

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 19
Number of usable nucleotide sites            : 19


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.563311 +/-     1.821142

Average gene diversity over loci              :     0.187543 +/-     0.106096

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    18.448768
  S.D. Theta(Hom) :     3.243366

         Theta(k) :    20.605975
95 % confidence interval limits for theta(k) : [  14.167681,  29.640070 ]

         Theta(S) :     3.402266
    S.D. Theta(S) :     1.075424

        Theta(Pi) :     3.563311
   S.D. Theta(Pi) :     2.015828

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     530     328.435
       1    1076    1148.405
       2    1842    2016.221
       3    2146    2370.990
       4    2095    2102.170
       5    1855    1499.913
       6     829     897.808
       7     502     464.132
       8     238     211.760
       9      45      86.727
      10      17      32.327
      11       0      11.096
      12       0       3.543
      13       0       1.062
      14       0       0.301
      15       0       0.081
      16       0       0.021
      17       0       0.005
      18       0       0.001
      19       0       0.000
      20       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.563
Mismatch observed variance          :        3.650
Tau                                 :        3.816
Theta0                              :        0.016
Theta1                              :       41.514

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00099947
          P(Sim. Ssd >= Obs. Ssd): 0.80000000
    Harpending's Raggedness index: 0.01845852
        P(Sim. Rag. >= Obs. Rag.): 0.50000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        530.0        489.3        478.5
          1.0       1076.0        929.0       1077.7
          2.0       1842.0       1798.6       1809.0
          3.0       2146.0       2263.3       2205.4
          4.0       2095.0       2071.3       2068.6
          5.0       1855.0       1552.6       1567.5
          6.0        829.0       1004.9        994.3
          7.0        502.0        563.5        541.9
          8.0        238.0        290.2        258.8
          9.0         45.0        126.5        109.9
         10.0         17.0         53.6         42.1
         11.0          0.0         17.9         14.6
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.816           2.590           4.777           3.627
                   Theta0:           0.016           0.002           1.552           0.255               
                   Theta1:          41.514          13.105       99999.000       40090.570
Mean No. of pairwise diff.           3.563           2.791           4.493           3.733
           No. pol. sites:              19              57              89          79.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        530.0        196.0       1207.0        478.5
          1.0       1076.0        408.0       1624.0       1077.7
          2.0       1842.0       1072.0       2743.0       1809.0
          3.0       2146.0       1630.0       2788.0       2205.4
          4.0       2095.0       1567.0       3004.0       2068.6
          5.0       1855.0        860.0       2236.0       1567.5
          6.0        829.0        485.0       1710.0        994.3
          7.0        502.0        236.0       1085.0        541.9
          8.0        238.0         15.0        594.0        258.8
          9.0         45.0          0.0        320.0        109.9
         10.0         17.0          0.0        224.0         42.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.816           2.590           4.777           3.627
                   Theta0:           0.016           0.002           1.552           0.255               
                   Theta1:          41.514          13.105       99999.000       40090.570
Mean No. of pairwise diff.           3.563           2.791           4.493           3.733
           No. pol. sites:              19              57              89          79.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        530.0        196.0       1207.0        478.5
          1.0       1076.0        408.0       1624.0       1077.7
          2.0       1842.0       1072.0       2743.0       1809.0
          3.0       2146.0       1630.0       2788.0       2205.4
          4.0       2095.0       1567.0       3004.0       2068.6
          5.0       1855.0        860.0       2236.0       1567.5
          6.0        829.0        485.0       1710.0        994.3
          7.0        502.0        236.0       1085.0        541.9
          8.0        238.0         15.0        594.0        258.8
          9.0         45.0          0.0        320.0        109.9
         10.0         17.0          0.0        224.0         42.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.816           2.590           4.777           3.627
                   Theta0:           0.016           0.002           1.552           0.255               
                   Theta1:          41.514          13.105       99999.000       40090.570
Mean No. of pairwise diff.           3.563           2.791           4.493           3.733
           No. pol. sites:              19              57              89          79.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        530.0        196.0       1207.0        478.5
          1.0       1076.0        408.0       1624.0       1077.7
          2.0       1842.0       1072.0       2743.0       1809.0
          3.0       2146.0       1630.0       2788.0       2205.4
          4.0       2095.0       1567.0       3004.0       2068.6
          5.0       1855.0        860.0       2236.0       1567.5
          6.0        829.0        485.0       1710.0        994.3
          7.0        502.0        236.0       1085.0        541.9
          8.0        238.0         15.0        594.0        258.8
          9.0         45.0          0.0        320.0        109.9
         10.0         17.0          0.0        224.0         42.1
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_50_16_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            19      19.000       0.000
  No. of usable loci            19      19.000       0.000
  No. of polym. loci            19      19.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_50_16_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.34550     0.34550     0.00000     0.34550
     2       0.18121     0.18121     0.00000     0.18121
     3       0.49897     0.49897     0.00000     0.49897
     4       0.17038     0.17038     0.00000     0.17038
     5       0.07732     0.07732     0.00000     0.07732
     6       0.27964     0.27964     0.00000     0.27964
     7       0.42810     0.42810     0.00000     0.42810
     8       0.07732     0.07732     0.00000     0.07732
     9       0.05226     0.05226     0.00000     0.05226
    10       0.21262     0.21262     0.00000     0.21262
    11       0.36725     0.36725     0.00000     0.36725
    12       0.03946     0.03946     0.00000     0.03946
    13       0.14819     0.14819     0.00000     0.14819
    14       0.14819     0.14819     0.00000     0.14819
    15       0.06488     0.06488     0.00000     0.06488
    16       0.01333     0.01333     0.00000     0.01333
    17       0.01333     0.01333     0.00000     0.01333
    18       0.29718     0.29718     0.00000     0.29718
    19       0.14819     0.14819     0.00000     0.14819
--------------------------------------------------------
  Mean       0.18754     0.18754     0.00000     0.18754
  s.d.       0.14501     0.14501     0.00000     0.14501
--------------------------------------------------------
<A NAME="31_01_17at14_50_16_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_50_16_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      20.60597    20.60597     0.00000
            Theta_k_lower      14.16768    14.16768     0.00000
            Theta_k_upper      29.64007    29.64007     0.00000
                  Theta_H      18.44877    18.44877     0.00000
             s.d. Theta_H       3.24337     3.24337     0.00000
                  Theta_S       3.40227     3.40227     0.00000
             s.d. Theta_S       1.07542     1.07542     0.00000
                 Theta_pi       3.56331     3.56331     0.00000
            s.d. Theta_pi       2.01583     2.01583     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_50_16_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.81641     3.81641     0.00000
              Tau qt 2.5%       2.58984     2.58984     0.00000
                Tau qt 5%       2.58984     2.58984     0.00000
               Tau qt 95%       4.56641     4.56641     0.00000
             Tau qt 97.5%       4.56641     4.56641     0.00000
                   Theta0       0.01562     0.01562     0.00000
           Theta0 qt 2.5%       0.00176     0.00176     0.00000
             Theta0 qt 5%       0.00176     0.00176     0.00000
            Theta0 qt 95%       0.87188     0.87188     0.00000
          Theta0 qt 97.5%       0.87188     0.87188     0.00000
                   Theta1      41.51367    41.51367     0.00000
           Theta1 qt 2.5%      13.10547    13.10547     0.00000
             Theta1 qt 5%      13.10547    13.10547     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00100     0.00100     0.00000
      Model (SSD) p-value       0.80000     0.80000     0.00000
         Raggedness index       0.01846     0.01846     0.00000
       Raggedness p-value       0.50000     0.50000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 379 (31/01/17 at 14:50:16))
Total computing time for this run : 0h 0m 0s 127 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

