
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_54_57"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 313 (31/01/17 at 14:54:57)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_54_57_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_54_57_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 49
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.15937
     2       150         2   0.23266
     3       150         2   0.05226
     4       150         2   0.01333
     5       150         2   0.31400
     6       150         2   0.19186
     7       150         2   0.01333
     8       150         2   0.31400
     9       150         2   0.44743
    10       150         2   0.11356
    11       150         2   0.48823
    12       150         2   0.12528
    13       150         2   0.10166
    14       150         2   0.36018
    15       150         2   0.05226
    16       150         2   0.50336
    17       150         2   0.24242
    18       150         2   0.17038
    19       150         2   0.15937
    20       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.20341
  s.d.     0.000     0.000   0.15657
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0494
Gene diversity           :  0.9570 +/-  0.0073

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 49

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.068277 +/-     2.040639

Average gene diversity over loci              :     0.203414 +/-     0.112939

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    20.564815
  S.D. Theta(Hom) :     3.886240

         Theta(k) :    24.926798
95 % confidence interval limits for theta(k) : [  17.352333,  35.483223 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     4.068277
   S.D. Theta(Pi) :     2.258790

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     481     216.923
       1     945     834.970
       2    1345    1621.908
       3    1785    2123.036
       4    1777    2110.547
       5    1811    1703.335
       6    1529    1165.505
       7     990     697.569
       8     412     374.115
       9      90     183.381
      10      10      83.551
      11       0      35.908
      12       0      14.748
      13       0       5.854
      14       0       2.267
      15       0       0.863
      16       0       0.325
      17       0       0.122
      18       0       0.045
      19       0       0.017
      20       0       0.006


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.068
Mismatch observed variance          :        4.417
Tau                                 :        4.654
Theta0                              :        0.007
Theta1                              :       24.575

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00125843
          P(Sim. Ssd >= Obs. Ssd): 0.60000000
    Harpending's Raggedness index: 0.01108602
        P(Sim. Rag. >= Obs. Rag.): 0.80000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        481.0        509.9        520.9
          1.0        945.0        878.0        808.0
          2.0       1345.0       1221.2       1297.3
          3.0       1785.0       1710.3       1751.8
          4.0       1777.0       1865.8       1917.7
          5.0       1811.0       1685.8       1730.8
          6.0       1529.0       1330.2       1319.5
          7.0        990.0        837.0        868.2
          8.0        412.0        529.6        501.8
          9.0         90.0        347.7        258.4
         10.0         10.0        160.5        119.9
         11.0          0.0         60.5         50.6
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.654           2.559           5.930           4.350
                   Theta0:           0.007           0.000           2.823           0.465               
                   Theta1:          24.575          20.740       99999.000       20045.783
Mean No. of pairwise diff.           4.068           3.077           6.026           4.322
           No. pol. sites:              20              52              85          65.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        481.0        321.0        848.0        520.9
          1.0        945.0        272.0       1637.0        808.0
          2.0       1345.0        324.0       2224.0       1297.3
          3.0       1785.0        762.0       2351.0       1751.8
          4.0       1777.0       1394.0       3101.0       1917.7
          5.0       1811.0       1163.0       2369.0       1730.8
          6.0       1529.0        464.0       2239.0       1319.5
          7.0        990.0        267.0       1380.0        868.2
          8.0        412.0         43.0       1330.0        501.8
          9.0         90.0          3.0       1210.0        258.4
         10.0         10.0          0.0        809.0        119.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.654           2.559           5.930           4.350
                   Theta0:           0.007           0.000           2.823           0.465               
                   Theta1:          24.575          20.740       99999.000       20045.783
Mean No. of pairwise diff.           4.068           3.077           6.026           4.322
           No. pol. sites:              20              52              85          65.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        481.0        321.0        848.0        520.9
          1.0        945.0        272.0       1637.0        808.0
          2.0       1345.0        324.0       2224.0       1297.3
          3.0       1785.0        762.0       2351.0       1751.8
          4.0       1777.0       1394.0       3101.0       1917.7
          5.0       1811.0       1163.0       2369.0       1730.8
          6.0       1529.0        464.0       2239.0       1319.5
          7.0        990.0        267.0       1380.0        868.2
          8.0        412.0         43.0       1330.0        501.8
          9.0         90.0          3.0       1210.0        258.4
         10.0         10.0          0.0        809.0        119.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.654           2.559           5.930           4.350
                   Theta0:           0.007           0.000           2.823           0.465               
                   Theta1:          24.575          20.740       99999.000       20045.783
Mean No. of pairwise diff.           4.068           3.077           6.026           4.322
           No. pol. sites:              20              52              85          65.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        481.0        321.0        848.0        520.9
          1.0        945.0        272.0       1637.0        808.0
          2.0       1345.0        324.0       2224.0       1297.3
          3.0       1785.0        762.0       2351.0       1751.8
          4.0       1777.0       1394.0       3101.0       1917.7
          5.0       1811.0       1163.0       2369.0       1730.8
          6.0       1529.0        464.0       2239.0       1319.5
          7.0        990.0        267.0       1380.0        868.2
          8.0        412.0         43.0       1330.0        501.8
          9.0         90.0          3.0       1210.0        258.4
         10.0         10.0          0.0        809.0        119.9
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_54_57_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_54_57_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.15937     0.15937     0.00000     0.15937
     2       0.23266     0.23266     0.00000     0.23266
     3       0.05226     0.05226     0.00000     0.05226
     4       0.01333     0.01333     0.00000     0.01333
     5       0.31400     0.31400     0.00000     0.31400
     6       0.19186     0.19186     0.00000     0.19186
     7       0.01333     0.01333     0.00000     0.01333
     8       0.31400     0.31400     0.00000     0.31400
     9       0.44743     0.44743     0.00000     0.44743
    10       0.11356     0.11356     0.00000     0.11356
    11       0.48823     0.48823     0.00000     0.48823
    12       0.12528     0.12528     0.00000     0.12528
    13       0.10166     0.10166     0.00000     0.10166
    14       0.36018     0.36018     0.00000     0.36018
    15       0.05226     0.05226     0.00000     0.05226
    16       0.50336     0.50336     0.00000     0.50336
    17       0.24242     0.24242     0.00000     0.24242
    18       0.17038     0.17038     0.00000     0.17038
    19       0.15937     0.15937     0.00000     0.15937
    20       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.20341     0.20341     0.00000     0.20341
  s.d.       0.15657     0.15657     0.00000     0.15657
--------------------------------------------------------
<A NAME="31_01_17at14_54_57_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_54_57_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      24.92680    24.92680     0.00000
            Theta_k_lower      17.35233    17.35233     0.00000
            Theta_k_upper      35.48322    35.48322     0.00000
                  Theta_H      20.56481    20.56481     0.00000
             s.d. Theta_H       3.88624     3.88624     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       4.06828     4.06828     0.00000
            s.d. Theta_pi       2.25879     2.25879     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_54_57_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.65430     4.65430     0.00000
              Tau qt 2.5%       2.55859     2.55859     0.00000
                Tau qt 5%       2.55859     2.55859     0.00000
               Tau qt 95%       5.45117     5.45117     0.00000
             Tau qt 97.5%       5.45117     5.45117     0.00000
                   Theta0       0.00703     0.00703     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.59785     1.59785     0.00000
          Theta0 qt 97.5%       1.59785     1.59785     0.00000
                   Theta1      24.57520    24.57520     0.00000
           Theta1 qt 2.5%      20.73975    20.73975     0.00000
             Theta1 qt 5%      20.73975    20.73975     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00126     0.00126     0.00000
      Model (SSD) p-value       0.60000     0.60000     0.00000
         Raggedness index       0.01109     0.01109     0.00000
       Raggedness p-value       0.80000     0.80000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 313 (31/01/17 at 14:54:57))
Total computing time for this run : 0h 0m 0s 122 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

