
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_50_10"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 347 (31/01/17 at 14:50:10)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_50_10_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_50_10_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 43
No. of loci              : 19
No. of usable loci       : 19 loci with less than  5.00 % missing data
No. of polymorphic sites : 19


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.10166
     2       150         2   0.34550
     3       150         2   0.17038
     4       150         2   0.02649
     5       150         2   0.49763
     6       150         2   0.13682
     7       150         2   0.08957
     8       150         2   0.28850
     9       150         2   0.26139
    10       150         2   0.36018
    11       150         2   0.10166
    12       150         2   0.12528
    13       150         2   0.38085
    14       150         2   0.21262
    15       150         2   0.02649
    16       150         2   0.01333
    17       150         2   0.17038
    18       150         2   0.32215
    19       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.19180
  s.d.     0.000     0.000   0.14265
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0410
Gene diversity           :  0.9655 +/-  0.0051

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 43

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 19
Number of usable nucleotide sites            : 19


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.644206 +/-     1.856352

Average gene diversity over loci              :     0.191800 +/-     0.108147

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    26.219714
  S.D. Theta(Hom) :     4.216253

         Theta(k) :    19.804082
95 % confidence interval limits for theta(k) : [  13.577772,  28.555190 ]

         Theta(S) :     3.402266
    S.D. Theta(S) :     1.075424

        Theta(Pi) :     3.644206
   S.D. Theta(Pi) :     2.054802

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     386     319.061
       1    1247    1112.950
       2    2049    1957.905
       3    1968    2319.248
       4    1569    2084.476
       5    1694    1519.182
       6    1362     937.381
       7     687     505.052
       8     181     243.338
       9      20     106.897
      10      12      43.528
      11       0      16.669
      12       0       6.080
      13       0       2.137
      14       0       0.730
      15       0       0.245
      16       0       0.081
      17       0       0.027
      18       0       0.009
      19       0       0.003
      20       0       0.001


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.644
Mismatch observed variance          :        3.875
Tau                                 :        2.777
Theta0                              :        1.167
Theta1                              :       75.391

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00382838
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.01932987
        P(Sim. Rag. >= Obs. Rag.): 0.40000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        386.0        352.7        446.7
          1.0       1247.0        969.6       1136.5
          2.0       2049.0       1607.9       1820.6
          3.0       1968.0       1900.8       2087.2
          4.0       1569.0       1841.5       1888.4
          5.0       1694.0       1518.5       1440.3
          6.0       1362.0       1115.4        971.2
          7.0        687.0        802.4        600.5
          8.0        181.0        511.3        350.3
          9.0         20.0        289.2        196.9
         10.0         12.0        141.3        108.3
         11.0          0.0         74.7         58.9
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.777           2.754           5.920           3.835
                   Theta0:           1.167           0.000           2.134           0.638               
                   Theta1:          75.391          19.319       99999.000       50025.728
Mean No. of pairwise diff.           3.644           2.681           5.707           4.180
           No. pol. sites:              19              89             108          98.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        386.0        181.0        751.0        446.7
          1.0       1247.0        283.0       2156.0       1136.5
          2.0       2049.0        692.0       2548.0       1820.6
          3.0       1968.0       1067.0       2595.0       2087.2
          4.0       1569.0       1337.0       2150.0       1888.4
          5.0       1694.0        948.0       2308.0       1440.3
          6.0       1362.0        405.0       1711.0        971.2
          7.0        687.0        130.0       1568.0        600.5
          8.0        181.0         11.0       1310.0        350.3
          9.0         20.0          0.0        769.0        196.9
         10.0         12.0          0.0        431.0        108.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.777           2.754           5.920           3.835
                   Theta0:           1.167           0.000           2.134           0.638               
                   Theta1:          75.391          19.319       99999.000       50025.728
Mean No. of pairwise diff.           3.644           2.681           5.707           4.180
           No. pol. sites:              19              89             108          98.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        386.0        181.0        751.0        446.7
          1.0       1247.0        283.0       2156.0       1136.5
          2.0       2049.0        692.0       2548.0       1820.6
          3.0       1968.0       1067.0       2595.0       2087.2
          4.0       1569.0       1337.0       2150.0       1888.4
          5.0       1694.0        948.0       2308.0       1440.3
          6.0       1362.0        405.0       1711.0        971.2
          7.0        687.0        130.0       1568.0        600.5
          8.0        181.0         11.0       1310.0        350.3
          9.0         20.0          0.0        769.0        196.9
         10.0         12.0          0.0        431.0        108.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.777           2.754           5.920           3.835
                   Theta0:           1.167           0.000           2.134           0.638               
                   Theta1:          75.391          19.319       99999.000       50025.728
Mean No. of pairwise diff.           3.644           2.681           5.707           4.180
           No. pol. sites:              19              89             108          98.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        386.0        181.0        751.0        446.7
          1.0       1247.0        283.0       2156.0       1136.5
          2.0       2049.0        692.0       2548.0       1820.6
          3.0       1968.0       1067.0       2595.0       2087.2
          4.0       1569.0       1337.0       2150.0       1888.4
          5.0       1694.0        948.0       2308.0       1440.3
          6.0       1362.0        405.0       1711.0        971.2
          7.0        687.0        130.0       1568.0        600.5
          8.0        181.0         11.0       1310.0        350.3
          9.0         20.0          0.0        769.0        196.9
         10.0         12.0          0.0        431.0        108.3
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_50_10_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            19      19.000       0.000
  No. of usable loci            19      19.000       0.000
  No. of polym. loci            19      19.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_50_10_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.10166     0.10166     0.00000     0.10166
     2       0.34550     0.34550     0.00000     0.34550
     3       0.17038     0.17038     0.00000     0.17038
     4       0.02649     0.02649     0.00000     0.02649
     5       0.49763     0.49763     0.00000     0.49763
     6       0.13682     0.13682     0.00000     0.13682
     7       0.08957     0.08957     0.00000     0.08957
     8       0.28850     0.28850     0.00000     0.28850
     9       0.26139     0.26139     0.00000     0.26139
    10       0.36018     0.36018     0.00000     0.36018
    11       0.10166     0.10166     0.00000     0.10166
    12       0.12528     0.12528     0.00000     0.12528
    13       0.38085     0.38085     0.00000     0.38085
    14       0.21262     0.21262     0.00000     0.21262
    15       0.02649     0.02649     0.00000     0.02649
    16       0.01333     0.01333     0.00000     0.01333
    17       0.17038     0.17038     0.00000     0.17038
    18       0.32215     0.32215     0.00000     0.32215
    19       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.19180     0.19180     0.00000     0.19180
  s.d.       0.14265     0.14265     0.00000     0.14265
--------------------------------------------------------
<A NAME="31_01_17at14_50_10_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_50_10_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      19.80408    19.80408     0.00000
            Theta_k_lower      13.57777    13.57777     0.00000
            Theta_k_upper      28.55519    28.55519     0.00000
                  Theta_H      26.21971    26.21971     0.00000
             s.d. Theta_H       4.21625     4.21625     0.00000
                  Theta_S       3.40227     3.40227     0.00000
             s.d. Theta_S       1.07542     1.07542     0.00000
                 Theta_pi       3.64421     3.64421     0.00000
            s.d. Theta_pi       2.05480     2.05480     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_50_10_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       2.77734     2.77734     0.00000
              Tau qt 2.5%       2.75391     2.75391     0.00000
                Tau qt 5%       2.75391     2.75391     0.00000
               Tau qt 95%       4.87695     4.87695     0.00000
             Tau qt 97.5%       4.87695     4.87695     0.00000
                   Theta0       1.16699     1.16699     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.83691     1.83691     0.00000
          Theta0 qt 97.5%       1.83691     1.83691     0.00000
                   Theta1      75.39062    75.39062     0.00000
           Theta1 qt 2.5%      19.31885    19.31885     0.00000
             Theta1 qt 5%      19.31885    19.31885     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00383     0.00383     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.01933     0.01933     0.00000
       Raggedness p-value       0.40000     0.40000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 347 (31/01/17 at 14:50:10))
Total computing time for this run : 0h 0m 0s 128 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

