
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_29_18"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 6 (31/01/17 at 16:29:18)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_29_18_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_29_18_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 60
No. of loci              : 76
No. of usable loci       : 76 loci with less than  5.00 % missing data
No. of polymorphic sites : 76


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.01333
     3       150         2   0.01333
     4       150         2   0.01333
     5       150         2   0.01333
     6       150         2   0.03946
     7       150         2   0.01333
     8       150         2   0.02649
     9       150         2   0.01333
    10       150         2   0.05226
    11       150         2   0.01333
    12       150         2   0.03946
    13       150         2   0.01333
    14       150         2   0.01333
    15       150         2   0.02649
    16       150         2   0.11356
    17       150         2   0.01333
    18       150         2   0.03946
    19       150         2   0.01333
    20       150         2   0.01333
    21       150         2   0.01333
    22       150         2   0.02649
    23       150         2   0.01333
    24       150         2   0.01333
    25       150         2   0.01333
    26       150         2   0.05226
    27       150         2   0.01333
    28       150         2   0.02649
    29       150         2   0.03946
    30       150         2   0.06488
    31       150         2   0.01333
    32       150         2   0.01333
    33       150         2   0.01333
    34       150         2   0.01333
    35       150         2   0.01333
    36       150         2   0.02649
    37       150         2   0.03946
    38       150         2   0.03946
    39       150         2   0.02649
    40       150         2   0.01333
    41       150         2   0.01333
    42       150         2   0.01333
    43       150         2   0.02649
    44       150         2   0.01333
    45       150         2   0.01333
    46       150         2   0.01333
    47       150         2   0.01333
    48       150         2   0.01333
    49       150         2   0.01333
    50       150         2   0.02649
    51       150         2   0.03946
    52       150         2   0.01333
    53       150         2   0.02649
    54       150         2   0.01333
    55       150         2   0.01333
    56       150         2   0.02649
    57       150         2   0.02649
    58       150         2   0.01333
    59       150         2   0.01333
    60       150         2   0.01333
    61       150         2   0.02649
    62       150         2   0.01333
    63       150         2   0.01333
    64       150         2   0.02649
    65       150         2   0.02649
    66       150         2   0.02649
    67       150         2   0.01333
    68       150         2   0.02649
    69       150         2   0.01333
    70       150         2   0.01333
    71       150         2   0.01333
    72       150         2   0.05226
    73       150         2   0.01333
    74       150         2   0.01333
    75       150         2   0.05226
    76       150         2   0.02649
-------------------------------------------------
  Mean   150.000     2.000   0.02273
  s.d.     0.000     0.000   0.01624
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.1679
Gene diversity           :  0.8377 +/-  0.0311

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 60

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 76
Number of usable nucleotide sites            : 76


Distance method : Pairwise difference
Mean number of pairwise differences           :     1.727338 +/-     1.012292

Average gene diversity over loci              :     0.022728 +/-     0.014744

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     4.183971
  S.D. Theta(Hom) :     1.044920

         Theta(k) :    36.572596
95 % confidence interval limits for theta(k) : [  25.948187,  51.263916 ]

         Theta(S) :    13.609063
    S.D. Theta(S) :     3.399804

        Theta(Pi) :     1.727338
   S.D. Theta(Pi) :     1.120509

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1814    1990.939
       1    3735    3430.201
       2    2952    2959.085
       3    1596    1704.326
       4     692     737.403
       5     252     255.679
       6     100      74.013
       7      25      18.401
       8       8       4.012
       9       1       0.779
      10       0       0.137
      11       0       0.022
      12       0       0.003
      13       0       0.000
      14       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        1.727
Mismatch observed variance          :        1.732
Tau                                 :        1.691
Theta0                              :        0.000
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00103629
          P(Sim. Ssd >= Obs. Ssd): 0.40000000
    Harpending's Raggedness index: 0.05751052
        P(Sim. Rag. >= Obs. Rag.): 0.20000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1814.0       1901.9       2059.2
          1.0       3735.0       3437.4       3482.8
          2.0       2952.0       3005.0       2945.4
          3.0       1596.0       1758.3       1660.6
          4.0        692.0        746.8        702.2
          5.0        252.0        250.5        237.5
          6.0        100.0         64.0         67.0
          7.0         25.0          9.6         16.2
          8.0          8.0          1.5          3.4
          9.0          1.0          0.0          0.6
         10.0          0.0          0.0          0.1
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.691           1.566           2.137           1.735
                   Theta0:           0.000           0.000           0.030           0.005               
                   Theta1:       99999.000           8.482       99999.000       80026.104
Mean No. of pairwise diff.           1.727           1.558           2.118           1.738
           No. pol. sites:              76             102             117         110.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1814.0       1233.0       2628.0       2059.2
          1.0       3735.0       2854.0       3757.0       3482.8
          2.0       2952.0       2423.0       3454.0       2945.4
          3.0       1596.0       1387.0       2259.0       1660.6
          4.0        692.0        509.0       1169.0        702.2
          5.0        252.0        131.0        449.0        237.5
          6.0        100.0         15.0        131.0         67.0
          7.0         25.0          0.0         31.0         16.2
          8.0          8.0          0.0          6.0          3.4
          9.0          1.0          0.0          0.0          0.6
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.691           1.566           2.137           1.735
                   Theta0:           0.000           0.000           0.030           0.005               
                   Theta1:       99999.000           8.482       99999.000       80026.104
Mean No. of pairwise diff.           1.727           1.558           2.118           1.738
           No. pol. sites:              76             102             117         110.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1814.0       1233.0       2628.0       2059.2
          1.0       3735.0       2854.0       3757.0       3482.8
          2.0       2952.0       2423.0       3454.0       2945.4
          3.0       1596.0       1387.0       2259.0       1660.6
          4.0        692.0        509.0       1169.0        702.2
          5.0        252.0        131.0        449.0        237.5
          6.0        100.0         15.0        131.0         67.0
          7.0         25.0          0.0         31.0         16.2
          8.0          8.0          0.0          6.0          3.4
          9.0          1.0          0.0          0.0          0.6
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.691           1.566           2.137           1.735
                   Theta0:           0.000           0.000           0.030           0.005               
                   Theta1:       99999.000           8.482       99999.000       80026.104
Mean No. of pairwise diff.           1.727           1.558           2.118           1.738
           No. pol. sites:              76             102             117         110.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1814.0       1233.0       2628.0       2059.2
          1.0       3735.0       2854.0       3757.0       3482.8
          2.0       2952.0       2423.0       3454.0       2945.4
          3.0       1596.0       1387.0       2259.0       1660.6
          4.0        692.0        509.0       1169.0        702.2
          5.0        252.0        131.0        449.0        237.5
          6.0        100.0         15.0        131.0         67.0
          7.0         25.0          0.0         31.0         16.2
          8.0          8.0          0.0          6.0          3.4
          9.0          1.0          0.0          0.0          0.6
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_29_18_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            76      76.000       0.000
  No. of usable loci            76      76.000       0.000
  No. of polym. loci            76      76.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_29_18_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.01333     0.01333     0.00000     0.01333
     3       0.01333     0.01333     0.00000     0.01333
     4       0.01333     0.01333     0.00000     0.01333
     5       0.01333     0.01333     0.00000     0.01333
     6       0.03946     0.03946     0.00000     0.03946
     7       0.01333     0.01333     0.00000     0.01333
     8       0.02649     0.02649     0.00000     0.02649
     9       0.01333     0.01333     0.00000     0.01333
    10       0.05226     0.05226     0.00000     0.05226
    11       0.01333     0.01333     0.00000     0.01333
    12       0.03946     0.03946     0.00000     0.03946
    13       0.01333     0.01333     0.00000     0.01333
    14       0.01333     0.01333     0.00000     0.01333
    15       0.02649     0.02649     0.00000     0.02649
    16       0.11356     0.11356     0.00000     0.11356
    17       0.01333     0.01333     0.00000     0.01333
    18       0.03946     0.03946     0.00000     0.03946
    19       0.01333     0.01333     0.00000     0.01333
    20       0.01333     0.01333     0.00000     0.01333
    21       0.01333     0.01333     0.00000     0.01333
    22       0.02649     0.02649     0.00000     0.02649
    23       0.01333     0.01333     0.00000     0.01333
    24       0.01333     0.01333     0.00000     0.01333
    25       0.01333     0.01333     0.00000     0.01333
    26       0.05226     0.05226     0.00000     0.05226
    27       0.01333     0.01333     0.00000     0.01333
    28       0.02649     0.02649     0.00000     0.02649
    29       0.03946     0.03946     0.00000     0.03946
    30       0.06488     0.06488     0.00000     0.06488
    31       0.01333     0.01333     0.00000     0.01333
    32       0.01333     0.01333     0.00000     0.01333
    33       0.01333     0.01333     0.00000     0.01333
    34       0.01333     0.01333     0.00000     0.01333
    35       0.01333     0.01333     0.00000     0.01333
    36       0.02649     0.02649     0.00000     0.02649
    37       0.03946     0.03946     0.00000     0.03946
    38       0.03946     0.03946     0.00000     0.03946
    39       0.02649     0.02649     0.00000     0.02649
    40       0.01333     0.01333     0.00000     0.01333
    41       0.01333     0.01333     0.00000     0.01333
    42       0.01333     0.01333     0.00000     0.01333
    43       0.02649     0.02649     0.00000     0.02649
    44       0.01333     0.01333     0.00000     0.01333
    45       0.01333     0.01333     0.00000     0.01333
    46       0.01333     0.01333     0.00000     0.01333
    47       0.01333     0.01333     0.00000     0.01333
    48       0.01333     0.01333     0.00000     0.01333
    49       0.01333     0.01333     0.00000     0.01333
    50       0.02649     0.02649     0.00000     0.02649
    51       0.03946     0.03946     0.00000     0.03946
    52       0.01333     0.01333     0.00000     0.01333
    53       0.02649     0.02649     0.00000     0.02649
    54       0.01333     0.01333     0.00000     0.01333
    55       0.01333     0.01333     0.00000     0.01333
    56       0.02649     0.02649     0.00000     0.02649
    57       0.02649     0.02649     0.00000     0.02649
    58       0.01333     0.01333     0.00000     0.01333
    59       0.01333     0.01333     0.00000     0.01333
    60       0.01333     0.01333     0.00000     0.01333
    61       0.02649     0.02649     0.00000     0.02649
    62       0.01333     0.01333     0.00000     0.01333
    63       0.01333     0.01333     0.00000     0.01333
    64       0.02649     0.02649     0.00000     0.02649
    65       0.02649     0.02649     0.00000     0.02649
    66       0.02649     0.02649     0.00000     0.02649
    67       0.01333     0.01333     0.00000     0.01333
    68       0.02649     0.02649     0.00000     0.02649
    69       0.01333     0.01333     0.00000     0.01333
    70       0.01333     0.01333     0.00000     0.01333
    71       0.01333     0.01333     0.00000     0.01333
    72       0.05226     0.05226     0.00000     0.05226
    73       0.01333     0.01333     0.00000     0.01333
    74       0.01333     0.01333     0.00000     0.01333
    75       0.05226     0.05226     0.00000     0.05226
    76       0.02649     0.02649     0.00000     0.02649
--------------------------------------------------------
  Mean       0.02273     0.02273     0.00000     0.02273
  s.d.       0.01624     0.01624     0.00000     0.01624
--------------------------------------------------------
<A NAME="31_01_17at16_29_18_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
    72             2       2.000       0.000           2
    73             2       2.000       0.000           2
    74             2       2.000       0.000           2
    75             2       2.000       0.000           2
    76             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_29_18_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      36.57260    36.57260     0.00000
            Theta_k_lower      25.94819    25.94819     0.00000
            Theta_k_upper      51.26392    51.26392     0.00000
                  Theta_H       4.18397     4.18397     0.00000
             s.d. Theta_H       1.04492     1.04492     0.00000
                  Theta_S      13.60906    13.60906     0.00000
             s.d. Theta_S       3.39980     3.39980     0.00000
                 Theta_pi       1.72734     1.72734     0.00000
            s.d. Theta_pi       1.12051     1.12051     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_29_18_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.69141     1.69141     0.00000
              Tau qt 2.5%       1.56641     1.56641     0.00000
                Tau qt 5%       1.56641     1.56641     0.00000
               Tau qt 95%       1.80664     1.80664     0.00000
             Tau qt 97.5%       1.80664     1.80664     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.01934     0.01934     0.00000
          Theta0 qt 97.5%       0.01934     0.01934     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%       8.48242     8.48242     0.00000
             Theta1 qt 5%       8.48242     8.48242     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00104     0.00104     0.00000
      Model (SSD) p-value       0.40000     0.40000     0.00000
         Raggedness index       0.05751     0.05751     0.00000
       Raggedness p-value       0.20000     0.20000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 6 (31/01/17 at 16:29:18))
Total computing time for this run : 0h 0m 0s 108 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

