
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_51_42"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 121 (31/01/17 at 14:51:42)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_51_42_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_51_42_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 41
No. of loci              : 19
No. of usable loci       : 19 loci with less than  5.00 % missing data
No. of polymorphic sites : 19


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.31400
     2       150         2   0.08957
     3       150         2   0.10166
     4       150         2   0.25199
     5       150         2   0.31400
     6       150         2   0.06488
     7       150         2   0.13682
     8       150         2   0.43320
     9       150         2   0.14819
    10       150         2   0.10166
    11       150         2   0.31400
    12       150         2   0.46389
    13       150         2   0.05226
    14       150         2   0.15937
    15       150         2   0.49047
    16       150         2   0.23266
    17       150         2   0.03946
    18       150         2   0.08957
    19       150         2   0.18121
-------------------------------------------------
  Mean   150.000     2.000   0.20941
  s.d.     0.000     0.000   0.14293
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0446
Gene diversity           :  0.9618 +/-  0.0050

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 41

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 19
Number of usable nucleotide sites            : 19


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.978881 +/-     2.001828

Average gene diversity over loci              :     0.209415 +/-     0.116623

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    23.467286
  S.D. Theta(Hom) :     3.394117

         Theta(k) :    18.258484
95 % confidence interval limits for theta(k) : [  12.442238,  26.463077 ]

         Theta(S) :     3.402266
    S.D. Theta(S) :     1.075424

        Theta(Pi) :     3.978881
   S.D. Theta(Pi) :     2.215830

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     427     224.450
       1     856     864.507
       2    1361    1674.907
       3    2122    2178.096
       4    2183    2140.887
       5    1706    1698.471
       6    1149    1134.423
       7     781     657.141
       8     426     337.646
       9     151     156.659
      10      13      66.621
      11       0      26.304
      12       0       9.753
      13       0       3.431
      14       0       1.156
      15       0       0.377
      16       0       0.119
      17       0       0.037
      18       0       0.011
      19       0       0.003
      20       0       0.001


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.979
Mismatch observed variance          :        4.160
Tau                                 :        4.211
Theta0                              :        0.014
Theta1                              :       49.902

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00049784
          P(Sim. Ssd >= Obs. Ssd): 0.70000000
    Harpending's Raggedness index: 0.01534244
        P(Sim. Rag. >= Obs. Rag.): 0.50000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        427.0        363.9        366.8
          1.0        856.0        688.6        834.4
          2.0       1361.0       1165.0       1509.9
          3.0       2122.0       1584.9       2016.8
          4.0       2183.0       1990.3       2081.2
          5.0       1706.0       1903.9       1737.4
          6.0       1149.0       1523.2       1214.7
          7.0        781.0        983.1        729.8
          8.0        426.0        571.0        384.2
          9.0        151.0        255.4        179.9
         10.0         13.0         95.8         75.9
         11.0          0.0         36.2         29.1
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.211           3.127           5.654           4.509
                   Theta0:           0.014           0.000           1.378           0.295               
                   Theta1:          49.902          19.014       99999.000       40021.777
Mean No. of pairwise diff.           3.979           3.371           5.280           4.445
           No. pol. sites:              19              79             110          94.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        427.0        181.0        716.0        366.8
          1.0        856.0        280.0       1405.0        834.4
          2.0       1361.0        565.0       1983.0       1509.9
          3.0       2122.0       1085.0       2274.0       2016.8
          4.0       2183.0       1576.0       2707.0       2081.2
          5.0       1706.0       1474.0       2326.0       1737.4
          6.0       1149.0        982.0       1899.0       1214.7
          7.0        781.0        450.0       1456.0        729.8
          8.0        426.0        118.0       1022.0        384.2
          9.0        151.0          0.0        498.0        179.9
         10.0         13.0          0.0        195.0         75.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.211           3.127           5.654           4.509
                   Theta0:           0.014           0.000           1.378           0.295               
                   Theta1:          49.902          19.014       99999.000       40021.777
Mean No. of pairwise diff.           3.979           3.371           5.280           4.445
           No. pol. sites:              19              79             110          94.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        427.0        181.0        716.0        366.8
          1.0        856.0        280.0       1405.0        834.4
          2.0       1361.0        565.0       1983.0       1509.9
          3.0       2122.0       1085.0       2274.0       2016.8
          4.0       2183.0       1576.0       2707.0       2081.2
          5.0       1706.0       1474.0       2326.0       1737.4
          6.0       1149.0        982.0       1899.0       1214.7
          7.0        781.0        450.0       1456.0        729.8
          8.0        426.0        118.0       1022.0        384.2
          9.0        151.0          0.0        498.0        179.9
         10.0         13.0          0.0        195.0         75.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.211           3.127           5.654           4.509
                   Theta0:           0.014           0.000           1.378           0.295               
                   Theta1:          49.902          19.014       99999.000       40021.777
Mean No. of pairwise diff.           3.979           3.371           5.280           4.445
           No. pol. sites:              19              79             110          94.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        427.0        181.0        716.0        366.8
          1.0        856.0        280.0       1405.0        834.4
          2.0       1361.0        565.0       1983.0       1509.9
          3.0       2122.0       1085.0       2274.0       2016.8
          4.0       2183.0       1576.0       2707.0       2081.2
          5.0       1706.0       1474.0       2326.0       1737.4
          6.0       1149.0        982.0       1899.0       1214.7
          7.0        781.0        450.0       1456.0        729.8
          8.0        426.0        118.0       1022.0        384.2
          9.0        151.0          0.0        498.0        179.9
         10.0         13.0          0.0        195.0         75.9
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_51_42_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            19      19.000       0.000
  No. of usable loci            19      19.000       0.000
  No. of polym. loci            19      19.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_51_42_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.31400     0.31400     0.00000     0.31400
     2       0.08957     0.08957     0.00000     0.08957
     3       0.10166     0.10166     0.00000     0.10166
     4       0.25199     0.25199     0.00000     0.25199
     5       0.31400     0.31400     0.00000     0.31400
     6       0.06488     0.06488     0.00000     0.06488
     7       0.13682     0.13682     0.00000     0.13682
     8       0.43320     0.43320     0.00000     0.43320
     9       0.14819     0.14819     0.00000     0.14819
    10       0.10166     0.10166     0.00000     0.10166
    11       0.31400     0.31400     0.00000     0.31400
    12       0.46389     0.46389     0.00000     0.46389
    13       0.05226     0.05226     0.00000     0.05226
    14       0.15937     0.15937     0.00000     0.15937
    15       0.49047     0.49047     0.00000     0.49047
    16       0.23266     0.23266     0.00000     0.23266
    17       0.03946     0.03946     0.00000     0.03946
    18       0.08957     0.08957     0.00000     0.08957
    19       0.18121     0.18121     0.00000     0.18121
--------------------------------------------------------
  Mean       0.20941     0.20941     0.00000     0.20941
  s.d.       0.14293     0.14293     0.00000     0.14293
--------------------------------------------------------
<A NAME="31_01_17at14_51_42_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_51_42_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      18.25848    18.25848     0.00000
            Theta_k_lower      12.44224    12.44224     0.00000
            Theta_k_upper      26.46308    26.46308     0.00000
                  Theta_H      23.46729    23.46729     0.00000
             s.d. Theta_H       3.39412     3.39412     0.00000
                  Theta_S       3.40227     3.40227     0.00000
             s.d. Theta_S       1.07542     1.07542     0.00000
                 Theta_pi       3.97888     3.97888     0.00000
            s.d. Theta_pi       2.21583     2.21583     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_51_42_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.21094     4.21094     0.00000
              Tau qt 2.5%       3.12695     3.12695     0.00000
                Tau qt 5%       3.12695     3.12695     0.00000
               Tau qt 95%       5.50781     5.50781     0.00000
             Tau qt 97.5%       5.50781     5.50781     0.00000
                   Theta0       0.01406     0.01406     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.74531     0.74531     0.00000
          Theta0 qt 97.5%       0.74531     0.74531     0.00000
                   Theta1      49.90234    49.90234     0.00000
           Theta1 qt 2.5%      19.01367    19.01367     0.00000
             Theta1 qt 5%      19.01367    19.01367     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00050     0.00050     0.00000
      Model (SSD) p-value       0.70000     0.70000     0.00000
         Raggedness index       0.01534     0.01534     0.00000
       Raggedness p-value       0.50000     0.50000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 121 (31/01/17 at 14:51:42))
Total computing time for this run : 0h 0m 0s 151 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

