
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_48_09"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 395 (31/01/17 at 14:48:09)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_48_09_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_48_09_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 47
No. of loci              : 19
No. of usable loci       : 19 loci with less than  5.00 % missing data
No. of polymorphic sites : 19


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.07732
     2       150         2   0.15937
     3       150         2   0.03946
     4       150         2   0.02649
     5       150         2   0.27964
     6       150         2   0.33011
     7       150         2   0.20233
     8       150         2   0.10166
     9       150         2   0.19186
    10       150         2   0.25199
    11       150         2   0.35293
    12       150         2   0.06488
    13       150         2   0.07732
    14       150         2   0.35293
    15       150         2   0.21262
    16       150         2   0.37414
    17       150         2   0.25199
    18       150         2   0.21262
    19       150         2   0.02649
-------------------------------------------------
  Mean   150.000     2.000   0.18874
  s.d.     0.000     0.000   0.11772
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0550
Gene diversity           :  0.9513 +/-  0.0083

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 47

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 19
Number of usable nucleotide sites            : 19


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.586130 +/-     1.831076

Average gene diversity over loci              :     0.188744 +/-     0.106675

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    17.915174
  S.D. Theta(Hom) :     3.444995

         Theta(k) :    23.134022
95 % confidence interval limits for theta(k) : [  16.029990,  33.058962 ]

         Theta(S) :     3.402266
    S.D. Theta(S) :     1.075424

        Theta(Pi) :     3.586130
   S.D. Theta(Pi) :     2.026824

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     544     312.696
       1    1177    1115.433
       2    1518    1992.042
       3    2143    2375.007
       4    2379    2126.852
       5    1609    1526.127
       6    1052     914.123
       7     511     470.187
       8     198     212.034
       9      41      85.176
      10       3      30.866
      11       0      10.194
      12       0       3.095
      13       0       0.870
      14       0       0.228
      15       0       0.056
      16       0       0.013
      17       0       0.003
      18       0       0.001
      19       0       0.000
      20       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.586
Mismatch observed variance          :        3.608
Tau                                 :        3.938
Theta0                              :        0.007
Theta1                              :       36.494

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00154633
          P(Sim. Ssd >= Obs. Ssd): 0.60000000
    Harpending's Raggedness index: 0.01828300
        P(Sim. Rag. >= Obs. Rag.): 0.60000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        544.0        447.1        487.1
          1.0       1177.0        955.2       1030.6
          2.0       1518.0       1613.1       1728.0
          3.0       2143.0       2029.7       2144.8
          4.0       2379.0       2092.5       2060.8
          5.0       1609.0       1690.3       1603.4
          6.0       1052.0       1202.9       1045.4
          7.0        511.0        683.5        585.9
          8.0        198.0        312.6        287.8
          9.0         41.0        105.6        125.8
         10.0          3.0         27.2         49.5
         11.0          0.0         12.6         17.7
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.938           2.521           5.260           3.889
                   Theta0:           0.007           0.000           2.180           0.265               
                   Theta1:          36.494          26.052       99999.000       50072.965
Mean No. of pairwise diff.           3.586           2.811           4.857           3.847
           No. pol. sites:              19              58              85          72.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        544.0        229.0        604.0        487.1
          1.0       1177.0        391.0       1874.0       1030.6
          2.0       1518.0        583.0       2897.0       1728.0
          3.0       2143.0        976.0       2828.0       2144.8
          4.0       2379.0       1461.0       2864.0       2060.8
          5.0       1609.0        929.0       2766.0       1603.4
          6.0       1052.0        362.0       2312.0       1045.4
          7.0        511.0         70.0       1323.0        585.9
          8.0        198.0          2.0        759.0        287.8
          9.0         41.0          0.0        382.0        125.8
         10.0          3.0          0.0        141.0         49.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.938           2.521           5.260           3.889
                   Theta0:           0.007           0.000           2.180           0.265               
                   Theta1:          36.494          26.052       99999.000       50072.965
Mean No. of pairwise diff.           3.586           2.811           4.857           3.847
           No. pol. sites:              19              58              85          72.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        544.0        229.0        604.0        487.1
          1.0       1177.0        391.0       1874.0       1030.6
          2.0       1518.0        583.0       2897.0       1728.0
          3.0       2143.0        976.0       2828.0       2144.8
          4.0       2379.0       1461.0       2864.0       2060.8
          5.0       1609.0        929.0       2766.0       1603.4
          6.0       1052.0        362.0       2312.0       1045.4
          7.0        511.0         70.0       1323.0        585.9
          8.0        198.0          2.0        759.0        287.8
          9.0         41.0          0.0        382.0        125.8
         10.0          3.0          0.0        141.0         49.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.938           2.521           5.260           3.889
                   Theta0:           0.007           0.000           2.180           0.265               
                   Theta1:          36.494          26.052       99999.000       50072.965
Mean No. of pairwise diff.           3.586           2.811           4.857           3.847
           No. pol. sites:              19              58              85          72.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        544.0        229.0        604.0        487.1
          1.0       1177.0        391.0       1874.0       1030.6
          2.0       1518.0        583.0       2897.0       1728.0
          3.0       2143.0        976.0       2828.0       2144.8
          4.0       2379.0       1461.0       2864.0       2060.8
          5.0       1609.0        929.0       2766.0       1603.4
          6.0       1052.0        362.0       2312.0       1045.4
          7.0        511.0         70.0       1323.0        585.9
          8.0        198.0          2.0        759.0        287.8
          9.0         41.0          0.0        382.0        125.8
         10.0          3.0          0.0        141.0         49.5
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_48_09_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            19      19.000       0.000
  No. of usable loci            19      19.000       0.000
  No. of polym. loci            19      19.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_48_09_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.07732     0.07732     0.00000     0.07732
     2       0.15937     0.15937     0.00000     0.15937
     3       0.03946     0.03946     0.00000     0.03946
     4       0.02649     0.02649     0.00000     0.02649
     5       0.27964     0.27964     0.00000     0.27964
     6       0.33011     0.33011     0.00000     0.33011
     7       0.20233     0.20233     0.00000     0.20233
     8       0.10166     0.10166     0.00000     0.10166
     9       0.19186     0.19186     0.00000     0.19186
    10       0.25199     0.25199     0.00000     0.25199
    11       0.35293     0.35293     0.00000     0.35293
    12       0.06488     0.06488     0.00000     0.06488
    13       0.07732     0.07732     0.00000     0.07732
    14       0.35293     0.35293     0.00000     0.35293
    15       0.21262     0.21262     0.00000     0.21262
    16       0.37414     0.37414     0.00000     0.37414
    17       0.25199     0.25199     0.00000     0.25199
    18       0.21262     0.21262     0.00000     0.21262
    19       0.02649     0.02649     0.00000     0.02649
--------------------------------------------------------
  Mean       0.18874     0.18874     0.00000     0.18874
  s.d.       0.11772     0.11772     0.00000     0.11772
--------------------------------------------------------
<A NAME="31_01_17at14_48_09_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_48_09_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      23.13402    23.13402     0.00000
            Theta_k_lower      16.02999    16.02999     0.00000
            Theta_k_upper      33.05896    33.05896     0.00000
                  Theta_H      17.91517    17.91517     0.00000
             s.d. Theta_H       3.44500     3.44500     0.00000
                  Theta_S       3.40227     3.40227     0.00000
             s.d. Theta_S       1.07542     1.07542     0.00000
                 Theta_pi       3.58613     3.58613     0.00000
            s.d. Theta_pi       2.02682     2.02682     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_48_09_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.93750     3.93750     0.00000
              Tau qt 2.5%       2.52148     2.52148     0.00000
                Tau qt 5%       2.52148     2.52148     0.00000
               Tau qt 95%       5.16992     5.16992     0.00000
             Tau qt 97.5%       5.16992     5.16992     0.00000
                   Theta0       0.00703     0.00703     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.40957     0.40957     0.00000
          Theta0 qt 97.5%       0.40957     0.40957     0.00000
                   Theta1      36.49414    36.49414     0.00000
           Theta1 qt 2.5%      26.05225    26.05225     0.00000
             Theta1 qt 5%      26.05225    26.05225     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00155     0.00155     0.00000
      Model (SSD) p-value       0.60000     0.60000     0.00000
         Raggedness index       0.01828     0.01828     0.00000
       Raggedness p-value       0.60000     0.60000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 395 (31/01/17 at 14:48:09))
Total computing time for this run : 0h 0m 0s 156 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

