
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_44_34"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 21 (31/01/17 at 16:44:34)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_44_34_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_44_34_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 70
No. of loci              : 27
No. of usable loci       : 27 loci with less than  5.00 % missing data
No. of polymorphic sites : 27


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.10166
     2       150         2   0.49897
     3       150         2   0.32215
     4       150         2   0.12528
     5       150         2   0.08957
     6       150         2   0.13682
     7       150         2   0.44743
     8       150         2   0.28850
     9       150         2   0.01333
    10       150         2   0.08957
    11       150         2   0.42282
    12       150         2   0.50013
    13       150         2   0.25199
    14       150         2   0.03946
    15       150         2   0.37414
    16       150         2   0.45602
    17       150         2   0.07732
    18       150         2   0.01333
    19       150         2   0.02649
    20       150         2   0.26139
    21       150         2   0.41736
    22       150         2   0.03946
    23       150         2   0.05226
    24       150         2   0.11356
    25       150         2   0.29718
    26       150         2   0.41736
    27       150         2   0.20233
-------------------------------------------------
  Mean   150.000     2.000   0.22503
  s.d.     0.000     0.000   0.16891
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0249
Gene diversity           :  0.9817 +/-  0.0036

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 70

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 27
Number of usable nucleotide sites            : 27


Distance method : Pairwise difference
Mean number of pairwise differences           :     6.075884 +/-     2.908867

Average gene diversity over loci              :     0.225033 +/-     0.119253

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    51.657198
  S.D. Theta(Hom) :    10.736385

         Theta(k) :    50.488748
95 % confidence interval limits for theta(k) : [  36.176731,  70.272284 ]

         Theta(S) :     4.834799
    S.D. Theta(S) :     1.410378

        Theta(Pi) :     6.075884
   S.D. Theta(Pi) :     3.219834

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     205       0.000
       1     299       0.000
       2     485       0.000
       3     501       0.000
       4    1035       0.000
       5    1530       0.000
       6    2409       0.000
       7    1708       0.000
       8    1498       0.000
       9     741       0.000
      10     403       0.000
      11     183       0.000
      12     112       0.000
      13      38       0.000
      14      28       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        6.076
Mismatch observed variance          :        5.889
Tau                                 :        6.467
Theta0                              :        0.000
Theta1                              :     1180.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00649226
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.02105295
        P(Sim. Rag. >= Obs. Rag.): 0.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        205.0         35.2         26.7
          1.0        299.0        148.5        121.0
          2.0        485.0        433.0        370.2
          3.0        501.0        842.9        786.9
          4.0       1035.0       1329.6       1266.0
          5.0       1530.0       1665.4       1633.8
          6.0       2409.0       1766.6       1759.0
          7.0       1708.0       1608.2       1624.0
          8.0       1498.0       1293.5       1312.2
          9.0        741.0        922.5        942.6
         10.0        403.0        538.0        609.5
         11.0        183.0        318.3        358.2
         12.0        112.0        154.6        193.0
         13.0         38.0         68.4         96.0
         14.0         28.0         27.9         44.3
         15.0          0.0         13.6         19.1
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           6.467           4.621           7.129           6.117
                   Theta0:           0.000           0.000           1.132           0.233               
                   Theta1:        1180.000         122.500       99999.000       60097.400
Mean No. of pairwise diff.           6.076           5.591           6.919           6.270
           No. pol. sites:              27             237             280         257.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        205.0         14.0         69.0         26.7
          1.0        299.0         43.0        330.0        121.0
          2.0        485.0        204.0        730.0        370.2
          3.0        501.0        521.0       1247.0        786.9
          4.0       1035.0        950.0       1668.0       1266.0
          5.0       1530.0       1423.0       1813.0       1633.8
          6.0       2409.0       1586.0       1935.0       1759.0
          7.0       1708.0       1346.0       1859.0       1624.0
          8.0       1498.0        982.0       1692.0       1312.2
          9.0        741.0        721.0       1280.0        942.6
         10.0        403.0        417.0        819.0        609.5
         11.0        183.0        185.0        533.0        358.2
         12.0        112.0         49.0        218.0        193.0
         13.0         38.0         20.0        144.0         96.0
         14.0         28.0          3.0         96.0         44.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           6.467           4.621           7.129           6.117
                   Theta0:           0.000           0.000           1.132           0.233               
                   Theta1:        1180.000         122.500       99999.000       60097.400
Mean No. of pairwise diff.           6.076           5.591           6.919           6.270
           No. pol. sites:              27             237             280         257.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        205.0         14.0         69.0         26.7
          1.0        299.0         43.0        330.0        121.0
          2.0        485.0        204.0        730.0        370.2
          3.0        501.0        521.0       1247.0        786.9
          4.0       1035.0        950.0       1668.0       1266.0
          5.0       1530.0       1423.0       1813.0       1633.8
          6.0       2409.0       1586.0       1935.0       1759.0
          7.0       1708.0       1346.0       1859.0       1624.0
          8.0       1498.0        982.0       1692.0       1312.2
          9.0        741.0        721.0       1280.0        942.6
         10.0        403.0        417.0        819.0        609.5
         11.0        183.0        185.0        533.0        358.2
         12.0        112.0         49.0        218.0        193.0
         13.0         38.0         20.0        144.0         96.0
         14.0         28.0          3.0         96.0         44.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           6.467           4.621           7.129           6.117
                   Theta0:           0.000           0.000           1.132           0.233               
                   Theta1:        1180.000         122.500       99999.000       60097.400
Mean No. of pairwise diff.           6.076           5.591           6.919           6.270
           No. pol. sites:              27             237             280         257.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        205.0         14.0         69.0         26.7
          1.0        299.0         43.0        330.0        121.0
          2.0        485.0        204.0        730.0        370.2
          3.0        501.0        521.0       1247.0        786.9
          4.0       1035.0        950.0       1668.0       1266.0
          5.0       1530.0       1423.0       1813.0       1633.8
          6.0       2409.0       1586.0       1935.0       1759.0
          7.0       1708.0       1346.0       1859.0       1624.0
          8.0       1498.0        982.0       1692.0       1312.2
          9.0        741.0        721.0       1280.0        942.6
         10.0        403.0        417.0        819.0        609.5
         11.0        183.0        185.0        533.0        358.2
         12.0        112.0         49.0        218.0        193.0
         13.0         38.0         20.0        144.0         96.0
         14.0         28.0          3.0         96.0         44.3
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_44_34_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            27      27.000       0.000
  No. of usable loci            27      27.000       0.000
  No. of polym. loci            27      27.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_44_34_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.10166     0.10166     0.00000     0.10166
     2       0.49897     0.49897     0.00000     0.49897
     3       0.32215     0.32215     0.00000     0.32215
     4       0.12528     0.12528     0.00000     0.12528
     5       0.08957     0.08957     0.00000     0.08957
     6       0.13682     0.13682     0.00000     0.13682
     7       0.44743     0.44743     0.00000     0.44743
     8       0.28850     0.28850     0.00000     0.28850
     9       0.01333     0.01333     0.00000     0.01333
    10       0.08957     0.08957     0.00000     0.08957
    11       0.42282     0.42282     0.00000     0.42282
    12       0.50013     0.50013     0.00000     0.50013
    13       0.25199     0.25199     0.00000     0.25199
    14       0.03946     0.03946     0.00000     0.03946
    15       0.37414     0.37414     0.00000     0.37414
    16       0.45602     0.45602     0.00000     0.45602
    17       0.07732     0.07732     0.00000     0.07732
    18       0.01333     0.01333     0.00000     0.01333
    19       0.02649     0.02649     0.00000     0.02649
    20       0.26139     0.26139     0.00000     0.26139
    21       0.41736     0.41736     0.00000     0.41736
    22       0.03946     0.03946     0.00000     0.03946
    23       0.05226     0.05226     0.00000     0.05226
    24       0.11356     0.11356     0.00000     0.11356
    25       0.29718     0.29718     0.00000     0.29718
    26       0.41736     0.41736     0.00000     0.41736
    27       0.20233     0.20233     0.00000     0.20233
--------------------------------------------------------
  Mean       0.22503     0.22503     0.00000     0.22503
  s.d.       0.16891     0.16891     0.00000     0.16891
--------------------------------------------------------
<A NAME="31_01_17at16_44_34_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_44_34_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      50.48875    50.48875     0.00000
            Theta_k_lower      36.17673    36.17673     0.00000
            Theta_k_upper      70.27228    70.27228     0.00000
                  Theta_H      51.65720    51.65720     0.00000
             s.d. Theta_H      10.73638    10.73638     0.00000
                  Theta_S       4.83480     4.83480     0.00000
             s.d. Theta_S       1.41038     1.41038     0.00000
                 Theta_pi       6.07588     6.07588     0.00000
            s.d. Theta_pi       3.21983     3.21983     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_44_34_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       6.46680     6.46680     0.00000
              Tau qt 2.5%       4.62109     4.62109     0.00000
                Tau qt 5%       4.62109     4.62109     0.00000
               Tau qt 95%       6.59375     6.59375     0.00000
             Tau qt 97.5%       6.59375     6.59375     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.12148     1.12148     0.00000
          Theta0 qt 97.5%       1.12148     1.12148     0.00000
                   Theta1    1180.00000  1180.00000     0.00000
           Theta1 qt 2.5%     122.50000   122.50000     0.00000
             Theta1 qt 5%     122.50000   122.50000     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00649     0.00649     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.02105     0.02105     0.00000
       Raggedness p-value       0.00000     0.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 21 (31/01/17 at 16:44:34))
Total computing time for this run : 0h 0m 0s 191 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

