
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_53_50"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 15 (31/01/17 at 14:53:50)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_53_50_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_53_50_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 49
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.47105
     3       150         2   0.10166
     4       150         2   0.06488
     5       150         2   0.28850
     6       150         2   0.45181
     7       150         2   0.18121
     8       150         2   0.05226
     9       150         2   0.50300
    10       150         2   0.25199
    11       150         2   0.01333
    12       150         2   0.48322
    13       150         2   0.07732
    14       150         2   0.07732
    15       150         2   0.50112
    16       150         2   0.41736
    17       150         2   0.03946
    18       150         2   0.26139
    19       150         2   0.45602
    20       150         2   0.06488
-------------------------------------------------
  Mean   150.000     2.000   0.23855
  s.d.     0.000     0.000   0.19069
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0343
Gene diversity           :  0.9722 +/-  0.0041

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 49

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.771096 +/-     2.345212

Average gene diversity over loci              :     0.238555 +/-     0.129796

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    33.150624
  S.D. Theta(Hom) :     5.208908

         Theta(k) :    24.926798
95 % confidence interval limits for theta(k) : [  17.352333,  35.483223 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     4.771096
   S.D. Theta(Pi) :     2.595923

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     311       0.000
       1     517       0.000
       2     770       0.000
       3    1449       0.000
       4    1915       0.000
       5    2147       0.000
       6    1759       0.000
       7    1095       0.000
       8     753       0.000
       9     334       0.000
      10     106       0.000
      11      19       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.771
Mismatch observed variance          :        4.626
Tau                                 :        5.107
Theta0                              :        0.005
Theta1                              :      108.594

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00135867
          P(Sim. Ssd >= Obs. Ssd): 0.20000000
    Harpending's Raggedness index: 0.01427239
        P(Sim. Rag. >= Obs. Rag.): 0.30000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        311.0        204.8        165.6
          1.0        517.0        421.5        425.6
          2.0        770.0        942.8        927.7
          3.0       1449.0       1512.5       1504.6
          4.0       1915.0       1909.0       1884.9
          5.0       2147.0       1964.7       1908.5
          6.0       1759.0       1654.2       1617.3
          7.0       1095.0       1180.9       1177.2
          8.0        753.0        735.2        750.6
          9.0        334.0        384.1        425.7
         10.0        106.0        161.2        217.4
         11.0         19.0         68.1        101.0
         12.0          0.0         24.9         43.0
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           5.107           4.021           6.182           5.063
                   Theta0:           0.005           0.000           0.527           0.092               
                   Theta1:         108.594          37.056       99999.000       30098.281
Mean No. of pairwise diff.           4.771           4.406           5.813           4.890
           No. pol. sites:              20             116             149         132.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        311.0         95.0        350.0        165.6
          1.0        517.0        217.0        625.0        425.6
          2.0        770.0        648.0       1367.0        927.7
          3.0       1449.0        891.0       2104.0       1504.6
          4.0       1915.0       1383.0       2467.0       1884.9
          5.0       2147.0       1741.0       2482.0       1908.5
          6.0       1759.0       1247.0       1922.0       1617.3
          7.0       1095.0        869.0       1643.0       1177.2
          8.0        753.0        349.0       1286.0        750.6
          9.0        334.0         92.0        762.0        425.7
         10.0        106.0          7.0        418.0        217.4
         11.0         19.0          0.0        230.0        101.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           5.107           4.021           6.182           5.063
                   Theta0:           0.005           0.000           0.527           0.092               
                   Theta1:         108.594          37.056       99999.000       30098.281
Mean No. of pairwise diff.           4.771           4.406           5.813           4.890
           No. pol. sites:              20             116             149         132.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        311.0         95.0        350.0        165.6
          1.0        517.0        217.0        625.0        425.6
          2.0        770.0        648.0       1367.0        927.7
          3.0       1449.0        891.0       2104.0       1504.6
          4.0       1915.0       1383.0       2467.0       1884.9
          5.0       2147.0       1741.0       2482.0       1908.5
          6.0       1759.0       1247.0       1922.0       1617.3
          7.0       1095.0        869.0       1643.0       1177.2
          8.0        753.0        349.0       1286.0        750.6
          9.0        334.0         92.0        762.0        425.7
         10.0        106.0          7.0        418.0        217.4
         11.0         19.0          0.0        230.0        101.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           5.107           4.021           6.182           5.063
                   Theta0:           0.005           0.000           0.527           0.092               
                   Theta1:         108.594          37.056       99999.000       30098.281
Mean No. of pairwise diff.           4.771           4.406           5.813           4.890
           No. pol. sites:              20             116             149         132.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        311.0         95.0        350.0        165.6
          1.0        517.0        217.0        625.0        425.6
          2.0        770.0        648.0       1367.0        927.7
          3.0       1449.0        891.0       2104.0       1504.6
          4.0       1915.0       1383.0       2467.0       1884.9
          5.0       2147.0       1741.0       2482.0       1908.5
          6.0       1759.0       1247.0       1922.0       1617.3
          7.0       1095.0        869.0       1643.0       1177.2
          8.0        753.0        349.0       1286.0        750.6
          9.0        334.0         92.0        762.0        425.7
         10.0        106.0          7.0        418.0        217.4
         11.0         19.0          0.0        230.0        101.0
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_53_50_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_53_50_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.47105     0.47105     0.00000     0.47105
     3       0.10166     0.10166     0.00000     0.10166
     4       0.06488     0.06488     0.00000     0.06488
     5       0.28850     0.28850     0.00000     0.28850
     6       0.45181     0.45181     0.00000     0.45181
     7       0.18121     0.18121     0.00000     0.18121
     8       0.05226     0.05226     0.00000     0.05226
     9       0.50300     0.50300     0.00000     0.50300
    10       0.25199     0.25199     0.00000     0.25199
    11       0.01333     0.01333     0.00000     0.01333
    12       0.48322     0.48322     0.00000     0.48322
    13       0.07732     0.07732     0.00000     0.07732
    14       0.07732     0.07732     0.00000     0.07732
    15       0.50112     0.50112     0.00000     0.50112
    16       0.41736     0.41736     0.00000     0.41736
    17       0.03946     0.03946     0.00000     0.03946
    18       0.26139     0.26139     0.00000     0.26139
    19       0.45602     0.45602     0.00000     0.45602
    20       0.06488     0.06488     0.00000     0.06488
--------------------------------------------------------
  Mean       0.23855     0.23855     0.00000     0.23855
  s.d.       0.19069     0.19069     0.00000     0.19069
--------------------------------------------------------
<A NAME="31_01_17at14_53_50_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_53_50_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      24.92680    24.92680     0.00000
            Theta_k_lower      17.35233    17.35233     0.00000
            Theta_k_upper      35.48322    35.48322     0.00000
                  Theta_H      33.15062    33.15062     0.00000
             s.d. Theta_H       5.20891     5.20891     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       4.77110     4.77110     0.00000
            s.d. Theta_pi       2.59592     2.59592     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_53_50_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       5.10742     5.10742     0.00000
              Tau qt 2.5%       4.02148     4.02148     0.00000
                Tau qt 5%       4.02148     4.02148     0.00000
               Tau qt 95%       5.45508     5.45508     0.00000
             Tau qt 97.5%       5.45508     5.45508     0.00000
                   Theta0       0.00527     0.00527     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.31465     0.31465     0.00000
          Theta0 qt 97.5%       0.31465     0.31465     0.00000
                   Theta1     108.59375   108.59375     0.00000
           Theta1 qt 2.5%      37.05566    37.05566     0.00000
             Theta1 qt 5%      37.05566    37.05566     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00136     0.00136     0.00000
      Model (SSD) p-value       0.20000     0.20000     0.00000
         Raggedness index       0.01427     0.01427     0.00000
       Raggedness p-value       0.30000     0.30000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 15 (31/01/17 at 14:53:50))
Total computing time for this run : 0h 0m 0s 160 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

