
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_33_00"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 49 (31/01/17 at 16:33:00)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_33_00_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_33_00_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 61
No. of loci              : 22
No. of usable loci       : 22 loci with less than  5.00 % missing data
No. of polymorphic sites : 22


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.18121
     3       150         2   0.39991
     4       150         2   0.01333
     5       150         2   0.03946
     6       150         2   0.39991
     7       150         2   0.25199
     8       150         2   0.15937
     9       150         2   0.02649
    10       150         2   0.17038
    11       150         2   0.47105
    12       150         2   0.03946
    13       150         2   0.02649
    14       150         2   0.17038
    15       150         2   0.17038
    16       150         2   0.05226
    17       150         2   0.38738
    18       150         2   0.23266
    19       150         2   0.01333
    20       150         2   0.24242
    21       150         2   0.20233
    22       150         2   0.03946
-------------------------------------------------
  Mean   150.000     2.000   0.16832
  s.d.     0.000     0.000   0.14484
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0475
Gene diversity           :  0.9589 +/-  0.0088

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 61

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 22
Number of usable nucleotide sites            : 22


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.702998 +/-     1.881930

Average gene diversity over loci              :     0.168318 +/-     0.094687

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    21.663912
  S.D. Theta(Hom) :     5.148258

         Theta(k) :    37.803161
95 % confidence interval limits for theta(k) : [  26.855382,  52.937022 ]

         Theta(S) :     3.939466
    S.D. Theta(S) :     1.201985

        Theta(Pi) :     3.702998
   S.D. Theta(Pi) :     2.083114

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     459       0.000
       1     583       0.000
       2    1406       0.000
       3    2373       0.000
       4    2793       0.000
       5    2163       0.000
       6    1018       0.000
       7     291       0.000
       8      68       0.000
       9      19       0.000
      10       2       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.703
Mismatch observed variance          :        2.693
Tau                                 :        3.984
Theta0                              :        0.004
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00805817
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.03277587
        P(Sim. Rag. >= Obs. Rag.): 0.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        459.0        199.2        207.3
          1.0        583.0        804.1        826.2
          2.0       1406.0       1659.1       1647.4
          3.0       2373.0       2276.7       2189.8
          4.0       2793.0       2250.3       2183.1
          5.0       2163.0       1751.4       1741.2
          6.0       1018.0       1122.6       1157.3
          7.0        291.0        613.8        659.3
          8.0         68.0        296.6        328.7
          9.0         19.0        123.5        145.6
         10.0          2.0         48.9         58.1
         11.0          0.0         19.0         21.1
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.984           3.422           4.215           3.916
                   Theta0:           0.004           0.000           0.304           0.036               
                   Theta1:       99999.000         152.906       99999.000       80039.734
Mean No. of pairwise diff.           3.703           3.420           4.256           3.942
           No. pol. sites:              22             191             225         212.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        459.0        136.0        300.0        207.3
          1.0        583.0        646.0       1204.0        826.2
          2.0       1406.0       1399.0       2129.0       1647.4
          3.0       2373.0       2048.0       2630.0       2189.8
          4.0       2793.0       2047.0       2553.0       2183.1
          5.0       2163.0       1436.0       1934.0       1741.2
          6.0       1018.0        785.0       1321.0       1157.3
          7.0        291.0        372.0        855.0        659.3
          8.0         68.0        134.0        455.0        328.7
          9.0         19.0         44.0        198.0        145.6
         10.0          2.0          8.0        108.0         58.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.984           3.422           4.215           3.916
                   Theta0:           0.004           0.000           0.304           0.036               
                   Theta1:       99999.000         152.906       99999.000       80039.734
Mean No. of pairwise diff.           3.703           3.420           4.256           3.942
           No. pol. sites:              22             191             225         212.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        459.0        136.0        300.0        207.3
          1.0        583.0        646.0       1204.0        826.2
          2.0       1406.0       1399.0       2129.0       1647.4
          3.0       2373.0       2048.0       2630.0       2189.8
          4.0       2793.0       2047.0       2553.0       2183.1
          5.0       2163.0       1436.0       1934.0       1741.2
          6.0       1018.0        785.0       1321.0       1157.3
          7.0        291.0        372.0        855.0        659.3
          8.0         68.0        134.0        455.0        328.7
          9.0         19.0         44.0        198.0        145.6
         10.0          2.0          8.0        108.0         58.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.984           3.422           4.215           3.916
                   Theta0:           0.004           0.000           0.304           0.036               
                   Theta1:       99999.000         152.906       99999.000       80039.734
Mean No. of pairwise diff.           3.703           3.420           4.256           3.942
           No. pol. sites:              22             191             225         212.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        459.0        136.0        300.0        207.3
          1.0        583.0        646.0       1204.0        826.2
          2.0       1406.0       1399.0       2129.0       1647.4
          3.0       2373.0       2048.0       2630.0       2189.8
          4.0       2793.0       2047.0       2553.0       2183.1
          5.0       2163.0       1436.0       1934.0       1741.2
          6.0       1018.0        785.0       1321.0       1157.3
          7.0        291.0        372.0        855.0        659.3
          8.0         68.0        134.0        455.0        328.7
          9.0         19.0         44.0        198.0        145.6
         10.0          2.0          8.0        108.0         58.1
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_33_00_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            22      22.000       0.000
  No. of usable loci            22      22.000       0.000
  No. of polym. loci            22      22.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_33_00_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.18121     0.18121     0.00000     0.18121
     3       0.39991     0.39991     0.00000     0.39991
     4       0.01333     0.01333     0.00000     0.01333
     5       0.03946     0.03946     0.00000     0.03946
     6       0.39991     0.39991     0.00000     0.39991
     7       0.25199     0.25199     0.00000     0.25199
     8       0.15937     0.15937     0.00000     0.15937
     9       0.02649     0.02649     0.00000     0.02649
    10       0.17038     0.17038     0.00000     0.17038
    11       0.47105     0.47105     0.00000     0.47105
    12       0.03946     0.03946     0.00000     0.03946
    13       0.02649     0.02649     0.00000     0.02649
    14       0.17038     0.17038     0.00000     0.17038
    15       0.17038     0.17038     0.00000     0.17038
    16       0.05226     0.05226     0.00000     0.05226
    17       0.38738     0.38738     0.00000     0.38738
    18       0.23266     0.23266     0.00000     0.23266
    19       0.01333     0.01333     0.00000     0.01333
    20       0.24242     0.24242     0.00000     0.24242
    21       0.20233     0.20233     0.00000     0.20233
    22       0.03946     0.03946     0.00000     0.03946
--------------------------------------------------------
  Mean       0.16832     0.16832     0.00000     0.16832
  s.d.       0.14484     0.14484     0.00000     0.14484
--------------------------------------------------------
<A NAME="31_01_17at16_33_00_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_33_00_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      37.80316    37.80316     0.00000
            Theta_k_lower      26.85538    26.85538     0.00000
            Theta_k_upper      52.93702    52.93702     0.00000
                  Theta_H      21.66391    21.66391     0.00000
             s.d. Theta_H       5.14826     5.14826     0.00000
                  Theta_S       3.93947     3.93947     0.00000
             s.d. Theta_S       1.20198     1.20198     0.00000
                 Theta_pi       3.70300     3.70300     0.00000
            s.d. Theta_pi       2.08311     2.08311     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_33_00_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.98438     3.98438     0.00000
              Tau qt 2.5%       3.42188     3.42188     0.00000
                Tau qt 5%       3.42188     3.42188     0.00000
               Tau qt 95%       4.20898     4.20898     0.00000
             Tau qt 97.5%       4.20898     4.20898     0.00000
                   Theta0       0.00352     0.00352     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.03340     0.03340     0.00000
          Theta0 qt 97.5%       0.03340     0.03340     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%     152.90625   152.90625     0.00000
             Theta1 qt 5%     152.90625   152.90625     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00806     0.00806     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.03278     0.03278     0.00000
       Raggedness p-value       0.00000     0.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 49 (31/01/17 at 16:33:00))
Total computing time for this run : 0h 0m 0s 160 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

