
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_55_09"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 369 (31/01/17 at 14:55:09)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_55_09_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_55_09_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 57
No. of loci              : 22
No. of usable loci       : 22 loci with less than  5.00 % missing data
No. of polymorphic sites : 22


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.27964
     2       150         2   0.19186
     3       150         2   0.07732
     4       150         2   0.08957
     5       150         2   0.50013
     6       150         2   0.25199
     7       150         2   0.01333
     8       150         2   0.12528
     9       150         2   0.24242
    10       150         2   0.33790
    11       150         2   0.18121
    12       150         2   0.08957
    13       150         2   0.30568
    14       150         2   0.32215
    15       150         2   0.03946
    16       150         2   0.01333
    17       150         2   0.30568
    18       150         2   0.45602
    19       150         2   0.22273
    20       150         2   0.08957
    21       150         2   0.50255
    22       150         2   0.27060
-------------------------------------------------
  Mean   150.000     2.000   0.22309
  s.d.     0.000     0.000   0.14819
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0308
Gene diversity           :  0.9757 +/-  0.0038

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 57

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 22
Number of usable nucleotide sites            : 22


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.908009 +/-     2.404449

Average gene diversity over loci              :     0.223091 +/-     0.120977

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    38.275970
  S.D. Theta(Hom) :     6.311694

         Theta(k) :    33.066749
95 % confidence interval limits for theta(k) : [  23.361695,  46.504216 ]

         Theta(S) :     3.939466
    S.D. Theta(S) :     1.201985

        Theta(Pi) :     4.908009
   S.D. Theta(Pi) :     2.661493

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     272       0.000
       1     455       0.000
       2     659       0.000
       3    1076       0.000
       4    1814       0.000
       5    2425       0.000
       6    2051       0.000
       7    1465       0.000
       8     738       0.000
       9     200       0.000
      10      20       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.908
Mismatch observed variance          :        3.963
Tau                                 :        5.340
Theta0                              :        0.000
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00577803
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.02002716
        P(Sim. Rag. >= Obs. Rag.): 0.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        272.0         54.1         53.7
          1.0        455.0        260.6        286.3
          2.0        659.0        737.2        764.3
          3.0       1076.0       1352.2       1360.3
          4.0       1814.0       1828.7       1815.8
          5.0       2425.0       1990.0       1939.2
          6.0       2051.0       1772.5       1725.9
          7.0       1465.0       1349.5       1316.5
          8.0        738.0        857.6        878.8
          9.0        200.0        491.8        521.4
         10.0         20.0        261.2        278.4
         11.0          0.0        119.8        135.2
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           5.340           5.162           5.588           5.348
                   Theta0:           0.000           0.000           0.102           0.013               
                   Theta1:       99999.000          71.578       99999.000       70048.180
Mean No. of pairwise diff.           4.908           5.077           5.610           5.335
           No. pol. sites:              22             239             267         251.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        272.0          6.0        105.0         53.7
          1.0        455.0        120.0        375.0        286.3
          2.0        659.0        577.0        974.0        764.3
          3.0       1076.0       1138.0       1572.0       1360.3
          4.0       1814.0       1660.0       1991.0       1815.8
          5.0       2425.0       1673.0       2211.0       1939.2
          6.0       2051.0       1394.0       1986.0       1725.9
          7.0       1465.0       1135.0       1592.0       1316.5
          8.0        738.0        702.0        976.0        878.8
          9.0        200.0        324.0        615.0        521.4
         10.0         20.0        134.0        366.0        278.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           5.340           5.162           5.588           5.348
                   Theta0:           0.000           0.000           0.102           0.013               
                   Theta1:       99999.000          71.578       99999.000       70048.180
Mean No. of pairwise diff.           4.908           5.077           5.610           5.335
           No. pol. sites:              22             239             267         251.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        272.0          6.0        105.0         53.7
          1.0        455.0        120.0        375.0        286.3
          2.0        659.0        577.0        974.0        764.3
          3.0       1076.0       1138.0       1572.0       1360.3
          4.0       1814.0       1660.0       1991.0       1815.8
          5.0       2425.0       1673.0       2211.0       1939.2
          6.0       2051.0       1394.0       1986.0       1725.9
          7.0       1465.0       1135.0       1592.0       1316.5
          8.0        738.0        702.0        976.0        878.8
          9.0        200.0        324.0        615.0        521.4
         10.0         20.0        134.0        366.0        278.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           5.340           5.162           5.588           5.348
                   Theta0:           0.000           0.000           0.102           0.013               
                   Theta1:       99999.000          71.578       99999.000       70048.180
Mean No. of pairwise diff.           4.908           5.077           5.610           5.335
           No. pol. sites:              22             239             267         251.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        272.0          6.0        105.0         53.7
          1.0        455.0        120.0        375.0        286.3
          2.0        659.0        577.0        974.0        764.3
          3.0       1076.0       1138.0       1572.0       1360.3
          4.0       1814.0       1660.0       1991.0       1815.8
          5.0       2425.0       1673.0       2211.0       1939.2
          6.0       2051.0       1394.0       1986.0       1725.9
          7.0       1465.0       1135.0       1592.0       1316.5
          8.0        738.0        702.0        976.0        878.8
          9.0        200.0        324.0        615.0        521.4
         10.0         20.0        134.0        366.0        278.4
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_55_09_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            22      22.000       0.000
  No. of usable loci            22      22.000       0.000
  No. of polym. loci            22      22.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_55_09_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.27964     0.27964     0.00000     0.27964
     2       0.19186     0.19186     0.00000     0.19186
     3       0.07732     0.07732     0.00000     0.07732
     4       0.08957     0.08957     0.00000     0.08957
     5       0.50013     0.50013     0.00000     0.50013
     6       0.25199     0.25199     0.00000     0.25199
     7       0.01333     0.01333     0.00000     0.01333
     8       0.12528     0.12528     0.00000     0.12528
     9       0.24242     0.24242     0.00000     0.24242
    10       0.33790     0.33790     0.00000     0.33790
    11       0.18121     0.18121     0.00000     0.18121
    12       0.08957     0.08957     0.00000     0.08957
    13       0.30568     0.30568     0.00000     0.30568
    14       0.32215     0.32215     0.00000     0.32215
    15       0.03946     0.03946     0.00000     0.03946
    16       0.01333     0.01333     0.00000     0.01333
    17       0.30568     0.30568     0.00000     0.30568
    18       0.45602     0.45602     0.00000     0.45602
    19       0.22273     0.22273     0.00000     0.22273
    20       0.08957     0.08957     0.00000     0.08957
    21       0.50255     0.50255     0.00000     0.50255
    22       0.27060     0.27060     0.00000     0.27060
--------------------------------------------------------
  Mean       0.22309     0.22309     0.00000     0.22309
  s.d.       0.14819     0.14819     0.00000     0.14819
--------------------------------------------------------
<A NAME="31_01_17at14_55_09_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_55_09_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      33.06675    33.06675     0.00000
            Theta_k_lower      23.36170    23.36170     0.00000
            Theta_k_upper      46.50422    46.50422     0.00000
                  Theta_H      38.27597    38.27597     0.00000
             s.d. Theta_H       6.31169     6.31169     0.00000
                  Theta_S       3.93947     3.93947     0.00000
             s.d. Theta_S       1.20198     1.20198     0.00000
                 Theta_pi       4.90801     4.90801     0.00000
            s.d. Theta_pi       2.66149     2.66149     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_55_09_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       5.33984     5.33984     0.00000
              Tau qt 2.5%       5.16211     5.16211     0.00000
                Tau qt 5%       5.16211     5.16211     0.00000
               Tau qt 95%       5.54102     5.54102     0.00000
             Tau qt 97.5%       5.54102     5.54102     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.02109     0.02109     0.00000
          Theta0 qt 97.5%       0.02109     0.02109     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      71.57812    71.57812     0.00000
             Theta1 qt 5%      71.57812    71.57812     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00578     0.00578     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.02003     0.02003     0.00000
       Raggedness p-value       0.00000     0.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 369 (31/01/17 at 14:55:09))
Total computing time for this run : 0h 0m 0s 140 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

