
<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_35_56"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 111 (31/01/17 at 16:35:56)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_35_56_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_35_56_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 73
No. of loci              : 23
No. of usable loci       : 23 loci with less than  5.00 % missing data
No. of polymorphic sites : 23


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.07732
     2       150         2   0.38085
     3       150         2   0.18121
     4       150         2   0.17038
     5       150         2   0.35293
     6       150         2   0.03946
     7       150         2   0.03946
     8       150         2   0.08957
     9       150         2   0.24242
    10       150         2   0.11356
    11       150         2   0.05226
    12       150         2   0.39991
    13       150         2   0.44743
    14       150         2   0.10166
    15       150         2   0.10166
    16       150         2   0.40591
    17       150         2   0.19186
    18       150         2   0.02649
    19       150         2   0.01333
    20       150         2   0.36018
    21       150         2   0.37414
    22       150         2   0.10166
    23       150         2   0.03946
-------------------------------------------------
  Mean   150.000     2.000   0.18709
  s.d.     0.000     0.000   0.14813
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0231
Gene diversity           :  0.9834 +/-  0.0035

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 73

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 23
Number of usable nucleotide sites            : 23


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.303087 +/-     2.142498

Average gene diversity over loci              :     0.187091 +/-     0.103110

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    57.536447
  S.D. Theta(Hom) :    12.702162

         Theta(k) :    55.466488
95 % confidence interval limits for theta(k) : [  39.815434,  77.123116 ]

         Theta(S) :     4.118532
    S.D. Theta(S) :     1.243883

        Theta(Pi) :     4.303087
   S.D. Theta(Pi) :     2.371538

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     185       0.000
       1     554       0.000
       2    1259       0.000
       3    2034       0.000
       4    2168       0.000
       5    1997       0.000
       6    1494       0.000
       7     845       0.000
       8     411       0.000
       9     191       0.000
      10      37       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.303
Mismatch observed variance          :        3.781
Tau                                 :        4.408
Theta0                              :        0.009
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00042485
          P(Sim. Ssd >= Obs. Ssd): 0.50000000
    Harpending's Raggedness index: 0.01775343
        P(Sim. Rag. >= Obs. Rag.): 0.60000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        185.0        168.8        135.0
          1.0        554.0        716.1        596.0
          2.0       1259.0       1549.1       1316.0
          3.0       2034.0       2165.8       1937.4
          4.0       2168.0       2185.5       2139.3
          5.0       1997.0       1759.4       1889.7
          6.0       1494.0       1205.5       1391.1
          7.0        845.0        726.0        877.7
          8.0        411.0        397.9        484.6
          9.0        191.0        183.9        237.8
         10.0         37.0         78.8        105.0
         11.0          0.0         28.4         42.2
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.408           3.115           4.508           3.931
                   Theta0:           0.009           0.000           0.631           0.070               
                   Theta1:       99999.000         129.781       99999.000       80027.562
Mean No. of pairwise diff.           4.303           3.830           4.481           4.136
           No. pol. sites:              23             203             230         214.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        185.0         78.0        300.0        135.0
          1.0        554.0        513.0        975.0        596.0
          2.0       1259.0       1170.0       1823.0       1316.0
          3.0       2034.0       1894.0       2545.0       1937.4
          4.0       2168.0       2020.0       2535.0       2139.3
          5.0       1997.0       1530.0       2052.0       1889.7
          6.0       1494.0        985.0       1443.0       1391.1
          7.0        845.0        486.0        878.0        877.7
          8.0        411.0        228.0        521.0        484.6
          9.0        191.0        128.0        246.0        237.8
         10.0         37.0         48.0        122.0        105.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.408           3.115           4.508           3.931
                   Theta0:           0.009           0.000           0.631           0.070               
                   Theta1:       99999.000         129.781       99999.000       80027.562
Mean No. of pairwise diff.           4.303           3.830           4.481           4.136
           No. pol. sites:              23             203             230         214.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        185.0         78.0        300.0        135.0
          1.0        554.0        513.0        975.0        596.0
          2.0       1259.0       1170.0       1823.0       1316.0
          3.0       2034.0       1894.0       2545.0       1937.4
          4.0       2168.0       2020.0       2535.0       2139.3
          5.0       1997.0       1530.0       2052.0       1889.7
          6.0       1494.0        985.0       1443.0       1391.1
          7.0        845.0        486.0        878.0        877.7
          8.0        411.0        228.0        521.0        484.6
          9.0        191.0        128.0        246.0        237.8
         10.0         37.0         48.0        122.0        105.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.408           3.115           4.508           3.931
                   Theta0:           0.009           0.000           0.631           0.070               
                   Theta1:       99999.000         129.781       99999.000       80027.562
Mean No. of pairwise diff.           4.303           3.830           4.481           4.136
           No. pol. sites:              23             203             230         214.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        185.0         78.0        300.0        135.0
          1.0        554.0        513.0        975.0        596.0
          2.0       1259.0       1170.0       1823.0       1316.0
          3.0       2034.0       1894.0       2545.0       1937.4
          4.0       2168.0       2020.0       2535.0       2139.3
          5.0       1997.0       1530.0       2052.0       1889.7
          6.0       1494.0        985.0       1443.0       1391.1
          7.0        845.0        486.0        878.0        877.7
          8.0        411.0        228.0        521.0        484.6
          9.0        191.0        128.0        246.0        237.8
         10.0         37.0         48.0        122.0        105.0
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_35_56_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            23      23.000       0.000
  No. of usable loci            23      23.000       0.000
  No. of polym. loci            23      23.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_35_56_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.07732     0.07732     0.00000     0.07732
     2       0.38085     0.38085     0.00000     0.38085
     3       0.18121     0.18121     0.00000     0.18121
     4       0.17038     0.17038     0.00000     0.17038
     5       0.35293     0.35293     0.00000     0.35293
     6       0.03946     0.03946     0.00000     0.03946
     7       0.03946     0.03946     0.00000     0.03946
     8       0.08957     0.08957     0.00000     0.08957
     9       0.24242     0.24242     0.00000     0.24242
    10       0.11356     0.11356     0.00000     0.11356
    11       0.05226     0.05226     0.00000     0.05226
    12       0.39991     0.39991     0.00000     0.39991
    13       0.44743     0.44743     0.00000     0.44743
    14       0.10166     0.10166     0.00000     0.10166
    15       0.10166     0.10166     0.00000     0.10166
    16       0.40591     0.40591     0.00000     0.40591
    17       0.19186     0.19186     0.00000     0.19186
    18       0.02649     0.02649     0.00000     0.02649
    19       0.01333     0.01333     0.00000     0.01333
    20       0.36018     0.36018     0.00000     0.36018
    21       0.37414     0.37414     0.00000     0.37414
    22       0.10166     0.10166     0.00000     0.10166
    23       0.03946     0.03946     0.00000     0.03946
--------------------------------------------------------
  Mean       0.18709     0.18709     0.00000     0.18709
  s.d.       0.14813     0.14813     0.00000     0.14813
--------------------------------------------------------
<A NAME="31_01_17at16_35_56_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_35_56_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      55.46649    55.46649     0.00000
            Theta_k_lower      39.81543    39.81543     0.00000
            Theta_k_upper      77.12312    77.12312     0.00000
                  Theta_H      57.53645    57.53645     0.00000
             s.d. Theta_H      12.70216    12.70216     0.00000
                  Theta_S       4.11853     4.11853     0.00000
             s.d. Theta_S       1.24388     1.24388     0.00000
                 Theta_pi       4.30309     4.30309     0.00000
            s.d. Theta_pi       2.37154     2.37154     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_35_56_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.40820     4.40820     0.00000
              Tau qt 2.5%       3.11523     3.11523     0.00000
                Tau qt 5%       3.11523     3.11523     0.00000
               Tau qt 95%       4.28320     4.28320     0.00000
             Tau qt 97.5%       4.28320     4.28320     0.00000
                   Theta0       0.00859     0.00859     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.04395     0.04395     0.00000
          Theta0 qt 97.5%       0.04395     0.04395     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%     129.78125   129.78125     0.00000
             Theta1 qt 5%     129.78125   129.78125     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00042     0.00042     0.00000
      Model (SSD) p-value       0.50000     0.50000     0.00000
         Raggedness index       0.01775     0.01775     0.00000
       Raggedness p-value       0.60000     0.60000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 111 (31/01/17 at 16:35:56))
Total computing time for this run : 0h 0m 0s 127 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

