
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_57_25"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 273 (31/01/17 at 14:57:25)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_57_25_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_57_25_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 44
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.06488
     2       150         2   0.26139
     3       150         2   0.02649
     4       150         2   0.48322
     5       150         2   0.23266
     6       150         2   0.03946
     7       150         2   0.12528
     8       150         2   0.48045
     9       150         2   0.10166
    10       150         2   0.10166
    11       150         2   0.28850
    12       150         2   0.33790
    13       150         2   0.03946
    14       150         2   0.10166
    15       150         2   0.33790
    16       150         2   0.06488
    17       150         2   0.07732
    18       150         2   0.30568
    19       150         2   0.18121
    20       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.18325
  s.d.     0.000     0.000   0.14835
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0950
Gene diversity           :  0.9111 +/-  0.0182

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 44

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.664966 +/-     1.865385

Average gene diversity over loci              :     0.183248 +/-     0.103240

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     8.884670
  S.D. Theta(Hom) :     2.187097

         Theta(k) :    20.605975
95 % confidence interval limits for theta(k) : [  14.167681,  29.640070 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     3.664966
   S.D. Theta(Pi) :     2.064801

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     994     382.419
       1    1054    1220.010
       2    1417    1992.233
       3    1670    2231.886
       4    1938    1942.084
       5    1897    1410.614
       6    1106     898.401
       7     713     520.592
       8     300     282.506
       9      66     146.823
      10      20      74.307
      11       0      37.043
      12       0      18.321
      13       0       9.027
      14       0       4.441
      15       0       2.183
      16       0       1.073
      17       0       0.527
      18       0       0.259
      19       0       0.127
      20       0       0.063


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.665
Mismatch observed variance          :        4.598
Tau                                 :        4.535
Theta0                              :        0.002
Theta1                              :       13.633

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00156424
          P(Sim. Ssd >= Obs. Ssd): 0.70000000
    Harpending's Raggedness index: 0.01025663
        P(Sim. Rag. >= Obs. Rag.): 0.80000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        994.0        837.7        843.6
          1.0       1054.0       1181.9       1068.6
          2.0       1417.0       1665.1       1455.5
          3.0       1670.0       1777.5       1777.3
          4.0       1938.0       1778.7       1822.8
          5.0       1897.0       1520.5       1567.6
          6.0       1106.0        984.5       1148.3
          7.0        713.0        723.0        729.3
          8.0        300.0        385.0        407.9
          9.0         66.0        150.9        203.5
         10.0         20.0        117.0         91.7
         11.0          0.0         39.7         37.6
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.535           1.695           7.439           4.254
                   Theta0:           0.002           0.000           0.986           0.189               
                   Theta1:          13.633           9.277       99999.000       10020.758
Mean No. of pairwise diff.           3.665           2.483           5.069           3.735
           No. pol. sites:              20              31              51          41.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        994.0        485.0       1228.0        843.6
          1.0       1054.0        675.0       2455.0       1068.6
          2.0       1417.0        962.0       2637.0       1455.5
          3.0       1670.0        850.0       3076.0       1777.3
          4.0       1938.0        738.0       2343.0       1822.8
          5.0       1897.0        723.0       2468.0       1567.6
          6.0       1106.0        261.0       2079.0       1148.3
          7.0        713.0         37.0       1662.0        729.3
          8.0        300.0          3.0        929.0        407.9
          9.0         66.0          0.0        458.0        203.5
         10.0         20.0          0.0        704.0         91.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.535           1.695           7.439           4.254
                   Theta0:           0.002           0.000           0.986           0.189               
                   Theta1:          13.633           9.277       99999.000       10020.758
Mean No. of pairwise diff.           3.665           2.483           5.069           3.735
           No. pol. sites:              20              31              51          41.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        994.0        485.0       1228.0        843.6
          1.0       1054.0        675.0       2455.0       1068.6
          2.0       1417.0        962.0       2637.0       1455.5
          3.0       1670.0        850.0       3076.0       1777.3
          4.0       1938.0        738.0       2343.0       1822.8
          5.0       1897.0        723.0       2468.0       1567.6
          6.0       1106.0        261.0       2079.0       1148.3
          7.0        713.0         37.0       1662.0        729.3
          8.0        300.0          3.0        929.0        407.9
          9.0         66.0          0.0        458.0        203.5
         10.0         20.0          0.0        704.0         91.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.535           1.695           7.439           4.254
                   Theta0:           0.002           0.000           0.986           0.189               
                   Theta1:          13.633           9.277       99999.000       10020.758
Mean No. of pairwise diff.           3.665           2.483           5.069           3.735
           No. pol. sites:              20              31              51          41.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        994.0        485.0       1228.0        843.6
          1.0       1054.0        675.0       2455.0       1068.6
          2.0       1417.0        962.0       2637.0       1455.5
          3.0       1670.0        850.0       3076.0       1777.3
          4.0       1938.0        738.0       2343.0       1822.8
          5.0       1897.0        723.0       2468.0       1567.6
          6.0       1106.0        261.0       2079.0       1148.3
          7.0        713.0         37.0       1662.0        729.3
          8.0        300.0          3.0        929.0        407.9
          9.0         66.0          0.0        458.0        203.5
         10.0         20.0          0.0        704.0         91.7
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_57_25_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_57_25_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.06488     0.06488     0.00000     0.06488
     2       0.26139     0.26139     0.00000     0.26139
     3       0.02649     0.02649     0.00000     0.02649
     4       0.48322     0.48322     0.00000     0.48322
     5       0.23266     0.23266     0.00000     0.23266
     6       0.03946     0.03946     0.00000     0.03946
     7       0.12528     0.12528     0.00000     0.12528
     8       0.48045     0.48045     0.00000     0.48045
     9       0.10166     0.10166     0.00000     0.10166
    10       0.10166     0.10166     0.00000     0.10166
    11       0.28850     0.28850     0.00000     0.28850
    12       0.33790     0.33790     0.00000     0.33790
    13       0.03946     0.03946     0.00000     0.03946
    14       0.10166     0.10166     0.00000     0.10166
    15       0.33790     0.33790     0.00000     0.33790
    16       0.06488     0.06488     0.00000     0.06488
    17       0.07732     0.07732     0.00000     0.07732
    18       0.30568     0.30568     0.00000     0.30568
    19       0.18121     0.18121     0.00000     0.18121
    20       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.18325     0.18325     0.00000     0.18325
  s.d.       0.14835     0.14835     0.00000     0.14835
--------------------------------------------------------
<A NAME="31_01_17at14_57_25_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_57_25_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      20.60597    20.60597     0.00000
            Theta_k_lower      14.16768    14.16768     0.00000
            Theta_k_upper      29.64007    29.64007     0.00000
                  Theta_H       8.88467     8.88467     0.00000
             s.d. Theta_H       2.18710     2.18710     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       3.66497     3.66497     0.00000
            s.d. Theta_pi       2.06480     2.06480     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_57_25_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.53516     4.53516     0.00000
              Tau qt 2.5%       1.69531     1.69531     0.00000
                Tau qt 5%       1.69531     1.69531     0.00000
               Tau qt 95%       5.13672     5.13672     0.00000
             Tau qt 97.5%       5.13672     5.13672     0.00000
                   Theta0       0.00176     0.00176     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.82793     0.82793     0.00000
          Theta0 qt 97.5%       0.82793     0.82793     0.00000
                   Theta1      13.63281    13.63281     0.00000
           Theta1 qt 2.5%       9.27734     9.27734     0.00000
             Theta1 qt 5%       9.27734     9.27734     0.00000
            Theta1 qt 95%      42.53906    42.53906     0.00000
          Theta1 qt 97.5%      42.53906    42.53906     0.00000
                      SSD       0.00156     0.00156     0.00000
      Model (SSD) p-value       0.70000     0.70000     0.00000
         Raggedness index       0.01026     0.01026     0.00000
       Raggedness p-value       0.80000     0.80000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 273 (31/01/17 at 14:57:25))
Total computing time for this run : 0h 0m 0s 121 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

