
<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_35_57"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 119 (31/01/17 at 16:35:57)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_35_57_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_35_57_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 58
No. of loci              : 23
No. of usable loci       : 23 loci with less than  5.00 % missing data
No. of polymorphic sites : 23


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.08957
     3       150         2   0.48045
     4       150         2   0.12528
     5       150         2   0.35293
     6       150         2   0.33790
     7       150         2   0.08957
     8       150         2   0.02649
     9       150         2   0.26139
    10       150         2   0.23266
    11       150         2   0.05226
    12       150         2   0.28850
    13       150         2   0.13682
    14       150         2   0.06488
    15       150         2   0.05226
    16       150         2   0.11356
    17       150         2   0.45181
    18       150         2   0.17038
    19       150         2   0.02649
    20       150         2   0.19186
    21       150         2   0.50112
    22       150         2   0.13682
    23       150         2   0.02649
-------------------------------------------------
  Mean   150.000     2.000   0.18360
  s.d.     0.000     0.000   0.15301
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0316
Gene diversity           :  0.9749 +/-  0.0045

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 58

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 23
Number of usable nucleotide sites            : 23


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.222819 +/-     2.107691

Average gene diversity over loci              :     0.183601 +/-     0.101435

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    37.107627
  S.D. Theta(Hom) :     7.057695

         Theta(k) :    34.205353
95 % confidence interval limits for theta(k) : [  24.201985,  48.048972 ]

         Theta(S) :     4.118532
    S.D. Theta(S) :     1.243883

        Theta(Pi) :     4.222819
   S.D. Theta(Pi) :     2.333010

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     280       0.000
       1     589       0.000
       2    1266       0.000
       3    1810       0.000
       4    2215       0.000
       5    2200       0.000
       6    1523       0.000
       7     837       0.000
       8     330       0.000
       9     108       0.000
      10      17       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.223
Mismatch observed variance          :        3.626
Tau                                 :        4.455
Theta0                              :        0.007
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00163930
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.01807986
        P(Sim. Rag. >= Obs. Rag.): 0.30000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        280.0        124.7        129.0
          1.0        589.0        571.7        575.4
          2.0       1266.0       1264.8       1283.6
          3.0       1810.0       1889.0       1909.2
          4.0       2215.0       2141.5       2129.7
          5.0       2200.0       1926.7       1900.6
          6.0       1523.0       1425.4       1413.4
          7.0        837.0        898.7        901.0
          8.0        330.0        509.2        502.5
          9.0        108.0        246.3        249.2
         10.0         17.0        104.9        111.2
         11.0          0.0         46.6         45.1
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.455           4.199           4.814           4.492
                   Theta0:           0.007           0.000           0.021           0.006               
                   Theta1:       99999.000          52.359       99999.000       60143.086
Mean No. of pairwise diff.           4.223           4.136           4.840           4.473
           No. pol. sites:              23             217             249         227.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        280.0         45.0        210.0        129.0
          1.0        589.0        362.0        744.0        575.4
          2.0       1266.0       1045.0       1446.0       1283.6
          3.0       1810.0       1617.0       2152.0       1909.2
          4.0       2215.0       1903.0       2415.0       2129.7
          5.0       2200.0       1786.0       2088.0       1900.6
          6.0       1523.0       1275.0       1645.0       1413.4
          7.0        837.0        678.0       1033.0        901.0
          8.0        330.0        335.0        656.0        502.5
          9.0        108.0        104.0        380.0        249.2
         10.0         17.0         31.0        199.0        111.2
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.455           4.199           4.814           4.492
                   Theta0:           0.007           0.000           0.021           0.006               
                   Theta1:       99999.000          52.359       99999.000       60143.086
Mean No. of pairwise diff.           4.223           4.136           4.840           4.473
           No. pol. sites:              23             217             249         227.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        280.0         45.0        210.0        129.0
          1.0        589.0        362.0        744.0        575.4
          2.0       1266.0       1045.0       1446.0       1283.6
          3.0       1810.0       1617.0       2152.0       1909.2
          4.0       2215.0       1903.0       2415.0       2129.7
          5.0       2200.0       1786.0       2088.0       1900.6
          6.0       1523.0       1275.0       1645.0       1413.4
          7.0        837.0        678.0       1033.0        901.0
          8.0        330.0        335.0        656.0        502.5
          9.0        108.0        104.0        380.0        249.2
         10.0         17.0         31.0        199.0        111.2
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.455           4.199           4.814           4.492
                   Theta0:           0.007           0.000           0.021           0.006               
                   Theta1:       99999.000          52.359       99999.000       60143.086
Mean No. of pairwise diff.           4.223           4.136           4.840           4.473
           No. pol. sites:              23             217             249         227.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        280.0         45.0        210.0        129.0
          1.0        589.0        362.0        744.0        575.4
          2.0       1266.0       1045.0       1446.0       1283.6
          3.0       1810.0       1617.0       2152.0       1909.2
          4.0       2215.0       1903.0       2415.0       2129.7
          5.0       2200.0       1786.0       2088.0       1900.6
          6.0       1523.0       1275.0       1645.0       1413.4
          7.0        837.0        678.0       1033.0        901.0
          8.0        330.0        335.0        656.0        502.5
          9.0        108.0        104.0        380.0        249.2
         10.0         17.0         31.0        199.0        111.2
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_35_57_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            23      23.000       0.000
  No. of usable loci            23      23.000       0.000
  No. of polym. loci            23      23.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_35_57_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.08957     0.08957     0.00000     0.08957
     3       0.48045     0.48045     0.00000     0.48045
     4       0.12528     0.12528     0.00000     0.12528
     5       0.35293     0.35293     0.00000     0.35293
     6       0.33790     0.33790     0.00000     0.33790
     7       0.08957     0.08957     0.00000     0.08957
     8       0.02649     0.02649     0.00000     0.02649
     9       0.26139     0.26139     0.00000     0.26139
    10       0.23266     0.23266     0.00000     0.23266
    11       0.05226     0.05226     0.00000     0.05226
    12       0.28850     0.28850     0.00000     0.28850
    13       0.13682     0.13682     0.00000     0.13682
    14       0.06488     0.06488     0.00000     0.06488
    15       0.05226     0.05226     0.00000     0.05226
    16       0.11356     0.11356     0.00000     0.11356
    17       0.45181     0.45181     0.00000     0.45181
    18       0.17038     0.17038     0.00000     0.17038
    19       0.02649     0.02649     0.00000     0.02649
    20       0.19186     0.19186     0.00000     0.19186
    21       0.50112     0.50112     0.00000     0.50112
    22       0.13682     0.13682     0.00000     0.13682
    23       0.02649     0.02649     0.00000     0.02649
--------------------------------------------------------
  Mean       0.18360     0.18360     0.00000     0.18360
  s.d.       0.15301     0.15301     0.00000     0.15301
--------------------------------------------------------
<A NAME="31_01_17at16_35_57_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_35_57_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      34.20535    34.20535     0.00000
            Theta_k_lower      24.20198    24.20198     0.00000
            Theta_k_upper      48.04897    48.04897     0.00000
                  Theta_H      37.10763    37.10763     0.00000
             s.d. Theta_H       7.05769     7.05769     0.00000
                  Theta_S       4.11853     4.11853     0.00000
             s.d. Theta_S       1.24388     1.24388     0.00000
                 Theta_pi       4.22282     4.22282     0.00000
            s.d. Theta_pi       2.33301     2.33301     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_35_57_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.45508     4.45508     0.00000
              Tau qt 2.5%       4.19922     4.19922     0.00000
                Tau qt 5%       4.19922     4.19922     0.00000
               Tau qt 95%       4.72070     4.72070     0.00000
             Tau qt 97.5%       4.72070     4.72070     0.00000
                   Theta0       0.00703     0.00703     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.01934     0.01934     0.00000
          Theta0 qt 97.5%       0.01934     0.01934     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      52.35938    52.35938     0.00000
             Theta1 qt 5%      52.35938    52.35938     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00164     0.00164     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.01808     0.01808     0.00000
       Raggedness p-value       0.30000     0.30000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 119 (31/01/17 at 16:35:57))
Total computing time for this run : 0h 0m 0s 138 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

