
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_01"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 104 (31/01/17 at 14:44:01)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_01_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_01_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 41
No. of loci              : 61
No. of usable loci       : 61 loci with less than  5.00 % missing data
No. of polymorphic sites : 61


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.02649
     2       150         2   0.01333
     3       150         2   0.10166
     4       150         2   0.18121
     5       150         2   0.01333
     6       150         2   0.01333
     7       150         2   0.01333
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.01333
    11       150         2   0.10166
    12       150         2   0.01333
    13       150         2   0.07732
    14       150         2   0.02649
    15       150         2   0.03946
    16       150         2   0.02649
    17       150         2   0.07732
    18       150         2   0.01333
    19       150         2   0.06488
    20       150         2   0.07732
    21       150         2   0.03946
    22       150         2   0.36018
    23       150         2   0.07732
    24       150         2   0.01333
    25       150         2   0.03946
    26       150         2   0.01333
    27       150         2   0.01333
    28       150         2   0.06488
    29       150         2   0.01333
    30       150         2   0.01333
    31       150         2   0.01333
    32       150         2   0.07732
    33       150         2   0.06488
    34       150         2   0.01333
    35       150         2   0.01333
    36       150         2   0.36018
    37       150         2   0.02649
    38       150         2   0.05226
    39       150         2   0.05226
    40       150         2   0.06488
    41       150         2   0.03946
    42       150         2   0.06488
    43       150         2   0.07732
    44       150         2   0.01333
    45       150         2   0.02649
    46       150         2   0.03946
    47       150         2   0.18121
    48       150         2   0.03946
    49       150         2   0.05226
    50       150         2   0.03946
    51       150         2   0.01333
    52       150         2   0.01333
    53       150         2   0.01333
    54       150         2   0.05226
    55       150         2   0.01333
    56       150         2   0.01333
    57       150         2   0.02649
    58       150         2   0.07732
    59       150         2   0.01333
    60       150         2   0.02649
    61       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.05173
  s.d.     0.000     0.000   0.06790
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.1238
Gene diversity           :  0.8821 +/-  0.0216

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 41

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 61
Number of usable nucleotide sites            : 61


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.155705 +/-     1.643387

Average gene diversity over loci              :     0.051733 +/-     0.029821

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     6.288111
  S.D. Theta(Hom) :     1.434522

         Theta(k) :    18.258484
95 % confidence interval limits for theta(k) : [  12.442238,  26.463077 ]

         Theta(S) :    10.923063
    S.D. Theta(S) :     2.795188

        Theta(Pi) :     3.155705
   S.D. Theta(Pi) :     1.819070

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1318     637.295
       1     983    1707.179
       2    2315    2363.764
       3    1858    2273.826
       4    1848    1726.124
       5    1382    1114.992
       6     791     645.492
       7     461     347.899
       8     173     179.477
       9      41      90.336
      10       5      44.897
      11       0      22.183
      12       0      10.933
      13       0       5.383
      14       0       2.650
      15       0       1.304
      16       0       0.642
      17       0       0.316
      18       0       0.155
      19       0       0.077
      20       0       0.038


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.156
Mismatch observed variance          :        4.095
Tau                                 :        3.814
Theta0                              :        0.005
Theta1                              :       10.885

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00401774
          P(Sim. Ssd >= Obs. Ssd): 0.50000000
    Harpending's Raggedness index: 0.02300136
        P(Sim. Rag. >= Obs. Rag.): 0.90000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1318.0       1041.0       1098.3
          1.0        983.0       1663.1       1451.6
          2.0       2315.0       2166.0       1878.8
          3.0       1858.0       2368.1       2035.7
          4.0       1848.0       1860.1       1798.2
          5.0       1382.0       1130.4       1315.9
          6.0        791.0        554.5        816.0
          7.0        461.0        227.6        437.6
          8.0        173.0         98.5        206.4
          9.0         41.0         46.1         86.8
         10.0          5.0         14.2         32.9
         11.0          0.0          4.8         11.4
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.814           1.469           4.232           3.099
                   Theta0:           0.005           0.000           0.772           0.082               
                   Theta1:          10.885          18.033       99999.000       20046.465
Mean No. of pairwise diff.           3.156           1.430           4.148           2.910
           No. pol. sites:              61              27              45          36.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1318.0        592.0       2281.0       1098.3
          1.0        983.0        463.0       4254.0       1451.6
          2.0       2315.0       1149.0       2875.0       1878.8
          3.0       1858.0       1226.0       3586.0       2035.7
          4.0       1848.0        416.0       2757.0       1798.2
          5.0       1382.0        106.0       2353.0       1315.9
          6.0        791.0         15.0       1087.0        816.0
          7.0        461.0          0.0        806.0        437.6
          8.0        173.0          0.0        534.0        206.4
          9.0         41.0          0.0        267.0         86.8
         10.0          5.0          0.0         86.0         32.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.814           1.469           4.232           3.099
                   Theta0:           0.005           0.000           0.772           0.082               
                   Theta1:          10.885          18.033       99999.000       20046.465
Mean No. of pairwise diff.           3.156           1.430           4.148           2.910
           No. pol. sites:              61              27              45          36.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1318.0        592.0       2281.0       1098.3
          1.0        983.0        463.0       4254.0       1451.6
          2.0       2315.0       1149.0       2875.0       1878.8
          3.0       1858.0       1226.0       3586.0       2035.7
          4.0       1848.0        416.0       2757.0       1798.2
          5.0       1382.0        106.0       2353.0       1315.9
          6.0        791.0         15.0       1087.0        816.0
          7.0        461.0          0.0        806.0        437.6
          8.0        173.0          0.0        534.0        206.4
          9.0         41.0          0.0        267.0         86.8
         10.0          5.0          0.0         86.0         32.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.814           1.469           4.232           3.099
                   Theta0:           0.005           0.000           0.772           0.082               
                   Theta1:          10.885          18.033       99999.000       20046.465
Mean No. of pairwise diff.           3.156           1.430           4.148           2.910
           No. pol. sites:              61              27              45          36.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1318.0        592.0       2281.0       1098.3
          1.0        983.0        463.0       4254.0       1451.6
          2.0       2315.0       1149.0       2875.0       1878.8
          3.0       1858.0       1226.0       3586.0       2035.7
          4.0       1848.0        416.0       2757.0       1798.2
          5.0       1382.0        106.0       2353.0       1315.9
          6.0        791.0         15.0       1087.0        816.0
          7.0        461.0          0.0        806.0        437.6
          8.0        173.0          0.0        534.0        206.4
          9.0         41.0          0.0        267.0         86.8
         10.0          5.0          0.0         86.0         32.9
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_01_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            61      61.000       0.000
  No. of usable loci            61      61.000       0.000
  No. of polym. loci            61      61.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_01_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.02649     0.02649     0.00000     0.02649
     2       0.01333     0.01333     0.00000     0.01333
     3       0.10166     0.10166     0.00000     0.10166
     4       0.18121     0.18121     0.00000     0.18121
     5       0.01333     0.01333     0.00000     0.01333
     6       0.01333     0.01333     0.00000     0.01333
     7       0.01333     0.01333     0.00000     0.01333
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.01333     0.01333     0.00000     0.01333
    11       0.10166     0.10166     0.00000     0.10166
    12       0.01333     0.01333     0.00000     0.01333
    13       0.07732     0.07732     0.00000     0.07732
    14       0.02649     0.02649     0.00000     0.02649
    15       0.03946     0.03946     0.00000     0.03946
    16       0.02649     0.02649     0.00000     0.02649
    17       0.07732     0.07732     0.00000     0.07732
    18       0.01333     0.01333     0.00000     0.01333
    19       0.06488     0.06488     0.00000     0.06488
    20       0.07732     0.07732     0.00000     0.07732
    21       0.03946     0.03946     0.00000     0.03946
    22       0.36018     0.36018     0.00000     0.36018
    23       0.07732     0.07732     0.00000     0.07732
    24       0.01333     0.01333     0.00000     0.01333
    25       0.03946     0.03946     0.00000     0.03946
    26       0.01333     0.01333     0.00000     0.01333
    27       0.01333     0.01333     0.00000     0.01333
    28       0.06488     0.06488     0.00000     0.06488
    29       0.01333     0.01333     0.00000     0.01333
    30       0.01333     0.01333     0.00000     0.01333
    31       0.01333     0.01333     0.00000     0.01333
    32       0.07732     0.07732     0.00000     0.07732
    33       0.06488     0.06488     0.00000     0.06488
    34       0.01333     0.01333     0.00000     0.01333
    35       0.01333     0.01333     0.00000     0.01333
    36       0.36018     0.36018     0.00000     0.36018
    37       0.02649     0.02649     0.00000     0.02649
    38       0.05226     0.05226     0.00000     0.05226
    39       0.05226     0.05226     0.00000     0.05226
    40       0.06488     0.06488     0.00000     0.06488
    41       0.03946     0.03946     0.00000     0.03946
    42       0.06488     0.06488     0.00000     0.06488
    43       0.07732     0.07732     0.00000     0.07732
    44       0.01333     0.01333     0.00000     0.01333
    45       0.02649     0.02649     0.00000     0.02649
    46       0.03946     0.03946     0.00000     0.03946
    47       0.18121     0.18121     0.00000     0.18121
    48       0.03946     0.03946     0.00000     0.03946
    49       0.05226     0.05226     0.00000     0.05226
    50       0.03946     0.03946     0.00000     0.03946
    51       0.01333     0.01333     0.00000     0.01333
    52       0.01333     0.01333     0.00000     0.01333
    53       0.01333     0.01333     0.00000     0.01333
    54       0.05226     0.05226     0.00000     0.05226
    55       0.01333     0.01333     0.00000     0.01333
    56       0.01333     0.01333     0.00000     0.01333
    57       0.02649     0.02649     0.00000     0.02649
    58       0.07732     0.07732     0.00000     0.07732
    59       0.01333     0.01333     0.00000     0.01333
    60       0.02649     0.02649     0.00000     0.02649
    61       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.05173     0.05173     0.00000     0.05173
  s.d.       0.06790     0.06790     0.00000     0.06790
--------------------------------------------------------
<A NAME="31_01_17at14_44_01_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_01_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      18.25848    18.25848     0.00000
            Theta_k_lower      12.44224    12.44224     0.00000
            Theta_k_upper      26.46308    26.46308     0.00000
                  Theta_H       6.28811     6.28811     0.00000
             s.d. Theta_H       1.43452     1.43452     0.00000
                  Theta_S      10.92306    10.92306     0.00000
             s.d. Theta_S       2.79519     2.79519     0.00000
                 Theta_pi       3.15570     3.15570     0.00000
            s.d. Theta_pi       1.81907     1.81907     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_01_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.81445     3.81445     0.00000
              Tau qt 2.5%       1.46875     1.46875     0.00000
                Tau qt 5%       1.46875     1.46875     0.00000
               Tau qt 95%       4.20703     4.20703     0.00000
             Tau qt 97.5%       4.20703     4.20703     0.00000
                   Theta0       0.00508     0.00508     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.02266     0.02266     0.00000
          Theta0 qt 97.5%       0.02266     0.02266     0.00000
                   Theta1      10.88501    10.88501     0.00000
           Theta1 qt 2.5%      18.03345    18.03345     0.00000
             Theta1 qt 5%      18.03345    18.03345     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00402     0.00402     0.00000
      Model (SSD) p-value       0.50000     0.50000     0.00000
         Raggedness index       0.02300     0.02300     0.00000
       Raggedness p-value       0.90000     0.90000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 104 (31/01/17 at 14:44:02))
Total computing time for this run : 0h 0m 0s 125 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

