
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_03"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 116 (31/01/17 at 14:44:03)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_03_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_03_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 50
No. of loci              : 71
No. of usable loci       : 71 loci with less than  5.00 % missing data
No. of polymorphic sites : 71


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.07732
     2       150         2   0.02649
     3       150         2   0.02649
     4       150         2   0.01333
     5       150         2   0.10166
     6       150         2   0.05226
     7       150         2   0.13682
     8       150         2   0.02649
     9       150         2   0.02649
    10       150         2   0.01333
    11       150         2   0.01333
    12       150         2   0.03946
    13       150         2   0.01333
    14       150         2   0.01333
    15       150         2   0.01333
    16       150         2   0.07732
    17       150         2   0.01333
    18       150         2   0.02649
    19       150         2   0.01333
    20       150         2   0.01333
    21       150         2   0.03946
    22       150         2   0.01333
    23       150         2   0.01333
    24       150         2   0.01333
    25       150         2   0.02649
    26       150         2   0.05226
    27       150         2   0.01333
    28       150         2   0.01333
    29       150         2   0.02649
    30       150         2   0.10166
    31       150         2   0.01333
    32       150         2   0.11356
    33       150         2   0.02649
    34       150         2   0.03946
    35       150         2   0.17038
    36       150         2   0.01333
    37       150         2   0.07732
    38       150         2   0.10166
    39       150         2   0.03946
    40       150         2   0.01333
    41       150         2   0.03946
    42       150         2   0.29718
    43       150         2   0.03946
    44       150         2   0.01333
    45       150         2   0.03946
    46       150         2   0.01333
    47       150         2   0.02649
    48       150         2   0.02649
    49       150         2   0.10166
    50       150         2   0.07732
    51       150         2   0.01333
    52       150         2   0.11356
    53       150         2   0.02649
    54       150         2   0.06488
    55       150         2   0.12528
    56       150         2   0.06488
    57       150         2   0.02649
    58       150         2   0.02649
    59       150         2   0.01333
    60       150         2   0.06488
    61       150         2   0.01333
    62       150         2   0.01333
    63       150         2   0.10166
    64       150         2   0.10166
    65       150         2   0.01333
    66       150         2   0.03946
    67       150         2   0.05226
    68       150         2   0.01333
    69       150         2   0.01333
    70       150         2   0.01333
    71       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.04593
  s.d.     0.000     0.000   0.04797
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0632
Gene diversity           :  0.9431 +/-  0.0124

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 50

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 71
Number of usable nucleotide sites            : 71


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.260761 +/-     1.689261

Average gene diversity over loci              :     0.045926 +/-     0.026336

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    15.002079
  S.D. Theta(Hom) :     3.730864

         Theta(k) :    25.857163
95 % confidence interval limits for theta(k) : [  18.038924,  36.741475 ]

         Theta(S) :    12.713730
    S.D. Theta(S) :     3.198456

        Theta(Pi) :     3.260761
   S.D. Theta(Pi) :     1.869848

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     636     460.879
       1    1362    1443.117
       2    2075    2280.232
       3    2376    2427.722
       4    2006    1962.835
       5    1368    1288.206
       6     783     716.694
       7     367     348.711
       8     150     152.003
       9      52      60.543
      10       0      22.408
      11       0       7.821
      12       0       2.608
      13       0       0.840
      14       0       0.264
      15       0       0.082
      16       0       0.025
      17       0       0.008
      18       0       0.002


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.261
Mismatch observed variance          :        3.446
Tau                                 :        3.426
Theta0                              :        0.039
Theta1                              :       35.801

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00003670
          P(Sim. Ssd >= Obs. Ssd): 1.00000000
    Harpending's Raggedness index: 0.01797451
        P(Sim. Rag. >= Obs. Rag.): 0.60000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        636.0        641.7        612.7
          1.0       1362.0       1341.6       1357.1
          2.0       2075.0       2042.1       2103.3
          3.0       2376.0       2350.0       2332.6
          4.0       2006.0       1973.7       1982.7
          5.0       1368.0       1413.9       1359.8
          6.0        783.0        816.4        780.3
          7.0        367.0        388.1        384.7
          8.0        150.0        140.4        166.2
          9.0         52.0         51.3         63.9
         10.0          0.0         13.6         22.1
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.426           2.551           4.199           3.330
                   Theta0:           0.039           0.000           1.078           0.180               
                   Theta1:          35.801          12.676       99999.000       20037.188
Mean No. of pairwise diff.           3.261           2.826           3.848           3.293
           No. pol. sites:              71              58              85          71.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        636.0        259.0       1171.0        612.7
          1.0       1362.0        680.0       1916.0       1357.1
          2.0       2075.0       1231.0       2731.0       2103.3
          3.0       2376.0       2039.0       2831.0       2332.6
          4.0       2006.0       1624.0       2489.0       1982.7
          5.0       1368.0        875.0       1928.0       1359.8
          6.0        783.0        412.0       1459.0        780.3
          7.0        367.0        170.0        656.0        384.7
          8.0        150.0         18.0        323.0        166.2
          9.0         52.0          0.0        149.0         63.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.426           2.551           4.199           3.330
                   Theta0:           0.039           0.000           1.078           0.180               
                   Theta1:          35.801          12.676       99999.000       20037.188
Mean No. of pairwise diff.           3.261           2.826           3.848           3.293
           No. pol. sites:              71              58              85          71.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        636.0        259.0       1171.0        612.7
          1.0       1362.0        680.0       1916.0       1357.1
          2.0       2075.0       1231.0       2731.0       2103.3
          3.0       2376.0       2039.0       2831.0       2332.6
          4.0       2006.0       1624.0       2489.0       1982.7
          5.0       1368.0        875.0       1928.0       1359.8
          6.0        783.0        412.0       1459.0        780.3
          7.0        367.0        170.0        656.0        384.7
          8.0        150.0         18.0        323.0        166.2
          9.0         52.0          0.0        149.0         63.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.426           2.551           4.199           3.330
                   Theta0:           0.039           0.000           1.078           0.180               
                   Theta1:          35.801          12.676       99999.000       20037.188
Mean No. of pairwise diff.           3.261           2.826           3.848           3.293
           No. pol. sites:              71              58              85          71.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        636.0        259.0       1171.0        612.7
          1.0       1362.0        680.0       1916.0       1357.1
          2.0       2075.0       1231.0       2731.0       2103.3
          3.0       2376.0       2039.0       2831.0       2332.6
          4.0       2006.0       1624.0       2489.0       1982.7
          5.0       1368.0        875.0       1928.0       1359.8
          6.0        783.0        412.0       1459.0        780.3
          7.0        367.0        170.0        656.0        384.7
          8.0        150.0         18.0        323.0        166.2
          9.0         52.0          0.0        149.0         63.9
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_03_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            71      71.000       0.000
  No. of usable loci            71      71.000       0.000
  No. of polym. loci            71      71.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_03_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.07732     0.07732     0.00000     0.07732
     2       0.02649     0.02649     0.00000     0.02649
     3       0.02649     0.02649     0.00000     0.02649
     4       0.01333     0.01333     0.00000     0.01333
     5       0.10166     0.10166     0.00000     0.10166
     6       0.05226     0.05226     0.00000     0.05226
     7       0.13682     0.13682     0.00000     0.13682
     8       0.02649     0.02649     0.00000     0.02649
     9       0.02649     0.02649     0.00000     0.02649
    10       0.01333     0.01333     0.00000     0.01333
    11       0.01333     0.01333     0.00000     0.01333
    12       0.03946     0.03946     0.00000     0.03946
    13       0.01333     0.01333     0.00000     0.01333
    14       0.01333     0.01333     0.00000     0.01333
    15       0.01333     0.01333     0.00000     0.01333
    16       0.07732     0.07732     0.00000     0.07732
    17       0.01333     0.01333     0.00000     0.01333
    18       0.02649     0.02649     0.00000     0.02649
    19       0.01333     0.01333     0.00000     0.01333
    20       0.01333     0.01333     0.00000     0.01333
    21       0.03946     0.03946     0.00000     0.03946
    22       0.01333     0.01333     0.00000     0.01333
    23       0.01333     0.01333     0.00000     0.01333
    24       0.01333     0.01333     0.00000     0.01333
    25       0.02649     0.02649     0.00000     0.02649
    26       0.05226     0.05226     0.00000     0.05226
    27       0.01333     0.01333     0.00000     0.01333
    28       0.01333     0.01333     0.00000     0.01333
    29       0.02649     0.02649     0.00000     0.02649
    30       0.10166     0.10166     0.00000     0.10166
    31       0.01333     0.01333     0.00000     0.01333
    32       0.11356     0.11356     0.00000     0.11356
    33       0.02649     0.02649     0.00000     0.02649
    34       0.03946     0.03946     0.00000     0.03946
    35       0.17038     0.17038     0.00000     0.17038
    36       0.01333     0.01333     0.00000     0.01333
    37       0.07732     0.07732     0.00000     0.07732
    38       0.10166     0.10166     0.00000     0.10166
    39       0.03946     0.03946     0.00000     0.03946
    40       0.01333     0.01333     0.00000     0.01333
    41       0.03946     0.03946     0.00000     0.03946
    42       0.29718     0.29718     0.00000     0.29718
    43       0.03946     0.03946     0.00000     0.03946
    44       0.01333     0.01333     0.00000     0.01333
    45       0.03946     0.03946     0.00000     0.03946
    46       0.01333     0.01333     0.00000     0.01333
    47       0.02649     0.02649     0.00000     0.02649
    48       0.02649     0.02649     0.00000     0.02649
    49       0.10166     0.10166     0.00000     0.10166
    50       0.07732     0.07732     0.00000     0.07732
    51       0.01333     0.01333     0.00000     0.01333
    52       0.11356     0.11356     0.00000     0.11356
    53       0.02649     0.02649     0.00000     0.02649
    54       0.06488     0.06488     0.00000     0.06488
    55       0.12528     0.12528     0.00000     0.12528
    56       0.06488     0.06488     0.00000     0.06488
    57       0.02649     0.02649     0.00000     0.02649
    58       0.02649     0.02649     0.00000     0.02649
    59       0.01333     0.01333     0.00000     0.01333
    60       0.06488     0.06488     0.00000     0.06488
    61       0.01333     0.01333     0.00000     0.01333
    62       0.01333     0.01333     0.00000     0.01333
    63       0.10166     0.10166     0.00000     0.10166
    64       0.10166     0.10166     0.00000     0.10166
    65       0.01333     0.01333     0.00000     0.01333
    66       0.03946     0.03946     0.00000     0.03946
    67       0.05226     0.05226     0.00000     0.05226
    68       0.01333     0.01333     0.00000     0.01333
    69       0.01333     0.01333     0.00000     0.01333
    70       0.01333     0.01333     0.00000     0.01333
    71       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.04593     0.04593     0.00000     0.04593
  s.d.       0.04797     0.04797     0.00000     0.04797
--------------------------------------------------------
<A NAME="31_01_17at14_44_03_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_03_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      25.85716    25.85716     0.00000
            Theta_k_lower      18.03892    18.03892     0.00000
            Theta_k_upper      36.74148    36.74148     0.00000
                  Theta_H      15.00208    15.00208     0.00000
             s.d. Theta_H       3.73086     3.73086     0.00000
                  Theta_S      12.71373    12.71373     0.00000
             s.d. Theta_S       3.19846     3.19846     0.00000
                 Theta_pi       3.26076     3.26076     0.00000
            s.d. Theta_pi       1.86985     1.86985     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_03_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.42578     3.42578     0.00000
              Tau qt 2.5%       2.55078     2.55078     0.00000
                Tau qt 5%       2.55078     2.55078     0.00000
               Tau qt 95%       3.94531     3.94531     0.00000
             Tau qt 97.5%       3.94531     3.94531     0.00000
                   Theta0       0.03867     0.03867     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.66445     0.66445     0.00000
          Theta0 qt 97.5%       0.66445     0.66445     0.00000
                   Theta1      35.80078    35.80078     0.00000
           Theta1 qt 2.5%      12.67578    12.67578     0.00000
             Theta1 qt 5%      12.67578    12.67578     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00004     0.00004     0.00000
      Model (SSD) p-value       1.00000     1.00000     0.00000
         Raggedness index       0.01797     0.01797     0.00000
       Raggedness p-value       0.60000     0.60000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 116 (31/01/17 at 14:44:03))
Total computing time for this run : 0h 0m 0s 108 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

