
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_47_26"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 104 (31/01/17 at 14:47:26)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_47_26_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_47_26_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 44
No. of loci              : 72
No. of usable loci       : 72 loci with less than  5.00 % missing data
No. of polymorphic sites : 72


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.19186
     2       150         2   0.19186
     3       150         2   0.21262
     4       150         2   0.08957
     5       150         2   0.01333
     6       150         2   0.01333
     7       150         2   0.06488
     8       150         2   0.02649
     9       150         2   0.05226
    10       150         2   0.13682
    11       150         2   0.02649
    12       150         2   0.01333
    13       150         2   0.13682
    14       150         2   0.05226
    15       150         2   0.02649
    16       150         2   0.01333
    17       150         2   0.01333
    18       150         2   0.01333
    19       150         2   0.01333
    20       150         2   0.08957
    21       150         2   0.01333
    22       150         2   0.01333
    23       150         2   0.05226
    24       150         2   0.01333
    25       150         2   0.01333
    26       150         2   0.01333
    27       150         2   0.02649
    28       150         2   0.02649
    29       150         2   0.01333
    30       150         2   0.21262
    31       150         2   0.02649
    32       150         2   0.01333
    33       150         2   0.05226
    34       150         2   0.01333
    35       150         2   0.11356
    36       150         2   0.05226
    37       150         2   0.01333
    38       150         2   0.05226
    39       150         2   0.02649
    40       150         2   0.01333
    41       150         2   0.15937
    42       150         2   0.18121
    43       150         2   0.19186
    44       150         2   0.17038
    45       150         2   0.02649
    46       150         2   0.21262
    47       150         2   0.01333
    48       150         2   0.01333
    49       150         2   0.08957
    50       150         2   0.07732
    51       150         2   0.01333
    52       150         2   0.19186
    53       150         2   0.01333
    54       150         2   0.13682
    55       150         2   0.01333
    56       150         2   0.01333
    57       150         2   0.01333
    58       150         2   0.05226
    59       150         2   0.05226
    60       150         2   0.01333
    61       150         2   0.06488
    62       150         2   0.02649
    63       150         2   0.14819
    64       150         2   0.03946
    65       150         2   0.02649
    66       150         2   0.01333
    67       150         2   0.03946
    68       150         2   0.01333
    69       150         2   0.02649
    70       150         2   0.01333
    71       150         2   0.02649
    72       150         2   0.13682
-------------------------------------------------
  Mean   150.000     2.000   0.06096
  s.d.     0.000     0.000   0.06381
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0509
Gene diversity           :  0.9554 +/-  0.0061

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 44

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 72
Number of usable nucleotide sites            : 72


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.389262 +/-     2.179852

Average gene diversity over loci              :     0.060962 +/-     0.033512

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    19.782840
  S.D. Theta(Hom) :     3.018358

         Theta(k) :    20.605975
95 % confidence interval limits for theta(k) : [  14.167681,  29.640070 ]

         Theta(S) :    12.892796
    S.D. Theta(S) :     3.238739

        Theta(Pi) :     4.389262
   S.D. Theta(Pi) :     2.412885

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     498       0.000
       1     371       0.000
       2    1001       0.000
       3    1984       0.000
       4    2206       0.000
       5    1775       0.000
       6    1506       0.000
       7     974       0.000
       8     598       0.000
       9     232       0.000
      10      30       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.389
Mismatch observed variance          :        4.351
Tau                                 :        4.639
Theta0                              :        0.012
Theta1                              :       97.500

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00208319
          P(Sim. Ssd >= Obs. Ssd): 0.20000000
    Harpending's Raggedness index: 0.01831176
        P(Sim. Rag. >= Obs. Rag.): 0.30000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        498.0        172.1        214.2
          1.0        371.0        462.7        579.8
          2.0       1001.0       1106.5       1195.0
          3.0       1984.0       1802.6       1782.5
          4.0       2206.0       2019.3       2039.3
          5.0       1775.0       1858.6       1881.5
          6.0       1506.0       1413.4       1451.6
          7.0        974.0       1023.5        961.6
          8.0        598.0        649.4        557.9
          9.0        232.0        379.8        287.9
         10.0         30.0        179.4        133.8
         11.0          0.0         73.8         56.5
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.639           4.090           6.234           4.666
                   Theta0:           0.012           0.000           0.775           0.079               
                   Theta1:          97.500          48.594       99999.000       60044.920
Mean No. of pairwise diff.           4.389           4.133           5.734           4.719
           No. pol. sites:              72             116             146         125.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        498.0         88.0        322.0        214.2
          1.0        371.0        190.0        668.0        579.8
          2.0       1001.0        636.0       1518.0       1195.0
          3.0       1984.0        779.0       2610.0       1782.5
          4.0       2206.0       1459.0       2353.0       2039.3
          5.0       1775.0       1712.0       2127.0       1881.5
          6.0       1506.0       1148.0       1701.0       1451.6
          7.0        974.0        611.0       1669.0        961.6
          8.0        598.0        302.0       1127.0        557.9
          9.0        232.0        150.0        884.0        287.9
         10.0         30.0         38.0        444.0        133.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.639           4.090           6.234           4.666
                   Theta0:           0.012           0.000           0.775           0.079               
                   Theta1:          97.500          48.594       99999.000       60044.920
Mean No. of pairwise diff.           4.389           4.133           5.734           4.719
           No. pol. sites:              72             116             146         125.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        498.0         88.0        322.0        214.2
          1.0        371.0        190.0        668.0        579.8
          2.0       1001.0        636.0       1518.0       1195.0
          3.0       1984.0        779.0       2610.0       1782.5
          4.0       2206.0       1459.0       2353.0       2039.3
          5.0       1775.0       1712.0       2127.0       1881.5
          6.0       1506.0       1148.0       1701.0       1451.6
          7.0        974.0        611.0       1669.0        961.6
          8.0        598.0        302.0       1127.0        557.9
          9.0        232.0        150.0        884.0        287.9
         10.0         30.0         38.0        444.0        133.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.639           4.090           6.234           4.666
                   Theta0:           0.012           0.000           0.775           0.079               
                   Theta1:          97.500          48.594       99999.000       60044.920
Mean No. of pairwise diff.           4.389           4.133           5.734           4.719
           No. pol. sites:              72             116             146         125.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        498.0         88.0        322.0        214.2
          1.0        371.0        190.0        668.0        579.8
          2.0       1001.0        636.0       1518.0       1195.0
          3.0       1984.0        779.0       2610.0       1782.5
          4.0       2206.0       1459.0       2353.0       2039.3
          5.0       1775.0       1712.0       2127.0       1881.5
          6.0       1506.0       1148.0       1701.0       1451.6
          7.0        974.0        611.0       1669.0        961.6
          8.0        598.0        302.0       1127.0        557.9
          9.0        232.0        150.0        884.0        287.9
         10.0         30.0         38.0        444.0        133.8
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_47_26_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            72      72.000       0.000
  No. of usable loci            72      72.000       0.000
  No. of polym. loci            72      72.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_47_26_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.19186     0.19186     0.00000     0.19186
     2       0.19186     0.19186     0.00000     0.19186
     3       0.21262     0.21262     0.00000     0.21262
     4       0.08957     0.08957     0.00000     0.08957
     5       0.01333     0.01333     0.00000     0.01333
     6       0.01333     0.01333     0.00000     0.01333
     7       0.06488     0.06488     0.00000     0.06488
     8       0.02649     0.02649     0.00000     0.02649
     9       0.05226     0.05226     0.00000     0.05226
    10       0.13682     0.13682     0.00000     0.13682
    11       0.02649     0.02649     0.00000     0.02649
    12       0.01333     0.01333     0.00000     0.01333
    13       0.13682     0.13682     0.00000     0.13682
    14       0.05226     0.05226     0.00000     0.05226
    15       0.02649     0.02649     0.00000     0.02649
    16       0.01333     0.01333     0.00000     0.01333
    17       0.01333     0.01333     0.00000     0.01333
    18       0.01333     0.01333     0.00000     0.01333
    19       0.01333     0.01333     0.00000     0.01333
    20       0.08957     0.08957     0.00000     0.08957
    21       0.01333     0.01333     0.00000     0.01333
    22       0.01333     0.01333     0.00000     0.01333
    23       0.05226     0.05226     0.00000     0.05226
    24       0.01333     0.01333     0.00000     0.01333
    25       0.01333     0.01333     0.00000     0.01333
    26       0.01333     0.01333     0.00000     0.01333
    27       0.02649     0.02649     0.00000     0.02649
    28       0.02649     0.02649     0.00000     0.02649
    29       0.01333     0.01333     0.00000     0.01333
    30       0.21262     0.21262     0.00000     0.21262
    31       0.02649     0.02649     0.00000     0.02649
    32       0.01333     0.01333     0.00000     0.01333
    33       0.05226     0.05226     0.00000     0.05226
    34       0.01333     0.01333     0.00000     0.01333
    35       0.11356     0.11356     0.00000     0.11356
    36       0.05226     0.05226     0.00000     0.05226
    37       0.01333     0.01333     0.00000     0.01333
    38       0.05226     0.05226     0.00000     0.05226
    39       0.02649     0.02649     0.00000     0.02649
    40       0.01333     0.01333     0.00000     0.01333
    41       0.15937     0.15937     0.00000     0.15937
    42       0.18121     0.18121     0.00000     0.18121
    43       0.19186     0.19186     0.00000     0.19186
    44       0.17038     0.17038     0.00000     0.17038
    45       0.02649     0.02649     0.00000     0.02649
    46       0.21262     0.21262     0.00000     0.21262
    47       0.01333     0.01333     0.00000     0.01333
    48       0.01333     0.01333     0.00000     0.01333
    49       0.08957     0.08957     0.00000     0.08957
    50       0.07732     0.07732     0.00000     0.07732
    51       0.01333     0.01333     0.00000     0.01333
    52       0.19186     0.19186     0.00000     0.19186
    53       0.01333     0.01333     0.00000     0.01333
    54       0.13682     0.13682     0.00000     0.13682
    55       0.01333     0.01333     0.00000     0.01333
    56       0.01333     0.01333     0.00000     0.01333
    57       0.01333     0.01333     0.00000     0.01333
    58       0.05226     0.05226     0.00000     0.05226
    59       0.05226     0.05226     0.00000     0.05226
    60       0.01333     0.01333     0.00000     0.01333
    61       0.06488     0.06488     0.00000     0.06488
    62       0.02649     0.02649     0.00000     0.02649
    63       0.14819     0.14819     0.00000     0.14819
    64       0.03946     0.03946     0.00000     0.03946
    65       0.02649     0.02649     0.00000     0.02649
    66       0.01333     0.01333     0.00000     0.01333
    67       0.03946     0.03946     0.00000     0.03946
    68       0.01333     0.01333     0.00000     0.01333
    69       0.02649     0.02649     0.00000     0.02649
    70       0.01333     0.01333     0.00000     0.01333
    71       0.02649     0.02649     0.00000     0.02649
    72       0.13682     0.13682     0.00000     0.13682
--------------------------------------------------------
  Mean       0.06096     0.06096     0.00000     0.06096
  s.d.       0.06381     0.06381     0.00000     0.06381
--------------------------------------------------------
<A NAME="31_01_17at14_47_26_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
    72             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_47_26_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      20.60597    20.60597     0.00000
            Theta_k_lower      14.16768    14.16768     0.00000
            Theta_k_upper      29.64007    29.64007     0.00000
                  Theta_H      19.78284    19.78284     0.00000
             s.d. Theta_H       3.01836     3.01836     0.00000
                  Theta_S      12.89280    12.89280     0.00000
             s.d. Theta_S       3.23874     3.23874     0.00000
                 Theta_pi       4.38926     4.38926     0.00000
            s.d. Theta_pi       2.41289     2.41289     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_47_26_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.63867     4.63867     0.00000
              Tau qt 2.5%       4.08984     4.08984     0.00000
                Tau qt 5%       4.08984     4.08984     0.00000
               Tau qt 95%       5.14648     5.14648     0.00000
             Tau qt 97.5%       5.14648     5.14648     0.00000
                   Theta0       0.01230     0.01230     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00703     0.00703     0.00000
          Theta0 qt 97.5%       0.00703     0.00703     0.00000
                   Theta1      97.50000    97.50000     0.00000
           Theta1 qt 2.5%      48.59375    48.59375     0.00000
             Theta1 qt 5%      48.59375    48.59375     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00208     0.00208     0.00000
      Model (SSD) p-value       0.20000     0.20000     0.00000
         Raggedness index       0.01831     0.01831     0.00000
       Raggedness p-value       0.30000     0.30000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 104 (31/01/17 at 14:47:26))
Total computing time for this run : 0h 0m 0s 132 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

