
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_49_13"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 27 (31/01/17 at 14:49:13)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_49_13_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_49_13_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 46
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.36725
     2       150         2   0.10166
     3       150         2   0.11356
     4       150         2   0.21262
     5       150         2   0.31400
     6       150         2   0.24242
     7       150         2   0.02649
     8       150         2   0.31400
     9       150         2   0.41172
    10       150         2   0.13682
    11       150         2   0.22273
    12       150         2   0.22273
    13       150         2   0.11356
    14       150         2   0.07732
    15       150         2   0.22273
    16       150         2   0.23266
    17       150         2   0.02649
    18       150         2   0.15937
    19       150         2   0.28850
    20       150         2   0.28850
-------------------------------------------------
  Mean   150.000     2.000   0.20476
  s.d.     0.000     0.000   0.10860
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0348
Gene diversity           :  0.9717 +/-  0.0036

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 46

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.095123 +/-     2.052290

Average gene diversity over loci              :     0.204756 +/-     0.113584

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    32.585485
  S.D. Theta(Hom) :     4.515681

         Theta(k) :    22.270418
95 % confidence interval limits for theta(k) : [  15.393436,  31.891169 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     4.095123
   S.D. Theta(Pi) :     2.271687

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     316     214.647
       1     869     826.302
       2    1558    1606.596
       3    1928    2107.156
       4    2033    2101.557
       5    1696    1704.222
       6    1148    1173.932
       7     830     708.949
       8     500     384.699
       9     286     191.401
      10      11      88.835
      11       0      39.046
      12       0      16.467
      13       0       6.738
      14       0       2.700
      15       0       1.067
      16       0       0.418
      17       0       0.163
      18       0       0.063
      19       0       0.025
      20       0       0.010


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.095
Mismatch observed variance          :        4.496
Tau                                 :        3.449
Theta0                              :        0.856
Theta1                              :       77.656

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00025632
          P(Sim. Ssd >= Obs. Ssd): 0.80000000
    Harpending's Raggedness index: 0.01340395
        P(Sim. Rag. >= Obs. Rag.): 0.80000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        316.0        311.3        320.7
          1.0        869.0        768.4        836.7
          2.0       1558.0       1470.4       1512.4
          3.0       1928.0       2121.3       1968.9
          4.0       2033.0       2195.7       1995.0
          5.0       1696.0       1778.9       1666.3
          6.0       1148.0       1157.3       1196.3
          7.0        830.0        721.7        762.1
          8.0        500.0        374.7        442.0
          9.0        286.0        166.7        238.5
         10.0         11.0         71.5        122.0
         11.0          0.0         27.4         60.0
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.449           3.004           5.650           4.066
                   Theta0:           0.856           0.000           1.471           0.157               
                   Theta1:          77.656          25.430       99999.000       50251.094
Mean No. of pairwise diff.           4.095           3.221           5.430           4.061
           No. pol. sites:              20              90             115         103.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        316.0        147.0        673.0        320.7
          1.0        869.0        292.0       1476.0        836.7
          2.0       1558.0        611.0       2179.0       1512.4
          3.0       1928.0       1073.0       2917.0       1968.9
          4.0       2033.0       1668.0       2972.0       1995.0
          5.0       1696.0       1233.0       2125.0       1666.3
          6.0       1148.0        725.0       1846.0       1196.3
          7.0        830.0        169.0       1557.0        762.1
          8.0        500.0         15.0       1011.0        442.0
          9.0        286.0          0.0        561.0        238.5
         10.0         11.0          0.0        268.0        122.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.449           3.004           5.650           4.066
                   Theta0:           0.856           0.000           1.471           0.157               
                   Theta1:          77.656          25.430       99999.000       50251.094
Mean No. of pairwise diff.           4.095           3.221           5.430           4.061
           No. pol. sites:              20              90             115         103.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        316.0        147.0        673.0        320.7
          1.0        869.0        292.0       1476.0        836.7
          2.0       1558.0        611.0       2179.0       1512.4
          3.0       1928.0       1073.0       2917.0       1968.9
          4.0       2033.0       1668.0       2972.0       1995.0
          5.0       1696.0       1233.0       2125.0       1666.3
          6.0       1148.0        725.0       1846.0       1196.3
          7.0        830.0        169.0       1557.0        762.1
          8.0        500.0         15.0       1011.0        442.0
          9.0        286.0          0.0        561.0        238.5
         10.0         11.0          0.0        268.0        122.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.449           3.004           5.650           4.066
                   Theta0:           0.856           0.000           1.471           0.157               
                   Theta1:          77.656          25.430       99999.000       50251.094
Mean No. of pairwise diff.           4.095           3.221           5.430           4.061
           No. pol. sites:              20              90             115         103.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        316.0        147.0        673.0        320.7
          1.0        869.0        292.0       1476.0        836.7
          2.0       1558.0        611.0       2179.0       1512.4
          3.0       1928.0       1073.0       2917.0       1968.9
          4.0       2033.0       1668.0       2972.0       1995.0
          5.0       1696.0       1233.0       2125.0       1666.3
          6.0       1148.0        725.0       1846.0       1196.3
          7.0        830.0        169.0       1557.0        762.1
          8.0        500.0         15.0       1011.0        442.0
          9.0        286.0          0.0        561.0        238.5
         10.0         11.0          0.0        268.0        122.0
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_49_13_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_49_13_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.36725     0.36725     0.00000     0.36725
     2       0.10166     0.10166     0.00000     0.10166
     3       0.11356     0.11356     0.00000     0.11356
     4       0.21262     0.21262     0.00000     0.21262
     5       0.31400     0.31400     0.00000     0.31400
     6       0.24242     0.24242     0.00000     0.24242
     7       0.02649     0.02649     0.00000     0.02649
     8       0.31400     0.31400     0.00000     0.31400
     9       0.41172     0.41172     0.00000     0.41172
    10       0.13682     0.13682     0.00000     0.13682
    11       0.22273     0.22273     0.00000     0.22273
    12       0.22273     0.22273     0.00000     0.22273
    13       0.11356     0.11356     0.00000     0.11356
    14       0.07732     0.07732     0.00000     0.07732
    15       0.22273     0.22273     0.00000     0.22273
    16       0.23266     0.23266     0.00000     0.23266
    17       0.02649     0.02649     0.00000     0.02649
    18       0.15937     0.15937     0.00000     0.15937
    19       0.28850     0.28850     0.00000     0.28850
    20       0.28850     0.28850     0.00000     0.28850
--------------------------------------------------------
  Mean       0.20476     0.20476     0.00000     0.20476
  s.d.       0.10860     0.10860     0.00000     0.10860
--------------------------------------------------------
<A NAME="31_01_17at14_49_13_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_49_13_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      22.27042    22.27042     0.00000
            Theta_k_lower      15.39344    15.39344     0.00000
            Theta_k_upper      31.89117    31.89117     0.00000
                  Theta_H      32.58548    32.58548     0.00000
             s.d. Theta_H       4.51568     4.51568     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       4.09512     4.09512     0.00000
            s.d. Theta_pi       2.27169     2.27169     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_49_13_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.44922     3.44922     0.00000
              Tau qt 2.5%       3.00391     3.00391     0.00000
                Tau qt 5%       3.00391     3.00391     0.00000
               Tau qt 95%       4.95898     4.95898     0.00000
             Tau qt 97.5%       4.95898     4.95898     0.00000
                   Theta0       0.85586     0.85586     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.04043     0.04043     0.00000
          Theta0 qt 97.5%       0.04043     0.04043     0.00000
                   Theta1      77.65625    77.65625     0.00000
           Theta1 qt 2.5%      25.42969    25.42969     0.00000
             Theta1 qt 5%      25.42969    25.42969     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00026     0.00026     0.00000
      Model (SSD) p-value       0.80000     0.80000     0.00000
         Raggedness index       0.01340     0.01340     0.00000
       Raggedness p-value       0.80000     0.80000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 27 (31/01/17 at 14:49:13))
Total computing time for this run : 0h 0m 0s 134 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

