
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_47_49"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 257 (31/01/17 at 14:47:49)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_47_49_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_47_49_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 37
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.27060
     2       150         2   0.19186
     3       150         2   0.01333
     4       150         2   0.36725
     5       150         2   0.28850
     6       150         2   0.01333
     7       150         2   0.01333
     8       150         2   0.05226
     9       150         2   0.17038
    10       150         2   0.08957
    11       150         2   0.11356
    12       150         2   0.13682
    13       150         2   0.03946
    14       150         2   0.10166
    15       150         2   0.36725
    16       150         2   0.02649
    17       150         2   0.01333
    18       150         2   0.48823
    19       150         2   0.06488
    20       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.14177
  s.d.     0.000     0.000   0.14299
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0696
Gene diversity           :  0.9366 +/-  0.0087

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 37

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.835436 +/-     1.503227

Average gene diversity over loci              :     0.141772 +/-     0.083196

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    13.259788
  S.D. Theta(Hom) :     2.103934

         Theta(k) :    15.387437
95 % confidence interval limits for theta(k) : [  10.339891,  22.570555 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     2.835436
   S.D. Theta(Pi) :     1.663927

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     708     680.850
       1    1829    1883.723
       2    2779    2623.880
       3    2230    2455.832
       4    1838    1739.565
       5     906     996.095
       6     539     481.084
       7     288     201.966
       8      46      75.409
       9      11      25.506
      10       1       7.937
      11       0       2.303
      12       0       0.631
      13       0       0.165
      14       0       0.041
      15       0       0.010
      16       0       0.002
      17       0       0.001
      18       0       0.000
      19       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.835
Mismatch observed variance          :        2.925
Tau                                 :        2.836
Theta0                              :        0.000
Theta1                              :      125.156

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00091079
          P(Sim. Ssd >= Obs. Ssd): 0.20000000
    Harpending's Raggedness index: 0.02995188
        P(Sim. Rag. >= Obs. Rag.): 0.60000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        708.0        727.9        724.4
          1.0       1829.0       1950.0       1885.9
          2.0       2779.0       2671.8       2624.6
          3.0       2230.0       2569.6       2464.0
          4.0       1838.0       1713.6       1741.1
          5.0        906.0        911.0        985.7
          6.0        539.0        398.8        465.3
          7.0        288.0        156.0        188.4
          8.0         46.0         53.8         66.7
          9.0         11.0         16.3         21.0
         10.0          1.0          4.6          6.0
         11.0          0.0          0.9          1.5
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.836           2.068           3.121           2.679
                   Theta0:           0.000           0.000           0.990           0.113               
                   Theta1:         125.156          16.680       99999.000       70062.847
Mean No. of pairwise diff.           2.835           2.297           3.077           2.733
           No. pol. sites:              20              95             127         109.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        708.0        355.0       1364.0        724.4
          1.0       1829.0       1472.0       2571.0       1885.9
          2.0       2779.0       2423.0       3066.0       2624.6
          3.0       2230.0       2310.0       2879.0       2464.0
          4.0       1838.0       1361.0       2129.0       1741.1
          5.0        906.0        594.0       1166.0        985.7
          6.0        539.0        155.0        609.0        465.3
          7.0        288.0         26.0        383.0        188.4
          8.0         46.0          1.0        165.0         66.7
          9.0         11.0          0.0         74.0         21.0
         10.0          1.0          0.0         30.0          6.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.836           2.068           3.121           2.679
                   Theta0:           0.000           0.000           0.990           0.113               
                   Theta1:         125.156          16.680       99999.000       70062.847
Mean No. of pairwise diff.           2.835           2.297           3.077           2.733
           No. pol. sites:              20              95             127         109.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        708.0        355.0       1364.0        724.4
          1.0       1829.0       1472.0       2571.0       1885.9
          2.0       2779.0       2423.0       3066.0       2624.6
          3.0       2230.0       2310.0       2879.0       2464.0
          4.0       1838.0       1361.0       2129.0       1741.1
          5.0        906.0        594.0       1166.0        985.7
          6.0        539.0        155.0        609.0        465.3
          7.0        288.0         26.0        383.0        188.4
          8.0         46.0          1.0        165.0         66.7
          9.0         11.0          0.0         74.0         21.0
         10.0          1.0          0.0         30.0          6.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.836           2.068           3.121           2.679
                   Theta0:           0.000           0.000           0.990           0.113               
                   Theta1:         125.156          16.680       99999.000       70062.847
Mean No. of pairwise diff.           2.835           2.297           3.077           2.733
           No. pol. sites:              20              95             127         109.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        708.0        355.0       1364.0        724.4
          1.0       1829.0       1472.0       2571.0       1885.9
          2.0       2779.0       2423.0       3066.0       2624.6
          3.0       2230.0       2310.0       2879.0       2464.0
          4.0       1838.0       1361.0       2129.0       1741.1
          5.0        906.0        594.0       1166.0        985.7
          6.0        539.0        155.0        609.0        465.3
          7.0        288.0         26.0        383.0        188.4
          8.0         46.0          1.0        165.0         66.7
          9.0         11.0          0.0         74.0         21.0
         10.0          1.0          0.0         30.0          6.0
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_47_49_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_47_49_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.27060     0.27060     0.00000     0.27060
     2       0.19186     0.19186     0.00000     0.19186
     3       0.01333     0.01333     0.00000     0.01333
     4       0.36725     0.36725     0.00000     0.36725
     5       0.28850     0.28850     0.00000     0.28850
     6       0.01333     0.01333     0.00000     0.01333
     7       0.01333     0.01333     0.00000     0.01333
     8       0.05226     0.05226     0.00000     0.05226
     9       0.17038     0.17038     0.00000     0.17038
    10       0.08957     0.08957     0.00000     0.08957
    11       0.11356     0.11356     0.00000     0.11356
    12       0.13682     0.13682     0.00000     0.13682
    13       0.03946     0.03946     0.00000     0.03946
    14       0.10166     0.10166     0.00000     0.10166
    15       0.36725     0.36725     0.00000     0.36725
    16       0.02649     0.02649     0.00000     0.02649
    17       0.01333     0.01333     0.00000     0.01333
    18       0.48823     0.48823     0.00000     0.48823
    19       0.06488     0.06488     0.00000     0.06488
    20       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.14177     0.14177     0.00000     0.14177
  s.d.       0.14299     0.14299     0.00000     0.14299
--------------------------------------------------------
<A NAME="31_01_17at14_47_49_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_47_49_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      15.38744    15.38744     0.00000
            Theta_k_lower      10.33989    10.33989     0.00000
            Theta_k_upper      22.57055    22.57055     0.00000
                  Theta_H      13.25979    13.25979     0.00000
             s.d. Theta_H       2.10393     2.10393     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       2.83544     2.83544     0.00000
            s.d. Theta_pi       1.66393     1.66393     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_47_49_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       2.83594     2.83594     0.00000
              Tau qt 2.5%       2.06836     2.06836     0.00000
                Tau qt 5%       2.06836     2.06836     0.00000
               Tau qt 95%       3.00195     3.00195     0.00000
             Tau qt 97.5%       3.00195     3.00195     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.08086     0.08086     0.00000
          Theta0 qt 97.5%       0.08086     0.08086     0.00000
                   Theta1     125.15625   125.15625     0.00000
           Theta1 qt 2.5%      16.67969    16.67969     0.00000
             Theta1 qt 5%      16.67969    16.67969     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00091     0.00091     0.00000
      Model (SSD) p-value       0.20000     0.20000     0.00000
         Raggedness index       0.02995     0.02995     0.00000
       Raggedness p-value       0.60000     0.60000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 257 (31/01/17 at 14:47:49))
Total computing time for this run : 0h 0m 0s 112 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

