
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_54_17"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 137 (31/01/17 at 14:54:17)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_54_17_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_54_17_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 38
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.27964
     3       150         2   0.40591
     4       150         2   0.17038
     5       150         2   0.05226
     6       150         2   0.11356
     7       150         2   0.31400
     8       150         2   0.02649
     9       150         2   0.08957
    10       150         2   0.32215
    11       150         2   0.12528
    12       150         2   0.03946
    13       150         2   0.34550
    14       150         2   0.13682
    15       150         2   0.10166
    16       150         2   0.21262
    17       150         2   0.27964
    18       150         2   0.23266
    19       150         2   0.49897
    20       150         2   0.02649
-------------------------------------------------
  Mean   150.000     2.000   0.18932
  s.d.     0.000     0.000   0.14048
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0668
Gene diversity           :  0.9395 +/-  0.0090

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 38

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.786398 +/-     1.918196

Average gene diversity over loci              :     0.189320 +/-     0.106163

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    13.987277
  S.D. Theta(Hom) :     2.387242

         Theta(k) :    16.078736
95 % confidence interval limits for theta(k) : [  10.845089,  23.508807 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     3.786398
   S.D. Theta(Pi) :     2.123258

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     676     315.527
       1     982    1079.095
       2    1653    1877.080
       3    1826    2221.985
       4    1959    2022.161
       5    1552    1516.814
       6    1339     982.633
       7     764     569.271
       8     247     303.208
       9     177     151.881
      10       0      72.885
      11       0      34.000
      12       0      15.585
      13       0       7.072
      14       0       3.192
      15       0       1.437
      16       0       0.646
      17       0       0.290
      18       0       0.130


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.786
Mismatch observed variance          :        4.452
Tau                                 :        4.201
Theta0                              :        0.139
Theta1                              :       19.248

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00070681
          P(Sim. Ssd >= Obs. Ssd): 0.70000000
    Harpending's Raggedness index: 0.01150423
        P(Sim. Rag. >= Obs. Rag.): 0.80000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        676.0        747.7        663.4
          1.0        982.0       1157.7       1001.3
          2.0       1653.0       1757.8       1512.6
          3.0       1826.0       1964.9       1900.3
          4.0       1959.0       1905.1       1932.0
          5.0       1552.0       1690.7       1620.7
          6.0       1339.0       1063.8       1149.7
          7.0        764.0        556.5        704.7
          8.0        247.0        226.0        380.0
          9.0        177.0         75.9        182.7
         10.0          0.0         21.8         79.3
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.201           2.234           4.922           3.662
                   Theta0:           0.139           0.000           1.452           0.283               
                   Theta1:          19.248          11.602       99999.000       10060.601
Mean No. of pairwise diff.           3.786           2.494           4.349           3.553
           No. pol. sites:              20              32              63          47.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        676.0        419.0       1138.0        663.4
          1.0        982.0        548.0       1840.0       1001.3
          2.0       1653.0        663.0       3806.0       1512.6
          3.0       1826.0       1386.0       2350.0       1900.3
          4.0       1959.0       1362.0       2294.0       1932.0
          5.0       1552.0        908.0       2698.0       1620.7
          6.0       1339.0        256.0       2030.0       1149.7
          7.0        764.0         34.0       1019.0        704.7
          8.0        247.0         17.0        568.0        380.0
          9.0        177.0          6.0        299.0        182.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.201           2.234           4.922           3.662
                   Theta0:           0.139           0.000           1.452           0.283               
                   Theta1:          19.248          11.602       99999.000       10060.601
Mean No. of pairwise diff.           3.786           2.494           4.349           3.553
           No. pol. sites:              20              32              63          47.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        676.0        419.0       1138.0        663.4
          1.0        982.0        548.0       1840.0       1001.3
          2.0       1653.0        663.0       3806.0       1512.6
          3.0       1826.0       1386.0       2350.0       1900.3
          4.0       1959.0       1362.0       2294.0       1932.0
          5.0       1552.0        908.0       2698.0       1620.7
          6.0       1339.0        256.0       2030.0       1149.7
          7.0        764.0         34.0       1019.0        704.7
          8.0        247.0         17.0        568.0        380.0
          9.0        177.0          6.0        299.0        182.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.201           2.234           4.922           3.662
                   Theta0:           0.139           0.000           1.452           0.283               
                   Theta1:          19.248          11.602       99999.000       10060.601
Mean No. of pairwise diff.           3.786           2.494           4.349           3.553
           No. pol. sites:              20              32              63          47.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        676.0        419.0       1138.0        663.4
          1.0        982.0        548.0       1840.0       1001.3
          2.0       1653.0        663.0       3806.0       1512.6
          3.0       1826.0       1386.0       2350.0       1900.3
          4.0       1959.0       1362.0       2294.0       1932.0
          5.0       1552.0        908.0       2698.0       1620.7
          6.0       1339.0        256.0       2030.0       1149.7
          7.0        764.0         34.0       1019.0        704.7
          8.0        247.0         17.0        568.0        380.0
          9.0        177.0          6.0        299.0        182.7
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_54_17_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_54_17_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.27964     0.27964     0.00000     0.27964
     3       0.40591     0.40591     0.00000     0.40591
     4       0.17038     0.17038     0.00000     0.17038
     5       0.05226     0.05226     0.00000     0.05226
     6       0.11356     0.11356     0.00000     0.11356
     7       0.31400     0.31400     0.00000     0.31400
     8       0.02649     0.02649     0.00000     0.02649
     9       0.08957     0.08957     0.00000     0.08957
    10       0.32215     0.32215     0.00000     0.32215
    11       0.12528     0.12528     0.00000     0.12528
    12       0.03946     0.03946     0.00000     0.03946
    13       0.34550     0.34550     0.00000     0.34550
    14       0.13682     0.13682     0.00000     0.13682
    15       0.10166     0.10166     0.00000     0.10166
    16       0.21262     0.21262     0.00000     0.21262
    17       0.27964     0.27964     0.00000     0.27964
    18       0.23266     0.23266     0.00000     0.23266
    19       0.49897     0.49897     0.00000     0.49897
    20       0.02649     0.02649     0.00000     0.02649
--------------------------------------------------------
  Mean       0.18932     0.18932     0.00000     0.18932
  s.d.       0.14048     0.14048     0.00000     0.14048
--------------------------------------------------------
<A NAME="31_01_17at14_54_17_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_54_17_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      16.07874    16.07874     0.00000
            Theta_k_lower      10.84509    10.84509     0.00000
            Theta_k_upper      23.50881    23.50881     0.00000
                  Theta_H      13.98728    13.98728     0.00000
             s.d. Theta_H       2.38724     2.38724     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       3.78640     3.78640     0.00000
            s.d. Theta_pi       2.12326     2.12326     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_54_17_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.20117     4.20117     0.00000
              Tau qt 2.5%       2.23438     2.23438     0.00000
                Tau qt 5%       2.23438     2.23438     0.00000
               Tau qt 95%       4.76953     4.76953     0.00000
             Tau qt 97.5%       4.76953     4.76953     0.00000
                   Theta0       0.13867     0.13867     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.71191     0.71191     0.00000
          Theta0 qt 97.5%       0.71191     0.71191     0.00000
                   Theta1      19.24805    19.24805     0.00000
           Theta1 qt 2.5%      11.60156    11.60156     0.00000
             Theta1 qt 5%      11.60156    11.60156     0.00000
            Theta1 qt 95%     310.18555   310.18555     0.00000
          Theta1 qt 97.5%     310.18555   310.18555     0.00000
                      SSD       0.00071     0.00071     0.00000
      Model (SSD) p-value       0.70000     0.70000     0.00000
         Raggedness index       0.01150     0.01150     0.00000
       Raggedness p-value       0.80000     0.80000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 137 (31/01/17 at 14:54:18))
Total computing time for this run : 0h 0m 0s 127 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

