
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_22"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 266 (31/01/17 at 14:44:22)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_22_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_22_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 54
No. of loci              : 71
No. of usable loci       : 71 loci with less than  5.00 % missing data
No. of polymorphic sites : 71


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.02649
     2       150         2   0.05226
     3       150         2   0.01333
     4       150         2   0.01333
     5       150         2   0.05226
     6       150         2   0.03946
     7       150         2   0.02649
     8       150         2   0.07732
     9       150         2   0.02649
    10       150         2   0.01333
    11       150         2   0.01333
    12       150         2   0.07732
    13       150         2   0.21262
    14       150         2   0.02649
    15       150         2   0.01333
    16       150         2   0.03946
    17       150         2   0.02649
    18       150         2   0.02649
    19       150         2   0.01333
    20       150         2   0.12528
    21       150         2   0.02649
    22       150         2   0.07732
    23       150         2   0.01333
    24       150         2   0.01333
    25       150         2   0.20233
    26       150         2   0.01333
    27       150         2   0.08957
    28       150         2   0.03946
    29       150         2   0.02649
    30       150         2   0.02649
    31       150         2   0.10166
    32       150         2   0.10166
    33       150         2   0.01333
    34       150         2   0.01333
    35       150         2   0.01333
    36       150         2   0.03946
    37       150         2   0.01333
    38       150         2   0.01333
    39       150         2   0.01333
    40       150         2   0.26139
    41       150         2   0.01333
    42       150         2   0.01333
    43       150         2   0.07732
    44       150         2   0.01333
    45       150         2   0.11356
    46       150         2   0.01333
    47       150         2   0.12528
    48       150         2   0.02649
    49       150         2   0.10166
    50       150         2   0.01333
    51       150         2   0.02649
    52       150         2   0.02649
    53       150         2   0.05226
    54       150         2   0.02649
    55       150         2   0.05226
    56       150         2   0.03946
    57       150         2   0.01333
    58       150         2   0.05226
    59       150         2   0.02649
    60       150         2   0.03946
    61       150         2   0.10166
    62       150         2   0.01333
    63       150         2   0.01333
    64       150         2   0.02649
    65       150         2   0.01333
    66       150         2   0.10166
    67       150         2   0.26139
    68       150         2   0.03946
    69       150         2   0.03946
    70       150         2   0.03946
    71       150         2   0.12528
-------------------------------------------------
  Mean   150.000     2.000   0.05166
  s.d.     0.000     0.000   0.05598
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0429
Gene diversity           :  0.9635 +/-  0.0072

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 54

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 71
Number of usable nucleotide sites            : 71


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.668009 +/-     1.866709

Average gene diversity over loci              :     0.051662 +/-     0.029102

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    24.673373
  S.D. Theta(Hom) :     5.324207

         Theta(k) :    29.820320
95 % confidence interval limits for theta(k) : [  20.964943,  42.104679 ]

         Theta(S) :    12.713730
    S.D. Theta(S) :     3.198456

        Theta(Pi) :     3.668009
   S.D. Theta(Pi) :     2.066266

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     408       0.000
       1     916       0.000
       2    1597       0.000
       3    2349       0.000
       4    2340       0.000
       5    1847       0.000
       6    1080       0.000
       7     435       0.000
       8     133       0.000
       9      53       0.000
      10      16       0.000
      11       0       0.000
      12       1       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.668
Mismatch observed variance          :        3.262
Tau                                 :        3.852
Theta0                              :        0.016
Theta1                              :      511.875

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00138180
          P(Sim. Ssd >= Obs. Ssd): 0.10000000
    Harpending's Raggedness index: 0.02109218
        P(Sim. Rag. >= Obs. Rag.): 0.60000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        408.0        247.4        253.3
          1.0        916.0        927.4        916.6
          2.0       1597.0       1829.8       1750.7
          3.0       2349.0       2352.5       2247.8
          4.0       2340.0       2227.5       2169.4
          5.0       1847.0       1659.5       1676.4
          6.0       1080.0       1032.0       1079.9
          7.0        435.0        535.7        596.4
          8.0        133.0        235.7        288.3
          9.0         53.0         91.1        123.9
         10.0         16.0         27.3         47.9
         11.0          0.0          6.9         16.8
         12.0          1.0          2.0          5.4
         13.0          0.0          0.2          1.6
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.852           3.152           4.193           3.684
                   Theta0:           0.016           0.000           0.687           0.081               
                   Theta1:         511.875          84.844       99999.000       70071.034
Mean No. of pairwise diff.           3.668           3.331           4.104           3.747
           No. pol. sites:              71             159             191         176.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        408.0        114.0        449.0        253.3
          1.0        916.0        582.0       1326.0        916.6
          2.0       1597.0       1509.0       2354.0       1750.7
          3.0       2349.0       2082.0       2531.0       2247.8
          4.0       2340.0       2006.0       2430.0       2169.4
          5.0       1847.0       1429.0       2016.0       1676.4
          6.0       1080.0        790.0       1346.0       1079.9
          7.0        435.0        347.0        749.0        596.4
          8.0        133.0        139.0        318.0        288.3
          9.0         53.0         46.0        161.0        123.9
         10.0         16.0          6.0         56.0         47.9
         11.0          0.0          0.0         20.0         16.8
         12.0          1.0          0.0          8.0          5.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.852           3.152           4.193           3.684
                   Theta0:           0.016           0.000           0.687           0.081               
                   Theta1:         511.875          84.844       99999.000       70071.034
Mean No. of pairwise diff.           3.668           3.331           4.104           3.747
           No. pol. sites:              71             159             191         176.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        408.0        114.0        449.0        253.3
          1.0        916.0        582.0       1326.0        916.6
          2.0       1597.0       1509.0       2354.0       1750.7
          3.0       2349.0       2082.0       2531.0       2247.8
          4.0       2340.0       2006.0       2430.0       2169.4
          5.0       1847.0       1429.0       2016.0       1676.4
          6.0       1080.0        790.0       1346.0       1079.9
          7.0        435.0        347.0        749.0        596.4
          8.0        133.0        139.0        318.0        288.3
          9.0         53.0         46.0        161.0        123.9
         10.0         16.0          6.0         56.0         47.9
         11.0          0.0          0.0         20.0         16.8
         12.0          1.0          0.0          8.0          5.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.852           3.152           4.193           3.684
                   Theta0:           0.016           0.000           0.687           0.081               
                   Theta1:         511.875          84.844       99999.000       70071.034
Mean No. of pairwise diff.           3.668           3.331           4.104           3.747
           No. pol. sites:              71             159             191         176.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        408.0        114.0        449.0        253.3
          1.0        916.0        582.0       1326.0        916.6
          2.0       1597.0       1509.0       2354.0       1750.7
          3.0       2349.0       2082.0       2531.0       2247.8
          4.0       2340.0       2006.0       2430.0       2169.4
          5.0       1847.0       1429.0       2016.0       1676.4
          6.0       1080.0        790.0       1346.0       1079.9
          7.0        435.0        347.0        749.0        596.4
          8.0        133.0        139.0        318.0        288.3
          9.0         53.0         46.0        161.0        123.9
         10.0         16.0          6.0         56.0         47.9
         11.0          0.0          0.0         20.0         16.8
         12.0          1.0          0.0          8.0          5.4
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_22_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            71      71.000       0.000
  No. of usable loci            71      71.000       0.000
  No. of polym. loci            71      71.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_22_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.02649     0.02649     0.00000     0.02649
     2       0.05226     0.05226     0.00000     0.05226
     3       0.01333     0.01333     0.00000     0.01333
     4       0.01333     0.01333     0.00000     0.01333
     5       0.05226     0.05226     0.00000     0.05226
     6       0.03946     0.03946     0.00000     0.03946
     7       0.02649     0.02649     0.00000     0.02649
     8       0.07732     0.07732     0.00000     0.07732
     9       0.02649     0.02649     0.00000     0.02649
    10       0.01333     0.01333     0.00000     0.01333
    11       0.01333     0.01333     0.00000     0.01333
    12       0.07732     0.07732     0.00000     0.07732
    13       0.21262     0.21262     0.00000     0.21262
    14       0.02649     0.02649     0.00000     0.02649
    15       0.01333     0.01333     0.00000     0.01333
    16       0.03946     0.03946     0.00000     0.03946
    17       0.02649     0.02649     0.00000     0.02649
    18       0.02649     0.02649     0.00000     0.02649
    19       0.01333     0.01333     0.00000     0.01333
    20       0.12528     0.12528     0.00000     0.12528
    21       0.02649     0.02649     0.00000     0.02649
    22       0.07732     0.07732     0.00000     0.07732
    23       0.01333     0.01333     0.00000     0.01333
    24       0.01333     0.01333     0.00000     0.01333
    25       0.20233     0.20233     0.00000     0.20233
    26       0.01333     0.01333     0.00000     0.01333
    27       0.08957     0.08957     0.00000     0.08957
    28       0.03946     0.03946     0.00000     0.03946
    29       0.02649     0.02649     0.00000     0.02649
    30       0.02649     0.02649     0.00000     0.02649
    31       0.10166     0.10166     0.00000     0.10166
    32       0.10166     0.10166     0.00000     0.10166
    33       0.01333     0.01333     0.00000     0.01333
    34       0.01333     0.01333     0.00000     0.01333
    35       0.01333     0.01333     0.00000     0.01333
    36       0.03946     0.03946     0.00000     0.03946
    37       0.01333     0.01333     0.00000     0.01333
    38       0.01333     0.01333     0.00000     0.01333
    39       0.01333     0.01333     0.00000     0.01333
    40       0.26139     0.26139     0.00000     0.26139
    41       0.01333     0.01333     0.00000     0.01333
    42       0.01333     0.01333     0.00000     0.01333
    43       0.07732     0.07732     0.00000     0.07732
    44       0.01333     0.01333     0.00000     0.01333
    45       0.11356     0.11356     0.00000     0.11356
    46       0.01333     0.01333     0.00000     0.01333
    47       0.12528     0.12528     0.00000     0.12528
    48       0.02649     0.02649     0.00000     0.02649
    49       0.10166     0.10166     0.00000     0.10166
    50       0.01333     0.01333     0.00000     0.01333
    51       0.02649     0.02649     0.00000     0.02649
    52       0.02649     0.02649     0.00000     0.02649
    53       0.05226     0.05226     0.00000     0.05226
    54       0.02649     0.02649     0.00000     0.02649
    55       0.05226     0.05226     0.00000     0.05226
    56       0.03946     0.03946     0.00000     0.03946
    57       0.01333     0.01333     0.00000     0.01333
    58       0.05226     0.05226     0.00000     0.05226
    59       0.02649     0.02649     0.00000     0.02649
    60       0.03946     0.03946     0.00000     0.03946
    61       0.10166     0.10166     0.00000     0.10166
    62       0.01333     0.01333     0.00000     0.01333
    63       0.01333     0.01333     0.00000     0.01333
    64       0.02649     0.02649     0.00000     0.02649
    65       0.01333     0.01333     0.00000     0.01333
    66       0.10166     0.10166     0.00000     0.10166
    67       0.26139     0.26139     0.00000     0.26139
    68       0.03946     0.03946     0.00000     0.03946
    69       0.03946     0.03946     0.00000     0.03946
    70       0.03946     0.03946     0.00000     0.03946
    71       0.12528     0.12528     0.00000     0.12528
--------------------------------------------------------
  Mean       0.05166     0.05166     0.00000     0.05166
  s.d.       0.05598     0.05598     0.00000     0.05598
--------------------------------------------------------
<A NAME="31_01_17at14_44_22_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_22_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      29.82032    29.82032     0.00000
            Theta_k_lower      20.96494    20.96494     0.00000
            Theta_k_upper      42.10468    42.10468     0.00000
                  Theta_H      24.67337    24.67337     0.00000
             s.d. Theta_H       5.32421     5.32421     0.00000
                  Theta_S      12.71373    12.71373     0.00000
             s.d. Theta_S       3.19846     3.19846     0.00000
                 Theta_pi       3.66801     3.66801     0.00000
            s.d. Theta_pi       2.06627     2.06627     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_22_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.85156     3.85156     0.00000
              Tau qt 2.5%       3.15234     3.15234     0.00000
                Tau qt 5%       3.15234     3.15234     0.00000
               Tau qt 95%       4.05469     4.05469     0.00000
             Tau qt 97.5%       4.05469     4.05469     0.00000
                   Theta0       0.01582     0.01582     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.08613     0.08613     0.00000
          Theta0 qt 97.5%       0.08613     0.08613     0.00000
                   Theta1     511.87500   511.87500     0.00000
           Theta1 qt 2.5%      84.84375    84.84375     0.00000
             Theta1 qt 5%      84.84375    84.84375     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00138     0.00138     0.00000
      Model (SSD) p-value       0.10000     0.10000     0.00000
         Raggedness index       0.02109     0.02109     0.00000
       Raggedness p-value       0.60000     0.60000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 266 (31/01/17 at 14:44:22))
Total computing time for this run : 0h 0m 0s 190 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

