
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_29_26"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 74 (31/01/17 at 16:29:26)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_29_26_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_29_26_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 45
No. of loci              : 54
No. of usable loci       : 54 loci with less than  5.00 % missing data
No. of polymorphic sites : 54


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.01333
     3       150         2   0.02649
     4       150         2   0.02649
     5       150         2   0.02649
     6       150         2   0.01333
     7       150         2   0.03946
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.02649
    11       150         2   0.01333
    12       150         2   0.01333
    13       150         2   0.01333
    14       150         2   0.01333
    15       150         2   0.02649
    16       150         2   0.01333
    17       150         2   0.01333
    18       150         2   0.03946
    19       150         2   0.02649
    20       150         2   0.01333
    21       150         2   0.01333
    22       150         2   0.02649
    23       150         2   0.01333
    24       150         2   0.01333
    25       150         2   0.01333
    26       150         2   0.07732
    27       150         2   0.03946
    28       150         2   0.01333
    29       150         2   0.02649
    30       150         2   0.01333
    31       150         2   0.01333
    32       150         2   0.02649
    33       150         2   0.06488
    34       150         2   0.02649
    35       150         2   0.01333
    36       150         2   0.01333
    37       150         2   0.02649
    38       150         2   0.01333
    39       150         2   0.03946
    40       150         2   0.01333
    41       150         2   0.01333
    42       150         2   0.01333
    43       150         2   0.03946
    44       150         2   0.01333
    45       150         2   0.01333
    46       150         2   0.06488
    47       150         2   0.07732
    48       150         2   0.02649
    49       150         2   0.01333
    50       150         2   0.01333
    51       150         2   0.02649
    52       150         2   0.01333
    53       150         2   0.06488
    54       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.02415
  s.d.     0.000     0.000   0.01710
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.2908
Gene diversity           :  0.7140 +/-  0.0424

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 45

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 54
Number of usable nucleotide sites            : 54


Distance method : Pairwise difference
Mean number of pairwise differences           :     1.304251 +/-     0.819939

Average gene diversity over loci              :     0.024153 +/-     0.016807

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     1.916094
  S.D. Theta(Hom) :     0.419800

         Theta(k) :    21.427915
95 % confidence interval limits for theta(k) : [  14.772787,  30.751806 ]

         Theta(S) :     9.669597
    S.D. Theta(S) :     2.512285

        Theta(Pi) :     1.304251
   S.D. Theta(Pi) :     0.907594

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    3196    3061.735
       1    3706    3937.291
       2    2537    2556.065
       3    1255    1118.695
       4     384     372.040
       5      88     100.513
       6       9      23.041
       7       0       4.625
       8       0       0.833
       9       0       0.137
      10       0       0.021
      11       0       0.003
      12       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        1.304
Mismatch observed variance          :        1.325
Tau                                 :        1.457
Theta0                              :        0.000
Theta1                              :        7.893

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00006665
          P(Sim. Ssd >= Obs. Ssd): 1.00000000
    Harpending's Raggedness index: 0.03301364
        P(Sim. Rag. >= Obs. Rag.): 1.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       3196.0       3007.2       3177.4
          1.0       3706.0       3694.3       3698.0
          2.0       2537.0       2664.6       2522.4
          3.0       1255.0       1347.1       1190.1
          4.0        384.0        377.1        426.7
          5.0         88.0         70.7        123.1
          6.0          9.0         12.3         29.7
          7.0          0.0          1.5          6.2
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.457           0.162           2.137           1.247
                   Theta0:           0.000           0.000           1.142           0.193               
                   Theta1:           7.893           2.210       99999.000       70005.729
Mean No. of pairwise diff.           1.304           0.777           1.953           1.343
           No. pol. sites:              54              16              33          20.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       3196.0       1103.0       5841.0       3177.4
          1.0       3706.0       2653.0       4403.0       3698.0
          2.0       2537.0       1375.0       3990.0       2522.4
          3.0       1255.0        708.0       2616.0       1190.1
          4.0        384.0         77.0        854.0        426.7
          5.0         88.0          0.0        335.0        123.1
          6.0          9.0          0.0         91.0         29.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.457           0.162           2.137           1.247
                   Theta0:           0.000           0.000           1.142           0.193               
                   Theta1:           7.893           2.210       99999.000       70005.729
Mean No. of pairwise diff.           1.304           0.777           1.953           1.343
           No. pol. sites:              54              16              33          20.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       3196.0       1103.0       5841.0       3177.4
          1.0       3706.0       2653.0       4403.0       3698.0
          2.0       2537.0       1375.0       3990.0       2522.4
          3.0       1255.0        708.0       2616.0       1190.1
          4.0        384.0         77.0        854.0        426.7
          5.0         88.0          0.0        335.0        123.1
          6.0          9.0          0.0         91.0         29.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.457           0.162           2.137           1.247
                   Theta0:           0.000           0.000           1.142           0.193               
                   Theta1:           7.893           2.210       99999.000       70005.729
Mean No. of pairwise diff.           1.304           0.777           1.953           1.343
           No. pol. sites:              54              16              33          20.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       3196.0       1103.0       5841.0       3177.4
          1.0       3706.0       2653.0       4403.0       3698.0
          2.0       2537.0       1375.0       3990.0       2522.4
          3.0       1255.0        708.0       2616.0       1190.1
          4.0        384.0         77.0        854.0        426.7
          5.0         88.0          0.0        335.0        123.1
          6.0          9.0          0.0         91.0         29.7
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_29_26_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            54      54.000       0.000
  No. of usable loci            54      54.000       0.000
  No. of polym. loci            54      54.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_29_26_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.01333     0.01333     0.00000     0.01333
     3       0.02649     0.02649     0.00000     0.02649
     4       0.02649     0.02649     0.00000     0.02649
     5       0.02649     0.02649     0.00000     0.02649
     6       0.01333     0.01333     0.00000     0.01333
     7       0.03946     0.03946     0.00000     0.03946
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.02649     0.02649     0.00000     0.02649
    11       0.01333     0.01333     0.00000     0.01333
    12       0.01333     0.01333     0.00000     0.01333
    13       0.01333     0.01333     0.00000     0.01333
    14       0.01333     0.01333     0.00000     0.01333
    15       0.02649     0.02649     0.00000     0.02649
    16       0.01333     0.01333     0.00000     0.01333
    17       0.01333     0.01333     0.00000     0.01333
    18       0.03946     0.03946     0.00000     0.03946
    19       0.02649     0.02649     0.00000     0.02649
    20       0.01333     0.01333     0.00000     0.01333
    21       0.01333     0.01333     0.00000     0.01333
    22       0.02649     0.02649     0.00000     0.02649
    23       0.01333     0.01333     0.00000     0.01333
    24       0.01333     0.01333     0.00000     0.01333
    25       0.01333     0.01333     0.00000     0.01333
    26       0.07732     0.07732     0.00000     0.07732
    27       0.03946     0.03946     0.00000     0.03946
    28       0.01333     0.01333     0.00000     0.01333
    29       0.02649     0.02649     0.00000     0.02649
    30       0.01333     0.01333     0.00000     0.01333
    31       0.01333     0.01333     0.00000     0.01333
    32       0.02649     0.02649     0.00000     0.02649
    33       0.06488     0.06488     0.00000     0.06488
    34       0.02649     0.02649     0.00000     0.02649
    35       0.01333     0.01333     0.00000     0.01333
    36       0.01333     0.01333     0.00000     0.01333
    37       0.02649     0.02649     0.00000     0.02649
    38       0.01333     0.01333     0.00000     0.01333
    39       0.03946     0.03946     0.00000     0.03946
    40       0.01333     0.01333     0.00000     0.01333
    41       0.01333     0.01333     0.00000     0.01333
    42       0.01333     0.01333     0.00000     0.01333
    43       0.03946     0.03946     0.00000     0.03946
    44       0.01333     0.01333     0.00000     0.01333
    45       0.01333     0.01333     0.00000     0.01333
    46       0.06488     0.06488     0.00000     0.06488
    47       0.07732     0.07732     0.00000     0.07732
    48       0.02649     0.02649     0.00000     0.02649
    49       0.01333     0.01333     0.00000     0.01333
    50       0.01333     0.01333     0.00000     0.01333
    51       0.02649     0.02649     0.00000     0.02649
    52       0.01333     0.01333     0.00000     0.01333
    53       0.06488     0.06488     0.00000     0.06488
    54       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.02415     0.02415     0.00000     0.02415
  s.d.       0.01710     0.01710     0.00000     0.01710
--------------------------------------------------------
<A NAME="31_01_17at16_29_26_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_29_26_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      21.42791    21.42791     0.00000
            Theta_k_lower      14.77279    14.77279     0.00000
            Theta_k_upper      30.75181    30.75181     0.00000
                  Theta_H       1.91609     1.91609     0.00000
             s.d. Theta_H       0.41980     0.41980     0.00000
                  Theta_S       9.66960     9.66960     0.00000
             s.d. Theta_S       2.51228     2.51228     0.00000
                 Theta_pi       1.30425     1.30425     0.00000
            s.d. Theta_pi       0.90759     0.90759     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_29_26_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.45703     1.45703     0.00000
              Tau qt 2.5%       0.16211     0.16211     0.00000
                Tau qt 5%       0.16211     0.16211     0.00000
               Tau qt 95%       1.78711     1.78711     0.00000
             Tau qt 97.5%       1.78711     1.78711     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.74004     0.74004     0.00000
          Theta0 qt 97.5%       0.74004     0.74004     0.00000
                   Theta1       7.89336     7.89336     0.00000
           Theta1 qt 2.5%       2.20977     2.20977     0.00000
             Theta1 qt 5%       2.20977     2.20977     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00007     0.00007     0.00000
      Model (SSD) p-value       1.00000     1.00000     0.00000
         Raggedness index       0.03301     0.03301     0.00000
       Raggedness p-value       1.00000     1.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 74 (31/01/17 at 16:29:26))
Total computing time for this run : 0h 0m 0s 84 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

