
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_47_19"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 64 (31/01/17 at 14:47:19)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_47_19_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_47_19_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 54
No. of loci              : 76
No. of usable loci       : 76 loci with less than  5.00 % missing data
No. of polymorphic sites : 76


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.01333
     3       150         2   0.01333
     4       150         2   0.14819
     5       150         2   0.01333
     6       150         2   0.01333
     7       150         2   0.01333
     8       150         2   0.01333
     9       150         2   0.12528
    10       150         2   0.05226
    11       150         2   0.01333
    12       150         2   0.03946
    13       150         2   0.02649
    14       150         2   0.01333
    15       150         2   0.02649
    16       150         2   0.03946
    17       150         2   0.07732
    18       150         2   0.03946
    19       150         2   0.25199
    20       150         2   0.01333
    21       150         2   0.03946
    22       150         2   0.01333
    23       150         2   0.01333
    24       150         2   0.03946
    25       150         2   0.07732
    26       150         2   0.02649
    27       150         2   0.05226
    28       150         2   0.03946
    29       150         2   0.11356
    30       150         2   0.03946
    31       150         2   0.03946
    32       150         2   0.01333
    33       150         2   0.03946
    34       150         2   0.05226
    35       150         2   0.01333
    36       150         2   0.06488
    37       150         2   0.02649
    38       150         2   0.01333
    39       150         2   0.14819
    40       150         2   0.10166
    41       150         2   0.01333
    42       150         2   0.01333
    43       150         2   0.02649
    44       150         2   0.08957
    45       150         2   0.01333
    46       150         2   0.03946
    47       150         2   0.01333
    48       150         2   0.01333
    49       150         2   0.01333
    50       150         2   0.02649
    51       150         2   0.05226
    52       150         2   0.07732
    53       150         2   0.01333
    54       150         2   0.25199
    55       150         2   0.02649
    56       150         2   0.01333
    57       150         2   0.01333
    58       150         2   0.01333
    59       150         2   0.01333
    60       150         2   0.02649
    61       150         2   0.01333
    62       150         2   0.01333
    63       150         2   0.01333
    64       150         2   0.02649
    65       150         2   0.15937
    66       150         2   0.05226
    67       150         2   0.15937
    68       150         2   0.01333
    69       150         2   0.11356
    70       150         2   0.25199
    71       150         2   0.02649
    72       150         2   0.02649
    73       150         2   0.02649
    74       150         2   0.02649
    75       150         2   0.02649
    76       150         2   0.08957
-------------------------------------------------
  Mean   150.000     2.000   0.04905
  s.d.     0.000     0.000   0.05618
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0507
Gene diversity           :  0.9557 +/-  0.0087

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 54

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 76
Number of usable nucleotide sites            : 76


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.727875 +/-     1.892750

Average gene diversity over loci              :     0.049051 +/-     0.027567

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    19.916881
  S.D. Theta(Hom) :     4.395061

         Theta(k) :    29.820320
95 % confidence interval limits for theta(k) : [  20.964943,  42.104679 ]

         Theta(S) :    13.609063
    S.D. Theta(S) :     3.399804

        Theta(Pi) :     3.727875
   S.D. Theta(Pi) :     2.095091

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     495       0.000
       1    1092       0.000
       2    1362       0.000
       3    2003       0.000
       4    2343       0.000
       5    1813       0.000
       6    1239       0.000
       7     671       0.000
       8     147       0.000
       9      10       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.728
Mismatch observed variance          :        3.560
Tau                                 :        4.168
Theta0                              :        0.000
Theta1                              :       39.453

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00199933
          P(Sim. Ssd >= Obs. Ssd): 0.20000000
    Harpending's Raggedness index: 0.01747457
        P(Sim. Rag. >= Obs. Rag.): 0.30000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        495.0        508.8        428.1
          1.0       1092.0       1164.8        898.5
          2.0       1362.0       1716.5       1562.3
          3.0       2003.0       1891.5       2037.1
          4.0       2343.0       1834.2       2063.7
          5.0       1813.0       1473.2       1695.0
          6.0       1239.0       1095.4       1167.1
          7.0        671.0        719.2        691.0
          8.0        147.0        424.8        358.6
          9.0         10.0        198.0        165.6
         10.0          0.0         95.2         68.9
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.168           2.432           5.676           4.003
                   Theta0:           0.000           0.000           2.016           0.252               
                   Theta1:          39.453           9.958       99999.000       30056.402
Mean No. of pairwise diff.           3.728           2.562           5.235           3.877
           No. pol. sites:              76              62              87          74.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        495.0        237.0        901.0        428.1
          1.0       1092.0        456.0       2329.0        898.5
          2.0       1362.0        799.0       3349.0       1562.3
          3.0       2003.0       1220.0       2664.0       2037.1
          4.0       2343.0       1181.0       2252.0       2063.7
          5.0       1813.0        692.0       2020.0       1695.0
          6.0       1239.0        285.0       1801.0       1167.1
          7.0        671.0         41.0       1405.0        691.0
          8.0        147.0         24.0       1026.0        358.6
          9.0         10.0          1.0        561.0        165.6
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.168           2.432           5.676           4.003
                   Theta0:           0.000           0.000           2.016           0.252               
                   Theta1:          39.453           9.958       99999.000       30056.402
Mean No. of pairwise diff.           3.728           2.562           5.235           3.877
           No. pol. sites:              76              62              87          74.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        495.0        237.0        901.0        428.1
          1.0       1092.0        456.0       2329.0        898.5
          2.0       1362.0        799.0       3349.0       1562.3
          3.0       2003.0       1220.0       2664.0       2037.1
          4.0       2343.0       1181.0       2252.0       2063.7
          5.0       1813.0        692.0       2020.0       1695.0
          6.0       1239.0        285.0       1801.0       1167.1
          7.0        671.0         41.0       1405.0        691.0
          8.0        147.0         24.0       1026.0        358.6
          9.0         10.0          1.0        561.0        165.6
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.168           2.432           5.676           4.003
                   Theta0:           0.000           0.000           2.016           0.252               
                   Theta1:          39.453           9.958       99999.000       30056.402
Mean No. of pairwise diff.           3.728           2.562           5.235           3.877
           No. pol. sites:              76              62              87          74.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        495.0        237.0        901.0        428.1
          1.0       1092.0        456.0       2329.0        898.5
          2.0       1362.0        799.0       3349.0       1562.3
          3.0       2003.0       1220.0       2664.0       2037.1
          4.0       2343.0       1181.0       2252.0       2063.7
          5.0       1813.0        692.0       2020.0       1695.0
          6.0       1239.0        285.0       1801.0       1167.1
          7.0        671.0         41.0       1405.0        691.0
          8.0        147.0         24.0       1026.0        358.6
          9.0         10.0          1.0        561.0        165.6
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_47_19_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            76      76.000       0.000
  No. of usable loci            76      76.000       0.000
  No. of polym. loci            76      76.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_47_19_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.01333     0.01333     0.00000     0.01333
     3       0.01333     0.01333     0.00000     0.01333
     4       0.14819     0.14819     0.00000     0.14819
     5       0.01333     0.01333     0.00000     0.01333
     6       0.01333     0.01333     0.00000     0.01333
     7       0.01333     0.01333     0.00000     0.01333
     8       0.01333     0.01333     0.00000     0.01333
     9       0.12528     0.12528     0.00000     0.12528
    10       0.05226     0.05226     0.00000     0.05226
    11       0.01333     0.01333     0.00000     0.01333
    12       0.03946     0.03946     0.00000     0.03946
    13       0.02649     0.02649     0.00000     0.02649
    14       0.01333     0.01333     0.00000     0.01333
    15       0.02649     0.02649     0.00000     0.02649
    16       0.03946     0.03946     0.00000     0.03946
    17       0.07732     0.07732     0.00000     0.07732
    18       0.03946     0.03946     0.00000     0.03946
    19       0.25199     0.25199     0.00000     0.25199
    20       0.01333     0.01333     0.00000     0.01333
    21       0.03946     0.03946     0.00000     0.03946
    22       0.01333     0.01333     0.00000     0.01333
    23       0.01333     0.01333     0.00000     0.01333
    24       0.03946     0.03946     0.00000     0.03946
    25       0.07732     0.07732     0.00000     0.07732
    26       0.02649     0.02649     0.00000     0.02649
    27       0.05226     0.05226     0.00000     0.05226
    28       0.03946     0.03946     0.00000     0.03946
    29       0.11356     0.11356     0.00000     0.11356
    30       0.03946     0.03946     0.00000     0.03946
    31       0.03946     0.03946     0.00000     0.03946
    32       0.01333     0.01333     0.00000     0.01333
    33       0.03946     0.03946     0.00000     0.03946
    34       0.05226     0.05226     0.00000     0.05226
    35       0.01333     0.01333     0.00000     0.01333
    36       0.06488     0.06488     0.00000     0.06488
    37       0.02649     0.02649     0.00000     0.02649
    38       0.01333     0.01333     0.00000     0.01333
    39       0.14819     0.14819     0.00000     0.14819
    40       0.10166     0.10166     0.00000     0.10166
    41       0.01333     0.01333     0.00000     0.01333
    42       0.01333     0.01333     0.00000     0.01333
    43       0.02649     0.02649     0.00000     0.02649
    44       0.08957     0.08957     0.00000     0.08957
    45       0.01333     0.01333     0.00000     0.01333
    46       0.03946     0.03946     0.00000     0.03946
    47       0.01333     0.01333     0.00000     0.01333
    48       0.01333     0.01333     0.00000     0.01333
    49       0.01333     0.01333     0.00000     0.01333
    50       0.02649     0.02649     0.00000     0.02649
    51       0.05226     0.05226     0.00000     0.05226
    52       0.07732     0.07732     0.00000     0.07732
    53       0.01333     0.01333     0.00000     0.01333
    54       0.25199     0.25199     0.00000     0.25199
    55       0.02649     0.02649     0.00000     0.02649
    56       0.01333     0.01333     0.00000     0.01333
    57       0.01333     0.01333     0.00000     0.01333
    58       0.01333     0.01333     0.00000     0.01333
    59       0.01333     0.01333     0.00000     0.01333
    60       0.02649     0.02649     0.00000     0.02649
    61       0.01333     0.01333     0.00000     0.01333
    62       0.01333     0.01333     0.00000     0.01333
    63       0.01333     0.01333     0.00000     0.01333
    64       0.02649     0.02649     0.00000     0.02649
    65       0.15937     0.15937     0.00000     0.15937
    66       0.05226     0.05226     0.00000     0.05226
    67       0.15937     0.15937     0.00000     0.15937
    68       0.01333     0.01333     0.00000     0.01333
    69       0.11356     0.11356     0.00000     0.11356
    70       0.25199     0.25199     0.00000     0.25199
    71       0.02649     0.02649     0.00000     0.02649
    72       0.02649     0.02649     0.00000     0.02649
    73       0.02649     0.02649     0.00000     0.02649
    74       0.02649     0.02649     0.00000     0.02649
    75       0.02649     0.02649     0.00000     0.02649
    76       0.08957     0.08957     0.00000     0.08957
--------------------------------------------------------
  Mean       0.04905     0.04905     0.00000     0.04905
  s.d.       0.05618     0.05618     0.00000     0.05618
--------------------------------------------------------
<A NAME="31_01_17at14_47_19_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
    72             2       2.000       0.000           2
    73             2       2.000       0.000           2
    74             2       2.000       0.000           2
    75             2       2.000       0.000           2
    76             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_47_19_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      29.82032    29.82032     0.00000
            Theta_k_lower      20.96494    20.96494     0.00000
            Theta_k_upper      42.10468    42.10468     0.00000
                  Theta_H      19.91688    19.91688     0.00000
             s.d. Theta_H       4.39506     4.39506     0.00000
                  Theta_S      13.60906    13.60906     0.00000
             s.d. Theta_S       3.39980     3.39980     0.00000
                 Theta_pi       3.72787     3.72787     0.00000
            s.d. Theta_pi       2.09509     2.09509     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_47_19_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.16797     4.16797     0.00000
              Tau qt 2.5%       2.43164     2.43164     0.00000
                Tau qt 5%       2.43164     2.43164     0.00000
               Tau qt 95%       5.08008     5.08008     0.00000
             Tau qt 97.5%       5.08008     5.08008     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.32871     0.32871     0.00000
          Theta0 qt 97.5%       0.32871     0.32871     0.00000
                   Theta1      39.45312    39.45312     0.00000
           Theta1 qt 2.5%       9.95850     9.95850     0.00000
             Theta1 qt 5%       9.95850     9.95850     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00200     0.00200     0.00000
      Model (SSD) p-value       0.20000     0.20000     0.00000
         Raggedness index       0.01747     0.01747     0.00000
       Raggedness p-value       0.30000     0.30000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 64 (31/01/17 at 14:47:20))
Total computing time for this run : 0h 0m 0s 143 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

