
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_51_44"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 127 (31/01/17 at 14:51:44)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_51_44_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_51_44_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 40
No. of loci              : 18
No. of usable loci       : 18 loci with less than  5.00 % missing data
No. of polymorphic sites : 18


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.14819
     2       150         2   0.44286
     3       150         2   0.07732
     4       150         2   0.34550
     5       150         2   0.31400
     6       150         2   0.10166
     7       150         2   0.46004
     8       150         2   0.14819
     9       150         2   0.18121
    10       150         2   0.07732
    11       150         2   0.08957
    12       150         2   0.49441
    13       150         2   0.05226
    14       150         2   0.05226
    15       150         2   0.05226
    16       150         2   0.25199
    17       150         2   0.49611
    18       150         2   0.07732
-------------------------------------------------
  Mean   150.000     2.000   0.21458
  s.d.     0.000     0.000   0.16667
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0488
Gene diversity           :  0.9576 +/-  0.0058

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 40

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 18
Number of usable nucleotide sites            : 18


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.862461 +/-     1.951256

Average gene diversity over loci              :     0.214581 +/-     0.119992

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    20.903324
  S.D. Theta(Hom) :     3.195146

         Theta(k) :    17.513851
95 % confidence interval limits for theta(k) : [  11.895992,  25.454453 ]

         Theta(S) :     3.223199
    S.D. Theta(S) :     1.032890

        Theta(Pi) :     3.862461
   S.D. Theta(Pi) :     2.159852

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     474     238.178
       1     974     913.608
       2    1345    1754.912
       3    1960    2251.003
       4    2293    2169.348
       5    1895    1675.727
       6    1200    1080.921
       7     638     598.976
       8     266     291.131
       9     122     126.113
      10       8      49.309
      11       0      17.582
      12       0       5.766
      13       0       1.752
      14       0       0.497
      15       0       0.132
      16       0       0.033
      17       0       0.008
      18       0       0.002
      19       0       0.000
      20       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.862
Mismatch observed variance          :        3.894
Tau                                 :        4.221
Theta0                              :        0.009
Theta1                              :       43.594

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00128162
          P(Sim. Ssd >= Obs. Ssd): 0.70000000
    Harpending's Raggedness index: 0.01606501
        P(Sim. Rag. >= Obs. Rag.): 0.70000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        474.0        357.6        395.0
          1.0        974.0        625.2        852.3
          2.0       1345.0       1473.3       1513.7
          3.0       1960.0       1847.7       2008.9
          4.0       2293.0       2151.2       2068.3
          5.0       1895.0       1895.3       1725.3
          6.0       1200.0       1335.6       1206.2
          7.0        638.0        796.3        724.9
          8.0        266.0        401.0        381.9
          9.0        122.0        180.6        179.0
         10.0          8.0         69.8         75.6
         11.0          0.0         26.6         29.0
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.221           3.352           6.008           4.448
                   Theta0:           0.009           0.000           0.069           0.017               
                   Theta1:          43.594          25.491       99999.000       50029.602
Mean No. of pairwise diff.           3.862           3.309           5.335           4.187
           No. pol. sites:              18              63              92          80.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        474.0        203.0        596.0        395.0
          1.0        974.0        314.0       1123.0        852.3
          2.0       1345.0        613.0       3020.0       1513.7
          3.0       1960.0        828.0       2542.0       2008.9
          4.0       2293.0       1407.0       2662.0       2068.3
          5.0       1895.0       1218.0       2699.0       1725.3
          6.0       1200.0        672.0       1977.0       1206.2
          7.0        638.0        238.0       1396.0        724.9
          8.0        266.0         66.0        925.0        381.9
          9.0        122.0          9.0        721.0        179.0
         10.0          8.0          0.0        410.0         75.6
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.221           3.352           6.008           4.448
                   Theta0:           0.009           0.000           0.069           0.017               
                   Theta1:          43.594          25.491       99999.000       50029.602
Mean No. of pairwise diff.           3.862           3.309           5.335           4.187
           No. pol. sites:              18              63              92          80.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        474.0        203.0        596.0        395.0
          1.0        974.0        314.0       1123.0        852.3
          2.0       1345.0        613.0       3020.0       1513.7
          3.0       1960.0        828.0       2542.0       2008.9
          4.0       2293.0       1407.0       2662.0       2068.3
          5.0       1895.0       1218.0       2699.0       1725.3
          6.0       1200.0        672.0       1977.0       1206.2
          7.0        638.0        238.0       1396.0        724.9
          8.0        266.0         66.0        925.0        381.9
          9.0        122.0          9.0        721.0        179.0
         10.0          8.0          0.0        410.0         75.6
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.221           3.352           6.008           4.448
                   Theta0:           0.009           0.000           0.069           0.017               
                   Theta1:          43.594          25.491       99999.000       50029.602
Mean No. of pairwise diff.           3.862           3.309           5.335           4.187
           No. pol. sites:              18              63              92          80.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        474.0        203.0        596.0        395.0
          1.0        974.0        314.0       1123.0        852.3
          2.0       1345.0        613.0       3020.0       1513.7
          3.0       1960.0        828.0       2542.0       2008.9
          4.0       2293.0       1407.0       2662.0       2068.3
          5.0       1895.0       1218.0       2699.0       1725.3
          6.0       1200.0        672.0       1977.0       1206.2
          7.0        638.0        238.0       1396.0        724.9
          8.0        266.0         66.0        925.0        381.9
          9.0        122.0          9.0        721.0        179.0
         10.0          8.0          0.0        410.0         75.6
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_51_44_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            18      18.000       0.000
  No. of usable loci            18      18.000       0.000
  No. of polym. loci            18      18.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_51_44_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.14819     0.14819     0.00000     0.14819
     2       0.44286     0.44286     0.00000     0.44286
     3       0.07732     0.07732     0.00000     0.07732
     4       0.34550     0.34550     0.00000     0.34550
     5       0.31400     0.31400     0.00000     0.31400
     6       0.10166     0.10166     0.00000     0.10166
     7       0.46004     0.46004     0.00000     0.46004
     8       0.14819     0.14819     0.00000     0.14819
     9       0.18121     0.18121     0.00000     0.18121
    10       0.07732     0.07732     0.00000     0.07732
    11       0.08957     0.08957     0.00000     0.08957
    12       0.49441     0.49441     0.00000     0.49441
    13       0.05226     0.05226     0.00000     0.05226
    14       0.05226     0.05226     0.00000     0.05226
    15       0.05226     0.05226     0.00000     0.05226
    16       0.25199     0.25199     0.00000     0.25199
    17       0.49611     0.49611     0.00000     0.49611
    18       0.07732     0.07732     0.00000     0.07732
--------------------------------------------------------
  Mean       0.21458     0.21458     0.00000     0.21458
  s.d.       0.16667     0.16667     0.00000     0.16667
--------------------------------------------------------
<A NAME="31_01_17at14_51_44_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_51_44_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      17.51385    17.51385     0.00000
            Theta_k_lower      11.89599    11.89599     0.00000
            Theta_k_upper      25.45445    25.45445     0.00000
                  Theta_H      20.90332    20.90332     0.00000
             s.d. Theta_H       3.19515     3.19515     0.00000
                  Theta_S       3.22320     3.22320     0.00000
             s.d. Theta_S       1.03289     1.03289     0.00000
                 Theta_pi       3.86246     3.86246     0.00000
            s.d. Theta_pi       2.15985     2.15985     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_51_44_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.22070     4.22070     0.00000
              Tau qt 2.5%       3.35156     3.35156     0.00000
                Tau qt 5%       3.35156     3.35156     0.00000
               Tau qt 95%       5.36719     5.36719     0.00000
             Tau qt 97.5%       5.36719     5.36719     0.00000
                   Theta0       0.00879     0.00879     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.04922     0.04922     0.00000
          Theta0 qt 97.5%       0.04922     0.04922     0.00000
                   Theta1      43.59375    43.59375     0.00000
           Theta1 qt 2.5%      25.49072    25.49072     0.00000
             Theta1 qt 5%      25.49072    25.49072     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00128     0.00128     0.00000
      Model (SSD) p-value       0.70000     0.70000     0.00000
         Raggedness index       0.01607     0.01607     0.00000
       Raggedness p-value       0.70000     0.70000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 127 (31/01/17 at 14:51:44))
Total computing time for this run : 0h 0m 0s 143 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

