
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_47_46"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 239 (31/01/17 at 14:47:46)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_47_46_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_47_46_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 45
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.05226
     2       150         2   0.19186
     3       150         2   0.34550
     4       150         2   0.35293
     5       150         2   0.15937
     6       150         2   0.06488
     7       150         2   0.45602
     8       150         2   0.02649
     9       150         2   0.17038
    10       150         2   0.17038
    11       150         2   0.22273
    12       150         2   0.27964
    13       150         2   0.03946
    14       150         2   0.05226
    15       150         2   0.43812
    16       150         2   0.06488
    17       150         2   0.01333
    18       150         2   0.25199
    19       150         2   0.33011
    20       150         2   0.02649
-------------------------------------------------
  Mean   150.000     2.000   0.18545
  s.d.     0.000     0.000   0.14392
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0416
Gene diversity           :  0.9648 +/-  0.0044

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 45

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.709083 +/-     1.884577

Average gene diversity over loci              :     0.185454 +/-     0.104302

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    25.708747
  S.D. Theta(Hom) :     3.559628

         Theta(k) :    21.427915
95 % confidence interval limits for theta(k) : [  14.772787,  30.751806 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     3.709083
   S.D. Theta(Pi) :     2.086044

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     393       0.000
       1     772       0.000
       2    1746       0.000
       3    2571       0.000
       4    2212       0.000
       5    1483       0.000
       6    1194       0.000
       7     481       0.000
       8     247       0.000
       9      60       0.000
      10      14       0.000
      11       2       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.709
Mismatch observed variance          :        3.444
Tau                                 :        3.775
Theta0                              :        0.005
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00159726
          P(Sim. Ssd >= Obs. Ssd): 0.10000000
    Harpending's Raggedness index: 0.02496098
        P(Sim. Rag. >= Obs. Rag.): 0.10000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        393.0        314.1        255.0
          1.0        772.0        960.2        963.7
          2.0       1746.0       1849.4       1821.5
          3.0       2571.0       2346.0       2295.5
          4.0       2212.0       2154.6       2169.6
          5.0       1483.0       1624.0       1640.5
          6.0       1194.0        994.6       1033.7
          7.0        481.0        522.9        558.3
          8.0        247.0        245.8        263.8
          9.0         60.0        101.6        110.8
         10.0         14.0         35.1         41.9
         11.0          2.0         14.9         14.4
         12.0          0.0          7.3          4.5
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.775           3.266           4.090           3.722
                   Theta0:           0.005           0.000           0.023           0.004               
                   Theta1:       99999.000          38.141       99999.000       70061.188
Mean No. of pairwise diff.           3.709           3.285           4.030           3.723
           No. pol. sites:              20             184             212         201.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        393.0        153.0        703.0        255.0
          1.0        772.0        706.0       1347.0        963.7
          2.0       1746.0       1500.0       2217.0       1821.5
          3.0       2571.0       2093.0       2830.0       2295.5
          4.0       2212.0       1936.0       2357.0       2169.6
          5.0       1483.0       1369.0       1904.0       1640.5
          6.0       1194.0        697.0       1233.0       1033.7
          7.0        481.0        333.0        691.0        558.3
          8.0        247.0        139.0        379.0        263.8
          9.0         60.0         42.0        187.0        110.8
         10.0         14.0          6.0         74.0         41.9
         11.0          2.0          0.0         52.0         14.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.775           3.266           4.090           3.722
                   Theta0:           0.005           0.000           0.023           0.004               
                   Theta1:       99999.000          38.141       99999.000       70061.188
Mean No. of pairwise diff.           3.709           3.285           4.030           3.723
           No. pol. sites:              20             184             212         201.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        393.0        153.0        703.0        255.0
          1.0        772.0        706.0       1347.0        963.7
          2.0       1746.0       1500.0       2217.0       1821.5
          3.0       2571.0       2093.0       2830.0       2295.5
          4.0       2212.0       1936.0       2357.0       2169.6
          5.0       1483.0       1369.0       1904.0       1640.5
          6.0       1194.0        697.0       1233.0       1033.7
          7.0        481.0        333.0        691.0        558.3
          8.0        247.0        139.0        379.0        263.8
          9.0         60.0         42.0        187.0        110.8
         10.0         14.0          6.0         74.0         41.9
         11.0          2.0          0.0         52.0         14.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.775           3.266           4.090           3.722
                   Theta0:           0.005           0.000           0.023           0.004               
                   Theta1:       99999.000          38.141       99999.000       70061.188
Mean No. of pairwise diff.           3.709           3.285           4.030           3.723
           No. pol. sites:              20             184             212         201.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        393.0        153.0        703.0        255.0
          1.0        772.0        706.0       1347.0        963.7
          2.0       1746.0       1500.0       2217.0       1821.5
          3.0       2571.0       2093.0       2830.0       2295.5
          4.0       2212.0       1936.0       2357.0       2169.6
          5.0       1483.0       1369.0       1904.0       1640.5
          6.0       1194.0        697.0       1233.0       1033.7
          7.0        481.0        333.0        691.0        558.3
          8.0        247.0        139.0        379.0        263.8
          9.0         60.0         42.0        187.0        110.8
         10.0         14.0          6.0         74.0         41.9
         11.0          2.0          0.0         52.0         14.4
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_47_46_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_47_46_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.05226     0.05226     0.00000     0.05226
     2       0.19186     0.19186     0.00000     0.19186
     3       0.34550     0.34550     0.00000     0.34550
     4       0.35293     0.35293     0.00000     0.35293
     5       0.15937     0.15937     0.00000     0.15937
     6       0.06488     0.06488     0.00000     0.06488
     7       0.45602     0.45602     0.00000     0.45602
     8       0.02649     0.02649     0.00000     0.02649
     9       0.17038     0.17038     0.00000     0.17038
    10       0.17038     0.17038     0.00000     0.17038
    11       0.22273     0.22273     0.00000     0.22273
    12       0.27964     0.27964     0.00000     0.27964
    13       0.03946     0.03946     0.00000     0.03946
    14       0.05226     0.05226     0.00000     0.05226
    15       0.43812     0.43812     0.00000     0.43812
    16       0.06488     0.06488     0.00000     0.06488
    17       0.01333     0.01333     0.00000     0.01333
    18       0.25199     0.25199     0.00000     0.25199
    19       0.33011     0.33011     0.00000     0.33011
    20       0.02649     0.02649     0.00000     0.02649
--------------------------------------------------------
  Mean       0.18545     0.18545     0.00000     0.18545
  s.d.       0.14392     0.14392     0.00000     0.14392
--------------------------------------------------------
<A NAME="31_01_17at14_47_46_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_47_46_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      21.42791    21.42791     0.00000
            Theta_k_lower      14.77279    14.77279     0.00000
            Theta_k_upper      30.75181    30.75181     0.00000
                  Theta_H      25.70875    25.70875     0.00000
             s.d. Theta_H       3.55963     3.55963     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       3.70908     3.70908     0.00000
            s.d. Theta_pi       2.08604     2.08604     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_47_46_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.77539     3.77539     0.00000
              Tau qt 2.5%       3.26562     3.26562     0.00000
                Tau qt 5%       3.26562     3.26562     0.00000
               Tau qt 95%       4.01172     4.01172     0.00000
             Tau qt 97.5%       4.01172     4.01172     0.00000
                   Theta0       0.00527     0.00527     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00703     0.00703     0.00000
          Theta0 qt 97.5%       0.00703     0.00703     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      38.14062    38.14062     0.00000
             Theta1 qt 5%      38.14062    38.14062     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00160     0.00160     0.00000
      Model (SSD) p-value       0.10000     0.10000     0.00000
         Raggedness index       0.02496     0.02496     0.00000
       Raggedness p-value       0.10000     0.10000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 239 (31/01/17 at 14:47:46))
Total computing time for this run : 0h 0m 0s 121 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

