
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_29_28"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 92 (31/01/17 at 16:29:28)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_29_28_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_29_28_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 53
No. of loci              : 66
No. of usable loci       : 66 loci with less than  5.00 % missing data
No. of polymorphic sites : 66


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.03946
     2       150         2   0.01333
     3       150         2   0.05226
     4       150         2   0.02649
     5       150         2   0.01333
     6       150         2   0.01333
     7       150         2   0.01333
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.01333
    11       150         2   0.07732
    12       150         2   0.01333
    13       150         2   0.02649
    14       150         2   0.01333
    15       150         2   0.01333
    16       150         2   0.02649
    17       150         2   0.02649
    18       150         2   0.01333
    19       150         2   0.01333
    20       150         2   0.02649
    21       150         2   0.03946
    22       150         2   0.01333
    23       150         2   0.03946
    24       150         2   0.02649
    25       150         2   0.02649
    26       150         2   0.01333
    27       150         2   0.01333
    28       150         2   0.01333
    29       150         2   0.01333
    30       150         2   0.02649
    31       150         2   0.01333
    32       150         2   0.01333
    33       150         2   0.01333
    34       150         2   0.01333
    35       150         2   0.02649
    36       150         2   0.01333
    37       150         2   0.01333
    38       150         2   0.01333
    39       150         2   0.01333
    40       150         2   0.01333
    41       150         2   0.01333
    42       150         2   0.01333
    43       150         2   0.01333
    44       150         2   0.01333
    45       150         2   0.01333
    46       150         2   0.01333
    47       150         2   0.01333
    48       150         2   0.01333
    49       150         2   0.02649
    50       150         2   0.01333
    51       150         2   0.02649
    52       150         2   0.01333
    53       150         2   0.01333
    54       150         2   0.01333
    55       150         2   0.01333
    56       150         2   0.02649
    57       150         2   0.01333
    58       150         2   0.01333
    59       150         2   0.02649
    60       150         2   0.02649
    61       150         2   0.01333
    62       150         2   0.01333
    63       150         2   0.01333
    64       150         2   0.05226
    65       150         2   0.01333
    66       150         2   0.03946
-------------------------------------------------
  Mean   150.000     2.000   0.01986
  s.d.     0.000     0.000   0.01207
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.2825
Gene diversity           :  0.7223 +/-  0.0421

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 53

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 66
Number of usable nucleotide sites            : 66


Distance method : Pairwise difference
Mean number of pairwise differences           :     1.310515 +/-     0.822823

Average gene diversity over loci              :     0.019856 +/-     0.013800

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     2.001100
  S.D. Theta(Hom) :     0.444353

         Theta(k) :    28.791891
95 % confidence interval limits for theta(k) : [  20.205559,  40.712272 ]

         Theta(S) :    11.818397
    S.D. Theta(S) :     2.996930

        Theta(Pi) :     1.310515
   S.D. Theta(Pi) :     0.910786

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    3103    3043.445
       1    3807    3931.282
       2    2616    2563.998
       3    1138    1127.593
       4     388     376.904
       5     101     102.375
       6      19      23.603
       7       3       4.767
       8       0       0.864
       9       0       0.143
      10       0       0.022
      11       0       0.003
      12       0       0.000
      13       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        1.311
Mismatch observed variance          :        1.332
Tau                                 :        1.383
Theta0                              :        0.000
Theta1                              :       15.286

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00003530
          P(Sim. Ssd >= Obs. Ssd): 1.00000000
    Harpending's Raggedness index: 0.03803984
        P(Sim. Rag. >= Obs. Rag.): 0.90000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       3103.0       3702.5       3089.9
          1.0       3807.0       3846.0       3820.4
          2.0       2616.0       2310.9       2560.1
          3.0       1138.0        896.7       1164.0
          4.0        388.0        333.1        399.4
          5.0        101.0         65.5        110.0
          6.0         19.0         19.2         25.3
          7.0          3.0          1.1          5.0
          8.0          0.0          0.0          0.9
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.383           0.322           2.051           1.202
                   Theta0:           0.000           0.000           0.334           0.058               
                   Theta1:          15.286           2.173       99999.000       60000.959
Mean No. of pairwise diff.           1.311           0.573           1.512           1.158
           No. pol. sites:              66              17              36          28.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       3103.0       1763.0       6367.0       3089.9
          1.0       3807.0       2792.0       4563.0       3820.4
          2.0       2616.0       1111.0       3500.0       2560.1
          3.0       1138.0        178.0       1454.0       1164.0
          4.0        388.0         13.0        713.0        399.4
          5.0        101.0          0.0        173.0        110.0
          6.0         19.0          0.0        120.0         25.3
          7.0          3.0          0.0          6.0          5.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.383           0.322           2.051           1.202
                   Theta0:           0.000           0.000           0.334           0.058               
                   Theta1:          15.286           2.173       99999.000       60000.959
Mean No. of pairwise diff.           1.311           0.573           1.512           1.158
           No. pol. sites:              66              17              36          28.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       3103.0       1763.0       6367.0       3089.9
          1.0       3807.0       2792.0       4563.0       3820.4
          2.0       2616.0       1111.0       3500.0       2560.1
          3.0       1138.0        178.0       1454.0       1164.0
          4.0        388.0         13.0        713.0        399.4
          5.0        101.0          0.0        173.0        110.0
          6.0         19.0          0.0        120.0         25.3
          7.0          3.0          0.0          6.0          5.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.383           0.322           2.051           1.202
                   Theta0:           0.000           0.000           0.334           0.058               
                   Theta1:          15.286           2.173       99999.000       60000.959
Mean No. of pairwise diff.           1.311           0.573           1.512           1.158
           No. pol. sites:              66              17              36          28.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       3103.0       1763.0       6367.0       3089.9
          1.0       3807.0       2792.0       4563.0       3820.4
          2.0       2616.0       1111.0       3500.0       2560.1
          3.0       1138.0        178.0       1454.0       1164.0
          4.0        388.0         13.0        713.0        399.4
          5.0        101.0          0.0        173.0        110.0
          6.0         19.0          0.0        120.0         25.3
          7.0          3.0          0.0          6.0          5.0
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_29_28_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            66      66.000       0.000
  No. of usable loci            66      66.000       0.000
  No. of polym. loci            66      66.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_29_28_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.03946     0.03946     0.00000     0.03946
     2       0.01333     0.01333     0.00000     0.01333
     3       0.05226     0.05226     0.00000     0.05226
     4       0.02649     0.02649     0.00000     0.02649
     5       0.01333     0.01333     0.00000     0.01333
     6       0.01333     0.01333     0.00000     0.01333
     7       0.01333     0.01333     0.00000     0.01333
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.01333     0.01333     0.00000     0.01333
    11       0.07732     0.07732     0.00000     0.07732
    12       0.01333     0.01333     0.00000     0.01333
    13       0.02649     0.02649     0.00000     0.02649
    14       0.01333     0.01333     0.00000     0.01333
    15       0.01333     0.01333     0.00000     0.01333
    16       0.02649     0.02649     0.00000     0.02649
    17       0.02649     0.02649     0.00000     0.02649
    18       0.01333     0.01333     0.00000     0.01333
    19       0.01333     0.01333     0.00000     0.01333
    20       0.02649     0.02649     0.00000     0.02649
    21       0.03946     0.03946     0.00000     0.03946
    22       0.01333     0.01333     0.00000     0.01333
    23       0.03946     0.03946     0.00000     0.03946
    24       0.02649     0.02649     0.00000     0.02649
    25       0.02649     0.02649     0.00000     0.02649
    26       0.01333     0.01333     0.00000     0.01333
    27       0.01333     0.01333     0.00000     0.01333
    28       0.01333     0.01333     0.00000     0.01333
    29       0.01333     0.01333     0.00000     0.01333
    30       0.02649     0.02649     0.00000     0.02649
    31       0.01333     0.01333     0.00000     0.01333
    32       0.01333     0.01333     0.00000     0.01333
    33       0.01333     0.01333     0.00000     0.01333
    34       0.01333     0.01333     0.00000     0.01333
    35       0.02649     0.02649     0.00000     0.02649
    36       0.01333     0.01333     0.00000     0.01333
    37       0.01333     0.01333     0.00000     0.01333
    38       0.01333     0.01333     0.00000     0.01333
    39       0.01333     0.01333     0.00000     0.01333
    40       0.01333     0.01333     0.00000     0.01333
    41       0.01333     0.01333     0.00000     0.01333
    42       0.01333     0.01333     0.00000     0.01333
    43       0.01333     0.01333     0.00000     0.01333
    44       0.01333     0.01333     0.00000     0.01333
    45       0.01333     0.01333     0.00000     0.01333
    46       0.01333     0.01333     0.00000     0.01333
    47       0.01333     0.01333     0.00000     0.01333
    48       0.01333     0.01333     0.00000     0.01333
    49       0.02649     0.02649     0.00000     0.02649
    50       0.01333     0.01333     0.00000     0.01333
    51       0.02649     0.02649     0.00000     0.02649
    52       0.01333     0.01333     0.00000     0.01333
    53       0.01333     0.01333     0.00000     0.01333
    54       0.01333     0.01333     0.00000     0.01333
    55       0.01333     0.01333     0.00000     0.01333
    56       0.02649     0.02649     0.00000     0.02649
    57       0.01333     0.01333     0.00000     0.01333
    58       0.01333     0.01333     0.00000     0.01333
    59       0.02649     0.02649     0.00000     0.02649
    60       0.02649     0.02649     0.00000     0.02649
    61       0.01333     0.01333     0.00000     0.01333
    62       0.01333     0.01333     0.00000     0.01333
    63       0.01333     0.01333     0.00000     0.01333
    64       0.05226     0.05226     0.00000     0.05226
    65       0.01333     0.01333     0.00000     0.01333
    66       0.03946     0.03946     0.00000     0.03946
--------------------------------------------------------
  Mean       0.01986     0.01986     0.00000     0.01986
  s.d.       0.01207     0.01207     0.00000     0.01207
--------------------------------------------------------
<A NAME="31_01_17at16_29_28_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_29_28_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      28.79189    28.79189     0.00000
            Theta_k_lower      20.20556    20.20556     0.00000
            Theta_k_upper      40.71227    40.71227     0.00000
                  Theta_H       2.00110     2.00110     0.00000
             s.d. Theta_H       0.44435     0.44435     0.00000
                  Theta_S      11.81840    11.81840     0.00000
             s.d. Theta_S       2.99693     2.99693     0.00000
                 Theta_pi       1.31051     1.31051     0.00000
            s.d. Theta_pi       0.91079     0.91079     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_29_28_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.38281     1.38281     0.00000
              Tau qt 2.5%       0.32227     0.32227     0.00000
                Tau qt 5%       0.32227     0.32227     0.00000
               Tau qt 95%       1.60156     1.60156     0.00000
             Tau qt 97.5%       1.60156     1.60156     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.22500     0.22500     0.00000
          Theta0 qt 97.5%       0.22500     0.22500     0.00000
                   Theta1      15.28594    15.28594     0.00000
           Theta1 qt 2.5%       2.17314     2.17314     0.00000
             Theta1 qt 5%       2.17314     2.17314     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00004     0.00004     0.00000
      Model (SSD) p-value       1.00000     1.00000     0.00000
         Raggedness index       0.03804     0.03804     0.00000
       Raggedness p-value       0.90000     0.90000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 92 (31/01/17 at 16:29:28))
Total computing time for this run : 0h 0m 0s 101 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

