
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_48_08"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 389 (31/01/17 at 14:48:08)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_48_08_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_48_08_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 42
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.30568
     3       150         2   0.36725
     4       150         2   0.01333
     5       150         2   0.02649
     6       150         2   0.02649
     7       150         2   0.20233
     8       150         2   0.49441
     9       150         2   0.49047
    10       150         2   0.30568
    11       150         2   0.05226
    12       150         2   0.21262
    13       150         2   0.10166
    14       150         2   0.01333
    15       150         2   0.25199
    16       150         2   0.15937
    17       150         2   0.37414
    18       150         2   0.07732
    19       150         2   0.10166
    20       150         2   0.13682
-------------------------------------------------
  Mean   150.000     2.000   0.18633
  s.d.     0.000     0.000   0.15852
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0732
Gene diversity           :  0.9331 +/-  0.0101

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 42

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.726622 +/-     1.892205

Average gene diversity over loci              :     0.186331 +/-     0.104724

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    12.440032
  S.D. Theta(Hom) :     2.191334

         Theta(k) :    19.021744
95 % confidence interval limits for theta(k) : [  13.002730,  27.496429 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     3.726622
   S.D. Theta(Pi) :     2.094488

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     748     309.534
       1    1048    1078.727
       2    1067    1902.669
       3    2011    2269.574
       4    2539    2065.149
       5    1944    1533.882
       6     850     972.384
       7     466     543.555
       8     344     274.882
       9     140     128.459
      10      18      56.487
      11       0      23.736
      12       0       9.655
      13       0       3.842
      14       0       1.507
      15       0       0.586
      16       0       0.227
      17       0       0.088
      18       0       0.034
      19       0       0.013
      20       0       0.005


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.727
Mismatch observed variance          :        4.120
Tau                                 :        4.219
Theta0                              :        0.002
Theta1                              :       27.969

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00637412
          P(Sim. Ssd >= Obs. Ssd): 0.10000000
    Harpending's Raggedness index: 0.02426562
        P(Sim. Rag. >= Obs. Rag.): 0.20000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        748.0        513.6        522.1
          1.0       1048.0        929.5        943.0
          2.0       1067.0       1285.2       1549.2
          3.0       2011.0       1815.1       1989.6
          4.0       2539.0       1820.4       2015.0
          5.0       1944.0       1594.0       1664.6
          6.0        850.0       1332.5       1156.0
          7.0        466.0        898.0        691.2
          8.0        344.0        497.3        362.6
          9.0        140.0        289.8        169.3
         10.0         18.0        127.4         71.2
         11.0          0.0         51.3         27.3
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.219           2.180           6.211           4.155
                   Theta0:           0.002           0.000           1.760           0.438               
                   Theta1:          27.969          18.760       99999.000       10057.671
Mean No. of pairwise diff.           3.727           2.934           5.401           4.220
           No. pol. sites:              20              60              84          68.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        748.0        215.0        998.0        522.1
          1.0       1048.0        264.0       1986.0        943.0
          2.0       1067.0        474.0       2444.0       1549.2
          3.0       2011.0        840.0       2367.0       1989.6
          4.0       2539.0       1176.0       2610.0       2015.0
          5.0       1944.0        792.0       2418.0       1664.6
          6.0        850.0        663.0       2256.0       1156.0
          7.0        466.0        313.0       1783.0        691.2
          8.0        344.0        115.0       1089.0        362.6
          9.0        140.0         45.0        637.0        169.3
         10.0         18.0          7.0        295.0         71.2
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.219           2.180           6.211           4.155
                   Theta0:           0.002           0.000           1.760           0.438               
                   Theta1:          27.969          18.760       99999.000       10057.671
Mean No. of pairwise diff.           3.727           2.934           5.401           4.220
           No. pol. sites:              20              60              84          68.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        748.0        215.0        998.0        522.1
          1.0       1048.0        264.0       1986.0        943.0
          2.0       1067.0        474.0       2444.0       1549.2
          3.0       2011.0        840.0       2367.0       1989.6
          4.0       2539.0       1176.0       2610.0       2015.0
          5.0       1944.0        792.0       2418.0       1664.6
          6.0        850.0        663.0       2256.0       1156.0
          7.0        466.0        313.0       1783.0        691.2
          8.0        344.0        115.0       1089.0        362.6
          9.0        140.0         45.0        637.0        169.3
         10.0         18.0          7.0        295.0         71.2
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.219           2.180           6.211           4.155
                   Theta0:           0.002           0.000           1.760           0.438               
                   Theta1:          27.969          18.760       99999.000       10057.671
Mean No. of pairwise diff.           3.727           2.934           5.401           4.220
           No. pol. sites:              20              60              84          68.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        748.0        215.0        998.0        522.1
          1.0       1048.0        264.0       1986.0        943.0
          2.0       1067.0        474.0       2444.0       1549.2
          3.0       2011.0        840.0       2367.0       1989.6
          4.0       2539.0       1176.0       2610.0       2015.0
          5.0       1944.0        792.0       2418.0       1664.6
          6.0        850.0        663.0       2256.0       1156.0
          7.0        466.0        313.0       1783.0        691.2
          8.0        344.0        115.0       1089.0        362.6
          9.0        140.0         45.0        637.0        169.3
         10.0         18.0          7.0        295.0         71.2
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_48_08_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_48_08_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.30568     0.30568     0.00000     0.30568
     3       0.36725     0.36725     0.00000     0.36725
     4       0.01333     0.01333     0.00000     0.01333
     5       0.02649     0.02649     0.00000     0.02649
     6       0.02649     0.02649     0.00000     0.02649
     7       0.20233     0.20233     0.00000     0.20233
     8       0.49441     0.49441     0.00000     0.49441
     9       0.49047     0.49047     0.00000     0.49047
    10       0.30568     0.30568     0.00000     0.30568
    11       0.05226     0.05226     0.00000     0.05226
    12       0.21262     0.21262     0.00000     0.21262
    13       0.10166     0.10166     0.00000     0.10166
    14       0.01333     0.01333     0.00000     0.01333
    15       0.25199     0.25199     0.00000     0.25199
    16       0.15937     0.15937     0.00000     0.15937
    17       0.37414     0.37414     0.00000     0.37414
    18       0.07732     0.07732     0.00000     0.07732
    19       0.10166     0.10166     0.00000     0.10166
    20       0.13682     0.13682     0.00000     0.13682
--------------------------------------------------------
  Mean       0.18633     0.18633     0.00000     0.18633
  s.d.       0.15852     0.15852     0.00000     0.15852
--------------------------------------------------------
<A NAME="31_01_17at14_48_08_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_48_08_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      19.02174    19.02174     0.00000
            Theta_k_lower      13.00273    13.00273     0.00000
            Theta_k_upper      27.49643    27.49643     0.00000
                  Theta_H      12.44003    12.44003     0.00000
             s.d. Theta_H       2.19133     2.19133     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       3.72662     3.72662     0.00000
            s.d. Theta_pi       2.09449     2.09449     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_48_08_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.21875     4.21875     0.00000
              Tau qt 2.5%       2.17969     2.17969     0.00000
                Tau qt 5%       2.17969     2.17969     0.00000
               Tau qt 95%       5.01367     5.01367     0.00000
             Tau qt 97.5%       5.01367     5.01367     0.00000
                   Theta0       0.00176     0.00176     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.49238     1.49238     0.00000
          Theta0 qt 97.5%       1.49238     1.49238     0.00000
                   Theta1      27.96875    27.96875     0.00000
           Theta1 qt 2.5%      18.75977    18.75977     0.00000
             Theta1 qt 5%      18.75977    18.75977     0.00000
            Theta1 qt 95%     283.90625   283.90625     0.00000
          Theta1 qt 97.5%     283.90625   283.90625     0.00000
                      SSD       0.00637     0.00637     0.00000
      Model (SSD) p-value       0.10000     0.10000     0.00000
         Raggedness index       0.02427     0.02427     0.00000
       Raggedness p-value       0.20000     0.20000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 389 (31/01/17 at 14:48:08))
Total computing time for this run : 0h 0m 0s 114 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

