
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_37"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 368 (31/01/17 at 14:44:37)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_37_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_37_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 55
No. of loci              : 70
No. of usable loci       : 70 loci with less than  5.00 % missing data
No. of polymorphic sites : 70


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.02649
     3       150         2   0.02649
     4       150         2   0.10166
     5       150         2   0.07732
     6       150         2   0.05226
     7       150         2   0.01333
     8       150         2   0.02649
     9       150         2   0.01333
    10       150         2   0.01333
    11       150         2   0.02649
    12       150         2   0.33790
    13       150         2   0.01333
    14       150         2   0.01333
    15       150         2   0.02649
    16       150         2   0.08957
    17       150         2   0.01333
    18       150         2   0.01333
    19       150         2   0.01333
    20       150         2   0.06488
    21       150         2   0.01333
    22       150         2   0.01333
    23       150         2   0.03946
    24       150         2   0.02649
    25       150         2   0.03946
    26       150         2   0.01333
    27       150         2   0.10166
    28       150         2   0.05226
    29       150         2   0.02649
    30       150         2   0.03946
    31       150         2   0.03946
    32       150         2   0.06488
    33       150         2   0.01333
    34       150         2   0.03946
    35       150         2   0.01333
    36       150         2   0.01333
    37       150         2   0.01333
    38       150         2   0.10166
    39       150         2   0.06488
    40       150         2   0.13682
    41       150         2   0.06488
    42       150         2   0.01333
    43       150         2   0.12528
    44       150         2   0.01333
    45       150         2   0.02649
    46       150         2   0.03946
    47       150         2   0.01333
    48       150         2   0.02649
    49       150         2   0.01333
    50       150         2   0.10166
    51       150         2   0.10166
    52       150         2   0.02649
    53       150         2   0.05226
    54       150         2   0.02649
    55       150         2   0.07732
    56       150         2   0.05226
    57       150         2   0.07732
    58       150         2   0.01333
    59       150         2   0.05226
    60       150         2   0.01333
    61       150         2   0.01333
    62       150         2   0.02649
    63       150         2   0.01333
    64       150         2   0.10166
    65       150         2   0.03946
    66       150         2   0.02649
    67       150         2   0.02649
    68       150         2   0.01333
    69       150         2   0.01333
    70       150         2   0.07732
-------------------------------------------------
  Mean   150.000     2.000   0.04462
  s.d.     0.000     0.000   0.04800
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0542
Gene diversity           :  0.9521 +/-  0.0095

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 55

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 70
Number of usable nucleotide sites            : 70


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.123311 +/-     1.629232

Average gene diversity over loci              :     0.044619 +/-     0.025763

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    18.254942
  S.D. Theta(Hom) :     4.065800

         Theta(k) :    30.875075
95 % confidence interval limits for theta(k) : [  21.743737,  43.533340 ]

         Theta(S) :    12.534663
    S.D. Theta(S) :     3.158166

        Theta(Pi) :     3.123311
   S.D. Theta(Pi) :     1.803403

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     535       0.000
       1    1567       0.000
       2    2286       0.000
       3    2354       0.000
       4    1974       0.000
       5    1396       0.000
       6     696       0.000
       7     297       0.000
       8      59       0.000
       9      11       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.123
Mismatch observed variance          :        3.011
Tau                                 :        3.109
Theta0                              :        0.147
Theta1                              :       71.406

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00030671
          P(Sim. Ssd >= Obs. Ssd): 1.00000000
    Harpending's Raggedness index: 0.02220811
        P(Sim. Rag. >= Obs. Rag.): 0.70000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        535.0        666.5        563.9
          1.0       1567.0       1584.8       1472.5
          2.0       2286.0       2390.5       2276.3
          3.0       2354.0       2526.3       2422.9
          4.0       1974.0       1980.3       1955.5
          5.0       1396.0       1166.2       1269.6
          6.0        696.0        553.3        689.5
          7.0        297.0        216.5        322.0
          8.0         59.0         68.7        132.0
          9.0         11.0         17.7         48.2
         10.0          0.0          3.2         15.9
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.109           2.289           3.938           3.063
                   Theta0:           0.147           0.000           0.403           0.045               
                   Theta1:          71.406          18.262       99999.000       60010.865
Mean No. of pairwise diff.           3.123           2.303           3.674           2.978
           No. pol. sites:              70              76              99          87.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        535.0        370.0       1324.0        563.9
          1.0       1567.0       1103.0       2352.0       1472.5
          2.0       2286.0       1633.0       2817.0       2276.3
          3.0       2354.0       2153.0       2813.0       2422.9
          4.0       1974.0       1455.0       2591.0       1955.5
          5.0       1396.0        675.0       1704.0       1269.6
          6.0        696.0        202.0       1044.0        689.5
          7.0        297.0         41.0        590.0        322.0
          8.0         59.0          2.0        226.0        132.0
          9.0         11.0          0.0         76.0         48.2
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.109           2.289           3.938           3.063
                   Theta0:           0.147           0.000           0.403           0.045               
                   Theta1:          71.406          18.262       99999.000       60010.865
Mean No. of pairwise diff.           3.123           2.303           3.674           2.978
           No. pol. sites:              70              76              99          87.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        535.0        370.0       1324.0        563.9
          1.0       1567.0       1103.0       2352.0       1472.5
          2.0       2286.0       1633.0       2817.0       2276.3
          3.0       2354.0       2153.0       2813.0       2422.9
          4.0       1974.0       1455.0       2591.0       1955.5
          5.0       1396.0        675.0       1704.0       1269.6
          6.0        696.0        202.0       1044.0        689.5
          7.0        297.0         41.0        590.0        322.0
          8.0         59.0          2.0        226.0        132.0
          9.0         11.0          0.0         76.0         48.2
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.109           2.289           3.938           3.063
                   Theta0:           0.147           0.000           0.403           0.045               
                   Theta1:          71.406          18.262       99999.000       60010.865
Mean No. of pairwise diff.           3.123           2.303           3.674           2.978
           No. pol. sites:              70              76              99          87.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        535.0        370.0       1324.0        563.9
          1.0       1567.0       1103.0       2352.0       1472.5
          2.0       2286.0       1633.0       2817.0       2276.3
          3.0       2354.0       2153.0       2813.0       2422.9
          4.0       1974.0       1455.0       2591.0       1955.5
          5.0       1396.0        675.0       1704.0       1269.6
          6.0        696.0        202.0       1044.0        689.5
          7.0        297.0         41.0        590.0        322.0
          8.0         59.0          2.0        226.0        132.0
          9.0         11.0          0.0         76.0         48.2
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_37_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            70      70.000       0.000
  No. of usable loci            70      70.000       0.000
  No. of polym. loci            70      70.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_37_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.02649     0.02649     0.00000     0.02649
     3       0.02649     0.02649     0.00000     0.02649
     4       0.10166     0.10166     0.00000     0.10166
     5       0.07732     0.07732     0.00000     0.07732
     6       0.05226     0.05226     0.00000     0.05226
     7       0.01333     0.01333     0.00000     0.01333
     8       0.02649     0.02649     0.00000     0.02649
     9       0.01333     0.01333     0.00000     0.01333
    10       0.01333     0.01333     0.00000     0.01333
    11       0.02649     0.02649     0.00000     0.02649
    12       0.33790     0.33790     0.00000     0.33790
    13       0.01333     0.01333     0.00000     0.01333
    14       0.01333     0.01333     0.00000     0.01333
    15       0.02649     0.02649     0.00000     0.02649
    16       0.08957     0.08957     0.00000     0.08957
    17       0.01333     0.01333     0.00000     0.01333
    18       0.01333     0.01333     0.00000     0.01333
    19       0.01333     0.01333     0.00000     0.01333
    20       0.06488     0.06488     0.00000     0.06488
    21       0.01333     0.01333     0.00000     0.01333
    22       0.01333     0.01333     0.00000     0.01333
    23       0.03946     0.03946     0.00000     0.03946
    24       0.02649     0.02649     0.00000     0.02649
    25       0.03946     0.03946     0.00000     0.03946
    26       0.01333     0.01333     0.00000     0.01333
    27       0.10166     0.10166     0.00000     0.10166
    28       0.05226     0.05226     0.00000     0.05226
    29       0.02649     0.02649     0.00000     0.02649
    30       0.03946     0.03946     0.00000     0.03946
    31       0.03946     0.03946     0.00000     0.03946
    32       0.06488     0.06488     0.00000     0.06488
    33       0.01333     0.01333     0.00000     0.01333
    34       0.03946     0.03946     0.00000     0.03946
    35       0.01333     0.01333     0.00000     0.01333
    36       0.01333     0.01333     0.00000     0.01333
    37       0.01333     0.01333     0.00000     0.01333
    38       0.10166     0.10166     0.00000     0.10166
    39       0.06488     0.06488     0.00000     0.06488
    40       0.13682     0.13682     0.00000     0.13682
    41       0.06488     0.06488     0.00000     0.06488
    42       0.01333     0.01333     0.00000     0.01333
    43       0.12528     0.12528     0.00000     0.12528
    44       0.01333     0.01333     0.00000     0.01333
    45       0.02649     0.02649     0.00000     0.02649
    46       0.03946     0.03946     0.00000     0.03946
    47       0.01333     0.01333     0.00000     0.01333
    48       0.02649     0.02649     0.00000     0.02649
    49       0.01333     0.01333     0.00000     0.01333
    50       0.10166     0.10166     0.00000     0.10166
    51       0.10166     0.10166     0.00000     0.10166
    52       0.02649     0.02649     0.00000     0.02649
    53       0.05226     0.05226     0.00000     0.05226
    54       0.02649     0.02649     0.00000     0.02649
    55       0.07732     0.07732     0.00000     0.07732
    56       0.05226     0.05226     0.00000     0.05226
    57       0.07732     0.07732     0.00000     0.07732
    58       0.01333     0.01333     0.00000     0.01333
    59       0.05226     0.05226     0.00000     0.05226
    60       0.01333     0.01333     0.00000     0.01333
    61       0.01333     0.01333     0.00000     0.01333
    62       0.02649     0.02649     0.00000     0.02649
    63       0.01333     0.01333     0.00000     0.01333
    64       0.10166     0.10166     0.00000     0.10166
    65       0.03946     0.03946     0.00000     0.03946
    66       0.02649     0.02649     0.00000     0.02649
    67       0.02649     0.02649     0.00000     0.02649
    68       0.01333     0.01333     0.00000     0.01333
    69       0.01333     0.01333     0.00000     0.01333
    70       0.07732     0.07732     0.00000     0.07732
--------------------------------------------------------
  Mean       0.04462     0.04462     0.00000     0.04462
  s.d.       0.04800     0.04800     0.00000     0.04800
--------------------------------------------------------
<A NAME="31_01_17at14_44_37_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_37_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      30.87507    30.87507     0.00000
            Theta_k_lower      21.74374    21.74374     0.00000
            Theta_k_upper      43.53334    43.53334     0.00000
                  Theta_H      18.25494    18.25494     0.00000
             s.d. Theta_H       4.06580     4.06580     0.00000
                  Theta_S      12.53466    12.53466     0.00000
             s.d. Theta_S       3.15817     3.15817     0.00000
                 Theta_pi       3.12331     3.12331     0.00000
            s.d. Theta_pi       1.80340     1.80340     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_37_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.10938     3.10938     0.00000
              Tau qt 2.5%       2.28906     2.28906     0.00000
                Tau qt 5%       2.28906     2.28906     0.00000
               Tau qt 95%       3.32422     3.32422     0.00000
             Tau qt 97.5%       3.32422     3.32422     0.00000
                   Theta0       0.14746     0.14746     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.02988     0.02988     0.00000
          Theta0 qt 97.5%       0.02988     0.02988     0.00000
                   Theta1      71.40625    71.40625     0.00000
           Theta1 qt 2.5%      18.26172    18.26172     0.00000
             Theta1 qt 5%      18.26172    18.26172     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00031     0.00031     0.00000
      Model (SSD) p-value       1.00000     1.00000     0.00000
         Raggedness index       0.02221     0.02221     0.00000
       Raggedness p-value       0.70000     0.70000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 368 (31/01/17 at 14:44:37))
Total computing time for this run : 0h 0m 0s 112 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

