
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_51_50"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 157 (31/01/17 at 14:51:50)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_51_50_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_51_50_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 53
No. of loci              : 22
No. of usable loci       : 22 loci with less than  5.00 % missing data
No. of polymorphic sites : 22


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.03946
     2       150         2   0.03946
     3       150         2   0.49047
     4       150         2   0.02649
     5       150         2   0.11356
     6       150         2   0.11356
     7       150         2   0.05226
     8       150         2   0.36725
     9       150         2   0.38085
    10       150         2   0.02649
    11       150         2   0.06488
    12       150         2   0.37414
    13       150         2   0.33011
    14       150         2   0.01333
    15       150         2   0.03946
    16       150         2   0.41172
    17       150         2   0.21262
    18       150         2   0.02649
    19       150         2   0.11356
    20       150         2   0.33011
    21       150         2   0.21262
    22       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.17237
  s.d.     0.000     0.000   0.15997
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0483
Gene diversity           :  0.9581 +/-  0.0074

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 53

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 22
Number of usable nucleotide sites            : 22


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.792215 +/-     1.920725

Average gene diversity over loci              :     0.172373 +/-     0.096639

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    21.201638
  S.D. Theta(Hom) :     4.146845

         Theta(k) :    28.791891
95 % confidence interval limits for theta(k) : [  20.205559,  40.712272 ]

         Theta(S) :     3.939466
    S.D. Theta(S) :     1.201985

        Theta(Pi) :     3.792215
   S.D. Theta(Pi) :     2.126057

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     468       0.000
       1     651       0.000
       2    1552       0.000
       3    2137       0.000
       4    2347       0.000
       5    2095       0.000
       6    1288       0.000
       7     497       0.000
       8     119       0.000
       9      20       0.000
      10       1       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.792
Mismatch observed variance          :        3.165
Tau                                 :        4.072
Theta0                              :        0.004
Theta1                              :      927.500

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00281125
          P(Sim. Ssd >= Obs. Ssd): 0.10000000
    Harpending's Raggedness index: 0.02182161
        P(Sim. Rag. >= Obs. Rag.): 0.30000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        468.0        204.4        200.7
          1.0        651.0        823.1        780.9
          2.0       1552.0       1725.8       1578.4
          3.0       2137.0       2246.0       2138.7
          4.0       2347.0       2138.7       2176.7
          5.0       2095.0       1685.6       1773.3
          6.0       1288.0       1114.6       1204.2
          7.0        497.0        651.6        701.0
          8.0        119.0        336.5        357.1
          9.0         20.0        153.4        161.7
         10.0          1.0         63.9         65.9
         11.0          0.0         22.2         24.4
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.072           3.416           4.814           3.851
                   Theta0:           0.004           0.000           0.513           0.063               
                   Theta1:         927.500         175.312       99999.000       70689.863
Mean No. of pairwise diff.           3.792           3.546           4.803           3.965
           No. pol. sites:              22             175             235         197.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        468.0         95.0        413.0        200.7
          1.0        651.0        412.0       1090.0        780.9
          2.0       1552.0        980.0       2034.0       1578.4
          3.0       2137.0       1702.0       2465.0       2138.7
          4.0       2347.0       1942.0       2279.0       2176.7
          5.0       2095.0       1379.0       2032.0       1773.3
          6.0       1288.0        837.0       1580.0       1204.2
          7.0        497.0        476.0       1056.0        701.0
          8.0        119.0        185.0        678.0        357.1
          9.0         20.0         51.0        347.0        161.7
         10.0          1.0         18.0        150.0         65.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.072           3.416           4.814           3.851
                   Theta0:           0.004           0.000           0.513           0.063               
                   Theta1:         927.500         175.312       99999.000       70689.863
Mean No. of pairwise diff.           3.792           3.546           4.803           3.965
           No. pol. sites:              22             175             235         197.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        468.0         95.0        413.0        200.7
          1.0        651.0        412.0       1090.0        780.9
          2.0       1552.0        980.0       2034.0       1578.4
          3.0       2137.0       1702.0       2465.0       2138.7
          4.0       2347.0       1942.0       2279.0       2176.7
          5.0       2095.0       1379.0       2032.0       1773.3
          6.0       1288.0        837.0       1580.0       1204.2
          7.0        497.0        476.0       1056.0        701.0
          8.0        119.0        185.0        678.0        357.1
          9.0         20.0         51.0        347.0        161.7
         10.0          1.0         18.0        150.0         65.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.072           3.416           4.814           3.851
                   Theta0:           0.004           0.000           0.513           0.063               
                   Theta1:         927.500         175.312       99999.000       70689.863
Mean No. of pairwise diff.           3.792           3.546           4.803           3.965
           No. pol. sites:              22             175             235         197.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        468.0         95.0        413.0        200.7
          1.0        651.0        412.0       1090.0        780.9
          2.0       1552.0        980.0       2034.0       1578.4
          3.0       2137.0       1702.0       2465.0       2138.7
          4.0       2347.0       1942.0       2279.0       2176.7
          5.0       2095.0       1379.0       2032.0       1773.3
          6.0       1288.0        837.0       1580.0       1204.2
          7.0        497.0        476.0       1056.0        701.0
          8.0        119.0        185.0        678.0        357.1
          9.0         20.0         51.0        347.0        161.7
         10.0          1.0         18.0        150.0         65.9
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_51_50_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            22      22.000       0.000
  No. of usable loci            22      22.000       0.000
  No. of polym. loci            22      22.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_51_50_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.03946     0.03946     0.00000     0.03946
     2       0.03946     0.03946     0.00000     0.03946
     3       0.49047     0.49047     0.00000     0.49047
     4       0.02649     0.02649     0.00000     0.02649
     5       0.11356     0.11356     0.00000     0.11356
     6       0.11356     0.11356     0.00000     0.11356
     7       0.05226     0.05226     0.00000     0.05226
     8       0.36725     0.36725     0.00000     0.36725
     9       0.38085     0.38085     0.00000     0.38085
    10       0.02649     0.02649     0.00000     0.02649
    11       0.06488     0.06488     0.00000     0.06488
    12       0.37414     0.37414     0.00000     0.37414
    13       0.33011     0.33011     0.00000     0.33011
    14       0.01333     0.01333     0.00000     0.01333
    15       0.03946     0.03946     0.00000     0.03946
    16       0.41172     0.41172     0.00000     0.41172
    17       0.21262     0.21262     0.00000     0.21262
    18       0.02649     0.02649     0.00000     0.02649
    19       0.11356     0.11356     0.00000     0.11356
    20       0.33011     0.33011     0.00000     0.33011
    21       0.21262     0.21262     0.00000     0.21262
    22       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.17237     0.17237     0.00000     0.17237
  s.d.       0.15997     0.15997     0.00000     0.15997
--------------------------------------------------------
<A NAME="31_01_17at14_51_50_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_51_50_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      28.79189    28.79189     0.00000
            Theta_k_lower      20.20556    20.20556     0.00000
            Theta_k_upper      40.71227    40.71227     0.00000
                  Theta_H      21.20164    21.20164     0.00000
             s.d. Theta_H       4.14684     4.14684     0.00000
                  Theta_S       3.93947     3.93947     0.00000
             s.d. Theta_S       1.20198     1.20198     0.00000
                 Theta_pi       3.79221     3.79221     0.00000
            s.d. Theta_pi       2.12606     2.12606     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_51_50_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.07227     4.07227     0.00000
              Tau qt 2.5%       3.41602     3.41602     0.00000
                Tau qt 5%       3.41602     3.41602     0.00000
               Tau qt 95%       4.07227     4.07227     0.00000
             Tau qt 97.5%       4.07227     4.07227     0.00000
                   Theta0       0.00352     0.00352     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.05801     0.05801     0.00000
          Theta0 qt 97.5%       0.05801     0.05801     0.00000
                   Theta1     927.50000   927.50000     0.00000
           Theta1 qt 2.5%     175.31250   175.31250     0.00000
             Theta1 qt 5%     175.31250   175.31250     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00281     0.00281     0.00000
      Model (SSD) p-value       0.10000     0.10000     0.00000
         Raggedness index       0.02182     0.02182     0.00000
       Raggedness p-value       0.30000     0.30000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 157 (31/01/17 at 14:51:50))
Total computing time for this run : 0h 0m 0s 130 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

