
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_50_09"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 343 (31/01/17 at 14:50:09)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_50_09_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_50_09_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 49
No. of loci              : 23
No. of usable loci       : 23 loci with less than  5.00 % missing data
No. of polymorphic sites : 23


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.45181
     3       150         2   0.14819
     4       150         2   0.01333
     5       150         2   0.08957
     6       150         2   0.17038
     7       150         2   0.01333
     8       150         2   0.01333
     9       150         2   0.35293
    10       150         2   0.26139
    11       150         2   0.01333
    12       150         2   0.39374
    13       150         2   0.07732
    14       150         2   0.03946
    15       150         2   0.01333
    16       150         2   0.03946
    17       150         2   0.48322
    18       150         2   0.19186
    19       150         2   0.01333
    20       150         2   0.06488
    21       150         2   0.28850
    22       150         2   0.27964
    23       150         2   0.03946
-------------------------------------------------
  Mean   150.000     2.000   0.15066
  s.d.     0.000     0.000   0.15621
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0340
Gene diversity           :  0.9724 +/-  0.0037

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 49

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 23
Number of usable nucleotide sites            : 23


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.465145 +/-     1.778387

Average gene diversity over loci              :     0.150658 +/-     0.085587

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    33.498567
  S.D. Theta(Hom) :     4.790729

         Theta(k) :    24.926798
95 % confidence interval limits for theta(k) : [  17.352333,  35.483223 ]

         Theta(S) :     4.118532
    S.D. Theta(S) :     1.243883

        Theta(Pi) :     3.465145
   S.D. Theta(Pi) :     1.968502

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     308       0.000
       1     844       0.000
       2    1986       0.000
       3    2516       0.000
       4    2877       0.000
       5    1376       0.000
       6     988       0.000
       7     205       0.000
       8      67       0.000
       9       8       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.465
Mismatch observed variance          :        2.527
Tau                                 :        3.627
Theta0                              :        0.000
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00609876
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.04037379
        P(Sim. Rag. >= Obs. Rag.): 0.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        308.0        251.6        297.3
          1.0        844.0       1015.3       1078.1
          2.0       1986.0       1952.2       1955.0
          3.0       2516.0       2453.3       2363.5
          4.0       2877.0       2196.4       2143.0
          5.0       1376.0       1562.1       1554.5
          6.0        988.0        934.7        939.7
          7.0        205.0        485.7        486.9
          8.0         67.0        222.7        220.7
          9.0          8.0         72.2         89.0
         10.0          0.0         21.8         32.3
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.627           3.156           4.115           3.635
                   Theta0:           0.000           0.000           0.025           0.004               
                   Theta1:       99999.000          76.578       99999.000       80143.008
Mean No. of pairwise diff.           3.465           3.207           4.047           3.634
           No. pol. sites:              23             185             202         194.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        308.0        141.0        465.0        297.3
          1.0        844.0        695.0       1645.0       1078.1
          2.0       1986.0       1489.0       2409.0       1955.0
          3.0       2516.0       2202.0       2816.0       2363.5
          4.0       2877.0       1769.0       2351.0       2143.0
          5.0       1376.0       1240.0       1935.0       1554.5
          6.0        988.0        610.0       1340.0        939.7
          7.0        205.0        324.0        648.0        486.9
          8.0         67.0         86.0        325.0        220.7
          9.0          8.0         19.0        105.0         89.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.627           3.156           4.115           3.635
                   Theta0:           0.000           0.000           0.025           0.004               
                   Theta1:       99999.000          76.578       99999.000       80143.008
Mean No. of pairwise diff.           3.465           3.207           4.047           3.634
           No. pol. sites:              23             185             202         194.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        308.0        141.0        465.0        297.3
          1.0        844.0        695.0       1645.0       1078.1
          2.0       1986.0       1489.0       2409.0       1955.0
          3.0       2516.0       2202.0       2816.0       2363.5
          4.0       2877.0       1769.0       2351.0       2143.0
          5.0       1376.0       1240.0       1935.0       1554.5
          6.0        988.0        610.0       1340.0        939.7
          7.0        205.0        324.0        648.0        486.9
          8.0         67.0         86.0        325.0        220.7
          9.0          8.0         19.0        105.0         89.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.627           3.156           4.115           3.635
                   Theta0:           0.000           0.000           0.025           0.004               
                   Theta1:       99999.000          76.578       99999.000       80143.008
Mean No. of pairwise diff.           3.465           3.207           4.047           3.634
           No. pol. sites:              23             185             202         194.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        308.0        141.0        465.0        297.3
          1.0        844.0        695.0       1645.0       1078.1
          2.0       1986.0       1489.0       2409.0       1955.0
          3.0       2516.0       2202.0       2816.0       2363.5
          4.0       2877.0       1769.0       2351.0       2143.0
          5.0       1376.0       1240.0       1935.0       1554.5
          6.0        988.0        610.0       1340.0        939.7
          7.0        205.0        324.0        648.0        486.9
          8.0         67.0         86.0        325.0        220.7
          9.0          8.0         19.0        105.0         89.0
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_50_09_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            23      23.000       0.000
  No. of usable loci            23      23.000       0.000
  No. of polym. loci            23      23.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_50_09_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.45181     0.45181     0.00000     0.45181
     3       0.14819     0.14819     0.00000     0.14819
     4       0.01333     0.01333     0.00000     0.01333
     5       0.08957     0.08957     0.00000     0.08957
     6       0.17038     0.17038     0.00000     0.17038
     7       0.01333     0.01333     0.00000     0.01333
     8       0.01333     0.01333     0.00000     0.01333
     9       0.35293     0.35293     0.00000     0.35293
    10       0.26139     0.26139     0.00000     0.26139
    11       0.01333     0.01333     0.00000     0.01333
    12       0.39374     0.39374     0.00000     0.39374
    13       0.07732     0.07732     0.00000     0.07732
    14       0.03946     0.03946     0.00000     0.03946
    15       0.01333     0.01333     0.00000     0.01333
    16       0.03946     0.03946     0.00000     0.03946
    17       0.48322     0.48322     0.00000     0.48322
    18       0.19186     0.19186     0.00000     0.19186
    19       0.01333     0.01333     0.00000     0.01333
    20       0.06488     0.06488     0.00000     0.06488
    21       0.28850     0.28850     0.00000     0.28850
    22       0.27964     0.27964     0.00000     0.27964
    23       0.03946     0.03946     0.00000     0.03946
--------------------------------------------------------
  Mean       0.15066     0.15066     0.00000     0.15066
  s.d.       0.15621     0.15621     0.00000     0.15621
--------------------------------------------------------
<A NAME="31_01_17at14_50_09_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_50_09_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      24.92680    24.92680     0.00000
            Theta_k_lower      17.35233    17.35233     0.00000
            Theta_k_upper      35.48322    35.48322     0.00000
                  Theta_H      33.49857    33.49857     0.00000
             s.d. Theta_H       4.79073     4.79073     0.00000
                  Theta_S       4.11853     4.11853     0.00000
             s.d. Theta_S       1.24388     1.24388     0.00000
                 Theta_pi       3.46515     3.46515     0.00000
            s.d. Theta_pi       1.96850     1.96850     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_50_09_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.62695     3.62695     0.00000
              Tau qt 2.5%       3.15625     3.15625     0.00000
                Tau qt 5%       3.15625     3.15625     0.00000
               Tau qt 95%       3.98242     3.98242     0.00000
             Tau qt 97.5%       3.98242     3.98242     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.01406     0.01406     0.00000
          Theta0 qt 97.5%       0.01406     0.01406     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      76.57812    76.57812     0.00000
             Theta1 qt 5%      76.57812    76.57812     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00610     0.00610     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.04037     0.04037     0.00000
       Raggedness p-value       0.00000     0.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 343 (31/01/17 at 14:50:09))
Total computing time for this run : 0h 0m 0s 105 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

