
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_03"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 117 (31/01/17 at 14:44:03)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_03_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_03_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 36
No. of loci              : 18
No. of usable loci       : 18 loci with less than  5.00 % missing data
No. of polymorphic sites : 18


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.20233
     2       150         2   0.19186
     3       150         2   0.01333
     4       150         2   0.01333
     5       150         2   0.03946
     6       150         2   0.39991
     7       150         2   0.10166
     8       150         2   0.01333
     9       150         2   0.03946
    10       150         2   0.20233
    11       150         2   0.13682
    12       150         2   0.38738
    13       150         2   0.19186
    14       150         2   0.17038
    15       150         2   0.18121
    16       150         2   0.19186
    17       150         2   0.18121
    18       150         2   0.05226
-------------------------------------------------
  Mean   150.000     2.000   0.15055
  s.d.     0.000     0.000   0.11490
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.1019
Gene diversity           :  0.9042 +/-  0.0172

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 36

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 18
Number of usable nucleotide sites            : 18


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.709978 +/-     1.448174

Average gene diversity over loci              :     0.150554 +/-     0.089055

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     8.118160
  S.D. Theta(Hom) :     1.771813

         Theta(k) :    14.713127
95 % confidence interval limits for theta(k) : [   9.847854,  21.654571 ]

         Theta(S) :     3.223199
    S.D. Theta(S) :     1.032890

        Theta(Pi) :     2.709978
   S.D. Theta(Pi) :     1.602988

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1071     767.158
       1    1808    2034.598
       2    2311    2715.461
       3    2518    2433.929
       4    1834    1650.002
       5    1041     903.540
       6     424     416.937
       7     133     167.051
       8      31      59.448
       9       4      19.134
      10       0       5.655
      11       0       1.555
      12       0       0.403
      13       0       0.099
      14       0       0.023
      15       0       0.005
      16       0       0.001
      17       0       0.000
      18       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.710
Mismatch observed variance          :        2.784
Tau                                 :        3.045
Theta0                              :        0.007
Theta1                              :       19.033

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00071050
          P(Sim. Ssd >= Obs. Ssd): 0.80000000
    Harpending's Raggedness index: 0.01931646
        P(Sim. Rag. >= Obs. Rag.): 0.90000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1071.0       1154.5        985.3
          1.0       1808.0       2039.8       1813.3
          2.0       2311.0       2673.7       2409.0
          3.0       2518.0       2375.9       2323.2
          4.0       1834.0       1491.7       1727.2
          5.0       1041.0        847.7       1038.9
          6.0        424.0        421.6        523.6
          7.0        133.0        130.8        226.9
          8.0         31.0         27.7         86.2
          9.0          4.0          8.8         29.1
         10.0          0.0          2.8          8.9
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.045           1.506           3.902           2.715
                   Theta0:           0.007           0.000           0.097           0.015               
                   Theta1:          19.033          12.471       99999.000       60015.160
Mean No. of pairwise diff.           2.710           1.504           3.608           2.550
           No. pol. sites:              18              37              55          44.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1071.0        412.0       2315.0        985.3
          1.0       1808.0        664.0       3915.0       1813.3
          2.0       2311.0       1479.0       3574.0       2409.0
          3.0       2518.0       1488.0       3661.0       2323.2
          4.0       1834.0        381.0       2502.0       1727.2
          5.0       1041.0         54.0       1967.0       1038.9
          6.0        424.0          3.0       1186.0        523.6
          7.0        133.0          0.0        322.0        226.9
          8.0         31.0          0.0        105.0         86.2
          9.0          4.0          0.0         88.0         29.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.045           1.506           3.902           2.715
                   Theta0:           0.007           0.000           0.097           0.015               
                   Theta1:          19.033          12.471       99999.000       60015.160
Mean No. of pairwise diff.           2.710           1.504           3.608           2.550
           No. pol. sites:              18              37              55          44.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1071.0        412.0       2315.0        985.3
          1.0       1808.0        664.0       3915.0       1813.3
          2.0       2311.0       1479.0       3574.0       2409.0
          3.0       2518.0       1488.0       3661.0       2323.2
          4.0       1834.0        381.0       2502.0       1727.2
          5.0       1041.0         54.0       1967.0       1038.9
          6.0        424.0          3.0       1186.0        523.6
          7.0        133.0          0.0        322.0        226.9
          8.0         31.0          0.0        105.0         86.2
          9.0          4.0          0.0         88.0         29.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.045           1.506           3.902           2.715
                   Theta0:           0.007           0.000           0.097           0.015               
                   Theta1:          19.033          12.471       99999.000       60015.160
Mean No. of pairwise diff.           2.710           1.504           3.608           2.550
           No. pol. sites:              18              37              55          44.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1071.0        412.0       2315.0        985.3
          1.0       1808.0        664.0       3915.0       1813.3
          2.0       2311.0       1479.0       3574.0       2409.0
          3.0       2518.0       1488.0       3661.0       2323.2
          4.0       1834.0        381.0       2502.0       1727.2
          5.0       1041.0         54.0       1967.0       1038.9
          6.0        424.0          3.0       1186.0        523.6
          7.0        133.0          0.0        322.0        226.9
          8.0         31.0          0.0        105.0         86.2
          9.0          4.0          0.0         88.0         29.1
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_03_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            18      18.000       0.000
  No. of usable loci            18      18.000       0.000
  No. of polym. loci            18      18.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_03_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.20233     0.20233     0.00000     0.20233
     2       0.19186     0.19186     0.00000     0.19186
     3       0.01333     0.01333     0.00000     0.01333
     4       0.01333     0.01333     0.00000     0.01333
     5       0.03946     0.03946     0.00000     0.03946
     6       0.39991     0.39991     0.00000     0.39991
     7       0.10166     0.10166     0.00000     0.10166
     8       0.01333     0.01333     0.00000     0.01333
     9       0.03946     0.03946     0.00000     0.03946
    10       0.20233     0.20233     0.00000     0.20233
    11       0.13682     0.13682     0.00000     0.13682
    12       0.38738     0.38738     0.00000     0.38738
    13       0.19186     0.19186     0.00000     0.19186
    14       0.17038     0.17038     0.00000     0.17038
    15       0.18121     0.18121     0.00000     0.18121
    16       0.19186     0.19186     0.00000     0.19186
    17       0.18121     0.18121     0.00000     0.18121
    18       0.05226     0.05226     0.00000     0.05226
--------------------------------------------------------
  Mean       0.15055     0.15055     0.00000     0.15055
  s.d.       0.11490     0.11490     0.00000     0.11490
--------------------------------------------------------
<A NAME="31_01_17at14_44_03_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_03_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      14.71313    14.71313     0.00000
            Theta_k_lower       9.84785     9.84785     0.00000
            Theta_k_upper      21.65457    21.65457     0.00000
                  Theta_H       8.11816     8.11816     0.00000
             s.d. Theta_H       1.77181     1.77181     0.00000
                  Theta_S       3.22320     3.22320     0.00000
             s.d. Theta_S       1.03289     1.03289     0.00000
                 Theta_pi       2.70998     2.70998     0.00000
            s.d. Theta_pi       1.60299     1.60299     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_03_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.04492     3.04492     0.00000
              Tau qt 2.5%       1.50586     1.50586     0.00000
                Tau qt 5%       1.50586     1.50586     0.00000
               Tau qt 95%       3.82227     3.82227     0.00000
             Tau qt 97.5%       3.82227     3.82227     0.00000
                   Theta0       0.00703     0.00703     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.05098     0.05098     0.00000
          Theta0 qt 97.5%       0.05098     0.05098     0.00000
                   Theta1      19.03320    19.03320     0.00000
           Theta1 qt 2.5%      12.47070    12.47070     0.00000
             Theta1 qt 5%      12.47070    12.47070     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00071     0.00071     0.00000
      Model (SSD) p-value       0.80000     0.80000     0.00000
         Raggedness index       0.01932     0.01932     0.00000
       Raggedness p-value       0.90000     0.90000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 117 (31/01/17 at 14:44:03))
Total computing time for this run : 0h 0m 0s 98 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

