
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_49_44"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 195 (31/01/17 at 14:49:44)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_49_44_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_49_44_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 45
No. of loci              : 19
No. of usable loci       : 19 loci with less than  5.00 % missing data
No. of polymorphic sites : 19


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.19186
     2       150         2   0.08957
     3       150         2   0.02649
     4       150         2   0.06488
     5       150         2   0.27060
     6       150         2   0.30568
     7       150         2   0.08957
     8       150         2   0.27060
     9       150         2   0.19186
    10       150         2   0.03946
    11       150         2   0.25199
    12       150         2   0.36018
    13       150         2   0.05226
    14       150         2   0.13682
    15       150         2   0.35293
    16       150         2   0.03946
    17       150         2   0.03946
    18       150         2   0.45181
    19       150         2   0.26139
-------------------------------------------------
  Mean   150.000     2.000   0.18352
  s.d.     0.000     0.000   0.13143
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0513
Gene diversity           :  0.9551 +/-  0.0068

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 45

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 19
Number of usable nucleotide sites            : 19


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.486890 +/-     1.787860

Average gene diversity over loci              :     0.183521 +/-     0.104157

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    19.606645
  S.D. Theta(Hom) :     3.313150

         Theta(k) :    21.427915
95 % confidence interval limits for theta(k) : [  14.772787,  30.751806 ]

         Theta(S) :     3.402266
    S.D. Theta(S) :     1.075424

        Theta(Pi) :     3.486890
   S.D. Theta(Pi) :     1.978988

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     502       0.000
       1    1063       0.000
       2    1604       0.000
       3    2508       0.000
       4    2450       0.000
       5    1578       0.000
       6     920       0.000
       7     445       0.000
       8      99       0.000
       9       6       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.487
Mismatch observed variance          :        3.094
Tau                                 :        3.719
Theta0                              :        0.005
Theta1                              :      163.750

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00228469
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.02382560
        P(Sim. Rag. >= Obs. Rag.): 0.40000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        502.0        281.8        329.9
          1.0       1063.0        915.4       1041.8
          2.0       1604.0       1773.6       1876.7
          3.0       2508.0       2324.5       2303.9
          4.0       2450.0       2131.1       2134.2
          5.0       1578.0       1646.5       1585.3
          6.0        920.0       1084.6        982.4
          7.0        445.0        586.7        522.1
          8.0         99.0        265.1        242.8
          9.0          6.0        111.0        100.4
         10.0          0.0         38.5         37.4
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.719           2.859           4.258           3.745
                   Theta0:           0.005           0.000           0.921           0.099               
                   Theta1:         163.750          41.143       99999.000       70025.571
Mean No. of pairwise diff.           3.487           3.040           4.337           3.803
           No. pol. sites:              19             123             155         139.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        502.0         91.0        620.0        329.9
          1.0       1063.0        500.0       1752.0       1041.8
          2.0       1604.0       1372.0       2634.0       1876.7
          3.0       2508.0       2024.0       2626.0       2303.9
          4.0       2450.0       1467.0       2393.0       2134.2
          5.0       1578.0        949.0       1957.0       1585.3
          6.0        920.0        715.0       1338.0        982.4
          7.0        445.0        429.0        832.0        522.1
          8.0         99.0        127.0        469.0        242.8
          9.0          6.0         31.0        218.0        100.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.719           2.859           4.258           3.745
                   Theta0:           0.005           0.000           0.921           0.099               
                   Theta1:         163.750          41.143       99999.000       70025.571
Mean No. of pairwise diff.           3.487           3.040           4.337           3.803
           No. pol. sites:              19             123             155         139.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        502.0         91.0        620.0        329.9
          1.0       1063.0        500.0       1752.0       1041.8
          2.0       1604.0       1372.0       2634.0       1876.7
          3.0       2508.0       2024.0       2626.0       2303.9
          4.0       2450.0       1467.0       2393.0       2134.2
          5.0       1578.0        949.0       1957.0       1585.3
          6.0        920.0        715.0       1338.0        982.4
          7.0        445.0        429.0        832.0        522.1
          8.0         99.0        127.0        469.0        242.8
          9.0          6.0         31.0        218.0        100.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.719           2.859           4.258           3.745
                   Theta0:           0.005           0.000           0.921           0.099               
                   Theta1:         163.750          41.143       99999.000       70025.571
Mean No. of pairwise diff.           3.487           3.040           4.337           3.803
           No. pol. sites:              19             123             155         139.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        502.0         91.0        620.0        329.9
          1.0       1063.0        500.0       1752.0       1041.8
          2.0       1604.0       1372.0       2634.0       1876.7
          3.0       2508.0       2024.0       2626.0       2303.9
          4.0       2450.0       1467.0       2393.0       2134.2
          5.0       1578.0        949.0       1957.0       1585.3
          6.0        920.0        715.0       1338.0        982.4
          7.0        445.0        429.0        832.0        522.1
          8.0         99.0        127.0        469.0        242.8
          9.0          6.0         31.0        218.0        100.4
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_49_44_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            19      19.000       0.000
  No. of usable loci            19      19.000       0.000
  No. of polym. loci            19      19.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_49_44_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.19186     0.19186     0.00000     0.19186
     2       0.08957     0.08957     0.00000     0.08957
     3       0.02649     0.02649     0.00000     0.02649
     4       0.06488     0.06488     0.00000     0.06488
     5       0.27060     0.27060     0.00000     0.27060
     6       0.30568     0.30568     0.00000     0.30568
     7       0.08957     0.08957     0.00000     0.08957
     8       0.27060     0.27060     0.00000     0.27060
     9       0.19186     0.19186     0.00000     0.19186
    10       0.03946     0.03946     0.00000     0.03946
    11       0.25199     0.25199     0.00000     0.25199
    12       0.36018     0.36018     0.00000     0.36018
    13       0.05226     0.05226     0.00000     0.05226
    14       0.13682     0.13682     0.00000     0.13682
    15       0.35293     0.35293     0.00000     0.35293
    16       0.03946     0.03946     0.00000     0.03946
    17       0.03946     0.03946     0.00000     0.03946
    18       0.45181     0.45181     0.00000     0.45181
    19       0.26139     0.26139     0.00000     0.26139
--------------------------------------------------------
  Mean       0.18352     0.18352     0.00000     0.18352
  s.d.       0.13143     0.13143     0.00000     0.13143
--------------------------------------------------------
<A NAME="31_01_17at14_49_44_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_49_44_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      21.42791    21.42791     0.00000
            Theta_k_lower      14.77279    14.77279     0.00000
            Theta_k_upper      30.75181    30.75181     0.00000
                  Theta_H      19.60665    19.60665     0.00000
             s.d. Theta_H       3.31315     3.31315     0.00000
                  Theta_S       3.40227     3.40227     0.00000
             s.d. Theta_S       1.07542     1.07542     0.00000
                 Theta_pi       3.48689     3.48689     0.00000
            s.d. Theta_pi       1.97899     1.97899     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_49_44_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.71875     3.71875     0.00000
              Tau qt 2.5%       2.85938     2.85938     0.00000
                Tau qt 5%       2.85938     2.85938     0.00000
               Tau qt 95%       4.25195     4.25195     0.00000
             Tau qt 97.5%       4.25195     4.25195     0.00000
                   Theta0       0.00527     0.00527     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.02461     0.02461     0.00000
          Theta0 qt 97.5%       0.02461     0.02461     0.00000
                   Theta1     163.75000   163.75000     0.00000
           Theta1 qt 2.5%      41.14258    41.14258     0.00000
             Theta1 qt 5%      41.14258    41.14258     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00228     0.00228     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.02383     0.02383     0.00000
       Raggedness p-value       0.40000     0.40000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 195 (31/01/17 at 14:49:44))
Total computing time for this run : 0h 0m 0s 121 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

