
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_19"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 240 (31/01/17 at 14:44:19)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_19_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_19_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 48
No. of loci              : 64
No. of usable loci       : 64 loci with less than  5.00 % missing data
No. of polymorphic sites : 64


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.02649
     2       150         2   0.05226
     3       150         2   0.03946
     4       150         2   0.03946
     5       150         2   0.02649
     6       150         2   0.01333
     7       150         2   0.02649
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.06488
    11       150         2   0.05226
    12       150         2   0.01333
    13       150         2   0.06488
    14       150         2   0.02649
    15       150         2   0.05226
    16       150         2   0.02649
    17       150         2   0.01333
    18       150         2   0.10166
    19       150         2   0.02649
    20       150         2   0.10166
    21       150         2   0.01333
    22       150         2   0.03946
    23       150         2   0.02649
    24       150         2   0.05226
    25       150         2   0.02649
    26       150         2   0.01333
    27       150         2   0.01333
    28       150         2   0.01333
    29       150         2   0.01333
    30       150         2   0.02649
    31       150         2   0.01333
    32       150         2   0.02649
    33       150         2   0.02649
    34       150         2   0.06488
    35       150         2   0.08957
    36       150         2   0.01333
    37       150         2   0.05226
    38       150         2   0.01333
    39       150         2   0.10166
    40       150         2   0.01333
    41       150         2   0.01333
    42       150         2   0.05226
    43       150         2   0.01333
    44       150         2   0.06488
    45       150         2   0.01333
    46       150         2   0.02649
    47       150         2   0.02649
    48       150         2   0.02649
    49       150         2   0.02649
    50       150         2   0.03946
    51       150         2   0.07732
    52       150         2   0.05226
    53       150         2   0.01333
    54       150         2   0.19186
    55       150         2   0.05226
    56       150         2   0.03946
    57       150         2   0.01333
    58       150         2   0.11356
    59       150         2   0.02649
    60       150         2   0.03946
    61       150         2   0.02649
    62       150         2   0.14819
    63       150         2   0.10166
    64       150         2   0.08957
-------------------------------------------------
  Mean   150.000     2.000   0.04273
  s.d.     0.000     0.000   0.03597
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0740
Gene diversity           :  0.9322 +/-  0.0147

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 48

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 64
Number of usable nucleotide sites            : 64


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.734676 +/-     1.459019

Average gene diversity over loci              :     0.042729 +/-     0.025234

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    12.248961
  S.D. Theta(Hom) :     3.088264

         Theta(k) :    24.019287
95 % confidence interval limits for theta(k) : [  16.682825,  34.256022 ]

         Theta(S) :    11.460263
    S.D. Theta(S) :     2.916262

        Theta(Pi) :     2.734676
   S.D. Theta(Pi) :     1.614993

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     758       0.000
       1    1986       0.000
       2    2504       0.000
       3    2465       0.000
       4    1865       0.000
       5    1032       0.000
       6     418       0.000
       7     133       0.000
       8      14       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.735
Mismatch observed variance          :        2.570
Tau                                 :        2.867
Theta0                              :        0.000
Theta1                              :       67.266

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00038183
          P(Sim. Ssd >= Obs. Ssd): 0.80000000
    Harpending's Raggedness index: 0.02645964
        P(Sim. Rag. >= Obs. Rag.): 0.70000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        758.0        768.3        763.7
          1.0       1986.0       1875.9       1872.7
          2.0       2504.0       2605.0       2591.1
          3.0       2465.0       2456.4       2444.0
          4.0       1865.0       1731.3       1740.7
          5.0       1032.0        939.9        994.6
          6.0        418.0        479.5        474.2
          7.0        133.0        209.6        193.9
          8.0         14.0         78.9         69.4
          9.0          0.0         25.4         22.1
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.867           1.945           4.113           2.698
                   Theta0:           0.000           0.000           0.731           0.144               
                   Theta1:          67.266          17.305       99999.000       50084.616
Mean No. of pairwise diff.           2.735           2.185           3.789           2.804
           No. pol. sites:              64              74             106          84.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        758.0        324.0       1346.0        763.7
          1.0       1986.0       1313.0       2435.0       1872.7
          2.0       2504.0       1455.0       3145.0       2591.1
          3.0       2465.0       1731.0       2960.0       2444.0
          4.0       1865.0       1365.0       2281.0       1740.7
          5.0       1032.0        509.0       1540.0        994.6
          6.0        418.0        100.0       1155.0        474.2
          7.0        133.0         11.0        708.0        193.9
          8.0         14.0          0.0        384.0         69.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.867           1.945           4.113           2.698
                   Theta0:           0.000           0.000           0.731           0.144               
                   Theta1:          67.266          17.305       99999.000       50084.616
Mean No. of pairwise diff.           2.735           2.185           3.789           2.804
           No. pol. sites:              64              74             106          84.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        758.0        324.0       1346.0        763.7
          1.0       1986.0       1313.0       2435.0       1872.7
          2.0       2504.0       1455.0       3145.0       2591.1
          3.0       2465.0       1731.0       2960.0       2444.0
          4.0       1865.0       1365.0       2281.0       1740.7
          5.0       1032.0        509.0       1540.0        994.6
          6.0        418.0        100.0       1155.0        474.2
          7.0        133.0         11.0        708.0        193.9
          8.0         14.0          0.0        384.0         69.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.867           1.945           4.113           2.698
                   Theta0:           0.000           0.000           0.731           0.144               
                   Theta1:          67.266          17.305       99999.000       50084.616
Mean No. of pairwise diff.           2.735           2.185           3.789           2.804
           No. pol. sites:              64              74             106          84.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        758.0        324.0       1346.0        763.7
          1.0       1986.0       1313.0       2435.0       1872.7
          2.0       2504.0       1455.0       3145.0       2591.1
          3.0       2465.0       1731.0       2960.0       2444.0
          4.0       1865.0       1365.0       2281.0       1740.7
          5.0       1032.0        509.0       1540.0        994.6
          6.0        418.0        100.0       1155.0        474.2
          7.0        133.0         11.0        708.0        193.9
          8.0         14.0          0.0        384.0         69.4
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_19_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            64      64.000       0.000
  No. of usable loci            64      64.000       0.000
  No. of polym. loci            64      64.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_19_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.02649     0.02649     0.00000     0.02649
     2       0.05226     0.05226     0.00000     0.05226
     3       0.03946     0.03946     0.00000     0.03946
     4       0.03946     0.03946     0.00000     0.03946
     5       0.02649     0.02649     0.00000     0.02649
     6       0.01333     0.01333     0.00000     0.01333
     7       0.02649     0.02649     0.00000     0.02649
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.06488     0.06488     0.00000     0.06488
    11       0.05226     0.05226     0.00000     0.05226
    12       0.01333     0.01333     0.00000     0.01333
    13       0.06488     0.06488     0.00000     0.06488
    14       0.02649     0.02649     0.00000     0.02649
    15       0.05226     0.05226     0.00000     0.05226
    16       0.02649     0.02649     0.00000     0.02649
    17       0.01333     0.01333     0.00000     0.01333
    18       0.10166     0.10166     0.00000     0.10166
    19       0.02649     0.02649     0.00000     0.02649
    20       0.10166     0.10166     0.00000     0.10166
    21       0.01333     0.01333     0.00000     0.01333
    22       0.03946     0.03946     0.00000     0.03946
    23       0.02649     0.02649     0.00000     0.02649
    24       0.05226     0.05226     0.00000     0.05226
    25       0.02649     0.02649     0.00000     0.02649
    26       0.01333     0.01333     0.00000     0.01333
    27       0.01333     0.01333     0.00000     0.01333
    28       0.01333     0.01333     0.00000     0.01333
    29       0.01333     0.01333     0.00000     0.01333
    30       0.02649     0.02649     0.00000     0.02649
    31       0.01333     0.01333     0.00000     0.01333
    32       0.02649     0.02649     0.00000     0.02649
    33       0.02649     0.02649     0.00000     0.02649
    34       0.06488     0.06488     0.00000     0.06488
    35       0.08957     0.08957     0.00000     0.08957
    36       0.01333     0.01333     0.00000     0.01333
    37       0.05226     0.05226     0.00000     0.05226
    38       0.01333     0.01333     0.00000     0.01333
    39       0.10166     0.10166     0.00000     0.10166
    40       0.01333     0.01333     0.00000     0.01333
    41       0.01333     0.01333     0.00000     0.01333
    42       0.05226     0.05226     0.00000     0.05226
    43       0.01333     0.01333     0.00000     0.01333
    44       0.06488     0.06488     0.00000     0.06488
    45       0.01333     0.01333     0.00000     0.01333
    46       0.02649     0.02649     0.00000     0.02649
    47       0.02649     0.02649     0.00000     0.02649
    48       0.02649     0.02649     0.00000     0.02649
    49       0.02649     0.02649     0.00000     0.02649
    50       0.03946     0.03946     0.00000     0.03946
    51       0.07732     0.07732     0.00000     0.07732
    52       0.05226     0.05226     0.00000     0.05226
    53       0.01333     0.01333     0.00000     0.01333
    54       0.19186     0.19186     0.00000     0.19186
    55       0.05226     0.05226     0.00000     0.05226
    56       0.03946     0.03946     0.00000     0.03946
    57       0.01333     0.01333     0.00000     0.01333
    58       0.11356     0.11356     0.00000     0.11356
    59       0.02649     0.02649     0.00000     0.02649
    60       0.03946     0.03946     0.00000     0.03946
    61       0.02649     0.02649     0.00000     0.02649
    62       0.14819     0.14819     0.00000     0.14819
    63       0.10166     0.10166     0.00000     0.10166
    64       0.08957     0.08957     0.00000     0.08957
--------------------------------------------------------
  Mean       0.04273     0.04273     0.00000     0.04273
  s.d.       0.03597     0.03597     0.00000     0.03597
--------------------------------------------------------
<A NAME="31_01_17at14_44_19_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_19_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      24.01929    24.01929     0.00000
            Theta_k_lower      16.68282    16.68282     0.00000
            Theta_k_upper      34.25602    34.25602     0.00000
                  Theta_H      12.24896    12.24896     0.00000
             s.d. Theta_H       3.08826     3.08826     0.00000
                  Theta_S      11.46026    11.46026     0.00000
             s.d. Theta_S       2.91626     2.91626     0.00000
                 Theta_pi       2.73468     2.73468     0.00000
            s.d. Theta_pi       1.61499     1.61499     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_19_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       2.86719     2.86719     0.00000
              Tau qt 2.5%       1.94531     1.94531     0.00000
                Tau qt 5%       1.94531     1.94531     0.00000
               Tau qt 95%       2.95703     2.95703     0.00000
             Tau qt 97.5%       2.95703     2.95703     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.51152     0.51152     0.00000
          Theta0 qt 97.5%       0.51152     0.51152     0.00000
                   Theta1      67.26562    67.26562     0.00000
           Theta1 qt 2.5%      17.30469    17.30469     0.00000
             Theta1 qt 5%      17.30469    17.30469     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00038     0.00038     0.00000
      Model (SSD) p-value       0.80000     0.80000     0.00000
         Raggedness index       0.02646     0.02646     0.00000
       Raggedness p-value       0.70000     0.70000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 240 (31/01/17 at 14:44:19))
Total computing time for this run : 0h 0m 0s 116 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

