
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at15_00_25"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 295 (31/01/17 at 15:00:25)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at15_00_25_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at15_00_25_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 46
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.11356
     2       150         2   0.28850
     3       150         2   0.50327
     4       150         2   0.03946
     5       150         2   0.07732
     6       150         2   0.47105
     7       150         2   0.08957
     8       150         2   0.08957
     9       150         2   0.50300
    10       150         2   0.11356
    11       150         2   0.46004
    12       150         2   0.23266
    13       150         2   0.03946
    14       150         2   0.01333
    15       150         2   0.50327
    16       150         2   0.12528
    17       150         2   0.05226
    18       150         2   0.17038
    19       150         2   0.11356
    20       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.20062
  s.d.     0.000     0.000   0.18312
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0427
Gene diversity           :  0.9638 +/-  0.0048

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 46

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.012438 +/-     2.016399

Average gene diversity over loci              :     0.200622 +/-     0.111598

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    24.874256
  S.D. Theta(Hom) :     3.649341

         Theta(k) :    22.270418
95 % confidence interval limits for theta(k) : [  15.393436,  31.891169 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     4.012438
   S.D. Theta(Pi) :     2.231958

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     405       0.000
       1     750       0.000
       2    1542       0.000
       3    1804       0.000
       4    2159       0.000
       5    1877       0.000
       6    1364       0.000
       7     928       0.000
       8     223       0.000
       9     122       0.000
      10       1       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.012
Mismatch observed variance          :        3.896
Tau                                 :        4.340
Theta0                              :        0.018
Theta1                              :       52.422

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00113437
          P(Sim. Ssd >= Obs. Ssd): 0.50000000
    Harpending's Raggedness index: 0.01598018
        P(Sim. Rag. >= Obs. Rag.): 0.70000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        405.0        353.4        338.5
          1.0        750.0        787.5        766.2
          2.0       1542.0       1467.3       1415.9
          3.0       1804.0       2125.8       1944.7
          4.0       2159.0       2153.6       2067.5
          5.0       1877.0       1744.0       1779.2
          6.0       1364.0       1225.7       1282.6
          7.0        928.0        728.5        794.7
          8.0        223.0        347.5        431.5
          9.0        122.0        155.0        208.4
         10.0          1.0         65.4         90.7
         11.0          0.0         18.2         35.9
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.340           2.975           5.395           4.221
                   Theta0:           0.018           0.000           0.014           0.004               
                   Theta1:          52.422          19.869       99999.000       70011.998
Mean No. of pairwise diff.           4.012           3.038           5.091           4.023
           No. pol. sites:              20              79             110          92.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        405.0        168.0        732.0        338.5
          1.0        750.0        211.0       1554.0        766.2
          2.0       1542.0        547.0       2622.0       1415.9
          3.0       1804.0       1274.0       2857.0       1944.7
          4.0       2159.0       1752.0       2655.0       2067.5
          5.0       1877.0       1046.0       2307.0       1779.2
          6.0       1364.0        552.0       1865.0       1282.6
          7.0        928.0        207.0       1390.0        794.7
          8.0        223.0         43.0        835.0        431.5
          9.0        122.0          0.0        444.0        208.4
         10.0          1.0          0.0        246.0         90.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.340           2.975           5.395           4.221
                   Theta0:           0.018           0.000           0.014           0.004               
                   Theta1:          52.422          19.869       99999.000       70011.998
Mean No. of pairwise diff.           4.012           3.038           5.091           4.023
           No. pol. sites:              20              79             110          92.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        405.0        168.0        732.0        338.5
          1.0        750.0        211.0       1554.0        766.2
          2.0       1542.0        547.0       2622.0       1415.9
          3.0       1804.0       1274.0       2857.0       1944.7
          4.0       2159.0       1752.0       2655.0       2067.5
          5.0       1877.0       1046.0       2307.0       1779.2
          6.0       1364.0        552.0       1865.0       1282.6
          7.0        928.0        207.0       1390.0        794.7
          8.0        223.0         43.0        835.0        431.5
          9.0        122.0          0.0        444.0        208.4
         10.0          1.0          0.0        246.0         90.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.340           2.975           5.395           4.221
                   Theta0:           0.018           0.000           0.014           0.004               
                   Theta1:          52.422          19.869       99999.000       70011.998
Mean No. of pairwise diff.           4.012           3.038           5.091           4.023
           No. pol. sites:              20              79             110          92.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        405.0        168.0        732.0        338.5
          1.0        750.0        211.0       1554.0        766.2
          2.0       1542.0        547.0       2622.0       1415.9
          3.0       1804.0       1274.0       2857.0       1944.7
          4.0       2159.0       1752.0       2655.0       2067.5
          5.0       1877.0       1046.0       2307.0       1779.2
          6.0       1364.0        552.0       1865.0       1282.6
          7.0        928.0        207.0       1390.0        794.7
          8.0        223.0         43.0        835.0        431.5
          9.0        122.0          0.0        444.0        208.4
         10.0          1.0          0.0        246.0         90.7
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at15_00_25_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at15_00_25_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.11356     0.11356     0.00000     0.11356
     2       0.28850     0.28850     0.00000     0.28850
     3       0.50327     0.50327     0.00000     0.50327
     4       0.03946     0.03946     0.00000     0.03946
     5       0.07732     0.07732     0.00000     0.07732
     6       0.47105     0.47105     0.00000     0.47105
     7       0.08957     0.08957     0.00000     0.08957
     8       0.08957     0.08957     0.00000     0.08957
     9       0.50300     0.50300     0.00000     0.50300
    10       0.11356     0.11356     0.00000     0.11356
    11       0.46004     0.46004     0.00000     0.46004
    12       0.23266     0.23266     0.00000     0.23266
    13       0.03946     0.03946     0.00000     0.03946
    14       0.01333     0.01333     0.00000     0.01333
    15       0.50327     0.50327     0.00000     0.50327
    16       0.12528     0.12528     0.00000     0.12528
    17       0.05226     0.05226     0.00000     0.05226
    18       0.17038     0.17038     0.00000     0.17038
    19       0.11356     0.11356     0.00000     0.11356
    20       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.20062     0.20062     0.00000     0.20062
  s.d.       0.18312     0.18312     0.00000     0.18312
--------------------------------------------------------
<A NAME="31_01_17at15_00_25_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at15_00_25_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      22.27042    22.27042     0.00000
            Theta_k_lower      15.39344    15.39344     0.00000
            Theta_k_upper      31.89117    31.89117     0.00000
                  Theta_H      24.87426    24.87426     0.00000
             s.d. Theta_H       3.64934     3.64934     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       4.01244     4.01244     0.00000
            s.d. Theta_pi       2.23196     2.23196     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at15_00_25_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.33984     4.33984     0.00000
              Tau qt 2.5%       2.97461     2.97461     0.00000
                Tau qt 5%       2.97461     2.97461     0.00000
               Tau qt 95%       5.31836     5.31836     0.00000
             Tau qt 97.5%       5.31836     5.31836     0.00000
                   Theta0       0.01758     0.01758     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.01230     0.01230     0.00000
          Theta0 qt 97.5%       0.01230     0.01230     0.00000
                   Theta1      52.42188    52.42188     0.00000
           Theta1 qt 2.5%      19.86938    19.86938     0.00000
             Theta1 qt 5%      19.86938    19.86938     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00113     0.00113     0.00000
      Model (SSD) p-value       0.50000     0.50000     0.00000
         Raggedness index       0.01598     0.01598     0.00000
       Raggedness p-value       0.70000     0.70000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 295 (31/01/17 at 15:00:25))
Total computing time for this run : 0h 0m 0s 130 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

