
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_29_27"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 84 (31/01/17 at 16:29:27)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_29_27_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_29_27_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 55
No. of loci              : 69
No. of usable loci       : 69 loci with less than  5.00 % missing data
No. of polymorphic sites : 69


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.02649
     2       150         2   0.01333
     3       150         2   0.01333
     4       150         2   0.01333
     5       150         2   0.01333
     6       150         2   0.01333
     7       150         2   0.02649
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.06488
    11       150         2   0.05226
    12       150         2   0.02649
    13       150         2   0.01333
    14       150         2   0.01333
    15       150         2   0.01333
    16       150         2   0.01333
    17       150         2   0.01333
    18       150         2   0.02649
    19       150         2   0.02649
    20       150         2   0.01333
    21       150         2   0.02649
    22       150         2   0.01333
    23       150         2   0.02649
    24       150         2   0.03946
    25       150         2   0.02649
    26       150         2   0.01333
    27       150         2   0.02649
    28       150         2   0.02649
    29       150         2   0.02649
    30       150         2   0.01333
    31       150         2   0.02649
    32       150         2   0.03946
    33       150         2   0.01333
    34       150         2   0.01333
    35       150         2   0.06488
    36       150         2   0.07732
    37       150         2   0.05226
    38       150         2   0.05226
    39       150         2   0.05226
    40       150         2   0.02649
    41       150         2   0.01333
    42       150         2   0.02649
    43       150         2   0.01333
    44       150         2   0.01333
    45       150         2   0.01333
    46       150         2   0.01333
    47       150         2   0.02649
    48       150         2   0.02649
    49       150         2   0.01333
    50       150         2   0.01333
    51       150         2   0.01333
    52       150         2   0.02649
    53       150         2   0.02649
    54       150         2   0.01333
    55       150         2   0.02649
    56       150         2   0.01333
    57       150         2   0.14819
    58       150         2   0.01333
    59       150         2   0.05226
    60       150         2   0.01333
    61       150         2   0.02649
    62       150         2   0.02649
    63       150         2   0.01333
    64       150         2   0.03946
    65       150         2   0.01333
    66       150         2   0.01333
    67       150         2   0.01333
    68       150         2   0.02649
    69       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.02586
  s.d.     0.000     0.000   0.02107
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.1124
Gene diversity           :  0.8936 +/-  0.0224

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 55

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 69
Number of usable nucleotide sites            : 69


Distance method : Pairwise difference
Mean number of pairwise differences           :     1.784340 +/-     1.037906

Average gene diversity over loci              :     0.025860 +/-     0.016650

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     7.144163
  S.D. Theta(Hom) :     1.852596

         Theta(k) :    30.875075
95 % confidence interval limits for theta(k) : [  21.743737,  43.533340 ]

         Theta(S) :    12.355596
    S.D. Theta(S) :     3.117869

        Theta(Pi) :     1.784340
   S.D. Theta(Pi) :     1.148861

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1189       0.000
       1    3556       0.000
       2    3769       0.000
       3    1930       0.000
       4     608       0.000
       5     114       0.000
       6       9       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        1.784
Mismatch observed variance          :        1.192
Tau                                 :        1.889
Theta0                              :        0.000
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00877590
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.08834681
        P(Sim. Rag. >= Obs. Rag.): 0.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1189.0       1779.8       1690.5
          1.0       3556.0       3319.2       3192.7
          2.0       3769.0       2992.8       3015.0
          3.0       1930.0       1844.9       1898.1
          4.0        608.0        838.2        896.2
          5.0        114.0        297.0        338.5
          6.0          9.0         83.1        106.6
          7.0          0.0         17.3         28.8
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.889           1.682           2.230           1.853
                   Theta0:           0.000           0.000           0.028           0.005               
                   Theta1:       99999.000           6.490       99999.000       60026.890
Mean No. of pairwise diff.           1.784           1.665           2.214           1.818
           No. pol. sites:              69             105             129         114.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1189.0        865.0       2346.0       1690.5
          1.0       3556.0       2788.0       3726.0       3192.7
          2.0       3769.0       2304.0       3426.0       3015.0
          3.0       1930.0       1635.0       2282.0       1898.1
          4.0        608.0        608.0       1162.0        896.2
          5.0        114.0        117.0        516.0        338.5
          6.0          9.0         20.0        137.0        106.6
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.889           1.682           2.230           1.853
                   Theta0:           0.000           0.000           0.028           0.005               
                   Theta1:       99999.000           6.490       99999.000       60026.890
Mean No. of pairwise diff.           1.784           1.665           2.214           1.818
           No. pol. sites:              69             105             129         114.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1189.0        865.0       2346.0       1690.5
          1.0       3556.0       2788.0       3726.0       3192.7
          2.0       3769.0       2304.0       3426.0       3015.0
          3.0       1930.0       1635.0       2282.0       1898.1
          4.0        608.0        608.0       1162.0        896.2
          5.0        114.0        117.0        516.0        338.5
          6.0          9.0         20.0        137.0        106.6
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.889           1.682           2.230           1.853
                   Theta0:           0.000           0.000           0.028           0.005               
                   Theta1:       99999.000           6.490       99999.000       60026.890
Mean No. of pairwise diff.           1.784           1.665           2.214           1.818
           No. pol. sites:              69             105             129         114.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1189.0        865.0       2346.0       1690.5
          1.0       3556.0       2788.0       3726.0       3192.7
          2.0       3769.0       2304.0       3426.0       3015.0
          3.0       1930.0       1635.0       2282.0       1898.1
          4.0        608.0        608.0       1162.0        896.2
          5.0        114.0        117.0        516.0        338.5
          6.0          9.0         20.0        137.0        106.6
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_29_27_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            69      69.000       0.000
  No. of usable loci            69      69.000       0.000
  No. of polym. loci            69      69.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_29_27_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.02649     0.02649     0.00000     0.02649
     2       0.01333     0.01333     0.00000     0.01333
     3       0.01333     0.01333     0.00000     0.01333
     4       0.01333     0.01333     0.00000     0.01333
     5       0.01333     0.01333     0.00000     0.01333
     6       0.01333     0.01333     0.00000     0.01333
     7       0.02649     0.02649     0.00000     0.02649
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.06488     0.06488     0.00000     0.06488
    11       0.05226     0.05226     0.00000     0.05226
    12       0.02649     0.02649     0.00000     0.02649
    13       0.01333     0.01333     0.00000     0.01333
    14       0.01333     0.01333     0.00000     0.01333
    15       0.01333     0.01333     0.00000     0.01333
    16       0.01333     0.01333     0.00000     0.01333
    17       0.01333     0.01333     0.00000     0.01333
    18       0.02649     0.02649     0.00000     0.02649
    19       0.02649     0.02649     0.00000     0.02649
    20       0.01333     0.01333     0.00000     0.01333
    21       0.02649     0.02649     0.00000     0.02649
    22       0.01333     0.01333     0.00000     0.01333
    23       0.02649     0.02649     0.00000     0.02649
    24       0.03946     0.03946     0.00000     0.03946
    25       0.02649     0.02649     0.00000     0.02649
    26       0.01333     0.01333     0.00000     0.01333
    27       0.02649     0.02649     0.00000     0.02649
    28       0.02649     0.02649     0.00000     0.02649
    29       0.02649     0.02649     0.00000     0.02649
    30       0.01333     0.01333     0.00000     0.01333
    31       0.02649     0.02649     0.00000     0.02649
    32       0.03946     0.03946     0.00000     0.03946
    33       0.01333     0.01333     0.00000     0.01333
    34       0.01333     0.01333     0.00000     0.01333
    35       0.06488     0.06488     0.00000     0.06488
    36       0.07732     0.07732     0.00000     0.07732
    37       0.05226     0.05226     0.00000     0.05226
    38       0.05226     0.05226     0.00000     0.05226
    39       0.05226     0.05226     0.00000     0.05226
    40       0.02649     0.02649     0.00000     0.02649
    41       0.01333     0.01333     0.00000     0.01333
    42       0.02649     0.02649     0.00000     0.02649
    43       0.01333     0.01333     0.00000     0.01333
    44       0.01333     0.01333     0.00000     0.01333
    45       0.01333     0.01333     0.00000     0.01333
    46       0.01333     0.01333     0.00000     0.01333
    47       0.02649     0.02649     0.00000     0.02649
    48       0.02649     0.02649     0.00000     0.02649
    49       0.01333     0.01333     0.00000     0.01333
    50       0.01333     0.01333     0.00000     0.01333
    51       0.01333     0.01333     0.00000     0.01333
    52       0.02649     0.02649     0.00000     0.02649
    53       0.02649     0.02649     0.00000     0.02649
    54       0.01333     0.01333     0.00000     0.01333
    55       0.02649     0.02649     0.00000     0.02649
    56       0.01333     0.01333     0.00000     0.01333
    57       0.14819     0.14819     0.00000     0.14819
    58       0.01333     0.01333     0.00000     0.01333
    59       0.05226     0.05226     0.00000     0.05226
    60       0.01333     0.01333     0.00000     0.01333
    61       0.02649     0.02649     0.00000     0.02649
    62       0.02649     0.02649     0.00000     0.02649
    63       0.01333     0.01333     0.00000     0.01333
    64       0.03946     0.03946     0.00000     0.03946
    65       0.01333     0.01333     0.00000     0.01333
    66       0.01333     0.01333     0.00000     0.01333
    67       0.01333     0.01333     0.00000     0.01333
    68       0.02649     0.02649     0.00000     0.02649
    69       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.02586     0.02586     0.00000     0.02586
  s.d.       0.02107     0.02107     0.00000     0.02107
--------------------------------------------------------
<A NAME="31_01_17at16_29_27_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_29_27_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      30.87507    30.87507     0.00000
            Theta_k_lower      21.74374    21.74374     0.00000
            Theta_k_upper      43.53334    43.53334     0.00000
                  Theta_H       7.14416     7.14416     0.00000
             s.d. Theta_H       1.85260     1.85260     0.00000
                  Theta_S      12.35560    12.35560     0.00000
             s.d. Theta_S       3.11787     3.11787     0.00000
                 Theta_pi       1.78434     1.78434     0.00000
            s.d. Theta_pi       1.14886     1.14886     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_29_27_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.88867     1.88867     0.00000
              Tau qt 2.5%       1.68164     1.68164     0.00000
                Tau qt 5%       1.68164     1.68164     0.00000
               Tau qt 95%       2.01562     2.01562     0.00000
             Tau qt 97.5%       2.01562     2.01562     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.01406     0.01406     0.00000
          Theta0 qt 97.5%       0.01406     0.01406     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%       6.49023     6.49023     0.00000
             Theta1 qt 5%       6.49023     6.49023     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00878     0.00878     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.08835     0.08835     0.00000
       Raggedness p-value       0.00000     0.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 84 (31/01/17 at 16:29:27))
Total computing time for this run : 0h 0m 0s 95 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

