
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_50_06"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 324 (31/01/17 at 14:50:06)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_50_06_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_50_06_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 44
No. of loci              : 65
No. of usable loci       : 65 loci with less than  5.00 % missing data
No. of polymorphic sites : 65


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.07732
     3       150         2   0.03946
     4       150         2   0.01333
     5       150         2   0.18121
     6       150         2   0.02649
     7       150         2   0.03946
     8       150         2   0.14819
     9       150         2   0.03946
    10       150         2   0.01333
    11       150         2   0.02649
    12       150         2   0.07732
    13       150         2   0.03946
    14       150         2   0.03946
    15       150         2   0.20233
    16       150         2   0.02649
    17       150         2   0.05226
    18       150         2   0.05226
    19       150         2   0.01333
    20       150         2   0.08957
    21       150         2   0.03946
    22       150         2   0.08957
    23       150         2   0.03946
    24       150         2   0.02649
    25       150         2   0.05226
    26       150         2   0.05226
    27       150         2   0.01333
    28       150         2   0.13682
    29       150         2   0.08957
    30       150         2   0.01333
    31       150         2   0.02649
    32       150         2   0.02649
    33       150         2   0.03946
    34       150         2   0.01333
    35       150         2   0.02649
    36       150         2   0.01333
    37       150         2   0.03946
    38       150         2   0.05226
    39       150         2   0.03946
    40       150         2   0.01333
    41       150         2   0.11356
    42       150         2   0.03946
    43       150         2   0.03946
    44       150         2   0.01333
    45       150         2   0.05226
    46       150         2   0.01333
    47       150         2   0.08957
    48       150         2   0.13682
    49       150         2   0.07732
    50       150         2   0.01333
    51       150         2   0.02649
    52       150         2   0.20233
    53       150         2   0.02649
    54       150         2   0.18121
    55       150         2   0.14819
    56       150         2   0.03946
    57       150         2   0.01333
    58       150         2   0.07732
    59       150         2   0.06488
    60       150         2   0.14819
    61       150         2   0.13682
    62       150         2   0.03946
    63       150         2   0.02649
    64       150         2   0.02649
    65       150         2   0.08957
-------------------------------------------------
  Mean   150.000     2.000   0.05982
  s.d.     0.000     0.000   0.05098
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0596
Gene diversity           :  0.9467 +/-  0.0106

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 44

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 65
Number of usable nucleotide sites            : 65


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.888412 +/-     1.962532

Average gene diversity over loci              :     0.059822 +/-     0.033421

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    16.156120
  S.D. Theta(Hom) :     3.657888

         Theta(k) :    20.605975
95 % confidence interval limits for theta(k) : [  14.167681,  29.640070 ]

         Theta(S) :    11.639330
    S.D. Theta(S) :     2.956600

        Theta(Pi) :     3.888412
   S.D. Theta(Pi) :     2.172333

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     596       0.000
       1     663       0.000
       2    1254       0.000
       3    2203       0.000
       4    1888       0.000
       5    2093       0.000
       6    1753       0.000
       7     662       0.000
       8      63       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.888
Mismatch observed variance          :        3.459
Tau                                 :        4.391
Theta0                              :        0.019
Theta1                              :       55.234

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00535231
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.02453761
        P(Sim. Rag. >= Obs. Rag.): 0.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        596.0        284.2        321.9
          1.0        663.0        685.2        736.3
          2.0       1254.0       1329.2       1377.8
          3.0       2203.0       1855.0       1916.2
          4.0       1888.0       2047.6       2062.5
          5.0       2093.0       1835.6       1796.8
          6.0       1753.0       1331.1       1311.1
          7.0        662.0        878.9        822.2
          8.0         63.0        518.3        451.8
          9.0          0.0        247.0        220.9
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.391           3.340           5.320           4.196
                   Theta0:           0.019           0.000           1.134           0.335               
                   Theta1:          55.234          34.219       99999.000       30102.935
Mean No. of pairwise diff.           3.888           3.719           4.910           4.338
           No. pol. sites:              65              84             112          99.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        596.0        199.0        467.0        321.9
          1.0        663.0        485.0       1107.0        736.3
          2.0       1254.0        901.0       1728.0       1377.8
          3.0       2203.0       1384.0       2347.0       1916.2
          4.0       1888.0       1748.0       2295.0       2062.5
          5.0       2093.0       1578.0       2316.0       1796.8
          6.0       1753.0        975.0       1687.0       1311.1
          7.0        662.0        630.0       1312.0        822.2
          8.0         63.0        277.0        792.0        451.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.391           3.340           5.320           4.196
                   Theta0:           0.019           0.000           1.134           0.335               
                   Theta1:          55.234          34.219       99999.000       30102.935
Mean No. of pairwise diff.           3.888           3.719           4.910           4.338
           No. pol. sites:              65              84             112          99.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        596.0        199.0        467.0        321.9
          1.0        663.0        485.0       1107.0        736.3
          2.0       1254.0        901.0       1728.0       1377.8
          3.0       2203.0       1384.0       2347.0       1916.2
          4.0       1888.0       1748.0       2295.0       2062.5
          5.0       2093.0       1578.0       2316.0       1796.8
          6.0       1753.0        975.0       1687.0       1311.1
          7.0        662.0        630.0       1312.0        822.2
          8.0         63.0        277.0        792.0        451.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.391           3.340           5.320           4.196
                   Theta0:           0.019           0.000           1.134           0.335               
                   Theta1:          55.234          34.219       99999.000       30102.935
Mean No. of pairwise diff.           3.888           3.719           4.910           4.338
           No. pol. sites:              65              84             112          99.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        596.0        199.0        467.0        321.9
          1.0        663.0        485.0       1107.0        736.3
          2.0       1254.0        901.0       1728.0       1377.8
          3.0       2203.0       1384.0       2347.0       1916.2
          4.0       1888.0       1748.0       2295.0       2062.5
          5.0       2093.0       1578.0       2316.0       1796.8
          6.0       1753.0        975.0       1687.0       1311.1
          7.0        662.0        630.0       1312.0        822.2
          8.0         63.0        277.0        792.0        451.8
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_50_06_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            65      65.000       0.000
  No. of usable loci            65      65.000       0.000
  No. of polym. loci            65      65.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_50_06_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.07732     0.07732     0.00000     0.07732
     3       0.03946     0.03946     0.00000     0.03946
     4       0.01333     0.01333     0.00000     0.01333
     5       0.18121     0.18121     0.00000     0.18121
     6       0.02649     0.02649     0.00000     0.02649
     7       0.03946     0.03946     0.00000     0.03946
     8       0.14819     0.14819     0.00000     0.14819
     9       0.03946     0.03946     0.00000     0.03946
    10       0.01333     0.01333     0.00000     0.01333
    11       0.02649     0.02649     0.00000     0.02649
    12       0.07732     0.07732     0.00000     0.07732
    13       0.03946     0.03946     0.00000     0.03946
    14       0.03946     0.03946     0.00000     0.03946
    15       0.20233     0.20233     0.00000     0.20233
    16       0.02649     0.02649     0.00000     0.02649
    17       0.05226     0.05226     0.00000     0.05226
    18       0.05226     0.05226     0.00000     0.05226
    19       0.01333     0.01333     0.00000     0.01333
    20       0.08957     0.08957     0.00000     0.08957
    21       0.03946     0.03946     0.00000     0.03946
    22       0.08957     0.08957     0.00000     0.08957
    23       0.03946     0.03946     0.00000     0.03946
    24       0.02649     0.02649     0.00000     0.02649
    25       0.05226     0.05226     0.00000     0.05226
    26       0.05226     0.05226     0.00000     0.05226
    27       0.01333     0.01333     0.00000     0.01333
    28       0.13682     0.13682     0.00000     0.13682
    29       0.08957     0.08957     0.00000     0.08957
    30       0.01333     0.01333     0.00000     0.01333
    31       0.02649     0.02649     0.00000     0.02649
    32       0.02649     0.02649     0.00000     0.02649
    33       0.03946     0.03946     0.00000     0.03946
    34       0.01333     0.01333     0.00000     0.01333
    35       0.02649     0.02649     0.00000     0.02649
    36       0.01333     0.01333     0.00000     0.01333
    37       0.03946     0.03946     0.00000     0.03946
    38       0.05226     0.05226     0.00000     0.05226
    39       0.03946     0.03946     0.00000     0.03946
    40       0.01333     0.01333     0.00000     0.01333
    41       0.11356     0.11356     0.00000     0.11356
    42       0.03946     0.03946     0.00000     0.03946
    43       0.03946     0.03946     0.00000     0.03946
    44       0.01333     0.01333     0.00000     0.01333
    45       0.05226     0.05226     0.00000     0.05226
    46       0.01333     0.01333     0.00000     0.01333
    47       0.08957     0.08957     0.00000     0.08957
    48       0.13682     0.13682     0.00000     0.13682
    49       0.07732     0.07732     0.00000     0.07732
    50       0.01333     0.01333     0.00000     0.01333
    51       0.02649     0.02649     0.00000     0.02649
    52       0.20233     0.20233     0.00000     0.20233
    53       0.02649     0.02649     0.00000     0.02649
    54       0.18121     0.18121     0.00000     0.18121
    55       0.14819     0.14819     0.00000     0.14819
    56       0.03946     0.03946     0.00000     0.03946
    57       0.01333     0.01333     0.00000     0.01333
    58       0.07732     0.07732     0.00000     0.07732
    59       0.06488     0.06488     0.00000     0.06488
    60       0.14819     0.14819     0.00000     0.14819
    61       0.13682     0.13682     0.00000     0.13682
    62       0.03946     0.03946     0.00000     0.03946
    63       0.02649     0.02649     0.00000     0.02649
    64       0.02649     0.02649     0.00000     0.02649
    65       0.08957     0.08957     0.00000     0.08957
--------------------------------------------------------
  Mean       0.05982     0.05982     0.00000     0.05982
  s.d.       0.05098     0.05098     0.00000     0.05098
--------------------------------------------------------
<A NAME="31_01_17at14_50_06_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_50_06_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      20.60597    20.60597     0.00000
            Theta_k_lower      14.16768    14.16768     0.00000
            Theta_k_upper      29.64007    29.64007     0.00000
                  Theta_H      16.15612    16.15612     0.00000
             s.d. Theta_H       3.65789     3.65789     0.00000
                  Theta_S      11.63933    11.63933     0.00000
             s.d. Theta_S       2.95660     2.95660     0.00000
                 Theta_pi       3.88841     3.88841     0.00000
            s.d. Theta_pi       2.17233     2.17233     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_50_06_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.39062     4.39062     0.00000
              Tau qt 2.5%       3.33984     3.33984     0.00000
                Tau qt 5%       3.33984     3.33984     0.00000
               Tau qt 95%       4.61328     4.61328     0.00000
             Tau qt 97.5%       4.61328     4.61328     0.00000
                   Theta0       0.01934     0.01934     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.09336     1.09336     0.00000
          Theta0 qt 97.5%       1.09336     1.09336     0.00000
                   Theta1      55.23438    55.23438     0.00000
           Theta1 qt 2.5%      34.21875    34.21875     0.00000
             Theta1 qt 5%      34.21875    34.21875     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00535     0.00535     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.02454     0.02454     0.00000
       Raggedness p-value       0.00000     0.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 324 (31/01/17 at 14:50:06))
Total computing time for this run : 0h 0m 0s 132 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

