
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_52_07"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 245 (31/01/17 at 14:52:07)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_52_07_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_52_07_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 44
No. of loci              : 18
No. of usable loci       : 18 loci with less than  5.00 % missing data
No. of polymorphic sites : 18


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.07732
     2       150         2   0.50013
     3       150         2   0.21262
     4       150         2   0.07732
     5       150         2   0.20233
     6       150         2   0.30568
     7       150         2   0.01333
     8       150         2   0.02649
     9       150         2   0.36725
    10       150         2   0.22273
    11       150         2   0.03946
    12       150         2   0.32215
    13       150         2   0.08957
    14       150         2   0.02649
    15       150         2   0.37414
    16       150         2   0.08957
    17       150         2   0.45602
    18       150         2   0.10166
-------------------------------------------------
  Mean   150.000     2.000   0.19468
  s.d.     0.000     0.000   0.15815
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0575
Gene diversity           :  0.9488 +/-  0.0094

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 44

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 18
Number of usable nucleotide sites            : 18


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.504251 +/-     1.795422

Average gene diversity over loci              :     0.194681 +/-     0.110409

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    16.927562
  S.D. Theta(Hom) :     3.538598

         Theta(k) :    20.605975
95 % confidence interval limits for theta(k) : [  14.167681,  29.640070 ]

         Theta(S) :     3.223199
    S.D. Theta(S) :     1.032890

        Theta(Pi) :     3.504251
   S.D. Theta(Pi) :     1.987359

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     572     396.508
       1    1347    1277.609
       2    1892    2091.406
       3    2095    2326.347
       4    1799    1985.737
       5    1495    1393.833
       6    1076     842.611
       7     635     454.029
       8     212     224.095
       9      44     103.617
      10       8      45.724
      11       0      19.547
      12       0       8.189
      13       0       3.389
      14       0       1.393
      15       0       0.571
      16       0       0.233
      17       0       0.095
      18       0       0.039
      19       0       0.016
      20       0       0.007


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.504
Mismatch observed variance          :        3.981
Tau                                 :        2.988
Theta0                              :        0.859
Theta1                              :       30.234

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00068235
          P(Sim. Ssd >= Obs. Ssd): 0.90000000
    Harpending's Raggedness index: 0.01359255
        P(Sim. Rag. >= Obs. Rag.): 0.90000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        572.0        607.0        615.7
          1.0       1347.0       1246.8       1228.1
          2.0       1892.0       1854.6       1861.7
          3.0       2095.0       2247.5       2108.7
          4.0       1799.0       2117.5       1892.7
          5.0       1495.0       1580.9       1418.7
          6.0       1076.0        883.6        925.1
          7.0        635.0        431.7        542.1
          8.0        212.0        157.0        293.2
          9.0         44.0         39.8        149.7
         10.0          8.0          8.0         73.4
         11.0          0.0          0.6         35.1
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.988           2.105           5.166           3.632
                   Theta0:           0.859           0.000           0.833           0.085               
                   Theta1:          30.234          18.242       99999.000       50017.799
Mean No. of pairwise diff.           3.504           2.044           4.368           3.409
           No. pol. sites:              18              48              68          58.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        572.0        229.0       1239.0        615.7
          1.0       1347.0        394.0       2956.0       1228.1
          2.0       1892.0       1087.0       3134.0       1861.7
          3.0       2095.0       1495.0       3149.0       2108.7
          4.0       1799.0       1058.0       2616.0       1892.7
          5.0       1495.0        367.0       2396.0       1418.7
          6.0       1076.0         84.0       1732.0        925.1
          7.0        635.0         10.0       1100.0        542.1
          8.0        212.0          0.0        679.0        293.2
          9.0         44.0          0.0        269.0        149.7
         10.0          8.0          0.0         56.0         73.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.988           2.105           5.166           3.632
                   Theta0:           0.859           0.000           0.833           0.085               
                   Theta1:          30.234          18.242       99999.000       50017.799
Mean No. of pairwise diff.           3.504           2.044           4.368           3.409
           No. pol. sites:              18              48              68          58.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        572.0        229.0       1239.0        615.7
          1.0       1347.0        394.0       2956.0       1228.1
          2.0       1892.0       1087.0       3134.0       1861.7
          3.0       2095.0       1495.0       3149.0       2108.7
          4.0       1799.0       1058.0       2616.0       1892.7
          5.0       1495.0        367.0       2396.0       1418.7
          6.0       1076.0         84.0       1732.0        925.1
          7.0        635.0         10.0       1100.0        542.1
          8.0        212.0          0.0        679.0        293.2
          9.0         44.0          0.0        269.0        149.7
         10.0          8.0          0.0         56.0         73.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.988           2.105           5.166           3.632
                   Theta0:           0.859           0.000           0.833           0.085               
                   Theta1:          30.234          18.242       99999.000       50017.799
Mean No. of pairwise diff.           3.504           2.044           4.368           3.409
           No. pol. sites:              18              48              68          58.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        572.0        229.0       1239.0        615.7
          1.0       1347.0        394.0       2956.0       1228.1
          2.0       1892.0       1087.0       3134.0       1861.7
          3.0       2095.0       1495.0       3149.0       2108.7
          4.0       1799.0       1058.0       2616.0       1892.7
          5.0       1495.0        367.0       2396.0       1418.7
          6.0       1076.0         84.0       1732.0        925.1
          7.0        635.0         10.0       1100.0        542.1
          8.0        212.0          0.0        679.0        293.2
          9.0         44.0          0.0        269.0        149.7
         10.0          8.0          0.0         56.0         73.4
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_52_07_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            18      18.000       0.000
  No. of usable loci            18      18.000       0.000
  No. of polym. loci            18      18.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_52_07_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.07732     0.07732     0.00000     0.07732
     2       0.50013     0.50013     0.00000     0.50013
     3       0.21262     0.21262     0.00000     0.21262
     4       0.07732     0.07732     0.00000     0.07732
     5       0.20233     0.20233     0.00000     0.20233
     6       0.30568     0.30568     0.00000     0.30568
     7       0.01333     0.01333     0.00000     0.01333
     8       0.02649     0.02649     0.00000     0.02649
     9       0.36725     0.36725     0.00000     0.36725
    10       0.22273     0.22273     0.00000     0.22273
    11       0.03946     0.03946     0.00000     0.03946
    12       0.32215     0.32215     0.00000     0.32215
    13       0.08957     0.08957     0.00000     0.08957
    14       0.02649     0.02649     0.00000     0.02649
    15       0.37414     0.37414     0.00000     0.37414
    16       0.08957     0.08957     0.00000     0.08957
    17       0.45602     0.45602     0.00000     0.45602
    18       0.10166     0.10166     0.00000     0.10166
--------------------------------------------------------
  Mean       0.19468     0.19468     0.00000     0.19468
  s.d.       0.15815     0.15815     0.00000     0.15815
--------------------------------------------------------
<A NAME="31_01_17at14_52_07_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_52_07_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      20.60597    20.60597     0.00000
            Theta_k_lower      14.16768    14.16768     0.00000
            Theta_k_upper      29.64007    29.64007     0.00000
                  Theta_H      16.92756    16.92756     0.00000
             s.d. Theta_H       3.53860     3.53860     0.00000
                  Theta_S       3.22320     3.22320     0.00000
             s.d. Theta_S       1.03289     1.03289     0.00000
                 Theta_pi       3.50425     3.50425     0.00000
            s.d. Theta_pi       1.98736     1.98736     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_52_07_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       2.98828     2.98828     0.00000
              Tau qt 2.5%       2.10547     2.10547     0.00000
                Tau qt 5%       2.10547     2.10547     0.00000
               Tau qt 95%       4.64453     4.64453     0.00000
             Tau qt 97.5%       4.64453     4.64453     0.00000
                   Theta0       0.85938     0.85938     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00937     0.00937     0.00000
          Theta0 qt 97.5%       0.00937     0.00937     0.00000
                   Theta1      30.23438    30.23438     0.00000
           Theta1 qt 2.5%      18.24219    18.24219     0.00000
             Theta1 qt 5%      18.24219    18.24219     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00068     0.00068     0.00000
      Model (SSD) p-value       0.90000     0.90000     0.00000
         Raggedness index       0.01359     0.01359     0.00000
       Raggedness p-value       0.90000     0.90000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 245 (31/01/17 at 14:52:07))
Total computing time for this run : 0h 0m 0s 122 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

