
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_43_56"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 64 (31/01/17 at 14:43:56)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_43_56_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_43_56_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 55
No. of loci              : 72
No. of usable loci       : 72 loci with less than  5.00 % missing data
No. of polymorphic sites : 72


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.06488
     2       150         2   0.01333
     3       150         2   0.03946
     4       150         2   0.12528
     5       150         2   0.07732
     6       150         2   0.01333
     7       150         2   0.02649
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.01333
    11       150         2   0.12528
    12       150         2   0.01333
    13       150         2   0.01333
    14       150         2   0.12528
    15       150         2   0.01333
    16       150         2   0.01333
    17       150         2   0.01333
    18       150         2   0.02649
    19       150         2   0.05226
    20       150         2   0.02649
    21       150         2   0.02649
    22       150         2   0.02649
    23       150         2   0.05226
    24       150         2   0.03946
    25       150         2   0.10166
    26       150         2   0.01333
    27       150         2   0.10166
    28       150         2   0.01333
    29       150         2   0.08957
    30       150         2   0.01333
    31       150         2   0.01333
    32       150         2   0.06488
    33       150         2   0.01333
    34       150         2   0.03946
    35       150         2   0.02649
    36       150         2   0.02649
    37       150         2   0.05226
    38       150         2   0.02649
    39       150         2   0.01333
    40       150         2   0.01333
    41       150         2   0.01333
    42       150         2   0.02649
    43       150         2   0.12528
    44       150         2   0.01333
    45       150         2   0.01333
    46       150         2   0.19186
    47       150         2   0.02649
    48       150         2   0.02649
    49       150         2   0.20233
    50       150         2   0.02649
    51       150         2   0.01333
    52       150         2   0.08957
    53       150         2   0.01333
    54       150         2   0.07732
    55       150         2   0.01333
    56       150         2   0.01333
    57       150         2   0.05226
    58       150         2   0.03946
    59       150         2   0.01333
    60       150         2   0.03946
    61       150         2   0.02649
    62       150         2   0.01333
    63       150         2   0.01333
    64       150         2   0.02649
    65       150         2   0.01333
    66       150         2   0.01333
    67       150         2   0.02649
    68       150         2   0.01333
    69       150         2   0.02649
    70       150         2   0.02649
    71       150         2   0.12528
    72       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.04108
  s.d.     0.000     0.000   0.04246
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0951
Gene diversity           :  0.9110 +/-  0.0191

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 55

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 72
Number of usable nucleotide sites            : 72


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.957405 +/-     1.556664

Average gene diversity over loci              :     0.041075 +/-     0.023932

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     8.873921
  S.D. Theta(Hom) :     2.295700

         Theta(k) :    30.875075
95 % confidence interval limits for theta(k) : [  21.743737,  43.533340 ]

         Theta(S) :    12.892796
    S.D. Theta(S) :     3.238739

        Theta(Pi) :     2.957405
   S.D. Theta(Pi) :     1.723077

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     995     637.043
       1    1569    1779.655
       2    2010    2520.740
       3    2518    2420.031
       4    1873    1777.259
       5    1255    1069.094
       6     639     551.221
       7     235     251.884
       8      69     104.747
       9      12      40.520
      10       0      14.853
      11       0       5.239
      12       0       1.800
      13       0       0.608
      14       0       0.204
      15       0       0.068
      16       0       0.022
      17       0       0.007
      18       0       0.002


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.957
Mismatch observed variance          :        3.202
Tau                                 :        3.367
Theta0                              :        0.004
Theta1                              :       17.451

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00092691
          P(Sim. Ssd >= Obs. Ssd): 0.80000000
    Harpending's Raggedness index: 0.01724520
        P(Sim. Rag. >= Obs. Rag.): 0.80000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        995.0        760.9        905.1
          1.0       1569.0       1560.7       1565.9
          2.0       2010.0       2406.1       2172.7
          3.0       2518.0       2328.3       2265.1
          4.0       1873.0       1908.1       1841.7
          5.0       1255.0       1168.7       1217.1
          6.0        639.0        608.2        675.4
          7.0        235.0        243.1        322.5
          8.0         69.0        119.4        135.1
          9.0         12.0         53.1         50.4
         10.0          0.0         14.1         16.9
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.367           2.160           4.855           3.216
                   Theta0:           0.004           0.000           0.016           0.003               
                   Theta1:          17.451           7.041       99999.000       60053.891
Mean No. of pairwise diff.           2.957           2.005           4.524           3.025
           No. pol. sites:              72              38              72          47.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        995.0        275.0       2029.0        905.1
          1.0       1569.0        510.0       2840.0       1565.9
          2.0       2010.0       1134.0       3839.0       2172.7
          3.0       2518.0       1599.0       2920.0       2265.1
          4.0       1873.0        840.0       2865.0       1841.7
          5.0       1255.0        274.0       2032.0       1217.1
          6.0        639.0         52.0       1491.0        675.4
          7.0        235.0          0.0       1020.0        322.5
          8.0         69.0          0.0        628.0        135.1
          9.0         12.0          0.0        313.0         50.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.367           2.160           4.855           3.216
                   Theta0:           0.004           0.000           0.016           0.003               
                   Theta1:          17.451           7.041       99999.000       60053.891
Mean No. of pairwise diff.           2.957           2.005           4.524           3.025
           No. pol. sites:              72              38              72          47.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        995.0        275.0       2029.0        905.1
          1.0       1569.0        510.0       2840.0       1565.9
          2.0       2010.0       1134.0       3839.0       2172.7
          3.0       2518.0       1599.0       2920.0       2265.1
          4.0       1873.0        840.0       2865.0       1841.7
          5.0       1255.0        274.0       2032.0       1217.1
          6.0        639.0         52.0       1491.0        675.4
          7.0        235.0          0.0       1020.0        322.5
          8.0         69.0          0.0        628.0        135.1
          9.0         12.0          0.0        313.0         50.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.367           2.160           4.855           3.216
                   Theta0:           0.004           0.000           0.016           0.003               
                   Theta1:          17.451           7.041       99999.000       60053.891
Mean No. of pairwise diff.           2.957           2.005           4.524           3.025
           No. pol. sites:              72              38              72          47.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        995.0        275.0       2029.0        905.1
          1.0       1569.0        510.0       2840.0       1565.9
          2.0       2010.0       1134.0       3839.0       2172.7
          3.0       2518.0       1599.0       2920.0       2265.1
          4.0       1873.0        840.0       2865.0       1841.7
          5.0       1255.0        274.0       2032.0       1217.1
          6.0        639.0         52.0       1491.0        675.4
          7.0        235.0          0.0       1020.0        322.5
          8.0         69.0          0.0        628.0        135.1
          9.0         12.0          0.0        313.0         50.4
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_43_56_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            72      72.000       0.000
  No. of usable loci            72      72.000       0.000
  No. of polym. loci            72      72.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_43_56_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.06488     0.06488     0.00000     0.06488
     2       0.01333     0.01333     0.00000     0.01333
     3       0.03946     0.03946     0.00000     0.03946
     4       0.12528     0.12528     0.00000     0.12528
     5       0.07732     0.07732     0.00000     0.07732
     6       0.01333     0.01333     0.00000     0.01333
     7       0.02649     0.02649     0.00000     0.02649
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.01333     0.01333     0.00000     0.01333
    11       0.12528     0.12528     0.00000     0.12528
    12       0.01333     0.01333     0.00000     0.01333
    13       0.01333     0.01333     0.00000     0.01333
    14       0.12528     0.12528     0.00000     0.12528
    15       0.01333     0.01333     0.00000     0.01333
    16       0.01333     0.01333     0.00000     0.01333
    17       0.01333     0.01333     0.00000     0.01333
    18       0.02649     0.02649     0.00000     0.02649
    19       0.05226     0.05226     0.00000     0.05226
    20       0.02649     0.02649     0.00000     0.02649
    21       0.02649     0.02649     0.00000     0.02649
    22       0.02649     0.02649     0.00000     0.02649
    23       0.05226     0.05226     0.00000     0.05226
    24       0.03946     0.03946     0.00000     0.03946
    25       0.10166     0.10166     0.00000     0.10166
    26       0.01333     0.01333     0.00000     0.01333
    27       0.10166     0.10166     0.00000     0.10166
    28       0.01333     0.01333     0.00000     0.01333
    29       0.08957     0.08957     0.00000     0.08957
    30       0.01333     0.01333     0.00000     0.01333
    31       0.01333     0.01333     0.00000     0.01333
    32       0.06488     0.06488     0.00000     0.06488
    33       0.01333     0.01333     0.00000     0.01333
    34       0.03946     0.03946     0.00000     0.03946
    35       0.02649     0.02649     0.00000     0.02649
    36       0.02649     0.02649     0.00000     0.02649
    37       0.05226     0.05226     0.00000     0.05226
    38       0.02649     0.02649     0.00000     0.02649
    39       0.01333     0.01333     0.00000     0.01333
    40       0.01333     0.01333     0.00000     0.01333
    41       0.01333     0.01333     0.00000     0.01333
    42       0.02649     0.02649     0.00000     0.02649
    43       0.12528     0.12528     0.00000     0.12528
    44       0.01333     0.01333     0.00000     0.01333
    45       0.01333     0.01333     0.00000     0.01333
    46       0.19186     0.19186     0.00000     0.19186
    47       0.02649     0.02649     0.00000     0.02649
    48       0.02649     0.02649     0.00000     0.02649
    49       0.20233     0.20233     0.00000     0.20233
    50       0.02649     0.02649     0.00000     0.02649
    51       0.01333     0.01333     0.00000     0.01333
    52       0.08957     0.08957     0.00000     0.08957
    53       0.01333     0.01333     0.00000     0.01333
    54       0.07732     0.07732     0.00000     0.07732
    55       0.01333     0.01333     0.00000     0.01333
    56       0.01333     0.01333     0.00000     0.01333
    57       0.05226     0.05226     0.00000     0.05226
    58       0.03946     0.03946     0.00000     0.03946
    59       0.01333     0.01333     0.00000     0.01333
    60       0.03946     0.03946     0.00000     0.03946
    61       0.02649     0.02649     0.00000     0.02649
    62       0.01333     0.01333     0.00000     0.01333
    63       0.01333     0.01333     0.00000     0.01333
    64       0.02649     0.02649     0.00000     0.02649
    65       0.01333     0.01333     0.00000     0.01333
    66       0.01333     0.01333     0.00000     0.01333
    67       0.02649     0.02649     0.00000     0.02649
    68       0.01333     0.01333     0.00000     0.01333
    69       0.02649     0.02649     0.00000     0.02649
    70       0.02649     0.02649     0.00000     0.02649
    71       0.12528     0.12528     0.00000     0.12528
    72       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.04108     0.04108     0.00000     0.04108
  s.d.       0.04246     0.04246     0.00000     0.04246
--------------------------------------------------------
<A NAME="31_01_17at14_43_56_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
    72             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_43_56_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      30.87507    30.87507     0.00000
            Theta_k_lower      21.74374    21.74374     0.00000
            Theta_k_upper      43.53334    43.53334     0.00000
                  Theta_H       8.87392     8.87392     0.00000
             s.d. Theta_H       2.29570     2.29570     0.00000
                  Theta_S      12.89280    12.89280     0.00000
             s.d. Theta_S       3.23874     3.23874     0.00000
                 Theta_pi       2.95740     2.95740     0.00000
            s.d. Theta_pi       1.72308     1.72308     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_43_56_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.36719     3.36719     0.00000
              Tau qt 2.5%       2.16016     2.16016     0.00000
                Tau qt 5%       2.16016     2.16016     0.00000
               Tau qt 95%       4.06055     4.06055     0.00000
             Tau qt 97.5%       4.06055     4.06055     0.00000
                   Theta0       0.00352     0.00352     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00879     0.00879     0.00000
          Theta0 qt 97.5%       0.00879     0.00879     0.00000
                   Theta1      17.45117    17.45117     0.00000
           Theta1 qt 2.5%       7.04102     7.04102     0.00000
             Theta1 qt 5%       7.04102     7.04102     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00093     0.00093     0.00000
      Model (SSD) p-value       0.80000     0.80000     0.00000
         Raggedness index       0.01725     0.01725     0.00000
       Raggedness p-value       0.80000     0.80000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 64 (31/01/17 at 14:43:57))
Total computing time for this run : 0h 0m 0s 105 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

