
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_29_28"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 90 (31/01/17 at 16:29:28)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_29_28_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_29_28_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 50
No. of loci              : 67
No. of usable loci       : 67 loci with less than  5.00 % missing data
No. of polymorphic sites : 67


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.01333
     3       150         2   0.01333
     4       150         2   0.01333
     5       150         2   0.01333
     6       150         2   0.02649
     7       150         2   0.01333
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.01333
    11       150         2   0.01333
    12       150         2   0.01333
    13       150         2   0.01333
    14       150         2   0.01333
    15       150         2   0.01333
    16       150         2   0.01333
    17       150         2   0.01333
    18       150         2   0.01333
    19       150         2   0.01333
    20       150         2   0.01333
    21       150         2   0.01333
    22       150         2   0.02649
    23       150         2   0.01333
    24       150         2   0.06488
    25       150         2   0.03946
    26       150         2   0.01333
    27       150         2   0.01333
    28       150         2   0.02649
    29       150         2   0.01333
    30       150         2   0.01333
    31       150         2   0.01333
    32       150         2   0.01333
    33       150         2   0.01333
    34       150         2   0.01333
    35       150         2   0.01333
    36       150         2   0.01333
    37       150         2   0.01333
    38       150         2   0.01333
    39       150         2   0.01333
    40       150         2   0.02649
    41       150         2   0.01333
    42       150         2   0.02649
    43       150         2   0.01333
    44       150         2   0.01333
    45       150         2   0.01333
    46       150         2   0.03946
    47       150         2   0.01333
    48       150         2   0.01333
    49       150         2   0.02649
    50       150         2   0.03946
    51       150         2   0.02649
    52       150         2   0.02649
    53       150         2   0.03946
    54       150         2   0.01333
    55       150         2   0.01333
    56       150         2   0.03946
    57       150         2   0.03946
    58       150         2   0.01333
    59       150         2   0.01333
    60       150         2   0.01333
    61       150         2   0.01333
    62       150         2   0.02649
    63       150         2   0.01333
    64       150         2   0.01333
    65       150         2   0.01333
    66       150         2   0.01333
    67       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.01821
  s.d.     0.000     0.000   0.01009
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.3262
Gene diversity           :  0.6783 +/-  0.0450

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 50

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 67
Number of usable nucleotide sites            : 67


Distance method : Pairwise difference
Mean number of pairwise differences           :     1.220045 +/-     0.781031

Average gene diversity over loci              :     0.018210 +/-     0.012903

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     1.604823
  S.D. Theta(Hom) :     0.345437

         Theta(k) :    25.857163
95 % confidence interval limits for theta(k) : [  18.038924,  36.741475 ]

         Theta(S) :    11.997463
    S.D. Theta(S) :     3.037251

        Theta(Pi) :     1.220045
   S.D. Theta(Pi) :     0.864526

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    3595    3750.571
       1    4169    3742.491
       2    1918    2128.395
       3     694     946.663
       4     522     377.083
       5     223     143.698
       6      39      53.937
       7      10      20.157
       8       5       7.525
       9       0       2.808
      10       0       1.048
      11       0       0.391
      12       0       0.146
      13       0       0.054
      14       0       0.020
      15       0       0.008
      16       0       0.003


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        1.220
Mismatch observed variance          :        1.575
Tau                                 :        0.967
Theta0                              :        0.134
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00210339
          P(Sim. Ssd >= Obs. Ssd): 0.50000000
    Harpending's Raggedness index: 0.05644098
        P(Sim. Rag. >= Obs. Rag.): 0.50000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       3595.0       3557.4       3749.0
          1.0       4169.0       3993.7       4066.3
          2.0       1918.0       2403.3       2231.3
          3.0        694.0        901.0        827.6
          4.0        522.0        251.4        234.0
          5.0        223.0         53.2         54.0
          6.0         39.0         13.1         10.6
          7.0         10.0          1.7          1.8
          8.0          5.0          0.2          0.3
          9.0          0.0          0.0          0.0
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           0.967           0.617           1.477           1.113
                   Theta0:           0.134           0.000           0.007           0.002               
                   Theta1:       99999.000          10.562       99999.000       70009.672
Mean No. of pairwise diff.           1.220           0.813           1.389           1.151
           No. pol. sites:              67              55              79          69.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       3595.0       2437.0       4950.0       3749.0
          1.0       4169.0       3566.0       4485.0       4066.3
          2.0       1918.0       1836.0       3133.0       2231.3
          3.0        694.0        428.0       1299.0        827.6
          4.0        522.0         55.0        420.0        234.0
          5.0        223.0          0.0        101.0         54.0
          6.0         39.0          0.0         56.0         10.6
          7.0         10.0          0.0         11.0          1.8
          8.0          5.0          0.0          2.0          0.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           0.967           0.617           1.477           1.113
                   Theta0:           0.134           0.000           0.007           0.002               
                   Theta1:       99999.000          10.562       99999.000       70009.672
Mean No. of pairwise diff.           1.220           0.813           1.389           1.151
           No. pol. sites:              67              55              79          69.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       3595.0       2437.0       4950.0       3749.0
          1.0       4169.0       3566.0       4485.0       4066.3
          2.0       1918.0       1836.0       3133.0       2231.3
          3.0        694.0        428.0       1299.0        827.6
          4.0        522.0         55.0        420.0        234.0
          5.0        223.0          0.0        101.0         54.0
          6.0         39.0          0.0         56.0         10.6
          7.0         10.0          0.0         11.0          1.8
          8.0          5.0          0.0          2.0          0.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           0.967           0.617           1.477           1.113
                   Theta0:           0.134           0.000           0.007           0.002               
                   Theta1:       99999.000          10.562       99999.000       70009.672
Mean No. of pairwise diff.           1.220           0.813           1.389           1.151
           No. pol. sites:              67              55              79          69.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       3595.0       2437.0       4950.0       3749.0
          1.0       4169.0       3566.0       4485.0       4066.3
          2.0       1918.0       1836.0       3133.0       2231.3
          3.0        694.0        428.0       1299.0        827.6
          4.0        522.0         55.0        420.0        234.0
          5.0        223.0          0.0        101.0         54.0
          6.0         39.0          0.0         56.0         10.6
          7.0         10.0          0.0         11.0          1.8
          8.0          5.0          0.0          2.0          0.3
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_29_28_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            67      67.000       0.000
  No. of usable loci            67      67.000       0.000
  No. of polym. loci            67      67.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_29_28_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.01333     0.01333     0.00000     0.01333
     3       0.01333     0.01333     0.00000     0.01333
     4       0.01333     0.01333     0.00000     0.01333
     5       0.01333     0.01333     0.00000     0.01333
     6       0.02649     0.02649     0.00000     0.02649
     7       0.01333     0.01333     0.00000     0.01333
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.01333     0.01333     0.00000     0.01333
    11       0.01333     0.01333     0.00000     0.01333
    12       0.01333     0.01333     0.00000     0.01333
    13       0.01333     0.01333     0.00000     0.01333
    14       0.01333     0.01333     0.00000     0.01333
    15       0.01333     0.01333     0.00000     0.01333
    16       0.01333     0.01333     0.00000     0.01333
    17       0.01333     0.01333     0.00000     0.01333
    18       0.01333     0.01333     0.00000     0.01333
    19       0.01333     0.01333     0.00000     0.01333
    20       0.01333     0.01333     0.00000     0.01333
    21       0.01333     0.01333     0.00000     0.01333
    22       0.02649     0.02649     0.00000     0.02649
    23       0.01333     0.01333     0.00000     0.01333
    24       0.06488     0.06488     0.00000     0.06488
    25       0.03946     0.03946     0.00000     0.03946
    26       0.01333     0.01333     0.00000     0.01333
    27       0.01333     0.01333     0.00000     0.01333
    28       0.02649     0.02649     0.00000     0.02649
    29       0.01333     0.01333     0.00000     0.01333
    30       0.01333     0.01333     0.00000     0.01333
    31       0.01333     0.01333     0.00000     0.01333
    32       0.01333     0.01333     0.00000     0.01333
    33       0.01333     0.01333     0.00000     0.01333
    34       0.01333     0.01333     0.00000     0.01333
    35       0.01333     0.01333     0.00000     0.01333
    36       0.01333     0.01333     0.00000     0.01333
    37       0.01333     0.01333     0.00000     0.01333
    38       0.01333     0.01333     0.00000     0.01333
    39       0.01333     0.01333     0.00000     0.01333
    40       0.02649     0.02649     0.00000     0.02649
    41       0.01333     0.01333     0.00000     0.01333
    42       0.02649     0.02649     0.00000     0.02649
    43       0.01333     0.01333     0.00000     0.01333
    44       0.01333     0.01333     0.00000     0.01333
    45       0.01333     0.01333     0.00000     0.01333
    46       0.03946     0.03946     0.00000     0.03946
    47       0.01333     0.01333     0.00000     0.01333
    48       0.01333     0.01333     0.00000     0.01333
    49       0.02649     0.02649     0.00000     0.02649
    50       0.03946     0.03946     0.00000     0.03946
    51       0.02649     0.02649     0.00000     0.02649
    52       0.02649     0.02649     0.00000     0.02649
    53       0.03946     0.03946     0.00000     0.03946
    54       0.01333     0.01333     0.00000     0.01333
    55       0.01333     0.01333     0.00000     0.01333
    56       0.03946     0.03946     0.00000     0.03946
    57       0.03946     0.03946     0.00000     0.03946
    58       0.01333     0.01333     0.00000     0.01333
    59       0.01333     0.01333     0.00000     0.01333
    60       0.01333     0.01333     0.00000     0.01333
    61       0.01333     0.01333     0.00000     0.01333
    62       0.02649     0.02649     0.00000     0.02649
    63       0.01333     0.01333     0.00000     0.01333
    64       0.01333     0.01333     0.00000     0.01333
    65       0.01333     0.01333     0.00000     0.01333
    66       0.01333     0.01333     0.00000     0.01333
    67       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.01821     0.01821     0.00000     0.01821
  s.d.       0.01009     0.01009     0.00000     0.01009
--------------------------------------------------------
<A NAME="31_01_17at16_29_28_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_29_28_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      25.85716    25.85716     0.00000
            Theta_k_lower      18.03892    18.03892     0.00000
            Theta_k_upper      36.74148    36.74148     0.00000
                  Theta_H       1.60482     1.60482     0.00000
             s.d. Theta_H       0.34544     0.34544     0.00000
                  Theta_S      11.99746    11.99746     0.00000
             s.d. Theta_S       3.03725     3.03725     0.00000
                 Theta_pi       1.22004     1.22004     0.00000
            s.d. Theta_pi       0.86453     0.86453     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_29_28_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       0.96680     0.96680     0.00000
              Tau qt 2.5%       0.61719     0.61719     0.00000
                Tau qt 5%       0.61719     0.61719     0.00000
               Tau qt 95%       1.46680     1.46680     0.00000
             Tau qt 97.5%       1.46680     1.46680     0.00000
                   Theta0       0.13359     0.13359     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00527     0.00527     0.00000
          Theta0 qt 97.5%       0.00527     0.00527     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      10.56250    10.56250     0.00000
             Theta1 qt 5%      10.56250    10.56250     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00210     0.00210     0.00000
      Model (SSD) p-value       0.50000     0.50000     0.00000
         Raggedness index       0.05644     0.05644     0.00000
       Raggedness p-value       0.50000     0.50000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 90 (31/01/17 at 16:29:28))
Total computing time for this run : 0h 0m 0s 124 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

