
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_49_42"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 185 (31/01/17 at 14:49:42)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_49_42_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_49_42_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 45
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.15937
     2       150         2   0.50112
     3       150         2   0.11356
     4       150         2   0.20233
     5       150         2   0.50327
     6       150         2   0.03946
     7       150         2   0.01333
     8       150         2   0.22273
     9       150         2   0.50300
    10       150         2   0.15937
    11       150         2   0.06488
    12       150         2   0.19186
    13       150         2   0.23266
    14       150         2   0.06488
    15       150         2   0.03946
    16       150         2   0.46756
    17       150         2   0.08957
    18       150         2   0.07732
    19       150         2   0.50327
    20       150         2   0.07732
-------------------------------------------------
  Mean   150.000     2.000   0.21132
  s.d.     0.000     0.000   0.17963
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0427
Gene diversity           :  0.9638 +/-  0.0045

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 45

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.226309 +/-     2.109204

Average gene diversity over loci              :     0.211315 +/-     0.116734

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    24.874256
  S.D. Theta(Hom) :     3.363107

         Theta(k) :    21.427915
95 % confidence interval limits for theta(k) : [  14.772787,  30.751806 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     4.226309
   S.D. Theta(Pi) :     2.334685

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     405     195.586
       1     726     766.670
       2    1282    1519.966
       3    1978    2036.218
       4    2063    2078.683
       5    1818    1729.920
       6    1089    1226.827
       7     832     765.728
       8     622     431.409
       9     271     224.033
      10      89     109.169
      11       0      50.690
      12       0      22.723
      13       0       9.940
      14       0       4.279
      15       0       1.824
      16       0       0.773
      17       0       0.326
      18       0       0.138
      19       0       0.058
      20       0       0.024


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.226
Mismatch observed variance          :        4.756
Tau                                 :        3.965
Theta0                              :        0.482
Theta1                              :       51.016

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00076322
          P(Sim. Ssd >= Obs. Ssd): 0.40000000
    Harpending's Raggedness index: 0.01417097
        P(Sim. Rag. >= Obs. Rag.): 0.50000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        405.0        348.2        343.5
          1.0        726.0        753.2        760.0
          2.0       1282.0       1271.6       1383.9
          3.0       1978.0       1738.5       1890.0
          4.0       2063.0       1851.6       2015.1
          5.0       1818.0       1737.4       1754.6
          6.0       1089.0       1373.8       1292.6
          7.0        832.0        922.8        827.7
          8.0        622.0        567.5        470.5
          9.0        271.0        310.1        241.5
         10.0         89.0        170.6        113.6
         11.0          0.0         87.2         49.5
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.965           1.873           5.406           3.984
                   Theta0:           0.482           0.000           1.939           0.602               
                   Theta1:          51.016          15.459       99999.000       30113.354
Mean No. of pairwise diff.           4.226           3.240           5.823           4.459
           No. pol. sites:              20              78             106          92.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        405.0        149.0       1095.0        343.5
          1.0        726.0        223.0       1801.0        760.0
          2.0       1282.0        575.0       2148.0       1383.9
          3.0       1978.0       1123.0       2619.0       1890.0
          4.0       2063.0       1393.0       2425.0       2015.1
          5.0       1818.0       1183.0       2354.0       1754.6
          6.0       1089.0        851.0       2096.0       1292.6
          7.0        832.0        445.0       1510.0        827.7
          8.0        622.0        265.0        971.0        470.5
          9.0        271.0         93.0        749.0        241.5
         10.0         89.0         11.0        690.0        113.6
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.965           1.873           5.406           3.984
                   Theta0:           0.482           0.000           1.939           0.602               
                   Theta1:          51.016          15.459       99999.000       30113.354
Mean No. of pairwise diff.           4.226           3.240           5.823           4.459
           No. pol. sites:              20              78             106          92.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        405.0        149.0       1095.0        343.5
          1.0        726.0        223.0       1801.0        760.0
          2.0       1282.0        575.0       2148.0       1383.9
          3.0       1978.0       1123.0       2619.0       1890.0
          4.0       2063.0       1393.0       2425.0       2015.1
          5.0       1818.0       1183.0       2354.0       1754.6
          6.0       1089.0        851.0       2096.0       1292.6
          7.0        832.0        445.0       1510.0        827.7
          8.0        622.0        265.0        971.0        470.5
          9.0        271.0         93.0        749.0        241.5
         10.0         89.0         11.0        690.0        113.6
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.965           1.873           5.406           3.984
                   Theta0:           0.482           0.000           1.939           0.602               
                   Theta1:          51.016          15.459       99999.000       30113.354
Mean No. of pairwise diff.           4.226           3.240           5.823           4.459
           No. pol. sites:              20              78             106          92.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        405.0        149.0       1095.0        343.5
          1.0        726.0        223.0       1801.0        760.0
          2.0       1282.0        575.0       2148.0       1383.9
          3.0       1978.0       1123.0       2619.0       1890.0
          4.0       2063.0       1393.0       2425.0       2015.1
          5.0       1818.0       1183.0       2354.0       1754.6
          6.0       1089.0        851.0       2096.0       1292.6
          7.0        832.0        445.0       1510.0        827.7
          8.0        622.0        265.0        971.0        470.5
          9.0        271.0         93.0        749.0        241.5
         10.0         89.0         11.0        690.0        113.6
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_49_42_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_49_42_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.15937     0.15937     0.00000     0.15937
     2       0.50112     0.50112     0.00000     0.50112
     3       0.11356     0.11356     0.00000     0.11356
     4       0.20233     0.20233     0.00000     0.20233
     5       0.50327     0.50327     0.00000     0.50327
     6       0.03946     0.03946     0.00000     0.03946
     7       0.01333     0.01333     0.00000     0.01333
     8       0.22273     0.22273     0.00000     0.22273
     9       0.50300     0.50300     0.00000     0.50300
    10       0.15937     0.15937     0.00000     0.15937
    11       0.06488     0.06488     0.00000     0.06488
    12       0.19186     0.19186     0.00000     0.19186
    13       0.23266     0.23266     0.00000     0.23266
    14       0.06488     0.06488     0.00000     0.06488
    15       0.03946     0.03946     0.00000     0.03946
    16       0.46756     0.46756     0.00000     0.46756
    17       0.08957     0.08957     0.00000     0.08957
    18       0.07732     0.07732     0.00000     0.07732
    19       0.50327     0.50327     0.00000     0.50327
    20       0.07732     0.07732     0.00000     0.07732
--------------------------------------------------------
  Mean       0.21132     0.21132     0.00000     0.21132
  s.d.       0.17963     0.17963     0.00000     0.17963
--------------------------------------------------------
<A NAME="31_01_17at14_49_42_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_49_42_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      21.42791    21.42791     0.00000
            Theta_k_lower      14.77279    14.77279     0.00000
            Theta_k_upper      30.75181    30.75181     0.00000
                  Theta_H      24.87426    24.87426     0.00000
             s.d. Theta_H       3.36311     3.36311     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       4.22631     4.22631     0.00000
            s.d. Theta_pi       2.33469     2.33469     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_49_42_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.96484     3.96484     0.00000
              Tau qt 2.5%       1.87305     1.87305     0.00000
                Tau qt 5%       1.87305     1.87305     0.00000
               Tau qt 95%       5.14844     5.14844     0.00000
             Tau qt 97.5%       5.14844     5.14844     0.00000
                   Theta0       0.48164     0.48164     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.82988     1.82988     0.00000
          Theta0 qt 97.5%       1.82988     1.82988     0.00000
                   Theta1      51.01562    51.01562     0.00000
           Theta1 qt 2.5%      15.45898    15.45898     0.00000
             Theta1 qt 5%      15.45898    15.45898     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00076     0.00076     0.00000
      Model (SSD) p-value       0.40000     0.40000     0.00000
         Raggedness index       0.01417     0.01417     0.00000
       Raggedness p-value       0.50000     0.50000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 185 (31/01/17 at 14:49:42))
Total computing time for this run : 0h 0m 0s 136 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

