
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_07"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 143 (31/01/17 at 14:44:07)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_07_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_07_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 44
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.19186
     2       150         2   0.33011
     3       150         2   0.02649
     4       150         2   0.02649
     5       150         2   0.32215
     6       150         2   0.11356
     7       150         2   0.01333
     8       150         2   0.19186
     9       150         2   0.10166
    10       150         2   0.07732
    11       150         2   0.33011
    12       150         2   0.35293
    13       150         2   0.05226
    14       150         2   0.01333
    15       150         2   0.20233
    16       150         2   0.24242
    17       150         2   0.03946
    18       150         2   0.01333
    19       150         2   0.14819
    20       150         2   0.17038
-------------------------------------------------
  Mean   150.000     2.000   0.14798
  s.d.     0.000     0.000   0.11878
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0970
Gene diversity           :  0.9091 +/-  0.0181

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 44

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.959553 +/-     1.557604

Average gene diversity over loci              :     0.147978 +/-     0.086206

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     8.653297
  S.D. Theta(Hom) :     2.079564

         Theta(k) :    20.605975
95 % confidence interval limits for theta(k) : [  14.167681,  29.640070 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     2.959553
   S.D. Theta(Pi) :     1.724117

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1016     641.925
       1    1684    1784.233
       2    1730    2517.565
       3    2500    2411.492
       4    2194    1770.413
       5    1128    1067.211
       6     541     553.037
       7     299     254.891
       8      73     107.343
       9      10      42.237
      10       0      15.819
      11       0       5.726
      12       0       2.027
      13       0       0.708
      14       0       0.245
      15       0       0.085
      16       0       0.029
      17       0       0.010
      18       0       0.003


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.960
Mismatch observed variance          :        3.234
Tau                                 :        3.449
Theta0                              :        0.000
Theta1                              :       15.762

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00339103
          P(Sim. Ssd >= Obs. Ssd): 0.30000000
    Harpending's Raggedness index: 0.02185695
        P(Sim. Rag. >= Obs. Rag.): 0.40000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1016.0       1104.4        934.9
          1.0       1684.0       1507.7       1536.1
          2.0       1730.0       2084.0       2114.9
          3.0       2500.0       2300.4       2227.6
          4.0       2194.0       1774.3       1842.1
          5.0       1128.0       1260.1       1241.6
          6.0        541.0        692.8        703.6
          7.0        299.0        303.4        343.4
          8.0         73.0        110.1        147.0
          9.0         10.0         31.6         56.1
         10.0          0.0          4.7         19.3
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.449           2.316           4.369           3.439
                   Theta0:           0.000           0.000           0.012           0.004               
                   Theta1:          15.762           6.265       99999.000       10039.328
Mean No. of pairwise diff.           2.960           2.093           3.800           2.996
           No. pol. sites:              20              36              51          43.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1016.0        644.0       1919.0        934.9
          1.0       1684.0        923.0       2355.0       1536.1
          2.0       1730.0       1202.0       3141.0       2114.9
          3.0       2500.0       1831.0       2883.0       2227.6
          4.0       2194.0        957.0       2773.0       1842.1
          5.0       1128.0        533.0       1882.0       1241.6
          6.0        541.0         93.0       1270.0        703.6
          7.0        299.0          0.0        712.0        343.4
          8.0         73.0          0.0        340.0        147.0
          9.0         10.0          0.0        122.0         56.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.449           2.316           4.369           3.439
                   Theta0:           0.000           0.000           0.012           0.004               
                   Theta1:          15.762           6.265       99999.000       10039.328
Mean No. of pairwise diff.           2.960           2.093           3.800           2.996
           No. pol. sites:              20              36              51          43.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1016.0        644.0       1919.0        934.9
          1.0       1684.0        923.0       2355.0       1536.1
          2.0       1730.0       1202.0       3141.0       2114.9
          3.0       2500.0       1831.0       2883.0       2227.6
          4.0       2194.0        957.0       2773.0       1842.1
          5.0       1128.0        533.0       1882.0       1241.6
          6.0        541.0         93.0       1270.0        703.6
          7.0        299.0          0.0        712.0        343.4
          8.0         73.0          0.0        340.0        147.0
          9.0         10.0          0.0        122.0         56.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.449           2.316           4.369           3.439
                   Theta0:           0.000           0.000           0.012           0.004               
                   Theta1:          15.762           6.265       99999.000       10039.328
Mean No. of pairwise diff.           2.960           2.093           3.800           2.996
           No. pol. sites:              20              36              51          43.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1016.0        644.0       1919.0        934.9
          1.0       1684.0        923.0       2355.0       1536.1
          2.0       1730.0       1202.0       3141.0       2114.9
          3.0       2500.0       1831.0       2883.0       2227.6
          4.0       2194.0        957.0       2773.0       1842.1
          5.0       1128.0        533.0       1882.0       1241.6
          6.0        541.0         93.0       1270.0        703.6
          7.0        299.0          0.0        712.0        343.4
          8.0         73.0          0.0        340.0        147.0
          9.0         10.0          0.0        122.0         56.1
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_07_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_07_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.19186     0.19186     0.00000     0.19186
     2       0.33011     0.33011     0.00000     0.33011
     3       0.02649     0.02649     0.00000     0.02649
     4       0.02649     0.02649     0.00000     0.02649
     5       0.32215     0.32215     0.00000     0.32215
     6       0.11356     0.11356     0.00000     0.11356
     7       0.01333     0.01333     0.00000     0.01333
     8       0.19186     0.19186     0.00000     0.19186
     9       0.10166     0.10166     0.00000     0.10166
    10       0.07732     0.07732     0.00000     0.07732
    11       0.33011     0.33011     0.00000     0.33011
    12       0.35293     0.35293     0.00000     0.35293
    13       0.05226     0.05226     0.00000     0.05226
    14       0.01333     0.01333     0.00000     0.01333
    15       0.20233     0.20233     0.00000     0.20233
    16       0.24242     0.24242     0.00000     0.24242
    17       0.03946     0.03946     0.00000     0.03946
    18       0.01333     0.01333     0.00000     0.01333
    19       0.14819     0.14819     0.00000     0.14819
    20       0.17038     0.17038     0.00000     0.17038
--------------------------------------------------------
  Mean       0.14798     0.14798     0.00000     0.14798
  s.d.       0.11878     0.11878     0.00000     0.11878
--------------------------------------------------------
<A NAME="31_01_17at14_44_07_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_07_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      20.60597    20.60597     0.00000
            Theta_k_lower      14.16768    14.16768     0.00000
            Theta_k_upper      29.64007    29.64007     0.00000
                  Theta_H       8.65330     8.65330     0.00000
             s.d. Theta_H       2.07956     2.07956     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       2.95955     2.95955     0.00000
            s.d. Theta_pi       1.72412     1.72412     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_07_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.44922     3.44922     0.00000
              Tau qt 2.5%       2.31641     2.31641     0.00000
                Tau qt 5%       2.31641     2.31641     0.00000
               Tau qt 95%       4.33398     4.33398     0.00000
             Tau qt 97.5%       4.33398     4.33398     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00879     0.00879     0.00000
          Theta0 qt 97.5%       0.00879     0.00879     0.00000
                   Theta1      15.76172    15.76172     0.00000
           Theta1 qt 2.5%       6.26494     6.26494     0.00000
             Theta1 qt 5%       6.26494     6.26494     0.00000
            Theta1 qt 95%     220.44922   220.44922     0.00000
          Theta1 qt 97.5%     220.44922   220.44922     0.00000
                      SSD       0.00339     0.00339     0.00000
      Model (SSD) p-value       0.30000     0.30000     0.00000
         Raggedness index       0.02186     0.02186     0.00000
       Raggedness p-value       0.40000     0.40000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 143 (31/01/17 at 14:44:07))
Total computing time for this run : 0h 0m 0s 104 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

