
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_43_57"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 70 (31/01/17 at 14:43:57)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_43_57_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_43_57_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 53
No. of loci              : 70
No. of usable loci       : 70 loci with less than  5.00 % missing data
No. of polymorphic sites : 70


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.02649
     2       150         2   0.05226
     3       150         2   0.01333
     4       150         2   0.01333
     5       150         2   0.15937
     6       150         2   0.01333
     7       150         2   0.01333
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.10166
    11       150         2   0.01333
    12       150         2   0.10166
    13       150         2   0.02649
    14       150         2   0.01333
    15       150         2   0.01333
    16       150         2   0.01333
    17       150         2   0.01333
    18       150         2   0.01333
    19       150         2   0.10166
    20       150         2   0.02649
    21       150         2   0.03946
    22       150         2   0.05226
    23       150         2   0.01333
    24       150         2   0.03946
    25       150         2   0.02649
    26       150         2   0.03946
    27       150         2   0.13682
    28       150         2   0.01333
    29       150         2   0.10166
    30       150         2   0.01333
    31       150         2   0.02649
    32       150         2   0.07732
    33       150         2   0.05226
    34       150         2   0.08957
    35       150         2   0.02649
    36       150         2   0.15937
    37       150         2   0.05226
    38       150         2   0.06488
    39       150         2   0.01333
    40       150         2   0.06488
    41       150         2   0.01333
    42       150         2   0.01333
    43       150         2   0.03946
    44       150         2   0.12528
    45       150         2   0.01333
    46       150         2   0.02649
    47       150         2   0.08957
    48       150         2   0.05226
    49       150         2   0.01333
    50       150         2   0.01333
    51       150         2   0.08957
    52       150         2   0.01333
    53       150         2   0.01333
    54       150         2   0.05226
    55       150         2   0.02649
    56       150         2   0.06488
    57       150         2   0.01333
    58       150         2   0.06488
    59       150         2   0.01333
    60       150         2   0.03946
    61       150         2   0.03946
    62       150         2   0.01333
    63       150         2   0.02649
    64       150         2   0.13682
    65       150         2   0.01333
    66       150         2   0.07732
    67       150         2   0.02649
    68       150         2   0.01333
    69       150         2   0.10166
    70       150         2   0.03946
-------------------------------------------------
  Mean   150.000     2.000   0.04481
  s.d.     0.000     0.000   0.03941
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0406
Gene diversity           :  0.9658 +/-  0.0059

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 53

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 70
Number of usable nucleotide sites            : 70


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.136823 +/-     1.635137

Average gene diversity over loci              :     0.044812 +/-     0.025856

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    26.520177
  S.D. Theta(Hom) :     5.010307

         Theta(k) :    28.791891
95 % confidence interval limits for theta(k) : [  20.205559,  40.712272 ]

         Theta(S) :    12.534663
    S.D. Theta(S) :     3.158166

        Theta(Pi) :     3.136823
   S.D. Theta(Pi) :     1.809939

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     382       0.000
       1    1195       0.000
       2    2530       0.000
       3    2681       0.000
       4    2311       0.000
       5    1255       0.000
       6     574       0.000
       7     195       0.000
       8      41       0.000
       9      10       0.000
      10       1       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.137
Mismatch observed variance          :        2.469
Tau                                 :        3.217
Theta0                              :        0.000
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00227437
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.03483480
        P(Sim. Rag. >= Obs. Rag.): 0.10000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        382.0        423.5        448.0
          1.0       1195.0       1481.7       1440.9
          2.0       2530.0       2333.6       2317.5
          3.0       2681.0       2472.9       2485.0
          4.0       2311.0       1951.3       1998.4
          5.0       1255.0       1268.7       1285.7
          6.0        574.0        690.9        689.3
          7.0        195.0        337.1        316.8
          8.0         41.0        138.8        127.4
          9.0         10.0         54.6         45.5
         10.0          1.0         17.1         14.6
         11.0          0.0          3.8          4.3
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.217           2.686           3.686           3.153
                   Theta0:           0.000           0.000           0.032           0.004               
                   Theta1:       99999.000          54.000       99999.000       80014.781
Mean No. of pairwise diff.           3.137           2.942           3.689           3.226
           No. pol. sites:              70             171             205         181.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        382.0        190.0        630.0        448.0
          1.0       1195.0        948.0       1874.0       1440.9
          2.0       2530.0       1847.0       2667.0       2317.5
          3.0       2681.0       2356.0       2652.0       2485.0
          4.0       2311.0       1696.0       2284.0       1998.4
          5.0       1255.0       1044.0       1647.0       1285.7
          6.0        574.0        559.0        992.0        689.3
          7.0        195.0        211.0        522.0        316.8
          8.0         41.0         61.0        248.0        127.4
          9.0         10.0          9.0        119.0         45.5
         10.0          1.0          0.0         54.0         14.6
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.217           2.686           3.686           3.153
                   Theta0:           0.000           0.000           0.032           0.004               
                   Theta1:       99999.000          54.000       99999.000       80014.781
Mean No. of pairwise diff.           3.137           2.942           3.689           3.226
           No. pol. sites:              70             171             205         181.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        382.0        190.0        630.0        448.0
          1.0       1195.0        948.0       1874.0       1440.9
          2.0       2530.0       1847.0       2667.0       2317.5
          3.0       2681.0       2356.0       2652.0       2485.0
          4.0       2311.0       1696.0       2284.0       1998.4
          5.0       1255.0       1044.0       1647.0       1285.7
          6.0        574.0        559.0        992.0        689.3
          7.0        195.0        211.0        522.0        316.8
          8.0         41.0         61.0        248.0        127.4
          9.0         10.0          9.0        119.0         45.5
         10.0          1.0          0.0         54.0         14.6
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.217           2.686           3.686           3.153
                   Theta0:           0.000           0.000           0.032           0.004               
                   Theta1:       99999.000          54.000       99999.000       80014.781
Mean No. of pairwise diff.           3.137           2.942           3.689           3.226
           No. pol. sites:              70             171             205         181.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        382.0        190.0        630.0        448.0
          1.0       1195.0        948.0       1874.0       1440.9
          2.0       2530.0       1847.0       2667.0       2317.5
          3.0       2681.0       2356.0       2652.0       2485.0
          4.0       2311.0       1696.0       2284.0       1998.4
          5.0       1255.0       1044.0       1647.0       1285.7
          6.0        574.0        559.0        992.0        689.3
          7.0        195.0        211.0        522.0        316.8
          8.0         41.0         61.0        248.0        127.4
          9.0         10.0          9.0        119.0         45.5
         10.0          1.0          0.0         54.0         14.6
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_43_57_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            70      70.000       0.000
  No. of usable loci            70      70.000       0.000
  No. of polym. loci            70      70.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_43_57_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.02649     0.02649     0.00000     0.02649
     2       0.05226     0.05226     0.00000     0.05226
     3       0.01333     0.01333     0.00000     0.01333
     4       0.01333     0.01333     0.00000     0.01333
     5       0.15937     0.15937     0.00000     0.15937
     6       0.01333     0.01333     0.00000     0.01333
     7       0.01333     0.01333     0.00000     0.01333
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.10166     0.10166     0.00000     0.10166
    11       0.01333     0.01333     0.00000     0.01333
    12       0.10166     0.10166     0.00000     0.10166
    13       0.02649     0.02649     0.00000     0.02649
    14       0.01333     0.01333     0.00000     0.01333
    15       0.01333     0.01333     0.00000     0.01333
    16       0.01333     0.01333     0.00000     0.01333
    17       0.01333     0.01333     0.00000     0.01333
    18       0.01333     0.01333     0.00000     0.01333
    19       0.10166     0.10166     0.00000     0.10166
    20       0.02649     0.02649     0.00000     0.02649
    21       0.03946     0.03946     0.00000     0.03946
    22       0.05226     0.05226     0.00000     0.05226
    23       0.01333     0.01333     0.00000     0.01333
    24       0.03946     0.03946     0.00000     0.03946
    25       0.02649     0.02649     0.00000     0.02649
    26       0.03946     0.03946     0.00000     0.03946
    27       0.13682     0.13682     0.00000     0.13682
    28       0.01333     0.01333     0.00000     0.01333
    29       0.10166     0.10166     0.00000     0.10166
    30       0.01333     0.01333     0.00000     0.01333
    31       0.02649     0.02649     0.00000     0.02649
    32       0.07732     0.07732     0.00000     0.07732
    33       0.05226     0.05226     0.00000     0.05226
    34       0.08957     0.08957     0.00000     0.08957
    35       0.02649     0.02649     0.00000     0.02649
    36       0.15937     0.15937     0.00000     0.15937
    37       0.05226     0.05226     0.00000     0.05226
    38       0.06488     0.06488     0.00000     0.06488
    39       0.01333     0.01333     0.00000     0.01333
    40       0.06488     0.06488     0.00000     0.06488
    41       0.01333     0.01333     0.00000     0.01333
    42       0.01333     0.01333     0.00000     0.01333
    43       0.03946     0.03946     0.00000     0.03946
    44       0.12528     0.12528     0.00000     0.12528
    45       0.01333     0.01333     0.00000     0.01333
    46       0.02649     0.02649     0.00000     0.02649
    47       0.08957     0.08957     0.00000     0.08957
    48       0.05226     0.05226     0.00000     0.05226
    49       0.01333     0.01333     0.00000     0.01333
    50       0.01333     0.01333     0.00000     0.01333
    51       0.08957     0.08957     0.00000     0.08957
    52       0.01333     0.01333     0.00000     0.01333
    53       0.01333     0.01333     0.00000     0.01333
    54       0.05226     0.05226     0.00000     0.05226
    55       0.02649     0.02649     0.00000     0.02649
    56       0.06488     0.06488     0.00000     0.06488
    57       0.01333     0.01333     0.00000     0.01333
    58       0.06488     0.06488     0.00000     0.06488
    59       0.01333     0.01333     0.00000     0.01333
    60       0.03946     0.03946     0.00000     0.03946
    61       0.03946     0.03946     0.00000     0.03946
    62       0.01333     0.01333     0.00000     0.01333
    63       0.02649     0.02649     0.00000     0.02649
    64       0.13682     0.13682     0.00000     0.13682
    65       0.01333     0.01333     0.00000     0.01333
    66       0.07732     0.07732     0.00000     0.07732
    67       0.02649     0.02649     0.00000     0.02649
    68       0.01333     0.01333     0.00000     0.01333
    69       0.10166     0.10166     0.00000     0.10166
    70       0.03946     0.03946     0.00000     0.03946
--------------------------------------------------------
  Mean       0.04481     0.04481     0.00000     0.04481
  s.d.       0.03941     0.03941     0.00000     0.03941
--------------------------------------------------------
<A NAME="31_01_17at14_43_57_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_43_57_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      28.79189    28.79189     0.00000
            Theta_k_lower      20.20556    20.20556     0.00000
            Theta_k_upper      40.71227    40.71227     0.00000
                  Theta_H      26.52018    26.52018     0.00000
             s.d. Theta_H       5.01031     5.01031     0.00000
                  Theta_S      12.53466    12.53466     0.00000
             s.d. Theta_S       3.15817     3.15817     0.00000
                 Theta_pi       3.13682     3.13682     0.00000
            s.d. Theta_pi       1.80994     1.80994     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_43_57_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.21680     3.21680     0.00000
              Tau qt 2.5%       2.68555     2.68555     0.00000
                Tau qt 5%       2.68555     2.68555     0.00000
               Tau qt 95%       3.61523     3.61523     0.00000
             Tau qt 97.5%       3.61523     3.61523     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.01230     0.01230     0.00000
          Theta0 qt 97.5%       0.01230     0.01230     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      54.00000    54.00000     0.00000
             Theta1 qt 5%      54.00000    54.00000     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00227     0.00227     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.03483     0.03483     0.00000
       Raggedness p-value       0.10000     0.10000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 70 (31/01/17 at 14:43:57))
Total computing time for this run : 0h 0m 0s 108 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

