
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_29_19"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 14 (31/01/17 at 16:29:19)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_29_19_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_29_19_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 58
No. of loci              : 72
No. of usable loci       : 72 loci with less than  5.00 % missing data
No. of polymorphic sites : 72


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.01333
     3       150         2   0.01333
     4       150         2   0.03946
     5       150         2   0.01333
     6       150         2   0.01333
     7       150         2   0.02649
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.01333
    11       150         2   0.02649
    12       150         2   0.02649
    13       150         2   0.02649
    14       150         2   0.01333
    15       150         2   0.01333
    16       150         2   0.01333
    17       150         2   0.01333
    18       150         2   0.01333
    19       150         2   0.01333
    20       150         2   0.02649
    21       150         2   0.02649
    22       150         2   0.01333
    23       150         2   0.06488
    24       150         2   0.02649
    25       150         2   0.01333
    26       150         2   0.07732
    27       150         2   0.01333
    28       150         2   0.01333
    29       150         2   0.01333
    30       150         2   0.01333
    31       150         2   0.02649
    32       150         2   0.01333
    33       150         2   0.03946
    34       150         2   0.03946
    35       150         2   0.01333
    36       150         2   0.01333
    37       150         2   0.07732
    38       150         2   0.01333
    39       150         2   0.05226
    40       150         2   0.02649
    41       150         2   0.02649
    42       150         2   0.01333
    43       150         2   0.01333
    44       150         2   0.02649
    45       150         2   0.02649
    46       150         2   0.01333
    47       150         2   0.01333
    48       150         2   0.01333
    49       150         2   0.02649
    50       150         2   0.01333
    51       150         2   0.01333
    52       150         2   0.01333
    53       150         2   0.03946
    54       150         2   0.01333
    55       150         2   0.01333
    56       150         2   0.02649
    57       150         2   0.02649
    58       150         2   0.02649
    59       150         2   0.01333
    60       150         2   0.01333
    61       150         2   0.01333
    62       150         2   0.01333
    63       150         2   0.01333
    64       150         2   0.01333
    65       150         2   0.01333
    66       150         2   0.01333
    67       150         2   0.01333
    68       150         2   0.06488
    69       150         2   0.03946
    70       150         2   0.01333
    71       150         2   0.01333
    72       150         2   0.02649
-------------------------------------------------
  Mean   150.000     2.000   0.02200
  s.d.     0.000     0.000   0.01499
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.2180
Gene diversity           :  0.7872 +/-  0.0366

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 58

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 72
Number of usable nucleotide sites            : 72


Distance method : Pairwise difference
Mean number of pairwise differences           :     1.584251 +/-     0.947722

Average gene diversity over loci              :     0.022003 +/-     0.014570

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     2.914348
  S.D. Theta(Hom) :     0.685769

         Theta(k) :    34.205353
95 % confidence interval limits for theta(k) : [  24.201985,  48.048972 ]

         Theta(S) :    12.892796
    S.D. Theta(S) :     3.238739

        Theta(Pi) :     1.584251
   S.D. Theta(Pi) :     1.049037

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    2378    2381.253
       1    3688    3605.277
       2    2667    2793.437
       3    1467    1484.513
       4     674     612.568
       5     233     210.910
       6      56      63.644
       7      12      17.465
       8       0       4.486
       9       0       1.103
      10       0       0.264
      11       0       0.062
      12       0       0.015
      13       0       0.003
      14       0       0.001


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        1.584
Mismatch observed variance          :        1.676
Tau                                 :        1.580
Theta0                              :        0.000
Theta1                              :       42.122

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00033159
          P(Sim. Ssd >= Obs. Ssd): 0.70000000
    Harpending's Raggedness index: 0.04048090
        P(Sim. Rag. >= Obs. Rag.): 0.70000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       2378.0       2794.4       2424.6
          1.0       3688.0       3842.2       3624.5
          2.0       2667.0       2671.2       2822.0
          3.0       1467.0       1240.6       1477.2
          4.0        674.0        441.9        581.6
          5.0        233.0        127.9        183.4
          6.0         56.0         39.1         48.2
          7.0         12.0         14.0         10.9
          8.0          0.0          2.7          2.1
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.580           0.934           2.090           1.459
                   Theta0:           0.000           0.000           0.315           0.055               
                   Theta1:          42.122           4.092       99999.000       60008.854
Mean No. of pairwise diff.           1.584           0.870           1.860           1.403
           No. pol. sites:              72              38              62          51.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2378.0       1720.0       4329.0       2424.6
          1.0       3688.0       2857.0       4467.0       3624.5
          2.0       2667.0       1918.0       3202.0       2822.0
          3.0       1467.0        429.0       2051.0       1477.2
          4.0        674.0         32.0        913.0        581.6
          5.0        233.0          0.0        392.0        183.4
          6.0         56.0          0.0        163.0         48.2
          7.0         12.0          0.0        114.0         10.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.580           0.934           2.090           1.459
                   Theta0:           0.000           0.000           0.315           0.055               
                   Theta1:          42.122           4.092       99999.000       60008.854
Mean No. of pairwise diff.           1.584           0.870           1.860           1.403
           No. pol. sites:              72              38              62          51.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2378.0       1720.0       4329.0       2424.6
          1.0       3688.0       2857.0       4467.0       3624.5
          2.0       2667.0       1918.0       3202.0       2822.0
          3.0       1467.0        429.0       2051.0       1477.2
          4.0        674.0         32.0        913.0        581.6
          5.0        233.0          0.0        392.0        183.4
          6.0         56.0          0.0        163.0         48.2
          7.0         12.0          0.0        114.0         10.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.580           0.934           2.090           1.459
                   Theta0:           0.000           0.000           0.315           0.055               
                   Theta1:          42.122           4.092       99999.000       60008.854
Mean No. of pairwise diff.           1.584           0.870           1.860           1.403
           No. pol. sites:              72              38              62          51.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2378.0       1720.0       4329.0       2424.6
          1.0       3688.0       2857.0       4467.0       3624.5
          2.0       2667.0       1918.0       3202.0       2822.0
          3.0       1467.0        429.0       2051.0       1477.2
          4.0        674.0         32.0        913.0        581.6
          5.0        233.0          0.0        392.0        183.4
          6.0         56.0          0.0        163.0         48.2
          7.0         12.0          0.0        114.0         10.9
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_29_19_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            72      72.000       0.000
  No. of usable loci            72      72.000       0.000
  No. of polym. loci            72      72.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_29_19_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.01333     0.01333     0.00000     0.01333
     3       0.01333     0.01333     0.00000     0.01333
     4       0.03946     0.03946     0.00000     0.03946
     5       0.01333     0.01333     0.00000     0.01333
     6       0.01333     0.01333     0.00000     0.01333
     7       0.02649     0.02649     0.00000     0.02649
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.01333     0.01333     0.00000     0.01333
    11       0.02649     0.02649     0.00000     0.02649
    12       0.02649     0.02649     0.00000     0.02649
    13       0.02649     0.02649     0.00000     0.02649
    14       0.01333     0.01333     0.00000     0.01333
    15       0.01333     0.01333     0.00000     0.01333
    16       0.01333     0.01333     0.00000     0.01333
    17       0.01333     0.01333     0.00000     0.01333
    18       0.01333     0.01333     0.00000     0.01333
    19       0.01333     0.01333     0.00000     0.01333
    20       0.02649     0.02649     0.00000     0.02649
    21       0.02649     0.02649     0.00000     0.02649
    22       0.01333     0.01333     0.00000     0.01333
    23       0.06488     0.06488     0.00000     0.06488
    24       0.02649     0.02649     0.00000     0.02649
    25       0.01333     0.01333     0.00000     0.01333
    26       0.07732     0.07732     0.00000     0.07732
    27       0.01333     0.01333     0.00000     0.01333
    28       0.01333     0.01333     0.00000     0.01333
    29       0.01333     0.01333     0.00000     0.01333
    30       0.01333     0.01333     0.00000     0.01333
    31       0.02649     0.02649     0.00000     0.02649
    32       0.01333     0.01333     0.00000     0.01333
    33       0.03946     0.03946     0.00000     0.03946
    34       0.03946     0.03946     0.00000     0.03946
    35       0.01333     0.01333     0.00000     0.01333
    36       0.01333     0.01333     0.00000     0.01333
    37       0.07732     0.07732     0.00000     0.07732
    38       0.01333     0.01333     0.00000     0.01333
    39       0.05226     0.05226     0.00000     0.05226
    40       0.02649     0.02649     0.00000     0.02649
    41       0.02649     0.02649     0.00000     0.02649
    42       0.01333     0.01333     0.00000     0.01333
    43       0.01333     0.01333     0.00000     0.01333
    44       0.02649     0.02649     0.00000     0.02649
    45       0.02649     0.02649     0.00000     0.02649
    46       0.01333     0.01333     0.00000     0.01333
    47       0.01333     0.01333     0.00000     0.01333
    48       0.01333     0.01333     0.00000     0.01333
    49       0.02649     0.02649     0.00000     0.02649
    50       0.01333     0.01333     0.00000     0.01333
    51       0.01333     0.01333     0.00000     0.01333
    52       0.01333     0.01333     0.00000     0.01333
    53       0.03946     0.03946     0.00000     0.03946
    54       0.01333     0.01333     0.00000     0.01333
    55       0.01333     0.01333     0.00000     0.01333
    56       0.02649     0.02649     0.00000     0.02649
    57       0.02649     0.02649     0.00000     0.02649
    58       0.02649     0.02649     0.00000     0.02649
    59       0.01333     0.01333     0.00000     0.01333
    60       0.01333     0.01333     0.00000     0.01333
    61       0.01333     0.01333     0.00000     0.01333
    62       0.01333     0.01333     0.00000     0.01333
    63       0.01333     0.01333     0.00000     0.01333
    64       0.01333     0.01333     0.00000     0.01333
    65       0.01333     0.01333     0.00000     0.01333
    66       0.01333     0.01333     0.00000     0.01333
    67       0.01333     0.01333     0.00000     0.01333
    68       0.06488     0.06488     0.00000     0.06488
    69       0.03946     0.03946     0.00000     0.03946
    70       0.01333     0.01333     0.00000     0.01333
    71       0.01333     0.01333     0.00000     0.01333
    72       0.02649     0.02649     0.00000     0.02649
--------------------------------------------------------
  Mean       0.02200     0.02200     0.00000     0.02200
  s.d.       0.01499     0.01499     0.00000     0.01499
--------------------------------------------------------
<A NAME="31_01_17at16_29_19_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
    72             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_29_19_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      34.20535    34.20535     0.00000
            Theta_k_lower      24.20198    24.20198     0.00000
            Theta_k_upper      48.04897    48.04897     0.00000
                  Theta_H       2.91435     2.91435     0.00000
             s.d. Theta_H       0.68577     0.68577     0.00000
                  Theta_S      12.89280    12.89280     0.00000
             s.d. Theta_S       3.23874     3.23874     0.00000
                 Theta_pi       1.58425     1.58425     0.00000
            s.d. Theta_pi       1.04904     1.04904     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_29_19_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.58008     1.58008     0.00000
              Tau qt 2.5%       0.93359     0.93359     0.00000
                Tau qt 5%       0.93359     0.93359     0.00000
               Tau qt 95%       1.80664     1.80664     0.00000
             Tau qt 97.5%       1.80664     1.80664     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.22324     0.22324     0.00000
          Theta0 qt 97.5%       0.22324     0.22324     0.00000
                   Theta1      42.12188    42.12188     0.00000
           Theta1 qt 2.5%       4.09209     4.09209     0.00000
             Theta1 qt 5%       4.09209     4.09209     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00033     0.00033     0.00000
      Model (SSD) p-value       0.70000     0.70000     0.00000
         Raggedness index       0.04048     0.04048     0.00000
       Raggedness p-value       0.70000     0.70000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 14 (31/01/17 at 16:29:19))
Total computing time for this run : 0h 0m 0s 120 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

