
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_29_32"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 120 (31/01/17 at 16:29:32)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_29_32_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_29_32_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 57
No. of loci              : 70
No. of usable loci       : 70 loci with less than  5.00 % missing data
No. of polymorphic sites : 70


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.03946
     2       150         2   0.01333
     3       150         2   0.06488
     4       150         2   0.03946
     5       150         2   0.01333
     6       150         2   0.02649
     7       150         2   0.01333
     8       150         2   0.01333
     9       150         2   0.02649
    10       150         2   0.02649
    11       150         2   0.01333
    12       150         2   0.03946
    13       150         2   0.01333
    14       150         2   0.01333
    15       150         2   0.02649
    16       150         2   0.02649
    17       150         2   0.02649
    18       150         2   0.01333
    19       150         2   0.01333
    20       150         2   0.02649
    21       150         2   0.01333
    22       150         2   0.02649
    23       150         2   0.03946
    24       150         2   0.05226
    25       150         2   0.01333
    26       150         2   0.10166
    27       150         2   0.02649
    28       150         2   0.01333
    29       150         2   0.01333
    30       150         2   0.01333
    31       150         2   0.01333
    32       150         2   0.02649
    33       150         2   0.03946
    34       150         2   0.02649
    35       150         2   0.03946
    36       150         2   0.02649
    37       150         2   0.01333
    38       150         2   0.01333
    39       150         2   0.01333
    40       150         2   0.02649
    41       150         2   0.03946
    42       150         2   0.01333
    43       150         2   0.01333
    44       150         2   0.01333
    45       150         2   0.02649
    46       150         2   0.01333
    47       150         2   0.01333
    48       150         2   0.03946
    49       150         2   0.01333
    50       150         2   0.01333
    51       150         2   0.01333
    52       150         2   0.01333
    53       150         2   0.02649
    54       150         2   0.05226
    55       150         2   0.01333
    56       150         2   0.06488
    57       150         2   0.02649
    58       150         2   0.01333
    59       150         2   0.02649
    60       150         2   0.01333
    61       150         2   0.02649
    62       150         2   0.01333
    63       150         2   0.01333
    64       150         2   0.03946
    65       150         2   0.01333
    66       150         2   0.02649
    67       150         2   0.03946
    68       150         2   0.01333
    69       150         2   0.01333
    70       150         2   0.02649
-------------------------------------------------
  Mean   150.000     2.000   0.02467
  s.d.     0.000     0.000   0.01590
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.1839
Gene diversity           :  0.8216 +/-  0.0329

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 57

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 70
Number of usable nucleotide sites            : 70


Distance method : Pairwise difference
Mean number of pairwise differences           :     1.726980 +/-     1.012131

Average gene diversity over loci              :     0.024671 +/-     0.016005

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     3.694410
  S.D. Theta(Hom) :     0.903220

         Theta(k) :    33.066749
95 % confidence interval limits for theta(k) : [  23.361695,  46.504216 ]

         Theta(S) :    12.534663
    S.D. Theta(S) :     3.158166

        Theta(Pi) :     1.726980
   S.D. Theta(Pi) :     1.120331

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1994    2040.971
       1    3557    3423.145
       2    2843    2911.323
       3    1601    1678.498
       4     773     740.405
       5     305     267.580
       6      81      82.910
       7      18      22.776
       8       3       5.696
       9       0       1.326
      10       0       0.292
      11       0       0.062
      12       0       0.013
      13       0       0.003
      14       0       0.001
      15       0       0.000
      16       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        1.727
Mismatch observed variance          :        1.789
Tau                                 :        1.691
Theta0                              :        0.000
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00026868
          P(Sim. Ssd >= Obs. Ssd): 1.00000000
    Harpending's Raggedness index: 0.04367624
        P(Sim. Rag. >= Obs. Rag.): 0.50000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1994.0       1925.9       2059.2
          1.0       3557.0       3460.5       3482.8
          2.0       2843.0       2929.8       2945.4
          3.0       1601.0       1774.8       1660.6
          4.0        773.0        759.4        702.2
          5.0        305.0        246.2        237.5
          6.0         81.0         64.0         67.0
          7.0         18.0         12.2         16.2
          8.0          3.0          1.9          3.4
          9.0          0.0          0.3          0.6
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.691           1.395           2.166           1.661
                   Theta0:           0.000           0.000           0.007           0.002               
                   Theta1:       99999.000          64.859       99999.000       90005.586
Mean No. of pairwise diff.           1.727           1.452           2.050           1.736
           No. pol. sites:              70              96             125         107.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1994.0       1028.0       2775.0       2059.2
          1.0       3557.0       2851.0       3894.0       3482.8
          2.0       2843.0       2400.0       3483.0       2945.4
          3.0       1601.0       1367.0       2493.0       1660.6
          4.0        773.0        476.0       1067.0        702.2
          5.0        305.0         99.0        379.0        237.5
          6.0         81.0         10.0        125.0         67.0
          7.0         18.0          0.0         34.0         16.2
          8.0          3.0          0.0         10.0          3.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.691           1.395           2.166           1.661
                   Theta0:           0.000           0.000           0.007           0.002               
                   Theta1:       99999.000          64.859       99999.000       90005.586
Mean No. of pairwise diff.           1.727           1.452           2.050           1.736
           No. pol. sites:              70              96             125         107.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1994.0       1028.0       2775.0       2059.2
          1.0       3557.0       2851.0       3894.0       3482.8
          2.0       2843.0       2400.0       3483.0       2945.4
          3.0       1601.0       1367.0       2493.0       1660.6
          4.0        773.0        476.0       1067.0        702.2
          5.0        305.0         99.0        379.0        237.5
          6.0         81.0         10.0        125.0         67.0
          7.0         18.0          0.0         34.0         16.2
          8.0          3.0          0.0         10.0          3.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.691           1.395           2.166           1.661
                   Theta0:           0.000           0.000           0.007           0.002               
                   Theta1:       99999.000          64.859       99999.000       90005.586
Mean No. of pairwise diff.           1.727           1.452           2.050           1.736
           No. pol. sites:              70              96             125         107.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1994.0       1028.0       2775.0       2059.2
          1.0       3557.0       2851.0       3894.0       3482.8
          2.0       2843.0       2400.0       3483.0       2945.4
          3.0       1601.0       1367.0       2493.0       1660.6
          4.0        773.0        476.0       1067.0        702.2
          5.0        305.0         99.0        379.0        237.5
          6.0         81.0         10.0        125.0         67.0
          7.0         18.0          0.0         34.0         16.2
          8.0          3.0          0.0         10.0          3.4
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_29_32_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            70      70.000       0.000
  No. of usable loci            70      70.000       0.000
  No. of polym. loci            70      70.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_29_32_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.03946     0.03946     0.00000     0.03946
     2       0.01333     0.01333     0.00000     0.01333
     3       0.06488     0.06488     0.00000     0.06488
     4       0.03946     0.03946     0.00000     0.03946
     5       0.01333     0.01333     0.00000     0.01333
     6       0.02649     0.02649     0.00000     0.02649
     7       0.01333     0.01333     0.00000     0.01333
     8       0.01333     0.01333     0.00000     0.01333
     9       0.02649     0.02649     0.00000     0.02649
    10       0.02649     0.02649     0.00000     0.02649
    11       0.01333     0.01333     0.00000     0.01333
    12       0.03946     0.03946     0.00000     0.03946
    13       0.01333     0.01333     0.00000     0.01333
    14       0.01333     0.01333     0.00000     0.01333
    15       0.02649     0.02649     0.00000     0.02649
    16       0.02649     0.02649     0.00000     0.02649
    17       0.02649     0.02649     0.00000     0.02649
    18       0.01333     0.01333     0.00000     0.01333
    19       0.01333     0.01333     0.00000     0.01333
    20       0.02649     0.02649     0.00000     0.02649
    21       0.01333     0.01333     0.00000     0.01333
    22       0.02649     0.02649     0.00000     0.02649
    23       0.03946     0.03946     0.00000     0.03946
    24       0.05226     0.05226     0.00000     0.05226
    25       0.01333     0.01333     0.00000     0.01333
    26       0.10166     0.10166     0.00000     0.10166
    27       0.02649     0.02649     0.00000     0.02649
    28       0.01333     0.01333     0.00000     0.01333
    29       0.01333     0.01333     0.00000     0.01333
    30       0.01333     0.01333     0.00000     0.01333
    31       0.01333     0.01333     0.00000     0.01333
    32       0.02649     0.02649     0.00000     0.02649
    33       0.03946     0.03946     0.00000     0.03946
    34       0.02649     0.02649     0.00000     0.02649
    35       0.03946     0.03946     0.00000     0.03946
    36       0.02649     0.02649     0.00000     0.02649
    37       0.01333     0.01333     0.00000     0.01333
    38       0.01333     0.01333     0.00000     0.01333
    39       0.01333     0.01333     0.00000     0.01333
    40       0.02649     0.02649     0.00000     0.02649
    41       0.03946     0.03946     0.00000     0.03946
    42       0.01333     0.01333     0.00000     0.01333
    43       0.01333     0.01333     0.00000     0.01333
    44       0.01333     0.01333     0.00000     0.01333
    45       0.02649     0.02649     0.00000     0.02649
    46       0.01333     0.01333     0.00000     0.01333
    47       0.01333     0.01333     0.00000     0.01333
    48       0.03946     0.03946     0.00000     0.03946
    49       0.01333     0.01333     0.00000     0.01333
    50       0.01333     0.01333     0.00000     0.01333
    51       0.01333     0.01333     0.00000     0.01333
    52       0.01333     0.01333     0.00000     0.01333
    53       0.02649     0.02649     0.00000     0.02649
    54       0.05226     0.05226     0.00000     0.05226
    55       0.01333     0.01333     0.00000     0.01333
    56       0.06488     0.06488     0.00000     0.06488
    57       0.02649     0.02649     0.00000     0.02649
    58       0.01333     0.01333     0.00000     0.01333
    59       0.02649     0.02649     0.00000     0.02649
    60       0.01333     0.01333     0.00000     0.01333
    61       0.02649     0.02649     0.00000     0.02649
    62       0.01333     0.01333     0.00000     0.01333
    63       0.01333     0.01333     0.00000     0.01333
    64       0.03946     0.03946     0.00000     0.03946
    65       0.01333     0.01333     0.00000     0.01333
    66       0.02649     0.02649     0.00000     0.02649
    67       0.03946     0.03946     0.00000     0.03946
    68       0.01333     0.01333     0.00000     0.01333
    69       0.01333     0.01333     0.00000     0.01333
    70       0.02649     0.02649     0.00000     0.02649
--------------------------------------------------------
  Mean       0.02467     0.02467     0.00000     0.02467
  s.d.       0.01590     0.01590     0.00000     0.01590
--------------------------------------------------------
<A NAME="31_01_17at16_29_32_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_29_32_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      33.06675    33.06675     0.00000
            Theta_k_lower      23.36170    23.36170     0.00000
            Theta_k_upper      46.50422    46.50422     0.00000
                  Theta_H       3.69441     3.69441     0.00000
             s.d. Theta_H       0.90322     0.90322     0.00000
                  Theta_S      12.53466    12.53466     0.00000
             s.d. Theta_S       3.15817     3.15817     0.00000
                 Theta_pi       1.72698     1.72698     0.00000
            s.d. Theta_pi       1.12033     1.12033     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_29_32_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.69141     1.69141     0.00000
              Tau qt 2.5%       1.39453     1.39453     0.00000
                Tau qt 5%       1.39453     1.39453     0.00000
               Tau qt 95%       1.82422     1.82422     0.00000
             Tau qt 97.5%       1.82422     1.82422     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00527     0.00527     0.00000
          Theta0 qt 97.5%       0.00527     0.00527     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      64.85938    64.85938     0.00000
             Theta1 qt 5%      64.85938    64.85938     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00027     0.00027     0.00000
      Model (SSD) p-value       1.00000     1.00000     0.00000
         Raggedness index       0.04368     0.04368     0.00000
       Raggedness p-value       0.50000     0.50000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 120 (31/01/17 at 16:29:32))
Total computing time for this run : 0h 0m 0s 111 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

