
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_47_58"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 324 (31/01/17 at 14:47:58)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_47_58_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_47_58_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 54
No. of loci              : 72
No. of usable loci       : 72 loci with less than  5.00 % missing data
No. of polymorphic sites : 72


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.06488
     3       150         2   0.02649
     4       150         2   0.02649
     5       150         2   0.02649
     6       150         2   0.01333
     7       150         2   0.10166
     8       150         2   0.01333
     9       150         2   0.10166
    10       150         2   0.01333
    11       150         2   0.06488
    12       150         2   0.01333
    13       150         2   0.01333
    14       150         2   0.05226
    15       150         2   0.02649
    16       150         2   0.02649
    17       150         2   0.07732
    18       150         2   0.05226
    19       150         2   0.05226
    20       150         2   0.01333
    21       150         2   0.10166
    22       150         2   0.01333
    23       150         2   0.15937
    24       150         2   0.13682
    25       150         2   0.01333
    26       150         2   0.01333
    27       150         2   0.01333
    28       150         2   0.06488
    29       150         2   0.06488
    30       150         2   0.05226
    31       150         2   0.01333
    32       150         2   0.02649
    33       150         2   0.01333
    34       150         2   0.01333
    35       150         2   0.33790
    36       150         2   0.02649
    37       150         2   0.11356
    38       150         2   0.01333
    39       150         2   0.02649
    40       150         2   0.06488
    41       150         2   0.02649
    42       150         2   0.02649
    43       150         2   0.01333
    44       150         2   0.02649
    45       150         2   0.03946
    46       150         2   0.01333
    47       150         2   0.03946
    48       150         2   0.01333
    49       150         2   0.01333
    50       150         2   0.01333
    51       150         2   0.01333
    52       150         2   0.07732
    53       150         2   0.02649
    54       150         2   0.01333
    55       150         2   0.01333
    56       150         2   0.13682
    57       150         2   0.06488
    58       150         2   0.03946
    59       150         2   0.05226
    60       150         2   0.01333
    61       150         2   0.03946
    62       150         2   0.01333
    63       150         2   0.01333
    64       150         2   0.08957
    65       150         2   0.02649
    66       150         2   0.10166
    67       150         2   0.07732
    68       150         2   0.10166
    69       150         2   0.33790
    70       150         2   0.02649
    71       150         2   0.08957
    72       150         2   0.05226
-------------------------------------------------
  Mean   150.000     2.000   0.05167
  s.d.     0.000     0.000   0.06050
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0522
Gene diversity           :  0.9542 +/-  0.0096

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 54

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 72
Number of usable nucleotide sites            : 72


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.719911 +/-     1.889286

Average gene diversity over loci              :     0.051665 +/-     0.029045

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    19.178368
  S.D. Theta(Hom) :     4.508651

         Theta(k) :    29.820320
95 % confidence interval limits for theta(k) : [  20.964943,  42.104679 ]

         Theta(S) :    12.892796
    S.D. Theta(S) :     3.238739

        Theta(Pi) :     3.719911
   S.D. Theta(Pi) :     2.091257

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     512       0.000
       1     988       0.000
       2    1304       0.000
       3    2304       0.000
       4    2248       0.000
       5    1676       0.000
       6    1449       0.000
       7     566       0.000
       8     118       0.000
       9      10       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.720
Mismatch observed variance          :        3.439
Tau                                 :        4.084
Theta0                              :        0.021
Theta1                              :       52.969

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00285334
          P(Sim. Ssd >= Obs. Ssd): 0.20000000
    Harpending's Raggedness index: 0.02162416
        P(Sim. Rag. >= Obs. Rag.): 0.20000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        512.0        326.6        374.5
          1.0        988.0        855.5        887.2
          2.0       1304.0       1607.2       1593.9
          3.0       2304.0       2011.9       2083.7
          4.0       2248.0       2113.4       2095.7
          5.0       1676.0       1676.3       1702.5
          6.0       1449.0       1169.6       1157.5
          7.0        566.0        689.5        676.1
          8.0        118.0        381.7        346.0
          9.0         10.0        196.9        157.5
         10.0          0.0         90.7         64.6
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.084           2.373           4.656           3.750
                   Theta0:           0.021           0.000           1.619           0.489               
                   Theta1:          52.969          22.010       99999.000       50023.754
Mean No. of pairwise diff.           3.720           3.403           4.692           4.041
           No. pol. sites:              72              78             101          94.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        512.0        144.0        563.0        374.5
          1.0        988.0        361.0       1522.0        887.2
          2.0       1304.0        969.0       2066.0       1593.9
          3.0       2304.0       1486.0       2890.0       2083.7
          4.0       2248.0       1658.0       2849.0       2095.7
          5.0       1676.0       1324.0       2221.0       1702.5
          6.0       1449.0        760.0       1517.0       1157.5
          7.0        566.0        286.0       1072.0        676.1
          8.0        118.0         93.0        701.0        346.0
          9.0         10.0         28.0        445.0        157.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.084           2.373           4.656           3.750
                   Theta0:           0.021           0.000           1.619           0.489               
                   Theta1:          52.969          22.010       99999.000       50023.754
Mean No. of pairwise diff.           3.720           3.403           4.692           4.041
           No. pol. sites:              72              78             101          94.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        512.0        144.0        563.0        374.5
          1.0        988.0        361.0       1522.0        887.2
          2.0       1304.0        969.0       2066.0       1593.9
          3.0       2304.0       1486.0       2890.0       2083.7
          4.0       2248.0       1658.0       2849.0       2095.7
          5.0       1676.0       1324.0       2221.0       1702.5
          6.0       1449.0        760.0       1517.0       1157.5
          7.0        566.0        286.0       1072.0        676.1
          8.0        118.0         93.0        701.0        346.0
          9.0         10.0         28.0        445.0        157.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.084           2.373           4.656           3.750
                   Theta0:           0.021           0.000           1.619           0.489               
                   Theta1:          52.969          22.010       99999.000       50023.754
Mean No. of pairwise diff.           3.720           3.403           4.692           4.041
           No. pol. sites:              72              78             101          94.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        512.0        144.0        563.0        374.5
          1.0        988.0        361.0       1522.0        887.2
          2.0       1304.0        969.0       2066.0       1593.9
          3.0       2304.0       1486.0       2890.0       2083.7
          4.0       2248.0       1658.0       2849.0       2095.7
          5.0       1676.0       1324.0       2221.0       1702.5
          6.0       1449.0        760.0       1517.0       1157.5
          7.0        566.0        286.0       1072.0        676.1
          8.0        118.0         93.0        701.0        346.0
          9.0         10.0         28.0        445.0        157.5
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_47_58_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            72      72.000       0.000
  No. of usable loci            72      72.000       0.000
  No. of polym. loci            72      72.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_47_58_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.06488     0.06488     0.00000     0.06488
     3       0.02649     0.02649     0.00000     0.02649
     4       0.02649     0.02649     0.00000     0.02649
     5       0.02649     0.02649     0.00000     0.02649
     6       0.01333     0.01333     0.00000     0.01333
     7       0.10166     0.10166     0.00000     0.10166
     8       0.01333     0.01333     0.00000     0.01333
     9       0.10166     0.10166     0.00000     0.10166
    10       0.01333     0.01333     0.00000     0.01333
    11       0.06488     0.06488     0.00000     0.06488
    12       0.01333     0.01333     0.00000     0.01333
    13       0.01333     0.01333     0.00000     0.01333
    14       0.05226     0.05226     0.00000     0.05226
    15       0.02649     0.02649     0.00000     0.02649
    16       0.02649     0.02649     0.00000     0.02649
    17       0.07732     0.07732     0.00000     0.07732
    18       0.05226     0.05226     0.00000     0.05226
    19       0.05226     0.05226     0.00000     0.05226
    20       0.01333     0.01333     0.00000     0.01333
    21       0.10166     0.10166     0.00000     0.10166
    22       0.01333     0.01333     0.00000     0.01333
    23       0.15937     0.15937     0.00000     0.15937
    24       0.13682     0.13682     0.00000     0.13682
    25       0.01333     0.01333     0.00000     0.01333
    26       0.01333     0.01333     0.00000     0.01333
    27       0.01333     0.01333     0.00000     0.01333
    28       0.06488     0.06488     0.00000     0.06488
    29       0.06488     0.06488     0.00000     0.06488
    30       0.05226     0.05226     0.00000     0.05226
    31       0.01333     0.01333     0.00000     0.01333
    32       0.02649     0.02649     0.00000     0.02649
    33       0.01333     0.01333     0.00000     0.01333
    34       0.01333     0.01333     0.00000     0.01333
    35       0.33790     0.33790     0.00000     0.33790
    36       0.02649     0.02649     0.00000     0.02649
    37       0.11356     0.11356     0.00000     0.11356
    38       0.01333     0.01333     0.00000     0.01333
    39       0.02649     0.02649     0.00000     0.02649
    40       0.06488     0.06488     0.00000     0.06488
    41       0.02649     0.02649     0.00000     0.02649
    42       0.02649     0.02649     0.00000     0.02649
    43       0.01333     0.01333     0.00000     0.01333
    44       0.02649     0.02649     0.00000     0.02649
    45       0.03946     0.03946     0.00000     0.03946
    46       0.01333     0.01333     0.00000     0.01333
    47       0.03946     0.03946     0.00000     0.03946
    48       0.01333     0.01333     0.00000     0.01333
    49       0.01333     0.01333     0.00000     0.01333
    50       0.01333     0.01333     0.00000     0.01333
    51       0.01333     0.01333     0.00000     0.01333
    52       0.07732     0.07732     0.00000     0.07732
    53       0.02649     0.02649     0.00000     0.02649
    54       0.01333     0.01333     0.00000     0.01333
    55       0.01333     0.01333     0.00000     0.01333
    56       0.13682     0.13682     0.00000     0.13682
    57       0.06488     0.06488     0.00000     0.06488
    58       0.03946     0.03946     0.00000     0.03946
    59       0.05226     0.05226     0.00000     0.05226
    60       0.01333     0.01333     0.00000     0.01333
    61       0.03946     0.03946     0.00000     0.03946
    62       0.01333     0.01333     0.00000     0.01333
    63       0.01333     0.01333     0.00000     0.01333
    64       0.08957     0.08957     0.00000     0.08957
    65       0.02649     0.02649     0.00000     0.02649
    66       0.10166     0.10166     0.00000     0.10166
    67       0.07732     0.07732     0.00000     0.07732
    68       0.10166     0.10166     0.00000     0.10166
    69       0.33790     0.33790     0.00000     0.33790
    70       0.02649     0.02649     0.00000     0.02649
    71       0.08957     0.08957     0.00000     0.08957
    72       0.05226     0.05226     0.00000     0.05226
--------------------------------------------------------
  Mean       0.05167     0.05167     0.00000     0.05167
  s.d.       0.06050     0.06050     0.00000     0.06050
--------------------------------------------------------
<A NAME="31_01_17at14_47_58_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
    72             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_47_58_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      29.82032    29.82032     0.00000
            Theta_k_lower      20.96494    20.96494     0.00000
            Theta_k_upper      42.10468    42.10468     0.00000
                  Theta_H      19.17837    19.17837     0.00000
             s.d. Theta_H       4.50865     4.50865     0.00000
                  Theta_S      12.89280    12.89280     0.00000
             s.d. Theta_S       3.23874     3.23874     0.00000
                 Theta_pi       3.71991     3.71991     0.00000
            s.d. Theta_pi       2.09126     2.09126     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_47_58_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.08398     4.08398     0.00000
              Tau qt 2.5%       2.37305     2.37305     0.00000
                Tau qt 5%       2.37305     2.37305     0.00000
               Tau qt 95%       4.50977     4.50977     0.00000
             Tau qt 97.5%       4.50977     4.50977     0.00000
                   Theta0       0.02109     0.02109     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.53457     1.53457     0.00000
          Theta0 qt 97.5%       1.53457     1.53457     0.00000
                   Theta1      52.96875    52.96875     0.00000
           Theta1 qt 2.5%      22.01050    22.01050     0.00000
             Theta1 qt 5%      22.01050    22.01050     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00285     0.00285     0.00000
      Model (SSD) p-value       0.20000     0.20000     0.00000
         Raggedness index       0.02162     0.02162     0.00000
       Raggedness p-value       0.20000     0.20000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 324 (31/01/17 at 14:47:59))
Total computing time for this run : 0h 0m 0s 118 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

