
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_47_36"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 171 (31/01/17 at 14:47:36)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_47_36_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_47_36_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 28
No. of loci              : 18
No. of usable loci       : 18 loci with less than  5.00 % missing data
No. of polymorphic sites : 18


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.02649
     2       150         2   0.02649
     3       150         2   0.03946
     4       150         2   0.15937
     5       150         2   0.25199
     6       150         2   0.03946
     7       150         2   0.12528
     8       150         2   0.14819
     9       150         2   0.14819
    10       150         2   0.08957
    11       150         2   0.28850
    12       150         2   0.03946
    13       150         2   0.01333
    14       150         2   0.21262
    15       150         2   0.03946
    16       150         2   0.01333
    17       150         2   0.13682
    18       150         2   0.19186
-------------------------------------------------
  Mean   150.000     2.000   0.11055
  s.d.     0.000     0.000   0.08695
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.2010
Gene diversity           :  0.8044 +/-  0.0320

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 28

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 18
Number of usable nucleotide sites            : 18


Distance method : Pairwise difference
Mean number of pairwise differences           :     1.989888 +/-     1.129837

Average gene diversity over loci              :     0.110549 +/-     0.069479

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     3.267364
  S.D. Theta(Hom) :     0.719774

         Theta(k) :     9.889483
95 % confidence interval limits for theta(k) : [   6.358882,  15.066083 ]

         Theta(S) :     3.223199
    S.D. Theta(S) :     1.032890

        Theta(Pi) :     1.989888
   S.D. Theta(Pi) :     1.250621

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    2186    1885.043
       1    2789    3086.812
       2    2267    2712.351
       3    2201    1737.089
       4     934     929.463
       5     400     449.967
       6     281     206.946
       7      69      92.978
       8      26      41.384
       9      22      18.359
      10       0       8.136
      11       0       3.605
      12       0       1.597
      13       0       0.708
      14       0       0.313
      15       0       0.139
      16       0       0.062
      17       0       0.027
      18       0       0.012


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        1.990
Mismatch observed variance          :        2.623
Tau                                 :        2.330
Theta0                              :        0.104
Theta1                              :        5.707

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00198738
          P(Sim. Ssd >= Obs. Ssd): 0.80000000
    Harpending's Raggedness index: 0.02075861
        P(Sim. Rag. >= Obs. Rag.): 0.80000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       2186.0       2426.8       2213.3
          1.0       2789.0       2582.0       2662.3
          2.0       2267.0       2416.9       2549.0
          3.0       2201.0       1870.2       1863.9
          4.0        934.0       1024.4       1075.8
          5.0        400.0        594.2        508.7
          6.0        281.0        141.2        203.1
          7.0         69.0         85.7         70.1
          8.0         26.0         25.9         21.3
          9.0         22.0          4.0          5.8
         10.0          0.0          3.7          1.4
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.330           0.750           3.660           2.470
                   Theta0:           0.104           0.000           0.410           0.063               
                   Theta1:           5.707           3.053       99999.000       20006.953
Mean No. of pairwise diff.           1.990           0.679           3.217           1.953
           No. pol. sites:              18              13              29          21.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2186.0       1221.0       5262.0       2213.3
          1.0       2789.0       1282.0       4421.0       2662.3
          2.0       2267.0       1312.0       3332.0       2549.0
          3.0       2201.0        180.0       2681.0       1863.9
          4.0        934.0          0.0       1918.0       1075.8
          5.0        400.0          0.0       1393.0        508.7
          6.0        281.0          0.0        570.0        203.1
          7.0         69.0          0.0        774.0         70.1
          8.0         26.0          0.0        243.0         21.3
          9.0         22.0          0.0         28.0          5.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.330           0.750           3.660           2.470
                   Theta0:           0.104           0.000           0.410           0.063               
                   Theta1:           5.707           3.053       99999.000       20006.953
Mean No. of pairwise diff.           1.990           0.679           3.217           1.953
           No. pol. sites:              18              13              29          21.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2186.0       1221.0       5262.0       2213.3
          1.0       2789.0       1282.0       4421.0       2662.3
          2.0       2267.0       1312.0       3332.0       2549.0
          3.0       2201.0        180.0       2681.0       1863.9
          4.0        934.0          0.0       1918.0       1075.8
          5.0        400.0          0.0       1393.0        508.7
          6.0        281.0          0.0        570.0        203.1
          7.0         69.0          0.0        774.0         70.1
          8.0         26.0          0.0        243.0         21.3
          9.0         22.0          0.0         28.0          5.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.330           0.750           3.660           2.470
                   Theta0:           0.104           0.000           0.410           0.063               
                   Theta1:           5.707           3.053       99999.000       20006.953
Mean No. of pairwise diff.           1.990           0.679           3.217           1.953
           No. pol. sites:              18              13              29          21.000
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2186.0       1221.0       5262.0       2213.3
          1.0       2789.0       1282.0       4421.0       2662.3
          2.0       2267.0       1312.0       3332.0       2549.0
          3.0       2201.0        180.0       2681.0       1863.9
          4.0        934.0          0.0       1918.0       1075.8
          5.0        400.0          0.0       1393.0        508.7
          6.0        281.0          0.0        570.0        203.1
          7.0         69.0          0.0        774.0         70.1
          8.0         26.0          0.0        243.0         21.3
          9.0         22.0          0.0         28.0          5.8
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_47_36_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            18      18.000       0.000
  No. of usable loci            18      18.000       0.000
  No. of polym. loci            18      18.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_47_36_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.02649     0.02649     0.00000     0.02649
     2       0.02649     0.02649     0.00000     0.02649
     3       0.03946     0.03946     0.00000     0.03946
     4       0.15937     0.15937     0.00000     0.15937
     5       0.25199     0.25199     0.00000     0.25199
     6       0.03946     0.03946     0.00000     0.03946
     7       0.12528     0.12528     0.00000     0.12528
     8       0.14819     0.14819     0.00000     0.14819
     9       0.14819     0.14819     0.00000     0.14819
    10       0.08957     0.08957     0.00000     0.08957
    11       0.28850     0.28850     0.00000     0.28850
    12       0.03946     0.03946     0.00000     0.03946
    13       0.01333     0.01333     0.00000     0.01333
    14       0.21262     0.21262     0.00000     0.21262
    15       0.03946     0.03946     0.00000     0.03946
    16       0.01333     0.01333     0.00000     0.01333
    17       0.13682     0.13682     0.00000     0.13682
    18       0.19186     0.19186     0.00000     0.19186
--------------------------------------------------------
  Mean       0.11055     0.11055     0.00000     0.11055
  s.d.       0.08695     0.08695     0.00000     0.08695
--------------------------------------------------------
<A NAME="31_01_17at14_47_36_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_47_36_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k       9.88948     9.88948     0.00000
            Theta_k_lower       6.35888     6.35888     0.00000
            Theta_k_upper      15.06608    15.06608     0.00000
                  Theta_H       3.26736     3.26736     0.00000
             s.d. Theta_H       0.71977     0.71977     0.00000
                  Theta_S       3.22320     3.22320     0.00000
             s.d. Theta_S       1.03289     1.03289     0.00000
                 Theta_pi       1.98989     1.98989     0.00000
            s.d. Theta_pi       1.25062     1.25062     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_47_36_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       2.33008     2.33008     0.00000
              Tau qt 2.5%       0.75000     0.75000     0.00000
                Tau qt 5%       0.75000     0.75000     0.00000
               Tau qt 95%       3.41016     3.41016     0.00000
             Tau qt 97.5%       3.41016     3.41016     0.00000
                   Theta0       0.10371     0.10371     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.16523     0.16523     0.00000
          Theta0 qt 97.5%       0.16523     0.16523     0.00000
                   Theta1       5.70679     5.70679     0.00000
           Theta1 qt 2.5%       3.05298     3.05298     0.00000
             Theta1 qt 5%       3.05298     3.05298     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00199     0.00199     0.00000
      Model (SSD) p-value       0.80000     0.80000     0.00000
         Raggedness index       0.02076     0.02076     0.00000
       Raggedness p-value       0.80000     0.80000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 171 (31/01/17 at 14:47:36))
Total computing time for this run : 0h 0m 0s 227 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

