
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_56_26"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 33 (31/01/17 at 14:56:26)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_56_26_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_56_26_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 46
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.07732
     2       150         2   0.25199
     3       150         2   0.01333
     4       150         2   0.41172
     5       150         2   0.02649
     6       150         2   0.06488
     7       150         2   0.50300
     8       150         2   0.46389
     9       150         2   0.01333
    10       150         2   0.27964
    11       150         2   0.50327
    12       150         2   0.19186
    13       150         2   0.24242
    14       150         2   0.46756
    15       150         2   0.05226
    16       150         2   0.07732
    17       150         2   0.22273
    18       150         2   0.42810
    19       150         2   0.20233
    20       150         2   0.06488
-------------------------------------------------
  Mean   150.000     2.000   0.22791
  s.d.     0.000     0.000   0.17858
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0444
Gene diversity           :  0.9620 +/-  0.0056

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 46

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.558300 +/-     2.253086

Average gene diversity over loci              :     0.227915 +/-     0.124697

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    23.589114
  S.D. Theta(Hom) :     3.863402

         Theta(k) :    22.270418
95 % confidence interval limits for theta(k) : [  15.393436,  31.891169 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     4.558300
   S.D. Theta(Pi) :     2.493949

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     425     139.908
       1     628     595.536
       2    1014    1279.755
       3    1457    1854.046
       4    2134    2041.104
       5    1874    1825.481
       6    1298    1385.367
       7    1187     920.567
       8     598     548.809
       9     475     299.459
      10      85     152.127
      11       0      73.029
      12       0      33.563
      13       0      14.935
      14       0       6.495
      15       0       2.782
      16       0       1.180
      17       0       0.497
      18       0       0.209
      19       0       0.088
      20       0       0.037


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.558
Mismatch observed variance          :        5.077
Tau                                 :        4.965
Theta0                              :        0.014
Theta1                              :       38.287

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00140905
          P(Sim. Ssd >= Obs. Ssd): 0.50000000
    Harpending's Raggedness index: 0.01423644
        P(Sim. Rag. >= Obs. Rag.): 0.60000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        425.0        344.7        350.3
          1.0        628.0        504.7        603.2
          2.0       1014.0        980.6       1075.8
          3.0       1457.0       1538.0       1586.5
          4.0       2134.0       1918.0       1877.3
          5.0       1874.0       1783.6       1822.4
          6.0       1298.0       1651.5       1490.5
          7.0       1187.0       1105.4       1050.6
          8.0        598.0        682.0        649.9
          9.0        475.0        384.9        358.0
         10.0         85.0        182.0        177.7
         11.0          0.0         67.8         80.2
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.965           3.842           5.484           4.933
                   Theta0:           0.014           0.000           0.738           0.156               
                   Theta1:          38.287          23.258       99999.000       10085.004
Mean No. of pairwise diff.           4.558           3.866           5.632           4.760
           No. pol. sites:              20              73              98          81.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        425.0        213.0        518.0        350.3
          1.0        628.0        188.0        904.0        603.2
          2.0       1014.0        457.0       1579.0       1075.8
          3.0       1457.0        922.0       2337.0       1586.5
          4.0       2134.0       1269.0       2334.0       1877.3
          5.0       1874.0       1461.0       2017.0       1822.4
          6.0       1298.0        919.0       2347.0       1490.5
          7.0       1187.0        377.0       1475.0       1050.6
          8.0        598.0        252.0       1111.0        649.9
          9.0        475.0        107.0        764.0        358.0
         10.0         85.0         14.0        515.0        177.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.965           3.842           5.484           4.933
                   Theta0:           0.014           0.000           0.738           0.156               
                   Theta1:          38.287          23.258       99999.000       10085.004
Mean No. of pairwise diff.           4.558           3.866           5.632           4.760
           No. pol. sites:              20              73              98          81.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        425.0        213.0        518.0        350.3
          1.0        628.0        188.0        904.0        603.2
          2.0       1014.0        457.0       1579.0       1075.8
          3.0       1457.0        922.0       2337.0       1586.5
          4.0       2134.0       1269.0       2334.0       1877.3
          5.0       1874.0       1461.0       2017.0       1822.4
          6.0       1298.0        919.0       2347.0       1490.5
          7.0       1187.0        377.0       1475.0       1050.6
          8.0        598.0        252.0       1111.0        649.9
          9.0        475.0        107.0        764.0        358.0
         10.0         85.0         14.0        515.0        177.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.965           3.842           5.484           4.933
                   Theta0:           0.014           0.000           0.738           0.156               
                   Theta1:          38.287          23.258       99999.000       10085.004
Mean No. of pairwise diff.           4.558           3.866           5.632           4.760
           No. pol. sites:              20              73              98          81.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        425.0        213.0        518.0        350.3
          1.0        628.0        188.0        904.0        603.2
          2.0       1014.0        457.0       1579.0       1075.8
          3.0       1457.0        922.0       2337.0       1586.5
          4.0       2134.0       1269.0       2334.0       1877.3
          5.0       1874.0       1461.0       2017.0       1822.4
          6.0       1298.0        919.0       2347.0       1490.5
          7.0       1187.0        377.0       1475.0       1050.6
          8.0        598.0        252.0       1111.0        649.9
          9.0        475.0        107.0        764.0        358.0
         10.0         85.0         14.0        515.0        177.7
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_56_26_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_56_26_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.07732     0.07732     0.00000     0.07732
     2       0.25199     0.25199     0.00000     0.25199
     3       0.01333     0.01333     0.00000     0.01333
     4       0.41172     0.41172     0.00000     0.41172
     5       0.02649     0.02649     0.00000     0.02649
     6       0.06488     0.06488     0.00000     0.06488
     7       0.50300     0.50300     0.00000     0.50300
     8       0.46389     0.46389     0.00000     0.46389
     9       0.01333     0.01333     0.00000     0.01333
    10       0.27964     0.27964     0.00000     0.27964
    11       0.50327     0.50327     0.00000     0.50327
    12       0.19186     0.19186     0.00000     0.19186
    13       0.24242     0.24242     0.00000     0.24242
    14       0.46756     0.46756     0.00000     0.46756
    15       0.05226     0.05226     0.00000     0.05226
    16       0.07732     0.07732     0.00000     0.07732
    17       0.22273     0.22273     0.00000     0.22273
    18       0.42810     0.42810     0.00000     0.42810
    19       0.20233     0.20233     0.00000     0.20233
    20       0.06488     0.06488     0.00000     0.06488
--------------------------------------------------------
  Mean       0.22791     0.22791     0.00000     0.22791
  s.d.       0.17858     0.17858     0.00000     0.17858
--------------------------------------------------------
<A NAME="31_01_17at14_56_26_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_56_26_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      22.27042    22.27042     0.00000
            Theta_k_lower      15.39344    15.39344     0.00000
            Theta_k_upper      31.89117    31.89117     0.00000
                  Theta_H      23.58911    23.58911     0.00000
             s.d. Theta_H       3.86340     3.86340     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       4.55830     4.55830     0.00000
            s.d. Theta_pi       2.49395     2.49395     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_56_26_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.96484     4.96484     0.00000
              Tau qt 2.5%       3.84180     3.84180     0.00000
                Tau qt 5%       3.84180     3.84180     0.00000
               Tau qt 95%       5.47070     5.47070     0.00000
             Tau qt 97.5%       5.47070     5.47070     0.00000
                   Theta0       0.01406     0.01406     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.51152     0.51152     0.00000
          Theta0 qt 97.5%       0.51152     0.51152     0.00000
                   Theta1      38.28735    38.28735     0.00000
           Theta1 qt 2.5%      23.25806    23.25806     0.00000
             Theta1 qt 5%      23.25806    23.25806     0.00000
            Theta1 qt 95%     182.97485   182.97485     0.00000
          Theta1 qt 97.5%     182.97485   182.97485     0.00000
                      SSD       0.00141     0.00141     0.00000
      Model (SSD) p-value       0.50000     0.50000     0.00000
         Raggedness index       0.01424     0.01424     0.00000
       Raggedness p-value       0.60000     0.60000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 33 (31/01/17 at 14:56:26))
Total computing time for this run : 0h 0m 0s 160 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

