
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_31_50"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 111 (31/01/17 at 16:31:50)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_31_50_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_31_50_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 60
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.17038
     2       150         2   0.44286
     3       150         2   0.14819
     4       150         2   0.01333
     5       150         2   0.32215
     6       150         2   0.30568
     7       150         2   0.02649
     8       150         2   0.33011
     9       150         2   0.17038
    10       150         2   0.02649
    11       150         2   0.01333
    12       150         2   0.32215
    13       150         2   0.11356
    14       150         2   0.10166
    15       150         2   0.23266
    16       150         2   0.01333
    17       150         2   0.05226
    18       150         2   0.37414
    19       150         2   0.20233
    20       150         2   0.03946
-------------------------------------------------
  Mean   150.000     2.000   0.17105
  s.d.     0.000     0.000   0.13869
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0305
Gene diversity           :  0.9760 +/-  0.0036

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 60

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.420940 +/-     1.759123

Average gene diversity over loci              :     0.171047 +/-     0.097359

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    38.886418
  S.D. Theta(Hom) :     6.186924

         Theta(k) :    36.572596
95 % confidence interval limits for theta(k) : [  25.948187,  51.263916 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     3.420940
   S.D. Theta(Pi) :     1.947179

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     268       0.000
       1     937       0.000
       2    2107       0.000
       3    2909       0.000
       4    2408       0.000
       5    1350       0.000
       6     684       0.000
       7     309       0.000
       8     137       0.000
       9      55       0.000
      10      10       0.000
      11       1       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.421
Mismatch observed variance          :        2.788
Tau                                 :        3.459
Theta0                              :        0.000
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00370087
          P(Sim. Ssd >= Obs. Ssd): 0.20000000
    Harpending's Raggedness index: 0.03565506
        P(Sim. Rag. >= Obs. Rag.): 0.10000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        268.0        311.7        351.7
          1.0        937.0       1113.6       1216.2
          2.0       2107.0       2084.1       2103.3
          3.0       2909.0       2457.3       2425.0
          4.0       2408.0       2142.1       2097.0
          5.0       1350.0       1492.1       1450.7
          6.0        684.0        864.9        836.3
          7.0        309.0        426.4        413.3
          8.0        137.0        182.3        178.7
          9.0         55.0         70.5         68.7
         10.0         10.0         21.6         23.8
         11.0          1.0          6.0          7.5
         12.0          0.0          1.9          2.2
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.459           2.959           4.271           3.472
                   Theta0:           0.000           0.000           0.192           0.021               
                   Theta1:       99999.000         101.812       99999.000       70811.688
Mean No. of pairwise diff.           3.421           3.035           4.174           3.513
           No. pol. sites:              20             176             202         191.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        268.0        105.0        595.0        351.7
          1.0        937.0        579.0       1552.0       1216.2
          2.0       2107.0       1371.0       2608.0       2103.3
          3.0       2909.0       2098.0       3014.0       2425.0
          4.0       2408.0       1903.0       2335.0       2097.0
          5.0       1350.0        953.0       2064.0       1450.7
          6.0        684.0        448.0       1379.0        836.3
          7.0        309.0        246.0        706.0        413.3
          8.0        137.0         81.0        298.0        178.7
          9.0         55.0         18.0        134.0         68.7
         10.0         10.0          3.0         53.0         23.8
         11.0          1.0          0.0         25.0          7.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.459           2.959           4.271           3.472
                   Theta0:           0.000           0.000           0.192           0.021               
                   Theta1:       99999.000         101.812       99999.000       70811.688
Mean No. of pairwise diff.           3.421           3.035           4.174           3.513
           No. pol. sites:              20             176             202         191.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        268.0        105.0        595.0        351.7
          1.0        937.0        579.0       1552.0       1216.2
          2.0       2107.0       1371.0       2608.0       2103.3
          3.0       2909.0       2098.0       3014.0       2425.0
          4.0       2408.0       1903.0       2335.0       2097.0
          5.0       1350.0        953.0       2064.0       1450.7
          6.0        684.0        448.0       1379.0        836.3
          7.0        309.0        246.0        706.0        413.3
          8.0        137.0         81.0        298.0        178.7
          9.0         55.0         18.0        134.0         68.7
         10.0         10.0          3.0         53.0         23.8
         11.0          1.0          0.0         25.0          7.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.459           2.959           4.271           3.472
                   Theta0:           0.000           0.000           0.192           0.021               
                   Theta1:       99999.000         101.812       99999.000       70811.688
Mean No. of pairwise diff.           3.421           3.035           4.174           3.513
           No. pol. sites:              20             176             202         191.300
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        268.0        105.0        595.0        351.7
          1.0        937.0        579.0       1552.0       1216.2
          2.0       2107.0       1371.0       2608.0       2103.3
          3.0       2909.0       2098.0       3014.0       2425.0
          4.0       2408.0       1903.0       2335.0       2097.0
          5.0       1350.0        953.0       2064.0       1450.7
          6.0        684.0        448.0       1379.0        836.3
          7.0        309.0        246.0        706.0        413.3
          8.0        137.0         81.0        298.0        178.7
          9.0         55.0         18.0        134.0         68.7
         10.0         10.0          3.0         53.0         23.8
         11.0          1.0          0.0         25.0          7.5
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_31_50_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_31_50_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.17038     0.17038     0.00000     0.17038
     2       0.44286     0.44286     0.00000     0.44286
     3       0.14819     0.14819     0.00000     0.14819
     4       0.01333     0.01333     0.00000     0.01333
     5       0.32215     0.32215     0.00000     0.32215
     6       0.30568     0.30568     0.00000     0.30568
     7       0.02649     0.02649     0.00000     0.02649
     8       0.33011     0.33011     0.00000     0.33011
     9       0.17038     0.17038     0.00000     0.17038
    10       0.02649     0.02649     0.00000     0.02649
    11       0.01333     0.01333     0.00000     0.01333
    12       0.32215     0.32215     0.00000     0.32215
    13       0.11356     0.11356     0.00000     0.11356
    14       0.10166     0.10166     0.00000     0.10166
    15       0.23266     0.23266     0.00000     0.23266
    16       0.01333     0.01333     0.00000     0.01333
    17       0.05226     0.05226     0.00000     0.05226
    18       0.37414     0.37414     0.00000     0.37414
    19       0.20233     0.20233     0.00000     0.20233
    20       0.03946     0.03946     0.00000     0.03946
--------------------------------------------------------
  Mean       0.17105     0.17105     0.00000     0.17105
  s.d.       0.13869     0.13869     0.00000     0.13869
--------------------------------------------------------
<A NAME="31_01_17at16_31_50_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_31_50_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      36.57260    36.57260     0.00000
            Theta_k_lower      25.94819    25.94819     0.00000
            Theta_k_upper      51.26392    51.26392     0.00000
                  Theta_H      38.88642    38.88642     0.00000
             s.d. Theta_H       6.18692     6.18692     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       3.42094     3.42094     0.00000
            s.d. Theta_pi       1.94718     1.94718     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_31_50_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.45898     3.45898     0.00000
              Tau qt 2.5%       2.95898     2.95898     0.00000
                Tau qt 5%       2.95898     2.95898     0.00000
               Tau qt 95%       3.73438     3.73438     0.00000
             Tau qt 97.5%       3.73438     3.73438     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.01406     0.01406     0.00000
          Theta0 qt 97.5%       0.01406     0.01406     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%     101.81250   101.81250     0.00000
             Theta1 qt 5%     101.81250   101.81250     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00370     0.00370     0.00000
      Model (SSD) p-value       0.20000     0.20000     0.00000
         Raggedness index       0.03566     0.03566     0.00000
       Raggedness p-value       0.10000     0.10000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 111 (31/01/17 at 16:31:50))
Total computing time for this run : 0h 0m 0s 130 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

