
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_24"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 280 (31/01/17 at 14:44:24)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_24_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_24_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 38
No. of loci              : 57
No. of usable loci       : 57 loci with less than  5.00 % missing data
No. of polymorphic sites : 57


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.02649
     3       150         2   0.01333
     4       150         2   0.05226
     5       150         2   0.10166
     6       150         2   0.06488
     7       150         2   0.18121
     8       150         2   0.03946
     9       150         2   0.05226
    10       150         2   0.01333
    11       150         2   0.01333
    12       150         2   0.12528
    13       150         2   0.02649
    14       150         2   0.05226
    15       150         2   0.02649
    16       150         2   0.01333
    17       150         2   0.01333
    18       150         2   0.12528
    19       150         2   0.02649
    20       150         2   0.01333
    21       150         2   0.03946
    22       150         2   0.02649
    23       150         2   0.03946
    24       150         2   0.05226
    25       150         2   0.18121
    26       150         2   0.01333
    27       150         2   0.02649
    28       150         2   0.12528
    29       150         2   0.07732
    30       150         2   0.01333
    31       150         2   0.02649
    32       150         2   0.12528
    33       150         2   0.06488
    34       150         2   0.02649
    35       150         2   0.01333
    36       150         2   0.02649
    37       150         2   0.20233
    38       150         2   0.02649
    39       150         2   0.02649
    40       150         2   0.02649
    41       150         2   0.05226
    42       150         2   0.02649
    43       150         2   0.02649
    44       150         2   0.01333
    45       150         2   0.01333
    46       150         2   0.02649
    47       150         2   0.03946
    48       150         2   0.01333
    49       150         2   0.03946
    50       150         2   0.12528
    51       150         2   0.01333
    52       150         2   0.10166
    53       150         2   0.19186
    54       150         2   0.01333
    55       150         2   0.03946
    56       150         2   0.01333
    57       150         2   0.02649
-------------------------------------------------
  Mean   150.000     2.000   0.05138
  s.d.     0.000     0.000   0.05063
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.1138
Gene diversity           :  0.8922 +/-  0.0195

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 38

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 57
Number of usable nucleotide sites            : 57


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.928591 +/-     1.544047

Average gene diversity over loci              :     0.051379 +/-     0.029984

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     7.027479
  S.D. Theta(Hom) :     1.564731

         Theta(k) :    16.078736
95 % confidence interval limits for theta(k) : [  10.845089,  23.508807 ]

         Theta(S) :    10.206797
    S.D. Theta(S) :     2.633604

        Theta(Pi) :     2.928591
   S.D. Theta(Pi) :     1.709111

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1205     911.385
       1    1885    2058.525
       2    2245    2462.327
       3    2104    2111.375
       4    1347    1484.154
       5    1034     926.220
       6     562     540.443
       7     428     304.634
       8     221     169.017
       9     102      93.173
      10      34      51.244
      11       8      28.162
      12       0      15.474
      13       0       8.502
      14       0       4.671
      15       0       2.566
      16       0       1.410
      17       0       0.775
      18       0       0.426
      19       0       0.234
      20       0       0.128
      21       0       0.071
      22       0       0.039


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.929
Mismatch observed variance          :        4.415
Tau                                 :        1.975
Theta0                              :        1.273
Theta1                              :       15.845

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00035819
          P(Sim. Ssd >= Obs. Ssd): 0.90000000
    Harpending's Raggedness index: 0.01270023
        P(Sim. Rag. >= Obs. Rag.): 1.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1205.0        976.2       1184.7
          1.0       1885.0       2085.0       1914.4
          2.0       2245.0       2419.2       2235.6
          3.0       2104.0       1837.7       1999.1
          4.0       1347.0       1275.4       1483.6
          5.0       1034.0        847.4        973.5
          6.0        562.0        705.1        591.8
          7.0        428.0        326.9        344.6
          8.0        221.0        127.6        196.2
          9.0        102.0         98.1        110.6
         10.0         34.0        165.2         62.1
         11.0          8.0        175.6         34.8
         12.0          0.0        100.5         19.5
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.975           0.424           5.018           2.493
                   Theta0:           1.273           0.000           5.850           0.894               
                   Theta1:          15.845          11.168       99999.000       60007.361
Mean No. of pairwise diff.           2.929           1.840           5.946           3.172
           No. pol. sites:              57              36              53          42.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1205.0        568.0       1843.0       1184.7
          1.0       1885.0       1280.0       3146.0       1914.4
          2.0       2245.0       1478.0       3259.0       2235.6
          3.0       2104.0       1025.0       3237.0       1999.1
          4.0       1347.0        301.0       2155.0       1483.6
          5.0       1034.0         48.0       1714.0        973.5
          6.0        562.0          0.0       1734.0        591.8
          7.0        428.0          0.0       1043.0        344.6
          8.0        221.0          0.0        366.0        196.2
          9.0        102.0          0.0        540.0        110.6
         10.0         34.0          0.0       1560.0         62.1
         11.0          8.0          0.0       1728.0         34.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.975           0.424           5.018           2.493
                   Theta0:           1.273           0.000           5.850           0.894               
                   Theta1:          15.845          11.168       99999.000       60007.361
Mean No. of pairwise diff.           2.929           1.840           5.946           3.172
           No. pol. sites:              57              36              53          42.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1205.0        568.0       1843.0       1184.7
          1.0       1885.0       1280.0       3146.0       1914.4
          2.0       2245.0       1478.0       3259.0       2235.6
          3.0       2104.0       1025.0       3237.0       1999.1
          4.0       1347.0        301.0       2155.0       1483.6
          5.0       1034.0         48.0       1714.0        973.5
          6.0        562.0          0.0       1734.0        591.8
          7.0        428.0          0.0       1043.0        344.6
          8.0        221.0          0.0        366.0        196.2
          9.0        102.0          0.0        540.0        110.6
         10.0         34.0          0.0       1560.0         62.1
         11.0          8.0          0.0       1728.0         34.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.975           0.424           5.018           2.493
                   Theta0:           1.273           0.000           5.850           0.894               
                   Theta1:          15.845          11.168       99999.000       60007.361
Mean No. of pairwise diff.           2.929           1.840           5.946           3.172
           No. pol. sites:              57              36              53          42.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1205.0        568.0       1843.0       1184.7
          1.0       1885.0       1280.0       3146.0       1914.4
          2.0       2245.0       1478.0       3259.0       2235.6
          3.0       2104.0       1025.0       3237.0       1999.1
          4.0       1347.0        301.0       2155.0       1483.6
          5.0       1034.0         48.0       1714.0        973.5
          6.0        562.0          0.0       1734.0        591.8
          7.0        428.0          0.0       1043.0        344.6
          8.0        221.0          0.0        366.0        196.2
          9.0        102.0          0.0        540.0        110.6
         10.0         34.0          0.0       1560.0         62.1
         11.0          8.0          0.0       1728.0         34.8
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_24_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            57      57.000       0.000
  No. of usable loci            57      57.000       0.000
  No. of polym. loci            57      57.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_24_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.02649     0.02649     0.00000     0.02649
     3       0.01333     0.01333     0.00000     0.01333
     4       0.05226     0.05226     0.00000     0.05226
     5       0.10166     0.10166     0.00000     0.10166
     6       0.06488     0.06488     0.00000     0.06488
     7       0.18121     0.18121     0.00000     0.18121
     8       0.03946     0.03946     0.00000     0.03946
     9       0.05226     0.05226     0.00000     0.05226
    10       0.01333     0.01333     0.00000     0.01333
    11       0.01333     0.01333     0.00000     0.01333
    12       0.12528     0.12528     0.00000     0.12528
    13       0.02649     0.02649     0.00000     0.02649
    14       0.05226     0.05226     0.00000     0.05226
    15       0.02649     0.02649     0.00000     0.02649
    16       0.01333     0.01333     0.00000     0.01333
    17       0.01333     0.01333     0.00000     0.01333
    18       0.12528     0.12528     0.00000     0.12528
    19       0.02649     0.02649     0.00000     0.02649
    20       0.01333     0.01333     0.00000     0.01333
    21       0.03946     0.03946     0.00000     0.03946
    22       0.02649     0.02649     0.00000     0.02649
    23       0.03946     0.03946     0.00000     0.03946
    24       0.05226     0.05226     0.00000     0.05226
    25       0.18121     0.18121     0.00000     0.18121
    26       0.01333     0.01333     0.00000     0.01333
    27       0.02649     0.02649     0.00000     0.02649
    28       0.12528     0.12528     0.00000     0.12528
    29       0.07732     0.07732     0.00000     0.07732
    30       0.01333     0.01333     0.00000     0.01333
    31       0.02649     0.02649     0.00000     0.02649
    32       0.12528     0.12528     0.00000     0.12528
    33       0.06488     0.06488     0.00000     0.06488
    34       0.02649     0.02649     0.00000     0.02649
    35       0.01333     0.01333     0.00000     0.01333
    36       0.02649     0.02649     0.00000     0.02649
    37       0.20233     0.20233     0.00000     0.20233
    38       0.02649     0.02649     0.00000     0.02649
    39       0.02649     0.02649     0.00000     0.02649
    40       0.02649     0.02649     0.00000     0.02649
    41       0.05226     0.05226     0.00000     0.05226
    42       0.02649     0.02649     0.00000     0.02649
    43       0.02649     0.02649     0.00000     0.02649
    44       0.01333     0.01333     0.00000     0.01333
    45       0.01333     0.01333     0.00000     0.01333
    46       0.02649     0.02649     0.00000     0.02649
    47       0.03946     0.03946     0.00000     0.03946
    48       0.01333     0.01333     0.00000     0.01333
    49       0.03946     0.03946     0.00000     0.03946
    50       0.12528     0.12528     0.00000     0.12528
    51       0.01333     0.01333     0.00000     0.01333
    52       0.10166     0.10166     0.00000     0.10166
    53       0.19186     0.19186     0.00000     0.19186
    54       0.01333     0.01333     0.00000     0.01333
    55       0.03946     0.03946     0.00000     0.03946
    56       0.01333     0.01333     0.00000     0.01333
    57       0.02649     0.02649     0.00000     0.02649
--------------------------------------------------------
  Mean       0.05138     0.05138     0.00000     0.05138
  s.d.       0.05063     0.05063     0.00000     0.05063
--------------------------------------------------------
<A NAME="31_01_17at14_44_24_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_24_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      16.07874    16.07874     0.00000
            Theta_k_lower      10.84509    10.84509     0.00000
            Theta_k_upper      23.50881    23.50881     0.00000
                  Theta_H       7.02748     7.02748     0.00000
             s.d. Theta_H       1.56473     1.56473     0.00000
                  Theta_S      10.20680    10.20680     0.00000
             s.d. Theta_S       2.63360     2.63360     0.00000
                 Theta_pi       2.92859     2.92859     0.00000
            s.d. Theta_pi       1.70911     1.70911     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_24_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.97461     1.97461     0.00000
              Tau qt 2.5%       0.42383     0.42383     0.00000
                Tau qt 5%       0.42383     0.42383     0.00000
               Tau qt 95%       4.35742     4.35742     0.00000
             Tau qt 97.5%       4.35742     4.35742     0.00000
                   Theta0       1.27266     1.27266     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.88438     1.88438     0.00000
          Theta0 qt 97.5%       1.88438     1.88438     0.00000
                   Theta1      15.84531    15.84531     0.00000
           Theta1 qt 2.5%      11.16758    11.16758     0.00000
             Theta1 qt 5%      11.16758    11.16758     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00036     0.00036     0.00000
      Model (SSD) p-value       0.90000     0.90000     0.00000
         Raggedness index       0.01270     0.01270     0.00000
       Raggedness p-value       1.00000     1.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 280 (31/01/17 at 14:44:25))
Total computing time for this run : 0h 0m 0s 180 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

