
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_29_37"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 156 (31/01/17 at 16:29:37)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_29_37_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_29_37_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 65
No. of loci              : 77
No. of usable loci       : 77 loci with less than  5.00 % missing data
No. of polymorphic sites : 77


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.01333
     3       150         2   0.01333
     4       150         2   0.01333
     5       150         2   0.03946
     6       150         2   0.01333
     7       150         2   0.01333
     8       150         2   0.02649
     9       150         2   0.01333
    10       150         2   0.01333
    11       150         2   0.01333
    12       150         2   0.01333
    13       150         2   0.01333
    14       150         2   0.01333
    15       150         2   0.01333
    16       150         2   0.01333
    17       150         2   0.01333
    18       150         2   0.01333
    19       150         2   0.01333
    20       150         2   0.01333
    21       150         2   0.02649
    22       150         2   0.01333
    23       150         2   0.03946
    24       150         2   0.07732
    25       150         2   0.01333
    26       150         2   0.02649
    27       150         2   0.02649
    28       150         2   0.01333
    29       150         2   0.01333
    30       150         2   0.01333
    31       150         2   0.03946
    32       150         2   0.01333
    33       150         2   0.01333
    34       150         2   0.03946
    35       150         2   0.01333
    36       150         2   0.02649
    37       150         2   0.01333
    38       150         2   0.01333
    39       150         2   0.01333
    40       150         2   0.01333
    41       150         2   0.02649
    42       150         2   0.02649
    43       150         2   0.01333
    44       150         2   0.01333
    45       150         2   0.01333
    46       150         2   0.01333
    47       150         2   0.01333
    48       150         2   0.01333
    49       150         2   0.01333
    50       150         2   0.01333
    51       150         2   0.01333
    52       150         2   0.01333
    53       150         2   0.02649
    54       150         2   0.02649
    55       150         2   0.01333
    56       150         2   0.01333
    57       150         2   0.01333
    58       150         2   0.01333
    59       150         2   0.01333
    60       150         2   0.01333
    61       150         2   0.01333
    62       150         2   0.01333
    63       150         2   0.03946
    64       150         2   0.02649
    65       150         2   0.01333
    66       150         2   0.02649
    67       150         2   0.01333
    68       150         2   0.01333
    69       150         2   0.01333
    70       150         2   0.03946
    71       150         2   0.03946
    72       150         2   0.01333
    73       150         2   0.01333
    74       150         2   0.01333
    75       150         2   0.02649
    76       150         2   0.03946
    77       150         2   0.10166
-------------------------------------------------
  Mean   150.000     2.000   0.02008
  s.d.     0.000     0.000   0.01448
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.1679
Gene diversity           :  0.8377 +/-  0.0311

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 65

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 77
Number of usable nucleotide sites            : 77


Distance method : Pairwise difference
Mean number of pairwise differences           :     1.545861 +/-     0.930324

Average gene diversity over loci              :     0.020076 +/-     0.013374

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     4.183971
  S.D. Theta(Hom) :     1.045251

         Theta(k) :    43.063016
95 % confidence interval limits for theta(k) : [  30.727796,  60.104460 ]

         Theta(S) :    13.788129
    S.D. Theta(S) :     3.440055

        Theta(Pi) :     1.545861
   S.D. Theta(Pi) :     1.029779

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1814       0.000
       1    4161       0.000
       2    3212       0.000
       3    1376       0.000
       4     511       0.000
       5      88       0.000
       6      13       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        1.546
Mismatch observed variance          :        1.211
Tau                                 :        1.609
Theta0                              :        0.000
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00524241
          P(Sim. Ssd >= Obs. Ssd): 0.10000000
    Harpending's Raggedness index: 0.08578476
        P(Sim. Rag. >= Obs. Rag.): 0.10000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1814.0       2060.8       2235.2
          1.0       4161.0       3823.0       3597.2
          2.0       3212.0       2997.1       2894.6
          3.0       1376.0       1531.3       1552.8
          4.0        511.0        579.3        624.8
          5.0         88.0        150.7        201.1
          6.0         13.0         29.3         53.9
          7.0          0.0          3.4         12.4
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.609           1.357           1.959           1.610
                   Theta0:           0.000           0.000           0.002           0.000               
                   Theta1:       99999.000          17.086       99999.000       80007.910
Mean No. of pairwise diff.           1.546           1.426           1.904           1.582
           No. pol. sites:              77              92             119         101.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1814.0       1276.0       2775.0       2235.2
          1.0       4161.0       3318.0       4165.0       3597.2
          2.0       3212.0       2233.0       3554.0       2894.6
          3.0       1376.0       1278.0       2036.0       1552.8
          4.0        511.0        347.0        868.0        624.8
          5.0         88.0         82.0        265.0        201.1
          6.0         13.0          8.0         66.0         53.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.609           1.357           1.959           1.610
                   Theta0:           0.000           0.000           0.002           0.000               
                   Theta1:       99999.000          17.086       99999.000       80007.910
Mean No. of pairwise diff.           1.546           1.426           1.904           1.582
           No. pol. sites:              77              92             119         101.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1814.0       1276.0       2775.0       2235.2
          1.0       4161.0       3318.0       4165.0       3597.2
          2.0       3212.0       2233.0       3554.0       2894.6
          3.0       1376.0       1278.0       2036.0       1552.8
          4.0        511.0        347.0        868.0        624.8
          5.0         88.0         82.0        265.0        201.1
          6.0         13.0          8.0         66.0         53.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.609           1.357           1.959           1.610
                   Theta0:           0.000           0.000           0.002           0.000               
                   Theta1:       99999.000          17.086       99999.000       80007.910
Mean No. of pairwise diff.           1.546           1.426           1.904           1.582
           No. pol. sites:              77              92             119         101.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1814.0       1276.0       2775.0       2235.2
          1.0       4161.0       3318.0       4165.0       3597.2
          2.0       3212.0       2233.0       3554.0       2894.6
          3.0       1376.0       1278.0       2036.0       1552.8
          4.0        511.0        347.0        868.0        624.8
          5.0         88.0         82.0        265.0        201.1
          6.0         13.0          8.0         66.0         53.9
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_29_37_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            77      77.000       0.000
  No. of usable loci            77      77.000       0.000
  No. of polym. loci            77      77.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_29_37_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.01333     0.01333     0.00000     0.01333
     3       0.01333     0.01333     0.00000     0.01333
     4       0.01333     0.01333     0.00000     0.01333
     5       0.03946     0.03946     0.00000     0.03946
     6       0.01333     0.01333     0.00000     0.01333
     7       0.01333     0.01333     0.00000     0.01333
     8       0.02649     0.02649     0.00000     0.02649
     9       0.01333     0.01333     0.00000     0.01333
    10       0.01333     0.01333     0.00000     0.01333
    11       0.01333     0.01333     0.00000     0.01333
    12       0.01333     0.01333     0.00000     0.01333
    13       0.01333     0.01333     0.00000     0.01333
    14       0.01333     0.01333     0.00000     0.01333
    15       0.01333     0.01333     0.00000     0.01333
    16       0.01333     0.01333     0.00000     0.01333
    17       0.01333     0.01333     0.00000     0.01333
    18       0.01333     0.01333     0.00000     0.01333
    19       0.01333     0.01333     0.00000     0.01333
    20       0.01333     0.01333     0.00000     0.01333
    21       0.02649     0.02649     0.00000     0.02649
    22       0.01333     0.01333     0.00000     0.01333
    23       0.03946     0.03946     0.00000     0.03946
    24       0.07732     0.07732     0.00000     0.07732
    25       0.01333     0.01333     0.00000     0.01333
    26       0.02649     0.02649     0.00000     0.02649
    27       0.02649     0.02649     0.00000     0.02649
    28       0.01333     0.01333     0.00000     0.01333
    29       0.01333     0.01333     0.00000     0.01333
    30       0.01333     0.01333     0.00000     0.01333
    31       0.03946     0.03946     0.00000     0.03946
    32       0.01333     0.01333     0.00000     0.01333
    33       0.01333     0.01333     0.00000     0.01333
    34       0.03946     0.03946     0.00000     0.03946
    35       0.01333     0.01333     0.00000     0.01333
    36       0.02649     0.02649     0.00000     0.02649
    37       0.01333     0.01333     0.00000     0.01333
    38       0.01333     0.01333     0.00000     0.01333
    39       0.01333     0.01333     0.00000     0.01333
    40       0.01333     0.01333     0.00000     0.01333
    41       0.02649     0.02649     0.00000     0.02649
    42       0.02649     0.02649     0.00000     0.02649
    43       0.01333     0.01333     0.00000     0.01333
    44       0.01333     0.01333     0.00000     0.01333
    45       0.01333     0.01333     0.00000     0.01333
    46       0.01333     0.01333     0.00000     0.01333
    47       0.01333     0.01333     0.00000     0.01333
    48       0.01333     0.01333     0.00000     0.01333
    49       0.01333     0.01333     0.00000     0.01333
    50       0.01333     0.01333     0.00000     0.01333
    51       0.01333     0.01333     0.00000     0.01333
    52       0.01333     0.01333     0.00000     0.01333
    53       0.02649     0.02649     0.00000     0.02649
    54       0.02649     0.02649     0.00000     0.02649
    55       0.01333     0.01333     0.00000     0.01333
    56       0.01333     0.01333     0.00000     0.01333
    57       0.01333     0.01333     0.00000     0.01333
    58       0.01333     0.01333     0.00000     0.01333
    59       0.01333     0.01333     0.00000     0.01333
    60       0.01333     0.01333     0.00000     0.01333
    61       0.01333     0.01333     0.00000     0.01333
    62       0.01333     0.01333     0.00000     0.01333
    63       0.03946     0.03946     0.00000     0.03946
    64       0.02649     0.02649     0.00000     0.02649
    65       0.01333     0.01333     0.00000     0.01333
    66       0.02649     0.02649     0.00000     0.02649
    67       0.01333     0.01333     0.00000     0.01333
    68       0.01333     0.01333     0.00000     0.01333
    69       0.01333     0.01333     0.00000     0.01333
    70       0.03946     0.03946     0.00000     0.03946
    71       0.03946     0.03946     0.00000     0.03946
    72       0.01333     0.01333     0.00000     0.01333
    73       0.01333     0.01333     0.00000     0.01333
    74       0.01333     0.01333     0.00000     0.01333
    75       0.02649     0.02649     0.00000     0.02649
    76       0.03946     0.03946     0.00000     0.03946
    77       0.10166     0.10166     0.00000     0.10166
--------------------------------------------------------
  Mean       0.02008     0.02008     0.00000     0.02008
  s.d.       0.01448     0.01448     0.00000     0.01448
--------------------------------------------------------
<A NAME="31_01_17at16_29_37_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
    72             2       2.000       0.000           2
    73             2       2.000       0.000           2
    74             2       2.000       0.000           2
    75             2       2.000       0.000           2
    76             2       2.000       0.000           2
    77             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_29_37_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      43.06302    43.06302     0.00000
            Theta_k_lower      30.72780    30.72780     0.00000
            Theta_k_upper      60.10446    60.10446     0.00000
                  Theta_H       4.18397     4.18397     0.00000
             s.d. Theta_H       1.04525     1.04525     0.00000
                  Theta_S      13.78813    13.78813     0.00000
             s.d. Theta_S       3.44006     3.44006     0.00000
                 Theta_pi       1.54586     1.54586     0.00000
            s.d. Theta_pi       1.02978     1.02978     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_29_37_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.60938     1.60938     0.00000
              Tau qt 2.5%       1.35742     1.35742     0.00000
                Tau qt 5%       1.35742     1.35742     0.00000
               Tau qt 95%       1.72852     1.72852     0.00000
             Tau qt 97.5%       1.72852     1.72852     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00000     0.00000     0.00000
          Theta0 qt 97.5%       0.00000     0.00000     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      17.08594    17.08594     0.00000
             Theta1 qt 5%      17.08594    17.08594     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00524     0.00524     0.00000
      Model (SSD) p-value       0.10000     0.10000     0.00000
         Raggedness index       0.08578     0.08578     0.00000
       Raggedness p-value       0.10000     0.10000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 156 (31/01/17 at 16:29:37))
Total computing time for this run : 0h 0m 0s 94 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

