
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_48_07"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 378 (31/01/17 at 14:48:07)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_48_07_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_48_07_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 44
No. of loci              : 62
No. of usable loci       : 62 loci with less than  5.00 % missing data
No. of polymorphic sites : 62


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.02649
     3       150         2   0.01333
     4       150         2   0.06488
     5       150         2   0.01333
     6       150         2   0.19186
     7       150         2   0.01333
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.10166
    11       150         2   0.02649
    12       150         2   0.11356
    13       150         2   0.01333
    14       150         2   0.01333
    15       150         2   0.03946
    16       150         2   0.14819
    17       150         2   0.01333
    18       150         2   0.01333
    19       150         2   0.01333
    20       150         2   0.01333
    21       150         2   0.08957
    22       150         2   0.11356
    23       150         2   0.01333
    24       150         2   0.01333
    25       150         2   0.06488
    26       150         2   0.10166
    27       150         2   0.01333
    28       150         2   0.01333
    29       150         2   0.11356
    30       150         2   0.01333
    31       150         2   0.06488
    32       150         2   0.01333
    33       150         2   0.06488
    34       150         2   0.01333
    35       150         2   0.06488
    36       150         2   0.05226
    37       150         2   0.22273
    38       150         2   0.01333
    39       150         2   0.02649
    40       150         2   0.36018
    41       150         2   0.13682
    42       150         2   0.26139
    43       150         2   0.06488
    44       150         2   0.01333
    45       150         2   0.26139
    46       150         2   0.14819
    47       150         2   0.06488
    48       150         2   0.01333
    49       150         2   0.05226
    50       150         2   0.14819
    51       150         2   0.06488
    52       150         2   0.01333
    53       150         2   0.05226
    54       150         2   0.08957
    55       150         2   0.06488
    56       150         2   0.01333
    57       150         2   0.14819
    58       150         2   0.06488
    59       150         2   0.02649
    60       150         2   0.06488
    61       150         2   0.03946
    62       150         2   0.03946
-------------------------------------------------
  Mean   150.000     2.000   0.06718
  s.d.     0.000     0.000   0.07256
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0577
Gene diversity           :  0.9486 +/-  0.0083

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 44

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 62
Number of usable nucleotide sites            : 62


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.165011 +/-     2.082615

Average gene diversity over loci              :     0.067178 +/-     0.037182

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    16.860767
  S.D. Theta(Hom) :     3.096566

         Theta(k) :    20.605975
95 % confidence interval limits for theta(k) : [  14.167681,  29.640070 ]

         Theta(S) :    11.102130
    S.D. Theta(S) :     2.835556

        Theta(Pi) :     4.165011
   S.D. Theta(Pi) :     2.305254

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     574     194.886
       1     652     772.310
       2    1304    1542.893
       3    1719    2074.437
       4    1985    2114.907
       5    1894    1747.105
       6    1446    1220.794
       7     974     744.043
       8     443     404.976
       9     161     200.661
      10      23      91.994
      11       0      39.583
      12       0      16.188
      13       0       6.364
      14       0       2.428
      15       0       0.907
      16       0       0.334
      17       0       0.122
      18       0       0.044
      19       0       0.016
      20       0       0.006


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.165
Mismatch observed variance          :        4.481
Tau                                 :        4.678
Theta0                              :        0.000
Theta1                              :       31.484

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00070054
          P(Sim. Ssd >= Obs. Ssd): 0.80000000
    Harpending's Raggedness index: 0.01190734
        P(Sim. Rag. >= Obs. Rag.): 0.80000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        574.0        344.7        430.8
          1.0        652.0        656.9        736.9
          2.0       1304.0       1348.1       1257.9
          3.0       1719.0       1942.6       1750.4
          4.0       1985.0       2194.6       1947.5
          5.0       1894.0       1918.4       1775.9
          6.0       1446.0       1396.3       1364.1
          7.0        974.0        779.8        903.0
          8.0        443.0        383.9        524.7
          9.0        161.0        142.4        271.4
         10.0         23.0         48.6        126.5
         11.0          0.0         14.5         53.7
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.678           3.164           5.072           4.185
                   Theta0:           0.000           0.000           1.225           0.192               
                   Theta1:          31.484          36.875       99999.000       55139.078
Mean No. of pairwise diff.           4.165           3.229           4.709           4.155
           No. pol. sites:              62              62              84          72.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        574.0        200.0        452.0        430.8
          1.0        652.0        245.0       1282.0        736.9
          2.0       1304.0        764.0       2679.0       1257.9
          3.0       1719.0       1283.0       2622.0       1750.4
          4.0       1985.0       1761.0       2577.0       1947.5
          5.0       1894.0       1381.0       2670.0       1775.9
          6.0       1446.0        781.0       1857.0       1364.1
          7.0        974.0        315.0       1247.0        903.0
          8.0        443.0        110.0        762.0        524.7
          9.0        161.0         16.0        369.0        271.4
         10.0         23.0          0.0        162.0        126.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.678           3.164           5.072           4.185
                   Theta0:           0.000           0.000           1.225           0.192               
                   Theta1:          31.484          36.875       99999.000       55139.078
Mean No. of pairwise diff.           4.165           3.229           4.709           4.155
           No. pol. sites:              62              62              84          72.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        574.0        200.0        452.0        430.8
          1.0        652.0        245.0       1282.0        736.9
          2.0       1304.0        764.0       2679.0       1257.9
          3.0       1719.0       1283.0       2622.0       1750.4
          4.0       1985.0       1761.0       2577.0       1947.5
          5.0       1894.0       1381.0       2670.0       1775.9
          6.0       1446.0        781.0       1857.0       1364.1
          7.0        974.0        315.0       1247.0        903.0
          8.0        443.0        110.0        762.0        524.7
          9.0        161.0         16.0        369.0        271.4
         10.0         23.0          0.0        162.0        126.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.678           3.164           5.072           4.185
                   Theta0:           0.000           0.000           1.225           0.192               
                   Theta1:          31.484          36.875       99999.000       55139.078
Mean No. of pairwise diff.           4.165           3.229           4.709           4.155
           No. pol. sites:              62              62              84          72.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        574.0        200.0        452.0        430.8
          1.0        652.0        245.0       1282.0        736.9
          2.0       1304.0        764.0       2679.0       1257.9
          3.0       1719.0       1283.0       2622.0       1750.4
          4.0       1985.0       1761.0       2577.0       1947.5
          5.0       1894.0       1381.0       2670.0       1775.9
          6.0       1446.0        781.0       1857.0       1364.1
          7.0        974.0        315.0       1247.0        903.0
          8.0        443.0        110.0        762.0        524.7
          9.0        161.0         16.0        369.0        271.4
         10.0         23.0          0.0        162.0        126.5
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_48_07_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            62      62.000       0.000
  No. of usable loci            62      62.000       0.000
  No. of polym. loci            62      62.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_48_07_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.02649     0.02649     0.00000     0.02649
     3       0.01333     0.01333     0.00000     0.01333
     4       0.06488     0.06488     0.00000     0.06488
     5       0.01333     0.01333     0.00000     0.01333
     6       0.19186     0.19186     0.00000     0.19186
     7       0.01333     0.01333     0.00000     0.01333
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.10166     0.10166     0.00000     0.10166
    11       0.02649     0.02649     0.00000     0.02649
    12       0.11356     0.11356     0.00000     0.11356
    13       0.01333     0.01333     0.00000     0.01333
    14       0.01333     0.01333     0.00000     0.01333
    15       0.03946     0.03946     0.00000     0.03946
    16       0.14819     0.14819     0.00000     0.14819
    17       0.01333     0.01333     0.00000     0.01333
    18       0.01333     0.01333     0.00000     0.01333
    19       0.01333     0.01333     0.00000     0.01333
    20       0.01333     0.01333     0.00000     0.01333
    21       0.08957     0.08957     0.00000     0.08957
    22       0.11356     0.11356     0.00000     0.11356
    23       0.01333     0.01333     0.00000     0.01333
    24       0.01333     0.01333     0.00000     0.01333
    25       0.06488     0.06488     0.00000     0.06488
    26       0.10166     0.10166     0.00000     0.10166
    27       0.01333     0.01333     0.00000     0.01333
    28       0.01333     0.01333     0.00000     0.01333
    29       0.11356     0.11356     0.00000     0.11356
    30       0.01333     0.01333     0.00000     0.01333
    31       0.06488     0.06488     0.00000     0.06488
    32       0.01333     0.01333     0.00000     0.01333
    33       0.06488     0.06488     0.00000     0.06488
    34       0.01333     0.01333     0.00000     0.01333
    35       0.06488     0.06488     0.00000     0.06488
    36       0.05226     0.05226     0.00000     0.05226
    37       0.22273     0.22273     0.00000     0.22273
    38       0.01333     0.01333     0.00000     0.01333
    39       0.02649     0.02649     0.00000     0.02649
    40       0.36018     0.36018     0.00000     0.36018
    41       0.13682     0.13682     0.00000     0.13682
    42       0.26139     0.26139     0.00000     0.26139
    43       0.06488     0.06488     0.00000     0.06488
    44       0.01333     0.01333     0.00000     0.01333
    45       0.26139     0.26139     0.00000     0.26139
    46       0.14819     0.14819     0.00000     0.14819
    47       0.06488     0.06488     0.00000     0.06488
    48       0.01333     0.01333     0.00000     0.01333
    49       0.05226     0.05226     0.00000     0.05226
    50       0.14819     0.14819     0.00000     0.14819
    51       0.06488     0.06488     0.00000     0.06488
    52       0.01333     0.01333     0.00000     0.01333
    53       0.05226     0.05226     0.00000     0.05226
    54       0.08957     0.08957     0.00000     0.08957
    55       0.06488     0.06488     0.00000     0.06488
    56       0.01333     0.01333     0.00000     0.01333
    57       0.14819     0.14819     0.00000     0.14819
    58       0.06488     0.06488     0.00000     0.06488
    59       0.02649     0.02649     0.00000     0.02649
    60       0.06488     0.06488     0.00000     0.06488
    61       0.03946     0.03946     0.00000     0.03946
    62       0.03946     0.03946     0.00000     0.03946
--------------------------------------------------------
  Mean       0.06718     0.06718     0.00000     0.06718
  s.d.       0.07256     0.07256     0.00000     0.07256
--------------------------------------------------------
<A NAME="31_01_17at14_48_07_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_48_07_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      20.60597    20.60597     0.00000
            Theta_k_lower      14.16768    14.16768     0.00000
            Theta_k_upper      29.64007    29.64007     0.00000
                  Theta_H      16.86077    16.86077     0.00000
             s.d. Theta_H       3.09657     3.09657     0.00000
                  Theta_S      11.10213    11.10213     0.00000
             s.d. Theta_S       2.83556     2.83556     0.00000
                 Theta_pi       4.16501     4.16501     0.00000
            s.d. Theta_pi       2.30525     2.30525     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_48_07_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.67773     4.67773     0.00000
              Tau qt 2.5%       3.16406     3.16406     0.00000
                Tau qt 5%       3.16406     3.16406     0.00000
               Tau qt 95%       4.88086     4.88086     0.00000
             Tau qt 97.5%       4.88086     4.88086     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.67324     0.67324     0.00000
          Theta0 qt 97.5%       0.67324     0.67324     0.00000
                   Theta1      31.48438    31.48438     0.00000
           Theta1 qt 2.5%      36.87500    36.87500     0.00000
             Theta1 qt 5%      36.87500    36.87500     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00070     0.00070     0.00000
      Model (SSD) p-value       0.80000     0.80000     0.00000
         Raggedness index       0.01191     0.01191     0.00000
       Raggedness p-value       0.80000     0.80000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 378 (31/01/17 at 14:48:07))
Total computing time for this run : 0h 0m 0s 164 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

