
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_46_23"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 19 (31/01/17 at 16:46:23)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_46_23_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_46_23_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 66
No. of loci              : 25
No. of usable loci       : 25 loci with less than  5.00 % missing data
No. of polymorphic sites : 25


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.10166
     3       150         2   0.43320
     4       150         2   0.39991
     5       150         2   0.06488
     6       150         2   0.23266
     7       150         2   0.47436
     8       150         2   0.33790
     9       150         2   0.12528
    10       150         2   0.01333
    11       150         2   0.19186
    12       150         2   0.47105
    13       150         2   0.12528
    14       150         2   0.06488
    15       150         2   0.01333
    16       150         2   0.21262
    17       150         2   0.49047
    18       150         2   0.13682
    19       150         2   0.12528
    20       150         2   0.50013
    21       150         2   0.12528
    22       150         2   0.26139
    23       150         2   0.45602
    24       150         2   0.13682
    25       150         2   0.08957
-------------------------------------------------
  Mean   150.000     2.000   0.22389
  s.d.     0.000     0.000   0.16908
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0275
Gene diversity           :  0.9791 +/-  0.0037

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 66

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 25
Number of usable nucleotide sites            : 25


Distance method : Pairwise difference
Mean number of pairwise differences           :     5.597315 +/-     2.702334

Average gene diversity over loci              :     0.223893 +/-     0.119649

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    44.921571
  S.D. Theta(Hom) :     8.376175

         Theta(k) :    44.468172
95 % confidence interval limits for theta(k) : [  31.760638,  62.023917 ]

         Theta(S) :     4.476665
    S.D. Theta(S) :     1.327330

        Theta(Pi) :     5.597315
   S.D. Theta(Pi) :     2.991222

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     234      45.289
       1     321     246.373
       2     551     672.534
       3    1009    1228.756
       4    1447    1691.193
       5    1867    1871.316
       6    1795    1735.053
       7    1591    1387.461
       8    1122     977.617
       9     644     617.155
      10     362     353.796
      11     149     186.269
      12      52      90.944
      13      19      41.532
      14      12      17.878
      15       0       7.307
      16       0       2.854
      17       0       1.072
      18       0       0.390
      19       0       0.138
      20       0       0.048
      21       0       0.016
      22       0       0.005
      23       0       0.002
      24       0       0.001
      25       0       0.000
      26       0       0.000
      27       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        5.597
Mismatch observed variance          :        5.831
Tau                                 :        5.949
Theta0                              :        0.002
Theta1                              :      102.148

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00057394
          P(Sim. Ssd >= Obs. Ssd): 0.70000000
    Harpending's Raggedness index: 0.01016415
        P(Sim. Rag. >= Obs. Rag.): 0.60000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        234.0        150.0        135.5
          1.0        321.0        322.3        268.8
          2.0        551.0        674.1        585.7
          3.0       1009.0       1150.5       1053.4
          4.0       1447.0       1673.5       1508.8
          5.0       1867.0       1856.6       1764.3
          6.0       1795.0       1766.8       1733.4
          7.0       1591.0       1408.8       1465.4
          8.0       1122.0        935.5       1086.1
          9.0        644.0        576.1        716.3
         10.0        362.0        343.3        425.5
         11.0        149.0        174.0        229.9
         12.0         52.0         87.9        113.9
         13.0         19.0         38.3         52.1
         14.0         12.0          8.8         22.1
         15.0          0.0          6.7          8.8
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           5.949           4.605           6.363           5.292
                   Theta0:           0.002           0.000           1.867           0.426               
                   Theta1:         102.148          32.217       99999.000       60028.688
Mean No. of pairwise diff.           5.597           4.750           6.180           5.493
           No. pol. sites:              25             120             149         134.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        234.0         77.0        338.0        135.5
          1.0        321.0        172.0        562.0        268.8
          2.0        551.0        288.0        959.0        585.7
          3.0       1009.0        634.0       1693.0       1053.4
          4.0       1447.0       1144.0       2240.0       1508.8
          5.0       1867.0       1516.0       2215.0       1764.3
          6.0       1795.0       1499.0       2251.0       1733.4
          7.0       1591.0        850.0       2195.0       1465.4
          8.0       1122.0        409.0       1339.0       1086.1
          9.0        644.0        302.0        830.0        716.3
         10.0        362.0        177.0        662.0        425.5
         11.0        149.0         64.0        457.0        229.9
         12.0         52.0         17.0        242.0        113.9
         13.0         19.0          8.0        113.0         52.1
         14.0         12.0          0.0         43.0         22.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           5.949           4.605           6.363           5.292
                   Theta0:           0.002           0.000           1.867           0.426               
                   Theta1:         102.148          32.217       99999.000       60028.688
Mean No. of pairwise diff.           5.597           4.750           6.180           5.493
           No. pol. sites:              25             120             149         134.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        234.0         77.0        338.0        135.5
          1.0        321.0        172.0        562.0        268.8
          2.0        551.0        288.0        959.0        585.7
          3.0       1009.0        634.0       1693.0       1053.4
          4.0       1447.0       1144.0       2240.0       1508.8
          5.0       1867.0       1516.0       2215.0       1764.3
          6.0       1795.0       1499.0       2251.0       1733.4
          7.0       1591.0        850.0       2195.0       1465.4
          8.0       1122.0        409.0       1339.0       1086.1
          9.0        644.0        302.0        830.0        716.3
         10.0        362.0        177.0        662.0        425.5
         11.0        149.0         64.0        457.0        229.9
         12.0         52.0         17.0        242.0        113.9
         13.0         19.0          8.0        113.0         52.1
         14.0         12.0          0.0         43.0         22.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           5.949           4.605           6.363           5.292
                   Theta0:           0.002           0.000           1.867           0.426               
                   Theta1:         102.148          32.217       99999.000       60028.688
Mean No. of pairwise diff.           5.597           4.750           6.180           5.493
           No. pol. sites:              25             120             149         134.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        234.0         77.0        338.0        135.5
          1.0        321.0        172.0        562.0        268.8
          2.0        551.0        288.0        959.0        585.7
          3.0       1009.0        634.0       1693.0       1053.4
          4.0       1447.0       1144.0       2240.0       1508.8
          5.0       1867.0       1516.0       2215.0       1764.3
          6.0       1795.0       1499.0       2251.0       1733.4
          7.0       1591.0        850.0       2195.0       1465.4
          8.0       1122.0        409.0       1339.0       1086.1
          9.0        644.0        302.0        830.0        716.3
         10.0        362.0        177.0        662.0        425.5
         11.0        149.0         64.0        457.0        229.9
         12.0         52.0         17.0        242.0        113.9
         13.0         19.0          8.0        113.0         52.1
         14.0         12.0          0.0         43.0         22.1
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_46_23_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            25      25.000       0.000
  No. of usable loci            25      25.000       0.000
  No. of polym. loci            25      25.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_46_23_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.10166     0.10166     0.00000     0.10166
     3       0.43320     0.43320     0.00000     0.43320
     4       0.39991     0.39991     0.00000     0.39991
     5       0.06488     0.06488     0.00000     0.06488
     6       0.23266     0.23266     0.00000     0.23266
     7       0.47436     0.47436     0.00000     0.47436
     8       0.33790     0.33790     0.00000     0.33790
     9       0.12528     0.12528     0.00000     0.12528
    10       0.01333     0.01333     0.00000     0.01333
    11       0.19186     0.19186     0.00000     0.19186
    12       0.47105     0.47105     0.00000     0.47105
    13       0.12528     0.12528     0.00000     0.12528
    14       0.06488     0.06488     0.00000     0.06488
    15       0.01333     0.01333     0.00000     0.01333
    16       0.21262     0.21262     0.00000     0.21262
    17       0.49047     0.49047     0.00000     0.49047
    18       0.13682     0.13682     0.00000     0.13682
    19       0.12528     0.12528     0.00000     0.12528
    20       0.50013     0.50013     0.00000     0.50013
    21       0.12528     0.12528     0.00000     0.12528
    22       0.26139     0.26139     0.00000     0.26139
    23       0.45602     0.45602     0.00000     0.45602
    24       0.13682     0.13682     0.00000     0.13682
    25       0.08957     0.08957     0.00000     0.08957
--------------------------------------------------------
  Mean       0.22389     0.22389     0.00000     0.22389
  s.d.       0.16908     0.16908     0.00000     0.16908
--------------------------------------------------------
<A NAME="31_01_17at16_46_23_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_46_23_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      44.46817    44.46817     0.00000
            Theta_k_lower      31.76064    31.76064     0.00000
            Theta_k_upper      62.02392    62.02392     0.00000
                  Theta_H      44.92157    44.92157     0.00000
             s.d. Theta_H       8.37617     8.37617     0.00000
                  Theta_S       4.47667     4.47667     0.00000
             s.d. Theta_S       1.32733     1.32733     0.00000
                 Theta_pi       5.59732     5.59732     0.00000
            s.d. Theta_pi       2.99122     2.99122     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_46_23_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       5.94922     5.94922     0.00000
              Tau qt 2.5%       4.60547     4.60547     0.00000
                Tau qt 5%       4.60547     4.60547     0.00000
               Tau qt 95%       5.91797     5.91797     0.00000
             Tau qt 97.5%       5.91797     5.91797     0.00000
                   Theta0       0.00176     0.00176     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.12324     1.12324     0.00000
          Theta0 qt 97.5%       1.12324     1.12324     0.00000
                   Theta1     102.14844   102.14844     0.00000
           Theta1 qt 2.5%      32.21680    32.21680     0.00000
             Theta1 qt 5%      32.21680    32.21680     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00057     0.00057     0.00000
      Model (SSD) p-value       0.70000     0.70000     0.00000
         Raggedness index       0.01016     0.01016     0.00000
       Raggedness p-value       0.60000     0.60000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 19 (31/01/17 at 16:46:23))
Total computing time for this run : 0h 0m 0s 190 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

