
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_40"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 388 (31/01/17 at 14:44:40)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_40_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_40_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 52
No. of loci              : 73
No. of usable loci       : 73 loci with less than  5.00 % missing data
No. of polymorphic sites : 73


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.02649
     2       150         2   0.02649
     3       150         2   0.12528
     4       150         2   0.02649
     5       150         2   0.02649
     6       150         2   0.01333
     7       150         2   0.01333
     8       150         2   0.03946
     9       150         2   0.12528
    10       150         2   0.01333
    11       150         2   0.02649
    12       150         2   0.01333
    13       150         2   0.01333
    14       150         2   0.01333
    15       150         2   0.08957
    16       150         2   0.01333
    17       150         2   0.03946
    18       150         2   0.08957
    19       150         2   0.02649
    20       150         2   0.03946
    21       150         2   0.11356
    22       150         2   0.01333
    23       150         2   0.01333
    24       150         2   0.02649
    25       150         2   0.02649
    26       150         2   0.12528
    27       150         2   0.02649
    28       150         2   0.02649
    29       150         2   0.01333
    30       150         2   0.02649
    31       150         2   0.15937
    32       150         2   0.01333
    33       150         2   0.06488
    34       150         2   0.01333
    35       150         2   0.02649
    36       150         2   0.01333
    37       150         2   0.05226
    38       150         2   0.08957
    39       150         2   0.02649
    40       150         2   0.35293
    41       150         2   0.08957
    42       150         2   0.01333
    43       150         2   0.02649
    44       150         2   0.05226
    45       150         2   0.01333
    46       150         2   0.02649
    47       150         2   0.03946
    48       150         2   0.03946
    49       150         2   0.01333
    50       150         2   0.03946
    51       150         2   0.03946
    52       150         2   0.01333
    53       150         2   0.01333
    54       150         2   0.03946
    55       150         2   0.11356
    56       150         2   0.01333
    57       150         2   0.01333
    58       150         2   0.01333
    59       150         2   0.01333
    60       150         2   0.08957
    61       150         2   0.08957
    62       150         2   0.01333
    63       150         2   0.03946
    64       150         2   0.08957
    65       150         2   0.02649
    66       150         2   0.03946
    67       150         2   0.13682
    68       150         2   0.02649
    69       150         2   0.02649
    70       150         2   0.01333
    71       150         2   0.02649
    72       150         2   0.18121
    73       150         2   0.03946
-------------------------------------------------
  Mean   150.000     2.000   0.04777
  s.d.     0.000     0.000   0.05380
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0443
Gene diversity           :  0.9621 +/-  0.0066

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 52

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 73
Number of usable nucleotide sites            : 73


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.487069 +/-     1.787938

Average gene diversity over loci              :     0.047768 +/-     0.027111

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    23.712105
  S.D. Theta(Hom) :     4.535333

         Theta(k) :    27.789032
95 % confidence interval limits for theta(k) : [  19.465082,  39.354984 ]

         Theta(S) :    13.071863
    S.D. Theta(S) :     3.279015

        Theta(Pi) :     3.487069
   S.D. Theta(Pi) :     1.979075

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     423       0.000
       1    1173       0.000
       2    1940       0.000
       3    2419       0.000
       4    2085       0.000
       5    1489       0.000
       6     911       0.000
       7     508       0.000
       8     192       0.000
       9      35       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.487
Mismatch observed variance          :        3.420
Tau                                 :        3.502
Theta0                              :        0.098
Theta1                              :      100.312

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00010993
          P(Sim. Ssd >= Obs. Ssd): 1.00000000
    Harpending's Raggedness index: 0.01977268
        P(Sim. Rag. >= Obs. Rag.): 0.80000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        423.0        415.0        403.2
          1.0       1173.0       1269.9       1158.4
          2.0       1940.0       2065.4       1985.2
          3.0       2419.0       2387.5       2341.2
          4.0       2085.0       2098.6       2090.6
          5.0       1489.0       1432.3       1499.6
          6.0        911.0        806.6        898.5
          7.0        508.0        405.0        462.2
          8.0        192.0        190.3        208.3
          9.0         35.0         69.2         83.5
         10.0          0.0         24.2         30.2
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.502           2.299           3.928           3.382
                   Theta0:           0.098           0.000           0.902           0.101               
                   Theta1:         100.312          39.238       99999.000       60238.113
Mean No. of pairwise diff.           3.487           2.678           3.934           3.428
           No. pol. sites:              73              96             125         108.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        423.0        149.0        726.0        403.2
          1.0       1173.0        686.0       2095.0       1158.4
          2.0       1940.0       1679.0       2861.0       1985.2
          3.0       2419.0       2083.0       2590.0       2341.2
          4.0       2085.0       1687.0       2444.0       2090.6
          5.0       1489.0        964.0       1689.0       1499.6
          6.0        911.0        361.0       1073.0        898.5
          7.0        508.0        122.0        567.0        462.2
          8.0        192.0         25.0        388.0        208.3
          9.0         35.0          0.0        203.0         83.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.502           2.299           3.928           3.382
                   Theta0:           0.098           0.000           0.902           0.101               
                   Theta1:         100.312          39.238       99999.000       60238.113
Mean No. of pairwise diff.           3.487           2.678           3.934           3.428
           No. pol. sites:              73              96             125         108.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        423.0        149.0        726.0        403.2
          1.0       1173.0        686.0       2095.0       1158.4
          2.0       1940.0       1679.0       2861.0       1985.2
          3.0       2419.0       2083.0       2590.0       2341.2
          4.0       2085.0       1687.0       2444.0       2090.6
          5.0       1489.0        964.0       1689.0       1499.6
          6.0        911.0        361.0       1073.0        898.5
          7.0        508.0        122.0        567.0        462.2
          8.0        192.0         25.0        388.0        208.3
          9.0         35.0          0.0        203.0         83.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.502           2.299           3.928           3.382
                   Theta0:           0.098           0.000           0.902           0.101               
                   Theta1:         100.312          39.238       99999.000       60238.113
Mean No. of pairwise diff.           3.487           2.678           3.934           3.428
           No. pol. sites:              73              96             125         108.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        423.0        149.0        726.0        403.2
          1.0       1173.0        686.0       2095.0       1158.4
          2.0       1940.0       1679.0       2861.0       1985.2
          3.0       2419.0       2083.0       2590.0       2341.2
          4.0       2085.0       1687.0       2444.0       2090.6
          5.0       1489.0        964.0       1689.0       1499.6
          6.0        911.0        361.0       1073.0        898.5
          7.0        508.0        122.0        567.0        462.2
          8.0        192.0         25.0        388.0        208.3
          9.0         35.0          0.0        203.0         83.5
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_40_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            73      73.000       0.000
  No. of usable loci            73      73.000       0.000
  No. of polym. loci            73      73.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_40_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.02649     0.02649     0.00000     0.02649
     2       0.02649     0.02649     0.00000     0.02649
     3       0.12528     0.12528     0.00000     0.12528
     4       0.02649     0.02649     0.00000     0.02649
     5       0.02649     0.02649     0.00000     0.02649
     6       0.01333     0.01333     0.00000     0.01333
     7       0.01333     0.01333     0.00000     0.01333
     8       0.03946     0.03946     0.00000     0.03946
     9       0.12528     0.12528     0.00000     0.12528
    10       0.01333     0.01333     0.00000     0.01333
    11       0.02649     0.02649     0.00000     0.02649
    12       0.01333     0.01333     0.00000     0.01333
    13       0.01333     0.01333     0.00000     0.01333
    14       0.01333     0.01333     0.00000     0.01333
    15       0.08957     0.08957     0.00000     0.08957
    16       0.01333     0.01333     0.00000     0.01333
    17       0.03946     0.03946     0.00000     0.03946
    18       0.08957     0.08957     0.00000     0.08957
    19       0.02649     0.02649     0.00000     0.02649
    20       0.03946     0.03946     0.00000     0.03946
    21       0.11356     0.11356     0.00000     0.11356
    22       0.01333     0.01333     0.00000     0.01333
    23       0.01333     0.01333     0.00000     0.01333
    24       0.02649     0.02649     0.00000     0.02649
    25       0.02649     0.02649     0.00000     0.02649
    26       0.12528     0.12528     0.00000     0.12528
    27       0.02649     0.02649     0.00000     0.02649
    28       0.02649     0.02649     0.00000     0.02649
    29       0.01333     0.01333     0.00000     0.01333
    30       0.02649     0.02649     0.00000     0.02649
    31       0.15937     0.15937     0.00000     0.15937
    32       0.01333     0.01333     0.00000     0.01333
    33       0.06488     0.06488     0.00000     0.06488
    34       0.01333     0.01333     0.00000     0.01333
    35       0.02649     0.02649     0.00000     0.02649
    36       0.01333     0.01333     0.00000     0.01333
    37       0.05226     0.05226     0.00000     0.05226
    38       0.08957     0.08957     0.00000     0.08957
    39       0.02649     0.02649     0.00000     0.02649
    40       0.35293     0.35293     0.00000     0.35293
    41       0.08957     0.08957     0.00000     0.08957
    42       0.01333     0.01333     0.00000     0.01333
    43       0.02649     0.02649     0.00000     0.02649
    44       0.05226     0.05226     0.00000     0.05226
    45       0.01333     0.01333     0.00000     0.01333
    46       0.02649     0.02649     0.00000     0.02649
    47       0.03946     0.03946     0.00000     0.03946
    48       0.03946     0.03946     0.00000     0.03946
    49       0.01333     0.01333     0.00000     0.01333
    50       0.03946     0.03946     0.00000     0.03946
    51       0.03946     0.03946     0.00000     0.03946
    52       0.01333     0.01333     0.00000     0.01333
    53       0.01333     0.01333     0.00000     0.01333
    54       0.03946     0.03946     0.00000     0.03946
    55       0.11356     0.11356     0.00000     0.11356
    56       0.01333     0.01333     0.00000     0.01333
    57       0.01333     0.01333     0.00000     0.01333
    58       0.01333     0.01333     0.00000     0.01333
    59       0.01333     0.01333     0.00000     0.01333
    60       0.08957     0.08957     0.00000     0.08957
    61       0.08957     0.08957     0.00000     0.08957
    62       0.01333     0.01333     0.00000     0.01333
    63       0.03946     0.03946     0.00000     0.03946
    64       0.08957     0.08957     0.00000     0.08957
    65       0.02649     0.02649     0.00000     0.02649
    66       0.03946     0.03946     0.00000     0.03946
    67       0.13682     0.13682     0.00000     0.13682
    68       0.02649     0.02649     0.00000     0.02649
    69       0.02649     0.02649     0.00000     0.02649
    70       0.01333     0.01333     0.00000     0.01333
    71       0.02649     0.02649     0.00000     0.02649
    72       0.18121     0.18121     0.00000     0.18121
    73       0.03946     0.03946     0.00000     0.03946
--------------------------------------------------------
  Mean       0.04777     0.04777     0.00000     0.04777
  s.d.       0.05380     0.05380     0.00000     0.05380
--------------------------------------------------------
<A NAME="31_01_17at14_44_40_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
    72             2       2.000       0.000           2
    73             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_40_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      27.78903    27.78903     0.00000
            Theta_k_lower      19.46508    19.46508     0.00000
            Theta_k_upper      39.35498    39.35498     0.00000
                  Theta_H      23.71211    23.71211     0.00000
             s.d. Theta_H       4.53533     4.53533     0.00000
                  Theta_S      13.07186    13.07186     0.00000
             s.d. Theta_S       3.27901     3.27901     0.00000
                 Theta_pi       3.48707     3.48707     0.00000
            s.d. Theta_pi       1.97907     1.97907     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_40_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.50195     3.50195     0.00000
              Tau qt 2.5%       2.29883     2.29883     0.00000
                Tau qt 5%       2.29883     2.29883     0.00000
               Tau qt 95%       3.82812     3.82812     0.00000
             Tau qt 97.5%       3.82812     3.82812     0.00000
                   Theta0       0.09824     0.09824     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.05977     0.05977     0.00000
          Theta0 qt 97.5%       0.05977     0.05977     0.00000
                   Theta1     100.31250   100.31250     0.00000
           Theta1 qt 2.5%      39.23828    39.23828     0.00000
             Theta1 qt 5%      39.23828    39.23828     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00011     0.00011     0.00000
      Model (SSD) p-value       1.00000     1.00000     0.00000
         Raggedness index       0.01977     0.01977     0.00000
       Raggedness p-value       0.80000     0.80000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 388 (31/01/17 at 14:44:40))
Total computing time for this run : 0h 0m 0s 136 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

