
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_30_49"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 196 (31/01/17 at 16:30:49)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_30_49_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_30_49_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 50
No. of loci              : 73
No. of usable loci       : 73 loci with less than  5.00 % missing data
No. of polymorphic sites : 73


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.01333
     3       150         2   0.01333
     4       150         2   0.01333
     5       150         2   0.01333
     6       150         2   0.01333
     7       150         2   0.10166
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.07732
    11       150         2   0.03946
    12       150         2   0.01333
    13       150         2   0.03946
    14       150         2   0.01333
    15       150         2   0.02649
    16       150         2   0.01333
    17       150         2   0.01333
    18       150         2   0.01333
    19       150         2   0.06488
    20       150         2   0.01333
    21       150         2   0.02649
    22       150         2   0.02649
    23       150         2   0.01333
    24       150         2   0.06488
    25       150         2   0.01333
    26       150         2   0.05226
    27       150         2   0.02649
    28       150         2   0.02649
    29       150         2   0.02649
    30       150         2   0.01333
    31       150         2   0.01333
    32       150         2   0.01333
    33       150         2   0.03946
    34       150         2   0.17038
    35       150         2   0.01333
    36       150         2   0.01333
    37       150         2   0.01333
    38       150         2   0.01333
    39       150         2   0.03946
    40       150         2   0.06488
    41       150         2   0.01333
    42       150         2   0.05226
    43       150         2   0.03946
    44       150         2   0.02649
    45       150         2   0.01333
    46       150         2   0.01333
    47       150         2   0.01333
    48       150         2   0.05226
    49       150         2   0.03946
    50       150         2   0.01333
    51       150         2   0.01333
    52       150         2   0.07732
    53       150         2   0.01333
    54       150         2   0.05226
    55       150         2   0.01333
    56       150         2   0.02649
    57       150         2   0.07732
    58       150         2   0.01333
    59       150         2   0.07732
    60       150         2   0.01333
    61       150         2   0.01333
    62       150         2   0.03946
    63       150         2   0.02649
    64       150         2   0.07732
    65       150         2   0.02649
    66       150         2   0.01333
    67       150         2   0.10166
    68       150         2   0.03946
    69       150         2   0.02649
    70       150         2   0.02649
    71       150         2   0.03946
    72       150         2   0.08957
    73       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.03297
  s.d.     0.000     0.000   0.02903
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.1374
Gene diversity           :  0.8684 +/-  0.0252

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 50

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 73
Number of usable nucleotide sites            : 73


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.406532 +/-     1.314585

Average gene diversity over loci              :     0.032966 +/-     0.019933

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     5.477451
  S.D. Theta(Hom) :     1.327240

         Theta(k) :    25.857163
95 % confidence interval limits for theta(k) : [  18.038924,  36.741475 ]

         Theta(S) :    13.071863
    S.D. Theta(S) :     3.279015

        Theta(Pi) :     2.406532
   S.D. Theta(Pi) :     1.455119

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1471    1231.147
       1    2492    2546.969
       2    2361    2751.751
       3    1982    2093.310
       4    1441    1277.669
       5     856     676.423
       6     377     327.504
       7     142     150.446
       8      44      67.184
       9       9      29.594
      10       0      12.958
      11       0       5.661
      12       0       2.471
      13       0       1.078
      14       0       0.470
      15       0       0.205
      16       0       0.090
      17       0       0.039
      18       0       0.017


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.407
Mismatch observed variance          :        3.006
Tau                                 :        1.795
Theta0                              :        0.923
Theta1                              :       13.691

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00062356
          P(Sim. Ssd >= Obs. Ssd): 0.90000000
    Harpending's Raggedness index: 0.01708612
        P(Sim. Rag. >= Obs. Rag.): 1.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1471.0       1685.0       1496.7
          1.0       2492.0       2389.7       2333.2
          2.0       2361.0       2480.7       2489.3
          3.0       1982.0       1930.3       1994.7
          4.0       1441.0       1174.1       1311.8
          5.0        856.0        739.9        755.8
          6.0        377.0        472.2        400.3
          7.0        142.0        200.3        201.7
          8.0         44.0         73.3         99.0
          9.0          9.0         25.1         47.9
         10.0          0.0          4.1         23.1
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.795           0.430           4.404           2.322
                   Theta0:           0.923           0.000           3.444           0.375               
                   Theta1:          13.691           1.946       99999.000       40038.840
Mean No. of pairwise diff.           2.407           0.928           3.780           2.383
           No. pol. sites:              73              18              41          31.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1471.0        546.0       4817.0       1496.7
          1.0       2492.0        542.0       4337.0       2333.2
          2.0       2361.0       1467.0       3621.0       2489.3
          3.0       1982.0        710.0       2975.0       1994.7
          4.0       1441.0        162.0       2705.0       1311.8
          5.0        856.0         22.0       2317.0        755.8
          6.0        377.0          0.0       1491.0        400.3
          7.0        142.0          0.0       1077.0        201.7
          8.0         44.0          0.0        520.0         99.0
          9.0          9.0          0.0        211.0         47.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.795           0.430           4.404           2.322
                   Theta0:           0.923           0.000           3.444           0.375               
                   Theta1:          13.691           1.946       99999.000       40038.840
Mean No. of pairwise diff.           2.407           0.928           3.780           2.383
           No. pol. sites:              73              18              41          31.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1471.0        546.0       4817.0       1496.7
          1.0       2492.0        542.0       4337.0       2333.2
          2.0       2361.0       1467.0       3621.0       2489.3
          3.0       1982.0        710.0       2975.0       1994.7
          4.0       1441.0        162.0       2705.0       1311.8
          5.0        856.0         22.0       2317.0        755.8
          6.0        377.0          0.0       1491.0        400.3
          7.0        142.0          0.0       1077.0        201.7
          8.0         44.0          0.0        520.0         99.0
          9.0          9.0          0.0        211.0         47.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           1.795           0.430           4.404           2.322
                   Theta0:           0.923           0.000           3.444           0.375               
                   Theta1:          13.691           1.946       99999.000       40038.840
Mean No. of pairwise diff.           2.407           0.928           3.780           2.383
           No. pol. sites:              73              18              41          31.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1471.0        546.0       4817.0       1496.7
          1.0       2492.0        542.0       4337.0       2333.2
          2.0       2361.0       1467.0       3621.0       2489.3
          3.0       1982.0        710.0       2975.0       1994.7
          4.0       1441.0        162.0       2705.0       1311.8
          5.0        856.0         22.0       2317.0        755.8
          6.0        377.0          0.0       1491.0        400.3
          7.0        142.0          0.0       1077.0        201.7
          8.0         44.0          0.0        520.0         99.0
          9.0          9.0          0.0        211.0         47.9
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_30_49_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            73      73.000       0.000
  No. of usable loci            73      73.000       0.000
  No. of polym. loci            73      73.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_30_49_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.01333     0.01333     0.00000     0.01333
     3       0.01333     0.01333     0.00000     0.01333
     4       0.01333     0.01333     0.00000     0.01333
     5       0.01333     0.01333     0.00000     0.01333
     6       0.01333     0.01333     0.00000     0.01333
     7       0.10166     0.10166     0.00000     0.10166
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.07732     0.07732     0.00000     0.07732
    11       0.03946     0.03946     0.00000     0.03946
    12       0.01333     0.01333     0.00000     0.01333
    13       0.03946     0.03946     0.00000     0.03946
    14       0.01333     0.01333     0.00000     0.01333
    15       0.02649     0.02649     0.00000     0.02649
    16       0.01333     0.01333     0.00000     0.01333
    17       0.01333     0.01333     0.00000     0.01333
    18       0.01333     0.01333     0.00000     0.01333
    19       0.06488     0.06488     0.00000     0.06488
    20       0.01333     0.01333     0.00000     0.01333
    21       0.02649     0.02649     0.00000     0.02649
    22       0.02649     0.02649     0.00000     0.02649
    23       0.01333     0.01333     0.00000     0.01333
    24       0.06488     0.06488     0.00000     0.06488
    25       0.01333     0.01333     0.00000     0.01333
    26       0.05226     0.05226     0.00000     0.05226
    27       0.02649     0.02649     0.00000     0.02649
    28       0.02649     0.02649     0.00000     0.02649
    29       0.02649     0.02649     0.00000     0.02649
    30       0.01333     0.01333     0.00000     0.01333
    31       0.01333     0.01333     0.00000     0.01333
    32       0.01333     0.01333     0.00000     0.01333
    33       0.03946     0.03946     0.00000     0.03946
    34       0.17038     0.17038     0.00000     0.17038
    35       0.01333     0.01333     0.00000     0.01333
    36       0.01333     0.01333     0.00000     0.01333
    37       0.01333     0.01333     0.00000     0.01333
    38       0.01333     0.01333     0.00000     0.01333
    39       0.03946     0.03946     0.00000     0.03946
    40       0.06488     0.06488     0.00000     0.06488
    41       0.01333     0.01333     0.00000     0.01333
    42       0.05226     0.05226     0.00000     0.05226
    43       0.03946     0.03946     0.00000     0.03946
    44       0.02649     0.02649     0.00000     0.02649
    45       0.01333     0.01333     0.00000     0.01333
    46       0.01333     0.01333     0.00000     0.01333
    47       0.01333     0.01333     0.00000     0.01333
    48       0.05226     0.05226     0.00000     0.05226
    49       0.03946     0.03946     0.00000     0.03946
    50       0.01333     0.01333     0.00000     0.01333
    51       0.01333     0.01333     0.00000     0.01333
    52       0.07732     0.07732     0.00000     0.07732
    53       0.01333     0.01333     0.00000     0.01333
    54       0.05226     0.05226     0.00000     0.05226
    55       0.01333     0.01333     0.00000     0.01333
    56       0.02649     0.02649     0.00000     0.02649
    57       0.07732     0.07732     0.00000     0.07732
    58       0.01333     0.01333     0.00000     0.01333
    59       0.07732     0.07732     0.00000     0.07732
    60       0.01333     0.01333     0.00000     0.01333
    61       0.01333     0.01333     0.00000     0.01333
    62       0.03946     0.03946     0.00000     0.03946
    63       0.02649     0.02649     0.00000     0.02649
    64       0.07732     0.07732     0.00000     0.07732
    65       0.02649     0.02649     0.00000     0.02649
    66       0.01333     0.01333     0.00000     0.01333
    67       0.10166     0.10166     0.00000     0.10166
    68       0.03946     0.03946     0.00000     0.03946
    69       0.02649     0.02649     0.00000     0.02649
    70       0.02649     0.02649     0.00000     0.02649
    71       0.03946     0.03946     0.00000     0.03946
    72       0.08957     0.08957     0.00000     0.08957
    73       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.03297     0.03297     0.00000     0.03297
  s.d.       0.02903     0.02903     0.00000     0.02903
--------------------------------------------------------
<A NAME="31_01_17at16_30_49_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
    72             2       2.000       0.000           2
    73             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_30_49_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      25.85716    25.85716     0.00000
            Theta_k_lower      18.03892    18.03892     0.00000
            Theta_k_upper      36.74148    36.74148     0.00000
                  Theta_H       5.47745     5.47745     0.00000
             s.d. Theta_H       1.32724     1.32724     0.00000
                  Theta_S      13.07186    13.07186     0.00000
             s.d. Theta_S       3.27901     3.27901     0.00000
                 Theta_pi       2.40653     2.40653     0.00000
            s.d. Theta_pi       1.45512     1.45512     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_30_49_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       1.79492     1.79492     0.00000
              Tau qt 2.5%       0.42969     0.42969     0.00000
                Tau qt 5%       0.42969     0.42969     0.00000
               Tau qt 95%       3.83008     3.83008     0.00000
             Tau qt 97.5%       3.83008     3.83008     0.00000
                   Theta0       0.92285     0.92285     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.28125     0.28125     0.00000
          Theta0 qt 97.5%       0.28125     0.28125     0.00000
                   Theta1      13.69141    13.69141     0.00000
           Theta1 qt 2.5%       1.94580     1.94580     0.00000
             Theta1 qt 5%       1.94580     1.94580     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00062     0.00062     0.00000
      Model (SSD) p-value       0.90000     0.90000     0.00000
         Raggedness index       0.01709     0.01709     0.00000
       Raggedness p-value       1.00000     1.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 196 (31/01/17 at 16:30:49))
Total computing time for this run : 0h 0m 0s 123 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

