
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_32_00"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 163 (31/01/17 at 16:32:00)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_32_00_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_32_00_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 64
No. of loci              : 21
No. of usable loci       : 21 loci with less than  5.00 % missing data
No. of polymorphic sites : 21


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.03946
     2       150         2   0.30568
     3       150         2   0.15937
     4       150         2   0.02649
     5       150         2   0.01333
     6       150         2   0.24242
     7       150         2   0.27060
     8       150         2   0.03946
     9       150         2   0.27964
    10       150         2   0.12528
    11       150         2   0.01333
    12       150         2   0.13682
    13       150         2   0.32215
    14       150         2   0.01333
    15       150         2   0.13682
    16       150         2   0.18121
    17       150         2   0.02649
    18       150         2   0.01333
    19       150         2   0.20233
    20       150         2   0.36018
    21       150         2   0.10166
-------------------------------------------------
  Mean   150.000     2.000   0.14330
  s.d.     0.000     0.000   0.11699
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0459
Gene diversity           :  0.9605 +/-  0.0093

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 64

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 21
Number of usable nucleotide sites            : 21


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.009396 +/-     1.579419

Average gene diversity over loci              :     0.143305 +/-     0.083251

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    22.645750
  S.D. Theta(Hom) :     5.925492

         Theta(k) :    41.695151
95 % confidence interval limits for theta(k) : [  29.721650,  58.237899 ]

         Theta(S) :     3.760399
    S.D. Theta(S) :     1.159953

        Theta(Pi) :     3.009396
   S.D. Theta(Pi) :     1.748264

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     441       0.000
       1    1489       0.000
       2    2465       0.000
       3    2697       0.000
       4    2182       0.000
       5    1236       0.000
       6     481       0.000
       7     150       0.000
       8      30       0.000
       9       4       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.009
Mismatch observed variance          :        2.429
Tau                                 :        3.113
Theta0                              :        0.005
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00121548
          P(Sim. Ssd >= Obs. Ssd): 0.30000000
    Harpending's Raggedness index: 0.03170646
        P(Sim. Rag. >= Obs. Rag.): 0.30000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        441.0        588.8        494.3
          1.0       1489.0       1645.3       1541.1
          2.0       2465.0       2473.7       2403.0
          3.0       2697.0       2510.6       2497.9
          4.0       2182.0       1894.7       1947.5
          5.0       1236.0       1121.4       1214.7
          6.0        481.0        566.4        631.3
          7.0        150.0        240.7        281.3
          8.0         30.0         95.0        109.6
          9.0          4.0         30.2         38.0
         10.0          0.0          6.4         11.8
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.113           2.609           3.410           3.021
                   Theta0:           0.005           0.000           0.004           0.000               
                   Theta1:       99999.000          28.883       99999.000       60013.055
Mean No. of pairwise diff.           3.009           2.556           3.393           2.999
           No. pol. sites:              21             155             190         174.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        441.0        378.0        903.0        494.3
          1.0       1489.0       1248.0       2104.0       1541.1
          2.0       2465.0       1896.0       3076.0       2403.0
          3.0       2697.0       2239.0       2873.0       2497.9
          4.0       2182.0       1635.0       2278.0       1947.5
          5.0       1236.0        710.0       1515.0       1214.7
          6.0        481.0        232.0        897.0        631.3
          7.0        150.0         67.0        332.0        281.3
          8.0         30.0         16.0        131.0        109.6
          9.0          4.0          2.0         49.0         38.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.113           2.609           3.410           3.021
                   Theta0:           0.005           0.000           0.004           0.000               
                   Theta1:       99999.000          28.883       99999.000       60013.055
Mean No. of pairwise diff.           3.009           2.556           3.393           2.999
           No. pol. sites:              21             155             190         174.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        441.0        378.0        903.0        494.3
          1.0       1489.0       1248.0       2104.0       1541.1
          2.0       2465.0       1896.0       3076.0       2403.0
          3.0       2697.0       2239.0       2873.0       2497.9
          4.0       2182.0       1635.0       2278.0       1947.5
          5.0       1236.0        710.0       1515.0       1214.7
          6.0        481.0        232.0        897.0        631.3
          7.0        150.0         67.0        332.0        281.3
          8.0         30.0         16.0        131.0        109.6
          9.0          4.0          2.0         49.0         38.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.113           2.609           3.410           3.021
                   Theta0:           0.005           0.000           0.004           0.000               
                   Theta1:       99999.000          28.883       99999.000       60013.055
Mean No. of pairwise diff.           3.009           2.556           3.393           2.999
           No. pol. sites:              21             155             190         174.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        441.0        378.0        903.0        494.3
          1.0       1489.0       1248.0       2104.0       1541.1
          2.0       2465.0       1896.0       3076.0       2403.0
          3.0       2697.0       2239.0       2873.0       2497.9
          4.0       2182.0       1635.0       2278.0       1947.5
          5.0       1236.0        710.0       1515.0       1214.7
          6.0        481.0        232.0        897.0        631.3
          7.0        150.0         67.0        332.0        281.3
          8.0         30.0         16.0        131.0        109.6
          9.0          4.0          2.0         49.0         38.0
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_32_00_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            21      21.000       0.000
  No. of usable loci            21      21.000       0.000
  No. of polym. loci            21      21.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_32_00_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.03946     0.03946     0.00000     0.03946
     2       0.30568     0.30568     0.00000     0.30568
     3       0.15937     0.15937     0.00000     0.15937
     4       0.02649     0.02649     0.00000     0.02649
     5       0.01333     0.01333     0.00000     0.01333
     6       0.24242     0.24242     0.00000     0.24242
     7       0.27060     0.27060     0.00000     0.27060
     8       0.03946     0.03946     0.00000     0.03946
     9       0.27964     0.27964     0.00000     0.27964
    10       0.12528     0.12528     0.00000     0.12528
    11       0.01333     0.01333     0.00000     0.01333
    12       0.13682     0.13682     0.00000     0.13682
    13       0.32215     0.32215     0.00000     0.32215
    14       0.01333     0.01333     0.00000     0.01333
    15       0.13682     0.13682     0.00000     0.13682
    16       0.18121     0.18121     0.00000     0.18121
    17       0.02649     0.02649     0.00000     0.02649
    18       0.01333     0.01333     0.00000     0.01333
    19       0.20233     0.20233     0.00000     0.20233
    20       0.36018     0.36018     0.00000     0.36018
    21       0.10166     0.10166     0.00000     0.10166
--------------------------------------------------------
  Mean       0.14330     0.14330     0.00000     0.14330
  s.d.       0.11699     0.11699     0.00000     0.11699
--------------------------------------------------------
<A NAME="31_01_17at16_32_00_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_32_00_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      41.69515    41.69515     0.00000
            Theta_k_lower      29.72165    29.72165     0.00000
            Theta_k_upper      58.23790    58.23790     0.00000
                  Theta_H      22.64575    22.64575     0.00000
             s.d. Theta_H       5.92549     5.92549     0.00000
                  Theta_S       3.76040     3.76040     0.00000
             s.d. Theta_S       1.15995     1.15995     0.00000
                 Theta_pi       3.00940     3.00940     0.00000
            s.d. Theta_pi       1.74826     1.74826     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_32_00_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.11328     3.11328     0.00000
              Tau qt 2.5%       2.60938     2.60938     0.00000
                Tau qt 5%       2.60938     2.60938     0.00000
               Tau qt 95%       3.29102     3.29102     0.00000
             Tau qt 97.5%       3.29102     3.29102     0.00000
                   Theta0       0.00527     0.00527     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00000     0.00000     0.00000
          Theta0 qt 97.5%       0.00000     0.00000     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      28.88281    28.88281     0.00000
             Theta1 qt 5%      28.88281    28.88281     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00122     0.00122     0.00000
      Model (SSD) p-value       0.30000     0.30000     0.00000
         Raggedness index       0.03171     0.03171     0.00000
       Raggedness p-value       0.30000     0.30000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 163 (31/01/17 at 16:32:00))
Total computing time for this run : 0h 0m 0s 104 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

