
<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_35_56"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 115 (31/01/17 at 16:35:56)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_35_56_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_35_56_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 71
No. of loci              : 21
No. of usable loci       : 21 loci with less than  5.00 % missing data
No. of polymorphic sites : 21


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.03946
     2       150         2   0.14819
     3       150         2   0.36725
     4       150         2   0.14819
     5       150         2   0.38738
     6       150         2   0.33790
     7       150         2   0.13682
     8       150         2   0.39991
     9       150         2   0.15937
    10       150         2   0.08957
    11       150         2   0.08957
    12       150         2   0.38085
    13       150         2   0.37414
    14       150         2   0.05226
    15       150         2   0.02649
    16       150         2   0.48045
    17       150         2   0.14819
    18       150         2   0.03946
    19       150         2   0.12528
    20       150         2   0.39374
    21       150         2   0.08957
-------------------------------------------------
  Mean   150.000     2.000   0.21019
  s.d.     0.000     0.000   0.15140
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0225
Gene diversity           :  0.9841 +/-  0.0028

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 71

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 21
Number of usable nucleotide sites            : 21


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.414049 +/-     2.190594

Average gene diversity over loci              :     0.210193 +/-     0.115466

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    59.907044
  S.D. Theta(Hom) :    11.139068

         Theta(k) :    52.101250
95 % confidence interval limits for theta(k) : [  37.356756,  72.488390 ]

         Theta(S) :     3.760399
    S.D. Theta(S) :     1.159953

        Theta(Pi) :     4.414049
   S.D. Theta(Pi) :     2.424776

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     178       0.000
       1     427       0.000
       2     895       0.000
       3    1848       0.000
       4    2605       0.000
       5    2336       0.000
       6    1556       0.000
       7     784       0.000
       8     330       0.000
       9     162       0.000
      10      50       0.000
      11       4       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.414
Mismatch observed variance          :        3.324
Tau                                 :        4.570
Theta0                              :        0.002
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00513393
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.02632950
        P(Sim. Rag. >= Obs. Rag.): 0.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        178.0        108.4        115.6
          1.0        427.0        473.6        528.2
          2.0        895.0       1118.5       1207.3
          3.0       1848.0       1751.0       1840.0
          4.0       2605.0       2067.1       2103.1
          5.0       2336.0       1972.5       1923.1
          6.0       1556.0       1509.4       1465.4
          7.0        784.0       1014.9        957.1
          8.0        330.0        587.0        547.0
          9.0        162.0        312.9        277.9
         10.0         50.0        154.3        127.0
         11.0          4.0         63.4         52.8
         12.0          0.0         26.6         20.1
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.570           4.078           5.039           4.655
                   Theta0:           0.002           0.000           0.146           0.032               
                   Theta1:       99999.000         122.438       99999.000       70130.328
Mean No. of pairwise diff.           4.414           4.096           5.024           4.701
           No. pol. sites:              21             214             251         235.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        178.0         15.0        190.0        115.6
          1.0        427.0        247.0        720.0        528.2
          2.0        895.0        907.0       1521.0       1207.3
          3.0       1848.0       1470.0       2276.0       1840.0
          4.0       2605.0       1881.0       2267.0       2103.1
          5.0       2336.0       1701.0       2311.0       1923.1
          6.0       1556.0       1077.0       1756.0       1465.4
          7.0        784.0        714.0       1291.0        957.1
          8.0        330.0        356.0        730.0        547.0
          9.0        162.0        147.0        426.0        277.9
         10.0         50.0         45.0        243.0        127.0
         11.0          4.0         11.0        148.0         52.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.570           4.078           5.039           4.655
                   Theta0:           0.002           0.000           0.146           0.032               
                   Theta1:       99999.000         122.438       99999.000       70130.328
Mean No. of pairwise diff.           4.414           4.096           5.024           4.701
           No. pol. sites:              21             214             251         235.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        178.0         15.0        190.0        115.6
          1.0        427.0        247.0        720.0        528.2
          2.0        895.0        907.0       1521.0       1207.3
          3.0       1848.0       1470.0       2276.0       1840.0
          4.0       2605.0       1881.0       2267.0       2103.1
          5.0       2336.0       1701.0       2311.0       1923.1
          6.0       1556.0       1077.0       1756.0       1465.4
          7.0        784.0        714.0       1291.0        957.1
          8.0        330.0        356.0        730.0        547.0
          9.0        162.0        147.0        426.0        277.9
         10.0         50.0         45.0        243.0        127.0
         11.0          4.0         11.0        148.0         52.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.570           4.078           5.039           4.655
                   Theta0:           0.002           0.000           0.146           0.032               
                   Theta1:       99999.000         122.438       99999.000       70130.328
Mean No. of pairwise diff.           4.414           4.096           5.024           4.701
           No. pol. sites:              21             214             251         235.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        178.0         15.0        190.0        115.6
          1.0        427.0        247.0        720.0        528.2
          2.0        895.0        907.0       1521.0       1207.3
          3.0       1848.0       1470.0       2276.0       1840.0
          4.0       2605.0       1881.0       2267.0       2103.1
          5.0       2336.0       1701.0       2311.0       1923.1
          6.0       1556.0       1077.0       1756.0       1465.4
          7.0        784.0        714.0       1291.0        957.1
          8.0        330.0        356.0        730.0        547.0
          9.0        162.0        147.0        426.0        277.9
         10.0         50.0         45.0        243.0        127.0
         11.0          4.0         11.0        148.0         52.8
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_35_56_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            21      21.000       0.000
  No. of usable loci            21      21.000       0.000
  No. of polym. loci            21      21.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_35_56_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.03946     0.03946     0.00000     0.03946
     2       0.14819     0.14819     0.00000     0.14819
     3       0.36725     0.36725     0.00000     0.36725
     4       0.14819     0.14819     0.00000     0.14819
     5       0.38738     0.38738     0.00000     0.38738
     6       0.33790     0.33790     0.00000     0.33790
     7       0.13682     0.13682     0.00000     0.13682
     8       0.39991     0.39991     0.00000     0.39991
     9       0.15937     0.15937     0.00000     0.15937
    10       0.08957     0.08957     0.00000     0.08957
    11       0.08957     0.08957     0.00000     0.08957
    12       0.38085     0.38085     0.00000     0.38085
    13       0.37414     0.37414     0.00000     0.37414
    14       0.05226     0.05226     0.00000     0.05226
    15       0.02649     0.02649     0.00000     0.02649
    16       0.48045     0.48045     0.00000     0.48045
    17       0.14819     0.14819     0.00000     0.14819
    18       0.03946     0.03946     0.00000     0.03946
    19       0.12528     0.12528     0.00000     0.12528
    20       0.39374     0.39374     0.00000     0.39374
    21       0.08957     0.08957     0.00000     0.08957
--------------------------------------------------------
  Mean       0.21019     0.21019     0.00000     0.21019
  s.d.       0.15140     0.15140     0.00000     0.15140
--------------------------------------------------------
<A NAME="31_01_17at16_35_56_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_35_56_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      52.10125    52.10125     0.00000
            Theta_k_lower      37.35676    37.35676     0.00000
            Theta_k_upper      72.48839    72.48839     0.00000
                  Theta_H      59.90704    59.90704     0.00000
             s.d. Theta_H      11.13907    11.13907     0.00000
                  Theta_S       3.76040     3.76040     0.00000
             s.d. Theta_S       1.15995     1.15995     0.00000
                 Theta_pi       4.41405     4.41405     0.00000
            s.d. Theta_pi       2.42478     2.42478     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_35_56_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.57031     4.57031     0.00000
              Tau qt 2.5%       4.07812     4.07812     0.00000
                Tau qt 5%       4.07812     4.07812     0.00000
               Tau qt 95%       4.97266     4.97266     0.00000
             Tau qt 97.5%       4.97266     4.97266     0.00000
                   Theta0       0.00176     0.00176     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.06504     0.06504     0.00000
          Theta0 qt 97.5%       0.06504     0.06504     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%     122.43750   122.43750     0.00000
             Theta1 qt 5%     122.43750   122.43750     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00513     0.00513     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.02633     0.02633     0.00000
       Raggedness p-value       0.00000     0.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 115 (31/01/17 at 16:35:57))
Total computing time for this run : 0h 0m 0s 179 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

