
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_47_45"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 231 (31/01/17 at 14:47:45)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_47_45_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_47_45_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 34
No. of loci              : 16
No. of usable loci       : 16 loci with less than  5.00 % missing data
No. of polymorphic sites : 16


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.17038
     2       150         2   0.14819
     3       150         2   0.01333
     4       150         2   0.17038
     5       150         2   0.17038
     6       150         2   0.22273
     7       150         2   0.12528
     8       150         2   0.10166
     9       150         2   0.41736
    10       150         2   0.14819
    11       150         2   0.11356
    12       150         2   0.33790
    13       150         2   0.01333
    14       150         2   0.01333
    15       150         2   0.30568
    16       150         2   0.23266
-------------------------------------------------
  Mean   150.000     2.000   0.16902
  s.d.     0.000     0.000   0.11516
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0969
Gene diversity           :  0.9092 +/-  0.0163

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 34

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 16
Number of usable nucleotide sites            : 16


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.704340 +/-     1.445698

Average gene diversity over loci              :     0.169021 +/-     0.100015

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     8.663587
  S.D. Theta(Hom) :     1.876921

         Theta(k) :    13.414024
95 % confidence interval limits for theta(k) : [   8.902311,  19.887208 ]

         Theta(S) :     2.865066
    S.D. Theta(S) :     0.947159

        Theta(Pi) :     2.704340
   S.D. Theta(Pi) :     1.600247

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1015     803.070
       1    1942    2069.272
       2    2384    2701.871
       3    2389    2389.508
       4    1827    1615.140
       5    1002     893.257
       6     395     422.873
       7     130     177.142
       8      73      67.406
       9      18      23.811
      10       0       7.953
      11       0       2.551
      12       0       0.796
      13       0       0.244
      14       0       0.074
      15       0       0.022
      16       0       0.007
      17       0       0.002
      18       0       0.001


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.704
Mismatch observed variance          :        2.886
Tau                                 :        2.967
Theta0                              :        0.005
Theta1                              :       20.273

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00037546
          P(Sim. Ssd >= Obs. Ssd): 0.90000000
    Harpending's Raggedness index: 0.01999076
        P(Sim. Rag. >= Obs. Rag.): 0.60000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1015.0        899.5        996.2
          1.0       1942.0       1740.4       1878.0
          2.0       2384.0       2093.1       2469.9
          3.0       2389.0       2265.5       2334.4
          4.0       1827.0       1838.5       1695.2
          5.0       1002.0       1193.7        994.6
          6.0        395.0        632.0        488.7
          7.0        130.0        301.0        206.3
          8.0         73.0        130.0         76.3
          9.0         18.0         48.1         25.1
         10.0          0.0         21.6          7.5
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.967           0.689           4.055           2.837
                   Theta0:           0.005           0.000           1.837           0.231               
                   Theta1:          20.273          10.107       99999.000       30054.602
Mean No. of pairwise diff.           2.704           2.526           3.764           3.021
           No. pol. sites:              16              37              62          50.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1015.0        519.0       1432.0        996.2
          1.0       1942.0        934.0       2687.0       1878.0
          2.0       2384.0       1559.0       2927.0       2469.9
          3.0       2389.0       1209.0       3013.0       2334.4
          4.0       1827.0        963.0       2683.0       1695.2
          5.0       1002.0        714.0       1990.0        994.6
          6.0        395.0        299.0       1047.0        488.7
          7.0        130.0         50.0        679.0        206.3
          8.0         73.0          3.0        363.0         76.3
          9.0         18.0          0.0        140.0         25.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.967           0.689           4.055           2.837
                   Theta0:           0.005           0.000           1.837           0.231               
                   Theta1:          20.273          10.107       99999.000       30054.602
Mean No. of pairwise diff.           2.704           2.526           3.764           3.021
           No. pol. sites:              16              37              62          50.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1015.0        519.0       1432.0        996.2
          1.0       1942.0        934.0       2687.0       1878.0
          2.0       2384.0       1559.0       2927.0       2469.9
          3.0       2389.0       1209.0       3013.0       2334.4
          4.0       1827.0        963.0       2683.0       1695.2
          5.0       1002.0        714.0       1990.0        994.6
          6.0        395.0        299.0       1047.0        488.7
          7.0        130.0         50.0        679.0        206.3
          8.0         73.0          3.0        363.0         76.3
          9.0         18.0          0.0        140.0         25.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.967           0.689           4.055           2.837
                   Theta0:           0.005           0.000           1.837           0.231               
                   Theta1:          20.273          10.107       99999.000       30054.602
Mean No. of pairwise diff.           2.704           2.526           3.764           3.021
           No. pol. sites:              16              37              62          50.500
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1015.0        519.0       1432.0        996.2
          1.0       1942.0        934.0       2687.0       1878.0
          2.0       2384.0       1559.0       2927.0       2469.9
          3.0       2389.0       1209.0       3013.0       2334.4
          4.0       1827.0        963.0       2683.0       1695.2
          5.0       1002.0        714.0       1990.0        994.6
          6.0        395.0        299.0       1047.0        488.7
          7.0        130.0         50.0        679.0        206.3
          8.0         73.0          3.0        363.0         76.3
          9.0         18.0          0.0        140.0         25.1
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_47_45_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            16      16.000       0.000
  No. of usable loci            16      16.000       0.000
  No. of polym. loci            16      16.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_47_45_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.17038     0.17038     0.00000     0.17038
     2       0.14819     0.14819     0.00000     0.14819
     3       0.01333     0.01333     0.00000     0.01333
     4       0.17038     0.17038     0.00000     0.17038
     5       0.17038     0.17038     0.00000     0.17038
     6       0.22273     0.22273     0.00000     0.22273
     7       0.12528     0.12528     0.00000     0.12528
     8       0.10166     0.10166     0.00000     0.10166
     9       0.41736     0.41736     0.00000     0.41736
    10       0.14819     0.14819     0.00000     0.14819
    11       0.11356     0.11356     0.00000     0.11356
    12       0.33790     0.33790     0.00000     0.33790
    13       0.01333     0.01333     0.00000     0.01333
    14       0.01333     0.01333     0.00000     0.01333
    15       0.30568     0.30568     0.00000     0.30568
    16       0.23266     0.23266     0.00000     0.23266
--------------------------------------------------------
  Mean       0.16902     0.16902     0.00000     0.16902
  s.d.       0.11516     0.11516     0.00000     0.11516
--------------------------------------------------------
<A NAME="31_01_17at14_47_45_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_47_45_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      13.41402    13.41402     0.00000
            Theta_k_lower       8.90231     8.90231     0.00000
            Theta_k_upper      19.88721    19.88721     0.00000
                  Theta_H       8.66359     8.66359     0.00000
             s.d. Theta_H       1.87692     1.87692     0.00000
                  Theta_S       2.86507     2.86507     0.00000
             s.d. Theta_S       0.94716     0.94716     0.00000
                 Theta_pi       2.70434     2.70434     0.00000
            s.d. Theta_pi       1.60025     1.60025     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_47_45_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       2.96680     2.96680     0.00000
              Tau qt 2.5%       0.68945     0.68945     0.00000
                Tau qt 5%       0.68945     0.68945     0.00000
               Tau qt 95%       3.74609     3.74609     0.00000
             Tau qt 97.5%       3.74609     3.74609     0.00000
                   Theta0       0.00527     0.00527     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.43594     0.43594     0.00000
          Theta0 qt 97.5%       0.43594     0.43594     0.00000
                   Theta1      20.27344    20.27344     0.00000
           Theta1 qt 2.5%      10.10742    10.10742     0.00000
             Theta1 qt 5%      10.10742    10.10742     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00038     0.00038     0.00000
      Model (SSD) p-value       0.90000     0.90000     0.00000
         Raggedness index       0.01999     0.01999     0.00000
       Raggedness p-value       0.60000     0.60000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 231 (31/01/17 at 14:47:45))
Total computing time for this run : 0h 0m 0s 108 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

