
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_49_45"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 204 (31/01/17 at 14:49:45)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_49_45_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_49_45_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 57
No. of loci              : 71
No. of usable loci       : 71 loci with less than  5.00 % missing data
No. of polymorphic sites : 71


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.17038
     2       150         2   0.17038
     3       150         2   0.07732
     4       150         2   0.14819
     5       150         2   0.01333
     6       150         2   0.01333
     7       150         2   0.01333
     8       150         2   0.03946
     9       150         2   0.02649
    10       150         2   0.21262
    11       150         2   0.03946
    12       150         2   0.01333
    13       150         2   0.01333
    14       150         2   0.01333
    15       150         2   0.02649
    16       150         2   0.02649
    17       150         2   0.01333
    18       150         2   0.02649
    19       150         2   0.14819
    20       150         2   0.02649
    21       150         2   0.05226
    22       150         2   0.02649
    23       150         2   0.01333
    24       150         2   0.01333
    25       150         2   0.02649
    26       150         2   0.08957
    27       150         2   0.01333
    28       150         2   0.03946
    29       150         2   0.05226
    30       150         2   0.11356
    31       150         2   0.01333
    32       150         2   0.06488
    33       150         2   0.01333
    34       150         2   0.01333
    35       150         2   0.08957
    36       150         2   0.01333
    37       150         2   0.01333
    38       150         2   0.19186
    39       150         2   0.02649
    40       150         2   0.01333
    41       150         2   0.02649
    42       150         2   0.19186
    43       150         2   0.01333
    44       150         2   0.05226
    45       150         2   0.06488
    46       150         2   0.01333
    47       150         2   0.01333
    48       150         2   0.07732
    49       150         2   0.11356
    50       150         2   0.02649
    51       150         2   0.18121
    52       150         2   0.03946
    53       150         2   0.01333
    54       150         2   0.02649
    55       150         2   0.01333
    56       150         2   0.10166
    57       150         2   0.08957
    58       150         2   0.01333
    59       150         2   0.01333
    60       150         2   0.10166
    61       150         2   0.02649
    62       150         2   0.03946
    63       150         2   0.02649
    64       150         2   0.01333
    65       150         2   0.19186
    66       150         2   0.08957
    67       150         2   0.02649
    68       150         2   0.02649
    69       150         2   0.12528
    70       150         2   0.01333
    71       150         2   0.03946
-------------------------------------------------
  Mean   150.000     2.000   0.05534
  s.d.     0.000     0.000   0.05608
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0324
Gene diversity           :  0.9741 +/-  0.0041

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 57

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 71
Number of usable nucleotide sites            : 71


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.929128 +/-     1.980219

Average gene diversity over loci              :     0.055340 +/-     0.030872

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    35.870912
  S.D. Theta(Hom) :     6.138437

         Theta(k) :    33.066749
95 % confidence interval limits for theta(k) : [  23.361695,  46.504216 ]

         Theta(S) :    12.713730
    S.D. Theta(S) :     3.198456

        Theta(Pi) :     3.929128
   S.D. Theta(Pi) :     2.191911

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     289       0.000
       1     820       0.000
       2    1399       0.000
       3    1950       0.000
       4    2461       0.000
       5    2008       0.000
       6    1400       0.000
       7     654       0.000
       8     169       0.000
       9      24       0.000
      10       1       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.929
Mismatch observed variance          :        3.251
Tau                                 :        4.186
Theta0                              :        0.000
Theta1                              :    31135.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00224044
          P(Sim. Ssd >= Obs. Ssd): 0.20000000
    Harpending's Raggedness index: 0.02058040
        P(Sim. Rag. >= Obs. Rag.): 0.70000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        289.0        153.7        170.3
          1.0        820.0        695.8        711.8
          2.0       1399.0       1546.4       1489.3
          3.0       1950.0       2185.9       2077.7
          4.0       2461.0       2351.1       2174.0
          5.0       2008.0       1843.3       1819.8
          6.0       1400.0       1156.1       1269.5
          7.0        654.0        666.9        759.1
          8.0        169.0        322.5        397.1
          9.0         24.0        158.5        184.7
         10.0          1.0         62.2         77.3
         11.0          0.0         21.5         29.4
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.186           3.738           4.633           4.078
                   Theta0:           0.000           0.000           0.037           0.007               
                   Theta1:       31135.000          60.938       99999.000       90005.194
Mean No. of pairwise diff.           3.929           3.684           4.571           4.078
           No. pol. sites:              71             196             223         211.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        289.0        105.0        231.0        170.3
          1.0        820.0        400.0        943.0        711.8
          2.0       1399.0       1137.0       1826.0       1489.3
          3.0       1950.0       1764.0       2681.0       2077.7
          4.0       2461.0       2031.0       2774.0       2174.0
          5.0       2008.0       1605.0       2195.0       1819.8
          6.0       1400.0        881.0       1514.0       1269.5
          7.0        654.0        408.0       1037.0        759.1
          8.0        169.0        155.0        481.0        397.1
          9.0         24.0         47.0        261.0        184.7
         10.0          1.0         12.0        135.0         77.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.186           3.738           4.633           4.078
                   Theta0:           0.000           0.000           0.037           0.007               
                   Theta1:       31135.000          60.938       99999.000       90005.194
Mean No. of pairwise diff.           3.929           3.684           4.571           4.078
           No. pol. sites:              71             196             223         211.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        289.0        105.0        231.0        170.3
          1.0        820.0        400.0        943.0        711.8
          2.0       1399.0       1137.0       1826.0       1489.3
          3.0       1950.0       1764.0       2681.0       2077.7
          4.0       2461.0       2031.0       2774.0       2174.0
          5.0       2008.0       1605.0       2195.0       1819.8
          6.0       1400.0        881.0       1514.0       1269.5
          7.0        654.0        408.0       1037.0        759.1
          8.0        169.0        155.0        481.0        397.1
          9.0         24.0         47.0        261.0        184.7
         10.0          1.0         12.0        135.0         77.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.186           3.738           4.633           4.078
                   Theta0:           0.000           0.000           0.037           0.007               
                   Theta1:       31135.000          60.938       99999.000       90005.194
Mean No. of pairwise diff.           3.929           3.684           4.571           4.078
           No. pol. sites:              71             196             223         211.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        289.0        105.0        231.0        170.3
          1.0        820.0        400.0        943.0        711.8
          2.0       1399.0       1137.0       1826.0       1489.3
          3.0       1950.0       1764.0       2681.0       2077.7
          4.0       2461.0       2031.0       2774.0       2174.0
          5.0       2008.0       1605.0       2195.0       1819.8
          6.0       1400.0        881.0       1514.0       1269.5
          7.0        654.0        408.0       1037.0        759.1
          8.0        169.0        155.0        481.0        397.1
          9.0         24.0         47.0        261.0        184.7
         10.0          1.0         12.0        135.0         77.3
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_49_45_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            71      71.000       0.000
  No. of usable loci            71      71.000       0.000
  No. of polym. loci            71      71.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_49_45_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.17038     0.17038     0.00000     0.17038
     2       0.17038     0.17038     0.00000     0.17038
     3       0.07732     0.07732     0.00000     0.07732
     4       0.14819     0.14819     0.00000     0.14819
     5       0.01333     0.01333     0.00000     0.01333
     6       0.01333     0.01333     0.00000     0.01333
     7       0.01333     0.01333     0.00000     0.01333
     8       0.03946     0.03946     0.00000     0.03946
     9       0.02649     0.02649     0.00000     0.02649
    10       0.21262     0.21262     0.00000     0.21262
    11       0.03946     0.03946     0.00000     0.03946
    12       0.01333     0.01333     0.00000     0.01333
    13       0.01333     0.01333     0.00000     0.01333
    14       0.01333     0.01333     0.00000     0.01333
    15       0.02649     0.02649     0.00000     0.02649
    16       0.02649     0.02649     0.00000     0.02649
    17       0.01333     0.01333     0.00000     0.01333
    18       0.02649     0.02649     0.00000     0.02649
    19       0.14819     0.14819     0.00000     0.14819
    20       0.02649     0.02649     0.00000     0.02649
    21       0.05226     0.05226     0.00000     0.05226
    22       0.02649     0.02649     0.00000     0.02649
    23       0.01333     0.01333     0.00000     0.01333
    24       0.01333     0.01333     0.00000     0.01333
    25       0.02649     0.02649     0.00000     0.02649
    26       0.08957     0.08957     0.00000     0.08957
    27       0.01333     0.01333     0.00000     0.01333
    28       0.03946     0.03946     0.00000     0.03946
    29       0.05226     0.05226     0.00000     0.05226
    30       0.11356     0.11356     0.00000     0.11356
    31       0.01333     0.01333     0.00000     0.01333
    32       0.06488     0.06488     0.00000     0.06488
    33       0.01333     0.01333     0.00000     0.01333
    34       0.01333     0.01333     0.00000     0.01333
    35       0.08957     0.08957     0.00000     0.08957
    36       0.01333     0.01333     0.00000     0.01333
    37       0.01333     0.01333     0.00000     0.01333
    38       0.19186     0.19186     0.00000     0.19186
    39       0.02649     0.02649     0.00000     0.02649
    40       0.01333     0.01333     0.00000     0.01333
    41       0.02649     0.02649     0.00000     0.02649
    42       0.19186     0.19186     0.00000     0.19186
    43       0.01333     0.01333     0.00000     0.01333
    44       0.05226     0.05226     0.00000     0.05226
    45       0.06488     0.06488     0.00000     0.06488
    46       0.01333     0.01333     0.00000     0.01333
    47       0.01333     0.01333     0.00000     0.01333
    48       0.07732     0.07732     0.00000     0.07732
    49       0.11356     0.11356     0.00000     0.11356
    50       0.02649     0.02649     0.00000     0.02649
    51       0.18121     0.18121     0.00000     0.18121
    52       0.03946     0.03946     0.00000     0.03946
    53       0.01333     0.01333     0.00000     0.01333
    54       0.02649     0.02649     0.00000     0.02649
    55       0.01333     0.01333     0.00000     0.01333
    56       0.10166     0.10166     0.00000     0.10166
    57       0.08957     0.08957     0.00000     0.08957
    58       0.01333     0.01333     0.00000     0.01333
    59       0.01333     0.01333     0.00000     0.01333
    60       0.10166     0.10166     0.00000     0.10166
    61       0.02649     0.02649     0.00000     0.02649
    62       0.03946     0.03946     0.00000     0.03946
    63       0.02649     0.02649     0.00000     0.02649
    64       0.01333     0.01333     0.00000     0.01333
    65       0.19186     0.19186     0.00000     0.19186
    66       0.08957     0.08957     0.00000     0.08957
    67       0.02649     0.02649     0.00000     0.02649
    68       0.02649     0.02649     0.00000     0.02649
    69       0.12528     0.12528     0.00000     0.12528
    70       0.01333     0.01333     0.00000     0.01333
    71       0.03946     0.03946     0.00000     0.03946
--------------------------------------------------------
  Mean       0.05534     0.05534     0.00000     0.05534
  s.d.       0.05608     0.05608     0.00000     0.05608
--------------------------------------------------------
<A NAME="31_01_17at14_49_45_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_49_45_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      33.06675    33.06675     0.00000
            Theta_k_lower      23.36170    23.36170     0.00000
            Theta_k_upper      46.50422    46.50422     0.00000
                  Theta_H      35.87091    35.87091     0.00000
             s.d. Theta_H       6.13844     6.13844     0.00000
                  Theta_S      12.71373    12.71373     0.00000
             s.d. Theta_S       3.19846     3.19846     0.00000
                 Theta_pi       3.92913     3.92913     0.00000
            s.d. Theta_pi       2.19191     2.19191     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_49_45_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.18555     4.18555     0.00000
              Tau qt 2.5%       3.73828     3.73828     0.00000
                Tau qt 5%       3.73828     3.73828     0.00000
               Tau qt 95%       4.41211     4.41211     0.00000
             Tau qt 97.5%       4.41211     4.41211     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.01934     0.01934     0.00000
          Theta0 qt 97.5%       0.01934     0.01934     0.00000
                   Theta1   31135.00000 31135.00000     0.00000
           Theta1 qt 2.5%      60.93750    60.93750     0.00000
             Theta1 qt 5%      60.93750    60.93750     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00224     0.00224     0.00000
      Model (SSD) p-value       0.20000     0.20000     0.00000
         Raggedness index       0.02058     0.02058     0.00000
       Raggedness p-value       0.70000     0.70000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 204 (31/01/17 at 14:49:45))
Total computing time for this run : 0h 0m 0s 121 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

