
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at15_00_45"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 373 (31/01/17 at 15:00:45)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at15_00_45_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at15_00_45_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 61
No. of loci              : 22
No. of usable loci       : 22 loci with less than  5.00 % missing data
No. of polymorphic sites : 22


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.44286
     3       150         2   0.15937
     4       150         2   0.15937
     5       150         2   0.01333
     6       150         2   0.18121
     7       150         2   0.42810
     8       150         2   0.35293
     9       150         2   0.02649
    10       150         2   0.02649
    11       150         2   0.50300
    12       150         2   0.38738
    13       150         2   0.02649
    14       150         2   0.43812
    15       150         2   0.25199
    16       150         2   0.02649
    17       150         2   0.01333
    18       150         2   0.41736
    19       150         2   0.13682
    20       150         2   0.18121
    21       150         2   0.39991
    22       150         2   0.05226
-------------------------------------------------
  Mean   150.000     2.000   0.21081
  s.d.     0.000     0.000   0.17730
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0276
Gene diversity           :  0.9790 +/-  0.0034

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 61

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 22
Number of usable nucleotide sites            : 22


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.637852 +/-     2.287535

Average gene diversity over loci              :     0.210811 +/-     0.115095

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    44.719045
  S.D. Theta(Hom) :     7.667088

         Theta(k) :    37.803161
95 % confidence interval limits for theta(k) : [  26.855382,  52.937022 ]

         Theta(S) :     3.939466
    S.D. Theta(S) :     1.201985

        Theta(Pi) :     4.637852
   S.D. Theta(Pi) :     2.532080

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     235       0.000
       1     566       0.000
       2     821       0.000
       3    1582       0.000
       4    2147       0.000
       5    2030       0.000
       6    1665       0.000
       7    1212       0.000
       8     641       0.000
       9     226       0.000
      10      50       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.638
Mismatch observed variance          :        4.159
Tau                                 :        4.922
Theta0                              :        0.000
Theta1                              :      268.438

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00128207
          P(Sim. Ssd >= Obs. Ssd): 0.20000000
    Harpending's Raggedness index: 0.01566934
        P(Sim. Rag. >= Obs. Rag.): 0.40000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        235.0         84.4        121.1
          1.0        566.0        352.5        433.0
          2.0        821.0        891.7       1004.0
          3.0       1582.0       1540.8       1618.3
          4.0       2147.0       1952.0       1977.0
          5.0       2030.0       1980.2       1939.2
          6.0       1665.0       1642.6       1587.6
          7.0       1212.0       1198.6       1114.9
          8.0        641.0        758.5        685.3
          9.0        226.0        409.4        374.6
         10.0         50.0        201.6        184.3
         11.0          0.0         98.8         82.4
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.922           4.037           5.727           4.905
                   Theta0:           0.000           0.000           0.580           0.062               
                   Theta1:         268.438          87.188       99999.000       80058.762
Mean No. of pairwise diff.           4.638           4.587           5.522           5.044
           No. pol. sites:              22             166             202         179.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        235.0         46.0        130.0        121.1
          1.0        566.0        186.0        458.0        433.0
          2.0        821.0        521.0       1220.0       1004.0
          3.0       1582.0       1077.0       1871.0       1618.3
          4.0       2147.0       1611.0       2209.0       1977.0
          5.0       2030.0       1789.0       2263.0       1939.2
          6.0       1665.0       1382.0       2049.0       1587.6
          7.0       1212.0        961.0       1575.0       1114.9
          8.0        641.0        547.0       1067.0        685.3
          9.0        226.0        215.0        583.0        374.6
         10.0         50.0         52.0        284.0        184.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.922           4.037           5.727           4.905
                   Theta0:           0.000           0.000           0.580           0.062               
                   Theta1:         268.438          87.188       99999.000       80058.762
Mean No. of pairwise diff.           4.638           4.587           5.522           5.044
           No. pol. sites:              22             166             202         179.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        235.0         46.0        130.0        121.1
          1.0        566.0        186.0        458.0        433.0
          2.0        821.0        521.0       1220.0       1004.0
          3.0       1582.0       1077.0       1871.0       1618.3
          4.0       2147.0       1611.0       2209.0       1977.0
          5.0       2030.0       1789.0       2263.0       1939.2
          6.0       1665.0       1382.0       2049.0       1587.6
          7.0       1212.0        961.0       1575.0       1114.9
          8.0        641.0        547.0       1067.0        685.3
          9.0        226.0        215.0        583.0        374.6
         10.0         50.0         52.0        284.0        184.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.922           4.037           5.727           4.905
                   Theta0:           0.000           0.000           0.580           0.062               
                   Theta1:         268.438          87.188       99999.000       80058.762
Mean No. of pairwise diff.           4.638           4.587           5.522           5.044
           No. pol. sites:              22             166             202         179.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        235.0         46.0        130.0        121.1
          1.0        566.0        186.0        458.0        433.0
          2.0        821.0        521.0       1220.0       1004.0
          3.0       1582.0       1077.0       1871.0       1618.3
          4.0       2147.0       1611.0       2209.0       1977.0
          5.0       2030.0       1789.0       2263.0       1939.2
          6.0       1665.0       1382.0       2049.0       1587.6
          7.0       1212.0        961.0       1575.0       1114.9
          8.0        641.0        547.0       1067.0        685.3
          9.0        226.0        215.0        583.0        374.6
         10.0         50.0         52.0        284.0        184.3
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at15_00_45_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            22      22.000       0.000
  No. of usable loci            22      22.000       0.000
  No. of polym. loci            22      22.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at15_00_45_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.44286     0.44286     0.00000     0.44286
     3       0.15937     0.15937     0.00000     0.15937
     4       0.15937     0.15937     0.00000     0.15937
     5       0.01333     0.01333     0.00000     0.01333
     6       0.18121     0.18121     0.00000     0.18121
     7       0.42810     0.42810     0.00000     0.42810
     8       0.35293     0.35293     0.00000     0.35293
     9       0.02649     0.02649     0.00000     0.02649
    10       0.02649     0.02649     0.00000     0.02649
    11       0.50300     0.50300     0.00000     0.50300
    12       0.38738     0.38738     0.00000     0.38738
    13       0.02649     0.02649     0.00000     0.02649
    14       0.43812     0.43812     0.00000     0.43812
    15       0.25199     0.25199     0.00000     0.25199
    16       0.02649     0.02649     0.00000     0.02649
    17       0.01333     0.01333     0.00000     0.01333
    18       0.41736     0.41736     0.00000     0.41736
    19       0.13682     0.13682     0.00000     0.13682
    20       0.18121     0.18121     0.00000     0.18121
    21       0.39991     0.39991     0.00000     0.39991
    22       0.05226     0.05226     0.00000     0.05226
--------------------------------------------------------
  Mean       0.21081     0.21081     0.00000     0.21081
  s.d.       0.17730     0.17730     0.00000     0.17730
--------------------------------------------------------
<A NAME="31_01_17at15_00_45_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at15_00_45_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      37.80316    37.80316     0.00000
            Theta_k_lower      26.85538    26.85538     0.00000
            Theta_k_upper      52.93702    52.93702     0.00000
                  Theta_H      44.71905    44.71905     0.00000
             s.d. Theta_H       7.66709     7.66709     0.00000
                  Theta_S       3.93947     3.93947     0.00000
             s.d. Theta_S       1.20198     1.20198     0.00000
                 Theta_pi       4.63785     4.63785     0.00000
            s.d. Theta_pi       2.53208     2.53208     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at15_00_45_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.92188     4.92188     0.00000
              Tau qt 2.5%       4.03711     4.03711     0.00000
                Tau qt 5%       4.03711     4.03711     0.00000
               Tau qt 95%       5.44141     5.44141     0.00000
             Tau qt 97.5%       5.44141     5.44141     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.02109     0.02109     0.00000
          Theta0 qt 97.5%       0.02109     0.02109     0.00000
                   Theta1     268.43750   268.43750     0.00000
           Theta1 qt 2.5%      87.18750    87.18750     0.00000
             Theta1 qt 5%      87.18750    87.18750     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00128     0.00128     0.00000
      Model (SSD) p-value       0.20000     0.20000     0.00000
         Raggedness index       0.01567     0.01567     0.00000
       Raggedness p-value       0.40000     0.40000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 373 (31/01/17 at 15:00:45))
Total computing time for this run : 0h 0m 0s 143 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

