
<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_35_52"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 95 (31/01/17 at 16:35:52)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_35_52_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_35_52_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 62
No. of loci              : 22
No. of usable loci       : 22 loci with less than  5.00 % missing data
No. of polymorphic sites : 22


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.27964
     3       150         2   0.43320
     4       150         2   0.01333
     5       150         2   0.17038
     6       150         2   0.21262
     7       150         2   0.29718
     8       150         2   0.01333
     9       150         2   0.07732
    10       150         2   0.15937
    11       150         2   0.44743
    12       150         2   0.01333
    13       150         2   0.11356
    14       150         2   0.22273
    15       150         2   0.25199
    16       150         2   0.01333
    17       150         2   0.30568
    18       150         2   0.36725
    19       150         2   0.06488
    20       150         2   0.38738
    21       150         2   0.32215
    22       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.19058
  s.d.     0.000     0.000   0.15030
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0327
Gene diversity           :  0.9738 +/-  0.0045

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 62

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 22
Number of usable nucleotide sites            : 22


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.192752 +/-     2.094649

Average gene diversity over loci              :     0.190580 +/-     0.105390

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    35.345767
  S.D. Theta(Hom) :     6.550803

         Theta(k) :    39.066381
95 % confidence interval limits for theta(k) : [  27.786208,  54.655947 ]

         Theta(S) :     3.939466
    S.D. Theta(S) :     1.201985

        Theta(Pi) :     4.192752
   S.D. Theta(Pi) :     2.318574

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     293       0.000
       1     549       0.000
       2    1406       0.000
       3    1766       0.000
       4    2195       0.000
       5    2207       0.000
       6    1497       0.000
       7     844       0.000
       8     300       0.000
       9      90       0.000
      10      28       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.193
Mismatch observed variance          :        3.620
Tau                                 :        4.430
Theta0                              :        0.000
Theta1                              :     1332.500

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00193925
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.01912981
        P(Sim. Rag. >= Obs. Rag.): 0.30000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        293.0        114.8        141.0
          1.0        549.0        540.5        595.8
          2.0       1406.0       1267.0       1309.2
          3.0       1766.0       1914.7       1928.4
          4.0       2195.0       2143.9       2133.4
          5.0       2207.0       1897.0       1889.1
          6.0       1497.0       1397.4       1394.2
          7.0        844.0        921.3        882.1
          8.0        300.0        533.2        488.4
          9.0         90.0        267.7        240.4
         10.0         28.0        116.9        106.5
         11.0          0.0         42.8         42.9
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.430           3.455           4.719           4.259
                   Theta0:           0.000           0.000           1.071           0.278               
                   Theta1:        1332.500         130.938       99999.000       70100.488
Mean No. of pairwise diff.           4.193           4.161           4.911           4.496
           No. pol. sites:              22             199             225         211.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        293.0         46.0        216.0        141.0
          1.0        549.0        375.0        698.0        595.8
          2.0       1406.0       1062.0       1535.0       1309.2
          3.0       1766.0       1643.0       2258.0       1928.4
          4.0       2195.0       1860.0       2450.0       2133.4
          5.0       2207.0       1707.0       2113.0       1889.1
          6.0       1497.0       1164.0       1546.0       1394.2
          7.0        844.0        659.0       1167.0        882.1
          8.0        300.0        364.0        797.0        488.4
          9.0         90.0        166.0        448.0        240.4
         10.0         28.0         65.0        196.0        106.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.430           3.455           4.719           4.259
                   Theta0:           0.000           0.000           1.071           0.278               
                   Theta1:        1332.500         130.938       99999.000       70100.488
Mean No. of pairwise diff.           4.193           4.161           4.911           4.496
           No. pol. sites:              22             199             225         211.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        293.0         46.0        216.0        141.0
          1.0        549.0        375.0        698.0        595.8
          2.0       1406.0       1062.0       1535.0       1309.2
          3.0       1766.0       1643.0       2258.0       1928.4
          4.0       2195.0       1860.0       2450.0       2133.4
          5.0       2207.0       1707.0       2113.0       1889.1
          6.0       1497.0       1164.0       1546.0       1394.2
          7.0        844.0        659.0       1167.0        882.1
          8.0        300.0        364.0        797.0        488.4
          9.0         90.0        166.0        448.0        240.4
         10.0         28.0         65.0        196.0        106.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.430           3.455           4.719           4.259
                   Theta0:           0.000           0.000           1.071           0.278               
                   Theta1:        1332.500         130.938       99999.000       70100.488
Mean No. of pairwise diff.           4.193           4.161           4.911           4.496
           No. pol. sites:              22             199             225         211.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        293.0         46.0        216.0        141.0
          1.0        549.0        375.0        698.0        595.8
          2.0       1406.0       1062.0       1535.0       1309.2
          3.0       1766.0       1643.0       2258.0       1928.4
          4.0       2195.0       1860.0       2450.0       2133.4
          5.0       2207.0       1707.0       2113.0       1889.1
          6.0       1497.0       1164.0       1546.0       1394.2
          7.0        844.0        659.0       1167.0        882.1
          8.0        300.0        364.0        797.0        488.4
          9.0         90.0        166.0        448.0        240.4
         10.0         28.0         65.0        196.0        106.5
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_35_52_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            22      22.000       0.000
  No. of usable loci            22      22.000       0.000
  No. of polym. loci            22      22.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_35_52_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.27964     0.27964     0.00000     0.27964
     3       0.43320     0.43320     0.00000     0.43320
     4       0.01333     0.01333     0.00000     0.01333
     5       0.17038     0.17038     0.00000     0.17038
     6       0.21262     0.21262     0.00000     0.21262
     7       0.29718     0.29718     0.00000     0.29718
     8       0.01333     0.01333     0.00000     0.01333
     9       0.07732     0.07732     0.00000     0.07732
    10       0.15937     0.15937     0.00000     0.15937
    11       0.44743     0.44743     0.00000     0.44743
    12       0.01333     0.01333     0.00000     0.01333
    13       0.11356     0.11356     0.00000     0.11356
    14       0.22273     0.22273     0.00000     0.22273
    15       0.25199     0.25199     0.00000     0.25199
    16       0.01333     0.01333     0.00000     0.01333
    17       0.30568     0.30568     0.00000     0.30568
    18       0.36725     0.36725     0.00000     0.36725
    19       0.06488     0.06488     0.00000     0.06488
    20       0.38738     0.38738     0.00000     0.38738
    21       0.32215     0.32215     0.00000     0.32215
    22       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.19058     0.19058     0.00000     0.19058
  s.d.       0.15030     0.15030     0.00000     0.15030
--------------------------------------------------------
<A NAME="31_01_17at16_35_52_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_35_52_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      39.06638    39.06638     0.00000
            Theta_k_lower      27.78621    27.78621     0.00000
            Theta_k_upper      54.65595    54.65595     0.00000
                  Theta_H      35.34577    35.34577     0.00000
             s.d. Theta_H       6.55080     6.55080     0.00000
                  Theta_S       3.93947     3.93947     0.00000
             s.d. Theta_S       1.20198     1.20198     0.00000
                 Theta_pi       4.19275     4.19275     0.00000
            s.d. Theta_pi       2.31857     2.31857     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_35_52_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.42969     4.42969     0.00000
              Tau qt 2.5%       3.45508     3.45508     0.00000
                Tau qt 5%       3.45508     3.45508     0.00000
               Tau qt 95%       4.70703     4.70703     0.00000
             Tau qt 97.5%       4.70703     4.70703     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.98789     0.98789     0.00000
          Theta0 qt 97.5%       0.98789     0.98789     0.00000
                   Theta1    1332.50000  1332.50000     0.00000
           Theta1 qt 2.5%     130.93750   130.93750     0.00000
             Theta1 qt 5%     130.93750   130.93750     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00194     0.00194     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.01913     0.01913     0.00000
       Raggedness p-value       0.30000     0.30000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 95 (31/01/17 at 16:35:52))
Total computing time for this run : 0h 0m 0s 197 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

