
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_47_38"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 186 (31/01/17 at 14:47:38)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_47_38_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_47_38_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 48
No. of loci              : 71
No. of usable loci       : 71 loci with less than  5.00 % missing data
No. of polymorphic sites : 71


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.12528
     2       150         2   0.01333
     3       150         2   0.21262
     4       150         2   0.15937
     5       150         2   0.01333
     6       150         2   0.12528
     7       150         2   0.01333
     8       150         2   0.01333
     9       150         2   0.15937
    10       150         2   0.13682
    11       150         2   0.02649
    12       150         2   0.01333
    13       150         2   0.10166
    14       150         2   0.01333
    15       150         2   0.02649
    16       150         2   0.03946
    17       150         2   0.12528
    18       150         2   0.13682
    19       150         2   0.01333
    20       150         2   0.02649
    21       150         2   0.01333
    22       150         2   0.02649
    23       150         2   0.18121
    24       150         2   0.05226
    25       150         2   0.14819
    26       150         2   0.02649
    27       150         2   0.03946
    28       150         2   0.02649
    29       150         2   0.01333
    30       150         2   0.02649
    31       150         2   0.11356
    32       150         2   0.05226
    33       150         2   0.02649
    34       150         2   0.05226
    35       150         2   0.01333
    36       150         2   0.01333
    37       150         2   0.01333
    38       150         2   0.02649
    39       150         2   0.01333
    40       150         2   0.12528
    41       150         2   0.03946
    42       150         2   0.05226
    43       150         2   0.01333
    44       150         2   0.01333
    45       150         2   0.21262
    46       150         2   0.14819
    47       150         2   0.01333
    48       150         2   0.05226
    49       150         2   0.14819
    50       150         2   0.06488
    51       150         2   0.01333
    52       150         2   0.14819
    53       150         2   0.03946
    54       150         2   0.02649
    55       150         2   0.01333
    56       150         2   0.02649
    57       150         2   0.01333
    58       150         2   0.05226
    59       150         2   0.01333
    60       150         2   0.01333
    61       150         2   0.14819
    62       150         2   0.03946
    63       150         2   0.27060
    64       150         2   0.01333
    65       150         2   0.01333
    66       150         2   0.11356
    67       150         2   0.18121
    68       150         2   0.01333
    69       150         2   0.01333
    70       150         2   0.14819
    71       150         2   0.02649
-------------------------------------------------
  Mean   150.000     2.000   0.06474
  s.d.     0.000     0.000   0.06457
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0382
Gene diversity           :  0.9682 +/-  0.0041

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 48

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 71
Number of usable nucleotide sites            : 71


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.596600 +/-     2.269673

Average gene diversity over loci              :     0.064741 +/-     0.035385

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    28.726922
  S.D. Theta(Hom) :     4.026697

         Theta(k) :    24.019287
95 % confidence interval limits for theta(k) : [  16.682825,  34.256022 ]

         Theta(S) :    12.713730
    S.D. Theta(S) :     3.198456

        Theta(Pi) :     4.596600
   S.D. Theta(Pi) :     2.512308

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     355       0.000
       1     300       0.000
       2     422       0.000
       3    1814       0.000
       4    3027       0.000
       5    1703       0.000
       6    1664       0.000
       7    1257       0.000
       8     366       0.000
       9     225       0.000
      10      40       0.000
      11       2       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.597
Mismatch observed variance          :        3.633
Tau                                 :        4.768
Theta0                              :        0.004
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.01370601
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.04961957
        P(Sim. Rag. >= Obs. Rag.): 0.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        355.0         59.5         94.8
          1.0        300.0        394.7        451.7
          2.0        422.0       1076.7       1077.5
          3.0       1814.0       1760.7       1713.5
          4.0       3027.0       2106.4       2043.8
          5.0       1703.0       1964.2       1950.2
          6.0       1664.0       1541.1       1550.8
          7.0       1257.0       1038.7       1057.0
          8.0        366.0        614.9        630.4
          9.0        225.0        341.5        334.2
         10.0         40.0        162.2        159.4
         11.0          2.0         75.1         69.2
         12.0          0.0         25.7         27.5
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.768           4.350           5.006           4.679
                   Theta0:           0.004           0.000           0.450           0.049               
                   Theta1:       99999.000        2261.500       99999.000       90225.250
Mean No. of pairwise diff.           4.597           4.387           5.063           4.790
           No. pol. sites:              71             223             242         231.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        355.0         28.0        106.0         94.8
          1.0        300.0        279.0        572.0        451.7
          2.0        422.0        861.0       1308.0       1077.5
          3.0       1814.0       1485.0       2057.0       1713.5
          4.0       3027.0       1907.0       2310.0       2043.8
          5.0       1703.0       1825.0       2180.0       1950.2
          6.0       1664.0       1352.0       1671.0       1550.8
          7.0       1257.0        820.0       1242.0       1057.0
          8.0        366.0        442.0        773.0        630.4
          9.0        225.0        237.0        455.0        334.2
         10.0         40.0        103.0        245.0        159.4
         11.0          2.0         34.0        134.0         69.2
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.768           4.350           5.006           4.679
                   Theta0:           0.004           0.000           0.450           0.049               
                   Theta1:       99999.000        2261.500       99999.000       90225.250
Mean No. of pairwise diff.           4.597           4.387           5.063           4.790
           No. pol. sites:              71             223             242         231.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        355.0         28.0        106.0         94.8
          1.0        300.0        279.0        572.0        451.7
          2.0        422.0        861.0       1308.0       1077.5
          3.0       1814.0       1485.0       2057.0       1713.5
          4.0       3027.0       1907.0       2310.0       2043.8
          5.0       1703.0       1825.0       2180.0       1950.2
          6.0       1664.0       1352.0       1671.0       1550.8
          7.0       1257.0        820.0       1242.0       1057.0
          8.0        366.0        442.0        773.0        630.4
          9.0        225.0        237.0        455.0        334.2
         10.0         40.0        103.0        245.0        159.4
         11.0          2.0         34.0        134.0         69.2
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.768           4.350           5.006           4.679
                   Theta0:           0.004           0.000           0.450           0.049               
                   Theta1:       99999.000        2261.500       99999.000       90225.250
Mean No. of pairwise diff.           4.597           4.387           5.063           4.790
           No. pol. sites:              71             223             242         231.200
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        355.0         28.0        106.0         94.8
          1.0        300.0        279.0        572.0        451.7
          2.0        422.0        861.0       1308.0       1077.5
          3.0       1814.0       1485.0       2057.0       1713.5
          4.0       3027.0       1907.0       2310.0       2043.8
          5.0       1703.0       1825.0       2180.0       1950.2
          6.0       1664.0       1352.0       1671.0       1550.8
          7.0       1257.0        820.0       1242.0       1057.0
          8.0        366.0        442.0        773.0        630.4
          9.0        225.0        237.0        455.0        334.2
         10.0         40.0        103.0        245.0        159.4
         11.0          2.0         34.0        134.0         69.2
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_47_38_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            71      71.000       0.000
  No. of usable loci            71      71.000       0.000
  No. of polym. loci            71      71.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_47_38_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.12528     0.12528     0.00000     0.12528
     2       0.01333     0.01333     0.00000     0.01333
     3       0.21262     0.21262     0.00000     0.21262
     4       0.15937     0.15937     0.00000     0.15937
     5       0.01333     0.01333     0.00000     0.01333
     6       0.12528     0.12528     0.00000     0.12528
     7       0.01333     0.01333     0.00000     0.01333
     8       0.01333     0.01333     0.00000     0.01333
     9       0.15937     0.15937     0.00000     0.15937
    10       0.13682     0.13682     0.00000     0.13682
    11       0.02649     0.02649     0.00000     0.02649
    12       0.01333     0.01333     0.00000     0.01333
    13       0.10166     0.10166     0.00000     0.10166
    14       0.01333     0.01333     0.00000     0.01333
    15       0.02649     0.02649     0.00000     0.02649
    16       0.03946     0.03946     0.00000     0.03946
    17       0.12528     0.12528     0.00000     0.12528
    18       0.13682     0.13682     0.00000     0.13682
    19       0.01333     0.01333     0.00000     0.01333
    20       0.02649     0.02649     0.00000     0.02649
    21       0.01333     0.01333     0.00000     0.01333
    22       0.02649     0.02649     0.00000     0.02649
    23       0.18121     0.18121     0.00000     0.18121
    24       0.05226     0.05226     0.00000     0.05226
    25       0.14819     0.14819     0.00000     0.14819
    26       0.02649     0.02649     0.00000     0.02649
    27       0.03946     0.03946     0.00000     0.03946
    28       0.02649     0.02649     0.00000     0.02649
    29       0.01333     0.01333     0.00000     0.01333
    30       0.02649     0.02649     0.00000     0.02649
    31       0.11356     0.11356     0.00000     0.11356
    32       0.05226     0.05226     0.00000     0.05226
    33       0.02649     0.02649     0.00000     0.02649
    34       0.05226     0.05226     0.00000     0.05226
    35       0.01333     0.01333     0.00000     0.01333
    36       0.01333     0.01333     0.00000     0.01333
    37       0.01333     0.01333     0.00000     0.01333
    38       0.02649     0.02649     0.00000     0.02649
    39       0.01333     0.01333     0.00000     0.01333
    40       0.12528     0.12528     0.00000     0.12528
    41       0.03946     0.03946     0.00000     0.03946
    42       0.05226     0.05226     0.00000     0.05226
    43       0.01333     0.01333     0.00000     0.01333
    44       0.01333     0.01333     0.00000     0.01333
    45       0.21262     0.21262     0.00000     0.21262
    46       0.14819     0.14819     0.00000     0.14819
    47       0.01333     0.01333     0.00000     0.01333
    48       0.05226     0.05226     0.00000     0.05226
    49       0.14819     0.14819     0.00000     0.14819
    50       0.06488     0.06488     0.00000     0.06488
    51       0.01333     0.01333     0.00000     0.01333
    52       0.14819     0.14819     0.00000     0.14819
    53       0.03946     0.03946     0.00000     0.03946
    54       0.02649     0.02649     0.00000     0.02649
    55       0.01333     0.01333     0.00000     0.01333
    56       0.02649     0.02649     0.00000     0.02649
    57       0.01333     0.01333     0.00000     0.01333
    58       0.05226     0.05226     0.00000     0.05226
    59       0.01333     0.01333     0.00000     0.01333
    60       0.01333     0.01333     0.00000     0.01333
    61       0.14819     0.14819     0.00000     0.14819
    62       0.03946     0.03946     0.00000     0.03946
    63       0.27060     0.27060     0.00000     0.27060
    64       0.01333     0.01333     0.00000     0.01333
    65       0.01333     0.01333     0.00000     0.01333
    66       0.11356     0.11356     0.00000     0.11356
    67       0.18121     0.18121     0.00000     0.18121
    68       0.01333     0.01333     0.00000     0.01333
    69       0.01333     0.01333     0.00000     0.01333
    70       0.14819     0.14819     0.00000     0.14819
    71       0.02649     0.02649     0.00000     0.02649
--------------------------------------------------------
  Mean       0.06474     0.06474     0.00000     0.06474
  s.d.       0.06457     0.06457     0.00000     0.06457
--------------------------------------------------------
<A NAME="31_01_17at14_47_38_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_47_38_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      24.01929    24.01929     0.00000
            Theta_k_lower      16.68282    16.68282     0.00000
            Theta_k_upper      34.25602    34.25602     0.00000
                  Theta_H      28.72692    28.72692     0.00000
             s.d. Theta_H       4.02670     4.02670     0.00000
                  Theta_S      12.71373    12.71373     0.00000
             s.d. Theta_S       3.19846     3.19846     0.00000
                 Theta_pi       4.59660     4.59660     0.00000
            s.d. Theta_pi       2.51231     2.51231     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_47_38_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.76758     4.76758     0.00000
              Tau qt 2.5%       4.34961     4.34961     0.00000
                Tau qt 5%       4.34961     4.34961     0.00000
               Tau qt 95%       4.89648     4.89648     0.00000
             Tau qt 97.5%       4.89648     4.89648     0.00000
                   Theta0       0.00352     0.00352     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.01934     0.01934     0.00000
          Theta0 qt 97.5%       0.01934     0.01934     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%    2261.50000  2261.50000     0.00000
             Theta1 qt 5%    2261.50000  2261.50000     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.01371     0.01371     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.04962     0.04962     0.00000
       Raggedness p-value       0.00000     0.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 186 (31/01/17 at 14:47:38))
Total computing time for this run : 0h 0m 0s 175 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

