
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_20"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 254 (31/01/17 at 14:44:20)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_20_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_20_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 43
No. of loci              : 62
No. of usable loci       : 62 loci with less than  5.00 % missing data
No. of polymorphic sites : 62


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.02649
     2       150         2   0.01333
     3       150         2   0.01333
     4       150         2   0.01333
     5       150         2   0.02649
     6       150         2   0.01333
     7       150         2   0.01333
     8       150         2   0.06488
     9       150         2   0.01333
    10       150         2   0.05226
    11       150         2   0.05226
    12       150         2   0.06488
    13       150         2   0.01333
    14       150         2   0.02649
    15       150         2   0.01333
    16       150         2   0.01333
    17       150         2   0.01333
    18       150         2   0.02649
    19       150         2   0.01333
    20       150         2   0.08957
    21       150         2   0.14819
    22       150         2   0.01333
    23       150         2   0.01333
    24       150         2   0.01333
    25       150         2   0.03946
    26       150         2   0.01333
    27       150         2   0.01333
    28       150         2   0.01333
    29       150         2   0.02649
    30       150         2   0.07732
    31       150         2   0.01333
    32       150         2   0.07732
    33       150         2   0.01333
    34       150         2   0.01333
    35       150         2   0.12528
    36       150         2   0.02649
    37       150         2   0.06488
    38       150         2   0.05226
    39       150         2   0.13682
    40       150         2   0.02649
    41       150         2   0.01333
    42       150         2   0.01333
    43       150         2   0.02649
    44       150         2   0.18121
    45       150         2   0.01333
    46       150         2   0.07732
    47       150         2   0.05226
    48       150         2   0.01333
    49       150         2   0.01333
    50       150         2   0.01333
    51       150         2   0.02649
    52       150         2   0.01333
    53       150         2   0.02649
    54       150         2   0.05226
    55       150         2   0.02649
    56       150         2   0.01333
    57       150         2   0.01333
    58       150         2   0.01333
    59       150         2   0.14819
    60       150         2   0.01333
    61       150         2   0.19186
    62       150         2   0.18121
-------------------------------------------------
  Mean   150.000     2.000   0.04249
  s.d.     0.000     0.000   0.04720
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.1283
Gene diversity           :  0.8776 +/-  0.0229

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 43

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 62
Number of usable nucleotide sites            : 62


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.634362 +/-     1.414946

Average gene diversity over loci              :     0.042490 +/-     0.025261

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     6.002020
  S.D. Theta(Hom) :     1.407646

         Theta(k) :    19.804082
95 % confidence interval limits for theta(k) : [  13.577772,  28.555190 ]

         Theta(S) :    11.102130
    S.D. Theta(S) :     2.835556

        Theta(Pi) :     2.634362
   S.D. Theta(Pi) :     1.566208

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    1368     830.139
       1    1143    2134.001
       2    3258    2763.448
       3    1947    2406.239
       4    2059    1587.010
       5     926     846.997
       6     279     381.751
       7     163     149.784
       8      30      52.362
       9       2      16.617
      10       0       4.864
      11       0       1.331
      12       0       0.345
      13       0       0.086
      14       0       0.021
      15       0       0.005
      16       0       0.001
      17       0       0.000
      18       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.634
Mismatch observed variance          :        2.716
Tau                                 :        2.979
Theta0                              :        0.004
Theta1                              :       19.102

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.01308852
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.06397586
        P(Sim. Rag. >= Obs. Rag.): 0.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       1368.0       1008.4       1016.4
          1.0       1143.0       1907.1       1878.5
          2.0       3258.0       2615.5       2459.3
          3.0       1947.0       2351.4       2324.7
          4.0       2059.0       1623.2       1691.1
          5.0        926.0        962.0        994.7
          6.0        279.0        460.9        490.0
          7.0        163.0        166.5        207.5
          8.0         30.0         66.6         77.0
          9.0          2.0         12.3         25.4
         10.0          0.0          1.1          7.6
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.979           1.963           4.199           2.864
                   Theta0:           0.004           0.000           0.211           0.025               
                   Theta1:          19.102          12.823       99999.000       40044.066
Mean No. of pairwise diff.           2.634           2.095           3.871           2.692
           No. pol. sites:              62              32              62          47.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1368.0        410.0       1525.0       1016.4
          1.0       1143.0        648.0       3025.0       1878.5
          2.0       3258.0       1625.0       3289.0       2459.3
          3.0       1947.0       1464.0       2874.0       2324.7
          4.0       2059.0        958.0       2750.0       1691.1
          5.0        926.0        215.0       2017.0        994.7
          6.0        279.0          8.0       1344.0        490.0
          7.0        163.0          0.0        602.0        207.5
          8.0         30.0          0.0        266.0         77.0
          9.0          2.0          0.0         68.0         25.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.979           1.963           4.199           2.864
                   Theta0:           0.004           0.000           0.211           0.025               
                   Theta1:          19.102          12.823       99999.000       40044.066
Mean No. of pairwise diff.           2.634           2.095           3.871           2.692
           No. pol. sites:              62              32              62          47.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1368.0        410.0       1525.0       1016.4
          1.0       1143.0        648.0       3025.0       1878.5
          2.0       3258.0       1625.0       3289.0       2459.3
          3.0       1947.0       1464.0       2874.0       2324.7
          4.0       2059.0        958.0       2750.0       1691.1
          5.0        926.0        215.0       2017.0        994.7
          6.0        279.0          8.0       1344.0        490.0
          7.0        163.0          0.0        602.0        207.5
          8.0         30.0          0.0        266.0         77.0
          9.0          2.0          0.0         68.0         25.4
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.979           1.963           4.199           2.864
                   Theta0:           0.004           0.000           0.211           0.025               
                   Theta1:          19.102          12.823       99999.000       40044.066
Mean No. of pairwise diff.           2.634           2.095           3.871           2.692
           No. pol. sites:              62              32              62          47.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       1368.0        410.0       1525.0       1016.4
          1.0       1143.0        648.0       3025.0       1878.5
          2.0       3258.0       1625.0       3289.0       2459.3
          3.0       1947.0       1464.0       2874.0       2324.7
          4.0       2059.0        958.0       2750.0       1691.1
          5.0        926.0        215.0       2017.0        994.7
          6.0        279.0          8.0       1344.0        490.0
          7.0        163.0          0.0        602.0        207.5
          8.0         30.0          0.0        266.0         77.0
          9.0          2.0          0.0         68.0         25.4
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_20_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            62      62.000       0.000
  No. of usable loci            62      62.000       0.000
  No. of polym. loci            62      62.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_20_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.02649     0.02649     0.00000     0.02649
     2       0.01333     0.01333     0.00000     0.01333
     3       0.01333     0.01333     0.00000     0.01333
     4       0.01333     0.01333     0.00000     0.01333
     5       0.02649     0.02649     0.00000     0.02649
     6       0.01333     0.01333     0.00000     0.01333
     7       0.01333     0.01333     0.00000     0.01333
     8       0.06488     0.06488     0.00000     0.06488
     9       0.01333     0.01333     0.00000     0.01333
    10       0.05226     0.05226     0.00000     0.05226
    11       0.05226     0.05226     0.00000     0.05226
    12       0.06488     0.06488     0.00000     0.06488
    13       0.01333     0.01333     0.00000     0.01333
    14       0.02649     0.02649     0.00000     0.02649
    15       0.01333     0.01333     0.00000     0.01333
    16       0.01333     0.01333     0.00000     0.01333
    17       0.01333     0.01333     0.00000     0.01333
    18       0.02649     0.02649     0.00000     0.02649
    19       0.01333     0.01333     0.00000     0.01333
    20       0.08957     0.08957     0.00000     0.08957
    21       0.14819     0.14819     0.00000     0.14819
    22       0.01333     0.01333     0.00000     0.01333
    23       0.01333     0.01333     0.00000     0.01333
    24       0.01333     0.01333     0.00000     0.01333
    25       0.03946     0.03946     0.00000     0.03946
    26       0.01333     0.01333     0.00000     0.01333
    27       0.01333     0.01333     0.00000     0.01333
    28       0.01333     0.01333     0.00000     0.01333
    29       0.02649     0.02649     0.00000     0.02649
    30       0.07732     0.07732     0.00000     0.07732
    31       0.01333     0.01333     0.00000     0.01333
    32       0.07732     0.07732     0.00000     0.07732
    33       0.01333     0.01333     0.00000     0.01333
    34       0.01333     0.01333     0.00000     0.01333
    35       0.12528     0.12528     0.00000     0.12528
    36       0.02649     0.02649     0.00000     0.02649
    37       0.06488     0.06488     0.00000     0.06488
    38       0.05226     0.05226     0.00000     0.05226
    39       0.13682     0.13682     0.00000     0.13682
    40       0.02649     0.02649     0.00000     0.02649
    41       0.01333     0.01333     0.00000     0.01333
    42       0.01333     0.01333     0.00000     0.01333
    43       0.02649     0.02649     0.00000     0.02649
    44       0.18121     0.18121     0.00000     0.18121
    45       0.01333     0.01333     0.00000     0.01333
    46       0.07732     0.07732     0.00000     0.07732
    47       0.05226     0.05226     0.00000     0.05226
    48       0.01333     0.01333     0.00000     0.01333
    49       0.01333     0.01333     0.00000     0.01333
    50       0.01333     0.01333     0.00000     0.01333
    51       0.02649     0.02649     0.00000     0.02649
    52       0.01333     0.01333     0.00000     0.01333
    53       0.02649     0.02649     0.00000     0.02649
    54       0.05226     0.05226     0.00000     0.05226
    55       0.02649     0.02649     0.00000     0.02649
    56       0.01333     0.01333     0.00000     0.01333
    57       0.01333     0.01333     0.00000     0.01333
    58       0.01333     0.01333     0.00000     0.01333
    59       0.14819     0.14819     0.00000     0.14819
    60       0.01333     0.01333     0.00000     0.01333
    61       0.19186     0.19186     0.00000     0.19186
    62       0.18121     0.18121     0.00000     0.18121
--------------------------------------------------------
  Mean       0.04249     0.04249     0.00000     0.04249
  s.d.       0.04720     0.04720     0.00000     0.04720
--------------------------------------------------------
<A NAME="31_01_17at14_44_20_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_20_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      19.80408    19.80408     0.00000
            Theta_k_lower      13.57777    13.57777     0.00000
            Theta_k_upper      28.55519    28.55519     0.00000
                  Theta_H       6.00202     6.00202     0.00000
             s.d. Theta_H       1.40765     1.40765     0.00000
                  Theta_S      11.10213    11.10213     0.00000
             s.d. Theta_S       2.83556     2.83556     0.00000
                 Theta_pi       2.63436     2.63436     0.00000
            s.d. Theta_pi       1.56621     1.56621     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_20_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       2.97852     2.97852     0.00000
              Tau qt 2.5%       1.96289     1.96289     0.00000
                Tau qt 5%       1.96289     1.96289     0.00000
               Tau qt 95%       4.05664     4.05664     0.00000
             Tau qt 97.5%       4.05664     4.05664     0.00000
                   Theta0       0.00352     0.00352     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.02109     0.02109     0.00000
          Theta0 qt 97.5%       0.02109     0.02109     0.00000
                   Theta1      19.10156    19.10156     0.00000
           Theta1 qt 2.5%      12.82349    12.82349     0.00000
             Theta1 qt 5%      12.82349    12.82349     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.01309     0.01309     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.06398     0.06398     0.00000
       Raggedness p-value       0.00000     0.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 254 (31/01/17 at 14:44:21))
Total computing time for this run : 0h 0m 0s 118 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

