
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_47_46"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 238 (31/01/17 at 14:47:46)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_47_46_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_47_46_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 55
No. of loci              : 73
No. of usable loci       : 73 loci with less than  5.00 % missing data
No. of polymorphic sites : 73


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.07732
     2       150         2   0.02649
     3       150         2   0.26139
     4       150         2   0.01333
     5       150         2   0.02649
     6       150         2   0.01333
     7       150         2   0.01333
     8       150         2   0.01333
     9       150         2   0.01333
    10       150         2   0.01333
    11       150         2   0.03946
    12       150         2   0.40591
    13       150         2   0.13682
    14       150         2   0.01333
    15       150         2   0.07732
    16       150         2   0.02649
    17       150         2   0.01333
    18       150         2   0.02649
    19       150         2   0.02649
    20       150         2   0.18121
    21       150         2   0.03946
    22       150         2   0.03946
    23       150         2   0.01333
    24       150         2   0.01333
    25       150         2   0.01333
    26       150         2   0.01333
    27       150         2   0.19186
    28       150         2   0.01333
    29       150         2   0.08957
    30       150         2   0.02649
    31       150         2   0.13682
    32       150         2   0.12528
    33       150         2   0.01333
    34       150         2   0.01333
    35       150         2   0.02649
    36       150         2   0.01333
    37       150         2   0.02649
    38       150         2   0.01333
    39       150         2   0.02649
    40       150         2   0.01333
    41       150         2   0.01333
    42       150         2   0.12528
    43       150         2   0.02649
    44       150         2   0.01333
    45       150         2   0.23266
    46       150         2   0.01333
    47       150         2   0.03946
    48       150         2   0.13682
    49       150         2   0.06488
    50       150         2   0.03946
    51       150         2   0.01333
    52       150         2   0.03946
    53       150         2   0.01333
    54       150         2   0.01333
    55       150         2   0.13682
    56       150         2   0.01333
    57       150         2   0.02649
    58       150         2   0.01333
    59       150         2   0.03946
    60       150         2   0.02649
    61       150         2   0.13682
    62       150         2   0.02649
    63       150         2   0.01333
    64       150         2   0.26139
    65       150         2   0.03946
    66       150         2   0.02649
    67       150         2   0.01333
    68       150         2   0.01333
    69       150         2   0.01333
    70       150         2   0.02649
    71       150         2   0.02649
    72       150         2   0.06488
    73       150         2   0.06488
-------------------------------------------------
  Mean   150.000     2.000   0.05544
  s.d.     0.000     0.000   0.07335
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0444
Gene diversity           :  0.9621 +/-  0.0063

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 55

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 73
Number of usable nucleotide sites            : 73


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.047427 +/-     2.031588

Average gene diversity over loci              :     0.055444 +/-     0.030805

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    23.650463
  S.D. Theta(Hom) :     4.321762

         Theta(k) :    30.875075
95 % confidence interval limits for theta(k) : [  21.743737,  43.533340 ]

         Theta(S) :    13.071863
    S.D. Theta(S) :     3.279015

        Theta(Pi) :     4.047427
   S.D. Theta(Pi) :     2.248772

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     424       0.000
       1     593       0.000
       2    1261       0.000
       3    2002       0.000
       4    2516       0.000
       5    1808       0.000
       6    1442       0.000
       7     779       0.000
       8     259       0.000
       9      82       0.000
      10       9       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.047
Mismatch observed variance          :        3.602
Tau                                 :        4.295
Theta0                              :        0.002
Theta1                              :      382.500

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00235609
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.02138034
        P(Sim. Rag. >= Obs. Rag.): 0.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        424.0        164.9        179.2
          1.0        593.0        588.2        673.4
          2.0       1261.0       1203.5       1411.9
          3.0       2002.0       1834.5       2006.9
          4.0       2516.0       2079.4       2148.7
          5.0       1808.0       1904.2       1843.4
          6.0       1442.0       1459.5       1318.8
          7.0        779.0        957.9        809.0
          8.0        259.0        537.9        434.3
          9.0         82.0        264.0        207.3
         10.0          9.0        113.9         89.0
         11.0          0.0         45.5         34.8
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.295           3.514           4.938           4.401
                   Theta0:           0.002           0.000           1.113           0.202               
                   Theta1:         382.500          63.496       99999.000       60045.343
Mean No. of pairwise diff.           4.047           3.882           4.878           4.509
           No. pol. sites:              73             161             204         183.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        424.0         65.0        290.0        179.2
          1.0        593.0        353.0        839.0        673.4
          2.0       1261.0        930.0       1658.0       1411.9
          3.0       2002.0       1501.0       2310.0       2006.9
          4.0       2516.0       1929.0       2295.0       2148.7
          5.0       1808.0       1669.0       2044.0       1843.4
          6.0       1442.0       1069.0       1715.0       1318.8
          7.0        779.0        596.0       1200.0        809.0
          8.0        259.0        279.0        750.0        434.3
          9.0         82.0        111.0        347.0        207.3
         10.0          9.0         37.0        184.0         89.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.295           3.514           4.938           4.401
                   Theta0:           0.002           0.000           1.113           0.202               
                   Theta1:         382.500          63.496       99999.000       60045.343
Mean No. of pairwise diff.           4.047           3.882           4.878           4.509
           No. pol. sites:              73             161             204         183.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        424.0         65.0        290.0        179.2
          1.0        593.0        353.0        839.0        673.4
          2.0       1261.0        930.0       1658.0       1411.9
          3.0       2002.0       1501.0       2310.0       2006.9
          4.0       2516.0       1929.0       2295.0       2148.7
          5.0       1808.0       1669.0       2044.0       1843.4
          6.0       1442.0       1069.0       1715.0       1318.8
          7.0        779.0        596.0       1200.0        809.0
          8.0        259.0        279.0        750.0        434.3
          9.0         82.0        111.0        347.0        207.3
         10.0          9.0         37.0        184.0         89.0
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.295           3.514           4.938           4.401
                   Theta0:           0.002           0.000           1.113           0.202               
                   Theta1:         382.500          63.496       99999.000       60045.343
Mean No. of pairwise diff.           4.047           3.882           4.878           4.509
           No. pol. sites:              73             161             204         183.100
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        424.0         65.0        290.0        179.2
          1.0        593.0        353.0        839.0        673.4
          2.0       1261.0        930.0       1658.0       1411.9
          3.0       2002.0       1501.0       2310.0       2006.9
          4.0       2516.0       1929.0       2295.0       2148.7
          5.0       1808.0       1669.0       2044.0       1843.4
          6.0       1442.0       1069.0       1715.0       1318.8
          7.0        779.0        596.0       1200.0        809.0
          8.0        259.0        279.0        750.0        434.3
          9.0         82.0        111.0        347.0        207.3
         10.0          9.0         37.0        184.0         89.0
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_47_46_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            73      73.000       0.000
  No. of usable loci            73      73.000       0.000
  No. of polym. loci            73      73.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_47_46_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.07732     0.07732     0.00000     0.07732
     2       0.02649     0.02649     0.00000     0.02649
     3       0.26139     0.26139     0.00000     0.26139
     4       0.01333     0.01333     0.00000     0.01333
     5       0.02649     0.02649     0.00000     0.02649
     6       0.01333     0.01333     0.00000     0.01333
     7       0.01333     0.01333     0.00000     0.01333
     8       0.01333     0.01333     0.00000     0.01333
     9       0.01333     0.01333     0.00000     0.01333
    10       0.01333     0.01333     0.00000     0.01333
    11       0.03946     0.03946     0.00000     0.03946
    12       0.40591     0.40591     0.00000     0.40591
    13       0.13682     0.13682     0.00000     0.13682
    14       0.01333     0.01333     0.00000     0.01333
    15       0.07732     0.07732     0.00000     0.07732
    16       0.02649     0.02649     0.00000     0.02649
    17       0.01333     0.01333     0.00000     0.01333
    18       0.02649     0.02649     0.00000     0.02649
    19       0.02649     0.02649     0.00000     0.02649
    20       0.18121     0.18121     0.00000     0.18121
    21       0.03946     0.03946     0.00000     0.03946
    22       0.03946     0.03946     0.00000     0.03946
    23       0.01333     0.01333     0.00000     0.01333
    24       0.01333     0.01333     0.00000     0.01333
    25       0.01333     0.01333     0.00000     0.01333
    26       0.01333     0.01333     0.00000     0.01333
    27       0.19186     0.19186     0.00000     0.19186
    28       0.01333     0.01333     0.00000     0.01333
    29       0.08957     0.08957     0.00000     0.08957
    30       0.02649     0.02649     0.00000     0.02649
    31       0.13682     0.13682     0.00000     0.13682
    32       0.12528     0.12528     0.00000     0.12528
    33       0.01333     0.01333     0.00000     0.01333
    34       0.01333     0.01333     0.00000     0.01333
    35       0.02649     0.02649     0.00000     0.02649
    36       0.01333     0.01333     0.00000     0.01333
    37       0.02649     0.02649     0.00000     0.02649
    38       0.01333     0.01333     0.00000     0.01333
    39       0.02649     0.02649     0.00000     0.02649
    40       0.01333     0.01333     0.00000     0.01333
    41       0.01333     0.01333     0.00000     0.01333
    42       0.12528     0.12528     0.00000     0.12528
    43       0.02649     0.02649     0.00000     0.02649
    44       0.01333     0.01333     0.00000     0.01333
    45       0.23266     0.23266     0.00000     0.23266
    46       0.01333     0.01333     0.00000     0.01333
    47       0.03946     0.03946     0.00000     0.03946
    48       0.13682     0.13682     0.00000     0.13682
    49       0.06488     0.06488     0.00000     0.06488
    50       0.03946     0.03946     0.00000     0.03946
    51       0.01333     0.01333     0.00000     0.01333
    52       0.03946     0.03946     0.00000     0.03946
    53       0.01333     0.01333     0.00000     0.01333
    54       0.01333     0.01333     0.00000     0.01333
    55       0.13682     0.13682     0.00000     0.13682
    56       0.01333     0.01333     0.00000     0.01333
    57       0.02649     0.02649     0.00000     0.02649
    58       0.01333     0.01333     0.00000     0.01333
    59       0.03946     0.03946     0.00000     0.03946
    60       0.02649     0.02649     0.00000     0.02649
    61       0.13682     0.13682     0.00000     0.13682
    62       0.02649     0.02649     0.00000     0.02649
    63       0.01333     0.01333     0.00000     0.01333
    64       0.26139     0.26139     0.00000     0.26139
    65       0.03946     0.03946     0.00000     0.03946
    66       0.02649     0.02649     0.00000     0.02649
    67       0.01333     0.01333     0.00000     0.01333
    68       0.01333     0.01333     0.00000     0.01333
    69       0.01333     0.01333     0.00000     0.01333
    70       0.02649     0.02649     0.00000     0.02649
    71       0.02649     0.02649     0.00000     0.02649
    72       0.06488     0.06488     0.00000     0.06488
    73       0.06488     0.06488     0.00000     0.06488
--------------------------------------------------------
  Mean       0.05544     0.05544     0.00000     0.05544
  s.d.       0.07335     0.07335     0.00000     0.07335
--------------------------------------------------------
<A NAME="31_01_17at14_47_46_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
    72             2       2.000       0.000           2
    73             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_47_46_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      30.87507    30.87507     0.00000
            Theta_k_lower      21.74374    21.74374     0.00000
            Theta_k_upper      43.53334    43.53334     0.00000
                  Theta_H      23.65046    23.65046     0.00000
             s.d. Theta_H       4.32176     4.32176     0.00000
                  Theta_S      13.07186    13.07186     0.00000
             s.d. Theta_S       3.27901     3.27901     0.00000
                 Theta_pi       4.04743     4.04743     0.00000
            s.d. Theta_pi       2.24877     2.24877     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_47_46_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.29492     4.29492     0.00000
              Tau qt 2.5%       3.51367     3.51367     0.00000
                Tau qt 5%       3.51367     3.51367     0.00000
               Tau qt 95%       4.91016     4.91016     0.00000
             Tau qt 97.5%       4.91016     4.91016     0.00000
                   Theta0       0.00176     0.00176     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.67324     0.67324     0.00000
          Theta0 qt 97.5%       0.67324     0.67324     0.00000
                   Theta1     382.50000   382.50000     0.00000
           Theta1 qt 2.5%      63.49609    63.49609     0.00000
             Theta1 qt 5%      63.49609    63.49609     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00236     0.00236     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.02138     0.02138     0.00000
       Raggedness p-value       0.00000     0.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 238 (31/01/17 at 14:47:46))
Total computing time for this run : 0h 0m 0s 169 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

