
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_17"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 222 (31/01/17 at 14:44:17)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_17_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_17_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 49
No. of loci              : 65
No. of usable loci       : 65 loci with less than  5.00 % missing data
No. of polymorphic sites : 65


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.01333
     3       150         2   0.05226
     4       150         2   0.11356
     5       150         2   0.01333
     6       150         2   0.01333
     7       150         2   0.03946
     8       150         2   0.03946
     9       150         2   0.03946
    10       150         2   0.01333
    11       150         2   0.02649
    12       150         2   0.05226
    13       150         2   0.01333
    14       150         2   0.01333
    15       150         2   0.03946
    16       150         2   0.05226
    17       150         2   0.03946
    18       150         2   0.01333
    19       150         2   0.02649
    20       150         2   0.06488
    21       150         2   0.01333
    22       150         2   0.05226
    23       150         2   0.03946
    24       150         2   0.01333
    25       150         2   0.01333
    26       150         2   0.06488
    27       150         2   0.02649
    28       150         2   0.24242
    29       150         2   0.01333
    30       150         2   0.07732
    31       150         2   0.01333
    32       150         2   0.01333
    33       150         2   0.07732
    34       150         2   0.03946
    35       150         2   0.01333
    36       150         2   0.03946
    37       150         2   0.03946
    38       150         2   0.05226
    39       150         2   0.02649
    40       150         2   0.05226
    41       150         2   0.02649
    42       150         2   0.01333
    43       150         2   0.01333
    44       150         2   0.10166
    45       150         2   0.01333
    46       150         2   0.03946
    47       150         2   0.01333
    48       150         2   0.01333
    49       150         2   0.05226
    50       150         2   0.07732
    51       150         2   0.03946
    52       150         2   0.11356
    53       150         2   0.02649
    54       150         2   0.11356
    55       150         2   0.01333
    56       150         2   0.01333
    57       150         2   0.02649
    58       150         2   0.02649
    59       150         2   0.01333
    60       150         2   0.01333
    61       150         2   0.05226
    62       150         2   0.01333
    63       150         2   0.01333
    64       150         2   0.01333
    65       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.03821
  s.d.     0.000     0.000   0.03725
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0759
Gene diversity           :  0.9303 +/-  0.0147

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 49

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 65
Number of usable nucleotide sites            : 65


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.483848 +/-     1.348687

Average gene diversity over loci              :     0.038213 +/-     0.022967

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    11.864113
  S.D. Theta(Hom) :     2.925093

         Theta(k) :    24.926798
95 % confidence interval limits for theta(k) : [  17.352333,  35.483223 ]

         Theta(S) :    11.639330
    S.D. Theta(S) :     2.956600

        Theta(Pi) :     2.483848
   S.D. Theta(Pi) :     1.492866

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     779       0.000
       1    2339       0.000
       2    3009       0.000
       3    2384       0.000
       4    1516       0.000
       5     804       0.000
       6     253       0.000
       7      82       0.000
       8       9       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.484
Mismatch observed variance          :        2.232
Tau                                 :        2.502
Theta0                              :        0.000
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00040216
          P(Sim. Ssd >= Obs. Ssd): 0.50000000
    Harpending's Raggedness index: 0.03901122
        P(Sim. Rag. >= Obs. Rag.): 0.30000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        779.0        886.9        915.6
          1.0       2339.0       2258.0       2290.6
          2.0       3009.0       2895.5       2865.4
          3.0       2384.0       2444.2       2389.7
          4.0       1516.0       1506.9       1494.7
          5.0        804.0        741.7        747.9
          6.0        253.0        310.3        311.9
          7.0         82.0         97.9        111.5
          8.0          9.0         28.1         34.9
          9.0          0.0          5.0          9.7
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.502           2.254           2.615           2.481
                   Theta0:           0.000           0.000           0.037           0.006               
                   Theta1:       99999.000          56.734       99999.000       50327.523
Mean No. of pairwise diff.           2.484           2.290           2.635           2.500
           No. pol. sites:              65             138             158         150.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        779.0        630.0       1176.0        915.6
          1.0       2339.0       2090.0       2602.0       2290.6
          2.0       3009.0       2648.0       3136.0       2865.4
          3.0       2384.0       2223.0       2774.0       2389.7
          4.0       1516.0       1223.0       1702.0       1494.7
          5.0        804.0        602.0        879.0        747.9
          6.0        253.0        200.0        459.0        311.9
          7.0         82.0         43.0        173.0        111.5
          8.0          9.0          3.0         51.0         34.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.502           2.254           2.615           2.481
                   Theta0:           0.000           0.000           0.037           0.006               
                   Theta1:       99999.000          56.734       99999.000       50327.523
Mean No. of pairwise diff.           2.484           2.290           2.635           2.500
           No. pol. sites:              65             138             158         150.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        779.0        630.0       1176.0        915.6
          1.0       2339.0       2090.0       2602.0       2290.6
          2.0       3009.0       2648.0       3136.0       2865.4
          3.0       2384.0       2223.0       2774.0       2389.7
          4.0       1516.0       1223.0       1702.0       1494.7
          5.0        804.0        602.0        879.0        747.9
          6.0        253.0        200.0        459.0        311.9
          7.0         82.0         43.0        173.0        111.5
          8.0          9.0          3.0         51.0         34.9
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.502           2.254           2.615           2.481
                   Theta0:           0.000           0.000           0.037           0.006               
                   Theta1:       99999.000          56.734       99999.000       50327.523
Mean No. of pairwise diff.           2.484           2.290           2.635           2.500
           No. pol. sites:              65             138             158         150.800
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        779.0        630.0       1176.0        915.6
          1.0       2339.0       2090.0       2602.0       2290.6
          2.0       3009.0       2648.0       3136.0       2865.4
          3.0       2384.0       2223.0       2774.0       2389.7
          4.0       1516.0       1223.0       1702.0       1494.7
          5.0        804.0        602.0        879.0        747.9
          6.0        253.0        200.0        459.0        311.9
          7.0         82.0         43.0        173.0        111.5
          8.0          9.0          3.0         51.0         34.9
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_17_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            65      65.000       0.000
  No. of usable loci            65      65.000       0.000
  No. of polym. loci            65      65.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_17_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.01333     0.01333     0.00000     0.01333
     3       0.05226     0.05226     0.00000     0.05226
     4       0.11356     0.11356     0.00000     0.11356
     5       0.01333     0.01333     0.00000     0.01333
     6       0.01333     0.01333     0.00000     0.01333
     7       0.03946     0.03946     0.00000     0.03946
     8       0.03946     0.03946     0.00000     0.03946
     9       0.03946     0.03946     0.00000     0.03946
    10       0.01333     0.01333     0.00000     0.01333
    11       0.02649     0.02649     0.00000     0.02649
    12       0.05226     0.05226     0.00000     0.05226
    13       0.01333     0.01333     0.00000     0.01333
    14       0.01333     0.01333     0.00000     0.01333
    15       0.03946     0.03946     0.00000     0.03946
    16       0.05226     0.05226     0.00000     0.05226
    17       0.03946     0.03946     0.00000     0.03946
    18       0.01333     0.01333     0.00000     0.01333
    19       0.02649     0.02649     0.00000     0.02649
    20       0.06488     0.06488     0.00000     0.06488
    21       0.01333     0.01333     0.00000     0.01333
    22       0.05226     0.05226     0.00000     0.05226
    23       0.03946     0.03946     0.00000     0.03946
    24       0.01333     0.01333     0.00000     0.01333
    25       0.01333     0.01333     0.00000     0.01333
    26       0.06488     0.06488     0.00000     0.06488
    27       0.02649     0.02649     0.00000     0.02649
    28       0.24242     0.24242     0.00000     0.24242
    29       0.01333     0.01333     0.00000     0.01333
    30       0.07732     0.07732     0.00000     0.07732
    31       0.01333     0.01333     0.00000     0.01333
    32       0.01333     0.01333     0.00000     0.01333
    33       0.07732     0.07732     0.00000     0.07732
    34       0.03946     0.03946     0.00000     0.03946
    35       0.01333     0.01333     0.00000     0.01333
    36       0.03946     0.03946     0.00000     0.03946
    37       0.03946     0.03946     0.00000     0.03946
    38       0.05226     0.05226     0.00000     0.05226
    39       0.02649     0.02649     0.00000     0.02649
    40       0.05226     0.05226     0.00000     0.05226
    41       0.02649     0.02649     0.00000     0.02649
    42       0.01333     0.01333     0.00000     0.01333
    43       0.01333     0.01333     0.00000     0.01333
    44       0.10166     0.10166     0.00000     0.10166
    45       0.01333     0.01333     0.00000     0.01333
    46       0.03946     0.03946     0.00000     0.03946
    47       0.01333     0.01333     0.00000     0.01333
    48       0.01333     0.01333     0.00000     0.01333
    49       0.05226     0.05226     0.00000     0.05226
    50       0.07732     0.07732     0.00000     0.07732
    51       0.03946     0.03946     0.00000     0.03946
    52       0.11356     0.11356     0.00000     0.11356
    53       0.02649     0.02649     0.00000     0.02649
    54       0.11356     0.11356     0.00000     0.11356
    55       0.01333     0.01333     0.00000     0.01333
    56       0.01333     0.01333     0.00000     0.01333
    57       0.02649     0.02649     0.00000     0.02649
    58       0.02649     0.02649     0.00000     0.02649
    59       0.01333     0.01333     0.00000     0.01333
    60       0.01333     0.01333     0.00000     0.01333
    61       0.05226     0.05226     0.00000     0.05226
    62       0.01333     0.01333     0.00000     0.01333
    63       0.01333     0.01333     0.00000     0.01333
    64       0.01333     0.01333     0.00000     0.01333
    65       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.03821     0.03821     0.00000     0.03821
  s.d.       0.03725     0.03725     0.00000     0.03725
--------------------------------------------------------
<A NAME="31_01_17at14_44_17_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_17_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      24.92680    24.92680     0.00000
            Theta_k_lower      17.35233    17.35233     0.00000
            Theta_k_upper      35.48322    35.48322     0.00000
                  Theta_H      11.86411    11.86411     0.00000
             s.d. Theta_H       2.92509     2.92509     0.00000
                  Theta_S      11.63933    11.63933     0.00000
             s.d. Theta_S       2.95660     2.95660     0.00000
                 Theta_pi       2.48385     2.48385     0.00000
            s.d. Theta_pi       1.49287     1.49287     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_17_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       2.50195     2.50195     0.00000
              Tau qt 2.5%       2.25391     2.25391     0.00000
                Tau qt 5%       2.25391     2.25391     0.00000
               Tau qt 95%       2.58789     2.58789     0.00000
             Tau qt 97.5%       2.58789     2.58789     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.02461     0.02461     0.00000
          Theta0 qt 97.5%       0.02461     0.02461     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      56.73438    56.73438     0.00000
             Theta1 qt 5%      56.73438    56.73438     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00040     0.00040     0.00000
      Model (SSD) p-value       0.50000     0.50000     0.00000
         Raggedness index       0.03901     0.03901     0.00000
       Raggedness p-value       0.30000     0.30000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 222 (31/01/17 at 14:44:17))
Total computing time for this run : 0h 0m 0s 95 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

