
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at16_31_33"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 19 (31/01/17 at 16:31:33)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at16_31_33_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at16_31_33_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 60
No. of loci              : 22
No. of usable loci       : 22 loci with less than  5.00 % missing data
No. of polymorphic sites : 22


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.22273
     3       150         2   0.27060
     4       150         2   0.02649
     5       150         2   0.03946
     6       150         2   0.25199
     7       150         2   0.18121
     8       150         2   0.01333
     9       150         2   0.08957
    10       150         2   0.19186
    11       150         2   0.01333
    12       150         2   0.02649
    13       150         2   0.24242
    14       150         2   0.34550
    15       150         2   0.05226
    16       150         2   0.15937
    17       150         2   0.18121
    18       150         2   0.07732
    19       150         2   0.08957
    20       150         2   0.36018
    21       150         2   0.36725
    22       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.14676
  s.d.     0.000     0.000   0.12179
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0397
Gene diversity           :  0.9667 +/-  0.0068

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 60

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 22
Number of usable nucleotide sites            : 22


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.228814 +/-     1.675316

Average gene diversity over loci              :     0.146764 +/-     0.084291

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    27.299845
  S.D. Theta(Hom) :     6.120810

         Theta(k) :    36.572596
95 % confidence interval limits for theta(k) : [  25.948187,  51.263916 ]

         Theta(S) :     3.939466
    S.D. Theta(S) :     1.201985

        Theta(Pi) :     3.228814
   S.D. Theta(Pi) :     1.854413

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     372       0.000
       1    1102       0.000
       2    2171       0.000
       3    2859       0.000
       4    2349       0.000
       5    1491       0.000
       6     651       0.000
       7     142       0.000
       8      32       0.000
       9       6       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.229
Mismatch observed variance          :        2.399
Tau                                 :        3.381
Theta0                              :        0.000
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00307723
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.03301362
        P(Sim. Rag. >= Obs. Rag.): 0.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        372.0        406.8        380.2
          1.0       1102.0       1283.3       1285.3
          2.0       2171.0       2187.1       2172.6
          3.0       2859.0       2446.2       2448.4
          4.0       2349.0       2060.8       2069.4
          5.0       1491.0       1379.0       1399.3
          6.0        651.0        776.7        788.5
          7.0        142.0        381.6        380.8
          8.0         32.0        164.8        160.9
          9.0          6.0         62.7         60.5
         10.0          0.0         19.4         20.4
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.381           2.748           3.742           3.369
                   Theta0:           0.000           0.000           0.076           0.017               
                   Theta1:       99999.000          39.078       99999.000       70021.102
Mean No. of pairwise diff.           3.229           2.861           3.552           3.366
           No. pol. sites:              22             163             198         187.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        372.0        231.0        596.0        380.2
          1.0       1102.0       1055.0       1897.0       1285.3
          2.0       2171.0       1862.0       2660.0       2172.6
          3.0       2859.0       2106.0       2604.0       2448.4
          4.0       2349.0       1768.0       2276.0       2069.4
          5.0       1491.0        999.0       1593.0       1399.3
          6.0        651.0        429.0        984.0        788.5
          7.0        142.0        186.0        496.0        380.8
          8.0         32.0         88.0        221.0        160.9
          9.0          6.0         25.0         89.0         60.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.381           2.748           3.742           3.369
                   Theta0:           0.000           0.000           0.076           0.017               
                   Theta1:       99999.000          39.078       99999.000       70021.102
Mean No. of pairwise diff.           3.229           2.861           3.552           3.366
           No. pol. sites:              22             163             198         187.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        372.0        231.0        596.0        380.2
          1.0       1102.0       1055.0       1897.0       1285.3
          2.0       2171.0       1862.0       2660.0       2172.6
          3.0       2859.0       2106.0       2604.0       2448.4
          4.0       2349.0       1768.0       2276.0       2069.4
          5.0       1491.0        999.0       1593.0       1399.3
          6.0        651.0        429.0        984.0        788.5
          7.0        142.0        186.0        496.0        380.8
          8.0         32.0         88.0        221.0        160.9
          9.0          6.0         25.0         89.0         60.5
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.381           2.748           3.742           3.369
                   Theta0:           0.000           0.000           0.076           0.017               
                   Theta1:       99999.000          39.078       99999.000       70021.102
Mean No. of pairwise diff.           3.229           2.861           3.552           3.366
           No. pol. sites:              22             163             198         187.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        372.0        231.0        596.0        380.2
          1.0       1102.0       1055.0       1897.0       1285.3
          2.0       2171.0       1862.0       2660.0       2172.6
          3.0       2859.0       2106.0       2604.0       2448.4
          4.0       2349.0       1768.0       2276.0       2069.4
          5.0       1491.0        999.0       1593.0       1399.3
          6.0        651.0        429.0        984.0        788.5
          7.0        142.0        186.0        496.0        380.8
          8.0         32.0         88.0        221.0        160.9
          9.0          6.0         25.0         89.0         60.5
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at16_31_33_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            22      22.000       0.000
  No. of usable loci            22      22.000       0.000
  No. of polym. loci            22      22.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at16_31_33_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.22273     0.22273     0.00000     0.22273
     3       0.27060     0.27060     0.00000     0.27060
     4       0.02649     0.02649     0.00000     0.02649
     5       0.03946     0.03946     0.00000     0.03946
     6       0.25199     0.25199     0.00000     0.25199
     7       0.18121     0.18121     0.00000     0.18121
     8       0.01333     0.01333     0.00000     0.01333
     9       0.08957     0.08957     0.00000     0.08957
    10       0.19186     0.19186     0.00000     0.19186
    11       0.01333     0.01333     0.00000     0.01333
    12       0.02649     0.02649     0.00000     0.02649
    13       0.24242     0.24242     0.00000     0.24242
    14       0.34550     0.34550     0.00000     0.34550
    15       0.05226     0.05226     0.00000     0.05226
    16       0.15937     0.15937     0.00000     0.15937
    17       0.18121     0.18121     0.00000     0.18121
    18       0.07732     0.07732     0.00000     0.07732
    19       0.08957     0.08957     0.00000     0.08957
    20       0.36018     0.36018     0.00000     0.36018
    21       0.36725     0.36725     0.00000     0.36725
    22       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.14676     0.14676     0.00000     0.14676
  s.d.       0.12179     0.12179     0.00000     0.12179
--------------------------------------------------------
<A NAME="31_01_17at16_31_33_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at16_31_33_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      36.57260    36.57260     0.00000
            Theta_k_lower      25.94819    25.94819     0.00000
            Theta_k_upper      51.26392    51.26392     0.00000
                  Theta_H      27.29984    27.29984     0.00000
             s.d. Theta_H       6.12081     6.12081     0.00000
                  Theta_S       3.93947     3.93947     0.00000
             s.d. Theta_S       1.20198     1.20198     0.00000
                 Theta_pi       3.22881     3.22881     0.00000
            s.d. Theta_pi       1.85441     1.85441     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at16_31_33_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.38086     3.38086     0.00000
              Tau qt 2.5%       2.74805     2.74805     0.00000
                Tau qt 5%       2.74805     2.74805     0.00000
               Tau qt 95%       3.62500     3.62500     0.00000
             Tau qt 97.5%       3.62500     3.62500     0.00000
                   Theta0       0.00000     0.00000     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.02988     0.02988     0.00000
          Theta0 qt 97.5%       0.02988     0.02988     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      39.07812    39.07812     0.00000
             Theta1 qt 5%      39.07812    39.07812     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00308     0.00308     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.03301     0.03301     0.00000
       Raggedness p-value       0.00000     0.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 19 (31/01/17 at 16:31:34))
Total computing time for this run : 0h 0m 0s 130 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

