
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_52_23"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 325 (31/01/17 at 14:52:23)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_52_23_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_52_23_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 47
No. of loci              : 18
No. of usable loci       : 18 loci with less than  5.00 % missing data
No. of polymorphic sites : 18


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.08957
     2       150         2   0.36018
     3       150         2   0.48823
     4       150         2   0.21262
     5       150         2   0.23266
     6       150         2   0.30568
     7       150         2   0.13682
     8       150         2   0.25199
     9       150         2   0.33790
    10       150         2   0.01333
    11       150         2   0.02649
    12       150         2   0.02649
    13       150         2   0.50112
    14       150         2   0.33011
    15       150         2   0.32215
    16       150         2   0.10166
    17       150         2   0.11356
    18       150         2   0.40591
-------------------------------------------------
  Mean   150.000     2.000   0.23647
  s.d.     0.000     0.000   0.15572
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0423
Gene diversity           :  0.9641 +/-  0.0050

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 47

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 18
Number of usable nucleotide sites            : 18


Distance method : Pairwise difference
Mean number of pairwise differences           :     4.256465 +/-     2.122282

Average gene diversity over loci              :     0.236470 +/-     0.130509

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    25.146819
  S.D. Theta(Hom) :     3.827886

         Theta(k) :    23.134022
95 % confidence interval limits for theta(k) : [  16.029990,  33.058962 ]

         Theta(S) :     3.223199
    S.D. Theta(S) :     1.032890

        Theta(Pi) :     4.256465
   S.D. Theta(Pi) :     2.349161

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     401       0.000
       1     394       0.000
       2    1403       0.000
       3    1635       0.000
       4    2161       0.000
       5    2162       0.000
       6    1614       0.000
       7     997       0.000
       8     330       0.000
       9      76       0.000
      10       2       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        4.256
Mismatch observed variance          :        3.698
Tau                                 :        4.584
Theta0                              :        0.004
Theta1                              :      196.875

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00289608
          P(Sim. Ssd >= Obs. Ssd): 0.00000000
    Harpending's Raggedness index: 0.02037553
        P(Sim. Rag. >= Obs. Rag.): 0.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        401.0        215.7        167.0
          1.0        394.0        640.8        562.8
          2.0       1403.0       1108.5       1216.2
          3.0       1635.0       1701.6       1825.7
          4.0       2161.0       2020.6       2077.7
          5.0       2162.0       1902.9       1898.9
          6.0       1614.0       1488.7       1448.6
          7.0        997.0       1010.6        948.0
          8.0        330.0        561.6        543.1
          9.0         76.0        305.8        276.6
         10.0          2.0        150.3        126.8
         11.0          0.0         50.2         52.9
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.584           3.104           5.551           4.475
                   Theta0:           0.004           0.000           1.450           0.327               
                   Theta1:         196.875          18.687       99999.000       60016.169
Mean No. of pairwise diff.           4.256           3.952           5.458           4.571
           No. pol. sites:              18             142             169         156.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        401.0         72.0        495.0        167.0
          1.0        394.0        245.0       1152.0        562.8
          2.0       1403.0        616.0       1656.0       1216.2
          3.0       1635.0       1235.0       2038.0       1825.7
          4.0       2161.0       1483.0       2312.0       2077.7
          5.0       2162.0       1302.0       2178.0       1898.9
          6.0       1614.0       1065.0       1783.0       1448.6
          7.0        997.0        686.0       1456.0        948.0
          8.0        330.0        351.0       1008.0        543.1
          9.0         76.0        148.0        622.0        276.6
         10.0          2.0         37.0        309.0        126.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.584           3.104           5.551           4.475
                   Theta0:           0.004           0.000           1.450           0.327               
                   Theta1:         196.875          18.687       99999.000       60016.169
Mean No. of pairwise diff.           4.256           3.952           5.458           4.571
           No. pol. sites:              18             142             169         156.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        401.0         72.0        495.0        167.0
          1.0        394.0        245.0       1152.0        562.8
          2.0       1403.0        616.0       1656.0       1216.2
          3.0       1635.0       1235.0       2038.0       1825.7
          4.0       2161.0       1483.0       2312.0       2077.7
          5.0       2162.0       1302.0       2178.0       1898.9
          6.0       1614.0       1065.0       1783.0       1448.6
          7.0        997.0        686.0       1456.0        948.0
          8.0        330.0        351.0       1008.0        543.1
          9.0         76.0        148.0        622.0        276.6
         10.0          2.0         37.0        309.0        126.8
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.584           3.104           5.551           4.475
                   Theta0:           0.004           0.000           1.450           0.327               
                   Theta1:         196.875          18.687       99999.000       60016.169
Mean No. of pairwise diff.           4.256           3.952           5.458           4.571
           No. pol. sites:              18             142             169         156.700
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        401.0         72.0        495.0        167.0
          1.0        394.0        245.0       1152.0        562.8
          2.0       1403.0        616.0       1656.0       1216.2
          3.0       1635.0       1235.0       2038.0       1825.7
          4.0       2161.0       1483.0       2312.0       2077.7
          5.0       2162.0       1302.0       2178.0       1898.9
          6.0       1614.0       1065.0       1783.0       1448.6
          7.0        997.0        686.0       1456.0        948.0
          8.0        330.0        351.0       1008.0        543.1
          9.0         76.0        148.0        622.0        276.6
         10.0          2.0         37.0        309.0        126.8
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_52_23_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            18      18.000       0.000
  No. of usable loci            18      18.000       0.000
  No. of polym. loci            18      18.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_52_23_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.08957     0.08957     0.00000     0.08957
     2       0.36018     0.36018     0.00000     0.36018
     3       0.48823     0.48823     0.00000     0.48823
     4       0.21262     0.21262     0.00000     0.21262
     5       0.23266     0.23266     0.00000     0.23266
     6       0.30568     0.30568     0.00000     0.30568
     7       0.13682     0.13682     0.00000     0.13682
     8       0.25199     0.25199     0.00000     0.25199
     9       0.33790     0.33790     0.00000     0.33790
    10       0.01333     0.01333     0.00000     0.01333
    11       0.02649     0.02649     0.00000     0.02649
    12       0.02649     0.02649     0.00000     0.02649
    13       0.50112     0.50112     0.00000     0.50112
    14       0.33011     0.33011     0.00000     0.33011
    15       0.32215     0.32215     0.00000     0.32215
    16       0.10166     0.10166     0.00000     0.10166
    17       0.11356     0.11356     0.00000     0.11356
    18       0.40591     0.40591     0.00000     0.40591
--------------------------------------------------------
  Mean       0.23647     0.23647     0.00000     0.23647
  s.d.       0.15572     0.15572     0.00000     0.15572
--------------------------------------------------------
<A NAME="31_01_17at14_52_23_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_52_23_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      23.13402    23.13402     0.00000
            Theta_k_lower      16.02999    16.02999     0.00000
            Theta_k_upper      33.05896    33.05896     0.00000
                  Theta_H      25.14682    25.14682     0.00000
             s.d. Theta_H       3.82789     3.82789     0.00000
                  Theta_S       3.22320     3.22320     0.00000
             s.d. Theta_S       1.03289     1.03289     0.00000
                 Theta_pi       4.25647     4.25647     0.00000
            s.d. Theta_pi       2.34916     2.34916     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_52_23_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.58398     4.58398     0.00000
              Tau qt 2.5%       3.10352     3.10352     0.00000
                Tau qt 5%       3.10352     3.10352     0.00000
               Tau qt 95%       4.95312     4.95312     0.00000
             Tau qt 97.5%       4.95312     4.95312     0.00000
                   Theta0       0.00352     0.00352     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       1.06699     1.06699     0.00000
          Theta0 qt 97.5%       1.06699     1.06699     0.00000
                   Theta1     196.87500   196.87500     0.00000
           Theta1 qt 2.5%      18.68652    18.68652     0.00000
             Theta1 qt 5%      18.68652    18.68652     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00290     0.00290     0.00000
      Model (SSD) p-value       0.00000     0.00000     0.00000
         Raggedness index       0.02038     0.02038     0.00000
       Raggedness p-value       0.00000     0.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 325 (31/01/17 at 14:52:23))
Total computing time for this run : 0h 0m 0s 141 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

