
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_37"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 366 (31/01/17 at 14:44:37)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_37_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_37_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 53
No. of loci              : 73
No. of usable loci       : 73 loci with less than  5.00 % missing data
No. of polymorphic sites : 73


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.19186
     2       150         2   0.03946
     3       150         2   0.03946
     4       150         2   0.03946
     5       150         2   0.01333
     6       150         2   0.02649
     7       150         2   0.13682
     8       150         2   0.05226
     9       150         2   0.02649
    10       150         2   0.02649
    11       150         2   0.03946
    12       150         2   0.01333
    13       150         2   0.05226
    14       150         2   0.01333
    15       150         2   0.01333
    16       150         2   0.03946
    17       150         2   0.15937
    18       150         2   0.01333
    19       150         2   0.05226
    20       150         2   0.02649
    21       150         2   0.06488
    22       150         2   0.02649
    23       150         2   0.01333
    24       150         2   0.02649
    25       150         2   0.01333
    26       150         2   0.08957
    27       150         2   0.01333
    28       150         2   0.02649
    29       150         2   0.01333
    30       150         2   0.11356
    31       150         2   0.11356
    32       150         2   0.01333
    33       150         2   0.01333
    34       150         2   0.02649
    35       150         2   0.06488
    36       150         2   0.10166
    37       150         2   0.02649
    38       150         2   0.01333
    39       150         2   0.02649
    40       150         2   0.06488
    41       150         2   0.10166
    42       150         2   0.03946
    43       150         2   0.08957
    44       150         2   0.02649
    45       150         2   0.01333
    46       150         2   0.08957
    47       150         2   0.01333
    48       150         2   0.01333
    49       150         2   0.02649
    50       150         2   0.01333
    51       150         2   0.01333
    52       150         2   0.03946
    53       150         2   0.01333
    54       150         2   0.11356
    55       150         2   0.02649
    56       150         2   0.01333
    57       150         2   0.03946
    58       150         2   0.01333
    59       150         2   0.02649
    60       150         2   0.10166
    61       150         2   0.01333
    62       150         2   0.10166
    63       150         2   0.01333
    64       150         2   0.06488
    65       150         2   0.01333
    66       150         2   0.02649
    67       150         2   0.01333
    68       150         2   0.08957
    69       150         2   0.03946
    70       150         2   0.01333
    71       150         2   0.03946
    72       150         2   0.06488
    73       150         2   0.02649
-------------------------------------------------
  Mean   150.000     2.000   0.04420
  s.d.     0.000     0.000   0.03925
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0810
Gene diversity           :  0.9252 +/-  0.0167

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 53

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 73
Number of usable nucleotide sites            : 73


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.226577 +/-     1.674339

Average gene diversity over loci              :     0.044200 +/-     0.025388

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    10.919707
  S.D. Theta(Hom) :     2.871033

         Theta(k) :    28.791891
95 % confidence interval limits for theta(k) : [  20.205559,  40.712272 ]

         Theta(S) :    13.071863
    S.D. Theta(S) :     3.279015

        Theta(Pi) :     3.226577
   S.D. Theta(Pi) :     1.853332

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     836     531.002
       1    1321    1551.403
       2    2317    2313.254
       3    1898    2357.389
       4    2018    1857.040
       5    1177    1213.912
       6    1031     690.924
       7     402     354.994
       8     128     169.415
       9      43      76.829
      10       4      33.701
      11       0      14.487
      12       0       6.157
      13       0       2.601
      14       0       1.096
      15       0       0.461
      16       0       0.194
      17       0       0.082
      18       0       0.034
      19       0       0.014
      20       0       0.006


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.227
Mismatch observed variance          :        3.753
Tau                                 :        2.973
Theta0                              :        0.612
Theta1                              :       20.684

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00229910
          P(Sim. Ssd >= Obs. Ssd): 0.40000000
    Harpending's Raggedness index: 0.02102230
        P(Sim. Rag. >= Obs. Rag.): 0.40000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        836.0        916.9        799.8
          1.0       1321.0       1518.4       1432.0
          2.0       2317.0       2007.8       2031.1
          3.0       1898.0       2164.3       2178.4
          4.0       2018.0       1813.0       1849.6
          5.0       1177.0       1367.0       1302.4
          6.0       1031.0        741.5        789.4
          7.0        402.0        398.8        424.2
          8.0        128.0        150.9        207.1
          9.0         43.0         58.3         93.8
         10.0          4.0         25.5         40.1
         11.0          0.0          9.9         16.4
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.973           1.688           5.859           3.370
                   Theta0:           0.612           0.000           1.308           0.316               
                   Theta1:          20.684           7.757       99999.000       20157.839
Mean No. of pairwise diff.           3.227           2.448           4.058           3.175
           No. pol. sites:              73              36              60          48.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        836.0        510.0       1790.0        799.8
          1.0       1321.0        737.0       2093.0       1432.0
          2.0       2317.0        854.0       2741.0       2031.1
          3.0       1898.0       1047.0       2988.0       2178.4
          4.0       2018.0       1373.0       2130.0       1849.6
          5.0       1177.0        740.0       2065.0       1302.4
          6.0       1031.0        200.0       1514.0        789.4
          7.0        402.0         63.0        997.0        424.2
          8.0        128.0          8.0        409.0        207.1
          9.0         43.0          0.0        252.0         93.8
         10.0          4.0          0.0        139.0         40.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.973           1.688           5.859           3.370
                   Theta0:           0.612           0.000           1.308           0.316               
                   Theta1:          20.684           7.757       99999.000       20157.839
Mean No. of pairwise diff.           3.227           2.448           4.058           3.175
           No. pol. sites:              73              36              60          48.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        836.0        510.0       1790.0        799.8
          1.0       1321.0        737.0       2093.0       1432.0
          2.0       2317.0        854.0       2741.0       2031.1
          3.0       1898.0       1047.0       2988.0       2178.4
          4.0       2018.0       1373.0       2130.0       1849.6
          5.0       1177.0        740.0       2065.0       1302.4
          6.0       1031.0        200.0       1514.0        789.4
          7.0        402.0         63.0        997.0        424.2
          8.0        128.0          8.0        409.0        207.1
          9.0         43.0          0.0        252.0         93.8
         10.0          4.0          0.0        139.0         40.1
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           2.973           1.688           5.859           3.370
                   Theta0:           0.612           0.000           1.308           0.316               
                   Theta1:          20.684           7.757       99999.000       20157.839
Mean No. of pairwise diff.           3.227           2.448           4.058           3.175
           No. pol. sites:              73              36              60          48.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        836.0        510.0       1790.0        799.8
          1.0       1321.0        737.0       2093.0       1432.0
          2.0       2317.0        854.0       2741.0       2031.1
          3.0       1898.0       1047.0       2988.0       2178.4
          4.0       2018.0       1373.0       2130.0       1849.6
          5.0       1177.0        740.0       2065.0       1302.4
          6.0       1031.0        200.0       1514.0        789.4
          7.0        402.0         63.0        997.0        424.2
          8.0        128.0          8.0        409.0        207.1
          9.0         43.0          0.0        252.0         93.8
         10.0          4.0          0.0        139.0         40.1
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_37_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            73      73.000       0.000
  No. of usable loci            73      73.000       0.000
  No. of polym. loci            73      73.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_37_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.19186     0.19186     0.00000     0.19186
     2       0.03946     0.03946     0.00000     0.03946
     3       0.03946     0.03946     0.00000     0.03946
     4       0.03946     0.03946     0.00000     0.03946
     5       0.01333     0.01333     0.00000     0.01333
     6       0.02649     0.02649     0.00000     0.02649
     7       0.13682     0.13682     0.00000     0.13682
     8       0.05226     0.05226     0.00000     0.05226
     9       0.02649     0.02649     0.00000     0.02649
    10       0.02649     0.02649     0.00000     0.02649
    11       0.03946     0.03946     0.00000     0.03946
    12       0.01333     0.01333     0.00000     0.01333
    13       0.05226     0.05226     0.00000     0.05226
    14       0.01333     0.01333     0.00000     0.01333
    15       0.01333     0.01333     0.00000     0.01333
    16       0.03946     0.03946     0.00000     0.03946
    17       0.15937     0.15937     0.00000     0.15937
    18       0.01333     0.01333     0.00000     0.01333
    19       0.05226     0.05226     0.00000     0.05226
    20       0.02649     0.02649     0.00000     0.02649
    21       0.06488     0.06488     0.00000     0.06488
    22       0.02649     0.02649     0.00000     0.02649
    23       0.01333     0.01333     0.00000     0.01333
    24       0.02649     0.02649     0.00000     0.02649
    25       0.01333     0.01333     0.00000     0.01333
    26       0.08957     0.08957     0.00000     0.08957
    27       0.01333     0.01333     0.00000     0.01333
    28       0.02649     0.02649     0.00000     0.02649
    29       0.01333     0.01333     0.00000     0.01333
    30       0.11356     0.11356     0.00000     0.11356
    31       0.11356     0.11356     0.00000     0.11356
    32       0.01333     0.01333     0.00000     0.01333
    33       0.01333     0.01333     0.00000     0.01333
    34       0.02649     0.02649     0.00000     0.02649
    35       0.06488     0.06488     0.00000     0.06488
    36       0.10166     0.10166     0.00000     0.10166
    37       0.02649     0.02649     0.00000     0.02649
    38       0.01333     0.01333     0.00000     0.01333
    39       0.02649     0.02649     0.00000     0.02649
    40       0.06488     0.06488     0.00000     0.06488
    41       0.10166     0.10166     0.00000     0.10166
    42       0.03946     0.03946     0.00000     0.03946
    43       0.08957     0.08957     0.00000     0.08957
    44       0.02649     0.02649     0.00000     0.02649
    45       0.01333     0.01333     0.00000     0.01333
    46       0.08957     0.08957     0.00000     0.08957
    47       0.01333     0.01333     0.00000     0.01333
    48       0.01333     0.01333     0.00000     0.01333
    49       0.02649     0.02649     0.00000     0.02649
    50       0.01333     0.01333     0.00000     0.01333
    51       0.01333     0.01333     0.00000     0.01333
    52       0.03946     0.03946     0.00000     0.03946
    53       0.01333     0.01333     0.00000     0.01333
    54       0.11356     0.11356     0.00000     0.11356
    55       0.02649     0.02649     0.00000     0.02649
    56       0.01333     0.01333     0.00000     0.01333
    57       0.03946     0.03946     0.00000     0.03946
    58       0.01333     0.01333     0.00000     0.01333
    59       0.02649     0.02649     0.00000     0.02649
    60       0.10166     0.10166     0.00000     0.10166
    61       0.01333     0.01333     0.00000     0.01333
    62       0.10166     0.10166     0.00000     0.10166
    63       0.01333     0.01333     0.00000     0.01333
    64       0.06488     0.06488     0.00000     0.06488
    65       0.01333     0.01333     0.00000     0.01333
    66       0.02649     0.02649     0.00000     0.02649
    67       0.01333     0.01333     0.00000     0.01333
    68       0.08957     0.08957     0.00000     0.08957
    69       0.03946     0.03946     0.00000     0.03946
    70       0.01333     0.01333     0.00000     0.01333
    71       0.03946     0.03946     0.00000     0.03946
    72       0.06488     0.06488     0.00000     0.06488
    73       0.02649     0.02649     0.00000     0.02649
--------------------------------------------------------
  Mean       0.04420     0.04420     0.00000     0.04420
  s.d.       0.03925     0.03925     0.00000     0.03925
--------------------------------------------------------
<A NAME="31_01_17at14_44_37_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
    71             2       2.000       0.000           2
    72             2       2.000       0.000           2
    73             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_37_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      28.79189    28.79189     0.00000
            Theta_k_lower      20.20556    20.20556     0.00000
            Theta_k_upper      40.71227    40.71227     0.00000
                  Theta_H      10.91971    10.91971     0.00000
             s.d. Theta_H       2.87103     2.87103     0.00000
                  Theta_S      13.07186    13.07186     0.00000
             s.d. Theta_S       3.27901     3.27901     0.00000
                 Theta_pi       3.22658     3.22658     0.00000
            s.d. Theta_pi       1.85333     1.85333     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_37_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       2.97266     2.97266     0.00000
              Tau qt 2.5%       1.68750     1.68750     0.00000
                Tau qt 5%       1.68750     1.68750     0.00000
               Tau qt 95%       4.55469     4.55469     0.00000
             Tau qt 97.5%       4.55469     4.55469     0.00000
                   Theta0       0.61172     0.61172     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.92441     0.92441     0.00000
          Theta0 qt 97.5%       0.92441     0.92441     0.00000
                   Theta1      20.68359    20.68359     0.00000
           Theta1 qt 2.5%       7.75693     7.75693     0.00000
             Theta1 qt 5%       7.75693     7.75693     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00230     0.00230     0.00000
      Model (SSD) p-value       0.40000     0.40000     0.00000
         Raggedness index       0.02102     0.02102     0.00000
       Raggedness p-value       0.40000     0.40000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 366 (31/01/17 at 14:44:37))
Total computing time for this run : 0h 0m 0s 122 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

