
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_12"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 186 (31/01/17 at 14:44:12)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_12_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_12_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 40
No. of loci              : 61
No. of usable loci       : 61 loci with less than  5.00 % missing data
No. of polymorphic sites : 61


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.07732
     2       150         2   0.03946
     3       150         2   0.01333
     4       150         2   0.01333
     5       150         2   0.01333
     6       150         2   0.01333
     7       150         2   0.03946
     8       150         2   0.01333
     9       150         2   0.06488
    10       150         2   0.01333
    11       150         2   0.01333
    12       150         2   0.01333
    13       150         2   0.12528
    14       150         2   0.01333
    15       150         2   0.03946
    16       150         2   0.01333
    17       150         2   0.05226
    18       150         2   0.05226
    19       150         2   0.01333
    20       150         2   0.05226
    21       150         2   0.01333
    22       150         2   0.19186
    23       150         2   0.02649
    24       150         2   0.05226
    25       150         2   0.02649
    26       150         2   0.06488
    27       150         2   0.03946
    28       150         2   0.01333
    29       150         2   0.01333
    30       150         2   0.02649
    31       150         2   0.03946
    32       150         2   0.01333
    33       150         2   0.05226
    34       150         2   0.01333
    35       150         2   0.01333
    36       150         2   0.01333
    37       150         2   0.02649
    38       150         2   0.07732
    39       150         2   0.02649
    40       150         2   0.07732
    41       150         2   0.06488
    42       150         2   0.01333
    43       150         2   0.19186
    44       150         2   0.07732
    45       150         2   0.05226
    46       150         2   0.02649
    47       150         2   0.06488
    48       150         2   0.01333
    49       150         2   0.02649
    50       150         2   0.06488
    51       150         2   0.06488
    52       150         2   0.06488
    53       150         2   0.02649
    54       150         2   0.01333
    55       150         2   0.01333
    56       150         2   0.01333
    57       150         2   0.01333
    58       150         2   0.01333
    59       150         2   0.01333
    60       150         2   0.11356
    61       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.04047
  s.d.     0.000     0.000   0.03878
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.2548
Gene diversity           :  0.7502 +/-  0.0386

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 40

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 61
Number of usable nucleotide sites            : 61


Distance method : Pairwise difference
Mean number of pairwise differences           :     2.468725 +/-     1.342020

Average gene diversity over loci              :     0.040471 +/-     0.024352

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :     2.330105
  S.D. Theta(Hom) :     0.511088

         Theta(k) :    17.513851
95 % confidence interval limits for theta(k) : [  11.895992,  25.454453 ]

         Theta(S) :    10.923063
    S.D. Theta(S) :     2.795188

        Theta(Pi) :     2.468725
   S.D. Theta(Pi) :     1.485487

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0    2792    1596.615
       1    1149    2641.320
       2    2173    2456.003
       3    2038    1758.828
       4     866    1112.826
       5    1271     667.407
       6     351     392.358
       7     303     229.202
       8     182     133.661
       9      23      77.913
      10      27      45.413
      11       0      26.469
      12       0      15.428
      13       0       8.992
      14       0       5.241
      15       0       3.055
      16       0       1.781
      17       0       1.038
      18       0       0.605
      19       0       0.353
      20       0       0.205


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        2.469
Mismatch observed variance          :        4.421
Tau                                 :        4.027
Theta0                              :        0.002
Theta1                              :        3.850

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.01299433
          P(Sim. Ssd >= Obs. Ssd): 0.60000000
    Harpending's Raggedness index: 0.04959320
        P(Sim. Rag. >= Obs. Rag.): 0.70000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0       2792.0       2637.8       2359.5
          1.0       1149.0       2094.9       2040.9
          2.0       2173.0       1865.3       1846.6
          3.0       2038.0       1751.1       1620.2
          4.0        866.0        885.4       1290.5
          5.0       1271.0       1014.5        906.3
          6.0        351.0        512.7        558.4
          7.0        303.0        143.1        303.5
          8.0        182.0        179.1        146.8
          9.0         23.0         88.7         63.8
         10.0         27.0          2.4         25.2
         11.0          0.0          0.0          9.1
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.027           0.000           6.621           3.099
                   Theta0:           0.002           0.000           0.005           0.002               
                   Theta1:           3.850           3.690       99999.000       30006.198
Mean No. of pairwise diff.           2.469           1.254           3.410           2.329
           No. pol. sites:              61              10              23          16.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2792.0       1202.0       3917.0       2359.5
          1.0       1149.0        610.0       4889.0       2040.9
          2.0       2173.0        426.0       3508.0       1846.6
          3.0       2038.0        435.0       4774.0       1620.2
          4.0        866.0        140.0       1925.0       1290.5
          5.0       1271.0         16.0       2548.0        906.3
          6.0        351.0          0.0       1853.0        558.4
          7.0        303.0          0.0       1127.0        303.5
          8.0        182.0          0.0       1548.0        146.8
          9.0         23.0          0.0        810.0         63.8
         10.0         27.0          0.0         18.0         25.2
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.027           0.000           6.621           3.099
                   Theta0:           0.002           0.000           0.005           0.002               
                   Theta1:           3.850           3.690       99999.000       30006.198
Mean No. of pairwise diff.           2.469           1.254           3.410           2.329
           No. pol. sites:              61              10              23          16.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2792.0       1202.0       3917.0       2359.5
          1.0       1149.0        610.0       4889.0       2040.9
          2.0       2173.0        426.0       3508.0       1846.6
          3.0       2038.0        435.0       4774.0       1620.2
          4.0        866.0        140.0       1925.0       1290.5
          5.0       1271.0         16.0       2548.0        906.3
          6.0        351.0          0.0       1853.0        558.4
          7.0        303.0          0.0       1127.0        303.5
          8.0        182.0          0.0       1548.0        146.8
          9.0         23.0          0.0        810.0         63.8
         10.0         27.0          0.0         18.0         25.2
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.027           0.000           6.621           3.099
                   Theta0:           0.002           0.000           0.005           0.002               
                   Theta1:           3.850           3.690       99999.000       30006.198
Mean No. of pairwise diff.           2.469           1.254           3.410           2.329
           No. pol. sites:              61              10              23          16.600
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0       2792.0       1202.0       3917.0       2359.5
          1.0       1149.0        610.0       4889.0       2040.9
          2.0       2173.0        426.0       3508.0       1846.6
          3.0       2038.0        435.0       4774.0       1620.2
          4.0        866.0        140.0       1925.0       1290.5
          5.0       1271.0         16.0       2548.0        906.3
          6.0        351.0          0.0       1853.0        558.4
          7.0        303.0          0.0       1127.0        303.5
          8.0        182.0          0.0       1548.0        146.8
          9.0         23.0          0.0        810.0         63.8
         10.0         27.0          0.0         18.0         25.2
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_12_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            61      61.000       0.000
  No. of usable loci            61      61.000       0.000
  No. of polym. loci            61      61.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_12_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.07732     0.07732     0.00000     0.07732
     2       0.03946     0.03946     0.00000     0.03946
     3       0.01333     0.01333     0.00000     0.01333
     4       0.01333     0.01333     0.00000     0.01333
     5       0.01333     0.01333     0.00000     0.01333
     6       0.01333     0.01333     0.00000     0.01333
     7       0.03946     0.03946     0.00000     0.03946
     8       0.01333     0.01333     0.00000     0.01333
     9       0.06488     0.06488     0.00000     0.06488
    10       0.01333     0.01333     0.00000     0.01333
    11       0.01333     0.01333     0.00000     0.01333
    12       0.01333     0.01333     0.00000     0.01333
    13       0.12528     0.12528     0.00000     0.12528
    14       0.01333     0.01333     0.00000     0.01333
    15       0.03946     0.03946     0.00000     0.03946
    16       0.01333     0.01333     0.00000     0.01333
    17       0.05226     0.05226     0.00000     0.05226
    18       0.05226     0.05226     0.00000     0.05226
    19       0.01333     0.01333     0.00000     0.01333
    20       0.05226     0.05226     0.00000     0.05226
    21       0.01333     0.01333     0.00000     0.01333
    22       0.19186     0.19186     0.00000     0.19186
    23       0.02649     0.02649     0.00000     0.02649
    24       0.05226     0.05226     0.00000     0.05226
    25       0.02649     0.02649     0.00000     0.02649
    26       0.06488     0.06488     0.00000     0.06488
    27       0.03946     0.03946     0.00000     0.03946
    28       0.01333     0.01333     0.00000     0.01333
    29       0.01333     0.01333     0.00000     0.01333
    30       0.02649     0.02649     0.00000     0.02649
    31       0.03946     0.03946     0.00000     0.03946
    32       0.01333     0.01333     0.00000     0.01333
    33       0.05226     0.05226     0.00000     0.05226
    34       0.01333     0.01333     0.00000     0.01333
    35       0.01333     0.01333     0.00000     0.01333
    36       0.01333     0.01333     0.00000     0.01333
    37       0.02649     0.02649     0.00000     0.02649
    38       0.07732     0.07732     0.00000     0.07732
    39       0.02649     0.02649     0.00000     0.02649
    40       0.07732     0.07732     0.00000     0.07732
    41       0.06488     0.06488     0.00000     0.06488
    42       0.01333     0.01333     0.00000     0.01333
    43       0.19186     0.19186     0.00000     0.19186
    44       0.07732     0.07732     0.00000     0.07732
    45       0.05226     0.05226     0.00000     0.05226
    46       0.02649     0.02649     0.00000     0.02649
    47       0.06488     0.06488     0.00000     0.06488
    48       0.01333     0.01333     0.00000     0.01333
    49       0.02649     0.02649     0.00000     0.02649
    50       0.06488     0.06488     0.00000     0.06488
    51       0.06488     0.06488     0.00000     0.06488
    52       0.06488     0.06488     0.00000     0.06488
    53       0.02649     0.02649     0.00000     0.02649
    54       0.01333     0.01333     0.00000     0.01333
    55       0.01333     0.01333     0.00000     0.01333
    56       0.01333     0.01333     0.00000     0.01333
    57       0.01333     0.01333     0.00000     0.01333
    58       0.01333     0.01333     0.00000     0.01333
    59       0.01333     0.01333     0.00000     0.01333
    60       0.11356     0.11356     0.00000     0.11356
    61       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.04047     0.04047     0.00000     0.04047
  s.d.       0.03878     0.03878     0.00000     0.03878
--------------------------------------------------------
<A NAME="31_01_17at14_44_12_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_12_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      17.51385    17.51385     0.00000
            Theta_k_lower      11.89599    11.89599     0.00000
            Theta_k_upper      25.45445    25.45445     0.00000
                  Theta_H       2.33010     2.33010     0.00000
             s.d. Theta_H       0.51109     0.51109     0.00000
                  Theta_S      10.92306    10.92306     0.00000
             s.d. Theta_S       2.79519     2.79519     0.00000
                 Theta_pi       2.46872     2.46872     0.00000
            s.d. Theta_pi       1.48549     1.48549     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_12_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.02734     4.02734     0.00000
              Tau qt 2.5%       0.00000     0.00000     0.00000
                Tau qt 5%       0.00000     0.00000     0.00000
               Tau qt 95%       4.74609     4.74609     0.00000
             Tau qt 97.5%       4.74609     4.74609     0.00000
                   Theta0       0.00176     0.00176     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00352     0.00352     0.00000
          Theta0 qt 97.5%       0.00352     0.00352     0.00000
                   Theta1       3.85010     3.85010     0.00000
           Theta1 qt 2.5%       3.69019     3.69019     0.00000
             Theta1 qt 5%       3.69019     3.69019     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.01299     0.01299     0.00000
      Model (SSD) p-value       0.60000     0.60000     0.00000
         Raggedness index       0.04959     0.04959     0.00000
       Raggedness p-value       0.70000     0.70000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 186 (31/01/17 at 14:44:12))
Total computing time for this run : 0h 0m 0s 110 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

