
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_52_18"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 301 (31/01/17 at 14:52:18)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_52_18_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_52_18_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 43
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.01333
     2       150         2   0.02649
     3       150         2   0.50255
     4       150         2   0.03946
     5       150         2   0.01333
     6       150         2   0.38085
     7       150         2   0.02649
     8       150         2   0.07732
     9       150         2   0.50255
    10       150         2   0.25199
    11       150         2   0.05226
    12       150         2   0.49611
    13       150         2   0.33011
    14       150         2   0.01333
    15       150         2   0.01333
    16       150         2   0.31400
    17       150         2   0.17038
    18       150         2   0.39374
    19       150         2   0.06488
    20       150         2   0.02649
-------------------------------------------------
  Mean   150.000     2.000   0.18545
  s.d.     0.000     0.000   0.18930
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0546
Gene diversity           :  0.9518 +/-  0.0070

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 43

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.708993 +/-     1.884538

Average gene diversity over loci              :     0.185450 +/-     0.104300

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    18.102511
  S.D. Theta(Hom) :     2.951313

         Theta(k) :    19.804082
95 % confidence interval limits for theta(k) : [  13.577772,  28.555190 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     3.708993
   S.D. Theta(Pi) :     2.086001

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     539     290.287
       1    1018    1046.041
       2    1713    1895.773
       3    1992    2305.790
       4    1985    2119.345
       5    1750    1571.929
       6    1150     981.294
       7     829     531.118
       8     160     254.879
       9      35     110.399
      10       4      43.805
      11       0      16.127
      12       0       5.572
      13       0       1.825
      14       0       0.572
      15       0       0.173
      16       0       0.051
      17       0       0.015
      18       0       0.004
      19       0       0.001
      20       0       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.709
Mismatch observed variance          :        3.855
Tau                                 :        4.088
Theta0                              :        0.033
Theta1                              :       28.398

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00086848
          P(Sim. Ssd >= Obs. Ssd): 0.70000000
    Harpending's Raggedness index: 0.01419584
        P(Sim. Rag. >= Obs. Rag.): 1.00000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        539.0        520.8        531.7
          1.0       1018.0       1049.3        986.6
          2.0       1713.0       1698.5       1615.1
          3.0       1992.0       2151.7       2040.7
          4.0       1985.0       2078.0       2024.7
          5.0       1750.0       1647.9       1635.8
          6.0       1150.0       1040.5       1110.3
          7.0        829.0        582.9        648.7
          8.0        160.0        263.0        332.5
          9.0         35.0         98.6        151.7
         10.0          4.0         31.2         62.3
         11.0          0.0         11.4         23.3
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.088           2.525           5.209           3.857
                   Theta0:           0.033           0.000           0.012           0.002               
                   Theta1:          28.398          13.555       99999.000       20153.370
Mean No. of pairwise diff.           3.709           2.779           4.741           3.689
           No. pol. sites:              20              57              75          65.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        539.0        299.0       1115.0        531.7
          1.0       1018.0        377.0       2624.0        986.6
          2.0       1713.0        794.0       2489.0       1615.1
          3.0       1992.0       1293.0       3076.0       2040.7
          4.0       1985.0       1103.0       2600.0       2024.7
          5.0       1750.0        811.0       2379.0       1635.8
          6.0       1150.0        518.0       2129.0       1110.3
          7.0        829.0        233.0       1358.0        648.7
          8.0        160.0         41.0        678.0        332.5
          9.0         35.0          3.0        391.0        151.7
         10.0          4.0          0.0        125.0         62.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.088           2.525           5.209           3.857
                   Theta0:           0.033           0.000           0.012           0.002               
                   Theta1:          28.398          13.555       99999.000       20153.370
Mean No. of pairwise diff.           3.709           2.779           4.741           3.689
           No. pol. sites:              20              57              75          65.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        539.0        299.0       1115.0        531.7
          1.0       1018.0        377.0       2624.0        986.6
          2.0       1713.0        794.0       2489.0       1615.1
          3.0       1992.0       1293.0       3076.0       2040.7
          4.0       1985.0       1103.0       2600.0       2024.7
          5.0       1750.0        811.0       2379.0       1635.8
          6.0       1150.0        518.0       2129.0       1110.3
          7.0        829.0        233.0       1358.0        648.7
          8.0        160.0         41.0        678.0        332.5
          9.0         35.0          3.0        391.0        151.7
         10.0          4.0          0.0        125.0         62.3
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           4.088           2.525           5.209           3.857
                   Theta0:           0.033           0.000           0.012           0.002               
                   Theta1:          28.398          13.555       99999.000       20153.370
Mean No. of pairwise diff.           3.709           2.779           4.741           3.689
           No. pol. sites:              20              57              75          65.400
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        539.0        299.0       1115.0        531.7
          1.0       1018.0        377.0       2624.0        986.6
          2.0       1713.0        794.0       2489.0       1615.1
          3.0       1992.0       1293.0       3076.0       2040.7
          4.0       1985.0       1103.0       2600.0       2024.7
          5.0       1750.0        811.0       2379.0       1635.8
          6.0       1150.0        518.0       2129.0       1110.3
          7.0        829.0        233.0       1358.0        648.7
          8.0        160.0         41.0        678.0        332.5
          9.0         35.0          3.0        391.0        151.7
         10.0          4.0          0.0        125.0         62.3
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_52_18_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_52_18_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.01333     0.01333     0.00000     0.01333
     2       0.02649     0.02649     0.00000     0.02649
     3       0.50255     0.50255     0.00000     0.50255
     4       0.03946     0.03946     0.00000     0.03946
     5       0.01333     0.01333     0.00000     0.01333
     6       0.38085     0.38085     0.00000     0.38085
     7       0.02649     0.02649     0.00000     0.02649
     8       0.07732     0.07732     0.00000     0.07732
     9       0.50255     0.50255     0.00000     0.50255
    10       0.25199     0.25199     0.00000     0.25199
    11       0.05226     0.05226     0.00000     0.05226
    12       0.49611     0.49611     0.00000     0.49611
    13       0.33011     0.33011     0.00000     0.33011
    14       0.01333     0.01333     0.00000     0.01333
    15       0.01333     0.01333     0.00000     0.01333
    16       0.31400     0.31400     0.00000     0.31400
    17       0.17038     0.17038     0.00000     0.17038
    18       0.39374     0.39374     0.00000     0.39374
    19       0.06488     0.06488     0.00000     0.06488
    20       0.02649     0.02649     0.00000     0.02649
--------------------------------------------------------
  Mean       0.18545     0.18545     0.00000     0.18545
  s.d.       0.18930     0.18930     0.00000     0.18930
--------------------------------------------------------
<A NAME="31_01_17at14_52_18_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_52_18_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      19.80408    19.80408     0.00000
            Theta_k_lower      13.57777    13.57777     0.00000
            Theta_k_upper      28.55519    28.55519     0.00000
                  Theta_H      18.10251    18.10251     0.00000
             s.d. Theta_H       2.95131     2.95131     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       3.70899     3.70899     0.00000
            s.d. Theta_pi       2.08600     2.08600     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_52_18_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       4.08789     4.08789     0.00000
              Tau qt 2.5%       2.52539     2.52539     0.00000
                Tau qt 5%       2.52539     2.52539     0.00000
               Tau qt 95%       4.94922     4.94922     0.00000
             Tau qt 97.5%       4.94922     4.94922     0.00000
                   Theta0       0.03320     0.03320     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00352     0.00352     0.00000
          Theta0 qt 97.5%       0.00352     0.00352     0.00000
                   Theta1      28.39844    28.39844     0.00000
           Theta1 qt 2.5%      13.55469    13.55469     0.00000
             Theta1 qt 5%      13.55469    13.55469     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00087     0.00087     0.00000
      Model (SSD) p-value       0.70000     0.70000     0.00000
         Raggedness index       0.01420     0.01420     0.00000
       Raggedness p-value       1.00000     1.00000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 301 (31/01/17 at 14:52:18))
Total computing time for this run : 0h 0m 0s 122 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

