
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_44_27"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 298 (31/01/17 at 14:44:27)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_44_27_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_44_27_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 52
No. of loci              : 70
No. of usable loci       : 70 loci with less than  5.00 % missing data
No. of polymorphic sites : 70


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.05226
     2       150         2   0.08957
     3       150         2   0.07732
     4       150         2   0.01333
     5       150         2   0.03946
     6       150         2   0.01333
     7       150         2   0.01333
     8       150         2   0.03946
     9       150         2   0.01333
    10       150         2   0.01333
    11       150         2   0.01333
    12       150         2   0.01333
    13       150         2   0.01333
    14       150         2   0.03946
    15       150         2   0.01333
    16       150         2   0.02649
    17       150         2   0.02649
    18       150         2   0.07732
    19       150         2   0.15937
    20       150         2   0.01333
    21       150         2   0.01333
    22       150         2   0.02649
    23       150         2   0.17038
    24       150         2   0.01333
    25       150         2   0.15937
    26       150         2   0.01333
    27       150         2   0.03946
    28       150         2   0.02649
    29       150         2   0.01333
    30       150         2   0.01333
    31       150         2   0.11356
    32       150         2   0.05226
    33       150         2   0.01333
    34       150         2   0.03946
    35       150         2   0.01333
    36       150         2   0.01333
    37       150         2   0.02649
    38       150         2   0.03946
    39       150         2   0.06488
    40       150         2   0.01333
    41       150         2   0.03946
    42       150         2   0.03946
    43       150         2   0.05226
    44       150         2   0.03946
    45       150         2   0.01333
    46       150         2   0.10166
    47       150         2   0.07732
    48       150         2   0.03946
    49       150         2   0.03946
    50       150         2   0.14819
    51       150         2   0.01333
    52       150         2   0.02649
    53       150         2   0.15937
    54       150         2   0.08957
    55       150         2   0.08957
    56       150         2   0.11356
    57       150         2   0.01333
    58       150         2   0.08957
    59       150         2   0.01333
    60       150         2   0.01333
    61       150         2   0.14819
    62       150         2   0.10166
    63       150         2   0.11356
    64       150         2   0.01333
    65       150         2   0.05226
    66       150         2   0.03946
    67       150         2   0.01333
    68       150         2   0.03946
    69       150         2   0.08957
    70       150         2   0.01333
-------------------------------------------------
  Mean   150.000     2.000   0.04964
  s.d.     0.000     0.000   0.04480
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0414
Gene diversity           :  0.9650 +/-  0.0066

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 52

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 70
Number of usable nucleotide sites            : 70


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.474541 +/-     1.782480

Average gene diversity over loci              :     0.049636 +/-     0.028186

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    25.852854
  S.D. Theta(Hom) :     5.325161

         Theta(k) :    27.789032
95 % confidence interval limits for theta(k) : [  19.465082,  39.354984 ]

         Theta(S) :    12.534663
    S.D. Theta(S) :     3.158166

        Theta(Pi) :     3.474541
   S.D. Theta(Pi) :     1.973033

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     391       0.000
       1    1038       0.000
       2    1854       0.000
       3    2506       0.000
       4    2311       0.000
       5    1680       0.000
       6     929       0.000
       7     387       0.000
       8      74       0.000
       9       5       0.000


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.475
Mismatch observed variance          :        2.920
Tau                                 :        3.639
Theta0                              :        0.002
Theta1                              :    99999.000

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00098409
          P(Sim. Ssd >= Obs. Ssd): 0.40000000
    Harpending's Raggedness index: 0.02327243
        P(Sim. Rag. >= Obs. Rag.): 0.40000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        391.0        253.6        293.3
          1.0       1038.0       1024.1       1067.6
          2.0       1854.0       1911.3       1943.1
          3.0       2506.0       2328.1       2357.9
          4.0       2311.0       2141.9       2145.9
          5.0       1680.0       1606.8       1562.4
          6.0        929.0        985.7        948.0
          7.0        387.0        528.2        493.0
          8.0         74.0        243.6        224.3
          9.0          5.0         98.0         90.7
         10.0          0.0         38.4         33.0
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.639           3.174           3.916           3.519
                   Theta0:           0.002           0.000           0.536           0.062               
                   Theta1:       99999.000          48.922       99999.000       70026.461
Mean No. of pairwise diff.           3.475           3.340           4.255           3.708
           No. pol. sites:              70             182             219         201.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        391.0        153.0        496.0        293.3
          1.0       1038.0        700.0       1379.0       1067.6
          2.0       1854.0       1592.0       2172.0       1943.1
          3.0       2506.0       2079.0       2653.0       2357.9
          4.0       2311.0       1959.0       2452.0       2145.9
          5.0       1680.0       1414.0       1885.0       1562.4
          6.0        929.0        808.0       1199.0        948.0
          7.0        387.0        393.0        830.0        493.0
          8.0         74.0        130.0        497.0        224.3
          9.0          5.0         35.0        266.0         90.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.639           3.174           3.916           3.519
                   Theta0:           0.002           0.000           0.536           0.062               
                   Theta1:       99999.000          48.922       99999.000       70026.461
Mean No. of pairwise diff.           3.475           3.340           4.255           3.708
           No. pol. sites:              70             182             219         201.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        391.0        153.0        496.0        293.3
          1.0       1038.0        700.0       1379.0       1067.6
          2.0       1854.0       1592.0       2172.0       1943.1
          3.0       2506.0       2079.0       2653.0       2357.9
          4.0       2311.0       1959.0       2452.0       2145.9
          5.0       1680.0       1414.0       1885.0       1562.4
          6.0        929.0        808.0       1199.0        948.0
          7.0        387.0        393.0        830.0        493.0
          8.0         74.0        130.0        497.0        224.3
          9.0          5.0         35.0        266.0         90.7
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.639           3.174           3.916           3.519
                   Theta0:           0.002           0.000           0.536           0.062               
                   Theta1:       99999.000          48.922       99999.000       70026.461
Mean No. of pairwise diff.           3.475           3.340           4.255           3.708
           No. pol. sites:              70             182             219         201.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        391.0        153.0        496.0        293.3
          1.0       1038.0        700.0       1379.0       1067.6
          2.0       1854.0       1592.0       2172.0       1943.1
          3.0       2506.0       2079.0       2653.0       2357.9
          4.0       2311.0       1959.0       2452.0       2145.9
          5.0       1680.0       1414.0       1885.0       1562.4
          6.0        929.0        808.0       1199.0        948.0
          7.0        387.0        393.0        830.0        493.0
          8.0         74.0        130.0        497.0        224.3
          9.0          5.0         35.0        266.0         90.7
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_44_27_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            70      70.000       0.000
  No. of usable loci            70      70.000       0.000
  No. of polym. loci            70      70.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_44_27_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.05226     0.05226     0.00000     0.05226
     2       0.08957     0.08957     0.00000     0.08957
     3       0.07732     0.07732     0.00000     0.07732
     4       0.01333     0.01333     0.00000     0.01333
     5       0.03946     0.03946     0.00000     0.03946
     6       0.01333     0.01333     0.00000     0.01333
     7       0.01333     0.01333     0.00000     0.01333
     8       0.03946     0.03946     0.00000     0.03946
     9       0.01333     0.01333     0.00000     0.01333
    10       0.01333     0.01333     0.00000     0.01333
    11       0.01333     0.01333     0.00000     0.01333
    12       0.01333     0.01333     0.00000     0.01333
    13       0.01333     0.01333     0.00000     0.01333
    14       0.03946     0.03946     0.00000     0.03946
    15       0.01333     0.01333     0.00000     0.01333
    16       0.02649     0.02649     0.00000     0.02649
    17       0.02649     0.02649     0.00000     0.02649
    18       0.07732     0.07732     0.00000     0.07732
    19       0.15937     0.15937     0.00000     0.15937
    20       0.01333     0.01333     0.00000     0.01333
    21       0.01333     0.01333     0.00000     0.01333
    22       0.02649     0.02649     0.00000     0.02649
    23       0.17038     0.17038     0.00000     0.17038
    24       0.01333     0.01333     0.00000     0.01333
    25       0.15937     0.15937     0.00000     0.15937
    26       0.01333     0.01333     0.00000     0.01333
    27       0.03946     0.03946     0.00000     0.03946
    28       0.02649     0.02649     0.00000     0.02649
    29       0.01333     0.01333     0.00000     0.01333
    30       0.01333     0.01333     0.00000     0.01333
    31       0.11356     0.11356     0.00000     0.11356
    32       0.05226     0.05226     0.00000     0.05226
    33       0.01333     0.01333     0.00000     0.01333
    34       0.03946     0.03946     0.00000     0.03946
    35       0.01333     0.01333     0.00000     0.01333
    36       0.01333     0.01333     0.00000     0.01333
    37       0.02649     0.02649     0.00000     0.02649
    38       0.03946     0.03946     0.00000     0.03946
    39       0.06488     0.06488     0.00000     0.06488
    40       0.01333     0.01333     0.00000     0.01333
    41       0.03946     0.03946     0.00000     0.03946
    42       0.03946     0.03946     0.00000     0.03946
    43       0.05226     0.05226     0.00000     0.05226
    44       0.03946     0.03946     0.00000     0.03946
    45       0.01333     0.01333     0.00000     0.01333
    46       0.10166     0.10166     0.00000     0.10166
    47       0.07732     0.07732     0.00000     0.07732
    48       0.03946     0.03946     0.00000     0.03946
    49       0.03946     0.03946     0.00000     0.03946
    50       0.14819     0.14819     0.00000     0.14819
    51       0.01333     0.01333     0.00000     0.01333
    52       0.02649     0.02649     0.00000     0.02649
    53       0.15937     0.15937     0.00000     0.15937
    54       0.08957     0.08957     0.00000     0.08957
    55       0.08957     0.08957     0.00000     0.08957
    56       0.11356     0.11356     0.00000     0.11356
    57       0.01333     0.01333     0.00000     0.01333
    58       0.08957     0.08957     0.00000     0.08957
    59       0.01333     0.01333     0.00000     0.01333
    60       0.01333     0.01333     0.00000     0.01333
    61       0.14819     0.14819     0.00000     0.14819
    62       0.10166     0.10166     0.00000     0.10166
    63       0.11356     0.11356     0.00000     0.11356
    64       0.01333     0.01333     0.00000     0.01333
    65       0.05226     0.05226     0.00000     0.05226
    66       0.03946     0.03946     0.00000     0.03946
    67       0.01333     0.01333     0.00000     0.01333
    68       0.03946     0.03946     0.00000     0.03946
    69       0.08957     0.08957     0.00000     0.08957
    70       0.01333     0.01333     0.00000     0.01333
--------------------------------------------------------
  Mean       0.04964     0.04964     0.00000     0.04964
  s.d.       0.04480     0.04480     0.00000     0.04480
--------------------------------------------------------
<A NAME="31_01_17at14_44_27_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
    21             2       2.000       0.000           2
    22             2       2.000       0.000           2
    23             2       2.000       0.000           2
    24             2       2.000       0.000           2
    25             2       2.000       0.000           2
    26             2       2.000       0.000           2
    27             2       2.000       0.000           2
    28             2       2.000       0.000           2
    29             2       2.000       0.000           2
    30             2       2.000       0.000           2
    31             2       2.000       0.000           2
    32             2       2.000       0.000           2
    33             2       2.000       0.000           2
    34             2       2.000       0.000           2
    35             2       2.000       0.000           2
    36             2       2.000       0.000           2
    37             2       2.000       0.000           2
    38             2       2.000       0.000           2
    39             2       2.000       0.000           2
    40             2       2.000       0.000           2
    41             2       2.000       0.000           2
    42             2       2.000       0.000           2
    43             2       2.000       0.000           2
    44             2       2.000       0.000           2
    45             2       2.000       0.000           2
    46             2       2.000       0.000           2
    47             2       2.000       0.000           2
    48             2       2.000       0.000           2
    49             2       2.000       0.000           2
    50             2       2.000       0.000           2
    51             2       2.000       0.000           2
    52             2       2.000       0.000           2
    53             2       2.000       0.000           2
    54             2       2.000       0.000           2
    55             2       2.000       0.000           2
    56             2       2.000       0.000           2
    57             2       2.000       0.000           2
    58             2       2.000       0.000           2
    59             2       2.000       0.000           2
    60             2       2.000       0.000           2
    61             2       2.000       0.000           2
    62             2       2.000       0.000           2
    63             2       2.000       0.000           2
    64             2       2.000       0.000           2
    65             2       2.000       0.000           2
    66             2       2.000       0.000           2
    67             2       2.000       0.000           2
    68             2       2.000       0.000           2
    69             2       2.000       0.000           2
    70             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_44_27_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      27.78903    27.78903     0.00000
            Theta_k_lower      19.46508    19.46508     0.00000
            Theta_k_upper      39.35498    39.35498     0.00000
                  Theta_H      25.85285    25.85285     0.00000
             s.d. Theta_H       5.32516     5.32516     0.00000
                  Theta_S      12.53466    12.53466     0.00000
             s.d. Theta_S       3.15817     3.15817     0.00000
                 Theta_pi       3.47454     3.47454     0.00000
            s.d. Theta_pi       1.97303     1.97303     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_44_27_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.63867     3.63867     0.00000
              Tau qt 2.5%       3.17383     3.17383     0.00000
                Tau qt 5%       3.17383     3.17383     0.00000
               Tau qt 95%       3.91211     3.91211     0.00000
             Tau qt 97.5%       3.91211     3.91211     0.00000
                   Theta0       0.00176     0.00176     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.05273     0.05273     0.00000
          Theta0 qt 97.5%       0.05273     0.05273     0.00000
                   Theta1   99999.00000 99999.00000     0.00000
           Theta1 qt 2.5%      48.92188    48.92188     0.00000
             Theta1 qt 5%      48.92188    48.92188     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00098     0.00098     0.00000
      Model (SSD) p-value       0.40000     0.40000     0.00000
         Raggedness index       0.02327     0.02327     0.00000
       Raggedness p-value       0.40000     0.40000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 298 (31/01/17 at 14:44:28))
Total computing time for this run : 0h 0m 0s 139 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

