
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>

</data>

<header>
Arlequin results
</header>
<data>

<A NAME="31_01_17at14_47_32"></A>

////////////////////////////////////////////////////////////////////
RUN NUMBER 149 (31/01/17 at 14:47:32)
////////////////////////////////////////////////////////////////////

<A NAME="31_01_17at14_47_32_run_information"></A>

Warning: The locus separator has been removed
-------


Project information:
--------------------
	NbSamples      =  1
	DataType       =  RFLP
	GenotypicData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise difference
        GammaA Value       = 0
        Theta estimators   : 
            Theta(Hom)
            Theta(S)
            Theta(k)
            Theta(Pi)

    Mismatch Distribution:
    ----------------------
        Molecular Distance             : Pairwise difference
        GammaA Value                   = 0
        Number of bootstrap replicates : 10


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="31_01_17at14_47_32_group0"></A>

===============================================================================
== Sample : 	Tus muestras
===============================================================================


===============================
== Standard diversity indices : (Tus muestras)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 150
No. haplotypes           : 43
No. of loci              : 20
No. of usable loci       : 20 loci with less than  5.00 % missing data
No. of polymorphic sites : 20


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Exp.
Locus#    copies   alleles       Het
-------------------------------------------------
     1       150         2   0.02649
     2       150         2   0.21262
     3       150         2   0.10166
     4       150         2   0.01333
     5       150         2   0.30568
     6       150         2   0.17038
     7       150         2   0.01333
     8       150         2   0.29718
     9       150         2   0.37414
    10       150         2   0.06488
    11       150         2   0.44743
    12       150         2   0.02649
    13       150         2   0.32215
    14       150         2   0.27964
    15       150         2   0.05226
    16       150         2   0.01333
    17       150         2   0.05226
    18       150         2   0.27964
    19       150         2   0.25199
    20       150         2   0.11356
-------------------------------------------------
  Mean   150.000     2.000   0.17092
  s.d.     0.000     0.000   0.13962
-------------------------------------------------


Haplotype-level computations
----------------------------

Sum of square freqs.     :  0.0653
Gene diversity           :  0.9409 +/-  0.0107

(Standard deviation is for the sampling process)

================================
== Molecular diversity indices : (Tus muestras)
================================

           <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 150.0000
No. of haplotypes                            : 43

Allowed level of missing data                : 5.0000 %
Number of observed indels                    : 0
Number of polymorphic sites                  : 20
Number of usable nucleotide sites            : 20


Distance method : Pairwise difference
Mean number of pairwise differences           :     3.418434 +/-     1.758031

Average gene diversity over loci              :     0.170922 +/-     0.097298

(Standard deviations are for both sampling and stochastic processes)

       Theta(Hom) :    14.378114
  S.D. Theta(Hom) :     2.994291

         Theta(k) :    19.804082
95 % confidence interval limits for theta(k) : [  13.577772,  28.555190 ]

         Theta(S) :     3.581332
    S.D. Theta(S) :     1.117771

        Theta(Pi) :     3.418434
   S.D. Theta(Pi) :     1.945970

==========================
== Mismatch distribution : (Tus muestras)
==========================

           <i>Li, W. H., 1977.</i>
           <i>Rogers, A., 1995.</i>
           <i>Harpending, R. C., 1994.</i>
           <i>Schneider, S., and L. Excoffier. 1999.</i>
           <i>Excoffier, L., 2004.</i>
Distance method : Pairwise difference

  #Diff.   Count  Exp. Count
  ------   -----  ----------
       0     660     406.716
       1    1375    1315.403
       2    1659    2151.612
       3    2166    2378.003
       4    2074    2002.659
       5    1634    1374.806
       6    1009     804.181
       7     385     413.952
       8     137     192.314
       9      61      82.340
      10      15      33.075
      11       0      12.658
      12       0       4.676
      13       0       1.685
      14       0       0.598
      15       0       0.210
      16       0       0.073
      17       0       0.025
      18       0       0.009
      19       0       0.003
      20       0       0.001


Estimated parameters from the sudden expansion model:
-----------------------------------------------------

Mismatch observed mean              :        3.418
Mismatch observed variance          :        3.701
Tau                                 :        3.840
Theta0                              :        0.014
Theta1                              :       22.305

Test of goodness-of-fit:
------------------------

         Sum of Squared deviation: 0.00099898
          P(Sim. Ssd >= Obs. Ssd): 0.80000000
    Harpending's Raggedness index: 0.01521946
        P(Sim. Rag. >= Obs. Rag.): 0.90000000

Mismatch distributions:
---------------------
-----------------------------------------------------
        #Diff     Observed    Simulated   Model Freq.
-----------------------------------------------------
          0.0        660.0        562.4        670.3
          1.0       1375.0       1035.0       1186.0
          2.0       1659.0       1891.5       1816.5
          3.0       2166.0       2286.6       2141.6
          4.0       2074.0       2323.3       1983.2
          5.0       1634.0       1582.5       1496.1
          6.0       1009.0        893.8        948.6
          7.0        385.0        394.9        517.8
          8.0        137.0        145.4        248.0
          9.0         61.0         42.0        105.7
         10.0         15.0         11.5         40.6
         11.0          0.0          4.9         14.2
-----------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.010
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.840           2.865           4.949           3.737
                   Theta0:           0.014           0.000           0.025           0.004               
                   Theta1:          22.305          17.617       99999.000       40056.131
Mean No. of pairwise diff.           3.418           2.766           4.457           3.466
           No. pol. sites:              20              43              70          55.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        660.0        372.0        826.0        670.3
          1.0       1375.0        235.0       1769.0       1186.0
          2.0       1659.0        857.0       3119.0       1816.5
          3.0       2166.0       1511.0       2880.0       2141.6
          4.0       2074.0       1826.0       2866.0       1983.2
          5.0       1634.0        969.0       2306.0       1496.1
          6.0       1009.0        359.0       1855.0        948.6
          7.0        385.0         89.0       1108.0        517.8
          8.0        137.0          6.0        468.0        248.0
          9.0         61.0          0.0        178.0        105.7
         10.0         15.0          0.0         36.0         40.6
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.050
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.840           2.865           4.949           3.737
                   Theta0:           0.014           0.000           0.025           0.004               
                   Theta1:          22.305          17.617       99999.000       40056.131
Mean No. of pairwise diff.           3.418           2.766           4.457           3.466
           No. pol. sites:              20              43              70          55.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        660.0        372.0        826.0        670.3
          1.0       1375.0        235.0       1769.0       1186.0
          2.0       1659.0        857.0       3119.0       1816.5
          3.0       2166.0       1511.0       2880.0       2141.6
          4.0       2074.0       1826.0       2866.0       1983.2
          5.0       1634.0        969.0       2306.0       1496.1
          6.0       1009.0        359.0       1855.0        948.6
          7.0        385.0         89.0       1108.0        517.8
          8.0        137.0          6.0        468.0        248.0
          9.0         61.0          0.0        178.0        105.7
         10.0         15.0          0.0         36.0         40.6
-----------------------------------------------------------------


Confidence intervals (Percentile method):	 alpha=	0.100
Based on 10 replicates

-------------------------------------------------------------------------------------------
                  Variable:       Est. val:     Low. bound:      Up. bound:     Mean Value:
-------------------------------------------------------------------------------------------
                      Tau:           3.840           2.865           4.949           3.737
                   Theta0:           0.014           0.000           0.025           0.004               
                   Theta1:          22.305          17.617       99999.000       40056.131
Mean No. of pairwise diff.           3.418           2.766           4.457           3.466
           No. pol. sites:              20              43              70          55.900
-------------------------------------------------------------------------------------------

Confidence region for the observed mismatch:

-----------------------------------------------------------------
        #Diff     Observed    Low. bound  Up. bound   Model Freq.
-----------------------------------------------------------------
          0.0        660.0        372.0        826.0        670.3
          1.0       1375.0        235.0       1769.0       1186.0
          2.0       1659.0        857.0       3119.0       1816.5
          3.0       2166.0       1511.0       2880.0       2141.6
          4.0       2074.0       1826.0       2866.0       1983.2
          5.0       1634.0        969.0       2306.0       1496.1
          6.0       1009.0        359.0       1855.0        948.6
          7.0        385.0         89.0       1108.0        517.8
          8.0        137.0          6.0        468.0        248.0
          9.0         61.0          0.0        178.0        105.7
         10.0         15.0          0.0         36.0         40.6
-----------------------------------------------------------------





================================================================================
== Summary of computations done within populations
================================================================================
<A NAME="31_01_17at14_47_32_comp_sum_Basic"></A>

----------------
Basic properties
----------------------------------------------------------
          Statistics  Tus muestras        Mean        s.d.
----------------------------------------------------------
  No. of gene copies           150     150.000       0.000
         No. of loci            20      20.000       0.000
  No. of usable loci            20      20.000       0.000
  No. of polym. loci            20      20.000       0.000
----------------------------------------------------------
<A NAME="31_01_17at14_47_32_comp_sum_het"></A>

-----------------------
Expected heterozygosity
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d.   Tot. Het.
--------------------------------------------------------
     1       0.02649     0.02649     0.00000     0.02649
     2       0.21262     0.21262     0.00000     0.21262
     3       0.10166     0.10166     0.00000     0.10166
     4       0.01333     0.01333     0.00000     0.01333
     5       0.30568     0.30568     0.00000     0.30568
     6       0.17038     0.17038     0.00000     0.17038
     7       0.01333     0.01333     0.00000     0.01333
     8       0.29718     0.29718     0.00000     0.29718
     9       0.37414     0.37414     0.00000     0.37414
    10       0.06488     0.06488     0.00000     0.06488
    11       0.44743     0.44743     0.00000     0.44743
    12       0.02649     0.02649     0.00000     0.02649
    13       0.32215     0.32215     0.00000     0.32215
    14       0.27964     0.27964     0.00000     0.27964
    15       0.05226     0.05226     0.00000     0.05226
    16       0.01333     0.01333     0.00000     0.01333
    17       0.05226     0.05226     0.00000     0.05226
    18       0.27964     0.27964     0.00000     0.27964
    19       0.25199     0.25199     0.00000     0.25199
    20       0.11356     0.11356     0.00000     0.11356
--------------------------------------------------------
  Mean       0.17092     0.17092     0.00000     0.17092
  s.d.       0.13962     0.13962     0.00000     0.13962
--------------------------------------------------------
<A NAME="31_01_17at14_47_32_comp_sum_numAll"></A>

-----------------
Number of alleles
--------------------------------------------------------
Locus#  Tus muestras        Mean        s.d. Tot. number
--------------------------------------------------------
     1             2       2.000       0.000           2
     2             2       2.000       0.000           2
     3             2       2.000       0.000           2
     4             2       2.000       0.000           2
     5             2       2.000       0.000           2
     6             2       2.000       0.000           2
     7             2       2.000       0.000           2
     8             2       2.000       0.000           2
     9             2       2.000       0.000           2
    10             2       2.000       0.000           2
    11             2       2.000       0.000           2
    12             2       2.000       0.000           2
    13             2       2.000       0.000           2
    14             2       2.000       0.000           2
    15             2       2.000       0.000           2
    16             2       2.000       0.000           2
    17             2       2.000       0.000           2
    18             2       2.000       0.000           2
    19             2       2.000       0.000           2
    20             2       2.000       0.000           2
--------------------------------------------------------
  Mean         2.000       2.000       0.000       2.000
  s.d.         0.000       0.000       0.000       0.000
--------------------------------------------------------
<A NAME="31_01_17at14_47_32_comp_sum_moldiv"></A>

---------------------------
Molecular diversity indexes
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------
                  Theta_k      19.80408    19.80408     0.00000
            Theta_k_lower      13.57777    13.57777     0.00000
            Theta_k_upper      28.55519    28.55519     0.00000
                  Theta_H      14.37811    14.37811     0.00000
             s.d. Theta_H       2.99429     2.99429     0.00000
                  Theta_S       3.58133     3.58133     0.00000
             s.d. Theta_S       1.11777     1.11777     0.00000
                 Theta_pi       3.41843     3.41843     0.00000
            s.d. Theta_pi       1.94597     1.94597     0.00000
---------------------------------------------------------------
<A NAME="31_01_17at14_47_32_comp_sum_mismatch"></A>

-----------------
Mismatch analysis
---------------------------------------------------------------
               Statistics  Tus muestras        Mean        s.d.
---------------------------------------------------------------

Demographic expansion
---------------------
                      Tau       3.83984     3.83984     0.00000
              Tau qt 2.5%       2.86523     2.86523     0.00000
                Tau qt 5%       2.86523     2.86523     0.00000
               Tau qt 95%       4.58008     4.58008     0.00000
             Tau qt 97.5%       4.58008     4.58008     0.00000
                   Theta0       0.01387     0.01387     0.00000
           Theta0 qt 2.5%       0.00000     0.00000     0.00000
             Theta0 qt 5%       0.00000     0.00000     0.00000
            Theta0 qt 95%       0.00703     0.00703     0.00000
          Theta0 qt 97.5%       0.00703     0.00703     0.00000
                   Theta1      22.30469    22.30469     0.00000
           Theta1 qt 2.5%      17.61719    17.61719     0.00000
             Theta1 qt 5%      17.61719    17.61719     0.00000
            Theta1 qt 95%   99999.00000 99999.00000     0.00000
          Theta1 qt 97.5%   99999.00000 99999.00000     0.00000
                      SSD       0.00100     0.00100     0.00000
      Model (SSD) p-value       0.80000     0.80000     0.00000
         Raggedness index       0.01522     0.01522     0.00000
       Raggedness p-value       0.90000     0.90000     0.00000
---------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 149 (31/01/17 at 14:47:32))
Total computing time for this run : 0h 0m 0s 108 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

